-- phpMyAdmin SQL Dump
-- version 5.2.1
-- https://www.phpmyadmin.net/
--
-- Host: 127.0.0.1
-- Generation Time: Sep 29, 2025 at 08:48 AM
-- Server version: 10.4.32-MariaDB
-- PHP Version: 8.2.12

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
START TRANSACTION;
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `gene_db`
--

-- --------------------------------------------------------

--
-- Table structure for table `final_gene_info`
--

CREATE TABLE `final_gene_info` (
  `id` int(11) UNSIGNED NOT NULL,
  `Gene` varchar(15) DEFAULT NULL,
  `Summary` varchar(2457) DEFAULT NULL,
  `Chromosome` varchar(4) DEFAULT NULL,
  `Gene type` varchar(34) DEFAULT NULL,
  `Gene start (bp)` int(9) DEFAULT NULL,
  `Gene end (bp)` int(9) DEFAULT NULL,
  `Gene % GC content` decimal(4,2) DEFAULT NULL,
  `Transcript count` int(2) DEFAULT NULL,
  `Ensembl_ID` varchar(15) DEFAULT NULL,
  `UniProt_ID` varchar(10) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;

--
-- Dumping data for table `final_gene_info`
--

INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1, 'A2ML1', 'This gene encodes a member of the alpha-macroglobulin superfamily. The encoded protein is thought to be an N-glycosylated monomeric protein that acts as an inhibitor of several proteases. It has been shown to form covalent interactions with proteases, and has been reported as the p170 antigen recognized by autoantibodies in the autoimmune disease paraneoplastic pemphigus (PNP; PMID:20805888). Mutations in these gene have also been associated with some cases of Noonan syndrome (NS; PMID:24939586) as well as some cases of otitis media (PMID:26121085). Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Aug 2015]', '12', 'protein_coding', 10191, 24554, 53.68, 4, 'ENSG00000166535', 'A8K2U0'),
(2, 'AARSD1', 'Predicted to enable Ser-tRNA(Ala) deacylase activity. Predicted to be involved in regulation of translational fidelity. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37312, 40291, 47.62, 6, 'ENSG00000266967', 'Q9BTE6'),
(3, 'ABCA12', 'The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intracellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, and White). This encoded protein is a member of the ABC1 subfamily, which is the only major ABC subfamily found exclusively in multicellular eukaryotes. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 90059, 93677, 69.14, 8, 'ENSG00000144452', 'Q86UK0'),
(4, 'ABCA13', 'In human, the ATP-binding cassette (ABC) family of transmembrane transporters has at least 48 genes and 7 gene subfamilies. This gene is a member of ABC gene subfamily A (ABCA). Genes within the ABCA family typically encode several thousand amino acids. Like other ABC transmembrane transporter proteins, this protein has 12 or more transmembrane alpha-helix domains that likely arrange to form a single central chamber with multiple substrate binding sites. It is also predicted to have two large extracellular domains and two nucleotide binding domains as is typical for ABCA proteins. Alternative splice variants have been described but their biological validity has not been demonstrated.[provided by RefSeq, Mar 2009]', '7', 'protein_coding', 19264364, 19273391, 55.28, 5, 'ENSG00000179869', 'Q86UQ4'),
(5, 'ABCA5', 'The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intracellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, and White). This encoded protein is a member of the ABC1 subfamily. Members of the ABC1 subfamily comprise the only major ABC subfamily found exclusively in multicellular eukaryotes. This gene is clustered among 4 other ABC1 family members on 17q24, but neither the substrate nor the function of this gene is known. Alternative splicing of this gene results in several transcript variants; however, not all variants have been fully described. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 20256558, 20305960, 39.83, 7, 'ENSG00000154265', 'Q8WWZ7'),
(6, 'ABCD3', 'This gene belongs to the CX3C subgroup of chemokines, characterized by the number of amino acids located between the conserved cysteine residues. This is the only member of the CX3C subgroup, which contains three amino acids between cysteine residues, resulting in a Cys-X-X-X-Cys configuration. The encoded protein contains an extended mucin-like stalk with a chemokine domain on top, and exists in both a membrane-anchored form where it acts as a binding molecule, or, in soluble form, as a chemotactic cytokine. The mature form of this protein can be cleaved at the cell surface, yielding different soluble forms that can interact with the G-protein coupled receptor, C-X3-C motif chemokine receptor 1 gene product. This gene plays a role in a wide range of diseases, including cancer, vasculitis, neuropathies, atherosclerosis, inflammatory diseases, and in human immunodeficiency virus infections. [provided by RefSeq, Sep 2017]', '16', 'miRNA', 53729838, 53729925, 47.73, 1, 'ENSG00000117528', 'P28288'),
(7, 'ABHD13', 'Predicted to enable palmitoyl-(protein) hydrolase activity. Predicted to be involved in protein depalmitoylation. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 24555958, 24582052, 45.46, 3, 'ENSG00000139826', 'Q7L211'),
(8, 'ABHD16B', 'Predicted to enable monoacylglycerol lipase activity and phospholipase activity. Predicted to be involved in monoacylglycerol catabolic process and phosphatidylserine catabolic process. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 182628, 183633, 49.70, 1, 'ENSG00000183260', 'Q9H3Z7'),
(9, 'ABHD3', 'This gene encodes a protein containing an alpha/beta hydrolase fold, which is a catalytic domain found in a very wide range of enzymes. The function of this protein has not been determined. [provided by RefSeq, Jul 2008]', '18', 'miRNA', 20114686, 20114751, 68.18, 1, 'ENSG00000158201', 'Q8WU67'),
(10, 'ABO', 'This gene encodes proteins related to the first discovered blood group system, ABO. Variation in the ABO gene (chromosome 9q34.2) is the basis of the ABO blood group, thus the presence of an allele determines the blood group in an individual. The \'O\' blood group is caused by a deletion of guanine-258 near the N-terminus of the protein which results in a frameshift and translation of an almost entirely different protein. Individuals with the A, B, and AB alleles express glycosyltransferase activities that convert the H antigen into the A or B antigen. Other minor alleles have been found for this gene. [provided by RefSeq, Apr 2022]', '9', 'protein_coding', 37243040, 37565827, 52.36, 23, 'ENSG00000281879', 'P16442'),
(11, 'ABO', 'This gene encodes proteins related to the first discovered blood group system, ABO. Variation in the ABO gene (chromosome 9q34.2) is the basis of the ABO blood group, thus the presence of an allele determines the blood group in an individual. The \'O\' blood group is caused by a deletion of guanine-258 near the N-terminus of the protein which results in a frameshift and translation of an almost entirely different protein. Individuals with the A, B, and AB alleles express glycosyltransferase activities that convert the H antigen into the A or B antigen. Other minor alleles have been found for this gene. [provided by RefSeq, Apr 2022]', '9', 'protein_coding', 37243040, 37565827, 52.36, 23, 'ENSG00000175164', 'P16442'),
(12, 'ABR', 'This gene encodes a protein that is similar to the protein encoded by the breakpoint cluster region gene located on chromosome 22. The protein encoded by this gene contains a GTPase-activating protein domain, a domain found in members of the Rho family of GTP-binding proteins. Functional studies in mice determined that this protein plays a role in vestibular morphogenesis. Alternatively spliced transcript variants have been reported for this gene. [provided by RefSeq, Feb 2012]', '17', 'protein_coding', 73909177, 73917496, 53.50, 5, 'ENSG00000276016', 'Q12979'),
(13, 'ABR', 'This gene encodes a protein that is similar to the protein encoded by the breakpoint cluster region gene located on chromosome 22. The protein encoded by this gene contains a GTPase-activating protein domain, a domain found in members of the Rho family of GTP-binding proteins. Functional studies in mice determined that this protein plays a role in vestibular morphogenesis. Alternatively spliced transcript variants have been reported for this gene. [provided by RefSeq, Feb 2012]', '17', 'protein_coding', 73909177, 73917496, 53.50, 5, 'ENSG00000278741', 'Q12979'),
(14, 'ABR', 'This gene encodes a protein that is similar to the protein encoded by the breakpoint cluster region gene located on chromosome 22. The protein encoded by this gene contains a GTPase-activating protein domain, a domain found in members of the Rho family of GTP-binding proteins. Functional studies in mice determined that this protein plays a role in vestibular morphogenesis. Alternatively spliced transcript variants have been reported for this gene. [provided by RefSeq, Feb 2012]', '17', 'protein_coding', 73909177, 73917496, 53.50, 5, 'ENSG00000159842', 'Q12979'),
(15, 'ABTB2', 'Predicted to enable protein heterodimerization activity. Predicted to act upstream of or within cellular response to toxic substance. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 172042079, 172188224, 48.99, 16, 'ENSG00000166016', 'Q8N961'),
(16, 'ACAA2', 'The encoded protein catalyzes the last step of the mitochondrial fatty acid beta-oxidation spiral. Unlike most mitochondrial matrix proteins, it contains a non-cleavable amino-terminal targeting signal. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 46601074, 46610782, 57.51, 13, 'ENSG00000167315', 'P42765'),
(17, 'ACADSB', 'Short/branched chain acyl-CoA dehydrogenase(ACADSB) is a member of the acyl-CoA dehydrogenase family of enzymes that catalyze the dehydrogenation of acyl-CoA derivatives in the metabolism of fatty acids or branch chained amino acids. Substrate specificity is the primary characteristic used to define members of this gene family. The ACADSB gene product has the greatest activity towards the short branched chain acyl-CoA derivative, (S)-2-methylbutyryl-CoA, but also reacts significantly with other 2-methyl branched chain substrates and with short straight chain acyl-CoAs. The cDNA encodes for a mitochondrial precursor protein which is cleaved upon mitochondrial import and predicted to yield a mature peptide of approximately 43.7-KDa. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 24181487, 24189106, 64.06, 2, 'ENSG00000196177', 'P45954'),
(18, 'ACO1', 'The protein encoded by this gene is a bifunctional, cytosolic protein that functions as an essential enzyme in the TCA cycle and interacts with mRNA to control the levels of iron inside cells. When cellular iron levels are high, this protein binds to a 4Fe-4S cluster and functions as an aconitase. Aconitases are iron-sulfur proteins that function to catalyze the conversion of citrate to isocitrate. When cellular iron levels are low, the protein binds to iron-responsive elements (IREs), which are stem-loop structures found in the 5\' UTR of ferritin mRNA, and in the 3\' UTR of transferrin receptor mRNA. When the protein binds to IRE, it results in repression of translation of ferritin mRNA, and inhibition of degradation of the otherwise rapidly degraded transferrin receptor mRNA. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. Alternative splicing results in multiple transcript variants [provided by RefSeq, Jan 2014]', '9', 'protein_coding', 38291918, 38398522, 47.79, 1, 'ENSG00000122729', 'P21399'),
(19, 'ACO2', 'The protein encoded by this gene belongs to the aconitase/IPM isomerase family. It is an enzyme that catalyzes the interconversion of citrate to isocitrate via cis-aconitate in the second step of the TCA cycle. This protein is encoded in the nucleus and functions in the mitochondrion. It was found to be one of the mitochondrial matrix proteins that are preferentially degraded by the serine protease 15(PRSS15), also known as Lon protease, after oxidative modification. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 3022625, 3024286, 63.72, 9, 'ENSG00000100412', 'Q99798'),
(20, 'ACOT9', 'The protein encoded by this gene is a mitochondrial acyl-CoA thioesterase of unknown function. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2010]', 'X', 'protein_coding', 100159244, 100168617, 52.79, 8, 'ENSG00000123130', 'Q9Y305'),
(21, 'ACRBP', 'The protein encoded by this gene is similar to proacrosin binding protein sp32 precursor found in mouse, guinea pig, and pig. This protein is located in the sperm acrosome and is thought to function as a binding protein to proacrosin for packaging and condensation of the acrosin zymogen in the acrosomal matrix. This protein is a member of the cancer/testis family of antigens and it is found to be immunogenic. In normal tissues, this mRNA is expressed only in testis, whereas it is detected in a range of different tumor types such as bladder, breast, lung, liver, and colon. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 37201612, 37231596, 37.99, 3, 'ENSG00000111644', 'Q8NEB7'),
(22, 'ACSS1', 'This gene encodes a mitochondrial acetyl-CoA synthetase enzyme. A similar protein in mice plays an important role in the tricarboxylic acid cycle by catalyzing the conversion of acetate to acetyl CoA. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Nov 2011]', '20', 'TR_J_gene', 22486228, 22486296, 46.38, 1, 'ENSG00000154930', 'Q9NUB1'),
(23, 'ACTL6A', 'This gene encodes a family member of actin-related proteins (ARPs), which share significant amino acid sequence identity to conventional actins. Both actins and ARPs have an actin fold, which is an ATP-binding cleft, as a common feature. The ARPs are involved in diverse cellular processes, including vesicular transport, spindle orientation, nuclear migration and chromatin remodeling. This gene encodes a 53 kDa subunit protein of the BAF (BRG1/brm-associated factor) complex in mammals, which is functionally related to SWI/SNF complex in S. cerevisiae and Drosophila; the latter is thought to facilitate transcriptional activation of specific genes by antagonizing chromatin-mediated transcriptional repression. Together with beta-actin, it is required for maximal ATPase activity of BRG1, and for the association of the BAF complex with chromatin/matrix. Three transcript variants that encode two different protein isoforms have been described. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 31488688, 31618588, 46.58, 29, 'ENSG00000136518', 'O96019'),
(24, 'ACTL7B', 'The protein encoded by this gene is a member of a family of actin-related proteins (ARPs) which share significant amino acid sequence identity to conventional actins. Both actins and ARPs have an actin fold, which is an ATP-binding cleft, as a common feature. The ARPs are involved in diverse cellular processes, including vesicular transport, spindle orientation, nuclear migration and chromatin remodeling. This gene (ACTL7B), and related gene, ACTL7A, are intronless, and are located approximately 4 kb apart in a head-to-head orientation within the familial dysautonomia candidate region on 9q31. Based on mutational analysis of the ACTL7B gene in patients with this disorder, it was concluded that it is unlikely to be involved in the pathogenesis of dysautonomia. Unlike ACTL7A, the ACTL7B gene is expressed predominantly in the testis, however, its exact function is not known. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 131058346, 131089885, 46.10, 6, 'ENSG00000148156', 'Q9Y614'),
(25, 'ACTR3B', 'This gene encodes a member of the actin-related proteins (ARP), which form multiprotein complexes and share 35-55% amino acid identity with conventional actin. The protein encoded by this gene may have a regulatory role in the actin cytoskeleton and induce cell-shape change and motility. Pseudogenes of this gene are located on chromosomes 2, 4, 10, 16, 22 and Y. Alternative splicing results in multiple transcript variants and protein isoforms. [provided by RefSeq, Jul 2012]', '7', 'protein_coding', 10694972, 10709782, 40.66, 1, 'ENSG00000133627', 'Q9P1U1'),
(26, 'ACTR5', 'Involved in several processes, including DNA repair; chromatin remodeling; and regulation of nucleobase-containing compound metabolic process. Located in cytoplasm and nucleus. Part of Ino80 complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 57940833, 57947706, 48.60, 2, 'ENSG00000101442', 'Q9H9F9'),
(27, 'ACYP1', 'This gene is a member of the acylphosphatase family. The encoded protein is a small cytosolic enzyme that catalyzes the hydrolysis of the carboxyl-phosphate bond of acylphosphates. Two isoenzymes have been isolated and described based on their tissue localization: erythrocyte (common) type acylphosphatase encoded by this gene, and muscle type acylphosphatase. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2014]', '14', 'protein_coding', 42950526, 42980528, 48.08, 7, 'ENSG00000119640', 'P07311'),
(28, 'ADA2', 'This gene encodes a member of a subfamily of the adenosine deaminase protein family. The encoded protein is one of two adenosine deaminases found in humans, which regulate levels of the signaling molecule, adenosine. The encoded protein is secreted from monocytes undergoing differentiation and may regulate cell proliferation and differentiation. This gene may be responsible for some of the phenotypic features associated with cat eye syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '22', 'IG_C_gene', 55245, 56751, 65.16, 1, 'ENSG00000093072', 'Q9NZK5'),
(29, 'ADAD1', 'Predicted to enable double-stranded RNA adenosine deaminase activity; double-stranded RNA binding activity; and tRNA-specific adenosine deaminase activity. Predicted to be involved in RNA processing and adenosine to inosine editing. Predicted to act upstream of or within spermatid development. Predicted to be located in male germ cell nucleus. Predicted to be active in cytoplasm and nucleolus. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 76736450, 76736548, 78.79, 1, 'ENSG00000164113', 'Q96M93'),
(30, 'ADAM21', 'This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family are membrane-anchored proteins structurally related to snake venom disintegrins, and have been implicated in a variety of biological processes involving cell-cell and cell-matrix interactions, including fertilization, muscle development, and neurogenesis. The expression of this gene expression is testis-specific. [provided by RefSeq, May 2011]', '14', 'protein_coding', 17644151, 17716824, 37.10, 5, 'ENSG00000139985', 'Q9UKJ8'),
(31, 'ADAM33', 'This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family are membrane-anchored proteins structurally related to snake venom disintegrins, and have been implicated in a variety of biological processes involving cell-cell and cell-matrix interactions, including fertilization, muscle development, and neurogenesis. This protein is a type I transmembrane protein implicated in asthma and bronchial hyperresponsiveness. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Sep 2013]', '20', 'protein_coding', 46334681, 46438190, 40.82, 22, 'ENSG00000149451', 'Q9BZ11'),
(32, 'ADAMTS13', 'This gene encodes a member of a family of proteins containing several distinct regions, including a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. The enzyme encoded by this gene specifically cleaves von Willebrand Factor (vWF). Defects in this gene are associated with thrombotic thrombocytopenic purpura. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '9', 'protein_coding', 9102630, 9134345, 37.35, 9, 'ENSG00000281244', 'Q76LX8'),
(33, 'ADAMTS13', 'This gene encodes a member of a family of proteins containing several distinct regions, including a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. The enzyme encoded by this gene specifically cleaves von Willebrand Factor (vWF). Defects in this gene are associated with thrombotic thrombocytopenic purpura. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '9', 'protein_coding', 9102630, 9134345, 37.35, 9, 'ENSG00000160323', 'Q76LX8'),
(34, 'ADAMTS16', 'This gene encodes a member of the ADAMTS (a disintegrin and metalloproteinase with thrombospondin motifs) protein family. ADAMTS family members share several distinct protein modules, including a propeptide region, a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. Individual members of this family differ in the number of C-terminal TS motifs, and some have unique C-terminal domains. The encoded preproprotein is proteolytically processed to generate the mature protein, which may inhibit chondrosarcoma cell proliferation and migration. This gene may regulate blood pressure. [provided by RefSeq, May 2016]', '5', 'protein_coding', 20923222, 20950697, 37.87, 14, 'ENSG00000145536', 'Q8TE57'),
(35, 'ADAMTS18', 'This gene encodes a member of the ADAMTS (a disintegrin and metalloproteinase with thrombospondin motifs) protein family. ADAMTS family members share several distinct protein modules, including a propeptide region, a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. Individual members of this family differ in the number of C-terminal TS motifs, and some have unique C-terminal domains. The encoded preproprotein is proteolytically processed to generate the mature protein, which may regulate hemostatic balance and function as a tumor suppressor. Mutations in this gene may be associated with microcornea, myopic chorioretinal atrophy, and telecanthus (MMCAT) and cone-rod dystrophy in human patients. [provided by RefSeq, May 2016]', '16', 'lncRNA', 144004916, 144008262, 40.45, 1, 'ENSG00000140873', 'Q8TE60'),
(36, 'ADAMTS5', 'This gene encodes a member of the ADAMTS (a disintegrin and metalloproteinase with thrombospondin motifs) protein family. Members of the family share several distinct protein modules, including a propeptide region, a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. Individual members of this family differ in the number of C-terminal TS motifs, and some have unique C-terminal domains. The encoded preproprotein is proteolytically processed to generate the mature enzyme. This enzyme contains two C-terminal TS motifs and functions as an aggrecanase that cleaves aggrecan, a major proteoglycan of cartilage, and may mediate cartilage destruction in osteoarthritis. [provided by RefSeq, Feb 2016]', '21', 'miRNA', 50048973, 50049061, 40.45, 1, 'ENSG00000154736', 'Q9UNA0'),
(37, 'ADCK5', 'Predicted to enable protein serine/threonine kinase activity. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 44573729, 44575371, 63.97, 1, 'ENSG00000285451', 'Q3MIX3'),
(38, 'ADCK5', 'Predicted to enable protein serine/threonine kinase activity. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 44573729, 44575371, 63.97, 1, 'ENSG00000173137', 'Q3MIX3'),
(39, 'ADCYAP1', 'This gene encodes a secreted proprotein that is further processed into multiple mature peptides. These peptides stimulate adenylate cyclase and increase cyclic adenosine monophosphate (cAMP) levels, resulting in the transcriptional activation of target genes. The products of this gene are key mediators of neuroendocrine stress responses. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2013]', '18', 'protein_coding', 29073035, 29168333, 45.51, 6, 'ENSG00000141433', 'P18509'),
(40, 'ADD3', 'Adducins are heteromeric proteins composed of different subunits referred to as adducin alpha, beta and gamma. The three subunits are encoded by distinct genes and belong to a family of membrane skeletal proteins involved in the assembly of spectrin-actin network in erythrocytes and at sites of cell-cell contact in epithelial tissues. While adducins alpha and gamma are ubiquitously expressed, the expression of adducin beta is restricted to brain and hematopoietic tissues. Adducin, originally purified from human erythrocytes, was found to be a heterodimer of adducins alpha and beta. Polymorphisms resulting in amino acid substitutions in these two subunits have been associated with the regulation of blood pressure in an animal model of hypertension. Heterodimers consisting of alpha and gamma subunits have also been described. Structurally, each subunit is comprised of two distinct domains. The amino-terminal region is protease resistant and globular in shape, while the carboxy-terminal region is protease sensitive. The latter contains multiple phosphorylation sites for protein kinase C, the binding site for calmodulin, and is required for association with spectrin and actin. Alternatively spliced adducin gamma transcripts encoding different isoforms have been described. The functions of the different isoforms are not known. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 95076749, 95217482, 41.01, 7, 'ENSG00000148700', 'Q9UEY8'),
(41, 'ADGRG2', 'This gene encodes a member of the G protein-coupled receptor family described as an epididymis-specific transmembrane protein. The encoded protein may be proteolytically processed as it contains a motif shown to be a protein scission motif in some members of this family (PMID: 11973329). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 16067139, 16180762, 39.41, 3, 'ENSG00000173698', 'Q8IZP9'),
(42, 'ADGRL4', 'Predicted to enable G protein-coupled receptor activity. Predicted to be involved in adenylate cyclase-activating G protein-coupled receptor signaling pathway. Predicted to be located in cytoplasmic vesicle. Predicted to be active in plasma membrane. Biomarker of glioblastoma and hypertrophic cardiomyopathy. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 23453287, 23486603, 41.69, 11, 'ENSG00000162618', 'Q9HBW9'),
(43, 'ADIPOR2', 'The adiponectin receptors, ADIPOR1 (MIM 607945) and ADIPOR2, serve as receptors for globular and full-length adiponectin (MIM 605441) and mediate increased AMPK (see MIM 602739) and PPAR-alpha (PPARA; MIM 170998) ligand activities, as well as fatty acid oxidation and glucose uptake by adiponectin (Yamauchi et al., 2003 [PubMed 12802337]).[supplied by OMIM, Mar 2008]', '12', 'miRNA', 28459869, 28459948, 48.75, 1, 'ENSG00000285070', 'Q86V24'),
(44, 'ADIPOR2', 'The adiponectin receptors, ADIPOR1 (MIM 607945) and ADIPOR2, serve as receptors for globular and full-length adiponectin (MIM 605441) and mediate increased AMPK (see MIM 602739) and PPAR-alpha (PPARA; MIM 170998) ligand activities, as well as fatty acid oxidation and glucose uptake by adiponectin (Yamauchi et al., 2003 [PubMed 12802337]).[supplied by OMIM, Mar 2008]', '12', 'miRNA', 28459869, 28459948, 48.75, 1, 'ENSG00000006831', 'Q86V24'),
(45, 'ADM', 'The protein encoded by this gene is a preprohormone which is cleaved to form two biologically active peptides, adrenomedullin and proadrenomedullin N-terminal 20 peptide. Adrenomedullin is a 52 aa peptide with several functions, including vasodilation, regulation of hormone secretion, promotion of angiogenesis, and antimicrobial activity. The antimicrobial activity is antibacterial, as the peptide has been shown to kill E. coli and S. aureus at low concentration. [provided by RefSeq, Aug 2014]', '11', 'protein_coding', 90515611, 90529584, 35.92, 4, 'ENSG00000148926', 'P35318'),
(46, 'ADM2', 'This gene encodes a member of the calcitonin gene-related peptide (CGRP)/calcitonin family of hormones that play a role in the regulation of cardiovascular homeostasis, prolactin release, anti-diuresis, anti-natriuresis, and regulation of food and water intake. The encoded protein is proteolytically processed to generate one or more biologically active peptides. [provided by RefSeq, Jul 2015]', '22', 'protein_coding', 47847233, 47914667, 35.55, 8, 'ENSG00000128165', 'Q7Z4H4'),
(47, 'ADNP', 'Vasoactive intestinal peptide is a neuroprotective factor that has a stimulatory effect on the growth of some tumor cells and an inhibitory effect on others. This gene encodes a protein that is upregulated by vasoactive intestinal peptide and may be involved in its stimulatory effect on certain tumor cells. The encoded protein contains one homeobox and nine zinc finger domains, suggesting that it functions as a transcription factor. This gene is also upregulated in normal proliferative tissues. Finally, the encoded protein may increase the viability of certain cell types through modulation of p53 activity. Alternatively spliced transcript variants encoding the same protein have been described. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 17673299, 17673394, 38.54, 1, 'ENSG00000101126', 'Q9H2P0'),
(48, 'ADPRHL1', 'ADP-ribosylation is a reversible posttranslational modification used to regulate protein function. ADP-ribosyltransferases (see ART1; MIM 601625) transfer ADP-ribose from NAD+ to the target protein, and ADP-ribosylhydrolases, such as ADPRHL1, reverse the reaction (Glowacki et al., 2002 [PubMed 12070318]).[supplied by OMIM, Mar 2008]', '13', 'protein_coding', 46599251, 46759118, 41.06, 5, 'ENSG00000153531', 'Q8NDY3'),
(49, 'ADRA1B', 'Alpha-1-adrenergic receptors (alpha-1-ARs) are members of the G protein-coupled receptor superfamily. They activate mitogenic responses and regulate growth and proliferation of many cells. There are 3 alpha-1-AR subtypes: alpha-1A, -1B and -1D, all of which signal through the Gq/11 family of G-proteins and different subtypes show different patterns of activation. This gene encodes alpha-1B-adrenergic receptor, which induces neoplastic transformation when transfected into NIH 3T3 fibroblasts and other cell lines. Thus, this normal cellular gene is identified as a protooncogene. This gene comprises 2 exons and a single large intron of at least 20 kb that interrupts the coding region. [provided by RefSeq, Jul 2008]', '5', 'miRNA', 171186184, 171186292, 41.28, 1, 'ENSG00000170214', 'P35368'),
(50, 'ADRB2', 'This gene encodes beta-2-adrenergic receptor which is a member of the G protein-coupled receptor superfamily. This receptor is directly associated with one of its ultimate effectors, the class C L-type calcium channel Ca(V)1.2. This receptor-channel complex also contains a G protein, an adenylyl cyclase, cAMP-dependent kinase, and the counterbalancing phosphatase, PP2A. The assembly of the signaling complex provides a mechanism that ensures specific and rapid signaling by this G protein-coupled receptor. This receptor is also a transcription regulator of the alpha-synuclein gene, and together, both genes are believed to be associated with risk of Parkinson\'s Disease. This gene is intronless. Different polymorphic forms, point mutations, and/or downregulation of this gene are associated with nocturnal asthma, obesity, type 2 diabetes and cardiovascular disease. [provided by RefSeq, Oct 2019]', '5', 'miRNA', 69565808, 69565868, 70.49, 1, 'ENSG00000169252', 'P07550'),
(51, 'ADRM1', 'This gene encodes a member of the adhesion regulating molecule 1 protein family. The encoded protein is a component of the proteasome where it acts as a ubiquitin receptor and recruits the deubiquitinating enzyme, ubiquitin carboxyl-terminal hydrolase L5. Increased levels of the encoded protein are associated with increased cell adhesion, which is likely an indirect effect of this intracellular protein. Dysregulation of this gene has been implicated in carcinogenesis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 54748918, 54798255, 44.09, 2, 'ENSG00000130706', 'Q16186'),
(52, 'AFG1L', 'This gene encodes a mitochondrial integral membrane protein that plays a role in mitochondrial protein homeostasis. The protein contains a P-loop motif and a five-domain structure that is conserved in fly, yeast, and bacteria. It functions to mediate the degradation of nuclear-encoded complex IV subunits. Two conserved estrogen receptor binding sites are located within 2.5 kb of this gene. Polymorphisms in this gene have been associated with bipolar disorder. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2016]', '6', 'snoRNA', 45104906, 45105042, 59.12, 1, 'ENSG00000135537', 'Q8WV93'),
(53, 'AGAP5', 'Predicted to enable GTPase activator activity and GTPase activity. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 51491227, 51501800, 50.21, 3, 'ENSG00000172650', 'A6NIR3'),
(54, 'AGAP9', 'Predicted to enable GTPase activator activity and GTPase activity. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 59646962, 59647020, 38.98, 1, 'ENSG00000204172', 'Q5VTM2'),
(55, 'AGBL3', 'Enables metallocarboxypeptidase activity. Involved in protein side chain deglutamylation. Predicted to be located in cytosol. Predicted to be active in centriole and cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 39655627, 39791666, 35.35, 12, 'ENSG00000146856', 'Q8NEM8'),
(56, 'AGO4', 'This gene encodes a member of the Argonaute family of proteins which contain PAZ and PIWI domains and play an integral role in RNA interference and short-interfering-RNA-mediated gene silencing. This gene is located on chromosome 1 in a cluster of related family members. [provided by RefSeq, Mar 2017]', '1', 'protein_coding', 77900, 81921, 43.96, 9, 'ENSG00000134698', 'Q9HCK5'),
(57, 'AGRP', 'This gene encodes an antagonist of the melanocortin-3 and melanocortin-4 receptor. It appears to regulate hypothalamic control of feeding behavior via melanocortin receptor and/or intracellular calcium regulation, and thus plays a role in weight homeostasis. Mutations in this gene have been associated with late on-set obesity. [provided by RefSeq, Dec 2009]', '16', 'miRNA', 64785626, 64785686, 55.74, 1, 'ENSG00000159723', 'O00253'),
(58, 'AHR', 'The protein encoded by this gene is a ligand-activated helix-loop-helix transcription factor involved in the regulation of biological responses to planar aromatic hydrocarbons. This receptor has been shown to regulate xenobiotic-metabolizing enzymes such as cytochrome P450. Before ligand binding, the encoded protein is sequestered in the cytoplasm; upon ligand binding, this protein moves to the nucleus and stimulates transcription of target genes. [provided by RefSeq, Sep 2015]', '7', 'protein_coding', 62643994, 62935150, 42.69, 5, 'ENSG00000106546', 'P35869'),
(59, 'AIF1L', 'Enables actin filament binding activity. Predicted to be involved in actin filament bundle assembly and ruffle assembly. Located in actin filament and focal adhesion. Biomarker of breast cancer. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 833818, 863864, 39.00, 1, 'ENSG00000126878', 'Q9BQI0'),
(60, 'AIFM2', 'This gene encodes a flavoprotein oxidoreductase that binds single stranded DNA and is thought to contribute to apoptosis in the presence of bacterial and viral DNA. The expression of this gene is also found to be induced by tumor suppressor protein p53 in colon cancer cells. [provided by RefSeq, Nov 2010]', '10', 'protein_coding', 110410, 365012, 40.15, 6, 'ENSG00000042286', 'Q9BRQ8'),
(61, 'AIFM3', 'Predicted to enable oxidoreductase activity, acting on NAD(P)H. Involved in execution phase of apoptosis. Located in cytosol; endoplasmic reticulum; and mitochondrial inner membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 804940, 805001, 80.65, 1, 'ENSG00000183773', 'Q96NN9'),
(62, 'AIRE', 'This gene encodes a transcriptional regulator that forms nuclear bodies and interacts with the transcriptional coactivator CREB binding protein. The encoded protein plays an important role in immunity by regulating the expression of autoantigens and negative selection of autoreactive T-cells in the thymus. Mutations in this gene cause the rare autosomal-recessive systemic autoimmune disease termed autoimmune polyendocrinopathy with candidiasis and ectodermal dystrophy (APECED). [provided by RefSeq, Jun 2012]', '21', 'protein_coding', 58481247, 58629091, 44.38, 6, 'ENSG00000160224', 'O43918'),
(63, 'AJM1', 'Predicted to be involved in cell-cell junction organization. Predicted to be located in adherens junction; apical plasma membrane; and cilium. Predicted to be active in apical junction complex and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 47589667, 47594411, 59.16, 2, 'ENSG00000232434', 'C9J069'),
(64, 'AKAIN1', 'Enables protein kinase A binding activity. Involved in protein localization. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '18', 'processed_pseudogene', 83907301, 83907781, 62.79, 1, 'ENSG00000231824', 'P0CW23'),
(65, 'AKAP12', 'The A-kinase anchor proteins (AKAPs) are a group of structurally diverse proteins, which have the common function of binding to the regulatory subunit of protein kinase A (PKA) and confining the holoenzyme to discrete locations within the cell. This gene encodes a member of the AKAP family. The encoded protein is expressed in endothelial cells, cultured fibroblasts, and osteosarcoma cells. It associates with protein kinases A and C and phosphatase, and serves as a scaffold protein in signal transduction. This protein and RII PKA colocalize at the cell periphery. This protein is a cell growth-related protein. Antibodies to this protein can be produced by patients with myasthenia gravis. Alternative splicing of this gene results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 29722775, 29738528, 45.02, 15, 'ENSG00000131016', 'Q02952'),
(66, 'AKAP14', 'The A-kinase anchor proteins (AKAPs) are a group of structurally diverse proteins, which have the common function of binding to the regulatory subunit of protein kinase A (PKA) and confining the holoenzyme to discrete locations within the cell. This gene encodes a member of the AKAP family. The protein anchors PKA in ciliary axonemes and, in this way, may play a role in regulating ciliary beat frequency. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 63752935, 63805376, 37.32, 7, 'ENSG00000186471', 'Q86UN6'),
(67, 'AKAP6', 'The A-kinase anchor proteins (AKAPs) are a group of structurally diverse proteins, which have the common function of binding to the regulatory subunit of protein kinase A (PKA) and confining the holoenzyme to discrete locations within the cell. This gene encodes a member of the AKAP family. The encoded protein is highly expressed in various brain regions and cardiac and skeletal muscle. It is specifically localized to the sarcoplasmic reticulum and nuclear membrane, and is involved in anchoring PKA to the nuclear membrane or sarcoplasmic reticulum. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 52497408, 52577060, 46.66, 1, 'ENSG00000151320', 'Q13023'),
(68, 'AKR1B15', 'Enables estradiol 17-beta-dehydrogenase [NAD(P)+] activity. Predicted to be involved in estrogen biosynthetic process. Located in cytosol and mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 3225030, 3232417, 61.97, 1, 'ENSG00000227471', 'C9JRZ8'),
(69, 'AKTIP', 'The mouse homolog of this gene produces fused toes and thymic hyperplasia in heterozygous mutant animals while homozygous mutants die in early development. This gene may play a role in apoptosis as these morphological abnormalities are caused by altered patterns of programmed cell death. The protein encoded by this gene is similar to the ubiquitin ligase domain of other ubiquitin-conjugating enzymes but lacks the conserved cysteine residue that enables those enzymes to conjugate ubiquitin to the target protein. This protein interacts directly with serine/threonine kinase protein kinase B (PKB)/Akt and modulates PKB activity by enhancing the phosphorylation of PKB\'s regulatory sites. Alternative splicing results in two transcript variants encoding the same protein. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 233671898, 233773300, 42.19, 1, 'ENSG00000166971', 'Q9H8T0'),
(70, 'ALAS2', 'The product of this gene specifies an erythroid-specific mitochondrially located enzyme. The encoded protein catalyzes the first step in the heme biosynthetic pathway. Defects in this gene cause X-linked pyridoxine-responsive sideroblastic anemia. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 4143168, 4187310, 49.97, 5, 'ENSG00000158578', 'P22557'),
(71, 'ALDH8A1', 'This gene encodes a member of the aldehyde dehydrogenase family of proteins. The encoded protein has been implicated in the synthesis of 9-cis-retinoic acid and in the breakdown of the amino acid tryptophan. This enzyme converts 9-cis-retinal into the retinoid X receptor ligand 9-cis-retinoic acid, and has approximately 40-fold higher activity with 9-cis-retinal than with all-trans-retinal. In addition, this enzyme has been shown to catalyze the conversion of 2-aminomuconic semialdehyde to 2-aminomuconate in the kynurenine pathway of tryptophan catabolism. [provided by RefSeq, Jul 2018]', '6', 'protein_coding', 171058414, 171460408, 39.52, 15, 'ENSG00000118514', 'Q9H2A2'),
(72, 'ALDOA', 'This gene encodes a member of the class I fructose-bisphosphate aldolase protein family. The encoded protein is a glycolytic enzyme that catalyzes the reversible conversion of fructose-1,6-bisphosphate to glyceraldehyde 3-phosphate and dihydroxyacetone phosphate. Three aldolase isozymes (A, B, and C), encoded by three different genes, are differentially expressed during development. Mutations in this gene have been associated with Glycogen Storage Disease XII, an autosomal recessive disorder associated with hemolytic anemia. Disruption of this gene also plays a role in the progression of multiple types of cancers. Related pseudogenes have been identified on chromosomes 3 and 10. [provided by RefSeq, Sep 2017]', '16', 'protein_coding', 88149959, 88168729, 37.88, 4, 'ENSG00000149925', 'P04075'),
(73, 'ALG5', 'This gene encodes a member of the glycosyltransferase 2 family. The encoded protein participates in glucosylation of the oligomannose core in N-linked glycosylation of proteins. The addition of glucose residues to the oligomannose core is necessary to ensure substrate recognition, and therefore, effectual transfer of the oligomannose core to the nascent glycoproteins. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2008]', '13', 'protein_coding', 111316184, 111344249, 50.50, 2, 'ENSG00000120697', 'Q9Y673'),
(74, 'ALK', 'This gene encodes a receptor tyrosine kinase, which belongs to the insulin receptor superfamily. This protein comprises an extracellular domain, an hydrophobic stretch corresponding to a single pass transmembrane region, and an intracellular kinase domain. It plays an important role in the development of the brain and exerts its effects on specific neurons in the nervous system. This gene has been found to be rearranged, mutated, or amplified in a series of tumours including anaplastic large cell lymphomas, neuroblastoma, and non-small cell lung cancer. The chromosomal rearrangements are the most common genetic alterations in this gene, which result in creation of multiple fusion genes in tumourigenesis, including ALK (chromosome 2)/EML4 (chromosome 2), ALK/RANBP2 (chromosome 2), ALK/ATIC (chromosome 2), ALK/TFG (chromosome 3), ALK/NPM1 (chromosome 5), ALK/SQSTM1 (chromosome 5), ALK/KIF5B (chromosome 10), ALK/CLTC (chromosome 17), ALK/TPM4 (chromosome 19), and ALK/MSN (chromosome X).[provided by RefSeq, Jan 2011]', '2', 'protein_coding', 154941073, 154969411, 38.82, 7, 'ENSG00000171094', 'Q9UM73'),
(75, 'ALKBH1', 'This gene encodes a homolog to the E. coli alkB gene product. The E. coli alkB protein is part of the adaptive response mechanism of DNA alkylation damage repair. It is involved in damage reversal by oxidative demethylation of 1-methyladenine and 3-methylcytosine. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 89324353, 89498187, 35.47, 15, 'ENSG00000100601', 'Q13686'),
(76, 'ALOX15', 'This gene encodes a member of the lipoxygenase family of proteins. The encoded enzyme acts on various polyunsaturated fatty acid substrates to generate various bioactive lipid mediators such as eicosanoids, hepoxilins, lipoxins, and other molecules. The encoded enzyme and its reaction products have been shown to regulate inflammation and immunity. Multiple pseudogenes of this gene have been identified in the human genome. [provided by RefSeq, Aug 2017]', '17', 'protein_coding', 176647387, 176659054, 58.70, 9, 'ENSG00000161905', 'P16050'),
(77, 'ALOX15B', 'This gene encodes a member of the lipoxygenase family of structurally related nonheme iron dioxygenases involved in the production of fatty acid hydroperoxides. The encoded protein converts arachidonic acid exclusively to 15S-hydroperoxyeicosatetraenoic acid, while metabolizing linoleic acid less effectively. This gene is located in a cluster of related genes and a pseudogene that spans approximately 100 kilobases on the short arm of chromosome 17. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 75472052, 75495445, 49.97, 4, 'ENSG00000179593', 'O15296'),
(78, 'ALPG', 'There are at least four distinct but related alkaline phosphatases: intestinal, placental, placental-like, and liver/bone/kidney (tissue non-specific). The product of this gene is a membrane bound glycosylated enzyme, localized to testis, thymus and certain germ cell tumors, that is closely related to both the placental and intestinal forms of alkaline phosphatase. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 36929, 51949, 48.82, 1, 'ENSG00000163286', 'P10696'),
(79, 'ALPK2', 'Predicted to enable ATP binding activity; protein serine kinase activity; and protein serine/threonine kinase activity. Involved in several processes, including epicardium morphogenesis; heart development; and negative regulation of Wnt signaling pathway involved in heart development. Acts upstream of or within regulation of gene expression. Located in basolateral plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 39405728, 39445805, 41.22, 21, 'ENSG00000198796', 'Q86TB3'),
(80, 'ALX1', 'The specific function of this gene has yet to be determined in humans; however,  in rodents, it is necessary for survival of the forebrain mesenchyme and may  also be involved in development of the cervix.  Mutations in the mouse gene lead  to neural tube defects such as acrania and meroanencephaly. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 58415384, 58546802, 39.98, 11, 'ENSG00000180318', 'Q15699'),
(81, 'AMDHD1', 'Predicted to enable imidazolonepropionase activity. Predicted to be involved in L-histidine catabolic process. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 76344474, 76348415, 37.16, 1, 'ENSG00000139344', 'Q96NU7'),
(82, 'AMTN', 'The mineralized portions of teeth, the dentin and enamel, are formed by mesenchyme-derived odontoblasts and epithelium-derived ameloblasts, respectively. As ameloblasts differentiate, they deposit specific proteins necessary for enamel formation, including amelogenin (AMELX; MIM 300391), enamelin (ENAM; MIM 606585), and ameloblastin (AMBN; MIM 601259), in the organic enamel matrix. Amelotin is specifically expressed in maturation-stage ameloblasts (Iwasaki et al., 2005 [PubMed 16304441]).[supplied by OMIM, Mar 2008]', '4', 'protein_coding', 43182184, 43224587, 49.09, 18, 'ENSG00000187689', 'Q6UX39'),
(83, 'ANAPC11', 'Enables cullin family protein binding activity and ubiquitin-ubiquitin ligase activity. Contributes to ubiquitin-protein transferase activity. Involved in anaphase-promoting complex-dependent catabolic process; positive regulation of mitotic metaphase/anaphase transition; and protein polyubiquitination. Located in nucleolus and nucleoplasm. Part of anaphase-promoting complex. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 83956, 128739, 38.52, 5, 'ENSG00000141552', 'Q9NYG5'),
(84, 'ANAPC4', 'A large protein complex, termed the anaphase-promoting complex (APC), or the cyclosome, promotes metaphase-anaphase transition by ubiquitinating its specific substrates such as mitotic cyclins and anaphase inhibitor, which are subsequently degraded by the 26S proteasome. Biochemical studies have shown that the vertebrate APC contains eight subunits. The composition of the APC is highly conserved in organisms from yeast to humans. The exact function of this gene product is not known. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2013]', '4', 'unprocessed_pseudogene', 56740179, 56741096, 47.71, 1, 'ENSG00000053900', 'Q9UJX5'),
(85, 'ANKEF1', 'Predicted to enable calcium ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 27203508, 27221768, 51.92, 14, 'ENSG00000132623', 'Q9NU02');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(86, 'ANKHD1', 'This gene encodes a protein with multiple ankyrin repeat domains and a single KH-domain. The protein is thought to function as a scaffolding protein, and it may be involved in the regulation of caspases and thereby play an antiapoptotic role in cell survival. Alternative splicing results in multiple transcript variants, one of which generates a fusion transcript (MASK-BP3) with the downstream eIF4E-binding protein 3 (EIF4EBP3) gene, resulting in a protein comprised of the ANKHD1 sequence for the majority of the protein and a different C-terminus due to an alternate reading frame for the EIF4EBP3 segments. [provided by RefSeq, Sep 2010]', '5', 'protein_coding', 81650459, 81663112, 55.06, 5, 'ENSG00000131503', 'Q8IWZ3'),
(87, 'ANKMY2', 'Predicted to enable enzyme binding activity and zinc ion binding activity. Predicted to be located in cilium. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 98106862, 98209638, 43.73, 2, 'ENSG00000106524', 'Q8IV38'),
(88, 'ANKRA2', 'Enables enzyme binding activity and low-density lipoprotein particle receptor binding activity. Involved in regulation of protein-containing complex assembly. Located in 3M complex; cytosol; and membrane. Part of protein-containing complex. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 400589, 411524, 59.93, 5, 'ENSG00000164331', 'Q9H9E1'),
(89, 'ANKRD10', 'Predicted to be involved in protein targeting to chloroplast. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 2481817, 2524702, 43.04, 9, 'ENSG00000088448', 'Q9NXR5'),
(90, 'ANKRD16', 'Predicted to be involved in tRNA modification. Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 63861498, 63862988, 69.89, 1, 'ENSG00000134461', 'Q6P6B7'),
(91, 'ANKRD29', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 1783517, 1861180, 46.63, 3, 'ENSG00000154065', 'Q8N6D5'),
(92, 'ANKS3', 'Predicted to be located in cytoplasm. Predicted to be active in cilium. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 33800, 56132, 35.88, 7, 'ENSG00000168096', 'Q6ZW76'),
(93, 'ANOS1', 'Mutations in this gene cause the X-linked Kallmann syndrome. The encoded protein is similar in sequence to proteins known to function in neural cell adhesion and axonal migration. In addition, this cell surface protein is N-glycosylated and may have anti-protease activity. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 17592136, 17628749, 45.03, 9, 'ENSG00000011201', 'P23352'),
(94, 'ANXA4', 'Annexin IV (ANX4) belongs to the annexin family of calcium-dependent phospholipid binding proteins. Although their functions are still not clearly defined, several members of the annexin family have been implicated in membrane-related events along exocytotic and endocytotic pathways. ANX4 has 45 to 59% identity with other members of its family and shares a similar size and exon-intron organization. Isolated from human placenta, ANX4 encodes a protein that has possible interactions with ATP, and has in vitro anticoagulant activity and also inhibits phospholipase A2 activity. ANX4 is almost exclusively expressed in epithelial cells. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2016]', '2', 'protein_coding', 28053228, 28054165, 44.35, 1, 'ENSG00000196975', 'P09525'),
(95, 'ANXA5', 'The Annexin 5 gene spans 29 kb containing 13 exons, and encodes a single transcript of approximately 1.6 kb and a protein product with a molecular weight of about 35 kDa.The protein encoded by this gene belongs to the annexin family of calcium-dependent phospholipid binding proteins some of which have been implicated in membrane-related events along exocytotic and endocytotic pathways. Annexin 5 is a phospholipase A2 and protein kinase C inhibitory protein with calcium channel activity and a potential role in cellular signal transduction, inflammation, growth and differentiation. Annexin 5 has also been described as placental anticoagulant protein I, vascular anticoagulant-alpha, endonexin II, lipocortin V, placental protein 4 and anchorin CII. Polymorphisms in this gene have been implicated in various obstetric complications. [provided by RefSeq, Dec 2019]', '4', 'miRNA', 95309923, 95309984, 45.16, 1, 'ENSG00000164111', 'P08758'),
(96, 'ANXA7', 'Annexin VII is a member of the annexin family of calcium-dependent phospholipid binding proteins.The Annexin VII gene contains 14 exons and spans approximately 34 kb of DNA. An alternatively spliced cassette exon results in two mRNA transcripts of 2.0 and 2.4 kb which are predicted to generate two protein isoforms differing in their N-terminal domain. The alternative splicing event is tissue specific and the mRNA containing the cassette exon is prevalent in brain, heart and skeletal muscle. The transcripts also differ in their 3\'-non coding regions by the use of two alternative poly(A) signals. Annexin VII encodes a protein with a molecular weight of approximately 51 kDa with a unique, highly hydrophobic N-terminal domain of 167 amino acids and a conserved C-terminal region of 299 amino acids. The latter domain is composed of alternating hydrophobic and hydrophilic segments. Structural analysis of the protein suggests that Annexin VII is a membrane binding protein with diverse properties, including voltage-sensitive calcium channel activity, ion selectivity and membrane fusion. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 179497, 196283, 47.10, 2, 'ENSG00000138279', 'P20073'),
(97, 'ANXA8L1', 'This gene encodes a member of the annexin family of evolutionarily conserved Ca2+ and phospholipid binding proteins. The encoded protein may function as an an anticoagulant that indirectly inhibits the thromboplastin-specific complex. Overexpression of this gene has been associated with acute myelocytic leukemia. A highly similar duplicated copy of this gene is found in close proximity on the long arm of chromosome 10. [provided by RefSeq, Apr 2014]', '10', 'transcribed_unprocessed_pseudogene', 1414829, 1419435, 44.76, 2, 'ENSG00000264230', 'Q5VT79'),
(98, 'AP5S1', 'Involved in double-strand break repair via homologous recombination and endosomal transport. Located in several cellular components, including late endosome; lysosome; and nucleoplasm. Part of AP-type membrane coat adaptor complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 181305593, 181305652, 53.33, 1, 'ENSG00000125843', 'Q9NUS5'),
(99, 'APCDD1', 'This locus encodes an inhibitor of the Wnt signaling pathway. Mutations at this locus have been associated with hereditary hypotrichosis simplex. Increased expression of this gene may also be associated with colorectal carcinogenesis.[provided by RefSeq, Sep 2010]', '18', 'protein_coding', 612428, 619014, 53.67, 5, 'ENSG00000154856', 'Q8J025'),
(100, 'APELA', 'This gene encodes a peptide hormone that binds to the Apelin receptor. The encoded protein is required for heart development in zebrafish and has been shown to maintain self-renewal of human embryonic stem cells through activation of the PI3K/AKT pathway. Experiments in human and mouse cell lines point to additional roles for the encoded protein in angiogenesis and regulation of vascular tone. [provided by RefSeq, Jul 2016]', '4', 'protein_coding', 155821024, 155854456, 39.66, 10, 'ENSG00000248329', 'P0DMC3'),
(101, 'APEX1', 'The APEX gene encodes the major AP endonuclease in human cells. It encodes the APEX endonuclease, a DNA repair enzyme with apurinic/apyrimidinic (AP) activity. Such AP activity sites occur frequently in DNA molecules by spontaneous hydrolysis, by DNA damaging agents or by DNA glycosylases that remove specific abnormal bases. The AP sites are the most frequent pre-mutagenic lesions that can prevent normal DNA replication. Splice variants have been found for this gene; all encode the same protein. Disruptions in the biological functions related to APEX are associated with many various malignancies and neurodegenerative diseases.[provided by RefSeq, Dec 2019]', '14', 'protein_coding', 101411286, 102074812, 36.28, 8, 'ENSG00000100823', 'P27695'),
(102, 'APEX1', 'The APEX gene encodes the major AP endonuclease in human cells. It encodes the APEX endonuclease, a DNA repair enzyme with apurinic/apyrimidinic (AP) activity. Such AP activity sites occur frequently in DNA molecules by spontaneous hydrolysis, by DNA damaging agents or by DNA glycosylases that remove specific abnormal bases. The AP sites are the most frequent pre-mutagenic lesions that can prevent normal DNA replication. Splice variants have been found for this gene; all encode the same protein. Disruptions in the biological functions related to APEX are associated with many various malignancies and neurodegenerative diseases.[provided by RefSeq, Dec 2019]', '14', 'protein_coding', 101411286, 102074812, 36.28, 8, 'ENSG00000291796', 'P27695'),
(103, 'APOBEC1', 'This gene encodes a member of the cytidine deaminase enzyme family. The encoded protein forms a multiple-protein editing holoenzyme with APOBEC1 complementation factor (ACF) and APOBEC1 stimulating protein (ASP). This holoenzyme is involved in the editing of C-to-U nucleotide bases in apolipoprotein B and neurofibromatosis-1 mRNAs. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2015]', '12', 'protein_coding', 59543, 125319, 50.51, 16, 'ENSG00000111701', 'P41238'),
(104, 'APOBEC3C', 'This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. It is thought that the proteins may be RNA editing enzymes and have roles in growth or cell cycle control. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 48178, 49072, 57.65, 1, 'ENSG00000244509', 'Q9NRW3'),
(105, 'APOBEC3F', 'This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster, thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. It is thought that the proteins may be RNA editing enzymes and have roles in growth or cell cycle control. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 20160593, 20186206, 36.98, 13, 'ENSG00000128394', 'Q8IUX4'),
(106, 'APOBEC3G', 'This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster, thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. The protein encoded by this gene catalyzes site-specific deamination of both RNA and single-stranded DNA. The encoded protein has been found to be a specific inhibitor of human immunodeficiency virus-1 (HIV-1) infectivity. [provided by RefSeq, Mar 2017]', '22', 'rRNA', 149485, 149603, 60.50, 1, 'ENSG00000239713', 'Q9HC16'),
(107, 'APOL2', 'This gene is a member of the apolipoprotein L gene family. The encoded protein is found in the cytoplasm, where it may affect the movement of lipids or allow the binding of lipids to organelles. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 75275, 138464, 50.47, 3, 'ENSG00000128335', 'Q9BQE5'),
(108, 'APOL4', 'This gene encodes a member of the apolipoprotein L family. The encoded protein may play a role in lipid exchange and transport throughout the body, as well as in reverse cholesterol transport from peripheral cells to the liver. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2020]', '22', 'protein_coding', 182143987, 182803024, 38.92, 6, 'ENSG00000100336', 'Q9BPW4'),
(109, 'APOL5', 'This gene is a member of the apolipoprotein L gene family. The encoded protein is found in the cytoplasm, where it may affect the movement of lipids or allow the binding of lipids to organelles. [provided by RefSeq, Jul 2008]', '22', 'miRNA', 50585786, 50585879, 56.38, 1, 'ENSG00000128313', 'Q9BWW9'),
(110, 'APOL5', 'This gene is a member of the apolipoprotein L gene family. The encoded protein is found in the cytoplasm, where it may affect the movement of lipids or allow the binding of lipids to organelles. [provided by RefSeq, Jul 2008]', '22', 'miRNA', 50585786, 50585879, 56.38, 1, 'ENSG00000277390', 'Q9BWW9'),
(111, 'APOOL', 'This gene encodes a protein which contains an apolipoprotein O superfamily domain. This domain is found on proteins in circulating lipoprotein complexes. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 2765554, 2770363, 56.94, 3, 'ENSG00000155008', 'Q6UXV4'),
(112, 'APP', 'This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]', '21', 'protein_coding', 34513142, 34615113, 44.23, 14, 'ENSG00000142192', 'P05067'),
(113, 'APPL2', 'This gene encodes amyloid precursor- like protein 2 (APLP2), which is a member of the APP (amyloid precursor protein) family including APP, APLP1 and APLP2. This protein is ubiquitously expressed. It contains heparin-, copper- and zinc- binding domains at the N-terminus, BPTI/Kunitz inhibitor and E2 domains in the middle region, and transmembrane and intracellular domains at the C-terminus. This protein interacts with major histocompatibility complex (MHC) class I molecules. The synergy of this protein and the APP is required to mediate neuromuscular transmission, spatial learning and synaptic plasticity. This protein has been implicated in the pathogenesis of Alzheimer\'s disease. Multiple alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Aug 2011]', '11', 'miRNA', 101059755, 101059838, 40.48, 1, 'ENSG00000136044', 'Q06481'),
(114, 'AQP1', 'This gene encodes a small integral membrane protein with six bilayer spanning domains that functions as a water channel protein. This protein permits passive transport of water along an osmotic gradient. This gene is a possible candidate for disorders involving imbalance in ocular fluid movement. [provided by RefSeq, Aug 2016]', '7', 'protein_coding', 63657810, 63696253, 59.63, 21, 'ENSG00000240583', 'P29972'),
(115, 'ARAF', 'Enables protein serine/threonine kinase activity. Involved in negative regulation of apoptotic process; regulation of TOR signaling; and regulation of protein metabolic process. Predicted to be active in cytosol and mitochondrion. Biomarker of high grade glioma. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 70902326, 71022449, 40.16, 5, 'ENSG00000078061', 'P10398'),
(116, 'AREL1', 'Enables ubiquitin protein ligase activity. Involved in negative regulation of apoptotic process; protein polyubiquitination; and ubiquitin-dependent protein catabolic process. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 100644195, 100675788, 43.59, 11, 'ENSG00000119682', 'O15033'),
(117, 'ARFGAP3', 'The protein encoded by this gene is a GTPase-activating protein (GAP) that associates with the Golgi apparatus and regulates the early secretory pathway of proteins. The encoded protein promotes hydrolysis of ADP-ribosylation factor 1 (ARF1)-bound GTP, which is required for the dissociation of coat proteins from Golgi-derived membranes and vesicles. Dissociation of the coat proteins is a prerequisite for the fusion of these vesicles with target compartments. The activity of this protein is sensitive to phospholipids. Multiple transcript variants encoding different isoforms have been found for this gene. This gene was originally known as ARFGAP1, but that is now the name of a related but different gene. [provided by RefSeq, Nov 2008]', '22', 'protein_coding', 140293693, 140404433, 45.22, 28, 'ENSG00000242247', 'Q9NP61'),
(118, 'ARG2', ' Arginase catalyzes the hydrolysis of arginine to ornithine and urea.  At least two isoforms of mammalian arginase exists (types I and II) which differ in their tissue distribution, subcellular localization, immunologic crossreactivity and physiologic function.  The type II isoform encoded by this gene, is located in the mitochondria and expressed in extra-hepatic tissues, especially kidney.  The physiologic role of this isoform is poorly understood; it is thought to play a role in nitric oxide and polyamine metabolism.  Transcript variants of the type II gene resulting from the use of alternative polyadenylation sites have been described. [provided by RefSeq, Jul 2008]', '14', 'miRNA', 174423795, 174423864, 48.57, 1, 'ENSG00000081181', 'P78540'),
(119, 'ARHGAP19-SLIT1', 'This locus represents naturally occurring read-through transcription between the neighboring Rho GTPase activating protein 19 (ARHGAP19) and slit homolog 1 (SLIT1) genes on chromosome 10. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is thus unlikely to produce a protein product. [provided by RefSeq, Feb 2011]', '10', 'protein_coding', 3247481, 3256236, 38.81, 12, 'ENSG00000269891', ''),
(120, 'ARHGAP23', 'The RHO (see ARHA; MIM 165390) family of small GTPases are involved in signal transduction through transmembrane receptors, and they are inactive in the GDP-bound form and active in the GTP-bound form. GTPase-activating proteins, such as ARHGAP23, inactivate RHO family proteins by stimulating their hydrolysis of GTP (Katoh and Katoh, 2004 [PubMed 15254754]).[supplied by OMIM, Mar 2008]', '17', 'protein_coding', 71254814, 71301522, 45.49, 40, 'ENSG00000273780', 'Q9P227'),
(121, 'ARHGAP23', 'The RHO (see ARHA; MIM 165390) family of small GTPases are involved in signal transduction through transmembrane receptors, and they are inactive in the GDP-bound form and active in the GTP-bound form. GTPase-activating proteins, such as ARHGAP23, inactivate RHO family proteins by stimulating their hydrolysis of GTP (Katoh and Katoh, 2004 [PubMed 15254754]).[supplied by OMIM, Mar 2008]', '17', 'protein_coding', 71254814, 71301522, 45.49, 40, 'ENSG00000275832', 'Q9P227'),
(122, 'ARHGAP29', 'Rap1 is a small GTPase that, through effectors, regulates Rho GTPase signaling. These effectors- Rasip1, Radil, and the protein encoded by this gene- translocate to the cell membrane, where they form a multiprotein complex. This complex is necessary for Rap1-induced inhibition of Rho signaling. Defects in this gene may be a cause of nonsyndromic cleft lip with or without cleft palate. [provided by RefSeq, Jun 2016]', '1', 'unprocessed_pseudogene', 55714799, 55715727, 45.32, 1, 'ENSG00000137962', 'Q52LW3'),
(123, 'ARHGAP36', 'Predicted to enable GTPase activator activity. Predicted to be involved in signal transduction. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 64340204, 64357534, 61.62, 10, 'ENSG00000147256', 'Q6ZRI8'),
(124, 'ARHGAP40', 'Predicted to enable GTPase activator activity. Predicted to be involved in regulation of actin filament polymerization and regulation of small GTPase mediated signal transduction. Predicted to be located in cytosol. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 25573980, 25574044, 40.00, 1, 'ENSG00000124143', 'Q5TG30'),
(125, 'ARHGEF10', 'This gene encodes a Rho guanine nucleotide exchange factor (GEF). Rho GEFs regulate the activity of small Rho GTPases by stimulating the exchange of guanine diphosphate (GDP) for guanine triphosphate (GTP) and may play a role in neural morphogenesis. Mutations in this gene are associated with slowed nerve conduction velocity (SNCV). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]', '8', 'miRNA', 32350109, 32350202, 65.96, 1, 'ENSG00000104728', 'O15013'),
(126, 'ARHGEF10', 'This gene encodes a Rho guanine nucleotide exchange factor (GEF). Rho GEFs regulate the activity of small Rho GTPases by stimulating the exchange of guanine diphosphate (GDP) for guanine triphosphate (GTP) and may play a role in neural morphogenesis. Mutations in this gene are associated with slowed nerve conduction velocity (SNCV). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]', '8', 'miRNA', 32350109, 32350202, 65.96, 1, 'ENSG00000274726', 'O15013'),
(127, 'ARHGEF17', 'Enables guanyl-nucleotide exchange factor activity. Acts upstream of or within actin cytoskeleton organization. Predicted to be located in cytosol. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 309372, 318565, 50.76, 6, 'ENSG00000110237', 'Q96PE2'),
(128, 'ARHGEF5', 'Rho GTPases play a fundamental role in numerous cellular processes initiated by extracellular stimuli that work through G protein coupled receptors. The encoded protein may form a complex with G proteins and stimulate Rho-dependent signals. This protein may be involved in the control of cytoskeletal organization. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 67482571, 67483547, 61.11, 1, 'ENSG00000050327', 'Q12774'),
(129, 'ARHGEF5', 'Rho GTPases play a fundamental role in numerous cellular processes initiated by extracellular stimuli that work through G protein coupled receptors. The encoded protein may form a complex with G proteins and stimulate Rho-dependent signals. This protein may be involved in the control of cytoskeletal organization. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 67482571, 67483547, 61.11, 1, 'ENSG00000284933', 'Q12774'),
(130, 'ARHGEF7', 'This gene encodes a protein that belongs to a family of cytoplasmic proteins that activate the Ras-like family of Rho proteins by exchanging bound GDP for GTP. It forms a complex with the small GTP binding protein Rac1 and recruits Rac1 to membrane ruffles and to focal adhesions. Multiple alternatively spliced transcript variants encoding different isoforms have been observed for this gene. [provided by RefSeq, Mar 2016]', '13', 'protein_coding', 35172072, 35216240, 42.85, 3, 'ENSG00000102606', 'Q14155'),
(131, 'ARL13A', 'Predicted to enable GTP binding activity and GTPase activity. Predicted to be involved in non-motile cilium assembly and receptor localization to non-motile cilium. Predicted to be active in ciliary membrane; motile cilium; and non-motile cilium. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 97153045, 97292637, 45.99, 2, 'ENSG00000174225', 'Q5H913'),
(132, 'ARL2', 'This gene encodes a small GTP-binding protein of the RAS superfamily which functions as an ADP-ribosylation factor (ARF). The encoded protein is one of a functionally distinct group of ARF-like genes. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 31115087, 31120446, 52.48, 1, 'ENSG00000213465', 'P36404'),
(133, 'ARL5C', 'Predicted to enable GTP binding activity. Predicted to be involved in intracellular protein transport; protein localization to Golgi membrane; and vesicle-mediated transport. Predicted to be active in trans-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 48904110, 48931840, 47.98, 5, 'ENSG00000141748', 'A6NH57'),
(134, 'ARMC3', 'Armadillo/beta-catenin (CTNNB1; MIM 116806)-like (ARM) domains are imperfect 45-amino acid repeats involved in protein-protein interactions. ARM domain-containing proteins, such as ARMC3, function in signal transduction, development, cell adhesion and mobility, and tumor initiation and metastasis (Li et al., 2006 [PubMed 16915934]).[supplied by OMIM, Mar 2008]', '10', 'protein_coding', 27545913, 27620529, 37.61, 2, 'ENSG00000165309', 'Q5W041'),
(135, 'ARMC5', 'This gene encodes a member of the ARM (armadillo/beta-catenin-like repeat) superfamily. The ARM repeat is a tandemly repeated sequence motif with approximately 40 amino acid long. This repeat is implicated in mediating protein-protein interactions. The encoded protein contains seven ARM repeats. Mutations in this gene are associated with primary bilateral macronodular adrenal hyperplasia, which is also known as ACTH-independent macronodular adrenal hyperplasia 2. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2014]', '16', 'miRNA', 133051996, 133052059, 64.06, 1, 'ENSG00000140691', 'Q96C12'),
(136, 'ARPC5L', 'Predicted to enable actin filament binding activity. Predicted to be involved in Arp2/3 complex-mediated actin nucleation and cell migration. Located in extracellular exosome and focal adhesion. [provided by Alliance of Genome Resources, May 2025]', '9', 'miRNA', 41233755, 41233835, 41.98, 1, 'ENSG00000136950', 'Q9BPX5'),
(137, 'ARR3', 'The protein encoded by this gene is a non-visual arrestin which binds to agonist-activated, phosphorylated G protein-coupled receptors. This binding uncouples the receptor from the heterotrimeric G protein, resulting in termination of the G protein-coupled receptor signaling. The encoded protein also is a part of the centrosome, interacting with gamma-tubulin to help regulate proper centrosome function. [provided by RefSeq, May 2016]', 'X', 'protein_coding', 64549920, 64589381, 42.72, 6, 'ENSG00000120500', 'P36575'),
(138, 'ASAP2', 'This gene encodes a multidomain protein containing an N-terminal alpha-helical region with a coiled-coil motif, followed by a pleckstrin homology (PH) domain, an Arf-GAP domain, an ankyrin homology region, a proline-rich region, and a C-terminal Src homology 3 (SH3) domain. The protein localizes in the Golgi apparatus and at the plasma membrane, where it colocalizes with protein tyrosine kinase 2-beta (PYK2). The encoded protein forms a stable complex with PYK2 in vivo. This interaction appears to be mediated by binding of its SH3 domain to the C-terminal proline-rich domain of PYK2. The encoded protein is tyrosine phosphorylated by activated PYK2. It has catalytic activity for class I and II ArfGAPs in vitro, and can bind the class III Arf ARF6 without immediate GAP activity. The encoded protein is believed to function as an ARF GAP that controls ARF-mediated vesicle budding when recruited to Golgi membranes. In addition, it functions as a substrate and downstream target for PYK2 and SRC, a pathway that may be involved in the regulation of vesicular transport. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2008]', '2', 'protein_coding', 125442881, 125592568, 42.92, 17, 'ENSG00000151693', 'O43150'),
(139, 'ASB13', 'The protein encoded by this gene is a member of the ankyrin repeat and SOCS box-containing (ASB) family of proteins. They contain ankyrin repeat sequence and a SOCS box domain. The SOCS box serves to couple suppressor of cytokine signalling (SOCS) proteins and their binding partners with the elongin B and C complex, possibly targeting them for degradation. Multiple alternatively spliced transcript variants, both protein-coding and not protein-coding, have been described for this gene. [provided by RefSeq, Nov 2010]', '10', 'protein_coding', 21090077, 21104722, 55.48, 13, 'ENSG00000196372', 'Q8WXK3'),
(140, 'ASB2', 'This gene encodes a member of the ankyrin repeat and SOCS box-containing (ASB) protein family. These proteins play a role in protein degradation by coupling suppressor of cytokine signalling (SOCS) proteins with the elongin BC complex. The encoded protein is a subunit of a multimeric E3 ubiquitin ligase complex that mediates the degradation of actin-binding proteins. This gene plays a role in retinoic acid-induced growth inhibition and differentiation of myeloid leukemia cells. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jan 2011]', '14', 'protein_coding', 167028, 196915, 47.58, 10, 'ENSG00000278693', 'Q96Q27'),
(141, 'ASB2', 'This gene encodes a member of the ankyrin repeat and SOCS box-containing (ASB) protein family. These proteins play a role in protein degradation by coupling suppressor of cytokine signalling (SOCS) proteins with the elongin BC complex. The encoded protein is a subunit of a multimeric E3 ubiquitin ligase complex that mediates the degradation of actin-binding proteins. This gene plays a role in retinoic acid-induced growth inhibition and differentiation of myeloid leukemia cells. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jan 2011]', '14', 'protein_coding', 167028, 196915, 47.58, 10, 'ENSG00000100628', 'Q96Q27'),
(142, 'ASB5', 'The protein encoded by this gene is a member of the ankyrin repeat and SOCS box-containing (ASB) family of proteins. They contain ankyrin repeat sequence and SOCS box domain. The SOCS box serves to couple suppressor of cytokine signalling (SOCS) proteins and their binding partners with the elongin B and C complex, possibly targeting them for degradation. Multiple alternatively spliced transcript variants have been described for this gene but their full length sequences are not known. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 93885199, 93909456, 36.37, 4, 'ENSG00000164122', 'Q8WWX0'),
(143, 'ASCC2', 'Enables K63-linked polyubiquitin modification-dependent protein binding activity. Involved in regulation of DNA-templated transcription; rescue of stalled ribosome; and ribosome-associated ubiquitin-dependent protein catabolic process. Located in nucleus. Part of DNA repair complex and RQC-trigger complex. Is active in cytosolic ribosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 9653, 28220, 52.59, 7, 'ENSG00000100325', 'Q9H1I8'),
(144, 'ASMT', 'This gene belongs to the methyltransferase superfamily, and is located in the pseudoautosomal region (PAR) at the end of the short arms of the X and Y chromosomes. The encoded enzyme catalyzes the final reaction in the synthesis of melatonin, and is abundant in the pineal gland. Alternatively spliced transcript variants have been noted for this gene. [provided by RefSeq, Jan 2010]', 'X, Y', 'snRNA', 492015, 492178, 54.88, 1, 'ENSG00000292336', 'P46597'),
(145, 'ASMT', 'This gene belongs to the methyltransferase superfamily, and is located in the pseudoautosomal region (PAR) at the end of the short arms of the X and Y chromosomes. The encoded enzyme catalyzes the final reaction in the synthesis of melatonin, and is abundant in the pineal gland. Alternatively spliced transcript variants have been noted for this gene. [provided by RefSeq, Jan 2010]', 'X, Y', 'snRNA', 492015, 492178, 54.88, 1, 'ENSG00000196433', 'P46597'),
(146, 'ASNS', 'The protein encoded by this gene is involved in the synthesis of asparagine. This gene complements a mutation in the temperature-sensitive hamster mutant ts11, which blocks progression through the G1 phase of the cell cycle at nonpermissive temperature. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, May 2010]', '7', 'protein_coding', 15345596, 15384413, 39.31, 2, 'ENSG00000070669', 'P08243'),
(147, 'ASTN2', 'This gene encodes a protein that is expressed in the brain and may function in neuronal migration, based on functional studies of the related astrotactin 1 gene in human and mouse. A deletion at this locus has been associated with schizophrenia. Multiple transcript variants encoding different proteins have been found for this locus. [provided by RefSeq, May 2010]', '9', 'protein_coding', 33161768, 33181412, 47.21, 2, 'ENSG00000148219', 'O75129'),
(148, 'ATG16L1', 'The protein encoded by this gene is part of a large protein complex that is necessary for autophagy, the major process by which intracellular components are targeted to lysosomes for degradation. Defects in this gene are a cause of susceptibility to inflammatory bowel disease type 10 (IBD10). Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jun 2010]', '2', 'protein_coding', 49145092, 49208638, 35.66, 13, 'ENSG00000281089', 'Q676U5'),
(149, 'ATG16L1', 'The protein encoded by this gene is part of a large protein complex that is necessary for autophagy, the major process by which intracellular components are targeted to lysosomes for degradation. Defects in this gene are a cause of susceptibility to inflammatory bowel disease type 10 (IBD10). Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jun 2010]', '2', 'protein_coding', 49145092, 49208638, 35.66, 13, 'ENSG00000085978', 'Q676U5'),
(150, 'ATL3', 'This gene encodes a member of a family of dynamin-like, integral membrane GTPases. The encoded protein is required for the proper formation of the network of interconnected tubules of the endoplasmic reticulum. Mutations in this gene may be associated with hereditary sensory neuropathy type IF. Alternatively spliced transcript variants that encode distinct isoforms have been described. [provided by RefSeq, Feb 2014]', '11', 'protein_coding', 54920462, 54931431, 51.41, 25, 'ENSG00000184743', 'Q6DD88'),
(151, 'ATN1', 'Dentatorubral pallidoluysian atrophy (DRPLA) is a rare neurodegenerative disorder characterized by cerebellar ataxia, myoclonic epilepsy, choreoathetosis, and dementia. The disorder is related to the expansion from 7-35 copies to 49-93 copies of a trinucleotide repeat (CAG/CAA) within this gene. The encoded protein includes a serine repeat and a region of alternating acidic and basic amino acids, as well as the variable glutamine repeat. Alternative splicing results in two transcripts variants that encode the same protein. [provided by RefSeq, Jul 2016]', '12', 'protein_coding', 56919602, 56971675, 38.91, 11, 'ENSG00000111676', 'P54259'),
(152, 'ATOH7', 'This intronless gene encodes a member of the basic helix-loop-helix family of transcription factors, with similarity to Drosophila atonal gene that controls photoreceptor development. Studies in mice suggest that this gene plays a central role in retinal ganglion cell and optic nerve formation. Mutations in this gene are associated with nonsyndromic congenital retinal nonattachment. [provided by RefSeq, Dec 2011]', '10', 'protein_coding', 37367960, 37427479, 58.59, 6, 'ENSG00000179774', 'Q8N100'),
(153, 'ATP10D', 'Enables glycosylceramide flippase activity. Predicted to be involved in phospholipid translocation. Located in endoplasmic reticulum; nucleoplasm; and plasma membrane. Part of phospholipid-translocating ATPase complex. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 29767369, 29770413, 47.62, 2, 'ENSG00000145246', 'Q9P241'),
(154, 'ATP11A', 'The protein encoded by this gene is an integral membrane ATPase. The encoded protein is probably phosphorylated in its intermediate state and likely drives the transport of ions such as calcium across membranes. [provided by RefSeq, Apr 2022]', '13', 'miRNA', 444467, 444543, 59.74, 1, 'ENSG00000068650', 'P98196'),
(155, 'ATP13A4', 'Predicted to enable ATPase-coupled monoatomic cation transmembrane transporter activity and polyamine transmembrane transporter activity. Predicted to be involved in intracellular calcium ion homeostasis and polyamine transmembrane transport. Predicted to be located in early endosome membrane and recycling endosome membrane. Predicted to be active in endoplasmic reticulum membrane and late endosome membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 140107777, 140125619, 41.49, 5, 'ENSG00000127249', 'Q4VNC1'),
(156, 'ATP23', 'The protein encoded by this gene is amplified in glioblastomas and interacts with the DNA binding subunit of DNA-dependent protein kinase. This kinase is involved in double-strand break repair (DSB), and higher expression of the encoded protein increases the efficiency of DSB. In addition, comparison to orthologous proteins strongly suggests that this protein is a metalloprotease important in the biosynthesis of mitochondrial ATPase. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 141589708, 141590762, 49.48, 1, 'ENSG00000262145', 'Q9Y6H3'),
(157, 'ATP23', 'The protein encoded by this gene is amplified in glioblastomas and interacts with the DNA binding subunit of DNA-dependent protein kinase. This kinase is involved in double-strand break repair (DSB), and higher expression of the encoded protein increases the efficiency of DSB. In addition, comparison to orthologous proteins strongly suggests that this protein is a metalloprotease important in the biosynthesis of mitochondrial ATPase. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 141589708, 141590762, 49.48, 1, 'ENSG00000166896', 'Q9Y6H3'),
(158, 'ATP4B', 'The protein encoded by this gene belongs to a family of P-type cation-transporting ATPases. The gastric H+, K+-ATPase is a heterodimer consisting of a high molecular weight catalytic alpha subunit and a smaller but heavily glycosylated beta subunit. This enzyme is a proton pump that catalyzes the hydrolysis of ATP coupled with the exchange of H(+) and K(+) ions across the plasma membrane. It is also responsible for gastric acid secretion. This gene encodes the beta subunit of the gastric H+, K+-ATPase. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 48171458, 48647497, 38.84, 8, 'ENSG00000186009', 'P51164'),
(159, 'ATP5F1A', 'This gene encodes a subunit of mitochondrial ATP synthase. Mitochondrial ATP synthase catalyzes ATP synthesis, using an electrochemical gradient of protons across the inner membrane during oxidative phosphorylation. ATP synthase is composed of two linked multi-subunit complexes: the soluble catalytic core, F1, and the membrane-spanning component, Fo, comprising the proton channel. The catalytic portion of mitochondrial ATP synthase consists of 5 different subunits (alpha, beta, gamma, delta, and epsilon) assembled with a stoichiometry of 3 alpha, 3 beta, and a single representative of the other 3. The proton channel consists of three main subunits (a, b, c). This gene encodes the alpha subunit of the catalytic core. Alternatively spliced transcript variants encoding the different isoforms have been identified. Pseudogenes of this gene are located on chromosomes 9, 2, and 16. [provided by RefSeq, Mar 2012]', '18', 'protein_coding', 431662, 433430, 40.42, 1, 'ENSG00000152234', 'P25705'),
(160, 'ATP5F1EP2', 'Predicted to enable hydrolase activity. Predicted to contribute to proton-transporting ATP synthase activity, rotational mechanism. Predicted to be involved in proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 81491439, 81752774, 36.03, 5, 'ENSG00000180389', 'Q5VTU8'),
(161, 'ATP6V1D', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of eukaryotic intracellular organelles. V-ATPase dependent organelle acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\", and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This gene encodes the V1 domain D subunit protein. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 65722205, 65871725, 40.37, 10, 'ENSG00000100554', 'Q9Y5K8'),
(162, 'ATP6V1E1', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of eukaryotic intracellular organelles. V-ATPase dependent organelle acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A, three B, and two G subunits, as well as a C, D, E, F, and H subunit. The V1 domain contains the ATP catalytic site. This gene encodes alternate transcriptional splice variants, encoding different V1 domain E subunit isoforms. Pseudogenes for this gene have been found in the genome. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 63521401, 63625128, 38.33, 2, 'ENSG00000131100', 'P36543'),
(163, 'ATP6V1E2', 'Predicted to enable proton-transporting ATPase activity, rotational mechanism. Predicted to be involved in regulation of macroautophagy. Predicted to act upstream of or within proton transmembrane transport. Predicted to be located in cytosol. Predicted to be part of proton-transporting two-sector ATPase complex, catalytic domain. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 1871393, 1982207, 47.15, 31, 'ENSG00000250565', 'Q96A05'),
(164, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000213760', 'O95670'),
(165, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000230900', 'O95670'),
(166, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000234668', 'O95670'),
(167, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000227587', 'O95670'),
(168, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000206445', 'O95670');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(169, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000226850', 'O95670'),
(170, 'ATP6V1G2', 'This gene encodes a component of vacuolar ATPase (V-ATPase), a multisubunit enzyme that mediates acidification of intracellular compartments of eukaryotic cells. V-ATPase dependent acidification is necessary for such intracellular processes as protein sorting, zymogen activation, receptor-mediated endocytosis, and synaptic vesicle proton gradient generation. V-ATPase is composed of a cytosolic V1 domain and a transmembrane V0 domain. The V1 domain consists of three A and three B subunits, two G subunits plus the C, D, E, F, and H subunits. The V1 domain contains the ATP catalytic site. The V0 domain consists of five different subunits: a, c, c\', c\'\', and d. Additional isoforms of many of the V1 and V0 subunit proteins are encoded by multiple genes or alternatively spliced transcript variants. This encoded protein is one of three V1 domain G subunit proteins. This gene had previous gene symbols of ATP6G and ATP6G2. Alternatively spliced transcript variants encoding different isoforms have been described. Read-through transcription also exists between this gene and the downstream DEAD (Asp-Glu-Ala-Asp) box polypeptide 39B (DDX39B) gene. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 268530, 280828, 51.50, 4, 'ENSG00000234920', 'O95670'),
(171, 'ATP6V1G2-DDX39B', 'This locus represents naturally occurring read-through transcription between the neighboring ATP6V1G2 (ATPase, H+ transporting, lysosomal 13kDa, V1 subunit G2) and DDX39B (DEAD box polypeptide 39B) genes located in the major histocompatibility complex class III region of chromosome 6. The read-through transcript and is a candidate for nonsense-mediated mRNA decay (NMD), and is thus unlikely to produce a protein product. [provided by RefSeq, Feb 2011]', '6', 'protein_coding', 109850945, 109880541, 49.11, 5, 'ENSG00000254870', 'F2Z307'),
(172, 'ATP7A', 'This gene encodes a transmembrane protein that functions in copper transport across membranes. This protein is localized to the trans Golgi network, where it is predicted to supply copper to copper-dependent enzymes in the secretory pathway. It relocalizes to the plasma membrane under conditions of elevated extracellular copper, and functions in the efflux of copper from cells. Mutations in this gene are associated with Menkes disease, X-linked distal spinal muscular atrophy, and occipital horn syndrome. Alternatively-spliced transcript variants have been observed. [provided by RefSeq, Aug 2013]', 'X', 'processed_pseudogene', 111889199, 111889474, 45.65, 1, 'ENSG00000165240', 'Q04656'),
(173, 'ATP7B', 'This gene is a member of the P-type cation transport ATPase family and encodes a protein with several membrane-spanning domains, an ATPase consensus sequence, a hinge domain, a phosphorylation site, and at least 2 putative copper-binding sites. This protein is a monomer, and functions as a copper-transporting ATPase which exports copper out of the cells, such as the efflux of hepatic copper into the bile. Alternate transcriptional splice variants, encoding different isoforms with distinct cellular localizations, have been characterized. Mutations in this gene have been associated with Wilson disease which is characterized by copper accumulation. [provided by RefSeq, Dec 2019]', '13', 'protein_coding', 57372477, 57385044, 53.75, 4, 'ENSG00000123191', 'P35670'),
(174, 'ATP9A', 'Enables protease binding activity. Involved in negative regulation of exosomal secretion; regulation of endocytic recycling; and regulation of retrograde transport, endosome to Golgi. Located in several cellular components, including endosome; perinuclear region of cytoplasm; and trans-Golgi network membrane. Implicated in neurodevelopmental disorder with poor growth and behavioral abnormalities. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 47380496, 47380565, 48.57, 1, 'ENSG00000054793', 'O75110'),
(175, 'ATR', 'The protein encoded by this gene is a serine/threonine kinase and DNA damage sensor, activating cell cycle checkpoint signaling upon DNA stress. The encoded protein can phosphorylate and activate several proteins involved in the inhibition of DNA replication and mitosis, and can promote DNA repair, recombination, and apoptosis. This protein is also important for fragile site stability and centrosome duplication. Defects in this gene are a cause of Seckel syndrome 1. [provided by RefSeq, Aug 2017]', '3', 'protein_coding', 133309410, 133352529, 56.13, 1, 'ENSG00000175054', 'P20848'),
(176, 'AUP1', 'The protein encoded this gene is involved in several pathways including quality control of misfolded proteins in the endoplasmic reticulum and lipid droplet accumulation. Lipid droplets are organelles in the cytoplasm that store neutral lipids such as cholesterol esters and trigylycerides to prevent the overabundance of free cholesterol and fatty acids in cells, but also to act as storage for other metabolic processes, such as membrane biogenesis. Reduced expression of this gene results in reduced lipid droplet clustering, a function that is dependent on ubiquitination of the protein. This protein contains multiple domains including a hydrophobic N-terminal domain, an acetyltranferase domain, a ubiquitin-binding CUE domain, and a UBE2B2-binding domain (G2BR). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]', '2', 'protein_coding', 11882622, 11908366, 45.05, 30, 'ENSG00000115307', 'Q9Y679'),
(177, 'AURKB', 'This gene encodes a member of the aurora kinase subfamily of serine/threonine kinases. The genes encoding the other two members of this subfamily are located on chromosomes 19 and 20. These kinases participate in the regulation of alignment and segregation of chromosomes during mitosis and meiosis through association with microtubules. A pseudogene of this gene is located on chromosome 8. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2015]', '17', 'miRNA', 147344629, 147344717, 29.21, 1, 'ENSG00000178999', 'Q96GD4'),
(178, 'AVPR2', 'This gene encodes the vasopressin receptor, type 2, also known as the V2 receptor, which belongs to the seven-transmembrane-domain G protein-coupled receptor (GPCR) superfamily, and couples to Gs thus stimulating adenylate cyclase. The subfamily that includes the V2 receptor, the V1a and V1b vasopressin receptors, the oxytocin receptor, and isotocin and mesotocin receptors in non-mammals, is well conserved, though several members signal via other G proteins. All bind similar cyclic nonapeptide hormones. The V2 receptor is expressed in the kidney tubule, predominantly in the distal convoluted tubule and collecting ducts, where its primary property is to respond to the pituitary hormone arginine vasopressin (AVP) by stimulating mechanisms that concentrate the urine and maintain water homeostasis in the organism. When the function of this gene is lost, the disease Nephrogenic Diabetes Insipidus (NDI) results. The V2 receptor is also expressed outside the kidney although its tissue localization is uncertain. When these \'extrarenal receptors\' are stimulated by infusion of a V2 selective agonist (dDAVP), a variety of clotting factors are released into the bloodstream. The physiologic importance of this property is not known - its absence does not appear to be detrimental in NDI patients. The gene expression has also been described in fetal lung tissue and lung cancer associated with alternative splicing. [provided by RefSeq, Jul 2008]', 'X', 'miRNA', 86542482, 86542552, 50.70, 1, 'ENSG00000126895', 'P30518'),
(179, 'AWAT1', 'The protein encoded by this gene belongs to the diacylglycerol acyltransferase family. It esterifies long chain (wax) alcohols with acyl-CoA-derived fatty acids to produce wax esters. Wax esters are enriched in sebum, suggesting that this enzyme plays a central role in lipid metabolism in skin. Consistent with this observation, this protein is predominantly expressed in the sebaceous gland of the skin. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 36147892, 36147988, 29.90, 1, 'ENSG00000204195', 'Q58HT5'),
(180, 'AXL', 'The protein encoded by this gene is a member of the Tyro3-Axl-Mer (TAM) receptor tyrosine kinase subfamily. The encoded protein possesses an extracellular domain which is composed of two immunoglobulin-like motifs at the N-terminal, followed by two fibronectin type-III motifs. It transduces signals from the extracellular matrix into the cytoplasm by binding to the vitamin K-dependent protein growth arrest-specific 6 (Gas6). This gene may be involved in several cellular functions including growth, migration, aggregation and anti-inflammation in multiple cell types. The encoded protein acts as a host cell receptor for multiple viruses, including Marburg, Ebola and Lassa viruses and is a candidate receptor for the SARS-CoV2 virus. [provided by RefSeq, Sep 2021]', '19', 'protein_coding', 55148112, 55191608, 39.09, 5, 'ENSG00000167601', 'P30530'),
(181, 'AZI2', 'The protein encoded by this gene belongs to the antizyme inhibitor family, which plays a role in cell growth and proliferation by maintaining polyamine homeostasis within the cell. Antizyme inhibitors are homologs of ornithine decarboxylase (ODC, the key enzyme in polyamine biosynthesis) that have lost the ability to decarboxylase ornithine; however, retain the ability to bind to antizymes. Antizymes negatively regulate intracellular polyamine levels by binding to ODC and targeting it for degradation, as well as by inhibiting polyamine uptake. Antizyme inhibitors function as positive regulators of polyamine levels by sequestering antizymes and neutralizing their effect. This gene encodes antizyme inhibitor 2, the second member of this gene family. Like antizyme inhibitor 1, antizyme inhibitor 2 interacts with all 3 antizymes and stimulates ODC activity and polyamine uptake. However, unlike antizyme inhibitor 1, which is ubiquitously expressed and localized in the nucleus and cytoplasm, antizyme inhibitor 2 is predominantly expressed in the brain and testis and localized in the endoplasmic reticulum-golgi intermediate compartment. Recent studies indicate that antizyme inhibitor 2 is also expressed in specific cell types in ovaries, adrenal glands and pancreas, and in mast cells. The exact function of this gene is not known, however, available data suggest its role in cell growth, spermiogenesis, vesicular trafficking and secretion. Accumulation of antizyme inhibitor 2 has also been observed in brains of patients with Alzheimer\'s disease. There has been confusion in literature and databases over the nomenclature of this gene, stemming from an earlier report that a human cDNA clone (identical to ODCp/AZIN2) had arginine decarboxylase (ADC) activity (PMID:14738999). Subsequent studies in human and mouse showed that antizyme inhibitor 2 was devoid of arginine decarboxylase activity (PMID:19956990). Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Sep 2014]', '1', 'protein_coding', 154474972, 154478227, 55.99, 1, 'ENSG00000163512', 'Q9H6S1'),
(182, 'B3GALNT2', 'This gene encodes a member of the glycosyltransferase 31 family. The encoded protein synthesizes GalNAc:beta-1,3GlcNAc, a novel carbohydrate structure, on N- and O-glycans. Alternatively spliced transcript variants that encode different isoforms have been described. [provided by RefSeq, Mar 2013]', '1', 'protein_coding', 1493319, 1507265, 51.83, 14, 'ENSG00000282880', 'Q8NCR0'),
(183, 'B3GALNT2', 'This gene encodes a member of the glycosyltransferase 31 family. The encoded protein synthesizes GalNAc:beta-1,3GlcNAc, a novel carbohydrate structure, on N- and O-glycans. Alternatively spliced transcript variants that encode different isoforms have been described. [provided by RefSeq, Mar 2013]', '1', 'protein_coding', 1493319, 1507265, 51.83, 14, 'ENSG00000162885', 'Q8NCR0'),
(184, 'B3GNT2', 'This gene encodes a member of the beta-1,3-N-acetylglucosaminyltransferase family. This enzyme is a type II transmembrane protein. It prefers the substrate of lacto-N-neotetraose, and is involved in the biosynthesis of poly-N-acetyllactosamine chains. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jan 2016]', '2', 'protein_coding', 90392257, 90409766, 33.62, 6, 'ENSG00000170340', 'Q9NY97'),
(185, 'B3GNT4', 'This gene encodes a member of the beta-1,3-N-acetylglucosaminyltransferase protein family. The encoded enzyme is involved in the biosynthesis of poly-N-acetyllactosamine chains and prefers lacto-N-neotetraose as a substrate. It is a type II transmembrane protein. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 1576739, 1743341, 38.61, 8, 'ENSG00000176383', 'Q9C0J1'),
(186, 'B3GNT5', 'This gene encodes a member of the beta-1,3-N-acetylglucosaminyltransferase family. This enzyme is a type II membrane protein. It exhibits strong activity to transfer GlcNAc to glycolipid substrates and is identified as the most likely candidate for lactotriaosylceramide synthase. This enzyme is essential for the expression of Lewis X epitopes on glycolipids. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 42696598, 42722597, 49.25, 1, 'ENSG00000176597', 'Q9BYG0'),
(187, 'B4GALNT1', 'GM2 and GD2 gangliosides are sialic acid-containing glycosphingolipids. GalNAc-T is the enzyme involved in the biosynthesis of G(M2) and G(D2) glycosphingolipids. GalNAc-T catalyzes the transfer of GalNAc into G(M3) and G(D3) by a beta-1,4 linkage, resulting in the synthesis of G(M2) and G(D2), respectively. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2013]', '12', 'protein_coding', 133230217, 133231558, 77.05, 1, 'ENSG00000135454', 'Q00973'),
(188, 'B4GALNT3', 'B4GALNT3 transfers N-acetylgalactosamine (GalNAc) onto glucosyl residues to form N,N-prime-diacetyllactosediamine (LacdiNAc, or LDN), a unique terminal structure of cell surface N-glycans (Ikehara et al., 2006 [PubMed 16728562]).[supplied by OMIM, Aug 2008]', '12', 'miRNA', 88745836, 88745905, 57.14, 1, 'ENSG00000139044', 'Q6L9W6'),
(189, 'BACH1', 'The protein encoded by this gene is a member of the RecQ DEAH helicase family and interacts with the BRCT repeats of breast cancer, type 1 (BRCA1). The bound complex is important in the normal double-strand break repair function of breast cancer, type 1 (BRCA1). This gene may be a target of germline cancer-inducing mutations. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 77249848, 77277900, 44.37, 2, 'ENSG00000156273', 'O14867'),
(190, 'BAIAP2L1', 'This gene encodes a member of the IMD (IRSp53/MIM homology domain) family. Members of this family can be subdivided in two groups, the IRSp53-like and MIM-like, based on the presence or absence of the SH3 (Src homology 3) domain. The protein encoded by this gene contains a conserved IMD, also known as F-actin bundling domain, at the N-terminus, and a canonical SH3 domain near the C-terminus, so it belongs to the IRSp53-like group. This protein is the substrate for insulin receptor tyrosine kinase and binds to the small GTPase Rac. It is involved in signal transduction pathways that link deformation of the plasma membrane and remodeling of the actin cytoskeleton. It also promotes actin assembly and membrane protrusions when overexpressed in mammalian cells, and is essential to the formation of a potent actin assembly complex during EHEC (Enterohemorrhagic Escherichia coli) pedestal formation. [provided by RefSeq, Oct 2009]', '7', 'miRNA', 100814491, 100814574, 63.10, 1, 'ENSG00000006453', 'Q9UHR4'),
(191, 'BAMBI', 'This gene encodes a transmembrane glycoprotein related to the type I receptors of the transforming growth factor-beta (TGF-beta) family, whose members play important roles in signal transduction in many developmental and pathological processes. The encoded protein however is a pseudoreceptor, lacking an intracellular serine/threonine kinase domain required for signaling. Similar proteins in frog, mouse and zebrafish function as negative regulators of TGF-beta, which has led to the suggestion that the encoded protein may function to limit the signaling range of the TGF-beta family during early embryogenesis. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 60914407, 60920653, 52.89, 1, 'ENSG00000095739', 'Q13145'),
(192, 'BANF1', 'The protein encoded by this gene was first identified by its ability to protect retroviruses from intramolecular integration and therefore promote intermolecular integration into the host cell genome. The protein forms a homodimer which localizes to both the nucleus and cytoplasm and is specifically associated with chromosomes during mitosis. This protein binds to double stranded DNA in a non-specific manner and also binds to LEM-domain containing proteins of the nuclear envelope. This protein is thought to facilitate nuclear reassembly by binding with both DNA and inner nuclear membrane proteins and thereby recruit chromatin to the nuclear periphery. Alternative splicing results in multiple transcript variants encoding the same protein.[provided by RefSeq, Jan 2009]', '11', 'protein_coding', 18486540, 18497225, 41.62, 3, 'ENSG00000175334', 'O75531'),
(193, 'BANK1', 'The protein encoded by this gene is a B-cell-specific scaffold protein that functions in B-cell receptor-induced calcium mobilization from intracellular stores. This protein can also promote Lyn-mediated tyrosine phosphorylation of inositol 1,4,5-trisphosphate receptors. Polymorphisms in this gene are associated with susceptibility to systemic lupus erythematosus. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]', '4', 'protein_coding', 49660674, 49691486, 45.54, 7, 'ENSG00000153064', 'Q8NDB2'),
(194, 'BATF2', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in defense response to protozoan; myeloid dendritic cell differentiation; and regulation of transcription by RNA polymerase II. Part of RNA polymerase II transcription regulator complex. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 44832, 248502, 38.81, 8, 'ENSG00000168062', 'Q8N1L9'),
(195, 'BBS10', 'This gene is a member of the Bardet-Biedl syndrome (BBS) gene family. Bardet-Biedl syndrome is an autosomal recessive disorder characterized by progressive retinal degeneration, obesity, polydactyly, renal malformation and cognitive disability. The proteins encoded by BBS gene family members are structurally diverse and the similar phenotypes exhibited by mutations in BBS gene family members is likely due to their shared roles in cilia formation and function. Many BBS proteins localize to the basal bodies, ciliary axonemes, and pericentriolar regions of cells. BBS proteins may also be involved in intracellular trafficking via microtubule-related transport. The protein encoded by this gene is likely not a ciliary protein but rather has distant sequence homology to type II chaperonins. As a molecular chaperone, this protein may affect the folding or stability of other ciliary or basal body proteins. Inhibition of this protein\'s expression impairs ciliogenesis in preadipocytes. Mutations in this gene cause Bardet-Biedl syndrome type 10. [provided by RefSeq, Jan 2010]', '12', 'protein_coding', 55399745, 55407788, 54.64, 4, 'ENSG00000179941', 'Q8TAM1'),
(196, 'BBS9', 'This gene is downregulated by parathyroid hormone in osteoblastic cells, and therefore is thought to be involved in parathyroid hormone action in bones. The exact function of this gene has not yet been determined. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jan 2017]', '7', 'protein_coding', 89797655, 89808575, 37.07, 2, 'ENSG00000122507', 'Q3SYG4'),
(197, 'BCAP31', 'This gene encodes a member of the B-cell receptor associated protein 31 superfamily. The encoded protein is a multi-pass transmembrane protein of the endoplasmic reticulum that is involved in the anterograde transport of membrane proteins from the endoplasmic reticulum to the Golgi and in caspase 8-mediated apoptosis. Microdeletions in this gene are associated with contiguous ABCD1/DXS1375E deletion syndrome (CADDS), a neonatal disorder. Alternative splicing of this gene results in multiple transcript variants. Two related pseudogenes have been identified on chromosome 16. [provided by RefSeq, Jan 2012]', 'X', 'protein_coding', 137554444, 137578925, 52.78, 14, 'ENSG00000185825', 'P51572'),
(198, 'BCAR3', 'Breast tumors are initially dependent on estrogens for growth and progression and can be inhibited by anti-estrogens such as tamoxifen. However, breast cancers progress to become anti-estrogen resistant. Breast cancer anti-estrogen resistance gene 3 was identified in the search for genes involved in the development of estrogen resistance. The gene encodes a component of intracellular signal transduction that causes estrogen-independent proliferation in human breast cancer cells. The protein contains a putative src homology 2 (SH2) domain, a hall mark of cellular tyrosine kinase signaling molecules, and is partly homologous to the cell division cycle protein CDC48. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2012]', '1', 'protein_coding', 13621498, 13711835, 37.08, 2, 'ENSG00000137936', 'O75815'),
(199, 'BCAS1', 'This gene resides in a region at 20q13 which is amplified in a variety of tumor types and associated with more aggressive tumor phenotypes. Among the genes identified from this region, it was found to be highly expressed in three amplified breast cancer cell lines and in one breast tumor without amplification at 20q13.2. However, this gene is not in the common region of maximal amplification and its expression was not detected in the breast cancer cell line MCF7, in which this region is highly amplified. Although not consistently expressed, this gene is a candidate oncogene. [provided by RefSeq, Apr 2016]', '20', 'protein_coding', 46033307, 46048180, 45.56, 3, 'ENSG00000064787', 'O75363'),
(200, 'BCHE', 'This gene encodes a cholinesterase enzyme and member of the type-B carboxylesterase/lipase family of proteins. The encoded enzyme exhibits broad substrate specificity and is involved in the detoxification of poisons including organophosphate nerve agents and pesticides, and the metabolism of drugs including cocaine, heroin and aspirin. Humans homozygous for certain mutations in this gene exhibit prolonged apnea after administration of the muscle relaxant succinylcholine. [provided by RefSeq, Jul 2016]', '3', 'miRNA', 34941837, 34941920, 79.76, 1, 'ENSG00000114200', 'P06276'),
(201, 'BCKDHA', 'The branched-chain alpha-keto acid (BCAA) dehydrogenase (BCKD) complex is an innter mitochondrial enzyme complex that catalyzes the second major step in the catabolism of the branched-chain amino acids leucine, isoleucine, and valine. The BCKD complex consists of three catalytic components: a heterotetrameric (alpha2-beta2) branched-chain alpha-keto acid decarboxylase (E1), a dihydrolipoyl transacylase (E2), and a dihydrolipoamide dehydrogenase (E3). This gene encodes the alpha subunit of the decarboxylase (E1) component. Mutations in this gene result in maple syrup urine disease, type IA. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Sep 2009]', '19', 'protein_coding', 150865679, 151101887, 43.50, 12, 'ENSG00000248098', 'P12694'),
(202, 'BCKDHB', 'This gene encodes the E1 beta subunit of branched-chain keto acid dehydrogenase, which is a multienzyme complex associated with the inner membrane of mitochondria. This enzyme complex functions in the catabolism of branched-chain amino acids. Mutations in this gene have been associated with maple syrup urine disease (MSUD), type 1B, a disease characterized by a maple syrup odor to the urine in addition to mental and physical retardation and feeding problems. Alternative splicing at this locus results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 131096476, 131123152, 53.82, 10, 'ENSG00000083123', 'P21953'),
(203, 'BCKDK', 'The branched-chain alpha-ketoacid dehydrogenase complex (BCKD) is an important regulator of the valine, leucine, and isoleucine catabolic pathways. The protein encoded by this gene is found in the mitochondrion, where it phosphorylates and inactivates BCKD. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2012]', '16', 'protein_coding', 31923438, 32173018, 41.21, 37, 'ENSG00000103507', 'O14874'),
(204, 'BCL2', 'This gene encodes an integral outer mitochondrial membrane protein that blocks the apoptotic death of some cells such as lymphocytes. Constitutive expression of BCL2, such as in the case of translocation of BCL2 to Ig heavy chain locus, is thought to be the cause of follicular lymphoma. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '18', 'protein_coding', 91434595, 91438584, 41.28, 2, 'ENSG00000171791', 'P10415'),
(205, 'BCL2L13', 'This gene encodes a mitochondrially-localized protein with conserved B-cell lymphoma 2 homology motifs. Overexpression of the encoded protein results in apoptosis. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Jul 2012]', '22', 'protein_coding', 55795556, 55796500, 44.87, 1, 'ENSG00000099968', 'Q9BXK5'),
(206, 'BCL2L14', 'The protein encoded by this gene belongs to the BCL2 protein family. BCL2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. Overexpression of this gene has been shown to induce apoptosis in cells. Three alternatively spliced transcript variants encoding two distinct isoforms have been reported for this gene. [provided by RefSeq, May 2009]', '12', 'protein_coding', 35854, 46795, 59.80, 12, 'ENSG00000281449', 'Q9BZR8'),
(207, 'BCL2L14', 'The protein encoded by this gene belongs to the BCL2 protein family. BCL2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. Overexpression of this gene has been shown to induce apoptosis in cells. Three alternatively spliced transcript variants encoding two distinct isoforms have been reported for this gene. [provided by RefSeq, May 2009]', '12', 'protein_coding', 35854, 46795, 59.80, 12, 'ENSG00000121380', 'Q9BZR8'),
(208, 'BCO2', 'This gene encodes an enzyme which oxidizes carotenoids such as beta-carotene during the biosynthesis of vitamin A. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]', '11', 'protein_coding', 49113407, 49123735, 50.86, 1, 'ENSG00000197580', 'Q9BYV7'),
(209, 'BCRP2', 'Predicted to be involved in negative regulation of fatty acid biosynthetic process. Part of Cul4-RING E3 ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 8413131, 8490411, 44.62, 19, 'ENSG00000169668', ''),
(210, 'BDP1', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, the mitotic cycle, and oncogenic transformation. This PTP contains a PEST motif, which often serves as a protein-protein interaction domain, and may be related to protein intracellular half-live. This protein can differentially dephosphorylate autophosphorylated tyrosine kinases that are overexpressed in tumor tissues, and it appears to regulate HER2, a member of the epidermal growth factor receptor family of receptor tyrosine kinases. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2008]', '2', 'protein_coding', 1753965, 1848609, 40.82, 15, 'ENSG00000274803', 'A6H8Y1'),
(211, 'BDP1', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, the mitotic cycle, and oncogenic transformation. This PTP contains a PEST motif, which often serves as a protein-protein interaction domain, and may be related to protein intracellular half-live. This protein can differentially dephosphorylate autophosphorylated tyrosine kinases that are overexpressed in tumor tissues, and it appears to regulate HER2, a member of the epidermal growth factor receptor family of receptor tyrosine kinases. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2008]', '2', 'protein_coding', 1753965, 1848609, 40.82, 15, 'ENSG00000273873', 'A6H8Y1'),
(212, 'BDP1', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, the mitotic cycle, and oncogenic transformation. This PTP contains a PEST motif, which often serves as a protein-protein interaction domain, and may be related to protein intracellular half-live. This protein can differentially dephosphorylate autophosphorylated tyrosine kinases that are overexpressed in tumor tissues, and it appears to regulate HER2, a member of the epidermal growth factor receptor family of receptor tyrosine kinases. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2008]', '2', 'protein_coding', 1753965, 1848609, 40.82, 15, 'ENSG00000145734', 'A6H8Y1'),
(213, 'BEGAIN', 'Predicted to enable kinase activity. Predicted to be involved in evoked excitatory postsynaptic potential and regulation of postsynaptic neurotransmitter receptor activity. Predicted to be located in dendrite; nucleus; and presynapse. Predicted to be active in glutamatergic synapse and postsynapse. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 76136333, 76141245, 63.83, 1, 'ENSG00000183092', 'Q9BUH8'),
(214, 'BEND3', 'Enables rDNA binding activity. Involved in negative regulation of macromolecule biosynthetic process; positive regulation of ATP metabolic process; and protein homooligomerization. Located in heterochromatin; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'lncRNA', 3594114, 3610103, 45.52, 11, 'ENSG00000178409', 'Q5T5X7'),
(215, 'BHMT2', 'Homocysteine is a sulfur-containing amino acid that plays a crucial role in methylation reactions. Transfer of the methyl group from betaine to homocysteine creates methionine, which donates the methyl group to methylate DNA, proteins, lipids, and other intracellular metabolites. The protein encoded by this gene is one of two methyl transferases that can catalyze the transfer of the methyl group from betaine to homocysteine. Anomalies in homocysteine metabolism have been implicated in disorders ranging from vascular disease to neural tube birth defects such as spina bifida. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2010]', '5', 'protein_coding', 50731580, 50756795, 41.04, 2, 'ENSG00000132840', 'Q9H2M3'),
(216, 'BICC1', 'This gene encodes an RNA-binding protein that is active in regulating gene expression by modulating protein translation during embryonic development. Mouse studies identified the corresponding protein to be under strict control during cell differentiation and to be a maternally provided gene product. [provided by RefSeq, Apr 2009]', '10', 'miRNA', 41169023, 41169078, 64.29, 1, 'ENSG00000122870', 'Q9H694'),
(217, 'BIRC7', 'This gene encodes a member of the inhibitor of apoptosis protein (IAP) family, and contains a single copy of a baculovirus IAP repeat (BIR) as well as a RING-type zinc finger domain. The BIR domain is essential for inhibitory activity and interacts with caspases, while the RING finger domain sometimes enhances antiapoptotic activity but does not inhibit apoptosis alone. Elevated levels of the encoded protein may be associated with cancer progression and play a role in chemotherapy sensitivity. Alternative splicing results in multiple transcript variants [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 15384799, 15493564, 38.99, 6, 'ENSG00000101197', 'Q96CA5'),
(218, 'BLK', 'This gene encodes a nonreceptor tyrosine-kinase of the src family of proto-oncogenes that are typically involved in cell proliferation and differentiation. The protein has a role in B-cell receptor signaling and B-cell development. The protein also stimulates insulin synthesis and secretion in response to glucose and enhances the expression of several pancreatic beta-cell transcription factors. [provided by RefSeq, Aug 2010]', '8', 'protein_coding', 35808016, 35857890, 41.67, 3, 'ENSG00000285369', 'P51451'),
(219, 'BLK', 'This gene encodes a nonreceptor tyrosine-kinase of the src family of proto-oncogenes that are typically involved in cell proliferation and differentiation. The protein has a role in B-cell receptor signaling and B-cell development. The protein also stimulates insulin synthesis and secretion in response to glucose and enhances the expression of several pancreatic beta-cell transcription factors. [provided by RefSeq, Aug 2010]', '8', 'protein_coding', 35808016, 35857890, 41.67, 3, 'ENSG00000136573', 'P51451'),
(220, 'BLMH', 'Bleomycin hydrolase (BMH) is a cytoplasmic cysteine peptidase that is highly conserved through evolution; however, the only known activity of the enzyme is metabolic inactivation of the glycopeptide bleomycin (BLM), an essential component of combination chemotherapy regimens for cancer. The protein contains the signature active site residues of the cysteine protease papain superfamily. [provided by RefSeq, Jul 2008]', '17', 'snoRNA', 29333163, 29333258, 55.21, 1, 'ENSG00000108578', 'Q13867'),
(221, 'BLOC1S5', 'This gene encodes a component of BLOC-1 (biogenesis of lysosome-related organelles complex 1). Components of this complex are involved in the biogenesis of organelles such as melanosomes and platelet-dense granules. A mouse model for Hermansky-Pudlak Syndrome is mutated in the murine version of this gene. Alternative splicing results in multiple transcript variants. Read-through transcription exists between this gene and the upstream EEF1E1 (eukaryotic translation elongation factor 1 epsilon 1) gene, as well as with the downstream TXNDC5 (thioredoxin domain containing 5) gene. [provided by RefSeq, Dec 2010]', '6', 'protein_coding', 100260992, 100325251, 36.47, 9, 'ENSG00000188428', 'Q8TDH9'),
(222, 'BLVRA', 'The protein encoded by this gene belongs to the biliverdin reductase family, members of which catalyze the conversion of biliverdin to bilirubin in the presence of NADPH or NADH. Mutations in this gene are associated with hyperbiliverdinemia. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2011]', '7', 'protein_coding', 54968727, 55173632, 36.22, 8, 'ENSG00000106605', 'P53004'),
(223, 'BMI1', 'This gene encodes a ring finger protein that is major component of the polycomb group complex 1 (PRC1). This complex functions through chromatin remodeling as an essential epigenetic repressor of multiple regulatory genes involved in embryonic development and self-renewal in somatic stem cells. This protein also plays a central role in DNA damage repair. This gene is an oncogene and aberrant expression is associated with numerous cancers and is associated with resistance to certain chemotherapies. A pseudogene of this gene is found on chromosome X. Read-through transcription also exists between this gene and the upstream COMM domain containing 3 (COMMD3) gene. [provided by RefSeq, Sep 2015]', '10', 'protein_coding', 120578764, 120581402, 48.65, 9, 'ENSG00000168283', 'P35226'),
(224, 'BORCS8-MEF2B', 'This gene represents numerous read-through transcripts that span GeneID:729991 and 100271849. Many read-through transcripts are predicted to be nonsense-mediated decay (NMD) candidates, and are thought to be non-coding. Some transcripts are predicted to be capable of translation reinitiation at a downstream AUG, resulting in expression of at least one isoform of myocyte enhancer factor 2B (MEF2B) from this read-through locus. At least one additional MEF2B variant and isoform can be expressed from a downstream promoter, and is annotated on GeneID:100271849. [provided by RefSeq, Oct 2010]', '19', 'protein_coding', 273220, 274152, 45.55, 1, 'ENSG00000064489', 'H3BNR1'),
(225, 'BPGM', '2,3-diphosphoglycerate (2,3-DPG) is a small molecule found at high concentrations in red blood cells where it binds to and decreases the oxygen affinity of hemoglobin. This gene encodes a multifunctional enzyme that catalyzes 2,3-DPG synthesis via its synthetase activity, and 2,3-DPG degradation via its phosphatase activity. The enzyme also has phosphoglycerate phosphomutase activity. Deficiency of this enzyme increases the affinity of cells for oxygen. Mutations in this gene result in hemolytic anemia. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 75714131, 75726490, 42.66, 1, 'ENSG00000172331', 'P07738'),
(226, 'BPI', 'This gene encodes a lipopolysaccharide binding protein. It is associated with human neutrophil granules and has antimicrobial activity against gram-negative organisms. [provided by RefSeq, Nov 2014]', '20', 'miRNA', 176629439, 176629500, 77.42, 1, 'ENSG00000101425', 'P17213'),
(227, 'BPIFA1', 'This gene is the human homolog of murine plunc, and like the mouse gene, is specifically expressed in the upper airways and nasopharyngeal regions. The encoded antimicrobial protein displays antibacterial activity against Gram-negative bacteria. It is thought to be involved in inflammatory responses to irritants in the upper airways and may also serve as a potential molecular marker for detection of micrometastasis in non-small-cell lung cancer. Multiple transcript variants resulting from alternative splicing in the 3\' UTR have been detected, but the full-length nature of only three are known. [provided by RefSeq, Aug 2014]', '20', 'miRNA', 53722166, 53722255, 44.44, 1, 'ENSG00000198183', 'Q9NP55'),
(228, 'BPIFA2', 'This gene encodes a member of the palate, lung and nasal epithelium clone (Plunc) family of proteins. Members of this family have been proposed to play a role in the local antibacterial response in nose, mouth and upper respiratory pathways. The encoded soluble salivary protein binds bacterial lipopolysaccharide (LPS) and inhibits bacterial growth. This gene is present in a gene cluster on chromosome 20. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '20', 'protein_coding', 173369969, 173377532, 42.24, 2, 'ENSG00000131050', 'Q96DR5'),
(229, 'BPY2', 'This gene is located in the nonrecombining portion of the Y chromosome, and expressed specifically in testis. The encoded protein interacts with ubiquitin protein ligase E3A and may be involved in male germ cell development and male infertility. Three nearly identical copies of this gene exist on chromosome Y; two copies are part of a palindromic region. This record represents the copy outside of the palidromic region. [provided by RefSeq, Jul 2008]', 'Y', 'lncRNA', 24485332, 24486463, 42.58, 1, 'ENSG00000183753', 'O14599'),
(230, 'BRF1', 'This gene encodes one of the three subunits of the RNA polymerase III transcription factor complex. This complex plays a central role in transcription initiation by RNA polymerase III on genes encoding tRNA, 5S rRNA, and other small structural RNAs. The gene product belongs to the TF2B family. Several alternatively spliced variants encoding different isoforms, that function at different promoters transcribed by RNA polymerase III, have been identified. [provided by RefSeq, Jun 2011]', '14', 'protein_coding', 119078635, 119150579, 42.74, 2, 'ENSG00000185024', 'Q92994'),
(231, 'BRI3', 'Enables amyloid-beta binding activity. Involved in negative regulation of neuron projection development and neuron differentiation. Located in several cellular components, including Golgi apparatus; lysosome; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 105357712, 105438513, 44.54, 12, 'ENSG00000164713', 'O95415'),
(232, 'BRINP1', 'This gene is located within a chromosomal region that shows loss of heterozygosity in some bladder cancers.  It contains a 5\' CpG island that may be a frequent target of hypermethylation, and it may undergo hypermethylation-based silencing in some bladder cancers. [provided by RefSeq, Jul 2008]', '9', 'miRNA', 65471102, 65471186, 48.24, 1, 'ENSG00000078725', 'O60477'),
(233, 'BRMS1', 'This gene reduces the metastatic potential, but not the tumorogenicity, of human breast cancer and melanoma cell lines. The protein encoded by this gene localizes primarily to the nucleus and is a component of the mSin3a family of histone deacetylase complexes (HDAC). The protein contains two coiled-coil motifs and several imperfect leucine zipper motifs. Alternative splicing results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 55124110, 55145423, 44.62, 7, 'ENSG00000174744', 'Q9HCU9'),
(234, 'BRPF3', 'Contributes to histone acetyltransferase activity. Involved in positive regulation of DNA replication. Located in nucleus. Part of MOZ/MORF histone acetyltransferase complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 165233, 169508, 55.29, 3, 'ENSG00000096070', 'Q9ULD4'),
(235, 'BRWD3', 'The protein encoded by this gene contains a bromodomain and several WD repeats. It is thought to have a chromatin-modifying function, and may thus play a role in transcription. Mutations in this gene are associated with a spectrum of cognitive disabilities and X-linked macrocephaly. This gene is also associated with translocations in patients with B-cell chronic lymphocytic leukemia. [provided by RefSeq, Jul 2017]', 'X', 'protein_coding', 12049844, 12211084, 40.68, 14, 'ENSG00000165288', 'Q6RI45'),
(236, 'BTBD16', 'This gene encodes a protein that contains a BTB/POZ domain. This domain mediates protein-protein interactions. A mutation in this gene may be associated with bipolar disorder. [provided by RefSeq, Sep 2016]', '10', 'protein_coding', 86576368, 86583478, 51.36, 2, 'ENSG00000138152', 'Q32M84'),
(237, 'BTBD7', 'Predicted to be involved in regulation of branching involved in salivary gland morphogenesis. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 52408282, 52553487, 38.59, 21, 'ENSG00000277222', 'Q9P203'),
(238, 'BTBD7', 'Predicted to be involved in regulation of branching involved in salivary gland morphogenesis. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 52408282, 52553487, 38.59, 21, 'ENSG00000011114', 'Q9P203'),
(239, 'BTF3P13', 'This locus defines a putative member of the BTF3 family of transcription factors and is thought to represent a pseudogene. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 116267760, 116279930, 39.87, 2, 'ENSG00000227118', ''),
(240, 'BTG3', 'The protein encoded by this gene is a member of the BTG/Tob family. This family has structurally related proteins that appear to have antiproliferative properties. This encoded protein might play a role in neurogenesis in the central nervous system. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008]', '21', 'protein_coding', 23180979, 23325162, 39.23, 1, 'ENSG00000154640', 'Q14201'),
(241, 'BTG3', 'The protein encoded by this gene is a member of the BTG/Tob family. This family has structurally related proteins that appear to have antiproliferative properties. This encoded protein might play a role in neurogenesis in the central nervous system. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008]', '21', 'protein_coding', 23180979, 23325162, 39.23, 1, 'ENSG00000281484', 'Q14201'),
(242, 'BTN2A1', 'This gene encodes a member of the immunoglobulin superfamily. The gene is located in a cluster of butyrophilin-like genes in the juxta-telomeric region of the major histocompatibility complex on chromosome 6. A pseudogene of this gene has been identified in this cluster. The encoded protein is an integral plasma membrane protein involved in lipid, fatty-acid, and sterol metabolism. Alterations in this gene may be associated with several disease states including metabolic syndrome. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2013]', '6', 'protein_coding', 152069, 168819, 47.07, 2, 'ENSG00000292170', 'Q7KYR7'),
(243, 'BTN2A1', 'This gene encodes a member of the immunoglobulin superfamily. The gene is located in a cluster of butyrophilin-like genes in the juxta-telomeric region of the major histocompatibility complex on chromosome 6. A pseudogene of this gene has been identified in this cluster. The encoded protein is an integral plasma membrane protein involved in lipid, fatty-acid, and sterol metabolism. Alterations in this gene may be associated with several disease states including metabolic syndrome. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2013]', '6', 'protein_coding', 152069, 168819, 47.07, 2, 'ENSG00000112763', 'Q7KYR7'),
(244, 'BTNL10P', 'Predicted to enable signaling receptor binding activity. Predicted to be involved in T cell receptor signaling pathway and regulation of cytokine production. Predicted to be located in membrane. Predicted to be active in external side of plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 85023, 236042, 38.54, 7, 'ENSG00000285153', 'A8MVZ5'),
(245, 'BTNL10P', 'Predicted to enable signaling receptor binding activity. Predicted to be involved in T cell receptor signaling pathway and regulation of cytokine production. Predicted to be located in membrane. Predicted to be active in external side of plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 85023, 236042, 38.54, 7, 'ENSG00000215811', 'A8MVZ5'),
(246, 'BTNL8', 'Predicted to enable signaling receptor binding activity. Predicted to be involved in T cell receptor signaling pathway and regulation of cytokine production. Predicted to be located in plasma membrane. Predicted to be active in external side of plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 5596109, 5612958, 44.09, 12, 'ENSG00000113303', 'Q6UX41');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(247, 'BTRC', 'This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbws class; in addition to an F-box, this protein contains multiple WD-40 repeats. The encoded protein mediates degradation of CD4 via its interaction with HIV-1 Vpu. It has also been shown to ubiquitinate phosphorylated NFKBIA (nuclear factor of kappa light polypeptide gene enhancer in B-cells inhibitor, alpha), targeting it for degradation and thus activating nuclear factor kappa-B. Alternatively spliced transcript variants have been described. A related pseudogene exists in chromosome 6. [provided by RefSeq, Mar 2012]', '10', 'protein_coding', 42795147, 42798704, 54.78, 2, 'ENSG00000166167', 'Q9Y297'),
(248, 'C11orf68', 'Enables RNA binding activity. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 34546854, 34560345, 52.56, 4, 'ENSG00000175573', 'Q9H3H3'),
(249, 'C14orf119', 'Located in cytosol and mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '14', 'TR_J_gene', 22503750, 22503814, 43.08, 1, 'ENSG00000179933', 'Q9NWQ9'),
(250, 'C14orf180', 'Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 40778242, 40808418, 54.23, 3, 'ENSG00000274126', 'Q8N912'),
(251, 'C14orf180', 'Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 40778242, 40808418, 54.23, 3, 'ENSG00000184601', 'Q8N912'),
(252, 'C17orf58', 'Located in collagen-containing extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 120684919, 120922870, 37.44, 12, 'ENSG00000186665', 'Q2M2W7'),
(253, 'C18orf15', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 17790761, 17940501, 35.09, 13, 'ENSG00000279020', 'Q96N68'),
(254, 'C18orf54', 'Predicted to act upstream of or within negative regulation of cell population proliferation. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '18', 'miRNA', 23419105, 23419190, 41.86, 1, 'ENSG00000166845', 'Q8IYD9'),
(255, 'C19orf18', 'Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 18418864, 18434562, 64.53, 15, 'ENSG00000177025', 'Q8NEA5'),
(256, 'C19orf67', 'Predicted to act upstream of or within in utero embryonic development. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 26755200, 26760786, 44.68, 2, 'ENSG00000288239', 'A6NJJ6'),
(257, 'C19orf67', 'Predicted to act upstream of or within in utero embryonic development. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 26755200, 26760786, 44.68, 2, 'ENSG00000188032', 'A6NJJ6'),
(258, 'C1QTNF4', 'Predicted to enable cytokine activity. Involved in positive regulation of cytokine production and positive regulation of signal transduction. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 20973036, 21061586, 45.68, 2, 'ENSG00000284838', 'Q9BXJ3'),
(259, 'C1QTNF4', 'Predicted to enable cytokine activity. Involved in positive regulation of cytokine production and positive regulation of signal transduction. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 20973036, 21061586, 45.68, 2, 'ENSG00000172247', 'Q9BXJ3'),
(260, 'C1R', 'This gene encodes a member of the peptidase S1 protein family. The encoded protein is a proteolytic subunit in the complement system C1 complex. The complement system acts as a mediator in the innate immune response by ultimately triggering phagocytosis, inflammation, and rupturing the bacterial cell wall. Mutations in this gene are associated with Ehlers-Danlos Syndrome. [provided by RefSeq, Dec 2018]', '12', 'protein_coding', 1481961, 1534651, 51.75, 4, 'ENSG00000288512', 'P00736'),
(261, 'C1R', 'This gene encodes a member of the peptidase S1 protein family. The encoded protein is a proteolytic subunit in the complement system C1 complex. The complement system acts as a mediator in the innate immune response by ultimately triggering phagocytosis, inflammation, and rupturing the bacterial cell wall. Mutations in this gene are associated with Ehlers-Danlos Syndrome. [provided by RefSeq, Dec 2018]', '12', 'protein_coding', 1481961, 1534651, 51.75, 4, 'ENSG00000159403', 'P00736'),
(262, 'C22orf39', 'Predicted to be involved in negative regulation of long-term synaptic potentiation. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '22', 'unitary_pseudogene', 30904348, 30910208, 50.59, 1, 'ENSG00000242259', 'Q6P5X5'),
(263, 'C2CD4C', 'Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 155456914, 155458620, 64.79, 1, 'ENSG00000183186', 'Q8TF44'),
(264, 'C2orf50', 'Located in cilium. [provided by Alliance of Genome Resources, May 2025]', '2', 'miRNA', 568089, 568198, 72.73, 1, '', 'Q96LR7'),
(265, 'C3orf38', 'Involved in positive regulation of apoptotic process. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 88706249, 88752776, 37.16, 1, 'ENSG00000179021', 'Q5JPI3'),
(266, 'C5AR1', 'Enables G protein-coupled receptor activity and complement component C5a receptor activity. Involved in several processes, including complement component C5a signaling pathway; mRNA transcription by RNA polymerase II; and positive regulation of ERK1 and ERK2 cascade. Located in apical part of cell and basolateral plasma membrane. Biomarker of Alzheimer\'s disease; asthma; chronic obstructive pulmonary disease; rhinitis; and severe acute respiratory syndrome. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 177511577, 177516961, 59.13, 23, 'ENSG00000197405', 'P21730'),
(267, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000204542', 'Q6UXA7'),
(268, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000206461', 'Q6UXA7'),
(269, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000224105', 'Q6UXA7'),
(270, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000225543', 'Q6UXA7'),
(271, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000229432', 'Q6UXA7'),
(272, 'C6orf15', 'Predicted to enable several functions, including collagen V binding activity; fibronectin binding activity; and glycosaminoglycan binding activity. Predicted to be involved in extracellular matrix organization. Predicted to be located in interstitial matrix. Predicted to be active in extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 325859, 329670, 49.90, 9, 'ENSG00000231624', 'Q6UXA7'),
(273, 'C8G', 'The protein encoded by this gene belongs to the lipocalin family. It is one of the three subunits that constitutes complement component 8 (C8), which is composed of a disulfide-linked C8 alpha-gamma heterodimer and a non-covalently associated C8 beta chain. C8 participates in the formation of the membrane attack complex (MAC) on bacterial cell membranes. While subunits alpha and beta play a role in complement-mediated bacterial killing, the gamma subunit is not required for the bactericidal activity. [provided by RefSeq, Jul 2011]', '9', 'protein_coding', 64300358, 64304770, 55.70, 3, 'ENSG00000176919', 'P07360'),
(274, 'CA10', 'This gene encodes a protein that belongs to the carbonic anhydrase family of zinc metalloenzymes, which catalyze the reversible hydration of carbon dioxide in various biological processes. The protein encoded by this gene is an acatalytic member of the alpha-carbonic anhydrase subgroup, and it is thought to play a role in the central nervous system, especially in brain development. Multiple transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 107152562, 107459564, 39.82, 3, 'ENSG00000154975', 'Q9NS85'),
(275, 'CA9', 'Carbonic anhydrases (CAs) are a large family of zinc metalloenzymes that catalyze the reversible hydration of carbon dioxide. They participate in a variety of biological processes, including respiration, calcification, acid-base balance, bone resorption, and the formation of aqueous humor, cerebrospinal fluid, saliva, and gastric acid. They show extensive diversity in tissue distribution and in their subcellular localization. CA IX is a transmembrane protein and is one of only two tumor-associated carbonic anhydrase isoenzymes known. It is expressed in all clear-cell renal cell carcinoma, but is not detected in normal kidney or most other normal tissues. It may be involved in cell proliferation and transformation. This gene was mapped to 17q21.2 by fluorescence in situ hybridization, however, radiation hybrid mapping localized it to 9p13-p12. [provided by RefSeq, Jun 2014]', '9', 'protein_coding', 4498881, 4539035, 57.38, 8, 'ENSG00000107159', 'Q16790'),
(276, 'CAB39L', 'Predicted to enable protein serine/threonine kinase activator activity. Predicted to be involved in intracellular signal transduction. Predicted to be located in cytoplasm. Predicted to be part of serine/threonine protein kinase complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'unprocessed_pseudogene', 107290, 108220, 41.57, 1, 'ENSG00000102547', 'Q9H9S4'),
(277, 'CABCOCO1', 'Predicted to enable calcium ion binding activity. Predicted to act upstream of or within sperm flagellum assembly. Predicted to be located in centrosome; cytoplasm; and sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 70375977, 70439237, 51.87, 6, 'ENSG00000183346', 'Q8IVU9'),
(278, 'CABLES2', 'Predicted to be involved in cell division and regulation of cell cycle. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 5570104, 5571696, 49.91, 1, 'ENSG00000149679', 'Q9BTV7'),
(279, 'CABP5', 'The product of this gene belongs to a subfamily of calcium binding proteins, which share similarity to calmodulin. Calcium binding proteins are an important component of calcium mediated cellular signal transduction. Expression of this gene is retina-specific. The mouse homolog of this protein has been shown to express in the inner nuclear layer of the retina, suggested its role in neuronal functioning. The specific function of this gene is unknown. [provided by RefSeq, Oct 2009]', '19', 'protein_coding', 80078646, 80092557, 42.93, 13, 'ENSG00000105507', 'Q9NP86'),
(280, 'CACHD1', 'Predicted to enable voltage-gated calcium channel activity. Predicted to be involved in calcium ion transmembrane transport. Predicted to be located in membrane. Predicted to be part of voltage-gated calcium channel complex. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 154379273, 154381574, 61.82, 16, 'ENSG00000158966', 'Q5VU97'),
(281, 'CACNA1I', 'This gene encodes the pore-forming alpha subunit of a voltage gated calcium channel. The encoded protein is a member of a subfamily of calcium channels referred to as is a low voltage-activated, T-type, calcium channel. The channel encoded by this protein is characterized by a slower activation and inactivation compared to other T-type calcium channels. This protein may be involved in calcium signaling in neurons. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Oct 2011]', '22', 'protein_coding', 39014257, 39020352, 52.56, 2, 'ENSG00000100346', 'Q9P0X4'),
(282, 'CALM3', 'This gene encodes a member of a family of proteins that binds calcium and functions as a enzymatic co-factor. Activity of this protein is important in the regulation of the cell cycle and cytokinesis. Multiple alternatively spliced transcript variants have been observed at this gene. [provided by RefSeq, Aug 2016]', '19', 'protein_coding', 38724486, 38729372, 60.73, 1, 'ENSG00000160014', 'P0DP25'),
(283, 'CALML5', 'This gene encodes a novel calcium binding protein expressed in the epidermis and related to the calmodulin family of calcium binding proteins. Functional studies with recombinant protein demonstrate it does bind calcium and undergoes a conformational change when it does so. Abundant expression is detected only in reconstructed epidermis and is restricted to differentiating keratinocytes. In addition, it can associate with transglutaminase 3, shown to be a key enzyme in the terminal differentiation of keratinocytes. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 325552, 330224, 62.19, 1, 'ENSG00000178372', 'Q9NZT1'),
(284, 'CAMSAP1', 'Enables microtubule minus-end binding activity and spectrin binding activity. Involved in several processes, including neuron projection development; regulation of cell morphogenesis; and regulation of microtubule polymerization. Located in microtubule minus-end. Implicated in complex cortical dysplasia with other brain malformations. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 65364, 70828, 55.30, 11, 'ENSG00000130559', 'Q5T5Y3'),
(285, 'CAND1', 'This gene encodes an essential regulator of Cullin-RING ubiquitin ligases, which are in involved in ubiquitinylation of proteins degraded by the Ub proteasome system. The encoded protein binds to unneddylated cullin-RING box protein complexes and acts as an inhibitor of cullin neddylation and of Skp1, cullin, and F box ubiquitin ligase complex assembly and activity. In mammalian cell culture, this protein predominantly localizes to the cytoplasm. Knockdown of this gene in preadipocytes results in blocked adipogenesis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2016]', '12', 'protein_coding', 98470367, 98658611, 41.92, 10, 'ENSG00000111530', 'Q86VP6'),
(286, 'CANX', 'This gene encodes a member of the calnexin family of molecular chaperones. The encoded protein is a calcium-binding, endoplasmic reticulum (ER)-associated protein that interacts transiently with newly synthesized N-linked glycoproteins, facilitating protein folding and assembly. It may also play a central role in the quality control of protein folding by retaining incorrectly folded protein subunits within the ER for degradation. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jun 2018]', '5', 'protein_coding', 9292894, 9369532, 54.29, 4, 'ENSG00000283777', 'P27824'),
(287, 'CANX', 'This gene encodes a member of the calnexin family of molecular chaperones. The encoded protein is a calcium-binding, endoplasmic reticulum (ER)-associated protein that interacts transiently with newly synthesized N-linked glycoproteins, facilitating protein folding and assembly. It may also play a central role in the quality control of protein folding by retaining incorrectly folded protein subunits within the ER for degradation. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jun 2018]', '5', 'protein_coding', 9292894, 9369532, 54.29, 4, 'ENSG00000127022', 'P27824'),
(288, 'CAPN11', 'Calpains constitute a family of intracellular calcium-dependent cysteine proteases. There are eight members in this superfamily. They consist of a variable 80 kDa subunit and an invariant 30 kDa subunit. This calpain protein appears to have protease activity and calcium-binding ability. A similar mouse protein may play a functional role in spermatogenesis and in the regulation of calcium-dependent signal transduction events during meiosis. [provided by RefSeq, Dec 2008]', '6', 'protein_coding', 46490750, 46542577, 44.74, 6, 'ENSG00000137225', 'Q9UMQ6'),
(289, 'CAPSL', 'Predicted to enable calcium ion binding activity. Predicted to be located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 30291990, 30326947, 52.41, 10, 'ENSG00000152611', 'Q8WWF8'),
(290, 'CARD19', 'Predicted to enable CARD domain binding activity. Predicted to be involved in negative regulation of canonical NF-kappaB signal transduction. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '9', 'snoRNA', 176941, 177006, 40.91, 1, 'ENSG00000165233', 'Q96LW7'),
(291, 'CASP8AP2', 'This protein is highly similar to FLASH, a mouse apoptotic protein identified by its interaction with the death-effector domain (DED) of caspase 8. Studies of FLASH protein suggested that this protein may be a component of the death-inducing signaling complex that includes Fas receptor, Fas-binding adapter FADD, and caspase 8, and plays a regulatory role in Fas-mediated apoptosis. Alternative splicing results in multiple transcript variants encoding the same protein.[provided by RefSeq, Nov 2008]', '6', 'protein_coding', 45615500, 45672780, 37.95, 9, 'ENSG00000288475', 'Q9UKL3'),
(292, 'CASP8AP2', 'This protein is highly similar to FLASH, a mouse apoptotic protein identified by its interaction with the death-effector domain (DED) of caspase 8. Studies of FLASH protein suggested that this protein may be a component of the death-inducing signaling complex that includes Fas receptor, Fas-binding adapter FADD, and caspase 8, and plays a regulatory role in Fas-mediated apoptosis. Alternative splicing results in multiple transcript variants encoding the same protein.[provided by RefSeq, Nov 2008]', '6', 'protein_coding', 45615500, 45672780, 37.95, 9, 'ENSG00000118412', 'Q9UKL3'),
(293, 'CASTOR1', 'Enables several functions, including arginine binding activity; identical protein binding activity; and protein sequestering activity. Involved in cellular response to L-arginine; cellular response to amino acid starvation; and regulation of TORC1 signaling. Located in GATOR2 complex and cytosol. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 480571, 484541, 47.92, 2, 'ENSG00000239282', 'Q8WTX7'),
(294, 'CAT', 'This gene encodes catalase, a key antioxidant enzyme in the bodies defense against oxidative stress. Catalase is a heme enzyme that is present in the peroxisome of nearly all aerobic cells. Catalase converts the reactive oxygen species hydrogen peroxide to water and oxygen and thereby mitigates the toxic effects of hydrogen peroxide. Oxidative stress is hypothesized to play a role in the development of many chronic or late-onset diseases such as diabetes, asthma, Alzheimer\'s disease, systemic lupus erythematosus, rheumatoid arthritis, and cancers. Polymorphisms in this gene have been associated with decreases in catalase activity but, to date, acatalasemia is the only disease known to be caused by this gene. [provided by RefSeq, Oct 2009]', '11', 'miRNA', 3917502, 3917563, 46.77, 1, 'ENSG00000121691', 'Q6IB77'),
(295, 'CATSPER1', 'Calcium ions play a primary role in the regulation of sperm motility. This gene belongs to a family of putative cation channels that are specific to spermatozoa and localize to the flagellum. The protein family features a single repeat with six membrane-spanning segments and a predicted calcium-selective pore region. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 17877847, 18361616, 39.79, 5, 'ENSG00000175294', 'Q8NEC5'),
(296, 'CATSPERB', 'Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in sperm principal piece. Predicted to be part of CatSper complex. Predicted to be active in cilium. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 23953734, 23969279, 45.00, 8, 'ENSG00000274338', 'Q9H7T0'),
(297, 'CATSPERB', 'Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in sperm principal piece. Predicted to be part of CatSper complex. Predicted to be active in cilium. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 23953734, 23969279, 45.00, 8, 'ENSG00000133962', 'Q9H7T0'),
(298, 'CATSPERZ', 'Predicted to be involved in flagellated sperm motility; male meiotic nuclear division; and sperm capacitation. Located in cytoplasm and sperm principal piece. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 14838302, 14846999, 44.22, 3, 'ENSG00000219435', 'Q9NTU4'),
(299, 'CAV2', 'The protein encoded by this gene is a major component of the inner surface of caveolae, small invaginations of the plasma membrane, and is involved in essential cellular functions, including signal transduction, lipid metabolism, cellular growth control and apoptosis. This protein may function as a tumor suppressor. This gene and related family member (CAV1) are located next to each other on chromosome 7, and express colocalizing proteins that form a stable hetero-oligomeric complex. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene. Additional isoforms resulting from the use of alternate in-frame translation initiation codons have also been described, and shown to have preferential localization in the cell (PMID:11238462). [provided by RefSeq, May 2011]', '7', 'protein_coding', 102918294, 103090222, 42.59, 4, 'ENSG00000105971', 'P51636'),
(300, 'CBLB', 'This gene encodes an E3 ubiquitin-protein ligase which promotes proteosome-mediated protein degradation by transferring ubiquitin from an E2 ubiquitin-conjugating enzyme to a substrate. The encoded protein is involved in the regulation of immune response by limiting T-cell receptor, B-cell receptor, and high affinity immunoglobulin epsilon receptor activation. Studies in mouse suggest that this gene is involved in antifungal host defense and that its inhibition leads to increased fungal killing. Manipulation of this gene may be beneficial in implementing immunotherapies for a variety of conditions, including cancer, autoimmune diseases, allergies, and infections. [provided by RefSeq, Sep 2017]', '3', 'unprocessed_pseudogene', 4752046, 4752944, 46.16, 1, 'ENSG00000114423', 'Q13191'),
(301, 'CBLN3', 'Members of the precerebellin family, such as CBLN3, contain a cerebellin motif (see CBLN1; MIM 600432) and a C-terminal C1q signature domain (see MIM 120550) that mediates trimeric assembly of atypical collagen complexes. However, precerebellins do not contain a collagen motif, suggesting that they are not conventional components of the extracellular matrix (Pang et al., 2000 [PubMed 10964938]).[supplied by OMIM, Aug 2009]', '14', 'protein_coding', 6141094, 6226932, 41.39, 9, 'ENSG00000139899', 'Q6UW01'),
(302, 'CBLN4', 'This gene encodes a member of a family of small secreted proteins containing C1Q domains. Members of this family are involved in regulation of neurexin signalling during synapse development. The mouse homolog of the protein encoded by this gene competes with netrin to bind to the deleted in colorectal cancer receptor. [provided by RefSeq, Aug 2012]', '20', 'miRNA', 39676721, 39676779, 33.90, 1, 'ENSG00000054803', 'Q9NTU7'),
(303, 'CBX6', 'Predicted to enable chromatin binding activity and methylated histone binding activity. Involved in negative regulation of transcription by RNA polymerase II. Located in nuclear body. Part of PcG protein complex. Biomarker of glioblastoma. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 254713, 270255, 44.99, 8, 'ENSG00000183741', 'O95503'),
(304, 'CBX7', 'This gene encodes a protein that contains the CHROMO (CHRomatin Organization MOdifier) domain. The encoded protein is a component of the Polycomb repressive complex 1 (PRC1), and is thought to control the lifespan of several normal human cells. [provided by RefSeq, Oct 2016]', '22', 'protein_coding', 39457012, 39492194, 57.01, 3, 'ENSG00000100307', 'O95931'),
(305, 'CBY2', 'Enables identical protein binding activity. Predicted to be located in cytoplasmic vesicle. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 36699115, 36749917, 51.13, 5, 'ENSG00000174015', 'Q8NA61'),
(306, 'CCDC106', 'Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 25664873, 25664939, 62.69, 1, 'ENSG00000173581', 'Q9BWC9'),
(307, 'CCDC107', 'This gene encodes a membrane protein which contains a coiled-coil domain in the central region. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2013]', '9', 'protein_coding', 153700492, 153724565, 54.21, 14, 'ENSG00000159884', 'Q8WV48'),
(308, 'CCDC112', 'Located in centriolar satellite. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 5638867, 5666595, 50.97, 5, 'ENSG00000164221', 'Q8NEF3'),
(309, 'CCDC125', 'Enables identical protein binding activity. Involved in activation of GTPase activity; negative regulation of Rho protein signal transduction; and negative regulation of cell motility. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 29920104, 29990289, 43.91, 10, 'ENSG00000183323', 'Q86Z20'),
(310, 'CCDC125', 'Enables identical protein binding activity. Involved in activation of GTPase activity; negative regulation of Rho protein signal transduction; and negative regulation of cell motility. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 29920104, 29990289, 43.91, 10, 'ENSG00000277868', 'Q86Z20'),
(311, 'CCDC137', 'Enables RNA binding activity. Located in chromosome; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 170888418, 170908644, 38.20, 5, 'ENSG00000185298', 'Q6PK04'),
(312, 'CCDC172', 'Predicted to be located in sperm midpiece. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 83940554, 84384880, 35.47, 4, 'ENSG00000182645', 'P0C7W6'),
(313, 'CCDC198', 'Predicted to be involved in energy homeostasis. Predicted to be part of cytoplasmic vesicle and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 70454595, 70480274, 55.18, 13, 'ENSG00000100557', 'Q9NVL8'),
(314, 'CCDC33', 'Predicted to be active in peroxisome. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 24487100, 24644978, 34.00, 4, 'ENSG00000288407', 'Q8N5R6'),
(315, 'CCDC33', 'Predicted to be active in peroxisome. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 24487100, 24644978, 34.00, 4, 'ENSG00000140481', 'Q8N5R6'),
(316, 'CCDC38', 'Predicted to enable ubiquitin-modified histone reader activity. Predicted to be involved in acrosome assembly and sperm flagellum assembly. Located in centrosome. [provided by Alliance of Genome Resources, May 2025]', '12', 'IG_V_pseudogene', 154821, 155292, 50.21, 1, 'ENSG00000165972', 'Q502W7'),
(317, 'CCDC40', 'This gene encodes a protein that is necessary for motile cilia function. It functions in correct left-right axis formation by regulating the assembly of the inner dynein arm and the dynein regulatory complexes, which control ciliary beat. Mutations in this gene cause ciliary dyskinesia type 15, a disorder due to defects in cilia motility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]', '17', 'protein_coding', 44600597, 44602174, 63.43, 1, 'ENSG00000291505', 'Q4G0X9'),
(318, 'CCDC40', 'This gene encodes a protein that is necessary for motile cilia function. It functions in correct left-right axis formation by regulating the assembly of the inner dynein arm and the dynein regulatory complexes, which control ciliary beat. Mutations in this gene cause ciliary dyskinesia type 15, a disorder due to defects in cilia motility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]', '17', 'protein_coding', 44600597, 44602174, 63.43, 1, 'ENSG00000141519', 'Q4G0X9'),
(319, 'CCDC43', 'Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 40126027, 40134622, 40.57, 1, 'ENSG00000180329', 'Q96MW1'),
(320, 'CCDC65', 'This gene encodes a sperm tail protein that is highly expressed in adult testis, spermatocytes and spermatids. The protein plays a critical role in the assembly of the nexin-dynein regulatory complex. Mutations in this gene result in primary ciliary dyskinesia. [provided by RefSeq, Nov 2013]', '12', 'snoRNA', 43417249, 43417438, 52.11, 1, 'ENSG00000139537', 'Q8IXS2'),
(321, 'CCDC70', 'Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 20187463, 20192938, 49.38, 2, 'ENSG00000123171', 'Q6NSX1'),
(322, 'CCDC81', 'Located in centrosome; ciliary basal body; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 25451594, 25585531, 49.00, 15, 'ENSG00000149201', 'Q6ZN84'),
(323, 'CCDC88B', 'This gene encodes a member of the hook-related protein family. Members of this family are characterized by an N-terminal potential microtubule binding domain, a central coiled-coiled and a C-terminal Hook-related domain. The encoded protein may be involved in linking organelles to microtubules. [provided by RefSeq, Oct 2009]', '11', 'miRNA', 72169467, 72169540, 35.14, 1, 'ENSG00000168071', 'A6NC98'),
(324, 'CCDC97', 'Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 49043848, 49045311, 64.89, 1, 'ENSG00000142039', 'Q96F63'),
(325, 'CCL15-CCL14', 'A cluster of CC chemokine genes exists on chromosome 17q11.2. The CC chemokines are secreted proteins characterized by two adjacent cysteines. The genes chemokine (C-C motif) ligand 14 and chemokine (C-C motif) ligand 15 are adjacent loci and express read-through transcripts spanning both loci. The read-through transcripts were originally interpreted as bicistronic transcripts, but they are represented as non-coding because they are candidates for nonsense-mediated mRNA decay (NMD). [provided by RefSeq, Dec 2009]', '17', 'protein_coding', 10666483, 11149569, 41.08, 21, 'ENSG00000282521', 'A0A8I5JUL6'),
(326, 'CCL15-CCL14', 'A cluster of CC chemokine genes exists on chromosome 17q11.2. The CC chemokines are secreted proteins characterized by two adjacent cysteines. The genes chemokine (C-C motif) ligand 14 and chemokine (C-C motif) ligand 15 are adjacent loci and express read-through transcripts spanning both loci. The read-through transcripts were originally interpreted as bicistronic transcripts, but they are represented as non-coding because they are candidates for nonsense-mediated mRNA decay (NMD). [provided by RefSeq, Dec 2009]', '17', 'protein_coding', 10666483, 11149569, 41.08, 21, 'ENSG00000275688', 'A0A8I5JUL6'),
(327, 'CCL16', 'This gene is one of several cytokine genes clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins involved in immunoregulatory and inflammatory processes. The CC cytokines are proteins characterized by two adjacent cysteines. The cytokine encoded by this gene displays chemotactic activity for lymphocytes and monocytes but not for neutrophils. This cytokine also shows a potent myelosuppressive activity and suppresses proliferation of myeloid progenitor cells. The expression of this gene is upregulated by IL-10. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 65892, 66818, 44.01, 1, 'ENSG00000275095', 'O15467'),
(328, 'CCL16', 'This gene is one of several cytokine genes clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins involved in immunoregulatory and inflammatory processes. The CC cytokines are proteins characterized by two adjacent cysteines. The cytokine encoded by this gene displays chemotactic activity for lymphocytes and monocytes but not for neutrophils. This cytokine also shows a potent myelosuppressive activity and suppresses proliferation of myeloid progenitor cells. The expression of this gene is upregulated by IL-10. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 65892, 66818, 44.01, 1, 'ENSG00000275152', 'O15467'),
(329, 'CCL18', 'This antimicrobial gene is one of several Cys-Cys (CC) cytokine genes clustered on the q arm of chromosome 17. Cytokines are a family of secreted proteins involved in immunoregulatory and inflammatory processes. The CC cytokines are proteins characterized by two adjacent cysteines. The cytokine encoded by this gene displays chemotactic activity for naive T cells, CD4+ and CD8+ T cells and nonactivated lymphocytes, but not for monocytes or granulocytes. This chemokine attracts naive T lymphocytes toward dendritic cells and activated macrophages in lymph nodes. It may play a role in both humoral and cell-mediated immunity responses. [provided by RefSeq, Sep 2014]', '17', 'snoRNA', 7577955, 7578091, 52.55, 1, 'ENSG00000278006', 'P55774'),
(330, 'CCL18', 'This antimicrobial gene is one of several Cys-Cys (CC) cytokine genes clustered on the q arm of chromosome 17. Cytokines are a family of secreted proteins involved in immunoregulatory and inflammatory processes. The CC cytokines are proteins characterized by two adjacent cysteines. The cytokine encoded by this gene displays chemotactic activity for naive T cells, CD4+ and CD8+ T cells and nonactivated lymphocytes, but not for monocytes or granulocytes. This chemokine attracts naive T lymphocytes toward dendritic cells and activated macrophages in lymph nodes. It may play a role in both humoral and cell-mediated immunity responses. [provided by RefSeq, Sep 2014]', '17', 'snoRNA', 7577955, 7578091, 52.55, 1, 'ENSG00000278167', 'P55774'),
(331, 'CCL18', 'This antimicrobial gene is one of several Cys-Cys (CC) cytokine genes clustered on the q arm of chromosome 17. Cytokines are a family of secreted proteins involved in immunoregulatory and inflammatory processes. The CC cytokines are proteins characterized by two adjacent cysteines. The cytokine encoded by this gene displays chemotactic activity for naive T cells, CD4+ and CD8+ T cells and nonactivated lymphocytes, but not for monocytes or granulocytes. This chemokine attracts naive T lymphocytes toward dendritic cells and activated macrophages in lymph nodes. It may play a role in both humoral and cell-mediated immunity responses. [provided by RefSeq, Sep 2014]', '17', 'snoRNA', 7577955, 7578091, 52.55, 1, 'ENSG00000275385', 'P55774'),
(332, 'CCL4L2', 'This gene is one of several cytokine genes that are clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins that function in inflammatory and immunoregulatory processes. The protein encoded by this family member is similar to the chemokine (C-C motif) ligand 4 product, which inhibits HIV entry by binding to the cellular receptor CCR5. The copy number of this gene varies among individuals, where most individuals have one to five copies. This gene copy contains a non-consensus splice acceptor site at the 3\' terminal exon found in other highly similar gene copies, and it thus uses other alternative splice sites for the 3\' terminal exon, resulting in multiple transcript variants. [provided by RefSeq, Apr 2014]', '17', 'protein_coding', 132769370, 132813339, 39.67, 7, 'ENSG00000293547', 'Q8NHW4'),
(333, 'CCL4L2', 'This gene is one of several cytokine genes that are clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins that function in inflammatory and immunoregulatory processes. The protein encoded by this family member is similar to the chemokine (C-C motif) ligand 4 product, which inhibits HIV entry by binding to the cellular receptor CCR5. The copy number of this gene varies among individuals, where most individuals have one to five copies. This gene copy contains a non-consensus splice acceptor site at the 3\' terminal exon found in other highly similar gene copies, and it thus uses other alternative splice sites for the 3\' terminal exon, resulting in multiple transcript variants. [provided by RefSeq, Apr 2014]', '17', 'protein_coding', 132769370, 132813339, 39.67, 7, 'ENSG00000282604', 'Q8NHW4'),
(334, 'CCL4L2', 'This gene is one of several cytokine genes that are clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins that function in inflammatory and immunoregulatory processes. The protein encoded by this family member is similar to the chemokine (C-C motif) ligand 4 product, which inhibits HIV entry by binding to the cellular receptor CCR5. The copy number of this gene varies among individuals, where most individuals have one to five copies. This gene copy contains a non-consensus splice acceptor site at the 3\' terminal exon found in other highly similar gene copies, and it thus uses other alternative splice sites for the 3\' terminal exon, resulting in multiple transcript variants. [provided by RefSeq, Apr 2014]', '17', 'protein_coding', 132769370, 132813339, 39.67, 7, 'ENSG00000276125', 'Q8NHW4'),
(335, 'CCL4L2', 'This gene is one of several cytokine genes that are clustered on the q-arm of chromosome 17. Cytokines are a family of secreted proteins that function in inflammatory and immunoregulatory processes. The protein encoded by this family member is similar to the chemokine (C-C motif) ligand 4 product, which inhibits HIV entry by binding to the cellular receptor CCR5. The copy number of this gene varies among individuals, where most individuals have one to five copies. This gene copy contains a non-consensus splice acceptor site at the 3\' terminal exon found in other highly similar gene copies, and it thus uses other alternative splice sites for the 3\' terminal exon, resulting in multiple transcript variants. [provided by RefSeq, Apr 2014]', '17', 'protein_coding', 132769370, 132813339, 39.67, 7, 'ENSG00000276070', 'Q8NHW4'),
(336, 'CCNL1', 'The protein encoded by this gene is one of the highly conserved mini-chromosome maintenance proteins (MCM) that are involved in the initiation of eukaryotic genome replication. The hexameric protein complex formed by MCM proteins is a key component of the pre-replication complex (pre_RC) and may be involved in the formation of replication forks and in the recruitment of other DNA replication related proteins. This protein forms a complex with MCM4, 6, and 7, and has been shown to regulate the helicase activity of the complex. This protein is phosphorylated, and thus regulated by, protein kinases CDC2 and CDC7. Multiple alternatively spliced transcript variants have been found, but the full-length nature of some variants has not been defined. [provided by RefSeq, Oct 2012]', '3', 'protein_coding', 36127809, 36144524, 53.28, 5, 'ENSG00000163660', 'Q9UK58'),
(337, 'CCT2', 'The protein encoded by this gene is a molecular chaperone that is a member of the chaperonin containing TCP1 complex (CCT), also known as the TCP1 ring complex (TRiC). This complex consists of two identical stacked rings, each containing eight different proteins. Unfolded polypeptides enter the central cavity of the complex and are folded in an ATP-dependent manner. The complex folds various proteins, including actin and tubulin. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2010]', '12', 'snoRNA', 100941126, 100941202, 41.56, 1, 'ENSG00000166226', 'P78371'),
(338, 'CCT8', 'This gene encodes the theta subunit of the CCT chaperonin, which is abundant in the eukaryotic cytosol and may be involved in the transport and assembly of newly synthesized proteins. Alternative splicing results in multiple transcript variants of this gene. A pseudogene related to this gene is located on chromosome 1. [provided by RefSeq, Sep 2013]', '21', 'miRNA', 62332487, 62332557, 74.65, 1, 'ENSG00000156261', 'P50990'),
(339, 'CCT8', 'This gene encodes the theta subunit of the CCT chaperonin, which is abundant in the eukaryotic cytosol and may be involved in the transport and assembly of newly synthesized proteins. Alternative splicing results in multiple transcript variants of this gene. A pseudogene related to this gene is located on chromosome 1. [provided by RefSeq, Sep 2013]', '21', 'miRNA', 62332487, 62332557, 74.65, 1, 'ENSG00000291533', 'P50990'),
(340, 'CCT8L1P', 'Predicted to enable unfolded protein binding activity. Predicted to be involved in protein folding. Predicted to be located in cytoplasm. Predicted to be part of chaperonin-containing T-complex. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 23641466, 23677472, 42.88, 8, 'ENSG00000020219', 'A6NM43'),
(341, 'CD109', 'This gene encodes a glycosyl phosphatidylinositol (GPI)-linked glycoprotein that localizes to the surface of platelets, activated T-cells, and endothelial cells. The protein binds to and negatively regulates signalling by transforming growth factor beta (TGF-beta). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2014]', '6', 'protein_coding', 20034368, 20056046, 39.08, 7, 'ENSG00000156535', 'Q6YHK3'),
(342, 'CD19', 'This gene encodes a member of the immunoglobulin gene superfamily. Expression of this cell surface protein is restricted to B cell lymphocytes. This protein is a reliable marker for pre-B cells but its expression diminishes during terminal B cell differentiation in antibody secreting plasma cells. The protein has two N-terminal extracellular Ig-like domains separated by a non-Ig-like domain, a hydrophobic transmembrane domain, and a large C-terminal cytoplasmic domain. This protein forms a complex with several membrane proteins including complement receptor type 2 (CD21) and tetraspanin (CD81) and this complex reduces the threshold for antigen-initiated B cell activation. Activation of this B-cell antigen receptor complex activates the phosphatidylinositol 3-kinase signalling pathway and the subsequent release of intracellular stores of calcium ions. This protein is a target of chimeric antigen receptor (CAR) T-cells used in the treatment of lymphoblastic leukemia. Mutations in this gene are associated with the disease common variable immunodeficiency 3 (CVID3) which results in a failure of B-cell differentiation and impaired secretion of immunoglobulins. CVID3 is characterized by hypogammaglobulinemia, an inability to mount an antibody response to antigen, and recurrent bacterial infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2020]', '16', 'IG_V_gene', 101235, 101757, 49.71, 1, 'ENSG00000177455', 'P15391'),
(343, 'CD24', 'This gene encodes a sialoglycoprotein that is expressed on mature granulocytes and B cells and modulates growth and differentiation signals to these cells. The precursor protein is cleaved to a short 32 amino acid mature peptide which is anchored via a glycosyl phosphatidylinositol (GPI) link to the cell surface. This gene was missing from previous genome assemblies, but is properly located on chromosome 6. Non-transcribed pseudogenes have been designated on chromosomes 1, 15, 20, and Y. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '6', 'protein_coding', 57798, 210052, 40.66, 12, 'ENSG00000272398', 'P25063'),
(344, 'CD2BP2', 'This gene encodes a bi-functional protein. In the cytoplasm, the encoded protein binds the cytoplasmic tail of human surface antigen CD2 via its C-terminal GYF domain, and regulate CD2-triggered T lymphocyte activation. In the nucleus, this protein is a component of the U5 small nuclear ribonucleoprotein complex and is involved in RNA splicing. A pseudogene has been identified on chromosome 7. Alternative splicing results in multiple transcript variants but their biological validity has not been determined. [provided by RefSeq, Nov 2008]', '16', 'unprocessed_pseudogene', 154658741, 154659643, 54.49, 1, 'ENSG00000169217', 'O95400'),
(345, 'CD300LD-AS1', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 111114559, 111305737, 44.55, 21, 'ENSG00000291923', 'Q96MU5'),
(346, 'CD300LD-AS1', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 111114559, 111305737, 44.55, 21, 'ENSG00000182352', 'Q96MU5'),
(347, 'CD68', 'This gene encodes a 110-kD transmembrane glycoprotein that is highly expressed by human monocytes and tissue macrophages. It is a member of the lysosomal/endosomal-associated membrane glycoprotein (LAMP) family. The protein primarily localizes to lysosomes and endosomes with a smaller fraction circulating to the cell surface. It is a type I integral membrane protein with a heavily glycosylated extracellular domain and binds to tissue- and organ-specific lectins or selectins. The protein is also a member of the scavenger receptor family. Scavenger receptors typically function to clear cellular debris, promote phagocytosis, and mediate the recruitment and activation of macrophages. Alternative splicing results in multiple transcripts encoding different isoforms. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 108854588, 108855986, 63.40, 1, 'ENSG00000129226', 'P34810'),
(348, 'CD70', 'The protein encoded by this gene is a cytokine that belongs to the tumor necrosis factor (TNF) ligand family. This cytokine is a ligand for TNFRSF27/CD27. It is a surface antigen on activated, but not on resting, T and B lymphocytes. It induces proliferation of costimulated T cells, enhances the generation of cytolytic T cells, and contributes to T cell activation. This cytokine is also reported to play a role in regulating B-cell activation, cytotoxic function of natural killer cells, and immunoglobulin sythesis. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 40615, 122817, 57.50, 3, 'ENSG00000125726', 'P32970'),
(349, 'CD99', 'The protein encoded by this gene is a cell surface glycoprotein involved in leukocyte migration, T-cell adhesion, ganglioside GM1 and transmembrane protein transport, and T-cell death by a caspase-independent pathway. In addition, the encoded protein may have the ability to rearrange the actin cytoskeleton and may also act as an oncosuppressor in osteosarcoma. This gene is found in the pseudoautosomal region of chromosomes X and Y and escapes X-chromosome inactivation. There is a related pseudogene located immediately adjacent to this locus. [provided by RefSeq, Mar 2016]', 'X, Y', 'snRNA', 318424, 318587, 53.66, 1, 'ENSG00000292348', 'P14209'),
(350, 'CD99', 'The protein encoded by this gene is a cell surface glycoprotein involved in leukocyte migration, T-cell adhesion, ganglioside GM1 and transmembrane protein transport, and T-cell death by a caspase-independent pathway. In addition, the encoded protein may have the ability to rearrange the actin cytoskeleton and may also act as an oncosuppressor in osteosarcoma. This gene is found in the pseudoautosomal region of chromosomes X and Y and escapes X-chromosome inactivation. There is a related pseudogene located immediately adjacent to this locus. [provided by RefSeq, Mar 2016]', 'X, Y', 'snRNA', 318424, 318587, 53.66, 1, 'ENSG00000002586', 'P14209');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(351, 'CDC16', 'The protein encoded by this gene functions as a protein ubiquitin ligase and is a component of the multiprotein APC complex. The APC complex is a cyclin degradation system that governs exit from mitosis by targeting cell cycle proteins for degredation by the 26S proteasome. Each component protein of the APC complex is highly conserved among eukaryotic organisms. This protein, and other APC complex proteins, contain a tetratricopeptide repeat (TPR) domain; a protein domain that is often involved in protein-protein interactions and the assembly of multiprotein complexes. Multiple alternatively spliced transcript variants, encoding distinct proteins, have been identified. [provided by RefSeq, Jan 2016]', '13', 'protein_coding', 9460319, 9528524, 42.79, 18, 'ENSG00000130177', 'Q13042'),
(352, 'CDC20B', 'Predicted to enable anaphase-promoting complex binding activity and ubiquitin ligase activator activity. Involved in de novo centriole assembly. Located in deuterosome. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 13972, 30766, 47.10, 2, 'ENSG00000164287', 'Q86Y33'),
(353, 'CDC25B', 'CDC25B is a member of the CDC25 family of phosphatases. CDC25B activates the cyclin dependent kinase CDC2 by removing two phosphate groups and it is required for entry into mitosis. CDC25B shuttles between the nucleus and the cytoplasm due to nuclear localization and nuclear export signals. The protein is nuclear in the M and G1 phases of the cell cycle and moves to the cytoplasm during S and G2. CDC25B has oncogenic properties, although its role in tumor formation has not been determined. Multiple transcript variants for this gene exist. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 6533618, 6535118, 57.50, 2, 'ENSG00000101224', 'P30305'),
(354, 'CDC26', 'The protein encoded by this gene is highly similar to Saccharomyces cerevisiae Cdc26, a component of cell cycle anaphase-promoting complex (APC). APC is composed of a group of highly conserved proteins and functions as a cell cycle-regulated ubiquitin-protein ligase. APC thus is responsible for the cell cycle regulated proteolysis of various proteins. [provided by RefSeq, Jul 2008]', '9', 'snoRNA', 62854621, 62854695, 36.00, 1, 'ENSG00000176386', 'Q8NHZ8'),
(355, 'CDC45', 'The protein encoded by this gene was identified by its strong similarity with Saccharomyces cerevisiae Cdc45, an essential protein required to the initiation of DNA replication. Cdc45 is a member of the highly conserved multiprotein complex including Cdc6/Cdc18, the minichromosome maintenance proteins (MCMs) and DNA polymerase, which is important for early steps of DNA replication in eukaryotes. This protein has been shown to interact with MCM7 and DNA polymerase alpha. Studies of the similar gene in Xenopus suggested that this protein play a pivotal role in the loading of DNA polymerase alpha onto chromatin. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '22', 'protein_coding', 40781, 42089, 60.28, 1, 'ENSG00000093009', 'O75419'),
(356, 'CDH10', 'This gene encodes a type II classical cadherin of the cadherin superfamily. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate the mature cadherin protein. These integral membrane proteins mediate calcium-dependent cell-cell adhesion and are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. The extracellular domain consists of 5 subdomains, each containing a cadherin motif, and appears to determine the specificity of the protein\'s homophilic cell adhesion activity. Type II (atypical) cadherins are defined based on their lack of a histidine-alanine-valine (HAV) cell adhesion recognition sequence specific to type I cadherins. This particular cadherin is predominantly expressed in brain and is putatively involved in synaptic adhesions, axon outgrowth and guidance. Mutations in this gene may be associated with lung squamous cell carcinoma and colorectal cancer in human patients. [provided by RefSeq, Nov 2015]', '5', 'protein_coding', 135808487, 135907546, 47.03, 10, 'ENSG00000040731', 'Q9Y6N8'),
(357, 'CDH11', 'This gene encodes a type II classical cadherin from the cadherin superfamily, integral membrane proteins that mediate calcium-dependent cell-cell adhesion. Mature cadherin proteins are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. Type II (atypical) cadherins are defined based on their lack of a HAV cell adhesion recognition sequence specific to type I cadherins. Expression of this particular cadherin in osteoblastic cell lines, and its upregulation during differentiation, suggests a specific function in bone development and maintenance. [provided by RefSeq, Jul 2008]', '16', 'miRNA', 5632467, 5632566, 37.00, 1, 'ENSG00000140937', 'P55287'),
(358, 'CDH18', 'This gene encodes a type II classical cadherin from the cadherin superfamily of integral membrane proteins that mediate calcium-dependent cell-cell adhesion. Mature cadherin proteins are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. Type II (atypical) cadherins are defined based on their lack of a HAV cell adhesion recognition sequence specific to type I cadherins. This particular cadherin is expressed specifically in the central nervous system and is putatively involved in synaptic adhesion, axon outgrowth and guidance. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2014]', '5', 'protein_coding', 20684014, 20823130, 38.21, 13, 'ENSG00000145526', 'Q13634'),
(359, 'CDH23', 'This gene is a member of the cadherin superfamily, whose genes encode calcium dependent cell-cell adhesion glycoproteins. The encoded protein is thought to be involved in stereocilia organization and hair bundle formation. The gene is located in a region containing the human deafness loci DFNB12 and USH1D. Usher syndrome 1D and nonsyndromic autosomal recessive deafness DFNB12 are caused by allelic mutations of this cadherin-like gene. Upregulation of this gene may also be associated with breast cancer. Alternative splice variants encoding different isoforms have been described. [provided by RefSeq, May 2013]', '10', 'miRNA', 40369846, 40369907, 64.52, 1, 'ENSG00000107736', 'Q9H251'),
(360, 'CDH4', 'This gene is a classical cadherin from the cadherin superfamily. The encoded protein is a calcium-dependent cell-cell adhesion glycoprotein comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. Based on studies in chicken and mouse, this cadherin is thought to play an important role during brain segmentation and neuronal outgrowth. In addition, a role in kidney and muscle development is indicated. Of particular interest are studies showing stable cis-heterodimers of cadherins 2 and 4 in cotransfected cell lines. Previously thought to interact in an exclusively homophilic manner, this is the first evidence of cadherin heterodimerization. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2011]', '20', 'protein_coding', 31557009, 31598833, 42.52, 7, 'ENSG00000179242', 'P55283'),
(361, 'CDH4', 'This gene is a classical cadherin from the cadherin superfamily. The encoded protein is a calcium-dependent cell-cell adhesion glycoprotein comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. Based on studies in chicken and mouse, this cadherin is thought to play an important role during brain segmentation and neuronal outgrowth. In addition, a role in kidney and muscle development is indicated. Of particular interest are studies showing stable cis-heterodimers of cadherins 2 and 4 in cotransfected cell lines. Previously thought to interact in an exclusively homophilic manner, this is the first evidence of cadherin heterodimerization. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2011]', '20', 'protein_coding', 31557009, 31598833, 42.52, 7, 'ENSG00000280641', 'P55283'),
(362, 'CDH9', 'This gene encodes a type II classical cadherin from the cadherin superfamily, integral membrane proteins that mediate calcium-dependent cell-cell adhesion. Mature cadherin proteins are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. The extracellular domain consists of 5 subdomains, each containing a cadherin motif, and appears to determine the specificity of the protein\'s homophilic cell adhesion activity. Type II (atypical) cadherins are defined based on their lack of a HAV cell adhesion recognition sequence specific to type I cadherins. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 43758680, 43807342, 44.31, 5, 'ENSG00000113100', 'Q9ULB4'),
(363, 'CDHR1', 'This gene belongs to the cadherin superfamily of calcium-dependent cell adhesion molecules. The encoded protein is a photoreceptor-specific cadherin that plays a role in outer segment disc morphogenesis. Mutations in this gene are associated with inherited retinal dystrophies. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2013]', '10', 'protein_coding', 1850, 18614, 47.14, 2, 'ENSG00000148600', 'Q96JP9'),
(364, 'CDHR5', 'This gene is a novel mucin-like gene that is a member of the cadherin superfamily. While encoding nonpolymorphic tandem repeats rich in proline, serine and threonine similar to mucin proteins, the gene also contains sequence encoding calcium-binding motifs found in all cadherins. The role of the hybrid extracellular region and the specific function of this protein have not yet been determined. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jan 2010]', '11', 'protein_coding', 93670930, 93673578, 42.36, 2, 'ENSG00000273572', 'Q9HBB8'),
(365, 'CDHR5', 'This gene is a novel mucin-like gene that is a member of the cadherin superfamily. While encoding nonpolymorphic tandem repeats rich in proline, serine and threonine similar to mucin proteins, the gene also contains sequence encoding calcium-binding motifs found in all cadherins. The role of the hybrid extracellular region and the specific function of this protein have not yet been determined. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jan 2010]', '11', 'protein_coding', 93670930, 93673578, 42.36, 2, 'ENSG00000099834', 'Q9HBB8'),
(366, 'CDK17', 'The protein encoded by this gene belongs to the cdc2/cdkx subfamily of the ser/thr family of protein kinases. It has similarity to a rat protein that is thought to play a role in terminally differentiated neurons. Alternatively spliced transcript variants encoding different isoforms have been found. [provided by RefSeq, Jul 2010]', '12', 'protein_coding', 28718, 46591, 50.37, 14, 'ENSG00000059758', 'Q00537'),
(367, 'CDK19', 'This gene encodes a protein that is one of the components of the Mediator co-activator complex. The Mediator complex is a multi-protein complex required for transcriptional activation by DNA binding transcription factors of genes transcribed by RNA polymerase II. The protein encoded by this gene is similar to cyclin-dependent kinase 8 which can also be a component of the Mediator complex. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2014]', '6', 'protein_coding', 3339261, 3379222, 41.43, 31, 'ENSG00000155111', 'Q9BWU1'),
(368, 'CDK4', 'The protein encoded by this gene is a member of the Ser/Thr protein kinase family. This protein is highly similar to the gene products of S. cerevisiae cdc28 and S. pombe cdc2. It is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression. The activity of this kinase is restricted to the G1-S phase, which is controlled by the regulatory subunits D-type cyclins and CDK inhibitor p16(INK4a). This kinase was shown to be responsible for the phosphorylation of retinoblastoma gene product (Rb). Mutations in this gene as well as in its related proteins including D-type cyclins, p16(INK4a) and Rb were all found to be associated with tumorigenesis of a variety of cancers. Multiple polyadenylation sites of this gene have been reported. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 20168, 64017, 39.68, 1, 'ENSG00000135446', 'P11802'),
(369, 'CDK5', 'This gene encodes a proline-directed serine/threonine kinase that is a member of the cyclin-dependent kinase family of proteins. Unlike other members of the family, the protein encoded by this gene does not directly control cell cycle regulation. Instead the protein, which is predominantly expressed at high levels in mammalian postmitotic central nervous system neurons, functions in diverse processes such as synaptic plasticity and neuronal migration through phosphorylation of proteins required for cytoskeletal organization, endocytosis and exocytosis, and apoptosis. In humans, an allelic variant of the gene that results in undetectable levels of the protein has been associated with lethal autosomal recessive lissencephaly-7. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2015]', '7', 'protein_coding', 77322017, 77474635, 39.90, 20, 'ENSG00000164885', 'Q00535'),
(370, 'CDK5RAP1', 'This gene encodes a regulator of cyclin-dependent kinase 5 activity. This protein has also been reported to modify RNA by adding a methylthio-group and may thus have a dual function as an RNA methylthiotransferase and as an inhibitor of cyclin-dependent kinase 5 activity. Alternative splicing results in multiple transcript variants that encode different isoforms. [provided by RefSeq, May 2013]', '20', 'protein_coding', 30202630, 30307551, 38.43, 5, 'ENSG00000101391', 'Q96SZ6'),
(371, 'CDK8', 'This gene encodes a member of the cyclin-dependent protein kinase (CDK) family. CDK family members are known to be important regulators of cell cycle progression. This kinase and its regulatory subunit, cyclin C, are components of the Mediator transcriptional regulatory complex, involved in both transcriptional activation and repression by phosphorylation of the carboxy-terminal domain of the largest subunit of RNA polymerase II. This kinase regulates transcription by targeting the cyclin-dependent kinase 7 subunits of the general transcription initiation factor IIH, thus providing a link between the Mediator complex and the basal transcription machinery. Multiple pseudogenes of this gene have been identified. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2016]', '13', 'protein_coding', 36949738, 37000763, 39.39, 28, 'ENSG00000132964', 'P49336'),
(372, 'CDKN1C', 'This gene is imprinted, with preferential expression of the maternal allele. The encoded protein is a tight-binding, strong inhibitor of several G1 cyclin/Cdk complexes and a negative regulator of cell proliferation. Mutations in this gene are implicated in sporadic cancers and Beckwith-Wiedemann syndorome, suggesting that this gene is a tumor suppressor candidate. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2010]', '11', 'miRNA', 6383906, 6383963, 44.83, 1, 'ENSG00000273707', 'P49918'),
(373, 'CDKN1C', 'This gene is imprinted, with preferential expression of the maternal allele. The encoded protein is a tight-binding, strong inhibitor of several G1 cyclin/Cdk complexes and a negative regulator of cell proliferation. Mutations in this gene are implicated in sporadic cancers and Beckwith-Wiedemann syndorome, suggesting that this gene is a tumor suppressor candidate. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2010]', '11', 'miRNA', 6383906, 6383963, 44.83, 1, 'ENSG00000129757', 'P49918'),
(374, 'CDKN2AIP', 'The protein encoded by this gene regulates the DNA damage response through several different signaling pathways. One such pathway is the p53-HDM2-p21(WAF1) pathway, which is critical to the DNA damage response. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2015]', '4', 'protein_coding', 12991612, 13018065, 48.02, 21, 'ENSG00000168564', 'Q9NXV6'),
(375, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000204539', 'Q15517'),
(376, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000237165', 'Q15517'),
(377, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000237123', 'Q15517'),
(378, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000206460', 'Q15517'),
(379, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000137197', 'Q15517'),
(380, 'CDSN', 'This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]', '6', 'protein_coding', 92116122, 92121917, 44.20, 7, 'ENSG00000237114', 'Q15517'),
(381, 'CDY1', 'This gene encodes a protein containing a chromodomain and a histone acetyltransferase catalytic domain. Chromodomain proteins are components of heterochromatin-like complexes and can act as gene repressors. This protein is localized to the nucleus of late spermatids where histone hyperacetylation takes place. Histone hyperacetylation is thought to facilitate the transition in which protamines replace histones as the major DNA-packaging protein. The human chromosome Y has two identical copies of this gene within a palindromic region; this record represents the more telomeric copy. Chromosome Y also contains a pair of closely related genes in another more telomeric palindrome as well as several related pseudogenes. Two protein isoforms are encoded by transcript variants of this gene. Additional transcript variants have been described, but their full-length nature has not been determined. [provided by RefSeq, Jul 2008]', 'Y', 'protein_coding', 21903618, 21918042, 36.96, 3, 'ENSG00000172288', 'Q9Y6F8'),
(382, 'CEBPB', 'This intronless gene encodes a transcription factor that contains a basic leucine zipper (bZIP) domain. The encoded protein functions as a homodimer but can also form heterodimers with CCAAT/enhancer-binding proteins alpha, delta, and gamma. Activity of this protein is important in the regulation of genes involved in immune and inflammatory responses, among other processes. The use of alternative in-frame AUG start codons results in multiple protein isoforms, each with distinct biological functions. [provided by RefSeq, Oct 2013]', '20', 'protein_coding', 231961245, 232344350, 42.48, 14, 'ENSG00000172216', 'P17676'),
(383, 'CEBPZ', 'This gene encodes a member of the CCAAT/enhancer-binding protein (C/EBP) family of transcription factors. The protein functions as a dominant-negative inhibitor by forming heterodimers with other C/EBP members, such as C/EBP and LAP (liver activator protein), and preventing their DNA binding activity. The protein is implicated in adipogenesis and erythropoiesis, is activated by endoplasmic reticulum stress, and promotes apoptosis. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in myxoid liposarcomas or Ewing sarcoma. Multiple alternatively spliced transcript variants encoding two isoforms with different length have been identified. [provided by RefSeq, Aug 2010]', '12', 'miRNA', 62789815, 62789885, 54.93, 1, 'ENSG00000115816', 'Q03701'),
(384, 'CEBPZOS', 'Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 124246249, 124336863, 39.20, 2, 'ENSG00000218739', 'A8MTT3'),
(385, 'CEL', 'The protein encoded by this gene is a glycoprotein secreted from the pancreas into the digestive tract and from the lactating mammary gland into human milk. The physiological role of this protein is in cholesterol and lipid-soluble vitamin ester hydrolysis and absorption. This encoded protein promotes large chylomicron production in the intestine. Also its presence in plasma suggests its interactions with cholesterol and oxidized lipoproteins to modulate the progression of atherosclerosis. In pancreatic tumoral cells, this encoded protein is thought to be sequestrated within the Golgi compartment and is probably not secreted. This gene contains a variable number of tandem repeat (VNTR) polymorphism in the coding region that may influence the function of the encoded protein. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 7306999, 7312463, 55.30, 11, 'ENSG00000170835', 'P19835'),
(386, 'CELF4', 'Members of the CELF/BRUNOL protein family contain two N-terminal RNA recognition motif (RRM) domains, one C-terminal RRM domain, and a divergent segment of 160-230 aa between the second and third RRM domains. Members of this protein family regulate pre-mRNA alternative splicing and may also be involved in mRNA editing, and translation. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 41798225, 41879482, 51.21, 14, 'ENSG00000101489', 'Q9BZC1'),
(387, 'CENATAC', 'This gene encodes a protein thought to contain a coiled coil motif. No function has been determined for the encoded protein. A pseudogene of this gene is located on chromosome 20. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '11', 'protein_coding', 490127, 496665, 50.99, 12, 'ENSG00000280975', 'Q86UT8'),
(388, 'CENATAC', 'This gene encodes a protein thought to contain a coiled coil motif. No function has been determined for the encoded protein. A pseudogene of this gene is located on chromosome 20. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '11', 'protein_coding', 490127, 496665, 50.99, 12, 'ENSG00000186166', 'Q86UT8'),
(389, 'CENPI', 'This gene encodes a centromere protein that is a component of the CENPA-NAC (nucleosome-associated) complex. This complex is critical for accurate chromosome alignment and segregation and it ensures proper mitotic progression. This protein regulates the recruitment of kinetochore-associated proteins that are required to generate the spindle checkpoint signal. The product of this gene is involved in the response of gonadal tissues to follicle-stimulating hormone. Mutations in this gene may be involved in human X-linked disorders of gonadal development and gametogenesis. Alternate splicing results in multiple transcript variants. A pseudogene of this gene is found on chromosome 13. [provided by RefSeq, Jan 2016]', 'X', 'protein_coding', 55192, 56291, 55.27, 1, 'ENSG00000102384', 'Q92674'),
(390, 'CENPW', 'Predicted to enable DNA binding activity and protein heterodimerization activity. Involved in chromosome segregation; kinetochore assembly; and mitotic cell cycle. Located in kinetochore and nucleoplasm. Part of inner kinetochore. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 70417107, 70460427, 41.62, 2, 'ENSG00000203760', 'Q5EE01'),
(391, 'CEP126', 'Involved in cilium assembly; cytoplasmic microtubule organization; and mitotic spindle organization. Located in centrosome; ciliary base; and midbody. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 70518569, 70532743, 37.60, 2, 'ENSG00000110318', 'Q9P2H0'),
(392, 'CEP164', 'This gene encodes a centrosomal protein involved in microtubule organization, DNA damage response, and chromosome segregation. The encoded protein is required for assembly of primary cilia and localizes to mature centrioles. Defects in this gene are a cause of nephronophthisis-related ciliopathies. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2012]', '11', 'unprocessed_pseudogene', 56512235, 56513171, 45.89, 1, 'ENSG00000110274', 'Q9UPV0'),
(393, 'CEP170', 'The product of this gene is a component of the centrosome, a non-membraneous organelle that functions as the major microtubule-organizing center in animal cells. During interphase, the encoded protein localizes to the sub-distal appendages of mature centrioles, which are microtubule-based structures thought to help organize centrosomes. During mitosis, the protein associates with spindle microtubules near the centrosomes. The protein interacts with and is phosphorylated by polo-like kinase 1, and functions in maintaining microtubule organization and cell morphology. The human genome contains a putative transcribed pseudogene. Several alternatively spliced transcript variants of this gene have been found, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 100119634, 100125508, 50.57, 5, 'ENSG00000276725', 'Q5SW79'),
(394, 'CEP170', 'The product of this gene is a component of the centrosome, a non-membraneous organelle that functions as the major microtubule-organizing center in animal cells. During interphase, the encoded protein localizes to the sub-distal appendages of mature centrioles, which are microtubule-based structures thought to help organize centrosomes. During mitosis, the protein associates with spindle microtubules near the centrosomes. The protein interacts with and is phosphorylated by polo-like kinase 1, and functions in maintaining microtubule organization and cell morphology. The human genome contains a putative transcribed pseudogene. Several alternatively spliced transcript variants of this gene have been found, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 100119634, 100125508, 50.57, 5, 'ENSG00000143702', 'Q5SW79'),
(395, 'CEP290', 'This gene encodes a protein with 13 putative coiled-coil domains, a region with homology to SMC chromosome segregation ATPases, six KID motifs, three tropomyosin homology domains and an ATP/GTP binding site motif A. The protein is localized to the centrosome and cilia and has sites for N-glycosylation, tyrosine sulfation, phosphorylation, N-myristoylation, and amidation. Mutations in this gene have been associated with Joubert syndrome and nephronophthisis and the presence of antibodies against this protein is associated with several forms of cancer. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 102125679, 102140426, 41.81, 4, 'ENSG00000198707', 'O15078'),
(396, 'CEP295', 'Enables microtubule binding activity. Involved in several processes, including centriole replication; positive regulation of protein acetylation; and regulation of centrosome duplication. Located in cytoplasm and microtubule cytoskeleton. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 84903, 87568, 34.47, 1, 'ENSG00000166004', 'Q9C0D2'),
(397, 'CEP55', 'Enables identical protein binding activity. Involved in cranial skeletal system development; establishment of protein localization; and midbody abscission. Acts upstream of or within mitotic cytokinesis. Located in Flemming body and centrosome. Implicated in multinucleated neurons, anhydramnios, renal dysplasia, cerebellar hypoplasia and hydranencephaly. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 13714322, 13784919, 37.99, 2, 'ENSG00000138180', 'Q53EZ4'),
(398, 'CEP76', 'This gene encodes a centrosomal protein which regulates centriole amplification by limiting centriole duplication to once per cell cycle. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2012]', '18', 'protein_coding', 6531026, 6535924, 59.20, 1, 'ENSG00000101624', 'Q8TAP6'),
(399, 'CER1', 'This gene encodes a cytokine member of the cysteine knot superfamily, characterized by nine conserved cysteines and a cysteine knot region. The cerberus-related cytokines, together with Dan and DRM/Gremlin, represent a group of bone morphogenetic protein (BMP) antagonists that can bind directly to BMPs and inhibit their activity. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 48085, 102804, 38.27, 1, 'ENSG00000147869', 'O95813'),
(400, 'CERCAM', 'Enables identical protein binding activity. Acts upstream of or within cell adhesion. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 858278, 860146, 56.13, 1, 'ENSG00000167123', 'Q5T4B2'),
(401, 'CETN1', 'The protein encoded by this gene plays important roles in the determination of centrosome position and segregation, and in the process of microtubule severing. This protein is localized to the centrosome of interphase cells, and redistributes to the region of the spindle poles during mitosis, reflecting the dynamic behavior of the centrosome during the cell cycle. [provided by RefSeq, Jan 2015]', '18', 'protein_coding', 52674736, 52680407, 47.13, 2, 'ENSG00000177143', 'Q12798'),
(402, 'CETN3', 'The protein encoded by this gene contains four EF-hand calcium binding domains, and is a member of the centrin protein family. Centrins are evolutionarily conserved proteins similar to the CDC31 protein of S. cerevisiae. Yeast CDC31 is located at the centrosome of interphase and mitotic cells, where it plays a fundamental role in centrosome duplication and separation. Multiple forms of the proteins similar to the yeast centrin have been identified in human and other mammalian cells, some of which have been shown to be associated with centrosome fractions. This protein appears to be one of the most abundant centrins associated with centrosome, which suggests a similar function to its yeast counterpart. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2014]', '5', 'protein_coding', 146116002, 148420998, 37.52, 24, 'ENSG00000153140', 'O15182'),
(403, 'CETP', 'The protein encoded by this gene is found in plasma, where it is involved in the transfer of cholesteryl ester from high density lipoprotein (HDL) to other lipoproteins. Defects in this gene are a cause of hyperalphalipoproteinemia 1 (HALP1). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2013]', '16', 'protein_coding', 47145221, 47186813, 50.27, 6, 'ENSG00000087237', 'P11597'),
(404, 'CFAP119', 'Predicted to act upstream of or within gene expression; single fertilization; and sperm flagellum assembly. Predicted to be located in acrosomal vesicle and sperm principal piece. Predicted to be part of radial spoke. Predicted to be active in axoneme. [provided by Alliance of Genome Resources, May 2025]', '16', 'miRNA', 57390132, 57390212, 33.33, 1, 'ENSG00000196118', 'A1A4V9'),
(405, 'CFAP410', 'Four alternatively spliced transcript variants encoding four different isoforms have been found for this nuclear gene. All isoforms contain leucine-rich repeats. Three of these isoforms are mitochondrial proteins and one of them lacks the target peptide, so is not located in mitochondrion. This gene is down-regulated in Down syndrome (DS) brain, which may represent mitochondrial dysfunction in DS patients. [provided by RefSeq, Sep 2012]', '21', 'protein_coding', 32329298, 32837684, 37.92, 15, 'ENSG00000160226', 'O43822'),
(406, 'CFAP47', 'While this gene is well-supported by transcript data, no functional information on its protein product is currently available. [provided by RefSeq, Dec 2009]', 'X', 'protein_coding', 93121496, 93134251, 59.63, 5, 'ENSG00000165164', 'Q6ZTR5'),
(407, 'CFC1', 'This gene encodes a member of the epidermal growth factor (EGF)- Cripto, Frl-1, and Cryptic (CFC) family, which are involved in signalling during embryonic development. Proteins in this family share a variant EGF-like motif, a conserved cysteine-rich domain, and a C-terminal hydrophobic region. The protein encoded by this gene is necessary for patterning the left-right embryonic axis. Mutations in this gene are associated with defects in organ development, including autosomal visceral heterotaxy and congenital heart disease. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jul 2012]', '2', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000136698', 'P0CG37'),
(408, 'CFDP1', 'Predicted to be involved in chromatin remodeling. Predicted to act upstream of or within several processes, including cell adhesion; negative regulation of fibroblast apoptotic process; and regulation of cell shape. Predicted to be located in kinetochore. Predicted to be part of Swr1 complex. [provided by Alliance of Genome Resources, May 2025]', '16', 'miRNA', 37847678, 37847774, 53.61, 1, 'ENSG00000153774', 'Q9UEE9'),
(409, 'CFHR5', 'This gene is a member of a small complement factor H (CFH) gene cluster on chromosome 1. Each member of this gene family contains multiple short consensus repeats (SCRs) typical of regulators of complement activation. The protein encoded by this gene has nine SCRs with the first two repeats having heparin binding properties, a region within repeats 5-7 having heparin binding and C reactive protein binding properties, and the C-terminal repeats being similar to a complement component 3 b (C3b) binding domain. This protein co-localizes with C3, binds C3b in a dose-dependent manner, and is recruited to tissues damaged by C-reactive protein. Allelic variations in this gene have been associated, but not causally linked, with two different forms of kidney disease: membranoproliferative glomerulonephritis type II (MPGNII) and hemolytic uraemic syndrome (HUS). [provided by RefSeq, Jan 2010]', '1', 'protein_coding', 41219223, 41261766, 50.86, 5, 'ENSG00000292002', 'Q9BXR6'),
(410, 'CFHR5', 'This gene is a member of a small complement factor H (CFH) gene cluster on chromosome 1. Each member of this gene family contains multiple short consensus repeats (SCRs) typical of regulators of complement activation. The protein encoded by this gene has nine SCRs with the first two repeats having heparin binding properties, a region within repeats 5-7 having heparin binding and C reactive protein binding properties, and the C-terminal repeats being similar to a complement component 3 b (C3b) binding domain. This protein co-localizes with C3, binds C3b in a dose-dependent manner, and is recruited to tissues damaged by C-reactive protein. Allelic variations in this gene have been associated, but not causally linked, with two different forms of kidney disease: membranoproliferative glomerulonephritis type II (MPGNII) and hemolytic uraemic syndrome (HUS). [provided by RefSeq, Jan 2010]', '1', 'protein_coding', 41219223, 41261766, 50.86, 5, 'ENSG00000134389', 'Q9BXR6'),
(411, 'CGB5', 'This gene is a member of the glycoprotein hormone beta chain family and encodes the beta 3 subunit of chorionic gonadotropin (CG). Glycoprotein hormones are heterodimers consisting of a common alpha subunit and an unique beta subunit which confers biological specificity. CG is produced by the trophoblastic cells of the placenta and stimulates the ovaries to synthesize the steroids that are essential for the maintenance of pregnancy. The beta subunit of CG is encoded by 6 genes which are arranged in tandem and inverted pairs on chromosome 19q13.3 and contiguous with the luteinizing hormone beta subunit gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 94026340, 94048930, 52.76, 2, 'ENSG00000189052', 'P0DN86'),
(412, 'CHAF1B', 'Chromatin assembly factor I (CAF-I) is required for the assembly of histone octamers onto newly-replicated DNA. CAF-I is composed of three protein subunits, p50, p60, and p150. The protein encoded by this gene corresponds to the p60 subunit and is required for chromatin assembly after replication. The encoded protein is differentially phosphorylated in a cell cycle-dependent manner. In addition, it is normally found in the nucleus except during mitosis, when it is released into the cytoplasm. This protein is a member of the WD-repeat HIR1 family and may also be involved in DNA repair. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 33488055, 33491716, 45.00, 3, 'ENSG00000159259', 'Q13112'),
(413, 'CHCHD10', 'This gene encodes a mitochondrial protein that is enriched at cristae junctions in the intermembrane space. It may play a role in cristae morphology maintenance or oxidative phosphorylation. Mutations in this gene cause frontotemporal dementia and/or amyotrophic lateral sclerosis-2. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 7 and 19. [provided by RefSeq, Aug 2014]', '22', 'protein_coding', 37551406, 37614179, 39.55, 10, 'ENSG00000250479', 'Q8WYQ3'),
(414, 'CHCHD10', 'This gene encodes a mitochondrial protein that is enriched at cristae junctions in the intermembrane space. It may play a role in cristae morphology maintenance or oxidative phosphorylation. Mutations in this gene cause frontotemporal dementia and/or amyotrophic lateral sclerosis-2. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 7 and 19. [provided by RefSeq, Aug 2014]', '22', 'protein_coding', 37551406, 37614179, 39.55, 10, 'ENSG00000273607', 'Q8WYQ3'),
(415, 'CHIA', 'The protein encoded by this gene degrades chitin, which is found in the cell wall of most fungi as well as in arthropods and some nematodes. The encoded protein can also stimulate interleukin 13 expression, and variations in this gene can lead to asthma susceptibility. Several transcript variants encoding a few different isoforms have been found for this gene. [provided by RefSeq, Apr 2012]', '1', 'protein_coding', 14081624, 14085875, 55.97, 3, 'ENSG00000134216', 'Q9BZP6'),
(416, 'CHID1', 'Enables oligosaccharide binding activity. Involved in negative regulation of cytokine production involved in inflammatory response. Located in several cellular components, including late endosome; lysosome; and trans-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 87859158, 87863533, 49.86, 1, 'ENSG00000177830', 'Q9BWS9'),
(417, 'CHKB', 'Choline kinase (CK) and ethanolamine kinase (EK) catalyze the phosphorylation of choline/ethanolamine to phosphocholine/phosphoethanolamine. This is the first enzyme in the biosynthesis of phosphatidylcholine/phosphatidylethanolamine in all animal cells. The highly purified CKs from mammalian sources and their recombinant gene products have been shown to have EK activity also, indicating that both activities reside on the same protein. The choline kinase-like protein encoded by CHKL belongs to the choline/ethanolamine kinase family; however, its exact function is not known. Read-through transcripts are expressed from this locus that include exons from the downstream CPT1B locus. [provided by RefSeq, Jun 2009]', '22', 'protein_coding', 288227, 309534, 53.55, 19, 'ENSG00000100288', 'Q9Y259'),
(418, 'CHL1', 'The protein encoded by this gene is a member of the L1 gene family of neural cell adhesion molecules. It is a neural recognition molecule that may be involved in signal transduction pathways. The deletion of one copy of this gene may be responsible for mental defects in patients with 3p- syndrome. This protein may also play a role in the growth of certain cancers. Alternate splicing results in both coding and non-coding variants. [provided by RefSeq, Nov 2011]', '3', 'protein_coding', 55547292, 55618880, 38.14, 6, 'ENSG00000134121', 'Q96FC9'),
(419, 'CHRNA7', 'The nicotinic acetylcholine receptors (nAChRs) are members of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. The nAChRs are thought to be hetero-pentamers composed of homologous subunits. The proposed structure for each subunit is a conserved N-terminal extracellular domain followed by three conserved transmembrane domains, a variable cytoplasmic loop, a fourth conserved transmembrane domain, and a short C-terminal extracellular region. The protein encoded by this gene forms a homo-oligomeric channel, displays marked permeability to calcium ions and is a major component of brain nicotinic receptors that are blocked by, and highly sensitive to, alpha-bungarotoxin. Once this receptor binds acetylcholine, it undergoes an extensive change in conformation that affects all subunits and leads to opening of an ion-conducting channel across the plasma membrane. This gene is located in a region identified as a major susceptibility locus for juvenile myoclonic epilepsy and a chromosomal location involved in the genetic transmission of schizophrenia. An evolutionarily recent partial duplication event in this region results in a hybrid containing sequence from this gene and a novel FAM7A gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]', '15', 'protein_coding', 14483, 48044, 37.95, 15, 'ENSG00000175344', 'P36544'),
(420, 'CHRNA7', 'The nicotinic acetylcholine receptors (nAChRs) are members of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. The nAChRs are thought to be hetero-pentamers composed of homologous subunits. The proposed structure for each subunit is a conserved N-terminal extracellular domain followed by three conserved transmembrane domains, a variable cytoplasmic loop, a fourth conserved transmembrane domain, and a short C-terminal extracellular region. The protein encoded by this gene forms a homo-oligomeric channel, displays marked permeability to calcium ions and is a major component of brain nicotinic receptors that are blocked by, and highly sensitive to, alpha-bungarotoxin. Once this receptor binds acetylcholine, it undergoes an extensive change in conformation that affects all subunits and leads to opening of an ion-conducting channel across the plasma membrane. This gene is located in a region identified as a major susceptibility locus for juvenile myoclonic epilepsy and a chromosomal location involved in the genetic transmission of schizophrenia. An evolutionarily recent partial duplication event in this region results in a hybrid containing sequence from this gene and a novel FAM7A gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]', '15', 'protein_coding', 14483, 48044, 37.95, 15, 'ENSG00000282088', 'P36544'),
(421, 'CHRNA7', 'The nicotinic acetylcholine receptors (nAChRs) are members of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. The nAChRs are thought to be hetero-pentamers composed of homologous subunits. The proposed structure for each subunit is a conserved N-terminal extracellular domain followed by three conserved transmembrane domains, a variable cytoplasmic loop, a fourth conserved transmembrane domain, and a short C-terminal extracellular region. The protein encoded by this gene forms a homo-oligomeric channel, displays marked permeability to calcium ions and is a major component of brain nicotinic receptors that are blocked by, and highly sensitive to, alpha-bungarotoxin. Once this receptor binds acetylcholine, it undergoes an extensive change in conformation that affects all subunits and leads to opening of an ion-conducting channel across the plasma membrane. This gene is located in a region identified as a major susceptibility locus for juvenile myoclonic epilepsy and a chromosomal location involved in the genetic transmission of schizophrenia. An evolutionarily recent partial duplication event in this region results in a hybrid containing sequence from this gene and a novel FAM7A gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]', '15', 'protein_coding', 14483, 48044, 37.95, 15, 'ENSG00000274542', 'P36544'),
(422, 'CHST11', 'The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to position 4 of the N-acetylgalactosamine (GalNAc) residue of chondroitin. Chondroitin sulfate constitutes the predominant proteoglycan present in cartilage, and is distributed on the surfaces of many cells and extracellular matrices. A chromosomal translocation involving this gene and IgH, t(12;14)(q23;q32), has been reported in a patient with B-cell chronic lymphocytic leukemia. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', '12', 'protein_coding', 275246, 290451, 47.95, 9, 'ENSG00000171310', 'Q9NPF2'),
(423, 'CHST2', 'This locus encodes a sulfotransferase protein. The encoded enzyme catalyzes the sulfation of a nonreducing N-acetylglucosamine residue, and may play a role in biosynthesis of 6-sulfosialyl Lewis X antigen. [provided by RefSeq, Aug 2011]', '3', 'protein_coding', 101061989, 101068131, 58.72, 6, 'ENSG00000175040', 'Q9Y4C5'),
(424, 'CHST6', 'The protein encoded by this gene is an enzyme that catalyzes the transfer of a sulfate group to the GlcNAc residues of keratan. Keratan sulfate helps maintain corneal transparency. Defects in this gene are a cause of macular corneal dystrophy (MCD). [provided by RefSeq, Jan 2010]', '16', 'protein_coding', 72273919, 72276036, 58.69, 5, 'ENSG00000183196', 'Q9GZX3'),
(425, 'CHURC1', 'Predicted to enable zinc ion binding activity. Predicted to be involved in fibroblast growth factor receptor signaling pathway. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 88033846, 88050160, 36.21, 10, 'ENSG00000258289', 'Q8WUH1'),
(426, 'CIAPIN1', 'CIAPIN1 is a cytokine-induced inhibitor of apoptosis with no relation to apoptosis regulatory molecules of the BCL2 (MIM 151430) or CASP (see MIM 147678) families. Expression of CIAPIN1 is dependent on growth factor stimulation (Shibayama et al., 2004 [PubMed 14970183]).[supplied by OMIM, Mar 2008]', '16', 'protein_coding', 40620680, 40665682, 51.40, 2, 'ENSG00000005194', 'Q6FI81'),
(427, 'CIB4', 'Enables calcium ion binding activity and magnesium ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 5402597, 5403547, 46.90, 1, 'ENSG00000157884', 'A0PJX0');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(428, 'CILK1', 'Eukaryotic protein kinases are enzymes that belong to a very extensive family of proteins which share a conserved catalytic core common with both serine/threonine and tyrosine protein kinases. This gene encodes an intestinal serine/threonine kinase harboring a dual phosphorylation site found in mitogen-activating protein (MAP) kinases. The protein localizes to the intestinal crypt region and is thought to be important in intestinal epithelial cell proliferation and differentiation. Alternative splicing has been observed at this locus and two variants, encoding the same isoform, have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 70326064, 70417074, 39.53, 4, 'ENSG00000112144', 'Q9UPZ9'),
(429, 'CIMAP3', 'Enables cytoskeletal protein binding activity and enzyme binding activity. Involved in positive regulation of kinase activity. Predicted to be located in ciliary basal body and trans-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '1', 'unprocessed_pseudogene', 55388529, 55389472, 40.89, 1, 'ENSG00000173947', 'Q8TCI5'),
(430, 'CIMIP1', 'Located in cilium. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 43473, 82770, 61.64, 17, 'ENSG00000124237', 'Q9H1P6'),
(431, 'CIMIP4', 'Predicted to be located in sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 131901008, 131959834, 47.94, 8, 'ENSG00000185264', 'O43247'),
(432, 'CISD3', 'CISD3 is a member of the CDGSH domain-containing family, which may play a role in regulating electron transport and oxidative phosphorylation (Wiley et al., 2007 [PubMed 17376863]).[supplied by OMIM, Apr 2008]', '17', 'protein_coding', 180899077, 180950906, 40.46, 11, 'ENSG00000274768', 'P0C7P0'),
(433, 'CISD3', 'CISD3 is a member of the CDGSH domain-containing family, which may play a role in regulating electron transport and oxidative phosphorylation (Wiley et al., 2007 [PubMed 17376863]).[supplied by OMIM, Apr 2008]', '17', 'protein_coding', 180899077, 180950906, 40.46, 11, 'ENSG00000277972', 'P0C7P0'),
(434, 'CIZ1', 'The protein encoded by this gene is a zinc finger DNA binding protein that interacts with CIP1, part of a complex with cyclin E. The encoded protein may regulate the cellular localization of CIP1. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2012]', '9', 'unprocessed_pseudogene', 143047213, 143047580, 43.48, 1, 'ENSG00000148337', 'Q9ULV3'),
(435, 'CKB', 'The protein encoded by this gene is a cytoplasmic enzyme involved in energy homeostasis. The encoded protein reversibly catalyzes the transfer of phosphate between ATP and various phosphogens such as creatine phosphate. It acts as a homodimer in brain as well as in other tissues, and as a heterodimer with a similar muscle isozyme in heart. The encoded protein is a member of the ATP:guanido phosphotransferase protein family. A pseudogene of this gene has been characterized. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 73667831, 73672112, 64.08, 2, 'ENSG00000166165', 'P12277'),
(436, 'CKLF', 'This gene encodes a member of the Kruppel-like factor subfamily of zinc finger proteins. The encoded protein is a transcriptional activator that binds directly to a specific recognition motif in the promoters of target genes. This protein acts downstream of multiple different signaling pathways and is regulated by post-translational modification. It may participate in both promoting and suppressing cell proliferation. Expression of this gene may be changed in a variety of different cancers and in cardiovascular disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2013]', '13', 'protein_coding', 105140982, 105168824, 65.88, 5, 'ENSG00000217555', 'Q13887'),
(437, 'CKLF-CMTM1', 'This locus represents naturally occurring read-through transcription between the neighboring CKLF (chemokine-like factor) and CMTM1 (CKLF-like MARVEL transmembrane domain containing 1) genes on chromosome 16. The read-through transcript produces a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Feb 2011]', '16', 'protein_coding', 500264, 501023, 45.79, 1, 'ENSG00000254788', 'H0YDF2'),
(438, 'CLBA1', 'Predicted to enable clathrin binding activity. Predicted to be involved in intracellular transport. Predicted to be part of AP-1 adaptor complex. Predicted to be active in trans-Golgi network membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 58023881, 58181616, 40.01, 2, 'ENSG00000140104', 'Q96F83'),
(439, 'CLDN12', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This gene is expressed in the inner ear and bladder epithelium, and it is over-expressed in colorectal carcinomas. This protein and claudin 2 are critical for vitamin D-dependent Ca2+ absorption between enterocytes. Multiple alternatively spliced transcript variants encoding the same protein have been found.[provided by RefSeq, Sep 2011]', '7', 'snoRNA', 98599635, 98599883, 39.36, 1, 'ENSG00000157224', 'P56749'),
(440, 'CLDN22', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This gene is intronless and overlaps the 3\' UTR of the WWC2 gene (GeneID: 80014) on the opposite strand. [provided by RefSeq, Aug 2010]', '4', 'protein_coding', 19875142, 19883932, 33.76, 4, 'ENSG00000177300', 'Q8N7P3'),
(441, 'CLDN23', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This gene is expressed in germinal center B-cells, placenta and stomach as well as in colon tumor. This gene is down-regulated in intestinal type gastric cancer. [provided by RefSeq, Aug 2010]', '8', 'protein_coding', 62260338, 62417431, 34.93, 19, 'ENSG00000285098', 'Q96B33'),
(442, 'CLDN23', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This gene is expressed in germinal center B-cells, placenta and stomach as well as in colon tumor. This gene is down-regulated in intestinal type gastric cancer. [provided by RefSeq, Aug 2010]', '8', 'protein_coding', 62260338, 62417431, 34.93, 19, 'ENSG00000253958', 'Q96B33'),
(443, 'CLDN24', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. The protein encoded by this gene is 75% identical to the mouse homolog. This gene is upstream of the CLDN22 gene, which overlaps the WWC2 gene on the opposite strand in the genome.[provided by RefSeq, Aug 2010]', '4', 'protein_coding', 1373422, 1374084, 48.87, 1, 'ENSG00000185758', 'A6NM45'),
(444, 'CLDN5', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets. Mutations in this gene have been found in patients with velocardiofacial syndrome. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, May 2018]', '22', 'unprocessed_pseudogene', 4793778, 4795284, 43.13, 1, 'ENSG00000184113', 'O00501'),
(445, 'CLDN6', 'Tight junctions represent one mode of cell-to-cell adhesion in epithelial or endothelial cell sheets, forming continuous seals around cells and serving as a physical barrier to prevent solutes and water from passing freely through the paracellular space. These junctions are comprised of sets of continuous networking strands in the outwardly facing cytoplasmic leaflet, with complementary grooves in the inwardly facing extracytoplasmic leaflet. This gene encodes a component of tight junction strands, which is a member of the claudin family. The protein is an integral membrane protein and is one of the entry cofactors for hepatitis C virus. The gene methylation may be involved in esophageal tumorigenesis. This gene is adjacent to another family member CLDN9 on chromosome 16.[provided by RefSeq, Aug 2010]', '16', 'protein_coding', 30130, 165741, 53.63, 12, 'ENSG00000184697', 'P56747'),
(446, 'CLDN7', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. Differential expression of this gene has been observed in different types of malignancies, including breast cancer, ovarian cancer, hepatocellular carcinomas, urinary tumors, prostate cancer, lung cancer, head and neck cancers, thyroid carcinomas, etc.. Alternatively spliced transcript variants encoding different isoforms have been found.[provided by RefSeq, May 2010]', '17', 'miRNA', 37657135, 37657215, 45.68, 1, 'ENSG00000288292', 'O95471'),
(447, 'CLDN7', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. Differential expression of this gene has been observed in different types of malignancies, including breast cancer, ovarian cancer, hepatocellular carcinomas, urinary tumors, prostate cancer, lung cancer, head and neck cancers, thyroid carcinomas, etc.. Alternatively spliced transcript variants encoding different isoforms have been found.[provided by RefSeq, May 2010]', '17', 'miRNA', 37657135, 37657215, 45.68, 1, 'ENSG00000181885', 'O95471'),
(448, 'CLDN8', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This protein plays important roles in the paracellular cation barrier of the distal renal tubule, and in the paracellular barrier to prevent sodium back-leakage in distal colon. Differential expression of this gene has been observed in colorectal carcinoma and renal cell tumors, and along with claudin-7, is an immunohistochemical marker for the differential diagnosis of chromophobe renal cell carcinoma and renal oncocytoma.[provided by RefSeq, May 2010]', '21', 'protein_coding', 137423, 138358, 41.56, 1, 'ENSG00000156284', 'P56748'),
(449, 'CLDN9', 'This gene encodes a member of the claudin family. Claudins are integral membrane proteins and components of tight junction strands. Tight junction strands serve as a physical barrier to prevent solutes and water from passing freely through the paracellular space between epithelial or endothelial cell sheets, and also play critical roles in maintaining cell polarity and signal transductions. This protein is one of the entry cofactors for hepatitis C virus. Mouse studies revealed that this gene is required for the preservation of sensory cells in the hearing organ and the gene deficiency is associated with deafness. [provided by RefSeq, Jun 2010]', '16', 'protein_coding', 32391698, 32483067, 45.15, 6, 'ENSG00000213937', 'O95484'),
(450, 'CLEC14A', 'This gene encodes a member of the C-type lectin/C-type lectin-like domain (CTL/CTLD) superfamily. Members of this family share a common protein fold and have diverse functions, such as cell adhesion, cell-cell signalling, glycoprotein turnover, and roles in inflammation and immune response. This family member plays a role in cell-cell adhesion and angiogenesis. It functions in filopodia formation, cell migration and tube formation. Due to its presence at higher levels in tumor endothelium than in normal tissue endothelium, it is considered to be a candidate for tumor vascular targeting. [provided by RefSeq, Jan 2012]', '14', 'protein_coding', 32689070, 32791724, 40.83, 7, 'ENSG00000176435', 'Q86T13'),
(451, 'CLEC18C', 'Enables polysaccharide binding activity. Predicted to be located in Golgi apparatus; endoplasmic reticulum; and endosome. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 68282660, 68333049, 43.30, 7, 'ENSG00000157335', 'Q8NCF0'),
(452, 'CLEC6A', 'The protein encoded by this gene is a type II membrane receptor with an extracellular C-type lectin-like domain fold. The extracellular portion binds structures with a high mannose content and has been shown to recognize several pathogens, including C. elegans, S. cerevisiae, M. tuberculosis, C. neoformans, and house dust mite. When stimulated, the encoded protein initiates signalling through the CARD9-Bcl10-Malt1 pathway, leading to the induction of cytokines. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2015]', '12', 'miRNA', 76868936, 76869019, 26.19, 1, 'ENSG00000205846', 'Q6EIG7'),
(453, 'CLEC7A', 'This gene encodes a member of the C-type lectin/C-type lectin-like domain (CTL/CTLD) superfamily. The encoded glycoprotein is a small type II membrane receptor with an extracellular C-type lectin-like domain fold and a cytoplasmic domain with an immunoreceptor tyrosine-based activation motif. It functions as a pattern-recognition receptor that recognizes a variety of beta-1,3-linked and beta-1,6-linked glucans from fungi and plants, and in this way plays a role in innate immune response. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. This gene is closely linked to other CTL/CTLD superfamily members on chromosome 12p13 in the natural killer gene complex region. [provided by RefSeq, Jul 2008]', '12', 'TR_J_gene', 22516273, 22516331, 49.15, 1, 'ENSG00000172243', 'Q9BXN2'),
(454, 'CLIC2', 'This gene encodes a chloride intracellular channel protein. Chloride channels are a diverse group of proteins that regulate fundamental cellular processes including stabilization of cell membrane potential, transepithelial transport, maintenance of intracellular pH, and regulation of cell volume. This protein plays a role in inhibiting the function of ryanodine receptor 2. A mutation in this gene is the cause of an X-linked form of cognitive disability. [provided by RefSeq, Jul 2017]', 'X', 'miRNA', 114209434, 114209529, 54.17, 1, 'ENSG00000155962', 'O15247'),
(455, 'CLIC6', 'This gene encodes a member of the chloride intracellular channel family of proteins. The gene is part of a large triplicated region found on chromosomes 1, 6, and 21. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Nov 2015]', '21', 'protein_coding', 35771974, 35950370, 44.08, 16, 'ENSG00000159212', 'Q96NY7'),
(456, 'CLPTM1L', 'The protein encoded by this gene is a membrane protein whose overexpression in cisplatin-sensitive cells causes apoptosis. Polymorphisms in this gene have been reported to increase susceptibility to several cancers, including lung, pancreatic, and breast cancers. [provided by RefSeq, Nov 2015]', '5', 'protein_coding', 49528003, 49546962, 53.50, 5, 'ENSG00000274811', 'Q96KA5'),
(457, 'CLPTM1L', 'The protein encoded by this gene is a membrane protein whose overexpression in cisplatin-sensitive cells causes apoptosis. Polymorphisms in this gene have been reported to increase susceptibility to several cancers, including lung, pancreatic, and breast cancers. [provided by RefSeq, Nov 2015]', '5', 'protein_coding', 49528003, 49546962, 53.50, 5, 'ENSG00000049656', 'Q96KA5'),
(458, 'CLTRN', 'This gene encodes a type 1 transmembrane protein that is important for trafficking amino acid transporters to the apical brush border of proximal tubules. The encoded protein binds to amino acid transporters and regulates their expression on the plasma membrane. It also plays a role in controlling insulin exocytosis by regulating formation of the SNARE (soluble N-ethylmaleimide-sensitive-factor attachment protein receptor) complex in pancreatic beta cells. The extracellular domain of the encoded protein may be cleaved and shed from the plasma membrane specifically in pancreatic beta cells. [provided by RefSeq, Jun 2013]', 'X', 'protein_coding', 9604680, 9621236, 44.45, 21, 'ENSG00000147003', 'Q9HBJ8'),
(459, 'CMKLR1', 'Enables adipokinetic hormone binding activity and adipokinetic hormone receptor activity. Involved in several processes, including negative regulation of NF-kappaB transcription factor activity; positive regulation of macrophage chemotaxis; and regulation of calcium-mediated signaling. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 93893973, 93956062, 40.30, 6, 'ENSG00000174600', 'Q99788'),
(460, 'CMKLR2', 'Enables adipokinetic hormone binding activity and adipokinetic hormone receptor activity. Predicted to be involved in glucose homeostasis and neuropeptide signaling pathway. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 14518557, 14774897, 43.19, 29, 'ENSG00000283448', 'P46091'),
(461, 'CMKLR2', 'Enables adipokinetic hormone binding activity and adipokinetic hormone receptor activity. Predicted to be involved in glucose homeostasis and neuropeptide signaling pathway. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 14518557, 14774897, 43.19, 29, 'ENSG00000183671', 'P46091'),
(462, 'CMTM3', 'This gene belongs to the chemokine-like factor gene superfamily, a novel family that is similar to the chemokine and the transmembrane 4 superfamilies of signaling molecules. This gene is one of several chemokine-like factor genes located in a cluster on chromosome 16. Alternatively spliced transcript variants containing different 5\' UTRs, but encoding the same protein, have been identified. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 58309715, 58367507, 41.45, 2, 'ENSG00000140931', 'Q96MX0'),
(463, 'CMTM7', 'This gene belongs to the chemokine-like factor gene superfamily, a novel family that is similar to the chemokine and transmembrane 4 superfamilies. This gene is one of several chemokine-like factor genes located in a cluster on chromosome 3. This gene acts as a tumor suppressor that regulates G1/S transition in the cell cycle, and epidermal growth factor receptor/protein kinase B signaling during tumor pathogenesis. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2016]', '3', 'protein_coding', 50819146, 50823787, 59.11, 4, 'ENSG00000153551', 'Q96FZ5'),
(464, 'CMTM8', 'This gene belongs to the chemokine-like factor gene superfamily, a novel family that is similar to the chemokine and the transmembrane 4 superfamilies. This gene is one of several chemokine-like factor genes located in a cluster on chromosome 3. This gene acts as a tumor suppressor, and plays a role in regulating the migration of tumor cells. The encoded protein is thought to function as a a negative regulator of epidermal growth factor-induced signaling. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2016]', '3', 'protein_coding', 108288044, 108339317, 49.34, 6, 'ENSG00000170293', 'Q8IZV2'),
(465, 'CNIH1', 'Predicted to enable signaling receptor binding activity. Predicted to be involved in immune response and signal transduction. Predicted to be located in ER to Golgi transport vesicle membrane; endoplasmic reticulum membrane; and endoplasmic reticulum-Golgi intermediate compartment membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 34787801, 36004667, 41.54, 17, 'ENSG00000100528', 'O95406'),
(466, 'CNKSR2', 'This gene encodes a multidomain protein that functions as a scaffold protein to mediate the mitogen-activated protein kinase pathways downstream from Ras. This gene product is induced by vitamin D and inhibits apoptosis in certain cancer cells. It may also play a role in ternary complex assembly of synaptic proteins at the postsynaptic membrane and coupling of signal transduction to membrane/cytoskeletal remodeling. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2009]', 'X', 'protein_coding', 58981208, 58995133, 48.57, 14, 'ENSG00000149970', 'Q8WXI2'),
(467, 'CNN3', 'This gene encodes a protein with a markedly acidic C terminus; the basic N-terminus is highly homologous to the N-terminus of a related gene, CNN1. Members of the CNN gene family all contain similar tandemly repeated motifs. This encoded protein is associated with the cytoskeleton but is not involved in contraction. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 5937228, 5950558, 42.59, 5, 'ENSG00000117519', 'Q15417'),
(468, 'CNNM2', 'This gene encodes a member of the ancient conserved domain containing protein family. Members of this protein family contain a cyclin box motif and have structural similarity to the cyclins. The encoded protein may play an important role in magnesium homeostasis by mediating the epithelial transport and renal reabsorption of Mg2+. Mutations in this gene are associated with renal hypomagnesemia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Dec 2011]', '10', 'miRNA', 479655, 479746, 83.70, 1, 'ENSG00000148842', 'Q9H8M5'),
(469, 'CNPY4', 'Predicted to enable signaling receptor binding activity. Involved in positive regulation of protein localization to plasma membrane. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 46080248, 46104334, 44.44, 20, 'ENSG00000166997', 'Q8N129'),
(470, 'CNTNAP2', 'This gene encodes a member of the neurexin family which functions in the vertebrate nervous system as cell adhesion molecules and receptors. This protein, like other neurexin proteins, contains epidermal growth factor repeats and laminin G domains. In addition, it includes an F5/8 type C domain, discoidin/neuropilin- and fibrinogen-like domains, thrombospondin N-terminal-like domains and a putative PDZ binding site. This protein is localized at the juxtaparanodes of myelinated axons, and mediates interactions between neurons and glia during nervous system development and is also involved in localization of potassium channels within differentiating axons. This gene encompasses almost 1.5% of chromosome 7 and is one of the largest genes in the human genome. It is directly bound and regulated by forkhead box protein P2, a transcription factor related to speech and language development. This gene has been implicated in multiple neurodevelopmental disorders, including Gilles de la Tourette syndrome, schizophrenia, epilepsy, autism, ADHD and intellectual disability. [provided by RefSeq, Jul 2017]', '7', 'protein_coding', 18173162, 18178117, 56.80, 3, 'ENSG00000174469', 'Q9UHC6'),
(471, 'CNTNAP2', 'This gene encodes a member of the neurexin family which functions in the vertebrate nervous system as cell adhesion molecules and receptors. This protein, like other neurexin proteins, contains epidermal growth factor repeats and laminin G domains. In addition, it includes an F5/8 type C domain, discoidin/neuropilin- and fibrinogen-like domains, thrombospondin N-terminal-like domains and a putative PDZ binding site. This protein is localized at the juxtaparanodes of myelinated axons, and mediates interactions between neurons and glia during nervous system development and is also involved in localization of potassium channels within differentiating axons. This gene encompasses almost 1.5% of chromosome 7 and is one of the largest genes in the human genome. It is directly bound and regulated by forkhead box protein P2, a transcription factor related to speech and language development. This gene has been implicated in multiple neurodevelopmental disorders, including Gilles de la Tourette syndrome, schizophrenia, epilepsy, autism, ADHD and intellectual disability. [provided by RefSeq, Jul 2017]', '7', 'protein_coding', 18173162, 18178117, 56.80, 3, 'ENSG00000278728', 'Q9UHC6'),
(472, 'COA3', 'This gene encodes a member of the cytochrome c oxidase assembly factor family. Studies of a related gene in fly suggest that the encoded protein is localized to mitochondria and is essential for cytochrome c oxidase function. [provided by RefSeq, Nov 2012]', '17', 'protein_coding', 34359480, 34402413, 52.39, 21, 'ENSG00000183978', 'Q9Y2R0'),
(473, 'COA8', 'This gene encodes a protein that localizes to the mitochondria, where it stimulates the release of cytochrome c, thereby promoting programmed cell death. Mutations in this gene have been found in individuals with mitochondrial complex IV deficiency. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]', '14', 'protein_coding', 65438549, 65534810, 42.78, 25, 'ENSG00000256053', 'Q96IL0'),
(474, 'COG6', 'This gene encodes a subunit of the conserved oligomeric Golgi complex that is required for maintaining normal structure and activity of the Golgi apparatus. The encoded protein is organized with conserved oligomeric Golgi complex components 5, 7 and 8 into a sub-complex referred to as lobe B. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Feb 2009]', '13', 'protein_coding', 93207241, 93229215, 43.11, 7, 'ENSG00000133103', 'Q9Y2V7'),
(475, 'COG7', 'The protein encoded by this gene resides in the golgi, and constitutes one of the 8 subunits of the conserved oligomeric Golgi (COG) complex, which is required for normal golgi morphology and localization. Mutations in this gene are associated with the congenital disorder of glycosylation type IIe.[provided by RefSeq, May 2010]', '16', 'IG_V_gene', 22819010, 22819756, 58.37, 1, 'ENSG00000168434', 'P83436'),
(476, 'COIL', 'The protein encoded by this gene is an integral component of Cajal bodies (also called coiled bodies). Cajal bodies are nuclear suborganelles of varying number and composition that are involved in the post-transcriptional modification of small nuclear and small nucleolar RNAs. The N-terminus of the coilin protein directs its self-oligomerization while the C-terminus influences the number of nuclear bodies assembled per cell. Differential methylation and phosphorylation of coilin likely influences its localization among nuclear bodies and the composition and assembly of Cajal bodies. This gene has pseudogenes on chromosome 4 and chromosome 14. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28677510, 28682932, 46.04, 2, 'ENSG00000121058', 'P38432'),
(477, 'COL23A1', 'COL23A1 is a member of the transmembrane collagens, a subfamily of the nonfibrillar collagens that contain a single pass hydrophobic transmembrane domain (Banyard et al., 2003 [PubMed 12644459]).[supplied by OMIM, Mar 2008]', '5', 'protein_coding', 130392, 144417, 48.25, 1, 'ENSG00000050767', 'Q86Y22'),
(478, 'COL24A1', 'This gene is a member of the collagen gene family and is thought to regulate type I collagen fibrillogenesis during fetal development. [provided by RefSeq, Mar 2017]', '1', 'protein_coding', 152086, 168847, 47.06, 2, 'ENSG00000171502', 'Q17RW2'),
(479, 'COLEC12', 'This gene encodes a member of the C-lectin family, proteins that possess collagen-like sequences and carbohydrate recognition domains. This protein is a scavenger receptor that displays several functions associated with host defense. It can bind to carbohydrate antigens on microorganisms, facilitating their recognition and removal. It also mediates the recognition, internalization, and degradation of oxidatively modified low density lipoprotein by vascular endothelial cells. [provided by RefSeq, May 2018]', '18', 'miRNA', 35720715, 35720808, 37.23, 1, 'ENSG00000158270', 'Q5KU26'),
(480, 'COMMD10', 'Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'snoRNA', 27255401, 27255526, 40.48, 1, 'ENSG00000145781', 'Q9Y6G5'),
(481, 'COMMD3', 'Predicted to be involved in sodium ion transport. Predicted to be located in extracellular region and ficolin-1-rich granule lumen. [provided by Alliance of Genome Resources, May 2025]', '10', 'TR_V_gene', 142554836, 142555318, 48.86, 1, 'ENSG00000148444', 'Q9UBI1'),
(482, 'COMMD3-BMI1', 'This locus represents naturally occurring read-through transcription between the neighboring COMM domain-containing protein 3 and polycomb complex protein BMI-1 genes on chromosome 10. The read-through transcript produces a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Feb 2011]', '10', 'protein_coding', 184649667, 184694963, 41.26, 11, 'ENSG00000269897', 'R4GMX3'),
(483, 'COPG2', 'Predicted to enable structural molecule activity. Involved in intra-Golgi vesicle-mediated transport. Part of COPI vesicle coat. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 330943, 333429, 45.15, 1, 'ENSG00000158623', 'Q9UBF2'),
(484, 'COQ4', 'This gene encodes a component of the coenzyme Q biosynthesis pathway. Coenzyme Q, an essential component of the electron transport chain, shuttles electrons between complexes I or II to complex III of the mitochondrial transport chain. This protein appears to play a structural role in stabilizing a complex that contains most of the coenzyme Q biosynthesis enzymes. Mutations in this gene are associated with mitochondrial disorders linked to coenzyme Q deficiency. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]', '9', 'protein_coding', 35812960, 35815354, 54.66, 7, 'ENSG00000167113', 'Q9Y3A0'),
(485, 'COQ5', 'Enables 2-methoxy-6-polyprenyl-1,4-benzoquinol methyltransferase activity. Involved in methylation and ubiquinone biosynthetic process. Located in mitochondrial inner membrane and mitochondrial matrix. Part of ubiquinone biosynthesis complex. Implicated in primary coenzyme Q10 deficiency 9. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 74653437, 74713117, 42.31, 18, 'ENSG00000110871', 'Q5HYK3'),
(486, 'CORO7-PAM16', 'This locus represents naturally occurring read-through transcription between the neighboring CORO7 (coronin 7) and PAM16 (presequence translocase-associated motor 16) genes on chromosome 16. The read-through transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jan 2011]', '16', 'protein_coding', 137227502, 137229640, 65.87, 1, 'ENSG00000103426', 'A0A0A6YYL4'),
(487, 'CORO7-PAM16', 'This locus represents naturally occurring read-through transcription between the neighboring CORO7 (coronin 7) and PAM16 (presequence translocase-associated motor 16) genes on chromosome 16. The read-through transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jan 2011]', '16', 'protein_coding', 137227502, 137229640, 65.87, 1, 'ENSG00000278760', 'A0A0A6YYL4'),
(488, 'COTL1', 'This gene encodes one of the numerous actin-binding proteins which regulate the actin cytoskeleton. This protein binds F-actin, and also interacts with 5-lipoxygenase, which is the first committed enzyme in leukotriene biosynthesis. Although this gene has been reported to map to chromosome 17 in the Smith-Magenis syndrome region, the best alignments for this gene are to chromosome 16. The Smith-Magenis syndrome region is the site of two related pseudogenes. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 449095, 454155, 47.07, 1, 'ENSG00000103187', 'Q14019'),
(489, 'COX18', 'This gene encodes a cytochrome c oxidase assembly protein. The encoded protein is essential for integral membrane protein insertion into the mitochondrial inner membrane. It is also required for cytochrome c oxidase assembly and activity. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014]', '4', 'protein_coding', 107168961, 107202522, 37.95, 15, 'ENSG00000163626', 'Q8N8Q8'),
(490, 'COX6B2', 'Predicted to be involved in oxidative phosphorylation. Located in mitochondrial crista. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 101098188, 101166126, 40.60, 8, 'ENSG00000160471', 'Q6YFQ2'),
(491, 'COX7A1', 'Cytochrome c oxidase (COX), the terminal component of the mitochondrial respiratory chain, catalyzes the electron transfer from reduced cytochrome c to oxygen. This component is a heteromeric complex consisting of 3 catalytic subunits encoded by mitochondrial genes and multiple structural subunits encoded by nuclear genes. The mitochondrially-encoded subunits function in electron transfer, and the nuclear-encoded subunits may function in the regulation and assembly of the complex. This nuclear gene encodes polypeptide 1 (muscle isoform) of subunit VIIa and the polypeptide 1 is present only in muscle tissues. Other polypeptides of subunit VIIa are present in both muscle and nonmuscle tissues, and are encoded by different genes. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 5408965, 5412971, 37.11, 1, 'ENSG00000161281', 'P24310'),
(492, 'COX7B2', 'Predicted to be involved in mitochondrial electron transport, cytochrome c to oxygen. Predicted to act upstream of or within spermatogenesis. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 61561569, 62297609, 40.79, 7, 'ENSG00000170516', 'Q8TF08'),
(493, 'CPA2', 'Three different forms of human pancreatic procarboxypeptidase A have been isolated. The encoded protein represents the A2 form, which is a monomeric protein with different biochemical properties from the A1 and A3 forms. The A2 form of pancreatic procarboxypeptidase acts on aromatic C-terminal residues and is a secreted protein. [provided by RefSeq, Dec 2008]', '7', 'protein_coding', 57469300, 57493867, 39.97, 11, 'ENSG00000158516', 'P48052'),
(494, 'CPA4', 'This gene is a member of the carboxypeptidase A/B subfamily, and it is located in a cluster with three other family members on chromosome 7. Carboxypeptidases are zinc-containing exopeptidases that catalyze the release of carboxy-terminal amino acids, and are synthesized as zymogens that are activated by proteolytic cleavage. This gene could be involved in the histone hyperacetylation pathway. It is imprinted and may be a strong candidate gene for prostate cancer aggressiveness. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 69931068, 69946574, 31.44, 6, 'ENSG00000128510', 'Q9UI42'),
(495, 'CPHXL', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 140678064, 140701150, 41.69, 1, 'ENSG00000283755', 'A0A1W2PPM1'),
(496, 'CPLX4', 'This gene likely encodes a member of the complexin family. The encoded protein may be involved in synaptic vesicle exocytosis. [provided by RefSeq, Jan 2009]', '18', 'protein_coding', 116085016, 116412762, 37.45, 7, 'ENSG00000166569', 'Q7Z7G2'),
(497, 'CPM', 'The protein encoded by this gene is a membrane-bound arginine/lysine carboxypeptidase. Its expression is associated with monocyte to macrophage differentiation. This encoded protein contains hydrophobic regions at the amino and carboxy termini and has 6 potential asparagine-linked glycosylation sites. The active site residues of carboxypeptidases A and B are conserved in this protein. Three alternatively spliced transcript variants encoding the same protein have been described for this gene. [provided by RefSeq, Jul 2008]', '12', 'snoRNA', 65755526, 65755659, 41.04, 1, 'ENSG00000135678', 'P14384'),
(498, 'CPNE2', 'Calcium-dependent membrane-binding proteins may regulate molecular events at the interface of the cell membrane and cytoplasm. This gene is one of several genes that encode a calcium-dependent protein containing two N-terminal type II C2 domains and an integrin A domain-like sequence in the C-terminus. Sequence analysis identified multiple alternatively spliced transcript variants but their full-length natures could not be determined. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 22829879, 22835037, 47.12, 17, 'ENSG00000140848', 'Q96FN4'),
(499, 'CPSF2', 'Predicted to enable RNA binding activity. Involved in mRNA 3\'-end processing by stem-loop binding activity and cleavage. Located in membrane. Part of mRNA cleavage and polyadenylation specificity factor complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 3667965, 3682246, 61.20, 5, 'ENSG00000165934', 'Q9P2I0'),
(500, 'CPT2', 'The protein encoded by this gene is a nuclear protein which is transported to the mitochondrial inner membrane. Together with carnitine palmitoyltransferase I, the encoded protein oxidizes long-chain fatty acids in the mitochondria. Defects in this gene are associated with mitochondrial long-chain fatty-acid (LCFA) oxidation disorders. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 26033092, 26033618, 55.22, 1, 'ENSG00000157184', 'P23786'),
(501, 'CPXCR1', 'This gene is one of several genes identified in a region of the X chromosome associated with an X-linked cleft palate (CPX) disorder. The encoded protein contains a motif similar to a motif found in zinc-finger proteins. Mutation analysis of this gene has not revealed any mutation which causes the CPX disorder. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 113399611, 113453488, 55.43, 5, 'ENSG00000147183', 'Q8N123'),
(502, 'CRADD', 'This gene encodes a protein containing a death domain (DD) motif. This protein recruits caspase 2/ICH1 to the cell death signal transduction complex, which includes tumor necrosis factor receptor 1 (TNFR1A) and RIPK1/RIP kinase, and acts in promoting apoptosis. A mutation in this gene was associated with cognitive disability. A related pseudogene is found on chromosome 3. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 380956, 381900, 39.26, 1, 'ENSG00000169372', 'P78560'),
(503, 'CRAMP1', 'Predicted to enable chromatin binding activity. Predicted to be involved in pattern specification process. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 54307859, 54390142, 34.78, 4, 'ENSG00000007545', 'Q96RY5'),
(504, 'CRAT', 'This gene encodes carnitine O-acetyltransferase, a member of the carnitine acyltransferase family and a key metabolic pathway enzyme which plays an important role in energy homeostasis and fat metabolism. This enzyme catalyzes the reversible transfer of acyl groups from an acyl-CoA thioester to carnitine and regulates the ratio of acyl-CoA/CoA. It is found in both the mitochondria and the peroxisome. Alternative splicing results in transcript variants encoding different isoforms that may localize to different subcellular compartments. [provided by RefSeq, Oct 2016]', '9', 'miRNA', 45425510, 45425573, 64.06, 1, 'ENSG00000095321', 'P43155'),
(505, 'CRHR1', 'This gene encodes a G-protein coupled receptor that binds neuropeptides of the corticotropin releasing hormone family that are major regulators of the hypothalamic-pituitary-adrenal pathway. The encoded protein is essential for the activation of signal transduction pathways that regulate diverse physiological processes including stress, reproduction, immune response and obesity. Alternative splicing results in multiple transcript variants. Naturally-occurring readthrough transcription between this gene and upstream GeneID:147081 results in transcripts that encode isoforms that share similarity with the products of this gene. [provided by RefSeq, Aug 2016]', '17', 'miRNA', 1713903, 1713987, 54.12, 1, 'ENSG00000278232', 'P34998'),
(506, 'CRHR1', 'This gene encodes a G-protein coupled receptor that binds neuropeptides of the corticotropin releasing hormone family that are major regulators of the hypothalamic-pituitary-adrenal pathway. The encoded protein is essential for the activation of signal transduction pathways that regulate diverse physiological processes including stress, reproduction, immune response and obesity. Alternative splicing results in multiple transcript variants. Naturally-occurring readthrough transcription between this gene and upstream GeneID:147081 results in transcripts that encode isoforms that share similarity with the products of this gene. [provided by RefSeq, Aug 2016]', '17', 'miRNA', 1713903, 1713987, 54.12, 1, 'ENSG00000276191', 'P34998'),
(507, 'CRHR1', 'This gene encodes a G-protein coupled receptor that binds neuropeptides of the corticotropin releasing hormone family that are major regulators of the hypothalamic-pituitary-adrenal pathway. The encoded protein is essential for the activation of signal transduction pathways that regulate diverse physiological processes including stress, reproduction, immune response and obesity. Alternative splicing results in multiple transcript variants. Naturally-occurring readthrough transcription between this gene and upstream GeneID:147081 results in transcripts that encode isoforms that share similarity with the products of this gene. [provided by RefSeq, Aug 2016]', '17', 'miRNA', 1713903, 1713987, 54.12, 1, 'ENSG00000120088', 'P34998'),
(508, 'CRISP3', 'This gene encodes a member of the cysteine-rich secretory protein (CRISP) family within the CRISP, antigen 5 and pathogenesis-related 1 proteins superfamily. The encoded protein has an N-terminal CRISP, antigen 5 and pathogenesis-related 1 proteins domain, a hinge region, and a C-terminal ion channel regulator domain. This protein contains cysteine residues, located in both the N- and C-terminal domains, that form eight disulfide bonds, a distinguishing characteristic of this family. This gene is expressed in the male reproductive tract where it plays a role in sperm function and fertilization, and the female reproductive tract where it plays a role in endometrial receptivity for embryo implantation. This gene is upregulated in certain types of prostate cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2016]', '6', 'protein_coding', 213259, 217539, 60.57, 4, 'ENSG00000096006', 'P54108'),
(509, 'CRNKL1', 'The crooked neck (crn) gene of Drosophila is essential for embryogenesis and is thought to be involved in cell cycle progression and pre-mRNA splicing. A protein encoded by this human locus has been found to localize to pre-mRNA splicing complexes in the nucleus and is necessary for pre-mRNA splicing. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 69689, 73929, 56.28, 16, 'ENSG00000101343', 'Q9BZJ0'),
(510, 'CRYBB1', 'Crystallins are separated into two classes: taxon-specific, or enzyme, and ubiquitous. The latter class constitutes the major proteins of vertebrate eye lens and maintains the transparency and refractive index of the lens. Since lens central fiber cells lose their nuclei during development, these crystallins are made and then retained throughout life, making them extremely stable proteins. Mammalian lens crystallins are divided into alpha, beta, and gamma families; beta and gamma crystallins are also considered as a superfamily. Alpha and beta families are further divided into acidic and basic groups. Seven protein regions exist in crystallins: four homologous motifs, a connecting peptide, and N- and C-terminal extensions. Beta-crystallins, the most heterogeneous, differ by the presence of the C-terminal extension (present in the basic group, none in the acidic group). Beta-crystallins form aggregates of different sizes and are able to self-associate to form dimers or to form heterodimers with other beta-crystallins. This gene, a beta basic group member, undergoes extensive cleavage at its N-terminal extension during lens maturation. It is also a member of a gene cluster with beta-A4, beta-B2, and beta-B3. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 63235883, 63240495, 64.27, 3, 'ENSG00000100122', 'P53674'),
(511, 'CRYBB3', 'Crystallins are separated into two classes: taxon-specific, or enzyme, and ubiquitous. The latter class constitutes the major proteins of vertebrate eye lens and maintains the transparency and refractive index of the lens. Since lens central fiber cells lose their nuclei during development, these crystallins are made and then retained throughout life, making them extremely stable proteins. Mammalian lens crystallins are divided into alpha, beta, and gamma families; beta and gamma crystallins are also considered as a superfamily. Alpha and beta families are further divided into acidic and basic groups. Seven protein regions exist in crystallins: four homologous motifs, a connecting peptide, and N- and C-terminal extensions. Beta-crystallins, the most heterogeneous, differ by the presence of the C-terminal extension (present in the basic group, none in the acidic group). Beta-crystallins form aggregates of different sizes and are able to self-associate to form dimers or to form heterodimers with other beta-crystallins. This gene, a beta basic group member, is part of a gene cluster with beta-A4, beta-B1, and beta-B2. Mutations in this gene result in cataract congenital nuclear autosomal recessive type 2. [provided by RefSeq, Feb 2013]', '22', 'protein_coding', 92770637, 92929331, 36.99, 5, 'ENSG00000100053', 'P26998'),
(512, 'CRYGA', 'Crystallins are separated into two classes: taxon-specific, or enzyme, and ubiquitous. The latter class constitutes the major proteins of vertebrate eye lens and maintains the transparency and refractive index of the lens. Since lens central fiber cells lose their nuclei during development, these crystallins are made and then retained throughout life, making them extremely stable proteins. Mammalian lens crystallins are divided into alpha, beta, and gamma families; beta and gamma crystallins are also considered as a superfamily. Alpha and beta families are further divided into acidic and basic groups. Seven protein regions exist in crystallins: four homologous motifs, a connecting peptide, and N- and C-terminal extensions. Gamma-crystallins are a homogeneous group of highly symmetrical, monomeric proteins typically lacking connecting peptides and terminal extensions. They are differentially regulated after early development. Four gamma-crystallin genes (gamma-A through gamma-D) and three pseudogenes (gamma-E, gamma-F, gamma-G) are tandemly organized in a genomic segment as a gene cluster. Whether due to aging or mutations in specific genes, gamma-crystallins have been involved in cataract formation. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 68109, 69113, 51.64, 1, 'ENSG00000168582', 'P11844'),
(513, 'CRYGC', 'This gene encodes a member of the beta/gamma-crystallin family of proteins. Crystallins constitute the major proteins of vertebrate eye lens and maintain the transparency and refractive index of the lens. This gene and several family members are present in a gene cluster on chromosome 2. Mutations in this gene have been shown to cause multiple types of cataract, including Coppock-like cataract and zonular pulverulent cataract, among others. [provided by RefSeq, Jan 2015]', '2', 'protein_coding', 89490719, 89557766, 52.33, 96, 'ENSG00000285011', 'P07315');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(514, 'CRYGC', 'This gene encodes a member of the beta/gamma-crystallin family of proteins. Crystallins constitute the major proteins of vertebrate eye lens and maintain the transparency and refractive index of the lens. This gene and several family members are present in a gene cluster on chromosome 2. Mutations in this gene have been shown to cause multiple types of cataract, including Coppock-like cataract and zonular pulverulent cataract, among others. [provided by RefSeq, Jan 2015]', '2', 'protein_coding', 89490719, 89557766, 52.33, 96, 'ENSG00000163254', 'P07315'),
(515, 'CRYM', 'Crystallins are separated into two classes: taxon-specific and ubiquitous. The former class is also called phylogenetically-restricted crystallins. The latter class constitutes the major proteins of vertebrate eye lens and maintains the transparency and refractive index of the lens. This gene encodes a taxon-specific crystallin protein that binds NADPH and has sequence similarity to bacterial ornithine cyclodeaminases. The encoded protein does not perform a structural role in lens tissue, and instead it binds thyroid hormone for possible regulatory or developmental roles. Mutations in this gene have been associated with autosomal dominant non-syndromic deafness. [provided by RefSeq, Sep 2014]', '16', 'protein_coding', 52602371, 52609051, 49.68, 2, 'ENSG00000103316', 'Q14894'),
(516, 'CRYZL1', 'This gene encodes a protein that has sequence similarity to zeta crystallin, also known as quinone oxidoreductase. This zeta crystallin-like protein also contains an NAD(P)H binding site. Alternatively spliced transcript variants have been observed but their full-length nature has not been completely determined. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 36777447, 36863538, 51.51, 4, 'ENSG00000205758', 'O95825'),
(517, 'CSAG1', 'This gene encodes a member of a family of tumor antigens. The protein is expressed in chondrosarcomas, but may also be expressed in normal tissues such as testis. Alternative splicing of this gene results in two transcript variants encoding the same protein. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 44113451, 44127452, 46.14, 1, 'ENSG00000198930', 'Q6PB30'),
(518, 'CSAG1', 'This gene encodes a member of a family of tumor antigens. The protein is expressed in chondrosarcomas, but may also be expressed in normal tissues such as testis. Alternative splicing of this gene results in two transcript variants encoding the same protein. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 44113451, 44127452, 46.14, 1, 'ENSG00000292188', 'Q6PB30'),
(519, 'CSN1S1', 'Predicted to be involved in several processes, including response to 11-deoxycorticosterone; response to dehydroepiandrosterone; and response to progesterone. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 30580533, 30634033, 38.92, 33, 'ENSG00000126545', 'P47710'),
(520, 'CSN1S2AP', 'This locus is the equivalent of casein gamma in mouse (GeneID 12993) and is found in a cluster of casein genes, similar to other mammals. In human, the potential open reading frame that matches the homologous protein from other species is prematurely truncated shortly after the signal peptide. Therefore, this locus appears to be a pseudogene. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 33407957, 33443763, 50.31, 1, 'ENSG00000283090', ''),
(521, 'CSN1S2AP', 'This locus is the equivalent of casein gamma in mouse (GeneID 12993) and is found in a cluster of casein genes, similar to other mammals. In human, the potential open reading frame that matches the homologous protein from other species is prematurely truncated shortly after the signal peptide. Therefore, this locus appears to be a pseudogene. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 33407957, 33443763, 50.31, 1, 'ENSG00000234124', ''),
(522, 'CSNK1A1L', 'Predicted to enable protein serine/threonine kinase activity. Predicted to be involved in negative regulation of canonical Wnt signaling pathway and signal transduction. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 23564732, 23568484, 49.64, 1, 'ENSG00000180138', 'Q8N752'),
(523, 'CSNK2A1', 'Casein kinase II is a serine/threonine protein kinase that phosphorylates acidic proteins such as casein. It is involved in various cellular processes, including cell cycle control, apoptosis, and circadian rhythm. The kinase exists as a tetramer and is composed of an alpha, an alpha-prime, and two beta subunits. The alpha subunits contain the catalytic activity while the beta subunits undergo autophosphorylation. The protein encoded by this gene represents the alpha subunit. Multiple transcript variants encoding different protein isoforms have been found for this gene. [provided by RefSeq, Apr 2018]', '20', 'protein_coding', 60869438, 61001440, 38.94, 6, 'ENSG00000101266', 'P68400'),
(524, 'CSNK2A2', 'This gene encodes the alpha\', or alpha 2, catalytic subunit of the protein kinase enzyme, casein kinase 2 (CK2). Casein kinase 2 is a serine/threonine protein kinase that phosphorylates acidic proteins such as casein. It is involved in various cellular processes, including cell cycle control, apoptosis, and circadian rhythms. This heterotetrameric kinase includes two catalytic subunits, either alpha or alpha\', and two regulatory beta subunits. The closely related gene paralog encoding the alpha, or alpha 1 subunit (CSNK2A1, Gene ID: 1457) is found on chromosome 20. An intronic variant in this gene (alpha 2) may be associated with leukocyte telomere length in a South Asian population. A related transcribed pseudogene is found on chromosome 11. [provided by RefSeq, Aug 2017]', '16', 'protein_coding', 8674, 30733, 51.96, 14, 'ENSG00000070770', 'P19784'),
(525, 'CSNKA2IP', 'Predicted to be located in male germ cell nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'unprocessed_pseudogene', 88997, 89932, 45.51, 1, '', 'A0A1B0GTH6'),
(526, 'CST1', 'The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions, where they appear to provide protective functions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes a cysteine proteinase inhibitor found in saliva, tears, urine, and seminal fluid. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 31114489, 31131393, 52.40, 8, 'ENSG00000170373', 'P01037'),
(527, 'CST11', 'The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes an epididymal-specific protein shown to have antimicrobial activity against E. coli. Alternative splicing yields two variants encoding distinct isoforms. [provided by RefSeq, Sep 2014]', '20', 'protein_coding', 49237946, 49278426, 45.85, 6, 'ENSG00000125831', 'Q9H112'),
(528, 'CST2', 'The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions, where they appear to provide protective functions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes a secreted thiol protease inhibitor found at high levels in saliva, tears and seminal plasma. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 8528874, 8732137, 39.59, 3, 'ENSG00000170369', 'P09228'),
(529, 'CST3', 'The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions, where they appear to provide protective functions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes the most abundant extracellular inhibitor of cysteine proteases, which is found in high concentrations in biological fluids and is expressed in virtually all organs of the body. A mutation in this gene has been associated with amyloid angiopathy. Expression of this protein in vascular wall smooth muscle cells is severely reduced in both atherosclerotic and aneurysmal aortic lesions, establishing its role in vascular disease. In addition, this protein has been shown to have an antimicrobial function, inhibiting the replication of herpes simplex virus. Alternative splicing results in multiple transcript variants encoding a single protein. [provided by RefSeq, Nov 2014]', '20', 'miRNA', 30731557, 30731641, 56.47, 1, 'ENSG00000101439', 'P01034'),
(530, 'CST9L', 'The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes a protein similar to mouse cystatin 9. Based on its testis-specific expression, it is likely to have a role in tissue reorganization during early testis development. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 25006230, 25058980, 52.00, 5, 'ENSG00000101435', 'Q9H4G1'),
(531, 'CSTF1', 'This gene encodes one of three subunits which combine to form cleavage stimulation factor (CSTF). CSTF is involved in the polyadenylation and 3\'end cleavage of pre-mRNAs. Similar to mammalian G protein beta subunits, this protein contains transducin-like repeats. Several transcript variants with different 5\' UTR, but encoding the same protein, have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 17406760, 17439677, 51.69, 3, 'ENSG00000101138', 'Q05048'),
(532, 'CSTP1', 'Predicted to enable metal ion binding activity and phosphoprotein phosphatase activity. Predicted to be involved in chromatin remodeling and regulation of transcription by RNA polymerase II. Located in cytosol and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 131008, 132561, 58.17, 1, 'ENSG00000228476', 'Q9BRF8'),
(533, 'CT45A10', 'Enables identical protein binding activity. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 100529072, 100559998, 45.24, 5, 'ENSG00000269586', 'P0DMU9'),
(534, 'CT45A3', 'Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'lncRNA', 102234, 104278, 60.40, 1, 'ENSG00000269096', 'Q8NHU0'),
(535, 'CT45A6', 'This gene represents one of a cluster of several similar genes located on the q arm of chromosome X. The genes in this cluster encode members of the cancer/testis (CT) family of antigens, and are distinct from other CT antigens. These antigens are thought to be novel therapeutic targets for human cancers. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', 'X', 'protein_coding', 136146702, 136211359, 42.07, 36, 'ENSG00000278289', 'P0DMU7'),
(536, 'CT55', 'Involved in spermatogenesis. Located in acrosomal vesicle and sperm flagellum. Implicated in X-linked spermatogenic failure 7. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 130184, 144634, 43.85, 10, 'ENSG00000169551', 'Q8WUE5'),
(537, 'CTAGE6', 'Predicted to be involved in endoplasmic reticulum to Golgi vesicle-mediated transport; protein secretion; and vesicle cargo loading. Predicted to be located in membrane. Predicted to be active in endoplasmic reticulum exit site and endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_J_gene', 22531939, 22531998, 53.33, 1, 'ENSG00000285123', 'Q86UF2'),
(538, 'CTAGE6', 'Predicted to be involved in endoplasmic reticulum to Golgi vesicle-mediated transport; protein secretion; and vesicle cargo loading. Predicted to be located in membrane. Predicted to be active in endoplasmic reticulum exit site and endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_J_gene', 22531939, 22531998, 53.33, 1, 'ENSG00000271321', 'Q86UF2'),
(539, 'CTF2P', 'The cytokine neuropoietin belongs to the IL-6 superfamily. This gene has been inactivated by mutation and is nonfunctional in humans. [provided by RefSeq, Mar 2013]', '16', 'protein_coding', 89829894, 89874436, 42.57, 6, 'ENSG00000230447', ''),
(540, 'CTNNA3', 'This gene encodes a protein that belongs to the vinculin/alpha-catenin family. The encoded protein plays a role in cell-cell adhesion in muscle cells. Mutations in this gene are associated with arrhythmogenic right ventricular dysplasia, familial 13. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2014]', '10', 'protein_coding', 165061, 167015, 49.00, 9, 'ENSG00000183230', 'Q9UI47'),
(541, 'CTPS2', 'The protein encoded by this gene catalyzes the formation of CTP from UTP with the concomitant deamination of glutamine to glutamate. This protein is the rate-limiting enzyme in the synthesis of cytosine nucleotides, which play an important role in various metabolic processes and provide the precursors necessary for the synthesis of RNA and DNA. Cancer cells that exhibit increased cell proliferation also exhibit an increased activity of this encoded protein. Thus, this protein is an attractive target for selective chemotherapy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', 'X', 'protein_coding', 75060573, 75088217, 40.50, 7, 'ENSG00000047230', 'Q9NRF8'),
(542, 'CTRB2', 'This gene encodes a member of the serine protease family of enzymes and forms a principal precursor of the pancreatic proteolytic enzymes. The encoded preproprotein is synthesized in the acinar cells of the pancreas and secreted into the small intestine where it undergoes proteolytic activation to generate a functional enzyme. This CTRB2 gene is located head-to-head with the related CTRB1 gene. Some human populations have an alternate haplotype which inverts a 16.6 Kb region containing portions of intron 1, exon 1, and the upstream sequence of the CTRB1 and CTRB2 genes. In this inversion haplotype exon 1 and flanking sequence is swapped in CTRB1 and CTRB2. This inversion is associated with differential gene expression and increased risk for chronic pancreatitis. The GRCh38 assembly represents the minor allele for SNP rs8048956 of the CTRB1 gene. SNP rs8048956 is diagnostic for this inversion. [provided by RefSeq, Jan 2021]', '16', 'protein_coding', 30554635, 30574665, 53.70, 22, 'ENSG00000168928', 'Q6GPI1'),
(543, 'CTRB2', 'This gene encodes a member of the serine protease family of enzymes and forms a principal precursor of the pancreatic proteolytic enzymes. The encoded preproprotein is synthesized in the acinar cells of the pancreas and secreted into the small intestine where it undergoes proteolytic activation to generate a functional enzyme. This CTRB2 gene is located head-to-head with the related CTRB1 gene. Some human populations have an alternate haplotype which inverts a 16.6 Kb region containing portions of intron 1, exon 1, and the upstream sequence of the CTRB1 and CTRB2 genes. In this inversion haplotype exon 1 and flanking sequence is swapped in CTRB1 and CTRB2. This inversion is associated with differential gene expression and increased risk for chronic pancreatitis. The GRCh38 assembly represents the minor allele for SNP rs8048956 of the CTRB1 gene. SNP rs8048956 is diagnostic for this inversion. [provided by RefSeq, Jan 2021]', '16', 'protein_coding', 30554635, 30574665, 53.70, 22, 'ENSG00000284810', 'Q6GPI1'),
(544, 'CTSA', 'This gene encodes a member of the peptidase S10 family of serine carboxypeptidases. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate two chains that comprise the heterodimeric active enzyme. This enzyme possesses deamidase, esterase and carboxypeptidase activities and acts as a scaffold in the lysosomal multienzyme complex. Mutations in this gene are associated with galactosialidosis. [provided by RefSeq, Nov 2015]', '20', 'protein_coding', 65961728, 65980137, 55.20, 8, 'ENSG00000064601', 'P10619'),
(545, 'CTSG', 'The protein encoded by this gene, a member of the peptidase S1 protein family, is found in azurophil granules of neutrophilic polymorphonuclear leukocytes. The encoded protease has a specificity similar to that of chymotrypsin C, and may participate in the killing and digestion of engulfed pathogens, and in connective tissue remodeling at sites of inflammation. In addition, the encoded protein is antimicrobial, with bacteriocidal activity against S. aureus and N. gonorrhoeae. Transcript variants utilizing alternative polyadenylation signals exist for this gene. [provided by RefSeq, Sep 2014]', '14', 'protein_coding', 948377, 951387, 58.85, 1, 'ENSG00000100448', 'P08311'),
(546, 'CTSL', 'The protein encoded by this gene is a lysosomal cysteine proteinase that plays a major role in intracellular protein catabolism. Its substrates include collagen and elastin, as well as alpha-1 protease inhibitor, a major controlling element of neutrophil elastase activity. The encoded protein has been implicated in several pathologic processes, including myofibril necrosis in myopathies and in myocardial ischemia, and in the renal tubular response to proteinuria. This protein, which is a member of the peptidase C1 family, is a dimer composed of disulfide-linked heavy and light chains, both produced from a single protein precursor. Additionally, this protein cleaves the S1 subunit of the SARS-CoV-2 spike protein, which is necessary for entry of the virus into the cell. [provided by RefSeq, Aug 2020]', '9', 'protein_coding', 31458080, 31467166, 58.18, 7, 'ENSG00000135047', 'P07711'),
(547, 'CTSZ', 'The protein encoded by this gene is a lysosomal cysteine proteinase and member of the peptidase C1 family. It exhibits both carboxy-monopeptidase and carboxy-dipeptidase activities. The encoded protein has also been known as cathepsin X and cathepsin P. This gene is expressed ubiquitously in cancer cell lines and primary tumors and, like other members of this family, may be involved in tumorigenesis. [provided by RefSeq, Oct 2008]', '20', 'protein_coding', 28659065, 28678959, 39.28, 12, 'ENSG00000101160', 'Q9UBR2'),
(548, 'CTTN-DT', 'Predicted to be involved in SRP-dependent cotranslational protein targeting to membrane, signal sequence recognition. Predicted to be part of signal recognition particle, endoplasmic reticulum targeting. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 16650158, 16654670, 42.32, 1, 'ENSG00000288166', ''),
(549, 'CTTN-DT', 'Predicted to be involved in SRP-dependent cotranslational protein targeting to membrane, signal sequence recognition. Predicted to be part of signal recognition particle, endoplasmic reticulum targeting. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 16650158, 16654670, 42.32, 1, 'ENSG00000246889', ''),
(550, 'CUEDC2', 'Predicted to enable ubiquitin binding activity. Acts upstream of or within negative regulation of cytokine production involved in inflammatory response and negative regulation of macrophage cytokine production. Located in cytosol; nuclear membrane; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 79970813, 80033949, 45.60, 12, 'ENSG00000107874', 'Q9H467'),
(551, 'CUL4B', 'This gene is a member of the cullin family. The encoded protein forms a complex that functions as an E3 ubiquitin ligase and catalyzes the polyubiquitination of specific protein substrates in the cell. The protein interacts with a ring finger protein, and is required for the proteolysis of several regulators of DNA replication including chromatin licensing and DNA replication factor 1 and cyclin E. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 93377883, 93408146, 41.97, 7, 'ENSG00000158290', 'Q13620'),
(552, 'CUL9', 'Predicted to enable ubiquitin protein ligase activity. Involved in microtubule cytoskeleton organization; protein ubiquitination; and regulation of mitotic nuclear division. Located in cytosol. Part of cullin-RING ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 18605815, 18611919, 62.54, 1, 'ENSG00000112659', 'Q8IWT3'),
(553, 'CWC22', 'Enables RNA binding activity. Involved in mRNA splicing, via spliceosome. Located in cytosol and nuclear speck. Part of U2-type catalytic step 1 spliceosome; U2-type catalytic step 2 spliceosome; and U2-type precatalytic spliceosome. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 32971, 35123, 58.76, 3, 'ENSG00000163510', 'Q9HCG8'),
(554, 'CX3CL1', 'This gene belongs to the CX3C subgroup of chemokines, characterized by the number of amino acids located between the conserved cysteine residues. This is the only member of the CX3C subgroup, which contains three amino acids between cysteine residues, resulting in a Cys-X-X-X-Cys configuration. The encoded protein contains an extended mucin-like stalk with a chemokine domain on top, and exists in both a membrane-anchored form where it acts as a binding molecule, or, in soluble form, as a chemotactic cytokine. The mature form of this protein can be cleaved at the cell surface, yielding different soluble forms that can interact with the G-protein coupled receptor, C-X3-C motif chemokine receptor 1 gene product. This gene plays a role in a wide range of diseases, including cancer, vasculitis, neuropathies, atherosclerosis, inflammatory diseases, and in human immunodeficiency virus infections. [provided by RefSeq, Sep 2017]', '16', 'protein_coding', 97851677, 97872542, 38.47, 17, 'ENSG00000006210', 'P78423'),
(555, 'CXADR', 'The protein encoded by this gene is a type I membrane receptor for group B coxsackieviruses and subgroup C adenoviruses. Several transcript variants encoding different isoforms have been found for this gene. Pseudogenes of this gene are found on chromosomes 15, 18, and 21. [provided by RefSeq, May 2011]', '21', 'miRNA', 8432530, 8432621, 83.70, 1, 'ENSG00000154639', 'P78310'),
(556, 'CXorf38', 'Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 19145569, 19192158, 54.55, 7, 'ENSG00000185753', 'Q8TB03'),
(557, 'CXorf66', 'The protein encoded by this gene is predicted to be a type I membrane protein, however, its exact function is not known. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 20389530, 20712644, 40.43, 5, 'ENSG00000203933', 'Q5JRM2'),
(558, 'CYB561', 'Predicted to enable transmembrane monodehydroascorbate reductase activity. Predicted to be involved in ascorbate homeostasis. Predicted to be located in chromaffin granule membrane. Predicted to be active in lysosomal membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 39147811, 39170532, 40.78, 7, 'ENSG00000008283', 'P49447'),
(559, 'CYB5A', 'The protein encoded by this gene is a membrane-bound cytochrome that reduces ferric hemoglobin (methemoglobin) to ferrous hemoglobin, which is required for stearyl-CoA-desaturase activity. Defects in this gene are a cause of type IV hereditary methemoglobinemia. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jun 2010]', '18', 'protein_coding', 26667020, 26723427, 36.41, 5, 'ENSG00000166347', 'P00167'),
(560, 'CYB5D2', 'Predicted to enable heme binding activity. Predicted to be involved in nervous system development. Predicted to act upstream of or within positive regulation of neuron differentiation. Predicted to be located in extracellular region. Predicted to be active in endomembrane system and membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 154389955, 154409168, 54.28, 16, 'ENSG00000167740', 'Q8WUJ1'),
(561, 'CYFIP1', 'This gene encodes a protein that regulates cytoskeletal dynamics and protein translation. The encoded protein is a component of the WAVE regulatory complex (WRC), which promotes actin polymerization. This protein also interacts with the synaptic functional regulator FMR1 protein and translation initiation factor 4E to inhibit protein translation. A large chromosomal deletion including this gene is associated with increased risk of schizophrenia and epilepsy in human patients. Reduced expression of this gene has been observed in various human cancers and the encoded protein may inhibit tumor invasion. [provided by RefSeq, Mar 2022]', '15', 'protein_coding', 15102584, 15168693, 43.09, 5, 'ENSG00000288461', 'Q7L576'),
(562, 'CYFIP1', 'This gene encodes a protein that regulates cytoskeletal dynamics and protein translation. The encoded protein is a component of the WAVE regulatory complex (WRC), which promotes actin polymerization. This protein also interacts with the synaptic functional regulator FMR1 protein and translation initiation factor 4E to inhibit protein translation. A large chromosomal deletion including this gene is associated with increased risk of schizophrenia and epilepsy in human patients. Reduced expression of this gene has been observed in various human cancers and the encoded protein may inhibit tumor invasion. [provided by RefSeq, Mar 2022]', '15', 'protein_coding', 15102584, 15168693, 43.09, 5, 'ENSG00000280618', 'Q7L576'),
(563, 'CYFIP1', 'This gene encodes a protein that regulates cytoskeletal dynamics and protein translation. The encoded protein is a component of the WAVE regulatory complex (WRC), which promotes actin polymerization. This protein also interacts with the synaptic functional regulator FMR1 protein and translation initiation factor 4E to inhibit protein translation. A large chromosomal deletion including this gene is associated with increased risk of schizophrenia and epilepsy in human patients. Reduced expression of this gene has been observed in various human cancers and the encoded protein may inhibit tumor invasion. [provided by RefSeq, Mar 2022]', '15', 'protein_coding', 15102584, 15168693, 43.09, 5, 'ENSG00000273749', 'Q7L576'),
(564, 'CYLC1', 'This gene encodes a sperm head cytoskeletal protein. The encoded protein is associated with the calyx of spermatozoa and spermatids. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2012]', 'X', 'protein_coding', 174312, 177520, 50.45, 9, 'ENSG00000183035', 'P35663'),
(565, 'CYLC2', 'Cylicin II (CYCL2) is specifically expressed in testis and is part of the cytoskeletal calyx of mammalian sperm heads.  Cylicin II may play a role in the morphogenesis of the sperm head. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 83512534, 83525704, 39.67, 1, 'ENSG00000155833', 'Q14093'),
(566, 'CYLD', 'This gene is encodes a cytoplasmic protein with three cytoskeletal-associated protein-glycine-conserved (CAP-GLY) domains that functions as a deubiquitinating enzyme. Mutations in this gene have been associated with cylindromatosis, multiple familial trichoepithelioma, and Brooke-Spiegler syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 69326913, 69330588, 54.49, 1, 'ENSG00000083799', 'Q9NQC7'),
(567, 'CYP11A1', 'This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the mitochondrial inner membrane and catalyzes the conversion of cholesterol to pregnenolone, the first and rate-limiting step in the synthesis of the steroid hormones. Two transcript variants encoding different isoforms have been found for this gene. The cellular location of the smaller isoform is unclear since it lacks the mitochondrial-targeting transit peptide. [provided by RefSeq, Jul 2008]', '15', 'protein_coding', 55160167, 55167297, 38.84, 8, 'ENSG00000288362', 'P05108'),
(568, 'CYP11A1', 'This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the mitochondrial inner membrane and catalyzes the conversion of cholesterol to pregnenolone, the first and rate-limiting step in the synthesis of the steroid hormones. Two transcript variants encoding different isoforms have been found for this gene. The cellular location of the smaller isoform is unclear since it lacks the mitochondrial-targeting transit peptide. [provided by RefSeq, Jul 2008]', '15', 'protein_coding', 55160167, 55167297, 38.84, 8, 'ENSG00000140459', 'P05108'),
(569, 'CYP24A1', 'This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This mitochondrial protein initiates the degradation of 1,25-dihydroxyvitamin D3, the physiologically active form of vitamin D3, by hydroxylation of the side chain. In regulating the level of vitamin D3, this enzyme plays a role in calcium homeostasis and the vitamin D endocrine system. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 42902086, 42919563, 45.56, 9, 'ENSG00000019186', 'Q07973'),
(570, 'CYP2A13', 'This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum. Although its endogenous substrate has not been determined, it is known to metabolize 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone, a major nitrosamine specific to tobacco. This gene is part of a large cluster of cytochrome P450 genes from the CYP2A, CYP2B and CYP2F subfamilies on chromosome 19q. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 42915989, 42925838, 50.96, 4, 'ENSG00000197838', 'Q16696'),
(571, 'CYP2G2P', 'Predicted to enable several functions, including heme binding activity; iron ion binding activity; and monooxygenase activity. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 159778498, 159789703, 40.84, 18, 'ENSG00000268101', ''),
(572, 'CYP3A7', 'This gene encodes a member of the cytochrome P450 superfamily of enzymes, which participate in drug metabolism and the synthesis of cholesterol, steroids and other lipids. This enzyme hydroxylates testosterone and dehydroepiandrosterone 3-sulphate, which is involved in the formation of estriol during pregnancy. This gene is part of a cluster of related genes on chromosome 7q21.1. Naturally-occurring readthrough transcription occurs between this gene and the downstream CYP3A51P pseudogene and is represented by GeneID:100861540. [provided by RefSeq, Jan 2015]', '7', 'protein_coding', 81175452, 81221377, 37.75, 4, 'ENSG00000160870', 'P24462'),
(573, 'CYSLTR1', 'This gene encodes a member of the G-protein coupled receptor 1 family. The encoded protein is a receptor for cysteinyl leukotrienes, and is involved in mediating bronchoconstriction via activation of a phosphatidylinositol-calcium second messenger system. Activation of the encoded receptor results in contraction and proliferation of bronchial smooth muscle cells, eosinophil migration, and damage to the mucus layer in the lung. Upregulation of this gene is associated with asthma and dysregulation may also be implicated in cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', 'X', 'miRNA', 113262920, 113262999, 56.25, 1, 'ENSG00000173198', 'Q9Y271'),
(574, 'CYSTM1', 'Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 56938199, 56961050, 42.99, 2, 'ENSG00000120306', 'Q9H1C7'),
(575, 'CYTL1', 'C17 is a cytokine-like protein specifically expressed in bone marrow and cord blood mononuclear cells that bear the CD34 (MIM 142230) surface marker (Liu et al., 2000 [PubMed 10857752]).[supplied by OMIM, Mar 2008]', '4', 'protein_coding', 39755025, 40106089, 37.04, 4, 'ENSG00000170891', 'Q9NRR1'),
(576, 'CYYR1', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 44275458, 44321494, 41.62, 10, 'ENSG00000166265', 'Q96J86'),
(577, 'DAGLB', 'Enables lipase activity. Involved in arachidonate metabolic process. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 50888916, 50931437, 41.67, 9, 'ENSG00000164535', 'Q8NCG7'),
(578, 'DAOA', 'This gene encodes a protein that may function as an activator of D-amino acid oxidase, which degrades the gliotransmitter D-serine, a potent activator of N-methyl-D-aspartate (NMDA) type glutamate receptors. Studies also suggest that one encoded isoform may play a role in mitochondrial function and dendritic arborization. Polymorphisms in this gene have been implicated in susceptibility to schizophrenia and bipolar affective disorder. Alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Mar 2011]', '13', 'protein_coding', 5014586, 5019458, 42.97, 3, 'ENSG00000182346', 'P59103'),
(579, 'DAZ3', 'This gene is a member of the DAZ gene family and is a candidate for the human Y-chromosomal azoospermia factor (AZF). Its expression is restricted to premeiotic germ cells, particularly in spermatogonia. It encodes an RNA-binding protein that is important for spermatogenesis. Four copies of this gene are found on chromosome Y within palindromic duplications; one pair of genes is part of the P2 palindrome and the second pair is part of the P1 palindrome. Each gene contains a 2.4 kb repeat including a 72-bp exon, called the DAZ repeat; the number of DAZ repeats is variable and there are several variations in the sequence of the DAZ repeat. Each copy of the gene also contains a 10.8 kb region that may be amplified; this region includes five exons that encode an RNA recognition motif (RRM) domain. This gene contains one copy of the 10.8 kb repeat. [provided by RefSeq, Jul 2008]', 'Y', 'lncRNA', 24570202, 24607025, 42.24, 1, 'ENSG00000187191', 'Q9NR90'),
(580, 'DAZAP1', 'In mammals, the Y chromosome directs the development of the testes and plays an important role in spermatogenesis. A high percentage of infertile men have deletions that map to regions of the Y chromosome. The DAZ (deleted in azoospermia) gene cluster maps to the AZFc region of the Y chromosome and is deleted in many azoospermic and severely oligospermic men. It is thought that the DAZ gene cluster arose from the transposition, amplification, and pruning of the ancestral autosomal gene DAZL also involved in germ cell development and gametogenesis. This gene encodes a RNA-binding protein with two RNP motifs that was originally identified by its interaction with the infertility factors DAZ and DAZL. Two isoforms are encoded by transcript variants of this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 49474207, 49486231, 53.99, 12, 'ENSG00000071626', 'Q96EP5'),
(581, 'DAZAP2', 'This gene encodes a proline-rich protein which interacts with the deleted in azoospermia (DAZ) and the deleted in azoospermia-like gene through the DAZ-like repeats. This protein also interacts with the transforming growth factor-beta signaling molecule SARA (Smad anchor for receptor activation), eukaryotic initiation factor 4G, and an E3 ubiquitinase that regulates its stability in splicing factor containing nuclear speckles. The encoded protein may function in various biological and pathological processes including spermatogenesis, cell signaling and transcription regulation, formation of stress granules during translation arrest, RNA splicing, and pathogenesis of multiple myeloma. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2008]', '12', 'protein_coding', 113979391, 114035572, 45.30, 5, 'ENSG00000183283', 'Q15038'),
(582, 'DBNDD2', 'Predicted to enable ATPase binding activity. Involved in negative regulation of protein kinase activity. Predicted to be located in cytoplasm. Predicted to be active in endoplasmic reticulum and lysosome. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 38561969, 38603586, 37.07, 12, 'ENSG00000244274', 'Q9BQY9'),
(583, 'DBNL', 'Enables cadherin binding activity. Predicted to be involved in several processes, including Rac protein signal transduction; nervous system development; and podosome assembly. Located in cytoplasm; intracellular membrane-bounded organelle; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 102419189, 102423136, 61.93, 6, 'ENSG00000136279', 'Q9UJU6'),
(584, 'DCAF5', 'Predicted to be involved in negative regulation of fatty acid biosynthetic process. Part of Cul4-RING E3 ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 159725585, 159756319, 38.63, 8, 'ENSG00000139990', 'Q96JK2'),
(585, 'DCAF8L1', 'This gene encodes a member of the WD repeat protein family. WD repeats are minimally conserved regions of approximately 40 amino acids typically bracketed by Gly-His and Trp-Asp (GH-WD), which may facilitate the formation of heterotrimeric or multi-protein complexes. Members of this family are involved in a variety of cellular processes, including cell cycle progression, signal transduction, apoptosis, and gene regulation. This gene appears to represent an intronless retrocopy of a related multi-exon gene located on chromosome 1. However, the CDS of this intronless gene remains intact, it is conserved in other primate species, it is known to be transcribed, and it is therefore thought to encode a functional protein. [provided by RefSeq, May 2010]', 'X', 'protein_coding', 92291167, 92353964, 40.45, 4, 'ENSG00000226372', 'A6NGE4'),
(586, 'DCLK2', 'This gene encodes a member of the protein kinase superfamily and the doublecortin family. The protein encoded by this gene contains two N-terminal doublecortin domains, which bind microtubules and regulate microtubule polymerization, a C-terminal serine/threonine protein kinase domain, which shows substantial homology to Ca2+/calmodulin-dependent protein kinase, and a serine/proline-rich domain in between the doublecortin and the protein kinase domains, which mediates multiple protein-protein interactions. The microtubule-polymerizing activity of the encoded protein is independent of its protein kinase activity. Mouse studies show that the DCX gene, another family member, and this gene share function in the establishment of hippocampal organization and that their absence results in a severe epileptic phenotype and lethality, as described in human patients with lissencephaly. Multiple alternatively spliced transcript variants have been identified. [provided by RefSeq, Sep 2010]', '4', 'protein_coding', 64934471, 64935893, 57.70, 3, 'ENSG00000170390', 'Q8N568'),
(587, 'DCLRE1A', 'This gene encodes a conserved protein that is involved in the repair of DNA interstrand cross-links. DNA cross-links suppress transcription, replication, and DNA segregation. The encoded protein is a regulator of the mitotic cell cycle checkpoint. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2012]', '10', 'miRNA', 213758067, 213758159, 39.78, 1, 'ENSG00000198924', 'Q6PJP8'),
(588, 'DCN', 'This gene encodes a member of the small leucine-rich proteoglycan family of proteins. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature protein. This protein plays a role in collagen fibril assembly. Binding of this protein to multiple cell surface receptors mediates its role in tumor suppression, including a stimulatory effect on autophagy and inflammation and an inhibitory effect on angiogenesis and tumorigenesis. This gene and the related gene biglycan are thought to be the result of a gene duplication. Mutations in this gene are associated with congenital stromal corneal dystrophy in human patients. [provided by RefSeq, Nov 2015]', '12', 'protein_coding', 189441446, 189475552, 37.02, 7, 'ENSG00000011465', 'P07585'),
(589, 'DCP1B', 'This gene encodes a member of a family of proteins that function in removing the 5\' cap from mRNAs, which is a step in regulated mRNA decay. This protein localizes to cytoplasmic foci which are the site of mRNA breakdown and turnover. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 108218392, 108234243, 35.41, 1, 'ENSG00000284850', 'Q8IZD4'),
(590, 'DCP1B', 'This gene encodes a member of a family of proteins that function in removing the 5\' cap from mRNAs, which is a step in regulated mRNA decay. This protein localizes to cytoplasmic foci which are the site of mRNA breakdown and turnover. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 108218392, 108234243, 35.41, 1, 'ENSG00000151065', 'Q8IZD4'),
(591, 'DCTN3', 'This gene encodes the smallest subunit of dynactin, a macromolecular complex consisting of 10 subunits ranging in size from 22 to 150 kD. Dynactin binds to both microtubules and cytoplasmic dynein. It is involved in a diverse array of cellular functions, including ER-to-Golgi transport, the centripetal movement of lysosomes and endosomes, spindle formation, cytokinesis, chromosome movement, nuclear positioning, and axonogenesis. This subunit, like most other dynactin subunits, exists only as a part of the dynactin complex. It is primarily an alpha-helical protein with very little coiled coil, and binds directly to the largest subunit (p150) of dynactin. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '9', 'lncRNA', 254689, 262186, 51.44, 1, 'ENSG00000137100', 'O75935'),
(592, 'DCTN5', 'This gene encodes a subunit of dynactin, a component of the cytoplasmic dynein motor machinery involved in minus-end-directed transport. The encoded protein is a component of the pointed-end subcomplex and is thought to bind membranous cargo. A pseudogene of this gene is located on the long arm of chromosome 1. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jan 2011]', '16', 'protein_coding', 211057, 216060, 48.50, 3, 'ENSG00000166847', 'Q9BTE1'),
(593, 'DCTPP1', 'The protein encoded by this gene is dCTP pyrophosphatase, which converts dCTP to dCMP and inorganic pyrophosphate. The encoded protein also displays weak activity against dTTP and dATP, but none against dGTP. This protein may be responsible for eliminating excess dCTP after DNA synthesis and may prevent overmethylation of CpG islands. Three transcript variants, one protein-coding and the other two non-protein coding, have been found for this gene. [provided by RefSeq, Dec 2015]', '16', 'protein_coding', 6409126, 6484190, 48.19, 10, 'ENSG00000179958', 'Q9H773'),
(594, 'DDI1', 'Predicted to enable aspartic-type endopeptidase activity. Involved in several processes, including cellular response to hydroxyurea; regulation of DNA stability; and regulation of protein stability. [provided by Alliance of Genome Resources, May 2025]', '11', 'transcribed_unprocessed_pseudogene', 290243, 291178, 44.02, 1, 'ENSG00000170967', 'Q8WTU0'),
(595, 'DDIT4', 'Predicted to enable 14-3-3 protein binding activity. Involved in negative regulation of TOR signaling and response to hypoxia. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 198916, 201430, 58.65, 2, 'ENSG00000168209', 'Q9NX09'),
(596, 'DDRGK1', 'The protein encoded by this gene interacts with components of the ubiquitin fold modifier 1 conjugation pathway and helps prevent apoptosis in ER-stressed secretory tissues. In addition, the encoded protein regulates nuclear factor-&#954;B activity. [provided by RefSeq, Dec 2015]', '20', 'protein_coding', 7566782, 8406861, 40.10, 15, 'ENSG00000198171', 'Q96HY6'),
(597, 'DDX18', 'DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene encodes a DEAD box protein, and it is activated by Myc protein. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 5523302, 5550862, 39.14, 1, 'ENSG00000088205', 'Q9NVP1'),
(598, 'DDX24', 'DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene encodes a DEAD box protein, which shows little similarity to any of the other known human DEAD box proteins, but shows a high similarity to mouse Ddx24 at the amino acid level. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 30104810, 30113537, 53.57, 5, 'ENSG00000273761', 'Q9GZR7'),
(599, 'DDX24', 'DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene encodes a DEAD box protein, which shows little similarity to any of the other known human DEAD box proteins, but shows a high similarity to mouse Ddx24 at the amino acid level. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 30104810, 30113537, 53.57, 5, 'ENSG00000089737', 'Q9GZR7'),
(600, 'DDX28', 'DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure, such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of the DEAD box protein family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene is intronless. It encodes an RNA-dependent ATPase. The encoded protein is localized in the mitochondria and the nucleus, and can be transported between the mitochondria and the nucleus. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 2160040, 2215515, 42.29, 9, 'ENSG00000182810', 'Q9NUL7');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(601, 'DDX41', 'DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure, such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of the DEAD box protein family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. The protein encoded by this gene is a member of the DEAD box protein family and interacts with several spliceosomal proteins. In addition, the encoded protein may recognize the bacterial second messengers cyclic di-GMP and cyclic di-AMP, resulting in the induction of genes involved in the innate immune response. [provided by RefSeq, Jan 2017]', '5', 'protein_coding', 9986126, 10058303, 37.26, 4, 'ENSG00000183258', 'Q9UJV9'),
(602, 'DDX43', ' The protein encoded by this gene is an ATP-dependent RNA helicase in the DEAD-box family and displays tumor-specific expression. [provided by RefSeq, Jul 2008]', '6', 'transcribed_unitary_pseudogene', 58996734, 59069338, 44.18, 1, 'ENSG00000080007', 'Q9NXZ2'),
(603, 'DDX51', 'Enables RNA binding activity. Predicted to be involved in rRNA processing. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 74493756, 74497106, 45.27, 4, 'ENSG00000288503', 'Q8N8A6'),
(604, 'DDX51', 'Enables RNA binding activity. Predicted to be involved in rRNA processing. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 74493756, 74497106, 45.27, 4, 'ENSG00000185163', 'Q8N8A6'),
(605, 'DDX55', 'This gene encodes a member of protein family containing a characteristic Asp-Glu-Ala-Asp (DEAD) motif. These proteins are putative RNA helicases, and may be involved in a range of nuclear processes including translational initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Multiple alternatively spliced transcript variants have been found for this gene. Pseudogenes have been identified on chromosomes 1 and 12. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 48890197, 48903402, 54.42, 19, 'ENSG00000111364', 'Q8NHQ9'),
(606, 'DEFA4', 'Defensins are a family of antimicrobial and cytotoxic peptides thought to be involved in host defense. They are abundant in the granules of neutrophils and also found in the epithelia of mucosal surfaces such as those of the intestine, respiratory tract, urinary tract, and vagina. Members of the defensin family are highly similar in protein sequence and distinguished by a conserved cysteine motif. Several alpha defensin genes are clustered on chromosome 8. This gene differs from other genes of this family by an extra 83-base segment that is apparently the result of a recent duplication within the coding region. The protein encoded by this gene, defensin, alpha 4, is found in the neutrophils; it exhibits corticostatic activity and inhibits corticotropin stimulated corticosterone production. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 31234152, 31269451, 50.24, 1, 'ENSG00000285318', 'P12838'),
(607, 'DEFA4', 'Defensins are a family of antimicrobial and cytotoxic peptides thought to be involved in host defense. They are abundant in the granules of neutrophils and also found in the epithelia of mucosal surfaces such as those of the intestine, respiratory tract, urinary tract, and vagina. Members of the defensin family are highly similar in protein sequence and distinguished by a conserved cysteine motif. Several alpha defensin genes are clustered on chromosome 8. This gene differs from other genes of this family by an extra 83-base segment that is apparently the result of a recent duplication within the coding region. The protein encoded by this gene, defensin, alpha 4, is found in the neutrophils; it exhibits corticostatic activity and inhibits corticotropin stimulated corticosterone production. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 31234152, 31269451, 50.24, 1, 'ENSG00000164821', 'P12838'),
(608, 'DEFB105B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 105, DEFB105A and DEFB105B, in tail-to-tail orientation. This gene, DEFB105B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 101451564, 101872562, 37.74, 7, 'ENSG00000285015', 'Q8NG35'),
(609, 'DEFB105B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 105, DEFB105A and DEFB105B, in tail-to-tail orientation. This gene, DEFB105B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 101451564, 101872562, 37.74, 7, 'ENSG00000274729', 'Q8NG35'),
(610, 'DEFB105B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 105, DEFB105A and DEFB105B, in tail-to-tail orientation. This gene, DEFB105B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 101451564, 101872562, 37.74, 7, 'ENSG00000186599', 'Q8NG35'),
(611, 'DEFB106A', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 106, DEFB106A and DEFB106B, in head-to-head orientation. This gene, DEFB106A, represents the more centromeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 29055805, 29073797, 38.32, 12, 'ENSG00000284937', 'Q8N104'),
(612, 'DEFB106A', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 106, DEFB106A and DEFB106B, in head-to-head orientation. This gene, DEFB106A, represents the more centromeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 29055805, 29073797, 38.32, 12, 'ENSG00000186579', 'Q8N104'),
(613, 'DEFB106B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 106, DEFB106A and DEFB106B, in head-to-head orientation. This gene, DEFB106B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 81489703, 81761645, 35.25, 6, 'ENSG00000285617', 'Q8N104'),
(614, 'DEFB106B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 106, DEFB106A and DEFB106B, in head-to-head orientation. This gene, DEFB106B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 81489703, 81761645, 35.25, 6, 'ENSG00000275794', 'Q8N104'),
(615, 'DEFB106B', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 106, DEFB106A and DEFB106B, in head-to-head orientation. This gene, DEFB106B, represents the more telomeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 81489703, 81761645, 35.25, 6, 'ENSG00000187082', 'Q8N104'),
(616, 'DEFB107A', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 107, DEFB107A and DEFB107B, in tail-to-tail orientation. This gene, DEFB107A, represents the more centromeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 55719627, 55732121, 40.18, 6, 'ENSG00000285432', 'Q8IZN7'),
(617, 'DEFB107A', 'Defensins form a family of antimicrobial and cytotoxic peptides made by neutrophils. Defensins are short, processed peptide molecules that are classified by structure into three groups: alpha-defensins, beta-defensins and theta-defensins. All beta-defensin genes are densely clustered in four to five syntenic chromosomal regions. Chromosome 8p23 contains at least two copies of the duplicated beta-defensin cluster. This duplication results in two identical copies of defensin, beta 107, DEFB107A and DEFB107B, in tail-to-tail orientation. This gene, DEFB107A, represents the more centromeric copy. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 55719627, 55732121, 40.18, 6, 'ENSG00000186572', 'Q8IZN7'),
(618, 'DEFB108C', 'Predicted to be involved in defense response to bacterium and innate immune response. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 118033618, 118258634, 36.26, 2, 'ENSG00000284958', 'A8MXU0'),
(619, 'DEFB108C', 'Predicted to be involved in defense response to bacterium and innate immune response. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 118033618, 118258634, 36.26, 2, 'ENSG00000276378', 'A8MXU0'),
(620, 'DEFB108C', 'Predicted to be involved in defense response to bacterium and innate immune response. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 118033618, 118258634, 36.26, 2, 'ENSG00000215371', 'A8MXU0'),
(621, 'DEFB109D', 'Predicted to be involved in defense response to bacterium. Predicted to be located in Golgi lumen and extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 34481758, 34512304, 54.89, 2, 'ENSG00000284926', 'P0DY26'),
(622, 'DEFB109D', 'Predicted to be involved in defense response to bacterium. Predicted to be located in Golgi lumen and extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 34481758, 34512304, 54.89, 2, 'ENSG00000254866', 'P0DY26'),
(623, 'DEFB118', 'This gene encodes a member of the beta subfamily of defensins. Beta-defensins are antimicrobial peptides that protect tissues and organs from infection by a variety of microorganisms. Expression of this gene is regulated by androgen, and the encoded protein binds to sperm and exhibits antibacterial activity against E. coli. This gene is found in a cluster with other beta-defensin genes on the long arm of chromosome 20. [provided by RefSeq, Nov 2014]', '20', 'protein_coding', 1850904, 1892267, 61.57, 18, 'ENSG00000131068', 'Q96PH6'),
(624, 'DEFB119', 'This gene encodes a member of the beta subfamily of defensins. Beta-defensins are antimicrobial peptides that protect tissues and organs from infection by a variety of microorganisms. This gene is found in a cluster with other beta-defensin genes on the long arm of chromosome 20. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2014]', '20', 'protein_coding', 107065182, 107115515, 47.60, 2, 'ENSG00000180483', 'Q8N690'),
(625, 'DEFB124', 'Defensins are cysteine-rich cationic polypeptides that are important in the host immunologic response to invading microorganisms. This antimicrobial protein is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 20q11.1. The encoded protein may serve to enhance innate immunity in the prostate. [provided by RefSeq, Nov 2014]', '20', 'protein_coding', 4377, 9347, 58.32, 5, 'ENSG00000180383', 'Q8NES8'),
(626, 'DEFB125', 'Defensins are cysteine-rich cationic polypeptides that are important in the host immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 20p13. [provided by RefSeq, Nov 2014]', '20', 'protein_coding', 34668994, 34718227, 45.91, 2, 'ENSG00000178591', 'Q8N687'),
(627, 'DEFB130B', 'Predicted to enable CCR6 chemokine receptor binding activity and chemoattractant activity. Predicted to be involved in cell chemotaxis and defense response to bacterium. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 155276211, 155334657, 38.46, 4, 'ENSG00000285249', 'P0DP73'),
(628, 'DEFB130B', 'Predicted to enable CCR6 chemokine receptor binding activity and chemoattractant activity. Predicted to be involved in cell chemotaxis and defense response to bacterium. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 155276211, 155334657, 38.46, 4, 'ENSG00000233050', 'P0DP73'),
(629, 'DEFB131B', 'Predicted to be involved in defense response to bacterium and innate immune response. Predicted to be located in extracellular region. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 179562886, 179588407, 39.25, 14, 'ENSG00000225805', 'A0A096LNP1'),
(630, 'DEFB134', 'Defensins are cysteine-rich cationic polypeptides that are important in the immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 8p23. [provided by RefSeq, Nov 2014]', '8', 'protein_coding', 122399551, 122433119, 35.55, 3, 'ENSG00000284754', 'Q4QY38'),
(631, 'DEFB134', 'Defensins are cysteine-rich cationic polypeptides that are important in the immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 8p23. [provided by RefSeq, Nov 2014]', '8', 'protein_coding', 122399551, 122433119, 35.55, 3, 'ENSG00000205882', 'Q4QY38'),
(632, 'DEFB136', 'Defensins are cysteine-rich cationic polypeptides that are important in the immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 8p23. [provided by RefSeq, Nov 2014]', '8', 'protein_coding', 92528773, 92540481, 40.17, 3, 'ENSG00000285087', 'Q30KP8'),
(633, 'DEFB136', 'Defensins are cysteine-rich cationic polypeptides that are important in the immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 8p23. [provided by RefSeq, Nov 2014]', '8', 'protein_coding', 92528773, 92540481, 40.17, 3, 'ENSG00000205884', 'Q30KP8'),
(634, 'DEFB4A', 'Defensins form a family of microbicidal and cytotoxic peptides made by neutrophils. Members of the defensin family are highly similar in protein sequence. This gene encodes defensin, beta 4, an antibiotic peptide which is locally regulated by inflammation. [provided by RefSeq, Jul 2008]', '8', 'miRNA', 30117086, 30117160, 50.67, 1, 'ENSG00000285181', 'O15263'),
(635, 'DEFB4A', 'Defensins form a family of microbicidal and cytotoxic peptides made by neutrophils. Members of the defensin family are highly similar in protein sequence. This gene encodes defensin, beta 4, an antibiotic peptide which is locally regulated by inflammation. [provided by RefSeq, Jul 2008]', '8', 'miRNA', 30117086, 30117160, 50.67, 1, 'ENSG00000171711', 'O15263'),
(636, 'DEFB4B', 'Defensins form a family of microbicidal and cytotoxic peptides made by neutrophils. Members of the defensin family are highly similar in protein sequence. This gene encodes defensin, beta 4, an antibiotic peptide which is locally regulated by inflammation. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 104936616, 105083332, 36.80, 3, 'ENSG00000285433', 'O15263'),
(637, 'DEFB4B', 'Defensins form a family of microbicidal and cytotoxic peptides made by neutrophils. Members of the defensin family are highly similar in protein sequence. This gene encodes defensin, beta 4, an antibiotic peptide which is locally regulated by inflammation. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 104936616, 105083332, 36.80, 3, 'ENSG00000275444', 'O15263'),
(638, 'DEFB4B', 'Defensins form a family of microbicidal and cytotoxic peptides made by neutrophils. Members of the defensin family are highly similar in protein sequence. This gene encodes defensin, beta 4, an antibiotic peptide which is locally regulated by inflammation. [provided by RefSeq, Oct 2014]', '8', 'protein_coding', 104936616, 105083332, 36.80, 3, 'ENSG00000177257', 'O15263'),
(639, 'DENND11', 'Predicted to enable guanyl-nucleotide exchange factor activity. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_V_gene', 142492132, 142492673, 51.85, 1, 'ENSG00000262599', 'A4D1U4'),
(640, 'DENND11', 'Predicted to enable guanyl-nucleotide exchange factor activity. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_V_gene', 142492132, 142492673, 51.85, 1, 'ENSG00000257093', 'A4D1U4'),
(641, 'DERA', 'Enables deoxyribose-phosphate aldolase activity. Involved in deoxyribonucleoside catabolic process. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'miRNA', 3485381, 3485469, 65.17, 1, 'ENSG00000023697', 'Q9Y315'),
(642, 'DGCR2', 'Deletions of the 22q11.2 have been associated with a wide range of developmental defects (notably DiGeorge syndrome, velocardiofacial syndrome, conotruncal anomaly face syndrome and isolated conotruncal cardiac defects) classified under the acronym CATCH 22. The DGCR2 gene encodes a novel putative adhesion receptor protein, which could play a role in neural crest cells migration, a process which has been proposed to be altered in DiGeorge syndrome. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010]', '22', 'protein_coding', 37231631, 37253403, 38.97, 13, 'ENSG00000070413', 'P98153'),
(643, 'DGCR6L', 'This gene, the result of a duplication at this locus, is one of two functional genes encoding nearly identical proteins that have similar expression patterns. The product of this gene is a protein that shares homology with the Drosophila gonadal protein, expressed in gonadal tissues and germ cells, and with the human laminin gamma-1 chain that functions in cell attachment and migration. This gene is located in a region of chromosome 22 implicated in the DiGeorge syndrome, one facet of a broader collection of anomalies referred to as the CATCH 22 syndrome. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 2969270, 2973484, 64.41, 5, 'ENSG00000128185', 'Q9BY27'),
(644, 'DGKH', 'This gene encodes a member of the diacylglycerol kinase (DGK) enzyme family. Members of this family are involved in regulating intracellular concentrations of diacylglycerol and phosphatidic acid. Variation in this gene has been associated with bipolar disorder. Alternatively spliced transcript variants have been identified. [provided by RefSeq, Jul 2014]', '13', 'protein_coding', 48053323, 48095131, 40.77, 4, 'ENSG00000102780', 'Q86XP1'),
(645, 'DHDH', 'This gene encodes an enzyme that belongs to the family of dihydrodiol dehydrogenases, which exist in multiple forms in mammalian tissues and are involved in the metabolism of xenobiotics and sugars. These enzymes catalyze the NADP1-linked oxidation of transdihydrodiols of aromatic hydrocarbons to corresponding catechols. This enzyme is a dimeric dihydrodiol dehydrogenase, and it differs from monomeric dihydrodiol dehydrogenases in its high substrate specificity for trans-dihydrodiols of aromatic hydrocarbons in the oxidative direction. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 101784443, 101818465, 41.92, 11, 'ENSG00000104808', 'Q9UQ10'),
(646, 'DHRS4', 'Enables identical protein binding activity; oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor; and oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor. Involved in several processes, including ketone metabolic process; positive regulation of reactive oxygen species metabolic process; and steroid metabolic process. Located in nucleus and peroxisomal membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 22928024, 23038523, 38.55, 13, 'ENSG00000284807', 'Q9BTZ2'),
(647, 'DHRS4', 'Enables identical protein binding activity; oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor; and oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor. Involved in several processes, including ketone metabolic process; positive regulation of reactive oxygen species metabolic process; and steroid metabolic process. Located in nucleus and peroxisomal membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 22928024, 23038523, 38.55, 13, 'ENSG00000157326', 'Q9BTZ2'),
(648, 'DHRS4L2', 'This gene encodes a member of the short chain dehydrogenase reductase family. The encoded protein may be an NADPH dependent retinol oxidoreductase. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Aug 2010]', '14', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000187630', 'Q6PKH6'),
(649, 'DHRS7', 'This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) family, which has over 46,000 members. Members in this family are enzymes that metabolize many different compounds, such as steroid hormones, prostaglandins, retinoids, lipids and xenobiotics. [provided by RefSeq, Apr 2016]', '14', 'protein_coding', 35648925, 35664956, 39.80, 6, 'ENSG00000100612', 'Q9Y394'),
(650, 'DHRS7C', 'Predicted to enable all-trans-retinol dehydrogenase (NAD+) activity. Predicted to be involved in D-glucose import; intracellular calcium ion homeostasis; and regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum. Predicted to be located in longitudinal sarcoplasmic reticulum and sarcoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 30472658, 30523567, 50.45, 22, 'ENSG00000184544', 'A6NNS2'),
(651, 'DHX35', 'DEAD box proteins characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of the DEAD box protein family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. The function of this gene product which is a member of this family, has not been determined. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jun 2010]', '20', 'miRNA', 41746772, 41746841, 62.86, 1, 'ENSG00000101452', 'Q9H5Z1'),
(652, 'DIPK1C', 'This gene encodes a member of the FAM69 family of cysteine-rich type II transmembrane proteins. These proteins localize to the endoplasmic reticulum but their specific functions are unknown. [provided by RefSeq, Nov 2011]', '18', 'protein_coding', 92826831, 93059493, 37.57, 9, 'ENSG00000187773', 'Q0P6D2'),
(653, 'DIS3L2', 'The protein encoded by this gene is similar in sequence to 3\'/5\' exonucleolytic subunits of the RNA exosome. The exosome is a large multimeric ribonucleotide complex responsible for degrading various RNA substrates. Several transcript variants, some protein-coding and some not, have been found for this gene. [provided by RefSeq, Mar 2012]', '2', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000144535', 'Q8IYB7'),
(654, 'DISP3', 'Involved in negative regulation of neuron differentiation; positive regulation of lipid metabolic process; and positive regulation of neural precursor cell proliferation. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 170941, 171501, 61.50, 1, 'ENSG00000204624', 'Q9P2K9'),
(655, 'DKC1', 'This gene functions in two distinct complexes. It plays an active role in telomerase stabilization and maintenance, as well as recognition of snoRNAs containing H/ACA sequences which provides stability during biogenesis and assembly into H/ACA small nucleolar RNA ribonucleoproteins (snoRNPs). This gene is highly conserved and widely expressed, and may play additional roles in nucleo-cytoplasmic shuttling, DNA damage response, and cell adhesion. Mutations have been associated with X-linked dyskeratosis congenita. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', 'X', 'protein_coding', 16467, 25055, 51.88, 10, 'ENSG00000130826', 'O60832'),
(656, 'DLGAP1-AS2', 'Biomarker of hepatocellular carcinoma. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 134549110, 134579875, 45.52, 4, 'ENSG00000262001', ''),
(657, 'DLGAP2', 'The product of this gene is a membrane-associated protein that may play a role in synapse organization and signalling in neuronal cells. This gene is biallelically expressed in the brain, however, only the paternal allele is expressed in the testis (PMID:18055845). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jun 2014]', '8', 'protein_coding', 64073181, 64079988, 63.53, 4, 'ENSG00000198010', 'Q9P1A6'),
(658, 'DLGAP2', 'The product of this gene is a membrane-associated protein that may play a role in synapse organization and signalling in neuronal cells. This gene is biallelically expressed in the brain, however, only the paternal allele is expressed in the testis (PMID:18055845). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jun 2014]', '8', 'protein_coding', 64073181, 64079988, 63.53, 4, 'ENSG00000282152', 'Q9P1A6'),
(659, 'DLGAP2', 'The product of this gene is a membrane-associated protein that may play a role in synapse organization and signalling in neuronal cells. This gene is biallelically expressed in the brain, however, only the paternal allele is expressed in the testis (PMID:18055845). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jun 2014]', '8', 'protein_coding', 64073181, 64079988, 63.53, 4, 'ENSG00000274161', 'Q9P1A6'),
(660, 'DLGAP2', 'The product of this gene is a membrane-associated protein that may play a role in synapse organization and signalling in neuronal cells. This gene is biallelically expressed in the brain, however, only the paternal allele is expressed in the testis (PMID:18055845). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jun 2014]', '8', 'protein_coding', 64073181, 64079988, 63.53, 4, 'ENSG00000282318', 'Q9P1A6'),
(661, 'DLGAP4', 'The product of this gene is a membrane-associated guanylate kinase found at the postsynaptic density in neuronal cells. It is a signaling molecule that can interact with potassium channels and receptors, as well as other signaling molecules. The protein encoded by this gene can interact with PSD-95 through its guanylate kinase domain and may be involved in clustering PSD-95 in the postsynaptic density region. The encoded protein is one of at least four similar proteins that have been found. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 109293788, 109316517, 43.89, 1, 'ENSG00000080845', 'Q9Y2H0'),
(662, 'DLGAP5', 'Predicted to enable microtubule binding activity. Predicted to be involved in several processes, including centrosome localization; kinetochore assembly; and mitotic spindle organization. Located in cytosol; nucleus; and spindle. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 86435256, 86521792, 39.86, 4, 'ENSG00000126787', 'Q15398'),
(663, 'DLK1', 'This gene encodes a transmembrane protein that contains multiple epidermal growth factor repeats that functions as a regulator of cell growth. The encoded protein is involved in the differentiation of several cell types including adipocytes. This gene is located in a region of chromosome 14 frequently showing unparental disomy, and is imprinted and expressed from the paternal allele. A single nucleotide variant in this gene is associated with child and adolescent obesity and shows polar overdominance, where heterozygotes carrying an active paternal allele express the phenotype, while mutant homozygotes are normal. [provided by RefSeq, Nov 2015]', '14', 'protein_coding', 30350773, 30355308, 57.83, 3, 'ENSG00000185559', 'P80370'),
(664, 'DMAC2', 'Involved in mitochondrial respiratory chain complex I assembly. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 69337996, 69343106, 47.96, 8, 'ENSG00000105341', 'Q9NW81'),
(665, 'DMGDH', 'This gene encodes an enzyme involved in the catabolism of choline, catalyzing the oxidative demethylation of dimethylglycine to form sarcosine. The enzyme is found as a monomer in the mitochondrial matrix, and uses flavin adenine dinucleotide and folate as cofactors. Mutation in this gene causes dimethylglycine dehydrogenase deficiency, characterized by a fishlike body odor, chronic muscle fatigue, and elevated levels of the muscle form of creatine kinase in serum. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '5', 'protein_coding', 1131131, 1146189, 48.66, 21, 'ENSG00000132837', 'Q9UI17'),
(666, 'DMRTC1', 'Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 131155383, 131224468, 39.50, 15, 'ENSG00000269502', 'Q5HYR2'),
(667, 'DNAAF6', 'Enables dynein intermediate chain binding activity. Involved in flagellated sperm motility; inner dynein arm assembly; and outer dynein arm assembly. Located in trans-Golgi network. Implicated in primary ciliary dyskinesia 36. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 115124762, 115180546, 40.18, 10, 'ENSG00000080572', 'Q9NQM4'),
(668, 'DNAH6', 'This gene belongs to the dynein family, whose members encode large proteins that are constituents of the microtubule-associated motor protein complex. This complex is composed of dynein heavy, intermediate and light chains, which can be axonemal or cytoplasmic. This protein is an axonemal dynein heavy chain. It is involved in producing force for ciliary beating by using energy from ATP hydrolysis. Mutations in this gene may cause primary ciliary dyskinesia (PCD) as well as heterotaxy. [provided by RefSeq, Jun 2016]', '2', 'protein_coding', 183099257, 183320777, 39.32, 10, 'ENSG00000115423', 'Q9C0G6'),
(669, 'DNAH9', 'This gene encodes the heavy chain subunit of axonemal dynein, a large multi-subunit molecular motor. Axonemal dynein attaches to microtubules and hydrolyzes ATP to mediate the movement of cilia and flagella. The gene expresses at least two transcript variants; additional variants have been described, but their full length nature has not been determined. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 120950763, 120977216, 35.56, 6, 'ENSG00000007174', 'Q9NYC9'),
(670, 'DNAJB14', 'Enables Hsp70 protein binding activity. Involved in chaperone cofactor-dependent protein refolding and protein-containing complex assembly. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 142728, 154503, 51.79, 2, 'ENSG00000164031', 'Q8TBM8'),
(671, 'DNAJC2', 'This gene is a member of the M-phase phosphoprotein (MPP) family. The gene encodes a phosphoprotein with a J domain and a Myb DNA-binding domain which localizes to both the nucleus and the cytosol. The protein is capable of forming a heterodimeric complex that associates with ribosomes, acting as a molecular chaperone for nascent polypeptide chains as they exit the ribosome. This protein was identified as a leukemia-associated antigen and expression of the gene is upregulated in leukemic blasts. Also, chromosomal aberrations involving this gene are associated with primary head and neck squamous cell tumors. This gene has a pseudogene on chromosome 6. Alternatively spliced variants which encode different protein isoforms have been described. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 28577565, 28599025, 46.20, 16, 'ENSG00000105821', 'Q99543'),
(672, 'DNAJC28', 'This gene encodes a member of the DnaJ heat shock protein family. The encoded protein, which contains a conserved N-terminal DnaJ domain, is thought to play a role in protein folding or act as a molecular chaperone protein. [provided by RefSeq, Oct 2016]', '21', 'protein_coding', 118836074, 118988547, 37.76, 4, 'ENSG00000177692', 'Q9NX36'),
(673, 'DNAJC28', 'This gene encodes a member of the DnaJ heat shock protein family. The encoded protein, which contains a conserved N-terminal DnaJ domain, is thought to play a role in protein folding or act as a molecular chaperone protein. [provided by RefSeq, Oct 2016]', '21', 'protein_coding', 118836074, 118988547, 37.76, 4, 'ENSG00000262911', 'Q9NX36'),
(674, 'DNALI1', 'This gene is the human homolog of the Chlamydomonas inner dynein arm gene, p28. The precise function of this gene is not known, however, it is a potential candidate for immotile cilia syndrome (ICS). Ultrastructural defects of the inner dynein arms are seen in patients with ICS. Immotile mutant strains of Chlamydomonas, a biflagellated algae, exhibit similar defects. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 2296001, 2341079, 47.92, 2, 'ENSG00000163879', 'O14645'),
(675, 'DNMT3L', 'CpG methylation is an epigenetic modification that is important for embryonic development, imprinting, and X-chromosome inactivation. Studies in mice have demonstrated that DNA methylation is required for mammalian development. This gene encodes a nuclear protein with similarity to DNA methyltransferases, but is not thought to function as a DNA methyltransferase as it does not contain the amino acid residues necessary for methyltransferase activity. However, it does stimulate de novo methylation by DNA cytosine methyltransferase 3 alpha and is thought to be required for the establishment of maternal genomic imprints. This protein also mediates transcriptional repression through interaction with histone deacetylase 1. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2012]', '21', 'protein_coding', 51240162, 51257655, 36.89, 3, 'ENSG00000142182', 'Q9UJW3'),
(676, 'DNPH1', 'This gene was identified on the basis of its stimulation by c-Myc protein. The latter is a transcription factor that participates in the regulation of cell proliferation, differentiation, and apoptosis. The exact function of this gene is not known but studies in rat suggest a role in cellular proliferation and c-Myc-mediated transformation. Two alternative transcripts encoding different proteins have been described. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 28097976, 28211965, 43.81, 23, 'ENSG00000112667', 'O43598'),
(677, 'DOC2B', 'There are at least two protein isoforms of the Double C2 protein,  namely alpha (DOC2A) and beta (DOC2B), which contain two C2-like domains.  DOC2A and DOC2B are encoded by different genes; these genes are at times confused with the unrelated DAB2 gene which was initially named DOC-2. DOC2B is expressed ubiquitously and is suggested to be involved in Ca(2+)-dependent intracellular vesicle trafficking in various types of cells. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 47032527, 47035621, 56.83, 1, 'ENSG00000272670', 'Q14184'),
(678, 'DOC2B', 'There are at least two protein isoforms of the Double C2 protein,  namely alpha (DOC2A) and beta (DOC2B), which contain two C2-like domains.  DOC2A and DOC2B are encoded by different genes; these genes are at times confused with the unrelated DAB2 gene which was initially named DOC-2. DOC2B is expressed ubiquitously and is suggested to be involved in Ca(2+)-dependent intracellular vesicle trafficking in various types of cells. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 47032527, 47035621, 56.83, 1, 'ENSG00000272636', 'Q14184'),
(679, 'DOCK7', 'The protein encoded by this gene is a guanine nucleotide exchange factor (GEF) that plays a role in axon formation and neuronal polarization. The encoded protein displays GEF activity toward RAC1 and RAC3 Rho small GTPases but not toward CDC42. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2012]', '1', 'protein_coding', 2621182, 2660056, 42.45, 32, 'ENSG00000116641', 'Q96N67'),
(680, 'DOK5', 'The protein encoded by this gene is a member of the DOK family of membrane proteins, which are adapter proteins involved in signal transduction. The encoded protein interacts with phosphorylated receptor tyrosine kinases to mediate neurite outgrowth and activation of the MAP kinase pathway. Unlike other DOK family proteins, this protein does not interact with RASGAP. This protein is up-regulated in patients with systemic sclerosis and is associated with fibrosis induced by insulin-like growth factor binding protein 5. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jun 2014]', '20', 'protein_coding', 60091911, 60169133, 37.99, 13, 'ENSG00000101134', 'Q9P104'),
(681, 'DOK6', 'DOK6 is a member of the DOK (see DOK1; MIM 602919) family of intracellular adaptors that play a role in the RET (MIM 164761) signaling cascade (Crowder et al., 2004 [PubMed 15286081]).[supplied by OMIM, Mar 2008]', '18', 'TR_J_gene', 22475316, 22475375, 43.33, 1, 'ENSG00000206052', 'Q6PKX4'),
(682, 'DPH1', 'The protein encoded by this gene is an enzyme involved in the biosynthesis of diphthamide, a modified histidine found only in elongation factor-2 (EEF2). Diphthamide residues in EEF2 are targeted for ADP-ribosylation by diphtheria toxin and Pseudomonas exotoxin A. Defects in this gene have been associated with both ovarian cancer and autosomal recessive intellectual disability with short stature, craniofacial, and ectodermal anomalies. [provided by RefSeq, Oct 2016]', '17', 'miRNA', 31160062, 31160119, 63.79, 1, 'ENSG00000108963', 'Q9BZG8'),
(683, 'DPH7', 'Diphthamide is a post-translationally modified histidine residue present in elongation factor 2, and is the target of diphtheria toxin. This gene encodes a protein that contains a WD-40 domain, and is thought to be involved in diphthamide biosynthesis. A similar protein in yeast functions as a methylesterase, converting methylated diphthine to diphthine, which can then undergo amidation to produce diphthamide. [provided by RefSeq, Oct 2016]', '9', 'protein_coding', 157381133, 157678157, 45.59, 9, 'ENSG00000148399', 'Q9BTV6'),
(684, 'DPM1', 'Dolichol-phosphate mannose (Dol-P-Man) serves as a donor of mannosyl residues on the lumenal side of the endoplasmic reticulum (ER). Lack of Dol-P-Man results in defective surface expression of GPI-anchored proteins. Dol-P-Man is synthesized from GDP-mannose and dolichol-phosphate on the cytosolic side of the ER by the enzyme dolichyl-phosphate mannosyltransferase. Human DPM1 lacks a carboxy-terminal transmembrane domain and signal sequence and is regulated by DPM2. Mutations in this gene are associated with congenital disorder of glycosylation type Ie. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2015]', '20', 'protein_coding', 31131433, 31135727, 58.95, 6, 'ENSG00000000419', 'O60762'),
(685, 'DPP10', 'This gene encodes a single-pass type II membrane protein that is a member of the S9B family in clan SC of the serine proteases. This protein has no detectable protease activity, most likely due to the absence of the conserved serine residue normally present in the catalytic domain of serine proteases. However, it does bind specific voltage-gated potassium channels and alters their expression and biophysical properties. Mutations in this gene have been associated with asthma. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 34613545, 34620523, 49.85, 11, 'ENSG00000175497', 'Q8N608'),
(686, 'DPPA2', 'Predicted to enable chromatin binding activity. Predicted to be involved in system development. Predicted to act upstream of or within several processes, including epigenetic regulation of gene expression; lung-associated mesenchyme development; and positive regulation of stem cell proliferation. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 34759740, 34762327, 55.91, 1, 'ENSG00000163530', 'Q7Z7J5'),
(687, 'DPPA3', 'This gene encodes a protein that in mice may function as a maternal factor during the preimplantation stage of development. In mice, this gene may play a role in transcriptional repression, cell division, and maintenance of cell pluripotentiality. In humans, related intronless loci are located on chromosomes 14 and X. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 106611930, 106679439, 35.16, 6, 'ENSG00000187569', 'Q6W0C5'),
(688, 'DPPA5', 'This gene encodes a protein that may function in the control of cell pluripotency and early embryogenesis. Expression of this gene is a specific marker for pluripotent stem cells. Pseudogenes of this gene are located on the short arm of chromosome 10 and the long arm of chromosomes 14 and 19. [provided by RefSeq, Dec 2010]', '6', 'protein_coding', 171309248, 171312139, 66.60, 1, 'ENSG00000203909', 'A6NC42'),
(689, 'DPRXP4', 'Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This pseudogene is a member of the DPRX homeobox gene family. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 666651, 680582, 56.84, 33, 'ENSG00000291714', ''),
(690, 'DPRXP4', 'Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This pseudogene is a member of the DPRX homeobox gene family. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 666651, 680582, 56.84, 33, 'ENSG00000264743', ''),
(691, 'DPY19L2P2', 'Predicted to enable mannosyltransferase activity. Predicted to be involved in spermatid development. Predicted to be located in membrane. Predicted to be active in nuclear inner membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 11836933, 11836986, 57.41, 1, 'ENSG00000282026', 'Q6ZN68'),
(692, 'DPY19L2P2', 'Predicted to enable mannosyltransferase activity. Predicted to be involved in spermatid development. Predicted to be located in membrane. Predicted to be active in nuclear inner membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 11836933, 11836986, 57.41, 1, 'ENSG00000290518', 'Q6ZN68'),
(693, 'DPY19L2P2', 'Predicted to enable mannosyltransferase activity. Predicted to be involved in spermatid development. Predicted to be located in membrane. Predicted to be active in nuclear inner membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 11836933, 11836986, 57.41, 1, 'ENSG00000170629', 'Q6ZN68'),
(694, 'DPY19L2P2', 'Predicted to enable mannosyltransferase activity. Predicted to be involved in spermatid development. Predicted to be located in membrane. Predicted to be active in nuclear inner membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 11836933, 11836986, 57.41, 1, 'ENSG00000291139', 'Q6ZN68'),
(695, 'DPYSL3', 'Enables filamin binding activity. Predicted to be involved in several processes, including actin filament organization; regulation of plasma membrane bounded cell projection organization; and response to axon injury. Predicted to act upstream of or within nervous system development. Predicted to be located in several cellular components, including exocytic vesicle; growth cone; and lamellipodium. Predicted to be part of filamentous actin. Predicted to be active in cytosol and synapse. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 120236640, 120319945, 48.07, 7, 'ENSG00000113657', 'Q14195'),
(696, 'DPYSL4', 'Enables filamin binding activity. Predicted to be involved in nervous system development. Predicted to be located in cytoplasm. Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 23980058, 23983915, 57.52, 2, 'ENSG00000151640', 'O14531'),
(697, 'DRD1', 'This gene encodes the D1 subtype of the dopamine receptor. The D1 subtype is the most abundant dopamine receptor in the central nervous system. This G-protein coupled receptor stimulates adenylyl cyclase and activates cyclic AMP-dependent protein kinases. D1 receptors regulate neuronal growth and development, mediate some behavioral responses, and modulate dopamine receptor D2-mediated events. Alternate transcription initiation sites result in two transcript variants of this gene. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 16574907, 16629062, 52.52, 7, 'ENSG00000184845', 'P21728'),
(698, 'DRGX', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within several processes, including detection of temperature stimulus; nervous system development; and sensory perception of mechanical stimulus. Predicted to be located in chromatin. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 149268, 178809, 63.82, 3, 'ENSG00000165606', 'A6NNA5');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(699, 'DSC3', 'This gene encodes a member of the desmocollin protein subfamily. Desmocollins, along with desmogleins, are cadherin-like transmembrane glycoproteins that are major components of the desmosome. Desmosomes are cell-cell junctions that help resist shearing forces and are found in high concentrations in cells subject to mechanical stress. This gene is found in a cluster with other desmocollin family members on chromosome 18. Mutations in this gene are associated with arrhythmogenic right ventricular dysplasia-11, and reduced protein expression has been described in several types of cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2015]', '18', 'protein_coding', 30248203, 30292056, 40.18, 13, 'ENSG00000134762', 'Q14574'),
(700, 'DSCAM', 'This gene is a member of the immunoglobulin superfamily of cell adhesion molecules (Ig-CAMs), and is involved in human central and peripheral nervous system development. This gene is a candidate for Down syndrome and congenital heart disease (DSCHD). A gene encoding a similar Ig-CAM protein is located on chromosome 11. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Oct 2012]', '21', 'protein_coding', 1623806, 2245605, 40.77, 10, 'ENSG00000171587', 'O60469'),
(701, 'DSCAM', 'This gene is a member of the immunoglobulin superfamily of cell adhesion molecules (Ig-CAMs), and is involved in human central and peripheral nervous system development. This gene is a candidate for Down syndrome and congenital heart disease (DSCHD). A gene encoding a similar Ig-CAM protein is located on chromosome 11. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Oct 2012]', '21', 'protein_coding', 1623806, 2245605, 40.77, 10, 'ENSG00000291561', 'O60469'),
(702, 'DSG4', 'This gene encodes a member of the desmoglein subgroup of desmosomal cadherins. The encoded preproprotein is proteolytically processed to generate the mature protein. This protein is a transmembrane component of desmosomes and may play a role in cell-cell adhesion in epithelial cells. Mutations in the gene are associated with localized autosomal recessive hypotrichosis and monilethrix, characterized by impaired hair growth. [provided by RefSeq, May 2016]', '18', 'protein_coding', 97434169, 97469128, 36.13, 2, 'ENSG00000175065', 'Q86SJ6'),
(703, 'DTWD2', 'Enables tRNA-uridine aminocarboxypropyltransferase activity. Involved in RNA glycosylation and tRNA modification. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 78462, 183684, 36.00, 13, 'ENSG00000169570', 'Q8NBA8'),
(704, 'DUSP14', 'Dual-specificity phosphatases (DUSPs) constitute a large heterogeneous subgroup of the type I cysteine-based protein-tyrosine phosphatase superfamily. DUSPs are characterized by their ability to dephosphorylate both tyrosine and serine/threonine residues. They have been implicated as major modulators of critical signaling pathways. DUSP14 contains the consensus DUSP C-terminal catalytic domain but lacks the N-terminal CH2 domain found in the MKP (mitogen-activated protein kinase phosphatase) class of DUSPs (see MIM 600714) (summary by Patterson et al., 2009 [PubMed 19228121]).[supplied by OMIM, Dec 2009]', '17', 'protein_coding', 5268345, 5505652, 38.96, 3, 'ENSG00000275932', 'O95147'),
(705, 'DUSP14', 'Dual-specificity phosphatases (DUSPs) constitute a large heterogeneous subgroup of the type I cysteine-based protein-tyrosine phosphatase superfamily. DUSPs are characterized by their ability to dephosphorylate both tyrosine and serine/threonine residues. They have been implicated as major modulators of critical signaling pathways. DUSP14 contains the consensus DUSP C-terminal catalytic domain but lacks the N-terminal CH2 domain found in the MKP (mitogen-activated protein kinase phosphatase) class of DUSPs (see MIM 600714) (summary by Patterson et al., 2009 [PubMed 19228121]).[supplied by OMIM, Dec 2009]', '17', 'protein_coding', 5268345, 5505652, 38.96, 3, 'ENSG00000276023', 'O95147'),
(706, 'DUSP16', 'This gene encodes a mitogen-activated protein kinase phosphatase that is a member of the dual specificity protein phosphatase subfamily. These phosphatases inactivate their target kinases by dephosphorylating both the phosphoserine/threonine and phosphotyrosine residues. The encoded protein specifically regulates the c-Jun amino-terminal kinase (JNK) and extracellular signal-regulated kinase (ERK) pathways.[provided by RefSeq, May 2010]', '12', 'protein_coding', 223017, 232224, 47.51, 2, 'ENSG00000280962', 'Q9BY84'),
(707, 'DUSP16', 'This gene encodes a mitogen-activated protein kinase phosphatase that is a member of the dual specificity protein phosphatase subfamily. These phosphatases inactivate their target kinases by dephosphorylating both the phosphoserine/threonine and phosphotyrosine residues. The encoded protein specifically regulates the c-Jun amino-terminal kinase (JNK) and extracellular signal-regulated kinase (ERK) pathways.[provided by RefSeq, May 2010]', '12', 'protein_coding', 223017, 232224, 47.51, 2, 'ENSG00000111266', 'Q9BY84'),
(708, 'DUX4L2', 'This gene is located within a D4Z4 repeat array in the subtelomeric region of chromosome 4q. The D4Z4 repeat is polymorphic in length and a similar D4Z4 repeat array has been identified on chromosome 10. Each D4Z4 repeat unit has an open reading frame (named DUX4) that encodes two homeoboxes; the repeat-array and ORF is conserved in other mammals. There is no evidence for transcription of the gene at this locus though RT-PCR and in vitro expression experiments indicate that a telomeric paralog of this gene is transcribed in some haplotypes. Contraction of the macrosatellite repeat causes autosomal dominant facioscapulohumeral muscular dystrophy (FSHD). [provided by RefSeq, Jun 2014]', '4', 'protein_coding', 58150902, 58161150, 46.81, 1, 'ENSG00000283834', 'P0CJ85'),
(709, 'DUX4L2', 'This gene is located within a D4Z4 repeat array in the subtelomeric region of chromosome 4q. The D4Z4 repeat is polymorphic in length and a similar D4Z4 repeat array has been identified on chromosome 10. Each D4Z4 repeat unit has an open reading frame (named DUX4) that encodes two homeoboxes; the repeat-array and ORF is conserved in other mammals. There is no evidence for transcription of the gene at this locus though RT-PCR and in vitro expression experiments indicate that a telomeric paralog of this gene is transcribed in some haplotypes. Contraction of the macrosatellite repeat causes autosomal dominant facioscapulohumeral muscular dystrophy (FSHD). [provided by RefSeq, Jun 2014]', '4', 'protein_coding', 58150902, 58161150, 46.81, 1, 'ENSG00000280457', 'P0CJ85'),
(710, 'DUX4L8', 'This gene is located within a D4Z4 repeat array in the subtelomeric region of chromosome 4q. The D4Z4 repeat is polymorphic in length and a similar D4Z4 repeat array has been identified on chromosome 10. Each D4Z4 repeat unit has an open reading frame (named DUX4) that encodes two homeoboxes; the repeat-array and ORF is conserved in other mammals. There is no evidence for transcription of the gene at this locus though RT-PCR and in vitro expression experiments indicate that a telomeric paralog of this gene is transcribed in some haplotypes. Contraction of the macrosatellite repeat causes autosomal dominant facioscapulohumeral muscular dystrophy (FSHD). [provided by RefSeq, Jun 2014]', '4', 'unprocessed_pseudogene', 13581060, 13582004, 40.95, 1, 'ENSG00000283363', ''),
(711, 'DUX4L8', 'This gene is located within a D4Z4 repeat array in the subtelomeric region of chromosome 4q. The D4Z4 repeat is polymorphic in length and a similar D4Z4 repeat array has been identified on chromosome 10. Each D4Z4 repeat unit has an open reading frame (named DUX4) that encodes two homeoboxes; the repeat-array and ORF is conserved in other mammals. There is no evidence for transcription of the gene at this locus though RT-PCR and in vitro expression experiments indicate that a telomeric paralog of this gene is transcribed in some haplotypes. Contraction of the macrosatellite repeat causes autosomal dominant facioscapulohumeral muscular dystrophy (FSHD). [provided by RefSeq, Jun 2014]', '4', 'unprocessed_pseudogene', 13581060, 13582004, 40.95, 1, 'ENSG00000283570', ''),
(712, 'DUX4L8', 'This gene is located within a D4Z4 repeat array in the subtelomeric region of chromosome 4q. The D4Z4 repeat is polymorphic in length and a similar D4Z4 repeat array has been identified on chromosome 10. Each D4Z4 repeat unit has an open reading frame (named DUX4) that encodes two homeoboxes; the repeat-array and ORF is conserved in other mammals. There is no evidence for transcription of the gene at this locus though RT-PCR and in vitro expression experiments indicate that a telomeric paralog of this gene is transcribed in some haplotypes. Contraction of the macrosatellite repeat causes autosomal dominant facioscapulohumeral muscular dystrophy (FSHD). [provided by RefSeq, Jun 2014]', '4', 'unprocessed_pseudogene', 13581060, 13582004, 40.95, 1, 'ENSG00000281720', ''),
(713, 'DUXAP7', 'Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This pseudogene is a member of the DUXA homeobox gene family. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 96760810, 96808100, 39.27, 9, 'ENSG00000232136', ''),
(714, 'DYNC1LI2', 'Cytoplasmic dynein is a microtubule-associated motor protein (Hughes et al., 1995 [PubMed 7738094]). See DYNC1H1 (MIM 600112) for general information about dyneins.[supplied by OMIM, Mar 2008]', '16', 'protein_coding', 34755047, 34758512, 54.15, 1, 'ENSG00000135720', 'O43237'),
(715, 'DYNC2I1', 'This gene encodes a member of the WD repeat protein family. WD repeats are minimally conserved regions of approximately 40 amino acids typically bracketed by gly-his and trp-asp (GH-WD) and may facilitate the formation of heterotrimeric or multiprotein complexes. Members of this family are involved in a variety of cellular processes including cell cycle progression, signal transduction, apoptosis, and gene regulation. The encoded protein contains four WD repeats and may play a role in the formation of cilia. Mutations in this gene have been associated with short-rib polydactyly and Jeune syndromes. [provided by RefSeq, Mar 2014]', '7', 'protein_coding', 96832254, 96995956, 39.59, 29, 'ENSG00000126870', 'Q8WVS4'),
(716, 'DYNLRB1', 'This gene is a member of the roadblock dynein light chain family. The encoded cytoplasmic protein is capable of binding intermediate chain proteins, interacts with transforming growth factor-beta, and has been implicated in the regulation of actin modulating proteins. Upregulation of this gene has been associated with hepatocellular carcinomas, suggesting that this gene may be involved in tumor progression. Alternative splicing results in multiple transcript variants. Pseudogenes of this gene have been defined on chromosomes 12 and 18. [provided by RefSeq, Aug 2013]', '20', 'protein_coding', 75564, 106271, 61.67, 4, 'ENSG00000125971', 'Q9NP97'),
(717, 'DZIP1', 'Enables BBSome binding activity and molecular adaptor activity. Involved in cilium organization; heart development; and protein-containing complex localization to centriolar satellite. Located in cytosol; microtubule organizing center; and nucleoplasm. Implicated in mitral valve prolapse and spermatogenic failure 47. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 21628089, 21630064, 67.56, 7, 'ENSG00000134874', 'Q86YF9'),
(718, 'DZIP3', 'Enables several functions, including phosphatase binding activity; polyubiquitin modification-dependent protein binding activity; and ubiquitin-protein transferase activity. Involved in protein polyubiquitination. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '3', 'unitary_pseudogene', 5468486, 5469422, 49.09, 1, 'ENSG00000198919', 'Q86Y13'),
(719, 'E2F6', 'This gene encodes a member of a family of transcription factors that play a crucial role in the control of the cell cycle. The protein encoded by this gene lacks the transactivation and tumor suppressor protein association domains found in other family members, and contains a modular suppression domain that functions in the inhibition of transcription. It interacts in a complex with chromatin modifying factors. There are pseudogenes for this gene on chromosomes 22 and X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2013]', '2', 'protein_coding', 32812763, 32820466, 43.38, 5, 'ENSG00000169016', 'O75461'),
(720, 'EARS2', 'This gene encodes a member of the class I family of aminoacyl-tRNA synthetases. These enzymes play a critical role in protein biosynthesis by charging tRNAs with their cognate amino acids. This protein is encoded by the nuclear genome but is likely to be imported to the mitochondrion where it is thought to catalyze the ligation of glutamate to tRNA molecules. Mutations in this gene have been associated with combined oxidative phosphorylation deficiency 12 (COXPD12). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2015]', '16', 'IG_C_gene', 22906342, 22906803, 57.14, 1, 'ENSG00000103356', 'Q5JPH6'),
(721, 'EBPL', 'Predicted to enable cholestenol delta-isomerase activity. Predicted to be involved in sterol metabolic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 5432108, 5521536, 46.64, 6, 'ENSG00000123179', 'Q9BY08'),
(722, 'EDAR', 'This gene encodes a member of the tumor necrosis factor receptor family. The encoded transmembrane protein is a receptor for the soluble ligand ectodysplasin A, and can activate the nuclear factor-kappaB, JNK, and caspase-independent cell death pathways. It is required for the development of hair, teeth, and other ectodermal derivatives. Mutations in this gene result in autosomal dominant and recessive forms of hypohidrotic ectodermal dysplasia. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 100357074, 100422055, 42.25, 4, 'ENSG00000135960', 'Q9UNE0'),
(723, 'EDIL3', 'The protein encoded by this gene is an integrin ligand. It plays an important role in mediating angiogenesis and may be important in vessel wall remodeling and development. It also influences endothelial cell behavior. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 47290023, 47322066, 49.69, 3, 'ENSG00000164176', 'O43854'),
(724, 'EEA1', 'Enables 1-phosphatidylinositol binding activity; GTP-dependent protein binding activity; and protein homodimerization activity. Involved in endocytosis and vesicle fusion. Located in cytosol and early endosome membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'miRNA', 28819582, 28819668, 37.93, 1, 'ENSG00000102189', 'Q15075'),
(725, 'EEF1A1', 'This gene encodes an isoform of the alpha subunit of the elongation factor-1 complex, which is responsible for the enzymatic delivery of aminoacyl tRNAs to the ribosome. This isoform (alpha 1) is expressed in brain, placenta, lung, liver, kidney, and pancreas, and the other isoform (alpha 2) is expressed in brain, heart and skeletal muscle. This isoform is identified as an autoantigen in 66% of patients with Felty syndrome. This gene has been found to have multiple copies on many chromosomes, some of which, if not all, represent different pseudogenes. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 143636019, 143656418, 47.93, 3, 'ENSG00000156508', 'P68104'),
(726, 'EEF1A1P5', 'Predicted to enable GTPase activity and translation elongation factor activity. Predicted to be involved in translational elongation. Located in extracellular exosome and nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'miRNA', 13590292, 13590346, 40.00, 1, 'ENSG00000196205', 'Q5VTE0'),
(727, 'EEF1AKMT1', 'Enables protein-lysine N-methyltransferase activity. Involved in peptidyl-lysine methylation. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 38006553, 38128819, 37.60, 2, 'ENSG00000150456', 'Q8WVE0'),
(728, 'EEF1AKMT3', 'Enables heat shock protein binding activity and protein-lysine N-methyltransferase activity. Involved in peptidyl-lysine methylation. Located in several cellular components, including centrosome; chromosome; and nucleoplasm. Part of protein-containing complex. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 150800403, 150805118, 57.55, 11, 'ENSG00000123427', 'Q96AZ1'),
(729, 'EEF1B2', 'This gene encodes a translation elongation factor. The protein is a guanine nucleotide exchange factor involved in the transfer of aminoacylated tRNAs to the ribosome. Alternative splicing results in three transcript variants which differ only in the 5\' UTR. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 56457, 57423, 50.47, 1, 'ENSG00000283391', 'P24534'),
(730, 'EEF1B2', 'This gene encodes a translation elongation factor. The protein is a guanine nucleotide exchange factor involved in the transfer of aminoacylated tRNAs to the ribosome. Alternative splicing results in three transcript variants which differ only in the 5\' UTR. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 56457, 57423, 50.47, 1, 'ENSG00000114942', 'P24534'),
(731, 'EFCAB6', 'This gene encodes a protein which directly binds the oncogene DJ-1 and androgen receptor to form a ternary complex in cells. This binding protein recruits histone-deacetylase complexes in order to repress transcription activity of androgen receptor. This protein may also play a role in spermatogenesis and fertilization. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]', '22', 'protein_coding', 62302093, 62308862, 64.09, 5, 'ENSG00000186976', 'Q5THR3'),
(732, 'EFCAB8', 'Predicted to enable calcium ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 30676389, 30687895, 56.10, 6, 'ENSG00000215529', 'A8MWE9'),
(733, 'EFHC2', 'This gene encodes a protein which contains three DM10 domains and three calcium-binding EF-hand motifs. A related protein is encoded by a gene on chromosome 6. It has been suggested that both proteins are involved in the development of epilepsy (PMID: 15258581, 16112844) and that this gene may be associated with fear recognition in individuals with Turner syndrome. [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 105465867, 105491034, 36.14, 11, 'ENSG00000183690', 'Q5JST6'),
(734, 'EFNB2', 'This gene encodes a member of the ephrin (EPH) family. The ephrins and EPH-related receptors comprise the largest subfamily of receptor protein-tyrosine kinases and have been implicated in mediating developmental events, especially in the nervous system and in erythropoiesis. Based on their structures and sequence relationships, ephrins are divided into the ephrin-A (EFNA) class, which are anchored to the membrane by a glycosylphosphatidylinositol linkage, and the ephrin-B (EFNB) class, which are transmembrane proteins. This gene encodes an EFNB class ephrin which binds to the EPHB4 and EPHA3 receptors. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 45333471, 45341284, 43.84, 14, 'ENSG00000125266', 'P52799'),
(735, 'EFS', 'The protein encoded by this gene is a member of the CAS (CRK-associated substrate) family of adaptor proteins which typically serve as scaffolds for the assembly of larger signaling complexes. These complexes form at the cell surface where integrin binding leads to the subsequent phosphorylation of a CAS protein. Additional binding of SRC family kinases leads to CAS hyperphosphorylation and the creation of binding sites for CRK and other proteins that cause actin cytoskeleton reorganization. This gene plays a role in integrin-mediated cell attachment, spreading, and migration and also plays a role in both normal and malignant cellular transformation. This broadly expressed gene has been shown to play a role in neurite outgrowth and its expression in the thymus and lymphocytes is important for T cell maturation and the development of immunological self-tolerance. Alternative splicing of this gene results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2020]', '14', 'transcribed_unprocessed_pseudogene', 23976872, 23991421, 53.25, 1, 'ENSG00000100842', 'O43281'),
(736, 'EGFL7', 'This gene encodes a secreted endothelial cell protein that contains two epidermal growth factor-like domains. The encoded protein may play a role in regulating vasculogenesis. This protein may be involved in the growth and proliferation of tumor cells. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]', '9', 'protein_coding', 5046526, 5047461, 45.51, 1, 'ENSG00000172889', 'Q9UHF1'),
(737, 'EGLN2', 'The hypoxia inducible factor (HIF) is a transcriptional complex that is involved in oxygen homeostasis. At normal oxygen levels, the alpha subunit of HIF is targeted for degration by prolyl hydroxylation. This gene encodes an enzyme responsible for this post-translational modification. Alternative splicing results in multiple transcript variants. Read-through transcription also exists between this gene and the upstream RAB4B (RAB4B, member RAS oncogene family) gene. [provided by RefSeq, Feb 2011]', '19', 'protein_coding', 790475, 798281, 65.53, 31, 'ENSG00000269858', 'Q96KS0'),
(738, 'EGLN3', 'Enables peptidyl-proline 4-dioxygenase activity. Involved in protein hydroxylation. Located in cytosol and nucleus. Implicated in renal cell carcinoma. Biomarker of clear cell renal cell carcinoma. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 112690038, 112887168, 50.37, 17, 'ENSG00000129521', 'Q9H6Z9'),
(739, 'EGR2', 'The protein encoded by this gene is a transcription factor with three tandem C2H2-type zinc fingers. Defects in this gene are associated with Charcot-Marie-Tooth disease type 1D (CMT1D), Charcot-Marie-Tooth disease type 4E (CMT4E), and with Dejerine-Sottas syndrome (DSS). Multiple transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2008]', '10', 'miRNA', 30584332, 30584417, 62.79, 1, 'ENSG00000122877', 'P11161'),
(740, 'EHD3', 'Predicted to enable nucleic acid binding activity. Involved in several processes, including cytosolic transport; endocytic recycling; and protein homooligomerization. Acts upstream of or within protein localization to plasma membrane and regulation of cardiac muscle cell membrane potential. Located in ciliary pocket membrane and recycling endosome membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 62454298, 62688386, 36.07, 37, 'ENSG00000013016', 'Q9NZN3'),
(741, 'EIF1AD', 'Predicted to enable RNA binding activity and translation initiation factor activity. Predicted to be involved in translational initiation. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 299168, 306920, 44.43, 2, 'ENSG00000175376', 'Q8N9N8'),
(742, 'EIF2B1', 'This gene encodes one of five subunits of eukaryotic translation initiation factor 2B (EIF2B), a GTP exchange factor for eukaryotic initiation factor 2 and an essential regulator for protein synthesis. Mutations in this gene and the genes encoding other EIF2B subunits have been associated with leukoencephalopathy with vanishing white matter. [provided by RefSeq, Oct 2009]', '12', 'protein_coding', 130021, 144329, 48.16, 1, 'ENSG00000111361', 'Q14232'),
(743, 'EIF2S1', 'The translation initiation factor EIF2 catalyzes the first regulated step of protein synthesis initiation, promoting the binding of the initiator tRNA to 40S ribosomal subunits. Binding occurs as a ternary complex of methionyl-tRNA, EIF2, and GTP. EIF2 is composed of 3 nonidentical subunits, the 36-kD EIF2-alpha subunit (EIF2S1), the 38-kD EIF2-beta subunit (EIF2S2; MIM 603908), and the 52-kD EIF2-gamma subunit (EIF2S3; MIM 300161). The rate of formation of the ternary complex is modulated by the phosphorylation state of EIF2-alpha (Ernst et al., 1987 [PubMed 2948954]).[supplied by OMIM, Feb 2010]', '14', 'unprocessed_pseudogene', 14570844, 14571788, 40.85, 1, 'ENSG00000134001', 'P05198'),
(744, 'EIF3C', 'Enables ribosome binding activity. Contributes to translation initiation factor activity. Involved in positive regulation of translation and translational initiation. Part of eukaryotic translation initiation factor 3 complex. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 880674, 882266, 51.04, 1, 'ENSG00000184110', 'Q99613'),
(745, 'EIF3G', 'This gene encodes a core subunit of the eukaryotic translation initiation factor 3 (eIF3) complex, which is required for initiation of protein translation. An N-terminal caspase cleavage product of the encoded protein may stimulate degradation of DNA. A mutation in this gene is associated with narcolepsy. [provided by RefSeq, Jul 2016]', '19', 'protein_coding', 128819663, 128881931, 50.31, 1, 'ENSG00000130811', 'O75821'),
(746, 'EIF3K', 'The 700-kD eukaryotic translation initiation factor-3 (eIF3) is the largest eIF and contains at least 12 subunits, including EIF2S12. eIF3 plays an essential role in translation by binding directly to the 40S ribosomal subunit and promoting formation of the 40S preinitiation complex (Mayeur et al., 2003 [PubMed 14519125]).[supplied by OMIM, Mar 2008]', '19', 'IG_V_gene', 106210936, 106211453, 52.32, 1, 'ENSG00000282986', 'Q9UBQ5'),
(747, 'EIF3K', 'The 700-kD eukaryotic translation initiation factor-3 (eIF3) is the largest eIF and contains at least 12 subunits, including EIF2S12. eIF3 plays an essential role in translation by binding directly to the 40S ribosomal subunit and promoting formation of the 40S preinitiation complex (Mayeur et al., 2003 [PubMed 14519125]).[supplied by OMIM, Mar 2008]', '19', 'IG_V_gene', 106210936, 106211453, 52.32, 1, 'ENSG00000178982', 'Q9UBQ5'),
(748, 'EIF4A3', 'This gene encodes a member of the DEAD box protein family. DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure, such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. The protein encoded by this gene is a nuclear matrix protein. Its amino acid sequence is highly similar to the amino acid sequences of the translation initiation factors eIF4AI and eIF4AII, two other members of the DEAD box protein family. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 59019429, 59026654, 44.08, 1, 'ENSG00000291508', 'P38919'),
(749, 'EIF4A3', 'This gene encodes a member of the DEAD box protein family. DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure, such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. The protein encoded by this gene is a nuclear matrix protein. Its amino acid sequence is highly similar to the amino acid sequences of the translation initiation factors eIF4AI and eIF4AII, two other members of the DEAD box protein family. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 59019429, 59026654, 44.08, 1, 'ENSG00000141543', 'P38919'),
(750, 'EIF5A', 'Enables U6 snRNA binding activity. Involved in several processes, including cellular response to virus; positive regulation of intrinsic apoptotic signaling pathway by p53 class mediator; and tumor necrosis factor-mediated signaling pathway. Located in annulate lamellae; cytoplasm; and nucleus. Part of nuclear pore. Biomarker of colorectal cancer. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 110523066, 110561722, 49.73, 1, 'ENSG00000288145', 'P63241'),
(751, 'EIF5A', 'Enables U6 snRNA binding activity. Involved in several processes, including cellular response to virus; positive regulation of intrinsic apoptotic signaling pathway by p53 class mediator; and tumor necrosis factor-mediated signaling pathway. Located in annulate lamellae; cytoplasm; and nucleus. Part of nuclear pore. Biomarker of colorectal cancer. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 110523066, 110561722, 49.73, 1, 'ENSG00000132507', 'P63241'),
(752, 'EIF5A2', 'Predicted to enable translation elongation factor activity. Predicted to be involved in translational elongation. Located in intracellular membrane-bounded organelle. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 52184876, 52192268, 59.92, 10, 'ENSG00000163577', 'Q9GZV4'),
(753, 'EIF5AL1', 'Predicted to enable translation elongation factor activity. Predicted to be involved in translational elongation. Predicted to be located in endoplasmic reticulum membrane and nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 1012823, 1036718, 63.40, 4, 'ENSG00000253626', 'Q6IS14'),
(754, 'ELAC2', 'The protein encoded by this gene has a C-terminal domain with tRNA 3&#8242; processing endoribonuclease activity, which catalyzes the removal of the 3\' trailer from precursor tRNAs. The protein also interacts with activated Smad family member 2 (Smad2) and its nuclear partner forkhead box H1 (also known as FAST-1), and reduced expression can suppress transforming growth factor-beta induced growth arrest. Mutations in this gene result in an increased risk of prostate cancer. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 101280105, 101386618, 39.31, 4, 'ENSG00000006744', 'Q9BQ52'),
(755, 'ELAPOR2', 'Predicted to enable BMP receptor binding activity. Predicted to be involved in negative regulation of nervous system development; positive regulation of BMP signaling pathway; and positive regulation of epidermis development. Predicted to be located in plasma membrane. Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 132996985, 133026604, 42.73, 10, 'ENSG00000164659', 'A8MWY0'),
(756, 'ELF1', 'This gene encodes an E26 transformation-specific related transcription factor. The encoded protein is primarily expressed in lymphoid cells and acts as both an enhancer and a repressor to regulate transcription of various genes. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2009]', '13', 'protein_coding', 137299631, 137302271, 59.22, 1, 'ENSG00000120690', 'P32519'),
(757, 'ELFN2', 'Predicted to enable protein phosphatase inhibitor activity. Predicted to be involved in synaptic membrane adhesion. Predicted to act upstream of or within chemical synaptic transmission; establishment of protein localization; and gene expression. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 26188710, 26189112, 55.09, 1, 'ENSG00000166897', 'Q5R3F8'),
(758, 'ELMOD1', 'Enables GTPase activator activity. Predicted to act upstream of or within cilium assembly and protein transport. Predicted to be active in Golgi apparatus; cilium; and glutamatergic synapse. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 41136960, 41196801, 49.84, 23, 'ENSG00000110675', 'Q8N336'),
(759, 'ELOA2', 'This gene encodes the transcriptionally active subunit of the SIII (or elongin) transcription elongation factor complex, which also includes two regulatory subunits, elongins B and C. This complex acts to increase the rate of RNA chain elongation by RNA polymerase II by suppressing transient pausing of the polymerase at many sites along the DNA template. Whereas a related protein with similar function, elongin A, is ubiquitously expressed, the encoded protein is specifically expressed in the testis, suggesting it may have a role in spermatogenesis. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 26511, 146600, 44.98, 23, 'ENSG00000206181', 'Q8IYF1'),
(760, 'ELOVL5', 'This gene belongs to the ELO family. It is highly expressed in the adrenal gland and testis, and encodes a multi-pass membrane protein that is localized in the endoplasmic reticulum. This protein is involved in the elongation of long-chain polyunsaturated fatty acids. Mutations in this gene have been associated with spinocerebellar ataxia-38 (SCA38). Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2014]', '6', 'protein_coding', 77778, 78311, 63.48, 2, 'ENSG00000012660', 'Q9NYP7'),
(761, 'ELOVL6', 'Fatty acid elongases (EC 6.2.1.3), such as ELOVL6, use malonyl-CoA as a 2-carbon donor in the first and rate-limiting step of fatty acid elongation (Moon et al., 2001 [PubMed 11567032]).[supplied by OMIM, Mar 2008]', '4', 'protein_coding', 165233, 169508, 55.29, 3, 'ENSG00000170522', 'Q9H5J4'),
(762, 'ELOVL7', 'Enables fatty acid elongase activity. Involved in fatty acid elongation, polyunsaturated fatty acid; fatty acid elongation, saturated fatty acid; and very long-chain fatty acid biosynthetic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 33109557, 33877180, 38.88, 29, 'ENSG00000164181', 'A1L3X0'),
(763, 'ELP1', 'The protein encoded by this gene is a scaffold protein and a regulator for three different kinases involved in proinflammatory signaling. The encoded protein can bind NF-kappa-B-inducing kinase and I-kappa-B kinases through separate domains and assemble them into an active kinase complex. Mutations in this gene have been associated with familial dysautonomia. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2016]', '9', 'scaRNA', 234014, 234278, 53.21, 1, 'ENSG00000070061', 'O95163'),
(764, 'EMD', 'Emerin is a serine-rich nuclear membrane protein and a member of the nuclear lamina-associated protein family. It mediates membrane anchorage to the cytoskeleton. Dreifuss-Emery muscular dystrophy is an X-linked inherited degenerative myopathy resulting from mutation in the emerin gene. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 196506879, 196515346, 44.72, 2, 'ENSG00000102119', 'P50402'),
(765, 'EMX2', 'This gene encodes a homeobox-containing transcription factor that is the homolog to the \'empty spiracles\' gene in Drosophila. Research on this gene in humans has focused on its expression in three tissues: dorsal telencephalon, olfactory neuroepithelium, and urogenetial system. It is expressed in the dorsal telencephalon during development in a low rostral-lateral to high caudal-medial gradient and is proposed to pattern the neocortex into defined functional areas. It is also expressed in embryonic and adult olfactory neuroepithelia where it complexes with eukaryotic translation initiation factor 4E (eIF4E) and possibly regulates mRNA transport or translation. In the developing urogenital system, it is expressed in epithelial tissues and is negatively regulated by HOXA10. Alternative splicing results in multiple transcript variants encoding distinct proteins.[provided by RefSeq, Sep 2009]', '10', 'protein_coding', 122789049, 122809720, 40.47, 3, 'ENSG00000170370', 'Q04743'),
(766, 'ENKUR', 'This gene encodes a protein that interacts with calmodulin and several transient receptor potential canonical cation channel proteins. The encoded protein may function as an adaptor to localize signal transduction machinery to calcium channels. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2012]', '10', 'protein_coding', 46734827, 46909245, 36.84, 4, 'ENSG00000151023', 'Q8TC29'),
(767, 'ENO2', 'This gene encodes one of the three enolase isoenzymes found in mammals. This isoenzyme, a homodimer, is found in mature neurons and cells of neuronal origin. A switch from alpha enolase to gamma enolase occurs in neural tissue during development in rats and primates. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 3063824, 3115187, 45.37, 19, 'ENSG00000111674', 'P09104'),
(768, 'ENOSF1', 'This gene can encode a mitochondrial enzyme that is thought to convert L-fuconate to 2-keto-3-deoxy-L-fuconate. This locus was originally identified as the source of antisense RNAs of the adjacent thymidylate synthase gene. Splice variants at this locus may contain an alternate 3\' exon that is complementary to the 3\'UTR and terminal intron of the thymidylate synthase (TS) RNA and may downregulate TS expression. [provided by RefSeq, Aug 2017]', '18', 'protein_coding', 30425623, 30429054, 57.17, 6, 'ENSG00000132199', 'Q7L5Y1'),
(769, 'ENOX1', 'The protein encoded by this gene is involved in plasma membrane electron transport pathways. The encoded protein has both a hydroquinone (NADH) oxidase activity and a protein disulfide-thiol interchange activity. The two activities cycle with a periodicity of 24 minutes, with one activity being at its peak when the other is at its lowest. [provided by RefSeq, Dec 2016]', '13', 'protein_coding', 24981979, 25062279, 38.61, 5, 'ENSG00000120658', 'Q8TC92'),
(770, 'ENPP5', 'This gene encodes a type-I transmembrane glycoprotein. Studies in rat suggest the encoded protein may play a role in neuronal cell communications. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2014]', '6', 'protein_coding', 133599, 139439, 36.09, 3, 'ENSG00000112796', 'Q9UJA9'),
(771, 'ENTR1', 'Involved in several processes, including endocytic recycling; positive regulation of cilium assembly; and positive regulation of protein localization to cilium. Located in several cellular components, including endosome; microtubule organizing center; and retromer complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 15627318, 15675012, 39.71, 2, 'ENSG00000165689', 'Q96C92'),
(772, 'EPB41L3', 'Predicted to enable cytoskeletal protein-membrane anchor activity. Predicted to be a structural constituent of cytoskeleton. Predicted to be involved in several processes, including nervous system development; paranodal junction maintenance; and protein localization to axon. Located in several cellular components, including cell-cell junction; ciliary basal body; and cytosol. Implicated in esophagus squamous cell carcinoma. Biomarker of meningioma. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 59636716, 59669037, 39.89, 2, 'ENSG00000082397', 'Q9Y2J2'),
(773, 'EPHA1', 'This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene is expressed in some human cancer cell lines and has been implicated in carcinogenesis. [provided by RefSeq, Jul 2008]', '7', 'TR_J_gene', 22505110, 22505167, 50.00, 1, 'ENSG00000284816', 'P21709'),
(774, 'EPHA1', 'This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene is expressed in some human cancer cell lines and has been implicated in carcinogenesis. [provided by RefSeq, Jul 2008]', '7', 'TR_J_gene', 22505110, 22505167, 50.00, 1, 'ENSG00000146904', 'P21709'),
(775, 'EPHB4', 'Ephrin receptors and their ligands, the ephrins, mediate numerous developmental processes, particularly in the nervous system. Based on their structures and sequence relationships, ephrins are divided into the ephrin-A (EFNA) class, which are anchored to the membrane by a glycosylphosphatidylinositol linkage, and the ephrin-B (EFNB) class, which are transmembrane proteins. The Eph family of receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. Ephrin receptors make up the largest subgroup of the receptor tyrosine kinase (RTK) family. The protein encoded by this gene binds to ephrin-B2 and plays an essential role in vascular development. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 78887127, 78939438, 35.92, 12, 'ENSG00000196411', 'P54760'),
(776, 'EPHB6', 'This gene encodes a member of a family of transmembrane proteins that function as receptors for ephrin-B family proteins. Unlike other members of this family, the encoded protein does not contain a functional kinase domain. Activity of this protein can influence cell adhesion and migration. Expression of this gene is downregulated during tumor progression, suggesting that the protein may suppress tumor invasion and metastasis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '7', 'protein_coding', 143574785, 143578649, 67.89, 14, 'ENSG00000275482', 'O15197'),
(777, 'EPHB6', 'This gene encodes a member of a family of transmembrane proteins that function as receptors for ephrin-B family proteins. Unlike other members of this family, the encoded protein does not contain a functional kinase domain. Activity of this protein can influence cell adhesion and migration. Expression of this gene is downregulated during tumor progression, suggesting that the protein may suppress tumor invasion and metastasis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '7', 'protein_coding', 143574785, 143578649, 67.89, 14, 'ENSG00000106123', 'O15197'),
(778, 'EPPIN-WFDC6', 'This locus represents naturally occurring read-through transcription between the neighboring SPINLW1 (serine peptidase inhibitor-like, with Kunitz and WAP domains 1) and WFDC6 (WAP four-disulfide core domain 6) genes. The read-through transcript encodes a fusion protein that shares sequence identity with the products of each individual gene. [provided by RefSeq, Nov 2010]', '20', 'protein_coding', 3575260, 3580920, 44.62, 12, 'ENSG00000249139', 'A0A494C0M2'),
(779, 'ERAP1', 'The protein encoded by this gene is an aminopeptidase involved in trimming HLA class I-binding precursors so that they can be presented on MHC class I molecules. The encoded protein acts as a monomer or as a heterodimer with ERAP2. This protein may also be involved in blood pressure regulation by inactivation of angiotensin II. Three transcript variants encoding two different isoforms have been found for this gene.[provided by RefSeq, Oct 2010]', '5', 'protein_coding', 498402, 542877, 47.62, 3, 'ENSG00000164307', 'Q9NZ08'),
(780, 'ERG', 'This gene encodes a member of the erythroblast transformation-specific (ETS) family of transcriptions factors. All members of this family are key regulators of embryonic development, cell proliferation, differentiation, angiogenesis, inflammation, and apoptosis. The protein encoded by this gene is mainly expressed in the nucleus. It contains an ETS DNA-binding domain and a PNT (pointed) domain which is implicated in the self-association of chimeric oncoproteins. This protein is required for platelet adhesion to the subendothelium, inducing vascular cell remodeling. It also regulates hematopoesis, and the differentiation and maturation of megakaryocytic cells. This gene is involved in chromosomal translocations, resulting in different fusion gene products, such as TMPSSR2-ERG and NDRG1-ERG in prostate cancer, EWS-ERG in Ewing\'s sarcoma and FUS-ERG in acute myeloid leukemia. More than two dozens of transcript variants generated from combinatorial usage of three alternative promoters and multiple alternative splicing events have been reported, but the full-length nature of many of these variants has not been determined. [provided by RefSeq, Apr 2014]', '21', 'protein_coding', 75204103, 75207161, 65.09, 6, 'ENSG00000157554', 'P11308'),
(781, 'ERMARD', 'The protein encoded by this gene contains 2 transmembrane domains near the C-terminus and is localized in the endoplasmic reticulum. Knockout of this gene in developing rat brain showed that it may be involved in neuronal migration. Mutations in this gene are associated with periventricular nodular heterotopia-6 (PVNH6). Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2013]', '6', 'protein_coding', 21549447, 21624034, 49.73, 5, 'ENSG00000276187', 'Q5T6L9'),
(782, 'ERMARD', 'The protein encoded by this gene contains 2 transmembrane domains near the C-terminus and is localized in the endoplasmic reticulum. Knockout of this gene in developing rat brain showed that it may be involved in neuronal migration. Mutations in this gene are associated with periventricular nodular heterotopia-6 (PVNH6). Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2013]', '6', 'protein_coding', 21549447, 21624034, 49.73, 5, 'ENSG00000130023', 'Q5T6L9'),
(783, 'ESF1', 'Enables RNA binding activity. Predicted to be involved in rRNA processing. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 86876906, 87059654, 37.29, 12, 'ENSG00000089048', 'Q9H501'),
(784, 'ESR2', 'This gene encodes a member of the family of estrogen receptors and superfamily of nuclear receptor transcription factors. The gene product contains an N-terminal DNA binding domain and C-terminal ligand binding domain and is localized to the nucleus, cytoplasm, and mitochondria. Upon binding to 17beta-estradiol or related ligands, the encoded protein forms homo- or hetero-dimers that interact with specific DNA sequences to activate transcription. Some isoforms dominantly inhibit the activity of other estrogen receptor family members. Several alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been fully characterized. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 119895837, 119920716, 46.85, 5, 'ENSG00000140009', 'Q92731'),
(785, 'ETS2', 'This gene encodes a transcription factor which regulates genes involved in development and apoptosis. The encoded protein is also a protooncogene and shown to be involved in regulation of telomerase. A pseudogene of this gene is located on the X chromosome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2012]', '21', 'protein_coding', 112485011, 112588205, 42.88, 8, 'ENSG00000157557', 'P15036'),
(786, 'ETV5', 'Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in cellular response to oxidative stress; negative regulation of transcription by RNA polymerase II; and positive regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 105107324, 105169130, 35.27, 14, 'ENSG00000244405', 'P41161'),
(787, 'EVA1A', 'Predicted to act upstream of or within several processes, including TOR signaling; cardiac left ventricle morphogenesis; and connective tissue replacement. Located in intracellular membrane-bounded organelle and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 122908056, 122913323, 36.77, 1, 'ENSG00000115363', 'Q9H8M9');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(788, 'EWSR1', 'This gene encodes a multifunctional protein that is involved in various cellular processes, including gene expression, cell signaling, and RNA processing and transport. The protein includes an N-terminal transcriptional activation domain and a C-terminal RNA-binding domain. Chromosomal translocations between this gene and various genes encoding transcription factors result in the production of chimeric proteins that are involved in tumorigenesis. These chimeric proteins usually consist of the N-terminal transcriptional activation domain of this protein fused to the C-terminal DNA-binding domain of the transcription factor protein. Mutations in this gene, specifically a t(11;22)(q24;q12) translocation, are known to cause Ewing sarcoma as well as neuroectodermal and various other tumors. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 1 and 14. [provided by RefSeq, Jul 2009]', '22', 'protein_coding', 230415942, 230545606, 39.21, 19, 'ENSG00000182944', 'Q01844'),
(789, 'EXOC2', 'The protein encoded by this gene is a component of the exocyst complex, a multi-protein complex essential for the polarized targeting of exocytic vesicles to specific docking sites on the plasma membrane. Though best characterized in yeast, the component proteins and the functions of the exocyst complex have been demonstrated to be highly conserved in higher eukaryotes. At least eight components of the exocyst complex, including this protein, are found to interact with the actin cytoskeletal remodeling and vesicle transport machinery. This interaction has been shown to mediate filopodia formation in fibroblasts. This protein has been shown to interact with the Ral subfamily of GTPases and thereby mediate exocytosis by tethering vesicles to the plasma membrane. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2012]', '6', 'protein_coding', 20535825, 20565809, 39.46, 5, 'ENSG00000112685', 'Q96KP1'),
(790, 'EXOC6', 'The protein encoded by this gene is highly similar to the Saccharomyces cerevisiae SEC15 gene product, which is essential for vesicular traffic from the Golgi apparatus to the cell surface in yeast. It is one of the components of a multiprotein complex required for exocytosis. The 5\' portion of this gene and two neighboring cytochrome p450 genes are included in a deletion that results in an autosomal-dominant form of nonsyndromic optic nerve aplasia (ONA). Alternative splicing and the use of alternative promoters results in multiple transcript variants. A paralogous gene encoding a similar protein is present on chromosome 2. [provided by RefSeq, Jan 2016]', '10', 'protein_coding', 58985686, 59008238, 53.07, 32, 'ENSG00000138190', 'Q8TAG9'),
(791, 'F10', 'This gene encodes the vitamin K-dependent coagulation factor X of the blood coagulation cascade. This factor undergoes multiple processing steps before its preproprotein is converted to a mature two-chain form by the excision of the tripeptide RKR. Two chains of the factor are held together by 1 or more disulfide bonds; the light chain contains 2 EGF-like domains, while the heavy chain contains the catalytic domain which is structurally homologous to those of the other hemostatic serine proteases. The mature factor is activated by the cleavage of the activation peptide by factor IXa (in the intrisic pathway), or by factor VIIa (in the extrinsic pathway). The activated factor then converts prothrombin to thrombin in the presence of factor Va, Ca+2, and phospholipid during blood clotting. Mutations of this gene result in factor X deficiency, a hemorrhagic condition of variable severity. Alternative splicing results in multiple transcript variants encoding different isoforms that may undergo similar proteolytic processing to generate mature polypeptides. [provided by RefSeq, Aug 2015]', '13', 'protein_coding', 47485275, 47593486, 37.61, 7, 'ENSG00000126218', 'P00742'),
(792, 'F3', 'This gene encodes coagulation factor III which is a cell surface glycoprotein. This factor enables cells to initiate the blood coagulation cascades, and it functions as the high-affinity receptor for the coagulation factor VII. The resulting complex provides a catalytic event that is responsible for initiation of the coagulation protease cascades by specific limited proteolysis. Unlike the other cofactors of these protease cascades, which circulate as nonfunctional precursors, this factor is a potent initiator that is fully functional when expressed on cell surfaces, for example, on monocytes. There are 3 distinct domains of this factor: extracellular, transmembrane, and cytoplasmic. Platelets and monocytes have been shown to express this coagulation factor under procoagulatory and proinflammatory stimuli, and a major role in HIV-associated coagulopathy has been described. Platelet-dependent monocyte expression of coagulation factor III has been described to be associated with Coronavirus Disease 2019 (COVID-19) severity and mortality. This protein is the only one in the coagulation pathway for which a congenital deficiency has not been described. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Aug 2020]', '1', 'protein_coding', 23045633, 23049672, 54.68, 1, 'ENSG00000117525', 'P13726'),
(793, 'F8', 'This gene encodes coagulation factor VIII, which participates in the intrinsic pathway of blood coagulation; factor VIII is a cofactor for factor IXa which, in the presence of Ca+2 and phospholipids, converts factor X to the activated form Xa.  This gene produces two alternatively spliced transcripts. Transcript variant 1 encodes a large glycoprotein, isoform a, which circulates in plasma and associates with von Willebrand factor in a noncovalent complex. This protein undergoes multiple cleavage events. Transcript variant 2 encodes a putative small protein, isoform b, which consists primarily of the phospholipid binding domain of factor VIIIc. This binding domain is essential for coagulant activity. Defects in this gene results in hemophilia A, a common recessive X-linked coagulation disorder. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 91563091, 91950228, 44.96, 8, 'ENSG00000185010', 'P00451'),
(794, 'F8A2', 'This gene is part of a region that is repeated three times on chromosome X, once in intron 22 of the F8 gene and twice closer to the Xq telomere. This record represents the middle copy. Although its function is unknown, the observation that this gene is conserved in the mouse implies it has some function. Unlike factor VIII, this gene is transcribed abundantly in a wide variety of cell types. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 70050438, 70058848, 32.73, 4, 'ENSG00000288709', 'P23610'),
(795, 'F8A3', 'This gene is part of a region that is repeated three times on chromosome X, once in intron 22 of the F8 gene and twice closer to the Xq telomere. This record represents the most telomeric copy. Although its function is unknown, the observation that this gene is conserved in the mouse implies it has some function. Unlike factor VIII, this gene is transcribed abundantly in a wide variety of cell types. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 111662621, 111997464, 38.48, 23, 'ENSG00000277150', 'P23610'),
(796, 'FAAP100', 'FAAP100 is a component of the Fanconi anemia (FA; MIM 277650) core complex and is required for core complex stability and FANCD2 (see MIM 227646) monoubiquitination (Ling et al., 2007 [PubMed 17396147]).[supplied by OMIM, Mar 2008]', '17', 'unprocessed_pseudogene', 5092676, 5093611, 45.51, 1, 'ENSG00000185504', 'Q0VG06'),
(797, 'FABP7', 'The gene encodes a small, highly conserved cytoplasmic protein that bind long-chain fatty acids and other hydrophobic ligands. The encoded protein is important in the establishment of the radial glial fiber in the developing brain. Alternative splicing and promoter usage results in multiple transcript variants encoding different isoforms. Pseudogenes of this gene are found on multiple chromosomes. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 102736905, 103019900, 39.03, 11, 'ENSG00000164434', 'O15540'),
(798, 'FAHD2B', 'Enables oxaloacetate tautomerase activity. Involved in oxaloacetate metabolic process. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 46161148, 46184476, 50.77, 2, 'ENSG00000291581', 'Q6P2I3'),
(799, 'FAHD2B', 'Enables oxaloacetate tautomerase activity. Involved in oxaloacetate metabolic process. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 46161148, 46184476, 50.77, 2, 'ENSG00000144199', 'Q6P2I3'),
(800, 'FAM107B', 'Predicted to act upstream of or within sensory perception of sound. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 122144058, 122159779, 41.44, 1, 'ENSG00000065809', 'Q9H098'),
(801, 'FAM124B', 'Located in mitochondrion and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'TR_V_gene', 21642889, 21643578, 43.48, 1, 'ENSG00000124019', 'Q9H5Z6'),
(802, 'FAM131B', 'Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000159784', 'Q86XD5'),
(803, 'FAM133B', 'Enables RNA binding activity. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 385233, 388463, 50.26, 2, 'ENSG00000234545', 'Q5BKY9'),
(804, 'FAM135A', 'Predicted to be involved in lipid metabolic process. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 48903180, 48911958, 52.04, 18, 'ENSG00000082269', 'Q9P2D6'),
(805, 'FAM181A', 'Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 104268336, 104268405, 48.57, 1, 'ENSG00000273533', 'Q8N9Y4'),
(806, 'FAM181A', 'Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 104268336, 104268405, 48.57, 1, 'ENSG00000140067', 'Q8N9Y4'),
(807, 'FAM200A', 'This gene encodes a protein of unknown function. The protein is weakly similar to transposase-like proteins in human and mouse. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 81158388, 81158461, 36.49, 1, 'ENSG00000221909', 'Q8TCP9'),
(808, 'FAM53B', 'Involved in positive regulation of canonical Wnt signaling pathway. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 33324387, 33359864, 41.36, 29, 'ENSG00000189319', 'Q14153'),
(809, 'FAM83F', 'Predicted to enable protein kinase binding activity. Predicted to be involved in signal transduction. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 106157, 120954, 48.91, 1, 'ENSG00000133477', 'Q8NEG4'),
(810, 'FAM83H', 'The protein encoded by this gene plays an important role in the structural development and calcification of tooth enamel. Defects in this gene are a cause of amelogenesis imperfecta type 3 (AI3). [provided by RefSeq, Mar 2010]', '8', 'protein_coding', 123681943, 123749003, 35.41, 1, 'ENSG00000180921', 'Q6ZRV2'),
(811, 'FAM83H', 'The protein encoded by this gene plays an important role in the structural development and calcification of tooth enamel. Defects in this gene are a cause of amelogenesis imperfecta type 3 (AI3). [provided by RefSeq, Mar 2010]', '8', 'protein_coding', 123681943, 123749003, 35.41, 1, 'ENSG00000273889', 'Q6ZRV2'),
(812, 'FAM90A12', 'FAM90A12 belongs to subfamily II of the primate-specific FAM90A gene family, which originated from multiple duplications and rearrangements (Bosch et al., 2007 [PubMed 17684299]). For background information on the FAM90A gene family, as well as information on the evolution of FAM90A genes, see FAM90A1 (MIM 613041).[supplied by OMIM, Jul 2010]', '8', 'protein_coding', 118722823, 118838683, 38.92, 14, 'ENSG00000292438', 'A8MX19'),
(813, 'FAM90A12', 'FAM90A12 belongs to subfamily II of the primate-specific FAM90A gene family, which originated from multiple duplications and rearrangements (Bosch et al., 2007 [PubMed 17684299]). For background information on the FAM90A gene family, as well as information on the evolution of FAM90A genes, see FAM90A1 (MIM 613041).[supplied by OMIM, Jul 2010]', '8', 'protein_coding', 118722823, 118838683, 38.92, 14, 'ENSG00000254229', 'A8MX19'),
(814, 'FAM90A3', 'FAM90A3 belongs to subfamily II of the primate-specific FAM90A gene family, which originated from multiple duplications and rearrangements (Bosch et al., 2007 [PubMed 17684299]). For background information on the FAM90A gene family, as well as information on the evolution of FAM90A genes, see FAM90A1 (MIM 613041).[supplied by OMIM, Oct 2009]', '8', 'protein_coding', 124777133, 124814885, 52.21, 3, 'ENSG00000285074', 'A0A8V8TPE2'),
(815, 'FAM90A3', 'FAM90A3 belongs to subfamily II of the primate-specific FAM90A gene family, which originated from multiple duplications and rearrangements (Bosch et al., 2007 [PubMed 17684299]). For background information on the FAM90A gene family, as well as information on the evolution of FAM90A genes, see FAM90A1 (MIM 613041).[supplied by OMIM, Oct 2009]', '8', 'protein_coding', 124777133, 124814885, 52.21, 3, 'ENSG00000233132', 'A0A8V8TPE2'),
(816, 'FAM9A', 'This gene is a member of a gene family which arose through duplication on the X chromosome. The encoded protein may be a nuclear protein that is localized to the nucleolus, and has some similarity to a synaptonemal complex protein. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 17787649, 18024561, 49.78, 20, 'ENSG00000183304', 'Q8IZU1'),
(817, 'FANCB', 'This gene encodes a member of the Fanconi anemia complementation group B. This protein is assembled into a nucleoprotein complex that is involved in the repair of DNA lesions. Mutations in this gene can cause chromosome instability and VACTERL syndrome with hydrocephalus. [provided by RefSeq, Apr 2016]', 'X', 'protein_coding', 64914361, 64944591, 41.09, 8, 'ENSG00000181544', 'Q8NB91'),
(818, 'FANCD2OS', 'This gene encodes a conserved protein of unknown function. [provided by RefSeq, Jan 2013]', '3', 'protein_coding', 41787662, 41789898, 50.87, 2, 'ENSG00000163705', 'Q96PS1'),
(819, 'FARSB', 'This gene encodes a highly conserved enzyme that belongs to the aminoacyl-tRNA synthetase class IIc subfamily. This enzyme comprises the regulatory beta subunits that form a tetramer with two catalytic alpha subunits. In the presence of ATP, this tetramer is responsible for attaching L-phenylalanine to the terminal adenosine of the appropriate tRNA. A pseudogene located on chromosome 10 has been identified. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2015]', '2', 'protein_coding', 737628, 1708476, 47.91, 15, 'ENSG00000116120', 'Q9NSD9'),
(820, 'FASTKD3', 'This gene encodes a member of a small family of Fas-activated serine/threonine kinase domain (FASTKD) containing proteins that share an amino terminal mitochondrial targeting domain and multiple carboxy terminal FAST domains as well as a putative RNA-binding RAP domain. The members of this family are ubiquitously expressed and are generally most abundant in mitochondria-enriched tissues such as heart, skeletal muscle and brown-adipose tissue. Some members of this protein family may play a role in apoptosis. The protein encoded by this gene interacts with components of the mitochondrial respiratory and translation networks. A pseudogene of this gene is also present on chromosome 5. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2013]', '5', 'protein_coding', 151176584, 151181465, 56.27, 1, 'ENSG00000124279', 'Q14CZ7'),
(821, 'FASTKD5', 'Enables rRNA binding activity. Involved in mitochondrial RNA processing. Located in mitochondrial nucleoid and ribonucleoprotein granule. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 154697855, 154858354, 34.96, 4, 'ENSG00000215251', 'Q7L8L6'),
(822, 'FAXDC2', 'Predicted to enable C-4 methylsterol oxidase activity. Involved in positive regulation of megakaryocyte differentiation and positive regulation of protein phosphorylation. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'processed_pseudogene', 133019486, 133020874, 48.31, 1, 'ENSG00000170271', 'Q96IV6'),
(823, 'FBL', 'This gene product is a component of a nucleolar small nuclear ribonucleoprotein (snRNP) particle thought to participate in the first step in processing preribosomal RNA. It is associated with the U3, U8, and U13 small nuclear RNAs and is located in the dense fibrillar component (DFC) of the nucleolus. The encoded protein contains an N-terminal repetitive domain that is rich in glycine and arginine residues, like fibrillarins in other species. Its central region resembles an RNA-binding domain and contains an RNP consensus sequence. Antisera from approximately 8% of humans with the autoimmune disease scleroderma recognize fibrillarin. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 25639258, 25717562, 39.65, 7, 'ENSG00000280548', 'P22087'),
(824, 'FBL', 'This gene product is a component of a nucleolar small nuclear ribonucleoprotein (snRNP) particle thought to participate in the first step in processing preribosomal RNA. It is associated with the U3, U8, and U13 small nuclear RNAs and is located in the dense fibrillar component (DFC) of the nucleolus. The encoded protein contains an N-terminal repetitive domain that is rich in glycine and arginine residues, like fibrillarins in other species. Its central region resembles an RNA-binding domain and contains an RNP consensus sequence. Antisera from approximately 8% of humans with the autoimmune disease scleroderma recognize fibrillarin. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 25639258, 25717562, 39.65, 7, 'ENSG00000105202', 'P22087'),
(825, 'FBRS', 'Fibrosin is a lymphokine secreted by activated lymphocytes that induces fibroblast proliferation (Prakash and Robbins, 1998 [PubMed 9809749]).[supplied by OMIM, Mar 2008]', '16', 'protein_coding', 266498, 293973, 37.87, 10, 'ENSG00000156860', 'Q9HAH7'),
(826, 'FBXL15', 'Involved in several processes, including G2/M transition of mitotic cell cycle; SCF-dependent proteasomal ubiquitin-dependent protein catabolic process; and positive regulation of BMP signaling pathway. Located in cytoplasm. Part of SCF ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 188947214, 189325304, 39.36, 15, 'ENSG00000107872', 'Q9H469'),
(827, 'FBXL5', 'This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbls class and, in addition to an F-box, contains several tandem leucine-rich repeats. Alternatively spliced transcript variants have been described for this locus. [provided by RefSeq, Aug 2010]', '4', 'protein_coding', 19383442, 19387190, 56.87, 5, 'ENSG00000118564', 'Q9UKA1'),
(828, 'FBXO15', 'Members of the F-box protein family, such as FBXO15, are characterized by an approximately 40-amino acid F-box motif. SCF complexes, formed by SKP1 (MIM 601434), cullin (see CUL1; MIM 603134), and F-box proteins, act as protein-ubiquitin ligases. F-box proteins interact with SKP1 through the F box, and they interact with ubiquitination targets through other protein interaction domains (Jin et al., 2004 [PubMed 15520277]).[supplied by OMIM, Mar 2008]', '18', 'protein_coding', 38965925, 38990859, 38.26, 2, 'ENSG00000141665', 'Q8NCQ5'),
(829, 'FBXO42', 'Members of the F-box protein family, such as FBXO42, are characterized by an approximately 40-amino acid F-box motif. SCF complexes, formed by SKP1 (SKP1A; MIM 601434), cullin (see CUL1; MIM 603134), and F-box proteins, act as protein-ubiquitin ligases. F-box proteins interact with SKP1 through the F box, and they interact with ubiquitination targets through other protein interaction domains (Jin et al., 2004 [PubMed 15520277]).[supplied by OMIM, Dec 2010]', '1', 'protein_coding', 293465, 297716, 55.97, 3, 'ENSG00000291375', 'Q6P3S6'),
(830, 'FBXO42', 'Members of the F-box protein family, such as FBXO42, are characterized by an approximately 40-amino acid F-box motif. SCF complexes, formed by SKP1 (SKP1A; MIM 601434), cullin (see CUL1; MIM 603134), and F-box proteins, act as protein-ubiquitin ligases. F-box proteins interact with SKP1 through the F box, and they interact with ubiquitination targets through other protein interaction domains (Jin et al., 2004 [PubMed 15520277]).[supplied by OMIM, Dec 2010]', '1', 'protein_coding', 293465, 297716, 55.97, 3, 'ENSG00000037637', 'Q6P3S6'),
(831, 'FBXO9', 'This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of the ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbxs class. Alternative splicing of this gene generates at least 3 transcript variants diverging at the 5\' terminus. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000112146', 'Q9UK97'),
(832, 'FBXW10', 'Members of the F-box protein family, such as FBXW10, are characterized by an approximately 40-amino acid F-box motif. SCF complexes, formed by SKP1 (MIM 601434), cullin (see CUL1; MIM 603134), and F-box proteins, act as protein-ubiquitin ligases. F-box proteins interact with SKP1 through the F box, and they interact with ubiquitination targets through other protein interaction domains (Jin et al., 2004 [PubMed 15520277]).[supplied by OMIM, Mar 2008]', '17', 'protein_coding', 5305, 153241, 48.78, 14, 'ENSG00000171931', 'Q5XX13'),
(833, 'FBXW11', 'This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbws class and, in addition to an F-box, contains multiple WD40 repeats. This gene contains at least 14 exons, and its alternative splicing generates 3 transcript variants diverging at the presence/absence of two alternate exons. [provided by RefSeq, Jul 2008]', '5', 'snoRNA', 73126080, 73126207, 50.78, 1, 'ENSG00000072803', 'Q9UKB1'),
(834, 'FBXW2', 'F-box proteins are an expanding family of eukaryotic proteins characterized by an approximately 40 amino acid motif, the F box. Some F-box proteins have been shown to be critical for the ubiquitin-mediated degradation of cellular regulatory proteins. In fact, F-box proteins are one of the four subunits of ubiquitin protein ligases, called SCFs. SCF ligases bring ubiquitin conjugating enzymes to substrates that are specifically recruited by the different F-box proteins. Mammalian F-box proteins are classified into three groups based on the presence of either WD-40 repeats, leucine-rich repeats, or the presence or absence of other protein-protein interacting domains. This gene encodes the second identified member of the F-box gene family and contains multiple WD-40 repeats. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 56961923, 56983845, 50.76, 5, 'ENSG00000119402', 'Q9UKT8'),
(835, 'FCSK', 'The protein encoded by this gene belongs to the GHMP (galacto-, homoserine, mevalonate and phosphomevalonate) kinase family and catalyzes the phosphorylation of L-fucose to form beta-L-fucose 1-phosphate. This enzyme catalyzes the first step in the utilization of free L-fucose in glycoprotein and glycolipid synthesis. L-fucose may be important in mediating a number of cell-cell interactions such as blood group antigen recognition, inflammation, and metastatis. While several transcript variants may exist for this gene, the full-length nature of only one has been described to date. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 125063302, 125090516, 52.71, 6, 'ENSG00000157353', 'Q8N0W3'),
(836, 'FERMT1', 'This gene encodes a member of the fermitin family, and contains a FERM domain and a pleckstrin homology domain. The encoded protein is involved in integrin signaling and linkage of the actin cytoskeleton to the extracellular matrix. Mutations in this gene have been linked to Kindler syndrome. [provided by RefSeq, Dec 2009]', '20', 'protein_coding', 135759846, 135768191, 42.77, 2, 'ENSG00000101311', 'Q9BQL6'),
(837, 'FEZ1', 'This gene is an ortholog of the C. elegans unc-76 gene, which is necessary for normal axonal bundling and elongation within axon bundles. Expression of this gene in C. elegans unc-76 mutants can restore to the mutants partial locomotion and axonal fasciculation, suggesting that it also functions in axonal outgrowth. The N-terminal half of the gene product is highly acidic. Alternatively spliced transcript variants encoding different isoforms of this protein have been described. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 46113686, 46113768, 53.01, 1, 'ENSG00000149557', 'Q99689'),
(838, 'FGD5', 'Predicted to enable guanyl-nucleotide exchange factor activity and small GTPase binding activity. Predicted to be involved in actin cytoskeleton organization; filopodium assembly; and regulation of cell shape. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'snoRNA', 49278940, 49279028, 41.57, 1, 'ENSG00000154783', 'Q6ZNL6'),
(839, 'FGD6', 'Predicted to enable guanyl-nucleotide exchange factor activity and small GTPase binding activity. Predicted to be involved in actin cytoskeleton organization; filopodium assembly; and regulation of cell shape. Predicted to be located in Golgi apparatus; lamellipodium; and ruffle. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 100473708, 100564414, 39.74, 11, 'ENSG00000180263', 'Q6ZV73'),
(840, 'FGF16', 'This gene encodes a member of a family of proteins that are involved in a variety of biological processes, including embryonic development, cell growth, morphogenesis, tissue repair, tumor growth and invasion. This gene is expressed in cardiac cells and is required for proper heart development. Mutation in this gene was also observed in individuals with metacarpal 4-5 fusion. [provided by RefSeq, Mar 2014]', 'X', 'protein_coding', 101009346, 101052116, 44.90, 6, 'ENSG00000196468', 'O43320'),
(841, 'FGF23', 'This gene encodes a member of the fibroblast growth factor family of proteins, which possess broad mitogenic and cell survival activities and are involved in a variety of biological processes. The product of this gene regulates phosphate homeostasis and transport in the kidney. The full-length, functional protein may be deactivated via cleavage into N-terminal and C-terminal chains. Mutation of this cleavage site causes autosomal dominant hypophosphatemic rickets (ADHR). Mutations in this gene are also associated with hyperphosphatemic familial tumoral calcinosis (HFTC). [provided by RefSeq, Feb 2013]', '12', 'protein_coding', 44826791, 44848087, 41.42, 8, 'ENSG00000118972', 'Q9GZV9'),
(842, 'FGF3', 'The protein encoded by this gene is a member of the fibroblast growth factor (FGF) family. FGF family members possess broad mitogenic and cell survival activities and are involved in a variety of biological processes including embryonic development, cell growth, morphogenesis, tissue repair, tumor growth and invasion. This gene was identified by its similarity with mouse fgf3/int-2, a proto-oncogene activated in virally induced mammary tumors in the mouse. Frequent amplification of this gene has been found in human tumors, which may be important for neoplastic transformation and tumor progression. Studies of the similar genes in mouse and chicken suggested the role in inner ear formation. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 657872, 660453, 47.06, 20, 'ENSG00000186895', 'P11487'),
(843, 'FHL1', 'This gene is a member of the Regulator of Complement Activation (RCA) gene cluster and encodes a protein with twenty short consensus repeat (SCR) domains. This protein is secreted into the bloodstream and has an essential role in the regulation of complement activation, restricting this innate defense mechanism to microbial infections. Mutations in this gene have been associated with hemolytic-uremic syndrome (HUS) and chronic hypocomplementemic nephropathy. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Oct 2011]', '1', 'protein_coding', 2799466, 2864146, 45.13, 4, 'ENSG00000022267', 'Q13642'),
(844, 'FHL2', 'This gene encodes a member of the four-and-a-half-LIM-only protein family. Family members contain two highly conserved, tandemly arranged, zinc finger domains with four highly conserved cysteines binding a zinc atom in each zinc finger. This protein is thought to have a role in the assembly of extracellular membranes. Also, this gene is down-regulated during transformation of normal myoblasts to rhabdomyosarcoma cells and the encoded protein may function as a link between presenilin-2 and an intracellular signaling pathway. Multiple alternatively spliced variants encoding different isoforms have been identified. [provided by RefSeq, Jan 2016]', '2', 'protein_coding', 141790217, 141791367, 48.39, 1, 'ENSG00000115641', 'Q14192'),
(845, 'FIGN', 'Predicted to enable ATP hydrolysis activity and microtubule severing ATPase activity. Predicted to be involved in cell division and microtubule cytoskeleton organization. Predicted to act upstream of or within locomotory behavior. Predicted to be located in nuclear matrix. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 31483002, 31490860, 59.32, 9, 'ENSG00000182263', 'Q5HY92'),
(846, 'FKBP2', 'The protein encoded by this gene is a member of the immunophilin protein family, which play a role in immunoregulation and basic cellular processes involving protein folding and trafficking. This encoded protein is a cis-trans prolyl isomerase that binds the immunosuppressants FK506 and rapamycin. It is thought to function as an ER chaperone and may also act as a component of membrane cytoskeletal scaffolds. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Sep 2008]', '11', 'miRNA', 241477905, 241477982, 28.21, 1, 'ENSG00000173486', 'P26885'),
(847, 'FLII', 'This gene encodes a protein with a gelsolin-like actin binding domain and an N-terminal leucine-rich repeat-protein protein interaction domain. The protein is similar to a Drosophila protein involved in early embryogenesis and the structural organization of indirect flight muscle. The gene is located within the Smith-Magenis syndrome region on chromosome 17. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 160755602, 161078902, 38.44, 5, 'ENSG00000284571', 'Q13045'),
(848, 'FLII', 'This gene encodes a protein with a gelsolin-like actin binding domain and an N-terminal leucine-rich repeat-protein protein interaction domain. The protein is similar to a Drosophila protein involved in early embryogenesis and the structural organization of indirect flight muscle. The gene is located within the Smith-Magenis syndrome region on chromosome 17. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 160755602, 161078902, 38.44, 5, 'ENSG00000177731', 'Q13045'),
(849, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000223654', 'O75955'),
(850, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000224740', 'O75955'),
(851, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000206480', 'O75955'),
(852, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000232280', 'O75955'),
(853, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000206379', 'O75955'),
(854, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000236271', 'O75955'),
(855, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000230143', 'O75955'),
(856, 'FLOT1', 'This gene encodes an protein that localizes to the caveolae, which are small domains on the inner cell membranes. This protein plays a role in vesicle trafficking and cell morphology. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000137312', 'O75955'),
(857, 'FLRT2', 'This gene encodes a member of the fibronectin leucine rich transmembrane (FLRT) family of cell adhesion molecules, which regulate early embryonic vascular and neural development. The encoded type I transmembrane protein has an extracellular region consisting of an N-terminal leucine-rich repeat domain and a type 3 fibronectin domain, followed by a transmembrane domain and a short C-terminal cytoplasmic tail domain. It functions as both a homophilic cell adhesion molecule and a heterophilic chemorepellent through its interaction with members of the uncoordinated-5 receptor family. Proteolytic removal of the extracellular region controls the migration of neurons in the developing cortex. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016]', '14', 'transcribed_unprocessed_pseudogene', 21207973, 21225554, 53.87, 1, 'ENSG00000185070', 'O43155'),
(858, 'FLT1', 'This gene encodes a member of the vascular endothelial growth factor receptor (VEGFR) family. VEGFR family members are receptor tyrosine kinases (RTKs) which contain an extracellular ligand-binding region with seven immunoglobulin (Ig)-like domains, a transmembrane segment, and a tyrosine kinase (TK) domain within the cytoplasmic domain. This protein binds to VEGFR-A, VEGFR-B and placental growth factor and plays an important role in angiogenesis and vasculogenesis. Expression of this receptor is found in vascular endothelial cells, placental trophoblast cells and peripheral blood monocytes. Multiple transcript variants encoding different isoforms have been found for this gene. Isoforms include a full-length transmembrane receptor isoform and shortened, soluble isoforms. The soluble isoforms are associated with the onset of pre-eclampsia.[provided by RefSeq, May 2009]', '13', 'protein_coding', 1374066, 1543711, 48.22, 16, 'ENSG00000102755', 'P17948'),
(859, 'FLT3', 'This gene encodes a class III receptor tyrosine kinase that regulates hematopoiesis. This receptor is activated by binding of the fms-related tyrosine kinase 3 ligand to the extracellular domain, which induces homodimer formation in the plasma membrane leading to autophosphorylation of the receptor. The activated receptor kinase subsequently phosphorylates and activates multiple cytoplasmic effector molecules in pathways involved in apoptosis, proliferation, and differentiation of hematopoietic cells in bone marrow. Mutations that result in the constitutive activation of this receptor result in acute myeloid leukemia and acute lymphoblastic leukemia. [provided by RefSeq, Jan 2015]', '13', 'protein_coding', 110878540, 110915069, 40.00, 11, 'ENSG00000122025', 'P36888'),
(860, 'FLT3LG', 'Dendritic cells (DCs) provide the key link between innate and adaptive immunity by recognizing pathogens and priming pathogen-specific immune responses. FLT3LG controls the development of DCs and is particularly important for plasmacytoid DCs and CD8 (see MIM 186910)-positive classical DCs and their CD103 (ITGAE; MIM 604682)-positive tissue counterparts (summary by Sathaliyawala et al., 2010 [PubMed 20933441]).[supplied by OMIM, Jan 2011]', '19', 'protein_coding', 30553764, 30558374, 58.06, 3, 'ENSG00000090554', 'P49771'),
(861, 'FLVCR2', 'This gene encodes a member of the major facilitator superfamily. The encoded transmembrane protein is a calcium transporter. Unlike the related protein feline leukemia virus subgroup C receptor 1, the protein encoded by this locus does not bind to feline leukemia virus subgroup C envelope protein. The encoded protein may play a role in development of brain vascular endothelial cells, as mutations at this locus have been associated with proliferative vasculopathy and hydranencephaly-hydrocephaly syndrome. Alternatively spliced transcript variants have been described.[provided by RefSeq, Aug 2010]', '14', 'protein_coding', 21141, 38201, 44.99, 5, 'ENSG00000119686', 'Q9UPI3'),
(862, 'FNDC9', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 70384604, 70414657, 43.91, 1, 'ENSG00000172568', 'Q8TBE3'),
(863, 'FNTB', 'Enables acetyltransferase activator activity; enzyme binding activity; and zinc ion binding activity. Contributes to protein farnesyltransferase activity. Involved in protein farnesylation and regulation of microtubule-based movement. Part of microtubule associated complex and protein farnesyltransferase complex. Implicated in anxiety disorder. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 10330229, 10330285, 50.88, 1, 'ENSG00000257365', 'P49356'),
(864, 'FOLH1', 'This gene encodes a type II transmembrane glycoprotein belonging to the M28 peptidase family. The protein acts as a glutamate carboxypeptidase on different alternative substrates, including the nutrient folate and the neuropeptide N-acetyl-l-aspartyl-l-glutamate and is expressed in a number of tissues such as prostate, central and peripheral nervous system and kidney. A mutation in this gene may be associated with impaired intestinal absorption of dietary folates, resulting in low blood folate levels and consequent hyperhomocysteinemia. Expression of this protein in the brain may be involved in a number of pathological conditions associated with glutamate excitotoxicity. In the prostate the protein is up-regulated in cancerous cells and is used as an effective diagnostic and prognostic indicator of prostate cancer. This gene likely arose from a duplication event of a nearby chromosomal region. Alternative splicing gives rise to multiple transcript variants encoding several different isoforms. [provided by RefSeq, Jul 2010]', '11', 'protein_coding', 68145, 281194, 45.86, 20, 'ENSG00000086205', 'Q04609'),
(865, 'FOXA3', 'This gene encodes a member of the forkhead class of DNA-binding proteins. These hepatocyte nuclear factors are transcriptional activators for liver-specific transcripts such as albumin and transthyretin, and they also interact with chromatin. Similar family members in mice have roles in the regulation of metabolism and in the differentiation of the pancreas and liver. The crystal structure of a similar protein in rat has been resolved. [provided by RefSeq, Jul 2008]', '19', 'miRNA', 97939056, 97939121, 31.82, 1, 'ENSG00000170608', 'P55318'),
(866, 'FOXC1', 'This gene belongs to the forkhead family of transcription factors which is characterized by a distinct DNA-binding forkhead domain.  The specific function of this gene has not yet been determined; however, it has been shown to play a role in the regulation of embryonic and ocular development.  Mutations in this gene cause various glaucoma phenotypes including primary congenital glaucoma, autosomal dominant iridogoniodysgenesis anomaly, and Axenfeld-Rieger anomaly. [provided by RefSeq, Jul 2008]', '6', 'unprocessed_pseudogene', 41050449, 41062444, 43.94, 1, 'ENSG00000054598', 'Q12948'),
(867, 'FOXJ1', 'This gene encodes a member of the forkhead family of transcription factors. Similar genes in zebrafish and mouse have been shown to regulate the transcription of genes that control the production of motile cilia. The mouse ortholog also functions in the determination of left-right asymmetry. Polymorphisms in this gene are associated with systemic lupus erythematosus and allergic rhinitis.[provided by RefSeq, Sep 2009]', '17', 'protein_coding', 49642209, 49658642, 58.14, 4, 'ENSG00000129654', 'Q92949'),
(868, 'FOXJ3', 'Enables DNA-binding transcription activator activity, RNA polymerase II-specific and sequence-specific double-stranded DNA binding activity. Involved in positive regulation of transcription by RNA polymerase II. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 149030127, 149086554, 35.95, 9, 'ENSG00000198815', 'Q9UPW0'),
(869, 'FOXL1', 'This gene encodes a member of the forkhead/winged helix-box (FOX) family of transcription factors. FOX transcription factors are characterized by a distinct DNA-binding forkhead domain and play critical roles in the regulation of multiple processes including metabolism, cell proliferation and gene expression during ontogenesis. [provided by RefSeq, Nov 2012]', '16', 'miRNA', 28724252, 28724321, 57.14, 1, 'ENSG00000176678', 'Q12952'),
(870, 'FOXR2', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 36541497, 36551447, 58.60, 2, 'ENSG00000189299', 'Q6PJQ5'),
(871, 'FOXRED2', 'Enables flavin adenine dinucleotide binding activity. Involved in ERAD pathway. Located in endoplasmic reticulum lumen. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 160388, 161570, 65.00, 1, 'ENSG00000100350', 'Q8IWF2'),
(872, 'FOXS1', 'The forkhead family of transcription factors belongs to the winged helix class of DNA-binding proteins. The protein encoded by this intronless gene contains a forkhead domain and is found predominantly in aorta and kidney. The function of the encoded protein is unknown. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 3804024, 3869038, 54.36, 10, 'ENSG00000179772', 'O43638'),
(873, 'FPR2', 'Enables amyloid-beta binding activity; scavenger receptor binding activity; and signaling receptor activity. Involved in several processes, including cellular response to amyloid-beta; positive regulation of ERK1 and ERK2 cascade; and regulation of defense response. Located in cytoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'unprocessed_pseudogene', 133575122, 133576031, 59.23, 1, 'ENSG00000171049', 'P25090'),
(874, 'FRA10AC1', 'The protein encoded by this gene is a nuclear phosphoprotein of unknown function. This gene contains a tandem CGG repeat region within a CpG island that normally consists of 8-14 repeats but can expand to over 200 repeats. The repeat region is within the 5\' UTR of some transcript variants, but is intronic to another variant. The expanded repeat allele is a fragile site and becomes hypermethylated, causing a reduction in gene expression. A disease phenotype has not been associated with expanded alleles. This gene is found within the rare FRA10A folate-sensitive fragile site. [provided by RefSeq, Dec 2016]', '10', 'protein_coding', 71754863, 71800286, 37.14, 4, 'ENSG00000148690', 'Q70Z53'),
(875, 'FREM2', 'This gene encodes an integral membrane protein containing numerous CSPG (chondroitin sulfate proteoglycan element) repeats and Calx-beta domains. The encoded protein localizes to the basement membrane, forming a ternary complex that plays a role in epidermal-dermal interactions. This protein is important for the integrity of skin and renal epithelia. Mutations in this gene are associated with Fraser syndrome. [provided by RefSeq, Apr 2014]', '13', 'protein_coding', 38687077, 38887131, 37.64, 2, 'ENSG00000150893', 'Q5SZK8'),
(876, 'FRG1', 'This gene maps to a location 100 kb centromeric of the repeat units on chromosome 4q35 which are deleted in facioscapulohumeral muscular dystrophy (FSHD). It is evolutionarily conserved and has related sequences on multiple human chromosomes but DNA sequence analysis did not reveal any homology to known genes. In vivo studies demonstrate the encoded protein is localized to the nucleolus. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 23730189, 23889400, 39.76, 5, 'ENSG00000283153', 'Q14331'),
(877, 'FRG1', 'This gene maps to a location 100 kb centromeric of the repeat units on chromosome 4q35 which are deleted in facioscapulohumeral muscular dystrophy (FSHD). It is evolutionarily conserved and has related sequences on multiple human chromosomes but DNA sequence analysis did not reveal any homology to known genes. In vivo studies demonstrate the encoded protein is localized to the nucleolus. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 23730189, 23889400, 39.76, 5, 'ENSG00000283630', 'Q14331'),
(878, 'FRG1', 'This gene maps to a location 100 kb centromeric of the repeat units on chromosome 4q35 which are deleted in facioscapulohumeral muscular dystrophy (FSHD). It is evolutionarily conserved and has related sequences on multiple human chromosomes but DNA sequence analysis did not reveal any homology to known genes. In vivo studies demonstrate the encoded protein is localized to the nucleolus. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 23730189, 23889400, 39.76, 5, 'ENSG00000275145', 'Q14331'),
(879, 'FRG1', 'This gene maps to a location 100 kb centromeric of the repeat units on chromosome 4q35 which are deleted in facioscapulohumeral muscular dystrophy (FSHD). It is evolutionarily conserved and has related sequences on multiple human chromosomes but DNA sequence analysis did not reveal any homology to known genes. In vivo studies demonstrate the encoded protein is localized to the nucleolus. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 23730189, 23889400, 39.76, 5, 'ENSG00000109536', 'Q14331'),
(880, 'FRK', 'The protein encoded by this gene belongs to the TYR family of protein kinases. This tyrosine kinase is a nuclear protein and may function during G1 and S phase of the cell cycle and suppress growth. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 134646811, 134679816, 39.66, 4, 'ENSG00000111816', 'P42685'),
(881, 'FRMD8P1', 'Predicted to be involved in negative regulation of canonical Wnt signaling pathway. Predicted to be located in cytoskeleton. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 57958437, 57974534, 45.79, 1, 'ENSG00000227942', 'Q9BZ68');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(882, 'FRRS1L', 'This gene encodes a component of the outer-core of an alpha-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor protein in the brain. The encoded protein is thought to interact with inner-core components of the receptor, and play a role in the modulation of glutamate signaling. Mutations in this gene are associated with early infantile epileptic encephalopathy 37. [provided by RefSeq, Jul 2016]', '9', 'protein_coding', 159803355, 159904756, 37.97, 4, 'ENSG00000260230', 'Q9P0K9'),
(883, 'FSHR', 'The protein encoded by this gene belongs to family 1 of G-protein coupled receptors. It is the receptor for follicle stimulating hormone and functions in gonad development. Mutations in this gene cause ovarian dysgenesis type 1, and also ovarian hyperstimulation syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2010]', '2', 'protein_coding', 50325, 69065, 53.31, 4, 'ENSG00000170820', 'P23945'),
(884, 'FST', ' Follistatin is a single-chain gonadal protein that specifically inhibits follicle-stimulating hormone release.  The single FST gene encodes two isoforms, FST317 and FST344 containing 317 and 344 amino acids respectively, resulting from alternative splicing of the precursor mRNA.  In a study in which 37 candidate genes were tested for linkage and association with polycystic ovary syndrome (PCOS) or hyperandrogenemia in 150 families, evidence was found for linkage between PCOS and follistatin. [provided by RefSeq, Jul 2008]', '5', 'TR_J_gene', 22522040, 22522102, 44.44, 1, 'ENSG00000134363', 'P19883'),
(885, 'FUNDC1', 'This gene encodes a protein with a FUN14 superfamily domain. The function of the encoded protein is not known. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 33403413, 33479348, 44.66, 7, 'ENSG00000069509', 'Q8IVP5'),
(886, 'FUT11', 'Enables fucosyltransferase activity. Involved in N-glycan fucosylation. Predicted to be located in Golgi cisterna membrane and Golgi membrane. Implicated in stomach cancer. Biomarker of stomach cancer. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 93496612, 93529092, 40.69, 3, 'ENSG00000196968', 'Q495W5'),
(887, 'FUT3', 'The Lewis histo-blood group system comprises a set of fucosylated glycosphingolipids that are synthesized by exocrine epithelial cells and circulate in body fluids. The glycosphingolipids function in embryogenesis, tissue differentiation, tumor metastasis, inflammation, and bacterial adhesion. They are secondarily absorbed to red blood cells giving rise to their Lewis phenotype. This gene is a member of the fucosyltransferase family, which catalyzes the addition of fucose to precursor polysaccharides in the last step of Lewis antigen biosynthesis. It encodes an enzyme with alpha(1,3)-fucosyltransferase and alpha(1,4)-fucosyltransferase activities. Mutations in this gene are responsible for the majority of Lewis antigen-negative phenotypes. Differences in the expression of this gene are associated with host susceptibility to viral infection. [provided by RefSeq, Aug 2020]', '19', 'protein_coding', 39498755, 39528311, 41.86, 12, 'ENSG00000292063', 'P21217'),
(888, 'FUT3', 'The Lewis histo-blood group system comprises a set of fucosylated glycosphingolipids that are synthesized by exocrine epithelial cells and circulate in body fluids. The glycosphingolipids function in embryogenesis, tissue differentiation, tumor metastasis, inflammation, and bacterial adhesion. They are secondarily absorbed to red blood cells giving rise to their Lewis phenotype. This gene is a member of the fucosyltransferase family, which catalyzes the addition of fucose to precursor polysaccharides in the last step of Lewis antigen biosynthesis. It encodes an enzyme with alpha(1,3)-fucosyltransferase and alpha(1,4)-fucosyltransferase activities. Mutations in this gene are responsible for the majority of Lewis antigen-negative phenotypes. Differences in the expression of this gene are associated with host susceptibility to viral infection. [provided by RefSeq, Aug 2020]', '19', 'protein_coding', 39498755, 39528311, 41.86, 12, 'ENSG00000171124', 'P21217'),
(889, 'FUT8', 'This gene encodes an enzyme belonging to the family of fucosyltransferases. The product of this gene catalyzes the transfer of fucose from GDP-fucose to N-linked type complex glycopeptides. This enzyme is distinct from other fucosyltransferases which catalyze alpha1-2, alpha1-3, and alpha1-4 fucose addition. The expression of this gene may contribute to the malignancy of cancer cells and to their invasive and metastatic capabilities. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2011]', '14', 'protein_coding', 57600656, 57622213, 42.81, 12, 'ENSG00000033170', 'Q9BYC5'),
(890, 'FUZ', 'This gene encodes a planar cell polarity protein that is involved in ciliogenesis and directional cell movement. Knockout studies in mice exhibit neural tube defects and defective cilia, and mutations in this gene are associated with neural tube defects in humans. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2012]', '19', 'protein_coding', 91264433, 91271326, 47.62, 1, 'ENSG00000010361', 'Q9BT04'),
(891, 'FXN', 'This nuclear gene encodes a mitochondrial protein which belongs to the FRATAXIN family. The protein functions in regulating mitochondrial iron transport and respiration. The expansion of intronic trinucleotide repeat GAA from 8-33 repeats to >90 repeats results in Friedreich ataxia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2016]', '9', 'protein_coding', 2948159, 2972165, 47.23, 25, 'ENSG00000165060', 'Q16595'),
(892, 'FXYD4', 'This gene encodes a member of a family of small membrane proteins that share a 35-amino acid signature sequence domain, beginning with the sequence PFXYD and containing 7 invariant and 6 highly conserved amino acids. The approved human gene nomenclature for the family is FXYD-domain containing ion transport regulator. FXYD4, originally named CHIF for channel-inducing factor, has been shown to modulate the properties of the Na,K-ATPase, as has FXYD2, also known as the gamma subunit of the Na,K-ATPase, and FXYD7. Transmembrane topology has been established for FXYD4 and two family members (FXYD1 and FXYD2), with the N-terminus extracellular and the C-terminus on the cytoplasmic side of the membrane. Alternatively spliced transcript variants encoding the same protein have been found.[provided by RefSeq, May 2010]', '10', 'protein_coding', 333625, 337512, 49.85, 1, 'ENSG00000150201', 'P59646'),
(893, 'FZD1', ' Members of the \'frizzled\' gene family encode 7-transmembrane domain proteins that are receptors for Wnt signaling proteins.  The FZD1 protein contains a signal peptide, a cysteine-rich domain in the N-terminal extracellular region, 7 transmembrane domains, and a C-terminal PDZ domain-binding motif.  The FZD1 transcript is expressed in various tissues. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 112299612, 112299690, 54.43, 1, 'ENSG00000157240', 'Q9UP38'),
(894, 'G3BP1', 'This gene encodes one of the DNA-unwinding enzymes which prefers partially unwound 3\'-tailed substrates and can also unwind partial RNA/DNA and RNA/RNA duplexes in an ATP-dependent fashion. This enzyme is a member of the heterogeneous nuclear RNA-binding proteins and is also an element of the Ras signal transduction pathway. It binds specifically to the Ras-GTPase-activating protein by associating with its SH3 domain. Several alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 141391995, 141432454, 62.40, 8, 'ENSG00000145907', 'Q13283'),
(895, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000232569', 'Q9UBS5'),
(896, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000206466', 'Q9UBS5'),
(897, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000232632', 'Q9UBS5'),
(898, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000206511', 'Q9UBS5'),
(899, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000237112', 'Q9UBS5'),
(900, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000237051', 'Q9UBS5'),
(901, 'GABBR1', 'This gene encodes a receptor for gamma-aminobutyric acid (GABA), which is the main inhibitory neurotransmitter in the mammalian central nervous system. This receptor functions as a heterodimer with GABA(B) receptor 2. Defects in this gene may underlie brain disorders such as schizophrenia and epilepsy. Alternative splicing generates multiple transcript variants, but the full-length nature of some of these variants has not been determined. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 6967337, 6967606, 57.78, 1, 'ENSG00000204681', 'Q9UBS5'),
(902, 'GABRA6', 'GABA is the major inhibitory neurotransmitter in the mammalian brain where it acts at GABA-A receptors, which are ligand-gated chloride channels. Chloride conductance of these channels can be modulated by agents such as benzodiazepines that bind to the GABA-A receptor. At least 16 distinct subunits of GABA-A receptors have been identified. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 67168, 84556, 59.97, 1, 'ENSG00000145863', 'Q16445'),
(903, 'GABRQ', 'The gamma-aminobutyric acid (GABA) A receptor is a multisubunit chloride channel that mediates the fastest inhibitory synaptic transmission in the central nervous system. This gene encodes the theta subunit of the GABA A receptor. The gene is mapped to chromosome Xq28 in a cluster of genes including those that encode the alpha 3 and epsilon subunits of the GABA A receptor. [provided by RefSeq, Jul 2017]', 'X', 'protein_coding', 57771492, 57782541, 48.38, 5, 'ENSG00000268089', 'Q9UN88'),
(904, 'GAL3ST1', 'Sulfonation, an important step in the metabolism of many drugs, xenobiotics, hormones, and neurotransmitters, is catalyzed by sulfotransferases. This gene encodes galactosylceramide sulfotransferase, which catalyzes the sulfation of membrane glycolipids including the final step in the synthesis of sulfatide, a major lipid component of the myelin sheath. This gene exhibits elevated expression in ovarian epithelial carcinoma and the encoded enzyme exhibits elevated activity in renal cell carcinoma. Mutations in this gene may be associated with reduced insulin resistance. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2015]', '22', 'protein_coding', 30092314, 30096915, 58.41, 12, 'ENSG00000128242', 'Q99999'),
(905, 'GAL3ST2', 'This gene encodes a member of the galactose-3-O-sulfotransferase protein family. The product of this gene catalyzes sulfonation by transferring a sulfate group to the hydroxyl at C-3 of nonreducing beta-galactosyl residues, and it can act on both type 1 and type 2 (Galbeta 1-3/1-4GlcNAc-R) oligosaccharides with similar efficiencies, and on core 1 glycans. This enzyme has been implicated in tumor metastasis processes. This gene is different from the GAL3ST3 gene located on chromosome 11, which has also been referred to as GAL3ST2 and encodes a related enzyme with distinct tissue distribution and substrate specificities, compared to galactose-3-O-sulfotransferase 2. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 143755089, 143757696, 41.33, 1, 'ENSG00000276126', 'Q9H3Q3'),
(906, 'GAL3ST2', 'This gene encodes a member of the galactose-3-O-sulfotransferase protein family. The product of this gene catalyzes sulfonation by transferring a sulfate group to the hydroxyl at C-3 of nonreducing beta-galactosyl residues, and it can act on both type 1 and type 2 (Galbeta 1-3/1-4GlcNAc-R) oligosaccharides with similar efficiencies, and on core 1 glycans. This enzyme has been implicated in tumor metastasis processes. This gene is different from the GAL3ST3 gene located on chromosome 11, which has also been referred to as GAL3ST2 and encodes a related enzyme with distinct tissue distribution and substrate specificities, compared to galactose-3-O-sulfotransferase 2. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 143755089, 143757696, 41.33, 1, 'ENSG00000154252', 'Q9H3Q3'),
(907, 'GAL3ST4', 'This gene encodes a member of the galactose-3-O-sulfotransferase protein family. The product of this gene catalyzes sulfonation by transferring a sulfate to the C-3\' position of galactose residues in O-linked glycoproteins. This enzyme is highly specific for core 1 structures, with asialofetuin, Gal-beta-1,3-GalNAc and Gal-beta-1,3 (GlcNAc-beta-1,6)GalNAc being good substrates. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 55349306, 55354719, 59.73, 10, 'ENSG00000197093', 'Q96RP7'),
(908, 'GALR1', 'The neuropeptide galanin elicits a range of biological effects by interaction with specific G-protein-coupled receptors.  Galanin receptors are seven-transmembrane proteins shown to activate a variety of intracellular second-messenger pathways.  GALR1 inhibits adenylyl cyclase via a G protein of the Gi/Go family.  GALR1 is widely expressed in the brain and spinal cord, as well as in peripheral sites such as the small intestine and heart. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 36819222, 36829104, 38.45, 2, 'ENSG00000166573', 'P47211'),
(909, 'GALR3', 'The neuropeptide galanin modulates a variety of physiologic processes including cognition/memory, sensory/pain processing, hormone secretion, and feeding behavior.  The human galanin receptors are G protein-coupled receptors that functionally couple to their intracellular effector through distinct signaling pathways.  GALR3 is found in many tissues and may be expressed as 1.4-, 2.4-, and 5-kb transcripts [provided by RefSeq, Jul 2008]', '22', 'unprocessed_pseudogene', 107290, 108220, 41.57, 1, 'ENSG00000128310', 'O60755'),
(910, 'GARRE1', 'Enables CCR4-NOT complex binding activity and small GTPase binding activity. Involved in Rac protein signal transduction. Located in P-body. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 180015303, 180015397, 38.95, 1, 'ENSG00000166398', 'O15063'),
(911, 'GARRE1', 'Enables CCR4-NOT complex binding activity and small GTPase binding activity. Involved in Rac protein signal transduction. Located in P-body. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 180015303, 180015397, 38.95, 1, 'ENSG00000279389', 'O15063'),
(912, 'GARRE1', 'Enables CCR4-NOT complex binding activity and small GTPase binding activity. Involved in Rac protein signal transduction. Located in P-body. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 180015303, 180015397, 38.95, 1, 'ENSG00000282735', 'O15063'),
(913, 'GARS1', 'This gene encodes glycyl-tRNA synthetase, one of the aminoacyl-tRNA synthetases that charge tRNAs with their cognate amino acids. The encoded enzyme is an (alpha)2 dimer which belongs to the class II family of tRNA synthetases. It has been shown to be a target of autoantibodies in the human autoimmune diseases, polymyositis or dermatomyositis. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2015]', '7', 'protein_coding', 22484421, 22508742, 41.13, 3, 'ENSG00000106105', 'P41250'),
(914, 'GAS7', 'This gene encodes a transcriptional coactivator that plays an essential role in regulating gene expression during early development and hematopoiesis. The encoded protein contains multiple conserved functional domains. One of these domains, the SET domain, is responsible for its histone H3 lysine 4 (H3K4) methyltransferase activity which mediates chromatin modifications associated with epigenetic transcriptional activation. This protein is processed by the enzyme Taspase 1 into two fragments, MLL-C and MLL-N. These fragments reassociate and further assemble into different multiprotein complexes that regulate the transcription of specific target genes, including many of the HOX genes. Multiple chromosomal translocations involving this gene are the cause of certain acute lymphoid leukemias and acute myeloid leukemias. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Oct 2010]', '11', 'protein_coding', 55112971, 55173177, 39.63, 5, 'ENSG00000007237', 'O60861'),
(915, 'GATA6', 'This gene is a member of a small family of zinc finger transcription factors that play an important role in the regulation of cellular differentiation and organogenesis during vertebrate development. This gene is expressed during early embryogenesis and localizes to endo- and mesodermally derived cells during later embryogenesis and thereby plays an important role in gut, lung, and heart development. Mutations in this gene are associated with several congenital defects. [provided by RefSeq, Mar 2012]', '18', 'protein_coding', 24365673, 24379604, 56.84, 33, 'ENSG00000141448', 'Q92908'),
(916, 'GBE1', 'The protein encoded by this gene is a glycogen branching enzyme that catalyzes the transfer of alpha-1,4-linked glucosyl units from the outer end of a glycogen chain to an alpha-1,6 position on the same or a neighboring glycogen chain. Branching of the chains is essential to increase the solubility of the glycogen molecule and, consequently, in reducing the osmotic pressure within cells. Highest level of this enzyme are found in liver and muscle. Mutations in this gene are associated with glycogen storage disease IV (also known as Andersen\'s disease). [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 64359148, 64372215, 56.99, 8, 'ENSG00000114480', 'Q04446'),
(917, 'GCLM', 'Glutamate-cysteine ligase, also known as gamma-glutamylcysteine synthetase, is the first rate limiting enzyme of glutathione synthesis. The enzyme consists of two subunits, a heavy catalytic subunit and a light regulatory subunit. Gamma glutamylcysteine synthetase deficiency has been implicated in some forms of hemolytic anemia. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Apr 2015]', '1', 'protein_coding', 68087897, 68212017, 40.41, 23, 'ENSG00000023909', 'P48507'),
(918, 'GCM2', 'This gene is a homolog of the Drosophila glial cells missing gene, which is thought to act as a binary switch between neuronal and glial cell determination. The protein encoded by this gene contains a conserved N-terminal GCM motif that has DNA-binding activity. The protein is a transcription factor that acts as a master regulator of parathyroid development. It has been suggested that this transcription factor might mediate the effect of calcium on parathyroid hormone expression and secretion in parathyroid cells. Mutations in this gene are associated with hypoparathyroidism. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 66016752, 66026479, 56.69, 2, 'ENSG00000124827', 'O75603'),
(919, 'GCSH', 'Degradation of glycine is brought about by the glycine cleavage system, which is composed of four mitochondrial protein components: P protein (a pyridoxal phosphate-dependent glycine decarboxylase), H protein (a lipoic acid-containing protein), T protein (a tetrahydrofolate-requiring enzyme), and L protein (a lipoamide dehydrogenase). The protein encoded by this gene is the H protein, which transfers the methylamine group of glycine from the P protein to the T protein. Defects in this gene are a cause of nonketotic hyperglycinemia (NKH). Two transcript variants, one protein-coding and the other probably not protein-coding,have been found for this gene. Also, several transcribed and non-transcribed pseudogenes of this gene exist throughout the genome.[provided by RefSeq, Jan 2010]', '16', 'protein_coding', 30440275, 30440869, 47.06, 1, 'ENSG00000140905', 'P23434'),
(920, 'GCSH', 'Degradation of glycine is brought about by the glycine cleavage system, which is composed of four mitochondrial protein components: P protein (a pyridoxal phosphate-dependent glycine decarboxylase), H protein (a lipoic acid-containing protein), T protein (a tetrahydrofolate-requiring enzyme), and L protein (a lipoamide dehydrogenase). The protein encoded by this gene is the H protein, which transfers the methylamine group of glycine from the P protein to the T protein. Defects in this gene are a cause of nonketotic hyperglycinemia (NKH). Two transcript variants, one protein-coding and the other probably not protein-coding,have been found for this gene. Also, several transcribed and non-transcribed pseudogenes of this gene exist throughout the genome.[provided by RefSeq, Jan 2010]', '16', 'protein_coding', 30440275, 30440869, 47.06, 1, 'ENSG00000291958', 'P23434'),
(921, 'GDA', 'This gene is a member of the kernel lipocalin superfamily whose members share relatively low sequence similarity but have highly conserved exon/intron structure and three-dimensional protein folding. Most lipocalins are clustered on the long arm of chromosome 9. The encoded glycoprotein has been previously referred to as pregnancy-associated endometrial alpha-2-globulin, placental protein 14, and glycodelin, but has been officially named progestagen-associated endometrial protein. Three distinct forms, with identical protein backbones but different glycosylation profiles, are found in amniotic fluid, follicular fluid and seminal plasma of the reproductive system. These glycoproteins have distinct and essential roles in regulating a uterine environment suitable for pregnancy and in the timing and occurrence of the appropriate sequence of events in the fertilization process. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2015]', '9', 'protein_coding', 19051, 23212, 47.28, 27, 'ENSG00000119125', 'Q9Y2T3'),
(922, 'GDF11', 'This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. This protein plays a role in the development of the nervous and other organ systems, and may regulate aging. [provided by RefSeq, Aug 2016]', '12', 'protein_coding', 13849247, 13996443, 40.67, 12, 'ENSG00000135414', 'O95390'),
(923, 'GDF2', 'This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. This protein regulates cartilage and bone development, angiogenesis and differentiation of cholinergic central nervous system neurons. Mutations in this gene are associated with hereditary hemorrhagic telangiectasia. [provided by RefSeq, Jul 2016]', '10', 'protein_coding', 32819954, 32850405, 44.38, 1, 'ENSG00000263761', 'Q9UK05'),
(924, 'GDPD3', 'Enables phosphoric diester hydrolase activity. Involved in N-acylethanolamine metabolic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 116671192, 116850251, 47.79, 30, 'ENSG00000102886', 'Q7L5L3'),
(925, 'GEMIN6', 'GEMIN6 is part of a large macromolecular complex, localized to both the cytoplasm and the nucleus, that plays a role in the cytoplasmic assembly of small nuclear ribonucleoproteins (snRNPs). Other members of this complex include SMN (MIM 600354), GEMIN2 (SIP1; MIM 602595), GEMIN3 (DDX20; MIM 606168), GEMIN4 (MIM 606969), and GEMIN5 (MIM 607005).[supplied by OMIM, Jul 2002]', '2', 'miRNA', 101046455, 101046538, 44.05, 1, 'ENSG00000152147', 'Q8WXD5'),
(926, 'GET1', 'This gene is located in the candidate region for congenital heart disease (CHD) in Down syndrome (DS). It encodes a basic protein that functions as a receptor that promotes insertion of tail-anchored proteins in the endoplasmic reticulum membrane. This gene is located at a maternally-methylated differentially methylated region (DMR); however, its transcription may be biallelic, not imprinted. Alternative splicing results in different transcript variants. A pseudogene has been defined on chromosome 4. [provided by RefSeq, Apr 2017]', '21', 'protein_coding', 37422666, 37436990, 53.09, 12, 'ENSG00000182093', 'O00258'),
(927, 'GGCT', 'The protein encoded by this gene catalyzes the formation of 5-oxoproline from gamma-glutamyl dipeptides, the penultimate step in glutathione catabolism, and may play a critical role in glutathione homeostasis. The encoded protein may also play a role in cell proliferation, and the expression of this gene is a potential marker for cancer. Pseudogenes of this gene are located on the long arm of chromosome 5 and the short arm of chromosomes 2 and 20. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Dec 2010]', '7', 'miRNA', 32732981, 32733077, 53.61, 1, 'ENSG00000006625', 'O75223'),
(928, 'GGT2P', 'This gene (GGT2P) appears to be a human duplication of the gamma-glutamyltransferase 1 (GGT1) gene. While GGT2P and GGT1 can potentially encode proteins of the same size and with 94% identity, the ability of GGT2P to encode a stable, functional enzyme has been disputed (see PMID:23682772). Since it is also expressed at a much lower level than GGT1, GGT2P is now thought to be a processed pseudogene. [provided by RefSeq, Jan 2022]', '22', 'protein_coding', 99546300, 99558536, 44.01, 2, 'ENSG00000133475', 'P36268'),
(929, 'GHRL', 'This gene encodes the ghrelin-obestatin preproprotein that is cleaved to yield two peptides, ghrelin and obestatin. Ghrelin is a powerful appetite stimulant and plays an important role in energy homeostasis. Its secretion is initiated when the stomach is empty, whereupon it binds to the growth hormone secretagogue receptor in the hypothalamus which results in the secretion of growth hormone (somatotropin). Ghrelin is thought to regulate multiple activities, including hunger, reward perception via the mesolimbic pathway, gastric acid secretion, gastrointestinal motility, and pancreatic glucose-stimulated insulin secretion. It was initially proposed that obestatin plays an opposing role to ghrelin by promoting satiety and thus decreasing food intake, but this action is still debated. Recent reports suggest multiple metabolic roles for obestatin, including regulating adipocyte function and glucose metabolism. Alternative splicing results in multiple transcript variants. In addition, antisense transcripts for this gene have been identified and may potentially regulate ghrelin-obestatin preproprotein expression. [provided by RefSeq, Nov 2014]', '3', 'protein_coding', 3308565, 3346652, 43.18, 1, 'ENSG00000157017', 'Q9UBU3'),
(930, 'GID8', 'Predicted to enable protein homodimerization activity. Involved in positive regulation of canonical Wnt signaling pathway and positive regulation of cell population proliferation. Located in cell junction; cytosol; and nucleoplasm. Part of ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'unprocessed_pseudogene', 41442770, 41443666, 49.72, 1, 'ENSG00000101193', 'Q9NWU2'),
(931, 'GIMAP1-GIMAP5', 'This locus represents naturally occurring readthrough transcription between the neighboring GIMAP1 (GTPase, IMAP family member 1) and GIMAP5 (GTPase, IMAP family member 5) genes on chromosome 7. Alternative splicing results in multiple readthrough transcript variants, one of which encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jan 2015]', '7', 'protein_coding', 10286955, 10288522, 60.71, 3, 'ENSG00000281887', 'A0A087WTJ2'),
(932, 'GINS3', 'This gene encodes a protein subunit of the GINS heterotetrameric complex, which is essential for the initiation of DNA replication and replisome progression in eukaryotes. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 984332, 998438, 62.24, 11, 'ENSG00000181938', 'Q9BRX5'),
(933, 'GIPC2', 'Enables identical protein binding activity. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 61897301, 61920269, 59.29, 6, 'ENSG00000137960', 'Q8TF65'),
(934, 'GJB1', 'This gene encodes a member of the gap junction protein family. The gap junction proteins are membrane-spanning proteins that assemble to form gap junction channels that facilitate the transfer of ions and small molecules between cells. According to sequence similarities at the nucleotide and amino acid levels, the gap junction proteins are divided into two categories, alpha and beta. Mutations in this gene cause X-linked Charcot-Marie-Tooth disease, an inherited peripheral neuropathy. Alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Oct 2008]', 'X', 'protein_coding', 104099214, 104157009, 42.85, 2, 'ENSG00000169562', 'P08034'),
(935, 'GJB2', 'This gene encodes a member of the gap junction protein family. The gap junctions were first characterized by electron microscopy as regionally specialized structures on plasma membranes of contacting adherent cells. These structures were shown to consist of cell-to-cell channels that facilitate the transfer of ions and small molecules between cells. The gap junction proteins, also known as connexins, purified from fractions of enriched gap junctions from different tissues differ. According to sequence similarities at the nucleotide and amino acid levels, the gap junction proteins are divided into two categories, alpha and beta. Mutations in this gene are responsible for as much as 50% of pre-lingual, recessive deafness. [provided by RefSeq, Oct 2008]', '13', 'protein_coding', 69400888, 69849087, 36.19, 5, 'ENSG00000165474', 'P29033'),
(936, 'GJB4', 'This gene encodes a transmembrane connexin protein that is a component of gap junctions. Mutations in this gene have been associated with erythrokeratodermia variabilis, progressive symmetric erythrokeratoderma and hearing impairment. [provided by RefSeq, Dec 2009]', '1', 'protein_coding', 31368601, 31373923, 41.97, 1, 'ENSG00000189433', 'Q9NTQ9'),
(937, 'GJB5', 'This gene encodes a member of the beta-type (group I) connexin family. The encoded protein is a gap junction protein involved in intercellular communication related to epidermal differentiation and environmental sensing. This gene has been linked to non-small cell lung cancer. [provided by RefSeq, Nov 2012]', '1', 'protein_coding', 352112, 447654, 40.08, 8, 'ENSG00000189280', 'O95377'),
(938, 'GJB6', 'Gap junctions allow the transport of ions and metabolites between the cytoplasm of adjacent cells. They are formed by two hemichannels, made up of six connexin proteins assembled in groups. Each connexin protein has four transmembrane segments, two extracellular loops, a cytoplasmic loop formed between the two inner transmembrane segments, and the N- and C-terminus both being in the cytoplasm. The specificity of the gap junction is determined by which connexin proteins comprise the hemichannel. In the past, connexin protein names were based on their molecular weight, however the new nomenclature uses sequential numbers based on which form (alpha or beta) of the gap junction is present. This gene encodes one of the connexin proteins. Mutations in this gene have been found in some forms of deafness and in some families with hidrotic ectodermal dysplasia. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 51386957, 51562831, 47.94, 7, 'ENSG00000121742', 'O95452'),
(939, 'GJC3', 'This gene encodes a gap junction protein. The encoded protein, also known as a connexin, plays a role in formation of gap junctions, which provide direct connections between neighboring cells. Mutations in this gene have been reported to be associated with nonsyndromic hearing loss.[provided by RefSeq, Feb 2010]', '7', 'protein_coding', 72881, 73972, 54.95, 1, 'ENSG00000176402', 'Q8NFK1'),
(940, 'GK5', 'Predicted to enable glycerol kinase activity. Predicted to be involved in glycerol metabolic process; glycerol-3-phosphate biosynthetic process; and triglyceride metabolic process. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 114439435, 114450279, 39.48, 19, 'ENSG00000175066', 'Q6ZS86'),
(941, 'GLE1', 'This gene encodes a predicted 75-kDa polypeptide with high sequence and structure homology to yeast Gle1p, which is nuclear protein with a leucine-rich nuclear export sequence essential for poly(A)+RNA export. Inhibition of human GLE1L by microinjection of antibodies against GLE1L in HeLa cells resulted in inhibition of poly(A)+RNA export. Immunoflourescence studies show that GLE1L is localized at the nuclear pore complexes. This localization suggests that GLE1L may act at a terminal step in the export of mature RNA messages to the cytoplasm. Two alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 445828, 447782, 49.05, 10, 'ENSG00000119392', 'Q53GS7'),
(942, 'GLI1', 'This gene encodes a member of the Kruppel family of zinc finger proteins. The encoded transcription factor is activated by the sonic hedgehog signal transduction cascade and regulates stem cell proliferation. The activity and nuclear localization of this protein is negatively regulated by p53 in an inhibitory loop. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2009]', '12', 'miRNA', 32583656, 32583752, 36.08, 1, 'ENSG00000111087', 'P08151'),
(943, 'GLOD5', 'This gene encodes a protein with a glyoxalase domain. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 120503279, 120534434, 45.87, 8, 'ENSG00000171433', 'A6NK44'),
(944, 'GLRA3', 'This gene encodes a member of the ligand-gated ion channel protein family. The encoded protein is a member of the glycine receptor subfamily. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2013]', '4', 'IG_V_gene', 208685, 209187, 48.31, 1, 'ENSG00000145451', 'O75311'),
(945, 'GLTP', 'The protein encoded by this gene is similar to bovine and porcine proteins which accelerate transfer of certain glycosphingolipids and glyceroglycolipids between membranes. It is thought to be a cytoplasmic protein. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 154818492, 154859252, 47.15, 16, 'ENSG00000139433', 'Q9NZD2'),
(946, 'GLYATL1B', 'Predicted to enable glutamine N-acyltransferase activity. Predicted to be involved in glutamine metabolic process. Predicted to be located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '11', 'transcribed_unitary_pseudogene', 69791872, 69813888, 35.44, 1, 'ENSG00000255151', 'A0A0U1RQE8'),
(947, 'GLYR1', 'Enables several functions, including chromatin-protein adaptor activity; methylated histone binding activity; and nucleosome binding activity. Involved in transcription elongation-coupled chromatin remodeling and transcription initiation-coupled chromatin remodeling. Located in cytosol and nucleoplasm. Part of nucleosome. Is active in chromatin. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 49773638, 49921950, 39.85, 1, 'ENSG00000140632', 'Q49A26'),
(948, 'GMDS', 'GDP-mannose 4,6-dehydratase (GMD; EC 4.2.1.47) catalyzes the conversion of GDP-mannose to GDP-4-keto-6-deoxymannose, the first step in the synthesis of GDP-fucose from GDP-mannose, using NADP+ as a cofactor. The second and third steps of the pathway are catalyzed by a single enzyme, GDP-keto-6-deoxymannose 3,5-epimerase, 4-reductase, designated FX in humans (MIM 137020).[supplied by OMIM, Aug 2009]', '6', 'protein_coding', 15231, 19311, 61.53, 8, 'ENSG00000112699', 'O60547'),
(949, 'GMPR2', 'This gene encodes an enzyme that catalyzes the irreversible and NADPH-dependent reductive deamination of guanosine monophosphate (GMP) to inosine monophosphate (IMP). The protein also functions in the re-utilization of free intracellular bases and purine nucleosides. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2017]', '14', 'protein_coding', 9786429, 9949443, 47.43, 4, 'ENSG00000284752', 'Q9P2T1'),
(950, 'GMPR2', 'This gene encodes an enzyme that catalyzes the irreversible and NADPH-dependent reductive deamination of guanosine monophosphate (GMP) to inosine monophosphate (IMP). The protein also functions in the re-utilization of free intracellular bases and purine nucleosides. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2017]', '14', 'protein_coding', 9786429, 9949443, 47.43, 4, 'ENSG00000100938', 'Q9P2T1'),
(951, 'GNAI2', 'The protein encoded by this gene is an alpha subunit of guanine nucleotide binding proteins (G proteins). The encoded protein contains the guanine nucleotide binding site and is involved in the hormonal regulation of adenylate cyclase. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2013]', '3', 'protein_coding', 53180704, 53195142, 50.63, 4, 'ENSG00000114353', 'P04899'),
(952, 'GNAQ', 'This locus encodes a guanine nucleotide-binding protein. The encoded protein, an alpha subunit in the Gq class, couples a seven-transmembrane domain receptor to activation of phospolipase C-beta. Mutations at this locus have been associated with problems in platelet activation and aggregation. A related pseudogene exists on chromosome 2.[provided by RefSeq, Nov 2010]', '9', 'protein_coding', 160550, 161824, 73.25, 1, 'ENSG00000156052', 'P50148'),
(953, 'GNAS-AS1', 'This gene produces a paternally-imprinted antisense RNA transcript that helps regulate the GNAS complex locus, which encodes the alpha subunit of the stimulatory G protein. Defects in this gene are a cause of pseudohypoparathyroidism type Ib.[provided by RefSeq, Jun 2010]', '20', 'protein_coding', 39994954, 40043534, 51.49, 3, 'ENSG00000235590', ''),
(954, 'GNE', 'The protein encoded by this gene is a bifunctional enzyme that initiates and regulates the biosynthesis of N-acetylneuraminic acid (NeuAc), a precursor of sialic acids. It is a rate-limiting enzyme in the sialic acid biosynthetic pathway. Sialic acid modification of cell surface molecules is crucial for their function in many biologic processes, including cell adhesion and signal transduction. Differential sialylation of cell surface molecules is also implicated in the tumorigenicity and metastatic behavior of malignant cells. Mutations in this gene are associated with sialuria, autosomal recessive inclusion body myopathy, and Nonaka myopathy. Alternative splicing of this gene results in transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 152391546, 152433368, 37.97, 2, 'ENSG00000159921', 'Q9Y223'),
(955, 'GNGT1', 'This gene encodes the gamma subunit of transducin, a guanine nucleotide-binding protein (G protein) that is found in rod outer segments. Transducin, also known as GMPase, mediates the activation of a cyclic GTP-specific (guanosine monophosphate) phosphodiesterase by rhodopsin. [provided by RefSeq, Jul 2016]', '7', 'protein_coding', 64987945, 64997018, 53.10, 5, 'ENSG00000127928', 'P63211'),
(956, 'GNL3L', 'The protein encoded by this gene appears to be a nucleolar GTPase that is essential for ribosomal pre-rRNA processing and cell proliferation. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, May 2010]', 'X', 'protein_coding', 29981, 44855, 48.87, 1, 'ENSG00000130119', 'Q9NVN8'),
(957, 'GNRH2', 'This gene is a member of the gonadotropin-releasing hormone (GnRH) gene family. Proteins encoded by members of this gene family are proteolytically cleaved to form neuropeptides which, in part, regulate reproductive functions by stimulating the production and release of the gonadotropins follicle-stimulating hormone (FSH) and luteinizing hormone (LH). The human GNRH2 gene is predicted to encode a preproprotein from which a mature neuropeptide of 10 amino acids is cleaved. However, while the human genome retains the sequence for a functional GNRH2 decapeptide, translation of the human GNRH2 gene has not yet been demonstrated and the GNRH2 gene of chimpanzees, gorilla, and Sumatran orangutan have a premature stop at codon eight of the decapeptide sequence which suggests GNRH2 was a pseudogene in the hominid lineage. The GNRH2 gene is also believed to be a pseudogene in many other mammalian species such as mouse and cow. The receptor for this gene (GNRHR2) is predicted to be a pseudogene in human as well as many other mammalian species. The closely related GNRH1 and GNRHR1 genes are functional in human and other mammals and are generally functional in vertebrates. [provided by RefSeq, Mar 2019]', '20', 'protein_coding', 19875517, 19941820, 54.09, 21, 'ENSG00000125787', 'O43555'),
(958, 'GOLGA8IP', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus and membrane. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100363431, 100401089, 38.43, 3, 'ENSG00000281356', 'A6NC78'),
(959, 'GOLGA8IP', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus and membrane. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100363431, 100401089, 38.43, 3, 'ENSG00000290526', 'A6NC78'),
(960, 'GOLGA8IP', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus and membrane. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100363431, 100401089, 38.43, 3, 'ENSG00000277561', 'A6NC78'),
(961, 'GOLGA8IP', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus and membrane. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100363431, 100401089, 38.43, 3, 'ENSG00000291261', 'A6NC78'),
(962, 'GOLGA8N', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 31201486, 31203450, 61.88, 4, 'ENSG00000283589', 'F8WBI6'),
(963, 'GOLGA8N', 'Predicted to be involved in Golgi organization. Predicted to be located in Golgi apparatus. Predicted to be active in Golgi cis cisterna; Golgi cisterna membrane; and cis-Golgi network. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 31201486, 31203450, 61.88, 4, 'ENSG00000232653', 'F8WBI6'),
(964, 'GOLM1', 'The Golgi complex plays a key role in the sorting and modification of proteins exported from the endoplasmic reticulum. The protein encoded by this gene is a type II Golgi transmembrane protein. It processes proteins synthesized in the rough endoplasmic reticulum and assists in the transport of protein cargo through the Golgi apparatus. The expression of this gene has been observed to be upregulated in response to viral infection. Alternatively spliced transcript variants encoding the same protein have been described for this gene. [provided by RefSeq, Sep 2009]', '9', 'IG_V_gene', 106107972, 106108464, 56.59, 1, 'ENSG00000135052', 'Q8NBJ4'),
(965, 'GON7', 'Involved in tRNA threonylcarbamoyladenosine modification. Located in cytosol; nucleolus; and nucleoplasm. Part of EKC/KEOPS complex. Implicated in Galloway-Mowat syndrome. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 3012923, 3014505, 64.50, 1, 'ENSG00000170270', 'Q9BXV9'),
(966, 'GON7', 'Involved in tRNA threonylcarbamoyladenosine modification. Located in cytosol; nucleolus; and nucleoplasm. Part of EKC/KEOPS complex. Implicated in Galloway-Mowat syndrome. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 3012923, 3014505, 64.50, 1, 'ENSG00000275619', 'Q9BXV9'),
(967, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000204438', 'O95872'),
(968, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000233210', 'O95872'),
(969, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000206408', 'O95872'),
(970, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000232312', 'O95872'),
(971, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000236011', 'O95872');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(972, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000223932', 'O95872'),
(973, 'GPANK1', 'This gene is located in a cluster of HLA-B-associated transcripts, which is included in the human major histocompatability complex III region. This gene encodes a protein which is thought to play a role in immunity. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Nov 2010]', '6', 'protein_coding', 90469639, 90513397, 38.24, 1, 'ENSG00000228605', 'O95872'),
(974, 'GPHA2', 'GPHA2 is a cystine knot-forming polypeptide and a subunit of the dimeric glycoprotein hormone family (Hsu et al., 2002 [PubMed 12089349]).[supplied by OMIM, Mar 2008]', '11', 'protein_coding', 44673069, 44689779, 39.85, 5, 'ENSG00000149735', 'Q96T91'),
(975, 'GPI', 'This gene encodes a member of the glucose phosphate isomerase protein family. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. In the cytoplasm, the gene product functions as a glycolytic enzyme (glucose-6-phosphate isomerase) that interconverts glucose-6-phosphate and fructose-6-phosphate. Extracellularly, the encoded protein (also referred to as neuroleukin) functions as a neurotrophic factor that promotes survival of skeletal motor neurons and sensory neurons, and as a lymphokine that induces immunoglobulin secretion. The encoded protein is also referred to as autocrine motility factor based on an additional function as a tumor-secreted cytokine and angiogenic factor. Defects in this gene are the cause of nonspherocytic hemolytic anemia and a severe enzyme deficiency can be associated with hydrops fetalis, immediate neonatal death and neurological impairment. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2016]', '19', 'protein_coding', 111346600, 111353013, 41.27, 4, 'ENSG00000105220', 'P06744'),
(976, 'GPI', 'This gene encodes a member of the glucose phosphate isomerase protein family. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. In the cytoplasm, the gene product functions as a glycolytic enzyme (glucose-6-phosphate isomerase) that interconverts glucose-6-phosphate and fructose-6-phosphate. Extracellularly, the encoded protein (also referred to as neuroleukin) functions as a neurotrophic factor that promotes survival of skeletal motor neurons and sensory neurons, and as a lymphokine that induces immunoglobulin secretion. The encoded protein is also referred to as autocrine motility factor based on an additional function as a tumor-secreted cytokine and angiogenic factor. Defects in this gene are the cause of nonspherocytic hemolytic anemia and a severe enzyme deficiency can be associated with hydrops fetalis, immediate neonatal death and neurological impairment. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2016]', '19', 'protein_coding', 111346600, 111353013, 41.27, 4, 'ENSG00000282019', 'P06744'),
(977, 'GPKOW', 'This gene encodes a putative RNA-binding protein containing G-patch and KOW (Kyprides, Ouzounis, Woese) domains. The encoded protein interacts directly with protein kinase A and protein kinase X and is also found associated with the spliceosome. [provided by RefSeq, Aug 2013]', 'X', 'protein_coding', 48617798, 48733148, 38.06, 7, 'ENSG00000068394', 'Q92917'),
(978, 'GPKOW', 'This gene encodes a putative RNA-binding protein containing G-patch and KOW (Kyprides, Ouzounis, Woese) domains. The encoded protein interacts directly with protein kinase A and protein kinase X and is also found associated with the spliceosome. [provided by RefSeq, Aug 2013]', 'X', 'protein_coding', 48617798, 48733148, 38.06, 7, 'ENSG00000292204', 'Q92917'),
(979, 'GPLD1', 'Many proteins are tethered to the extracellular face of eukaryotic plasma membranes by a glycosylphosphatidylinositol (GPI) anchor. The GPI-anchor is a glycolipid found on many blood cells. The protein encoded by this gene is a GPI degrading enzyme. Glycosylphosphatidylinositol specific phospholipase D1 hydrolyzes the inositol phosphate linkage in proteins anchored by phosphatidylinositol glycans, thereby releasing the attached protein from the plasma membrane. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 186046314, 186110318, 42.69, 14, 'ENSG00000112293', 'P80108'),
(980, 'GPNMB', 'The protein encoded by this gene is a type I transmembrane glycoprotein which shows homology to the pMEL17 precursor, a melanocyte-specific protein. GPNMB shows expression in the lowly metastatic human melanoma cell lines and xenografts but does not show expression in the highly metastatic cell lines. GPNMB may be involved in growth delay and reduction of metastatic potential. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 79144688, 79175318, 36.53, 1, 'ENSG00000136235', 'Q14956'),
(981, 'GPR119', 'This gene encodes a member of the rhodopsin subfamily of G-protein-coupled receptors that is expressed in the pancreas and gastrointestinal tract. The encoded protein is activated by lipid amides including lysophosphatidylcholine and oleoylethanolamide and may be involved in glucose homeostasis. This protein is a potential drug target in the treatment of type 2 diabetes.[provided by RefSeq, Jan 2010]', 'X', 'protein_coding', 10305073, 10307397, 58.97, 8, 'ENSG00000147262', 'Q8TDV5'),
(982, 'GPR12', 'Predicted to enable G protein-coupled receptor activity. Predicted to be involved in adenylate cyclase-activating G protein-coupled receptor signaling pathway and regulation of metabolic process. Predicted to act upstream of or within G protein-coupled receptor signaling pathway and intracellular calcium ion homeostasis. Predicted to be located in membrane. Predicted to be active in cytoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'unprocessed_pseudogene', 145891, 146817, 36.68, 1, 'ENSG00000132975', 'P47775'),
(983, 'GPR158', 'Enables G protein-coupled glycine receptor activity and enzyme activator activity. Involved in G protein-coupled receptor signaling pathway; protein localization to plasma membrane; and regulation of G protein-coupled receptor signaling pathway. Is active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 56608584, 56609497, 54.16, 5, 'ENSG00000151025', 'Q5T848'),
(984, 'GPR15LG', 'Enables chemokine activity. Involved in several processes, including defense response to other organism; mast cell degranulation; and negative regulation of cell cycle G1/S phase transition. Located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 90383721, 90513402, 39.02, 15, 'ENSG00000188373', 'Q6UWK7'),
(985, 'GPR174', 'This gene encodes a protein belonging to the G protein-coupled receptor superfamily. These proteins are characterized by the presence of seven alpha-helical transmembrane domains, and they activate or interact with various endogenous or exogenous ligands, including neurotransmitters, hormones, and odorant and taste substances. This family member is classified as an orphan receptor because the cognate ligand has not been identified. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 155208636, 155217078, 39.59, 2, 'ENSG00000147138', 'Q9BXC1'),
(986, 'GPR182', 'Adrenomedullin is a potent vasodilator peptide that exerts major effects on cardiovascular function. This gene encodes a seven-transmembrane protein that belongs to the family 1 of G-protein coupled receptors. Studies of the rat counterpart suggest that the encoded protein may function as a receptor for adrenomedullin. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 75205659, 75235758, 48.14, 25, 'ENSG00000166856', 'O15218'),
(987, 'GPR183', 'This gene was identified by the up-regulation of its expression upon Epstein-Barr virus infection of primary B lymphocytes. This gene is predicted to encode a G protein-coupled receptor that is most closely related to the thrombin receptor. Expression of this gene was detected in B-lymphocyte cell lines and lymphoid tissues but not in T-lymphocyte cell lines or peripheral blood T lymphocytes. The function of this gene is unknown. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 23626706, 23638473, 51.65, 3, 'ENSG00000169508', 'P32249'),
(988, 'GPR20', 'Enables G protein-coupled receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Located in plasma membrane. Part of receptor complex. [provided by Alliance of Genome Resources, May 2025]', '8', 'rRNA', 8212572, 8212724, 57.52, 1, 'ENSG00000291465', 'Q99678'),
(989, 'GPR20', 'Enables G protein-coupled receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Located in plasma membrane. Part of receptor complex. [provided by Alliance of Genome Resources, May 2025]', '8', 'rRNA', 8212572, 8212724, 57.52, 1, 'ENSG00000204882', 'Q99678'),
(990, 'GPR20', 'Enables G protein-coupled receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Located in plasma membrane. Part of receptor complex. [provided by Alliance of Genome Resources, May 2025]', '8', 'rRNA', 8212572, 8212724, 57.52, 1, 'ENSG00000275181', 'Q99678'),
(991, 'GPR21', 'This gene encodes a member of the G-protein-coupled receptor 1 family. G-protein coupled receptors are membrane proteins which activate signaling cascades as a response to extracellular stress. The encoded protein activates a Gq signal transduction pathway which mobilizes calcium. [provided by RefSeq, Nov 2012]', '9', 'protein_coding', 191495, 196334, 61.59, 1, 'ENSG00000188394', 'Q99679'),
(992, 'GPR22', 'This gene is a member of the G-protein coupled receptor 1 family and encodes a multi-pass membrane protein. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 606712, 613031, 55.70, 5, 'ENSG00000283812', 'Q99680'),
(993, 'GPR22', 'This gene is a member of the G-protein coupled receptor 1 family and encodes a multi-pass membrane protein. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 606712, 613031, 55.70, 5, 'ENSG00000172209', 'Q99680'),
(994, 'GPR32P1', 'Predicted to enable N-formyl peptide receptor activity and complement receptor activity. Predicted to be involved in several processes, including complement receptor mediated signaling pathway; phospholipase C-activating G protein-coupled receptor signaling pathway; and positive regulation of cytosolic calcium ion concentration. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 148923639, 149063163, 42.51, 21, 'ENSG00000269565', 'Q8NGA4'),
(995, 'GPR50', 'This gene product belongs to the G-protein coupled receptor 1 family. Even though this protein shares similarity with the melatonin receptors, it does not bind melatonin, however, it inhibits melatonin receptor 1A function through heterodimerization. Polymorphic variants of this gene have been associated with bipolar affective disorder in women. [provided by RefSeq, Jan 2010]', 'X', 'protein_coding', 297188, 305221, 61.25, 5, 'ENSG00000102195', 'Q13585'),
(996, 'GPR68', 'The protein encoded by this gene is a G protein-coupled receptor for sphingosylphosphorylcholine. The encoded protein is a proton-sensing receptor, inactive at pH 7.8 but active at pH 6.8. Mutations in this gene are a cause of amelogenesis imperfecta. [provided by RefSeq, Feb 2017]', '14', 'protein_coding', 52989340, 53094779, 38.01, 6, 'ENSG00000119714', 'Q15743'),
(997, 'GPRASP2', 'The protein encoded by this gene is a member of a family that regulates the activity of G protein-coupled receptors (GPCRs). The encoded protein has been shown to be capable of interacting with several GPCRs, including the M1 muscarinic acetylcholine receptor and the calcitonin receptor. Several transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, May 2010]', 'X', 'miRNA', 61815161, 61815240, 80.00, 1, 'ENSG00000158301', 'Q96D09'),
(998, 'GPRASP3', 'This locus represents naturally occurring readthrough transcription among the adjacent armadillo repeat containing, X-linked 5 (ARMCX5), G protein-coupled receptor associated sorting proteins 1 and 2 (GPRASP1 and GPRASP2), basic helix-loop-helix family member b9 (BHLHB9), and long intergenic non-protein coding RNA 630 (LINC00630) genes on chromosome X. Transcripts may make use of multiple alternative promoters and polyadenylation signals in this region. Readthrough transcripts may produce proteins identical to the proteins encoded by GPRASP2 or BHLHB9. [provided by RefSeq, Apr 2017]', 'X', 'protein_coding', 38472816, 38578859, 39.37, 9, 'ENSG00000198908', 'Q6PI77'),
(999, 'GPS2', 'This gene encodes a protein involved in G protein-mitogen-activated protein kinase (MAPK) signaling cascades. When overexpressed in mammalian cells, this gene could potently suppress a RAS- and MAPK-mediated signal and interfere with JNK activity, suggesting that the function of this gene may be signal repression. The encoded protein is an integral subunit of the NCOR1-HDAC3 (nuclear receptor corepressor 1-histone deacetylase 3) complex, and it was shown that the complex inhibits JNK activation through this subunit and thus could potentially provide an alternative mechanism for hormone-mediated antagonism of AP1 (activator protein 1) function. [provided by RefSeq, Jul 2008]', '17', 'TR_J_gene', 22490491, 22490553, 38.10, 1, 'ENSG00000288325', 'Q13227'),
(1000, 'GPS2', 'This gene encodes a protein involved in G protein-mitogen-activated protein kinase (MAPK) signaling cascades. When overexpressed in mammalian cells, this gene could potently suppress a RAS- and MAPK-mediated signal and interfere with JNK activity, suggesting that the function of this gene may be signal repression. The encoded protein is an integral subunit of the NCOR1-HDAC3 (nuclear receptor corepressor 1-histone deacetylase 3) complex, and it was shown that the complex inhibits JNK activation through this subunit and thus could potentially provide an alternative mechanism for hormone-mediated antagonism of AP1 (activator protein 1) function. [provided by RefSeq, Jul 2008]', '17', 'TR_J_gene', 22490491, 22490553, 38.10, 1, 'ENSG00000132522', 'Q13227'),
(1001, 'GPX7', 'Enables catalase activity. Predicted to be involved in cellular response to oxidative stress. Located in endoplasmic reticulum. Implicated in Barrett\'s adenocarcinoma and Barrett\'s esophagus. [provided by Alliance of Genome Resources, May 2025]', '1', 'miRNA', 59137758, 59137872, 39.13, 1, 'ENSG00000116157', 'Q96SL4'),
(1002, 'GPX8', 'Enables peroxidase activity. Predicted to be involved in cellular response to oxidative stress. Predicted to be located in endoplasmic reticulum lumen. [provided by Alliance of Genome Resources, May 2025]', '5', 'TR_J_gene', 22493403, 22493465, 52.38, 1, 'ENSG00000164294', 'Q8TED1'),
(1003, 'GREM1', 'This gene encodes a member of the BMP (bone morphogenic protein) antagonist family. Like BMPs, BMP antagonists contain cystine knots and typically form homo- and heterodimers. The CAN (cerberus and dan) subfamily of BMP antagonists, to which this gene belongs, is characterized by a C-terminal cystine knot with an eight-membered ring. The antagonistic effect of the secreted glycosylated protein encoded by this gene is likely due to its direct binding to BMP proteins. As an antagonist of BMP, this gene may play a role in regulating organogenesis, body patterning, and tissue differentiation. In mouse, this protein has been shown to relay the sonic hedgehog (SHH) signal from the polarizing region to the apical ectodermal ridge during limb bud outgrowth. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010]', '15', 'protein_coding', 101568353, 101686028, 38.13, 24, 'ENSG00000282046', 'O60565'),
(1004, 'GREM1', 'This gene encodes a member of the BMP (bone morphogenic protein) antagonist family. Like BMPs, BMP antagonists contain cystine knots and typically form homo- and heterodimers. The CAN (cerberus and dan) subfamily of BMP antagonists, to which this gene belongs, is characterized by a C-terminal cystine knot with an eight-membered ring. The antagonistic effect of the secreted glycosylated protein encoded by this gene is likely due to its direct binding to BMP proteins. As an antagonist of BMP, this gene may play a role in regulating organogenesis, body patterning, and tissue differentiation. In mouse, this protein has been shown to relay the sonic hedgehog (SHH) signal from the polarizing region to the apical ectodermal ridge during limb bud outgrowth. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010]', '15', 'protein_coding', 101568353, 101686028, 38.13, 24, 'ENSG00000276886', 'O60565'),
(1005, 'GREM1', 'This gene encodes a member of the BMP (bone morphogenic protein) antagonist family. Like BMPs, BMP antagonists contain cystine knots and typically form homo- and heterodimers. The CAN (cerberus and dan) subfamily of BMP antagonists, to which this gene belongs, is characterized by a C-terminal cystine knot with an eight-membered ring. The antagonistic effect of the secreted glycosylated protein encoded by this gene is likely due to its direct binding to BMP proteins. As an antagonist of BMP, this gene may play a role in regulating organogenesis, body patterning, and tissue differentiation. In mouse, this protein has been shown to relay the sonic hedgehog (SHH) signal from the polarizing region to the apical ectodermal ridge during limb bud outgrowth. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010]', '15', 'protein_coding', 101568353, 101686028, 38.13, 24, 'ENSG00000166923', 'O60565'),
(1006, 'GRHPR', 'This gene encodes an enzyme with hydroxypyruvate reductase, glyoxylate reductase, and D-glycerate dehydrogenase enzymatic activities. The enzyme has widespread tissue expression and has a role in metabolism. Type II hyperoxaluria is caused by mutations in this gene. [provided by RefSeq, Jul 2008]', '9', 'unprocessed_pseudogene', 55754977, 55755957, 44.95, 1, 'ENSG00000137106', 'Q9UBQ7'),
(1007, 'GRIK1', 'Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. This gene product belongs to the kainate family of glutamate receptors, which are composed of four subunits and function as ligand-activated ion channels. The subunit encoded by this gene is subject to RNA editing (CAG->CGG; Q->R) within the second transmembrane domain, which is thought to alter the properties of ion flow. Alternative splicing, resulting in transcript variants encoding different isoforms, has been noted for this gene. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 233799, 242013, 47.49, 5, 'ENSG00000171189', 'P39086'),
(1008, 'GSC2', 'Goosecoidlike (GSCL), a homeodomain-containing gene, resides in the critical region for VCFS/DGS on 22q11. Velocardiofacial syndrome (VCFS) is a developmental disorder characterized by conotruncal heart defects, craniofacial anomalies, and learning disabilities. VCFS is phenotypically related to DiGeorge syndrome (DGS) and both syndromes are associated with hemizygous 22q11 deletions. Because many of the tissues and structures affected in VCFS/DGS derive from the pharyngeal arches of the developing embryo, it is believed that haploinsufficiency of a gene involved in embryonic development may be responsible for its etiology. The gene is expressed in a limited number of adult tissues, as well as in early human development. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 172325000, 172454525, 48.30, 5, 'ENSG00000063515', 'O15499'),
(1009, 'GSDME', 'Hearing impairment is a heterogeneous condition with over 40 loci described. The protein encoded by this gene is expressed in fetal cochlea, however, its function is not known. Nonsyndromic hearing impairment is associated with a mutation in this gene. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 141838316, 141897832, 39.06, 5, 'ENSG00000105928', 'O60443'),
(1010, 'GSKIP', 'This gene encodes a protein that is involved as a negative regulator of GSK3-beta in the Wnt signaling pathway. The encoded protein may play a role in the retinoic acid signaling pathway by regulating the functional interactions between GSK3-beta, beta-catenin and cyclin D1, and it regulates the beta-catenin/N-cadherin pool. The encoded protein contains a GSK3-beta interacting domain (GID) in its C-terminus, which is similar to the GID of Axin. The protein also contains an evolutionarily conserved RII-binding domain, which facilitates binding with protein kinase-A and GSK3-beta, enabling its role as an A-kinase anchoring protein. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Dec 2012]', '14', 'protein_coding', 90524564, 90816479, 46.46, 15, 'ENSG00000100744', 'Q9P0R6'),
(1011, 'GSTA1', 'This gene encodes a member of a family of enzymes that function to add glutathione to target electrophilic compounds, including carcinogens, therapeutic drugs, environmental toxins, and products of oxidative stress. This action is an important step in detoxification of these compounds. This subfamily of enzymes has a particular role in protecting cells from reactive oxygen species and the products of peroxidation. Polymorphisms in this gene influence the ability of individuals to metabolize different drugs. This gene is located in a cluster of similar genes and pseudogenes on chromosome 6. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 32109714, 32167258, 47.43, 7, 'ENSG00000243955', 'P08263'),
(1012, 'GSTCD', 'Predicted to act upstream of or within respiratory system development and response to lipopolysaccharide. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 99356131, 99356228, 52.04, 1, 'ENSG00000138780', 'Q8NEC7'),
(1013, 'GSTT2', 'The protein encoded by this gene, glutathione S-transferase (GST) theta 2 (GSTT2), is a member of a superfamily of proteins that catalyze the conjugation of reduced glutathione to a variety of electrophilic and hydrophobic compounds. Human GSTs can be divided into five main classes: alpha, mu, pi, theta, and zeta. The theta class includes GSTT1, GSTT2, and GSTT2B. GSTT2 and GSTT2B are nearly identical to each other, and share 55% amino acid identity with GSTT1. All three genes may play a role in human carcinogenesis. The GSTT2 gene is a pseudogene in some populations. [provided by RefSeq, Sep 2015]', '22', 'protein_coding', 37010859, 37020183, 57.04, 4, 'ENSG00000099984', 'P0CG30'),
(1014, 'GSTT2', 'The protein encoded by this gene, glutathione S-transferase (GST) theta 2 (GSTT2), is a member of a superfamily of proteins that catalyze the conjugation of reduced glutathione to a variety of electrophilic and hydrophobic compounds. Human GSTs can be divided into five main classes: alpha, mu, pi, theta, and zeta. The theta class includes GSTT1, GSTT2, and GSTT2B. GSTT2 and GSTT2B are nearly identical to each other, and share 55% amino acid identity with GSTT1. All three genes may play a role in human carcinogenesis. The GSTT2 gene is a pseudogene in some populations. [provided by RefSeq, Sep 2015]', '22', 'protein_coding', 37010859, 37020183, 57.04, 4, 'ENSG00000277897', 'P0CG30'),
(1015, 'GSTT3P', 'Predicted to enable glutathione transferase activity. Predicted to act upstream of or within glutathione metabolic process. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 119177205, 119349761, 37.60, 1, 'ENSG00000224205', ''),
(1016, 'GSTT3P', 'Predicted to enable glutathione transferase activity. Predicted to act upstream of or within glutathione metabolic process. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 119177205, 119349761, 37.60, 1, 'ENSG00000281276', ''),
(1017, 'GSX1', 'Enables sequence-specific double-stranded DNA binding activity. Acts upstream of or within positive regulation of transcription by RNA polymerase II. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '13', 'miRNA', 62564912, 62565013, 50.98, 1, 'ENSG00000169840', 'Q9H4S2'),
(1018, 'GTF2A1', 'Accurate transcription initiation on TATA-containing class II genes involves the ordered assembly of RNA polymerase II (POLR2A; MIM 180660) and several general initiation factors (summarized by DeJong and Roeder, 1993 [PubMed 8224848]). One of these factors is TFIIA, which when purified from HeLa extracts consists of 35-, 19-, and 12-kD subunits.[supplied by OMIM, Jul 2010]', '14', 'unprocessed_pseudogene', 50758382, 50759192, 59.19, 1, 'ENSG00000165417', 'P52655'),
(1019, 'GTF2A1L', 'The assembly and stability of the RNA polymerase II transcription pre-initiation complex on a eukaryotic core promoter involve the effects of transcription factor IIA (TFIIA) on the interaction between TATA-binding protein (TBP) and DNA. This gene encodes a germ cell-specific counterpart of the large (alpha/beta) subunit of general transcription factor TFIIA that is able to stabilize the binding of TBP to DNA and may be uniquely important to testis biology. Alternative splicing for this locus has been observed and two variants, encoding distinct isoforms, have been identified. Co-transcription of this gene and the neighboring upstream gene generates a rare transcript (SALF), which encodes a fusion protein comprised of sequence sharing identity with each individual gene product. [provided by RefSeq, Mar 2014]', '2', 'miRNA', 110055329, 110055426, 50.00, 1, 'ENSG00000242441', 'Q9UNN4'),
(1020, 'GTF2F2', 'Predicted to enable RNA polymerase II general transcription initiation factor activity. Involved in positive regulation of transcription by RNA polymerase II; transcription elongation by RNA polymerase II; and transcription initiation at RNA polymerase II promoter. Located in microtubule cytoskeleton and nucleoplasm. Part of transcription factor TFIIF complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 58579172, 58624639, 36.65, 2, 'ENSG00000188342', 'P13984'),
(1021, 'GTF2H2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is within the telomeric copy of the duplication. Deletion of this gene sometimes accompanies deletion of the neighboring SMN1 gene in spinal muscular atrophy (SMA) patients but it is unclear if deletion of this gene contributes to the SMA phenotype. This gene encodes the 44 kDa subunit of RNA polymerase II transcription initiation factor IIH which is involved in basal transcription and nucleotide excision repair. Transcript variants for this gene have been described, but their full length nature has not been determined. A second copy of this gene within the centromeric copy of the duplication has been described in the literature. It is reported to be different by either two or four base pairs; however, no sequence data is currently available for the centromeric copy of the gene. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 67884656, 67918390, 40.88, 6, 'ENSG00000276910', 'Q13888'),
(1022, 'GTF2H2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is within the telomeric copy of the duplication. Deletion of this gene sometimes accompanies deletion of the neighboring SMN1 gene in spinal muscular atrophy (SMA) patients but it is unclear if deletion of this gene contributes to the SMA phenotype. This gene encodes the 44 kDa subunit of RNA polymerase II transcription initiation factor IIH which is involved in basal transcription and nucleotide excision repair. Transcript variants for this gene have been described, but their full length nature has not been determined. A second copy of this gene within the centromeric copy of the duplication has been described in the literature. It is reported to be different by either two or four base pairs; however, no sequence data is currently available for the centromeric copy of the gene. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 67884656, 67918390, 40.88, 6, 'ENSG00000275045', 'Q13888'),
(1023, 'GTF2H2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is within the telomeric copy of the duplication. Deletion of this gene sometimes accompanies deletion of the neighboring SMN1 gene in spinal muscular atrophy (SMA) patients but it is unclear if deletion of this gene contributes to the SMA phenotype. This gene encodes the 44 kDa subunit of RNA polymerase II transcription initiation factor IIH which is involved in basal transcription and nucleotide excision repair. Transcript variants for this gene have been described, but their full length nature has not been determined. A second copy of this gene within the centromeric copy of the duplication has been described in the literature. It is reported to be different by either two or four base pairs; however, no sequence data is currently available for the centromeric copy of the gene. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 67884656, 67918390, 40.88, 6, 'ENSG00000145736', 'Q13888'),
(1024, 'GTF2H2C_2', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in G protein-coupled receptor internalization and transcription initiation at RNA polymerase II promoter. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 36196744, 36232790, 47.04, 12, 'ENSG00000274675', 'Q6P1K8'),
(1025, 'GTF2H3', 'This gene encodes a member of the TFB4 family. The encoded protein is a subunit of the core-TFIIH basal transcription factor and localizes to the nucleus. The encoded protein is involved in RNA transcription by RNA polymerase II and nucleotide excision repair and associates with the Cdk-activating kinase complex. Alternative splicing results in multiple transcript variants. A related pseudogene has been identified on chromosome 14. [provided by RefSeq, Dec 2012]', '12', 'protein_coding', 148826611, 148828623, 50.57, 1, 'ENSG00000111358', 'Q13889'),
(1026, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000213780', 'Q92759'),
(1027, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000226384', 'Q92759'),
(1028, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000234370', 'Q92759'),
(1029, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000236895', 'Q92759'),
(1030, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000221974', 'Q92759'),
(1031, 'GTF2H4', 'Enables RNA polymerase II general transcription initiation factor activity. Involved in transcription by RNA polymerase II. Located in nuclear speck. Part of core TFIIH complex portion of holo TFIIH complex and transcription factor TFIID complex. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 64225941, 64230686, 59.06, 12, 'ENSG00000233149', 'Q92759'),
(1032, 'GTPBP1', 'This gene is upregulated by interferon-gamma and encodes a protein that is a member of the AGP11/GTPBP1 family of GTP-binding proteins. A structurally similar protein has been found in mouse, where disruption of the gene for that protein had no observable phenotype. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 154733378, 154834244, 37.63, 6, 'ENSG00000100226', 'O00178'),
(1033, 'GTSCR1', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 1074875, 1110511, 64.12, 3, 'ENSG00000263417', 'Q86UQ5'),
(1034, 'GTSF1L', 'Predicted to enable zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 517835, 533345, 43.50, 8, 'ENSG00000124196', 'Q9H1H1'),
(1035, 'GUCY1B1', 'This gene encodes the beta subunit of the soluble guanylate cyclase (sGC), which catalyzes the conversion of GTP (guanosine triphosphate) to cGMP (cyclic guanosine monophosphate). The encoded protein contains an HNOX domain, which serves as a receptor for ligands such as nitric oxide, oxygen and nitrovasodilator drugs. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]', '4', 'protein_coding', 2033, 18762, 47.02, 2, 'ENSG00000061918', 'Q02153'),
(1036, 'GUCY2D', 'This gene encodes a retina-specific guanylate cyclase, which is a member of the membrane guanylyl cyclase family. Like other membrane guanylyl cyclases, this enzyme has a hydrophobic amino-terminal signal sequence followed by a large extracellular domain, a single membrane spanning domain, a kinase homology domain, and a guanylyl cyclase catalytic domain. In contrast to other membrane guanylyl cyclases, this enzyme is not activated by natriuretic peptides. Mutations in this gene result in Leber congenital amaurosis and cone-rod dystrophy-6 diseases. [provided by RefSeq, Dec 2008]', '17', 'unprocessed_pseudogene', 124238420, 124239486, 41.14, 1, 'ENSG00000132518', 'Q02846'),
(1037, 'GUCY2F', 'The protein encoded by this gene is a guanylyl cyclase found predominantly in photoreceptors in the retina. The encoded protein is thought to be involved in resynthesis of cGMP after light activation of the visual signal transduction cascade, allowing a return to the dark state. This protein is a single-pass type I membrane protein. Defects in this gene may be a cause of X-linked retinitis pigmentosa. [provided by RefSeq, Dec 2008]', 'X', 'protein_coding', 321618, 343777, 43.29, 7, 'ENSG00000101890', 'P51841'),
(1038, 'GUSBP11', 'This transcribed pseudogene is similar to two functional genes. The 5\' portion of the pseudogene is related to glucuronidase, beta, and the 3\' portion is related to immunoglobulin lambda-like polypeptide 1. [provided by RefSeq, Jul 2011]', '22', 'protein_coding', 157146508, 157160760, 37.30, 26, 'ENSG00000272578', 'Q6P575'),
(1039, 'GUSBP11', 'This transcribed pseudogene is similar to two functional genes. The 5\' portion of the pseudogene is related to glucuronidase, beta, and the 3\' portion is related to immunoglobulin lambda-like polypeptide 1. [provided by RefSeq, Jul 2011]', '22', 'protein_coding', 157146508, 157160760, 37.30, 26, 'ENSG00000228315', 'Q6P575'),
(1040, 'GYS2', 'The protein encoded by this gene, liver glycogen synthase, catalyzes the rate-limiting step in the synthesis of glycogen - the transfer of a glucose molecule from UDP-glucose to a terminal branch of the glycogen molecule. Mutations in this gene cause glycogen storage disease type 0 (GSD-0) - a rare type of early childhood fasting hypoglycemia with decreased liver glycogen content. [provided by RefSeq, Dec 2009]', '12', 'protein_coding', 20560448, 20651130, 52.47, 9, 'ENSG00000111713', 'P54840'),
(1041, 'GZF1', 'Enables DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in negative regulation of transcription by RNA polymerase II. Located in nucleolus and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 96278261, 96400480, 37.80, 12, 'ENSG00000125812', 'Q9H116'),
(1042, 'GZMB', 'This gene encodes a member of the granzyme subfamily of proteins, part of the peptidase S1 family of serine proteases. The encoded preproprotein is secreted by natural killer (NK) cells and cytotoxic T lymphocytes (CTLs) and proteolytically processed to generate the active protease, which induces target cell apoptosis. This protein also processes cytokines and degrades extracellular matrix proteins, and these roles are implicated in chronic inflammation and wound healing. Expression of this gene may be elevated in human patients with cardiac fibrosis. [provided by RefSeq, Sep 2016]', '14', 'protein_coding', 132692628, 132737638, 36.68, 8, 'ENSG00000100453', 'P10144'),
(1043, 'GZMH', 'This gene encodes a member of the peptidase S1 family of serine proteases. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate a chymotrypsin-like protease. This protein is reported to be constitutively expressed in the NK (natural killer) cells of the immune system and may play a role in the cytotoxic arm of the innate immune response by inducing target cell death and by directly cleaving substrates in pathogen-infected cells. This gene is present in a gene cluster with another member of the granzyme subfamily on chromosome 14. [provided by RefSeq, Nov 2015]', '14', 'protein_coding', 729759, 731673, 38.22, 1, 'ENSG00000100450', 'P20718'),
(1044, 'H1-1', 'Histones are basic nuclear proteins responsible for nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H1 family. Transcripts from this gene lack polyA tails but instead contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6. [provided by RefSeq, Aug 2015]', '6', 'unprocessed_pseudogene', 133430394, 133431318, 59.14, 1, 'ENSG00000124610', 'Q02539'),
(1045, 'H2AC25', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H2A family. Transcripts from this gene contain a palindromic termination element. [provided by RefSeq, Aug 2015]', '1', 'protein_coding', 21534879, 21549326, 36.98, 2, 'ENSG00000284841', 'Q7L7L0'),
(1046, 'H2AC25', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H2A family. Transcripts from this gene contain a palindromic termination element. [provided by RefSeq, Aug 2015]', '1', 'protein_coding', 21534879, 21549326, 36.98, 2, 'ENSG00000181218', 'Q7L7L0'),
(1047, 'H2AC4', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. This structure consists of approximately 146 bp of DNA wrapped around a nucleosome, an octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H2A family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3. [provided by RefSeq, Aug 2015]', '6', 'miRNA', 134540185, 134540262, 28.21, 1, 'ENSG00000278463', 'P04908'),
(1048, 'H2AL1Q', 'Predicted to be a structural constituent of chromatin. Predicted to be involved in heterochromatin formation. Predicted to act upstream of or within chromosome organization. Predicted to be located in pericentric heterochromatin. Predicted to be part of nucleosome. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 30658431, 30670810, 59.44, 9, 'ENSG00000249467', 'A0A8Q3WKH5'),
(1049, 'H2AZ2', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene encodes a replication-independent histone that is a member of the histone H2A family. Several transcript variants encoding different isoforms, have been identified for this gene. [provided by RefSeq, Oct 2015]', '7', 'protein_coding', 65912457, 67763637, 36.26, 16, 'ENSG00000105968', 'Q71UI9'),
(1050, 'H2BC27P', 'Histones are basic nuclear proteins responsible for nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is a histone pseudogene found on chromosome 1. [provided by RefSeq, Oct 2015]', '1', 'protein_coding', 18842, 180082, 40.68, 14, 'ENSG00000285234', ''),
(1051, 'H2BC27P', 'Histones are basic nuclear proteins responsible for nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is a histone pseudogene found on chromosome 1. [provided by RefSeq, Oct 2015]', '1', 'protein_coding', 18842, 180082, 40.68, 14, 'ENSG00000181201', ''),
(1052, 'H2BC6', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. The protein has antibacterial and antifungal antimicrobial activity. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H2B family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3. [provided by RefSeq, Aug 2015]', '6', 'protein_coding', 134565838, 134764322, 42.07, 4, 'ENSG00000274290', 'P62807'),
(1053, 'H3C1', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. This structure consists of approximately 146 bp of DNA wrapped around a nucleosome, an octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H3 family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3. [provided by RefSeq, Aug 2015]', '6', 'protein_coding', 44910060, 44959035, 47.22, 19, 'ENSG00000275714', 'P68431'),
(1054, 'H3C3', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. This structure consists of approximately 146 bp of DNA wrapped around a nucleosome, an octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H3 family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3. [provided by RefSeq, Aug 2015]', '6', 'miRNA', 134541341, 134541437, 47.42, 1, 'ENSG00000287080', 'P68431'),
(1055, 'H4C4', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H4 family. Transcripts from this gene lack polyA tails but instead contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6. [provided by RefSeq, Aug 2015]', '6', 'protein_coding', 134769566, 134797232, 40.17, 8, 'ENSG00000277157', 'P62805'),
(1056, 'HACD4', 'Enables enzyme binding activity and very-long-chain (3R)-3-hydroxyacyl-CoA dehydratase activity. Involved in fatty acid elongation and very long-chain fatty acid biosynthetic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 8297, 25091, 47.09, 2, 'ENSG00000188921', 'Q5VWC8');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1057, 'HAND1', 'The protein encoded by this gene belongs to the basic helix-loop-helix family of transcription factors. This gene product is one of two closely related family members, the HAND proteins, which are asymmetrically expressed in the developing ventricular chambers and play an essential role in cardiac morphogenesis. Working in a complementary fashion, they function in the formation of the right ventricle and aortic arch arteries, implicating them as mediators of congenital heart disease. In addition, it has been suggested that this transcription factor may be required for early trophoblast differentiation. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 85729233, 86156943, 36.41, 5, 'ENSG00000113196', 'O96004'),
(1058, 'HAPLN2', 'Predicted to enable hyaluronic acid binding activity. Predicted to be involved in central nervous system development and skeletal system development. Predicted to act upstream of or within establishment of blood-nerve barrier and extracellular matrix assembly. Predicted to be located in extracellular matrix. Predicted to be active in extracellular space; perineuronal net; and synapse. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 9514358, 9530524, 42.77, 2, 'ENSG00000291756', 'Q9GZV7'),
(1059, 'HAPLN2', 'Predicted to enable hyaluronic acid binding activity. Predicted to be involved in central nervous system development and skeletal system development. Predicted to act upstream of or within establishment of blood-nerve barrier and extracellular matrix assembly. Predicted to be located in extracellular matrix. Predicted to be active in extracellular space; perineuronal net; and synapse. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 9514358, 9530524, 42.77, 2, 'ENSG00000132702', 'Q9GZV7'),
(1060, 'HAT1', 'The protein encoded by this gene is a type B histone acetyltransferase (HAT) that is involved in the rapid acetylation of newly synthesized cytoplasmic histones, which are in turn imported into the nucleus for de novo deposition onto nascent DNA chains. Histone acetylation, particularly of histone H4, plays an important role in replication-dependent chromatin assembly. Specifically, this HAT can acetylate soluble but not nucleosomal histone H4 at lysines 5 and 12, and to a lesser degree, histone H2A at lysine 5. Alternatively spliced transcript variants have been identified for this gene. [provided by RefSeq, Jun 2009]', '2', 'protein_coding', 31718648, 31721746, 52.95, 2, 'ENSG00000292096', 'O14929'),
(1061, 'HAT1', 'The protein encoded by this gene is a type B histone acetyltransferase (HAT) that is involved in the rapid acetylation of newly synthesized cytoplasmic histones, which are in turn imported into the nucleus for de novo deposition onto nascent DNA chains. Histone acetylation, particularly of histone H4, plays an important role in replication-dependent chromatin assembly. Specifically, this HAT can acetylate soluble but not nucleosomal histone H4 at lysines 5 and 12, and to a lesser degree, histone H2A at lysine 5. Alternatively spliced transcript variants have been identified for this gene. [provided by RefSeq, Jun 2009]', '2', 'protein_coding', 31718648, 31721746, 52.95, 2, 'ENSG00000128708', 'O14929'),
(1062, 'HAUS4', 'This gene encodes a subunit of the centrosome complex termed the human augmin complex. The encoded protein localizes to the spindle microtubules and may play a role in mitotic spindle assembly and maintenance of centrosome integrity during cell division. Alternate splicing results in multiple transcript variants. A pseudogene of this gene is found on chromosome 1. [provided by RefSeq, Oct 2009]', '14', 'protein_coding', 183248237, 183418380, 44.39, 4, 'ENSG00000092036', 'Q9H6D7'),
(1063, 'HBA1', 'The human alpha globin gene cluster located on chromosome 16 spans about 30 kb and includes seven loci: 5\'- zeta - pseudozeta - mu - pseudoalpha-1 - alpha-2 - alpha-1 - theta - 3\'. The alpha-2 (HBA2) and alpha-1 (HBA1) coding sequences are identical. These genes differ slightly over the 5\' untranslated regions and the introns, but they differ significantly over the 3\' untranslated regions. Two alpha chains plus two beta chains constitute HbA, which in normal adult life comprises about 97% of the total hemoglobin; alpha chains combine with delta chains to constitute HbA-2, which with HbF (fetal hemoglobin) makes up the remaining 3% of adult hemoglobin. Alpha thalassemias result from deletions of each of the alpha genes as well as deletions of both HBA2 and HBA1; some nondeletion alpha thalassemias have also been reported. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 2111786, 2125658, 48.94, 2, 'ENSG00000206172', 'P69905'),
(1064, 'HBB', 'The alpha (HBA) and beta (HBB) loci determine the structure of the 2 types of polypeptide chains in adult hemoglobin, Hb A. The normal adult hemoglobin tetramer consists of two alpha chains and two beta chains. Mutant beta globin causes sickle cell anemia. Absence of beta chain causes beta-zero-thalassemia. Reduced amounts of detectable beta globin causes beta-plus-thalassemia. The order of the genes in the beta-globin cluster is 5\'-epsilon -- gamma-G -- gamma-A -- delta -- beta--3\'. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 12661833, 12702777, 40.84, 12, 'ENSG00000244734', 'P68871'),
(1065, 'HBE1', 'The epsilon globin gene (HBE) is normally expressed in the embryonic yolk sac: two epsilon chains together with two zeta chains (an alpha-like globin) constitute the embryonic hemoglobin Hb Gower I; two epsilon chains together with two alpha chains form the embryonic Hb Gower II. Both of these embryonic hemoglobins are normally supplanted by fetal, and later, adult hemoglobin. The five beta-like globin genes are found within a 45 kb cluster on chromosome 11 in the following order:  5\'-epsilon - G-gamma - A-gamma - delta - beta-3\' [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 120815496, 120842951, 40.53, 8, 'ENSG00000213931', 'P02100'),
(1066, 'HBM', 'This gene encodes a transmembrane low-density lipoprotein receptor that binds and internalizes ligands in the process of receptor-mediated endocytosis. This protein also acts as a co-receptor with Frizzled protein family members for transducing signals by Wnt proteins and was originally cloned on the basis of its association with type 1 diabetes mellitus in humans. This protein plays a key role in skeletal homeostasis and many bone density related diseases are caused by mutations in this gene. Mutations in this gene also cause familial exudative vitreoretinopathy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]', '11', 'protein_coding', 20457681, 20461465, 50.54, 7, 'ENSG00000206177', 'Q6B0K9'),
(1067, 'HBP1', 'Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in negative regulation of lipid transport; negative regulation of reactive oxygen species biosynthetic process; and negative regulation of transcription by RNA polymerase II. Located in nuclear speck. Biomarker of osteoarthritis. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 93096217, 93113283, 49.97, 9, 'ENSG00000283847', 'O60381'),
(1068, 'HBP1', 'Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in negative regulation of lipid transport; negative regulation of reactive oxygen species biosynthetic process; and negative regulation of transcription by RNA polymerase II. Located in nuclear speck. Biomarker of osteoarthritis. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 93096217, 93113283, 49.97, 9, 'ENSG00000105856', 'O60381'),
(1069, 'HBS1L', 'This gene encodes a member of the GTP-binding elongation factor family. It is expressed in multiple tissues with the highest expression in heart and skeletal muscle. The intergenic region of this gene and the MYB gene has been identified to be a quantitative trait locus (QTL) controlling fetal hemoglobin level, and this region influnces erythrocyte, platelet, and monocyte counts as well as erythrocyte volume and hemoglobin content. DNA polymorphisms at this region associate with fetal hemoglobin levels and pain crises in sickle cell disease. A single nucleotide polymorphism in exon 1 of this gene is significantly associated with severity in beta-thalassemia/Hemoglobin E. Multiple alternatively spliced transcript variants encoding different protein isoforms have been found for this gene. [provided by RefSeq, May 2009]', '6', 'protein_coding', 45406057, 45410610, 55.82, 9, 'ENSG00000112339', 'Q9Y450'),
(1070, 'HBZ', 'Zeta-globin is an alpha-like hemoglobin. The zeta-globin polypeptide is synthesized in the yolk sac of the early embryo, while alpha-globin is produced throughout fetal and adult life. The zeta-globin gene is a member of the human alpha-globin gene cluster that includes five functional genes and two pseudogenes. The order of genes is: 5\' - zeta - pseudozeta - mu - pseudoalpha-1 - alpha-2 -alpha-1 - theta1 - 3\'. [provided by RefSeq, Nov 2009]', '16', 'protein_coding', 142157527, 142225592, 38.78, 12, 'ENSG00000130656', 'P02008'),
(1071, 'HCCS', 'The protein encoded by this gene is an enzyme that covalently links a heme group to the apoprotein of cytochrome c. Defects in this gene are a cause of microphthalmia syndromic type 7 (MCOPS7). Three transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jan 2010]', 'X', 'protein_coding', 4859665, 4863936, 56.95, 2, 'ENSG00000004961', 'P53701'),
(1072, 'HCFC1R1', 'Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 4648031, 4650190, 60.74, 1, 'ENSG00000103145', 'Q9NWW0'),
(1073, 'HCN3', 'This gene encodes a multi-pass membrane protein that functions as a voltage gated cation channel. The encoded protein is a member of a family of closely related cyclic adenosine monophosphate-binding channel proteins. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '1', 'protein_coding', 299410, 302017, 41.33, 1, 'ENSG00000143630', 'Q9P1Z3'),
(1074, 'HCN3', 'This gene encodes a multi-pass membrane protein that functions as a voltage gated cation channel. The encoded protein is a member of a family of closely related cyclic adenosine monophosphate-binding channel proteins. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '1', 'protein_coding', 299410, 302017, 41.33, 1, 'ENSG00000263324', 'Q9P1Z3'),
(1075, 'HDX', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in chromatin. [provided by Alliance of Genome Resources, May 2025]', 'X', 'miRNA', 21683510, 21683597, 42.05, 1, 'ENSG00000165259', 'Q7Z353'),
(1076, 'HELZ2', 'The protein encoded by this gene is a nuclear transcriptional co-activator for peroxisome proliferator activated receptor alpha. The encoded protein contains a zinc finger and is a helicase that appears to be part of the peroxisome proliferator activated receptor alpha interacting complex. This gene is a member of the DNA2/NAM7 helicase gene family. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 128172, 128253, 74.39, 1, 'ENSG00000130589', 'Q9BYK8'),
(1077, 'HES5', 'This gene encodes a member of a family of basic helix-loop-helix transcriptional repressors. The protein product of this gene, which is activated downstream of the Notch pathway, regulates cell differentiation in multiple tissues. Disruptions in the normal expression of this gene have been associated with developmental diseases and cancer. [provided by RefSeq, Dec 2008]', '1', 'protein_coding', 86505668, 86721122, 40.01, 2, 'ENSG00000197921', 'Q5TA89'),
(1078, 'HES5', 'This gene encodes a member of a family of basic helix-loop-helix transcriptional repressors. The protein product of this gene, which is activated downstream of the Notch pathway, regulates cell differentiation in multiple tissues. Disruptions in the normal expression of this gene have been associated with developmental diseases and cancer. [provided by RefSeq, Dec 2008]', '1', 'protein_coding', 86505668, 86721122, 40.01, 2, 'ENSG00000273529', 'Q5TA89'),
(1079, 'HES6', 'This gene encodes a member of a subfamily of basic helix-loop-helix transcription repressors that have homology to the Drosophila enhancer of split genes. Members of this gene family regulate cell differentiation in numerous cell types. The protein encoded by this gene functions as a cofactor, interacting with other transcription factors through a tetrapeptide domain in its C-terminus. Alternatively spliced transcript variants encoding different isoforms have been described.[provided by RefSeq, Dec 2008]', '2', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000144485', 'Q96HZ4'),
(1080, 'HIF1AN', 'Enables several functions, including 2-oxoglutarate-dependent dioxygenase activity; NF-kappaB binding activity; and transition metal ion binding activity. Involved in negative regulation of Notch signaling pathway and positive regulation of myoblast differentiation. Located in cytosol; nucleoplasm; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 12446512, 12658107, 40.65, 13, 'ENSG00000166135', 'Q9NWT6'),
(1081, 'HINT1', 'This gene encodes a protein that hydrolyzes purine nucleotide phosphoramidates substrates, including AMP-morpholidate, AMP-N-alanine methyl ester, AMP-alpha-acetyl lysine methyl ester, and AMP-NH2. The encoded protein interacts with these substrates via a histidine triad motif. This gene is considered a tumor suppressor gene. In addition, mutations in this gene can cause autosomal recessive neuromyotonia and axonal neuropathy. There are several related pseudogenes on chromosome 7. Several transcript variants have been observed. [provided by RefSeq, Dec 2015]', '5', 'lncRNA', 51685363, 51693456, 57.03, 1, 'ENSG00000169567', 'P49773'),
(1082, 'HINT2', 'Histidine triad proteins, such as HINT2, are nucleotide hydrolases and transferases that act on the alpha-phosphate of ribonucleotides (Brenner, 2002 [PubMed 12119013]).[supplied by OMIM, Mar 2008]', '9', 'protein_coding', 50958818, 50963929, 45.62, 1, 'ENSG00000137133', 'Q9BX68'),
(1083, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000204257', 'P28067'),
(1084, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000242361', 'P28067'),
(1085, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000241394', 'P28067'),
(1086, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000243719', 'P28067'),
(1087, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000243189', 'P28067'),
(1088, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000242685', 'P28067'),
(1089, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000239463', 'P28067'),
(1090, 'HLA-DMA', 'HLA-DMA belongs to the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DMA) and a beta chain (DMB), both anchored in the membrane. It is located in intracellular vesicles. DM plays a central role in the peptide loading of MHC class II molecules by helping to release the CLIP molecule from the peptide binding site. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 122476958, 122477926, 49.43, 1, 'ENSG00000243215', 'P28067'),
(1091, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000241106', 'P13765'),
(1092, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000243612', 'P13765'),
(1093, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000241386', 'P13765'),
(1094, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000243496', 'P13765'),
(1095, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000241910', 'P13765'),
(1096, 'HLA-DOB', 'HLA-DOB belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DOA) and a beta chain (DOB), both anchored in the membrane. It is located in intracellular vesicles. DO suppresses peptide loading of MHC class II molecules by inhibiting HLA-DM. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3686970, 3712600, 55.28, 3, 'ENSG00000239457', 'P13765'),
(1097, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000204642', 'P30511'),
(1098, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000234487', 'P30511'),
(1099, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000235220', 'P30511'),
(1100, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000137403', 'P30511'),
(1101, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000237508', 'P30511'),
(1102, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000206509', 'P30511'),
(1103, 'HLA-F', 'This gene belongs to the HLA class I heavy chain paralogues. It encodes a non-classical heavy chain that forms a heterodimer with a beta-2 microglobulin light chain, with the heavy chain anchored in the membrane. Unlike most other HLA heavy chains, this molecule is localized in the endoplasmic reticulum and Golgi apparatus, with a small amount present at the cell surface in some cell types. It contains a divergent peptide-binding groove, and is thought to bind a restricted subset of peptides for immune presentation. This gene exhibits few polymorphisms. Multiple transcript variants encoding different isoforms have been found for this gene. These variants lack a coding exon found in transcripts from other HLA paralogues due to an altered splice acceptor site, resulting in a shorter cytoplasmic domain. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 50018575, 50058298, 53.46, 2, 'ENSG00000229698', 'P30511'),
(1104, 'HLTF', 'This gene encodes a member of the SWI/SNF family. Members of this family have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein contains a RING finger DNA binding motif. Two transcript variants encoding the same protein have been found for this gene. However, use of an alternative translation start site produces an isoform that is truncated at the N-terminus compared to the full-length protein. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 236324147, 236507535, 42.77, 10, 'ENSG00000071794', 'Q14527'),
(1105, 'HMGCR', 'HMG-CoA reductase is the rate-limiting enzyme for cholesterol synthesis and is regulated via a negative feedback mechanism mediated by sterols and non-sterol metabolites derived from mevalonate, the product of the reaction catalyzed by reductase. Normally in mammalian cells this enzyme is suppressed by cholesterol derived from the internalization and degradation of low density lipoprotein (LDL) via the LDL receptor. Competitive inhibitors of the reductase induce the expression of LDL receptors in the liver, which in turn increases the catabolism of plasma LDL and lowers the plasma concentration of cholesterol, an important determinant of atherosclerosis. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2008]', '5', 'processed_pseudogene', 107272582, 107273656, 57.58, 1, 'ENSG00000113161', 'P04035'),
(1106, 'HMGXB3', 'This gene is one of the non-canonical high mobility group (HMG) genes. The encoded protein contains an HMG-box domain found in DNA binding proteins such as transcription factors and chromosomal proteins. [provided by RefSeq, Aug 2011]', '5', 'protein_coding', 96816269, 96875623, 38.74, 7, 'ENSG00000113716', 'Q12766'),
(1107, 'HNF4A', 'The protein encoded by this gene is a nuclear transcription factor which binds DNA as a homodimer. The encoded protein controls the expression of several genes, including hepatocyte nuclear factor 1 alpha, a transcription factor which regulates the expression of several hepatic genes. This gene may play a role in development of the liver, kidney, and intestines. Mutations in this gene have been associated with monogenic autosomal dominant non-insulin-dependent diabetes mellitus type I. Alternative splicing of this gene results in multiple transcript variants encoding several different isoforms. [provided by RefSeq, Apr 2012]', '20', 'protein_coding', 195712, 241065, 42.55, 3, 'ENSG00000101076', 'P41235'),
(1108, 'HNMT', 'In mammals, histamine is metabolized by two major pathways: N(tau)-methylation via histamine N-methyltransferase and oxidative deamination via diamine oxidase. This gene encodes the first enzyme which is found in the cytosol and uses S-adenosyl-L-methionine as the methyl donor. In the mammalian brain, the neurotransmitter activity of histamine is controlled by N(tau)-methylation as diamine oxidase is not found in the central nervous system. A common genetic polymorphism affects the activity levels of this gene product in red blood cells. Multiple alternatively spliced transcript variants that encode different proteins have been found for this gene. [provided by RefSeq, Jul 2008]', '2', 'unprocessed_pseudogene', 57018100, 57019021, 39.26, 1, 'ENSG00000150540', 'P50135'),
(1109, 'HNRNPCL4', 'Predicted to enable RNA binding activity. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 299692, 303195, 67.21, 3, 'ENSG00000282095', 'P0DMR1'),
(1110, 'HNRNPCL4', 'Predicted to enable RNA binding activity. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 299692, 303195, 67.21, 3, 'ENSG00000283231', 'P0DMR1'),
(1111, 'HNRNPCL4', 'Predicted to enable RNA binding activity. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 299692, 303195, 67.21, 3, 'ENSG00000273865', 'P0DMR1'),
(1112, 'HNRNPCL4', 'Predicted to enable RNA binding activity. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 299692, 303195, 67.21, 3, 'ENSG00000179412', 'P0DMR1'),
(1113, 'HNRNPH3', 'This gene belongs to the subfamily of ubiquitously expressed heterogeneous nuclear ribonucleoproteins (hnRNPs). The hnRNPs are RNA binding proteins and they complex with heterogeneous nuclear RNA (hnRNA). These proteins are associated with pre-mRNAs in the nucleus and appear to influence pre-mRNA processing and other aspects of mRNA metabolism and transport. While all of the hnRNPs are present in the nucleus, some seem to shuttle between the nucleus and the cytoplasm. The hnRNP proteins have distinct nucleic acid binding properties. The protein encoded by this gene has two repeats of quasi-RRM domains that bind to RNAs. It is localized in nuclear bodies of the nucleus. This protein is involved in the splicing process and it also participates in early heat shock-induced splicing arrest by transiently leaving the hnRNP complexes. Several alternatively spliced transcript variants have been noted for this gene, however, not all are fully characterized. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 26017032, 26017787, 51.72, 1, 'ENSG00000096746', 'P31942'),
(1114, 'HNRNPLL', 'HNRNPLL is a master regulator of activation-induced alternative splicing in T cells. In particular, it alters splicing of CD45 (PTPRC; MIM 151460), a tyrosine phosphatase essential for T-cell development and activation (Oberdoerffer et al., 2008 [PubMed 18669861]).[supplied by OMIM, Aug 2008]', '2', 'protein_coding', 98291650, 98401090, 46.78, 5, 'ENSG00000143889', 'Q8WVV9'),
(1115, 'HNRNPR', 'This gene encodes an RNA-binding protein that is a member of the spliceosome C complex, which functions in pre-mRNA processing and transport. The encoded protein also promotes transcription at the c-fos gene. Alternative splicing results in multiple transcript variants. There are pseudogenes for this gene on chromosomes 4, 11, and 10. [provided by RefSeq, Jul 2014]', '1', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000282958', 'O43390'),
(1116, 'HNRNPR', 'This gene encodes an RNA-binding protein that is a member of the spliceosome C complex, which functions in pre-mRNA processing and transport. The encoded protein also promotes transcription at the c-fos gene. Alternative splicing results in multiple transcript variants. There are pseudogenes for this gene on chromosomes 4, 11, and 10. [provided by RefSeq, Jul 2014]', '1', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000125944', 'O43390'),
(1117, 'HPS4', 'This gene encodes a protein component of biogenesis of lysosome-related organelles complexes (BLOC). BLOC complexes are important for the formation of endosomal-lysosomal organelles such as melanosomes and platelet dense granules. Mutations in this gene result in subtype 4 of Hermansky-Pudlak syndrome, a form of albinism. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '22', 'TR_V_gene', 21749178, 21749705, 43.56, 1, 'ENSG00000100099', 'Q9NQG7'),
(1118, 'HS1BP3', 'The protein encoded by this gene shares similarity with mouse Hs1bp3, an Hcls1/Hs1-interacting protein that may be involved in lymphocyte activation. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 23747562, 23751268, 56.78, 2, 'ENSG00000118960', 'Q53T59'),
(1119, 'HSBP1L1', 'Predicted to enable transcription corepressor activity. Predicted to be involved in cellular heat acclimation. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '18', 'TR_J_gene', 22533497, 22533560, 48.44, 1, 'ENSG00000226742', 'C9JCN9'),
(1120, 'HSDL2', 'Predicted to enable oxidoreductase activity. Involved in cholesterol homeostasis. Located in mitochondrion and peroxisome. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 30757423, 30762221, 53.91, 12, 'ENSG00000119471', 'Q6YN16'),
(1121, 'HSF2', 'The protein encoded by this gene belongs to the HSF family of transcription factors that bind specifically to the heat-shock promoter element and activate transcription. Heat shock transcription factors activate heat-shock response genes under conditions of heat or other stresses. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2011]', '6', 'protein_coding', 7404874, 7419860, 59.54, 4, 'ENSG00000025156', 'Q03933'),
(1122, 'HSF2BP', 'HSF2 binding protein (HSF2BP) associates with HSF2. The interaction occurs between the trimerization domain of HSF2 and the amino terminal hydrophilic region of HSF2BP that comprises two leucine zipper motifs. HSF2BP may therefore be involved in modulating HSF2 activation. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 10116777, 10130258, 37.42, 13, 'ENSG00000160207', 'O75031'),
(1123, 'HSFX2', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 168540768, 168543997, 42.17, 1, 'ENSG00000268738', 'Q9UBD0'),
(1124, 'HSPA12A', 'Predicted to enable ATP binding activity. Predicted to act upstream of or within several processes, including import into nucleus; liver development; and macrophage activation. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 53276993, 53277071, 58.23, 1, 'ENSG00000165868', 'O43301'),
(1125, 'HSPA1B', 'This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44355699, 44432845, 48.23, 17, 'ENSG00000232804', 'P0DMV9'),
(1126, 'HSPA1B', 'This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44355699, 44432845, 48.23, 17, 'ENSG00000231555', 'P0DMV9'),
(1127, 'HSPA1B', 'This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44355699, 44432845, 48.23, 17, 'ENSG00000224501', 'P0DMV9'),
(1128, 'HSPA1B', 'This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44355699, 44432845, 48.23, 17, 'ENSG00000212866', 'P0DMV9'),
(1129, 'HSPA1B', 'This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44355699, 44432845, 48.23, 17, 'ENSG00000204388', 'P0DMV9'),
(1130, 'HSPBAP1', 'This gene encodes a protein that binds to one of the small heat shock proteins, specifically hsp27. Hsp27 is involved with cell growth and differentiation. This encoded protein was found to be abnormally expressed in patients with intractable epilepsy, although how brain function is affected remains unknown. [provided by RefSeq, Sep 2011]', '3', 'protein_coding', 1212926, 1214106, 50.72, 1, 'ENSG00000169087', 'Q96EW2'),
(1131, 'HTATSF1', 'The protein encoded by this gene functions as a cofactor for the stimulation of transcriptional elongation by HIV-1 Tat, which binds to the HIV-1 promoter through Tat-TAR interaction. This protein may also serve as a dual-function factor to couple transcription and splicing and to facilitate their reciprocal activation. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 21082159, 21125270, 38.78, 10, 'ENSG00000102241', 'O43719'),
(1132, 'HTN1', 'This gene encodes a member of the histatin family of small, histidine-rich, cationic proteins. They function as antimicrobial peptides and are important components of the innate immune system. Histatins are found in saliva and exhibit antibacterial, antifungal activities and function in wound healing. [provided by RefSeq, Aug 2014]', '4', 'protein_coding', 67647085, 67674820, 42.96, 6, 'ENSG00000126550', 'P15515'),
(1133, 'HTN1', 'This gene encodes a member of the histatin family of small, histidine-rich, cationic proteins. They function as antimicrobial peptides and are important components of the innate immune system. Histatins are found in saliva and exhibit antibacterial, antifungal activities and function in wound healing. [provided by RefSeq, Aug 2014]', '4', 'protein_coding', 67647085, 67674820, 42.96, 6, 'ENSG00000283046', 'P15515'),
(1134, 'HTN3', 'This gene encodes a member of the histatin family of small, histidine-rich, cationic proteins. They function as antimicrobial peptides and are important components of the innate immune system. Histatins are found in saliva and exhibit antibacterial, antifungal activities and function in wound healing. [provided by RefSeq, Sep 2014]', '4', 'protein_coding', 65916810, 65919062, 61.47, 3, 'ENSG00000282967', 'P15516'),
(1135, 'HTN3', 'This gene encodes a member of the histatin family of small, histidine-rich, cationic proteins. They function as antimicrobial peptides and are important components of the innate immune system. Histatins are found in saliva and exhibit antibacterial, antifungal activities and function in wound healing. [provided by RefSeq, Sep 2014]', '4', 'protein_coding', 65916810, 65919062, 61.47, 3, 'ENSG00000205649', 'P15516'),
(1136, 'HTR2A', 'This gene encodes one of the receptors for serotonin, a neurotransmitter with many roles. Mutations in this gene are associated with susceptibility to schizophrenia and obsessive-compulsive disorder, and are also associated with response to the antidepressant citalopram in patients with major depressive disorder (MDD). MDD patients who also have a mutation in intron 2 of this gene show a significantly reduced response to citalopram as this antidepressant downregulates expression of this gene. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 57648392, 57711536, 54.59, 8, 'ENSG00000102468', 'P28223');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1137, 'HTRA1', 'This gene encodes a member of the trypsin family of serine proteases. This protein is a secreted enzyme that is proposed to regulate the availability of insulin-like growth factors (IGFs) by cleaving IGF-binding proteins. It has also been suggested to be a regulator of cell growth. Variations in the promoter region of this gene are the cause of susceptibility to age-related macular degeneration type 7. [provided by RefSeq, Jul 2008]', '10', 'miRNA', 30117079, 30117172, 52.13, 1, 'ENSG00000166033', 'Q92743'),
(1138, 'HYAL4', 'This gene encodes a protein which is similar in structure to hyaluronidases but lacks hyaluronidase activity. The encoded protein acts as a chondroitin-sulfate-specific endo-beta-N-acetylgalactosaminidase; that is, it exhibits hydrolytic activity toward chondroitin sulfate chains and degrades them into oligosaccharides. Proteoglycans are formed by the covalent linkage of chondroitin sulfate chains to protein. Proteoglycans are ubiquitous components of the extracellular matrix of connective tissues and are also found at the surface of many cell types where they participate in a variety of cellular processes such as cell proliferation, differentiation, migration, cell-cell recognition, extracellular matrix deposition, and tissue morphogenesis. The expression of this gene is highest in testes and placenta. [provided by RefSeq, Apr 2019]', '7', 'protein_coding', 53936676, 54048958, 43.07, 22, 'ENSG00000106302', 'Q2M3T9'),
(1139, 'HYMAI', 'This gene, which encodes a non-protein coding transcript, exhibits differential DNA methylation between the two parental alleles at an adjacent CpG island, and is expressed only from the paternal allele. It is believed to be one of the causative genes for transient neonatal diabetes mellitus (TNDM), which is a rare disease characterized by intrauterine growth retardation, dehydration, and failure to thrive due to a lack of normal insulin secretion. [provided by RefSeq, Oct 2010]', '6', 'protein_coding', 10300865, 10373130, 45.88, 3, 'ENSG00000283122', ''),
(1140, 'ICA1', 'This gene encodes a protein with an arfaptin homology domain that is found both in the cytosol and as membrane-bound form on the Golgi complex and immature secretory granules. This protein is believed to be an autoantigen in insulin-dependent diabetes mellitus and primary Sjogren\'s syndrome. Several transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Feb 2013]', '7', 'protein_coding', 303403, 331374, 39.66, 7, 'ENSG00000003147', 'Q05084'),
(1141, 'ICAM2', 'The protein encoded by this gene is a member of the intercellular adhesion molecule (ICAM) family. All ICAM proteins are type I transmembrane glycoproteins, contain 2-9 immunoglobulin-like C2-type domains, and bind to the leukocyte adhesion LFA-1 protein. This protein may play a role in lymphocyte recirculation by blocking LFA-1-dependent cell adhesion. It mediates adhesive interactions important for antigen-specific immune response, NK-cell mediated clearance, lymphocyte recirculation, and other cellular interactions important for immune response and surveillance. Several transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 100706121, 100707486, 59.08, 2, 'ENSG00000108622', 'P13598'),
(1142, 'ICAM4', 'This gene encodes the Landsteiner-Wiener (LW) blood group antigen(s) that belongs to the immunoglobulin (Ig) superfamily, and that shares similarity with the intercellular adhesion molecule (ICAM) protein family. This ICAM protein contains 2 Ig-like C2-type domains and binds to the leukocyte adhesion LFA-1 protein. The molecular basis of the LW(A)/LW(B) blood group antigens is a single aa variation at position 100; Gln-100=LW(A) and Arg-100=LW(B). Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 18466878, 18484648, 40.97, 22, 'ENSG00000105371', 'Q14773'),
(1143, 'ICE1', 'Enables protein-macromolecule adaptor activity. Involved in several processes, including positive regulation of intracellular protein transport; positive regulation of transcription by RNA polymerase III; and snRNA transcription. Located in Cajal body; euchromatin; and histone locus body. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 2426, 47592, 40.76, 7, 'ENSG00000164151', 'Q9Y2F5'),
(1144, 'ID1', 'The protein encoded by this gene is a helix-loop-helix (HLH) protein that can form heterodimers with members of the basic HLH family of transcription factors. The encoded protein has no DNA binding activity and therefore can inhibit the DNA binding and transcriptional activation ability of basic HLH proteins with which it interacts. This protein may play a role in cell growth, senescence, and differentiation. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'TR_V_gene', 21811502, 21811977, 42.44, 1, 'ENSG00000125968', 'P41134'),
(1145, 'IDH3B', 'Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. NAD(+)-dependent isocitrate dehydrogenases catalyze the allosterically regulated rate-limiting step of the tricarboxylic acid cycle. Each isozyme is a heterotetramer that is composed of two alpha subunits, one beta subunit, and one gamma subunit. The protein encoded by this gene is the beta subunit of one isozyme of NAD(+)-dependent isocitrate dehydrogenase. Multiple alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Sep 2016]', '20', 'protein_coding', 1842, 18600, 47.08, 2, 'ENSG00000101365', 'O43837'),
(1146, 'IER3IP1', 'This gene encodes a small protein that is localized to the endoplasmic reticulum (ER) and may play a role in the ER stress response by mediating cell differentiation and apoptosis. Transcription of this gene is regulated by tumor necrosis factor alpha and specificity protein 1 (Sp1). Mutations in this gene may play a role in microcephaly, epilepsy, and diabetes syndrome (MEDS), and a pseudogene of this gene is located on the long arm of chromosome 12. [provided by RefSeq, Dec 2011]', '18', 'TR_J_gene', 22483004, 22483069, 37.88, 1, 'ENSG00000134049', 'Q9Y5U9'),
(1147, 'IFI30', 'The protein encoded by this gene is a lysosomal thiol reductase that at low pH can reduce protein disulfide bonds. The enzyme is expressed constitutively in antigen-presenting cells and induced by gamma-interferon in other cell types. This enzyme has an important role in MHC class II-restricted antigen processing. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 155758992, 155807811, 36.48, 8, 'ENSG00000216490', 'P13284'),
(1148, 'IFIT2', 'Enables RNA binding activity. Involved in antiviral innate immune response and positive regulation of apoptotic process. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 37181579, 37218153, 42.21, 9, 'ENSG00000119922', 'P09913'),
(1149, 'IFNAR1', 'The protein encoded by this gene is a type I membrane protein that forms one of the two chains of a receptor for interferons alpha and beta. Binding and activation of the receptor stimulates Janus protein kinases, which in turn phosphorylate several proteins, including STAT1 and STAT2. The protein belongs to the type II cytokine receptor family and functions as an antiviral factor. [provided by RefSeq, Jul 2020]', '21', 'TR_J_gene', 22518446, 22518505, 50.00, 1, 'ENSG00000142166', 'P17181'),
(1150, 'IFNGR2', 'This gene (IFNGR2) encodes the non-ligand-binding beta chain of the gamma interferon receptor. Human interferon-gamma receptor is a heterodimer of IFNGR1 and IFNGR2. Defects in IFNGR2 are a cause of mendelian susceptibility to mycobacterial disease (MSMD), also known as familial disseminated atypical mycobacterial infection. MSMD is a genetically heterogeneous disease with autosomal recessive, autosomal dominant or X-linked inheritance. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 79481155, 79556077, 35.46, 30, 'ENSG00000159128', 'P38484'),
(1151, 'IFNGR2', 'This gene (IFNGR2) encodes the non-ligand-binding beta chain of the gamma interferon receptor. Human interferon-gamma receptor is a heterodimer of IFNGR1 and IFNGR2. Defects in IFNGR2 are a cause of mendelian susceptibility to mycobacterial disease (MSMD), also known as familial disseminated atypical mycobacterial infection. MSMD is a genetically heterogeneous disease with autosomal recessive, autosomal dominant or X-linked inheritance. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 79481155, 79556077, 35.46, 30, 'ENSG00000262795', 'P38484'),
(1152, 'IFRD1', 'This gene is an immediate early gene that encodes a protein related to interferon-gamma. This protein may function as a transcriptional co-activator/repressor that controls the growth and differentiation of specific cell types during embryonic development and tissue regeneration. Mutations in this gene are associated with sensory/motor neuropathy with ataxia. This gene may also be involved in modulating the pathogenesis of cystic fibrosis lung disease. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Oct 2010]', '7', 'protein_coding', 14812, 49543, 42.77, 9, 'ENSG00000006652', 'O00458'),
(1153, 'IFT140', 'This gene encodes one of the subunits of the intraflagellar transport (IFT) complex A. Intraflagellar transport is involved in the genesis, resorption and signaling of primary cilia. The primary cilium is a microtubule-based sensory organelle at the surface of most quiescent mammalian cells, that receives signals from its environment, such as the flow of fluid, light or odors, and transduces those signals to the nucleus. Loss of the corresponding protein in mouse results in renal cystic disease. [provided by RefSeq, Jun 2012]', '16', 'protein_coding', 91612, 126747, 51.48, 1, 'ENSG00000187535', 'Q96RY7'),
(1154, 'IFT56', 'Predicted to enable intraciliary transport particle B binding activity. Involved in cilium assembly. Located in cilium. Part of intraciliary transport particle B. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 123452193, 123535077, 36.64, 6, 'ENSG00000105948', 'A0AVF1'),
(1155, 'IFT57', 'Predicted to enable DNA binding activity. Acts upstream of or within apoptotic process and regulation of apoptotic process. Located in ciliary base and photoreceptor connecting cilium. Part of intraciliary transport particle B. Implicated in orofaciodigital syndrome XVIII. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 94148988, 94275068, 38.07, 5, 'ENSG00000114446', 'Q9NWB7'),
(1156, 'IGHA2', 'Contributes to immunoglobulin receptor binding activity. Involved in glomerular filtration and positive regulation of respiratory burst. Located in extracellular space. Part of monomeric IgA immunoglobulin complex and secretory dimeric IgA immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 31106107, 31112791, 57.82, 10, 'ENSG00000276173', 'P01877'),
(1157, 'IGHA2', 'Contributes to immunoglobulin receptor binding activity. Involved in glomerular filtration and positive regulation of respiratory burst. Located in extracellular space. Part of monomeric IgA immunoglobulin complex and secretory dimeric IgA immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 31106107, 31112791, 57.82, 10, 'ENSG00000211890', 'P01877'),
(1158, 'IGHEP1', 'This gene encodes a member of the inter-alpha-trypsin inhibitor family of proteins. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the heavy chain of the inter-alpha-trypsin inhibitor complex, which is secreted by hepatocytes into the blood. The heavy chain also interacts with hyaluronan, and this interaction may play a role in ovulation and fertilization, and has been implicated in multiple inflammatory diseases. This gene is present in a gene cluster on chromosome 3. [provided by RefSeq, Nov 2015]', '3', 'protein_coding', 128322544, 128334072, 51.72, 5, 'ENSG00000281947', 'P19827'),
(1159, 'IGHEP1', 'This gene encodes a member of the inter-alpha-trypsin inhibitor family of proteins. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the heavy chain of the inter-alpha-trypsin inhibitor complex, which is secreted by hepatocytes into the blood. The heavy chain also interacts with hyaluronan, and this interaction may play a role in ovulation and fertilization, and has been implicated in multiple inflammatory diseases. This gene is present in a gene cluster on chromosome 3. [provided by RefSeq, Nov 2015]', '3', 'protein_coding', 128322544, 128334072, 51.72, 5, 'ENSG00000253692', 'P19827'),
(1160, 'IGHV1OR21-1', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 39009865, 39038089, 37.34, 11, 'ENSG00000277282', 'A0A087WYE8'),
(1161, 'IGHV3-15', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 69051363, 69112987, 35.89, 5, 'ENSG00000211943', 'A0A0B4J1V0'),
(1162, 'IGHV3-15', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 69051363, 69112987, 35.89, 5, 'ENSG00000282290', 'A0A0B4J1V0'),
(1163, 'IGHV3-20', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 40626921, 40647561, 45.21, 3, 'ENSG00000211946', 'A0A0C4DH32'),
(1164, 'IGHV3-20', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 40626921, 40647561, 45.21, 3, 'ENSG00000282476', 'A0A0C4DH32'),
(1165, 'IGHV4-31', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 73212299, 73214851, 40.34, 1, 'ENSG00000231475', 'P0DP07'),
(1166, 'IGHV4-31', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 73212299, 73214851, 40.34, 1, 'ENSG00000282404', 'P0DP07'),
(1167, 'IGHV4-34', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 31844303, 31845604, 64.13, 1, 'ENSG00000211956', 'P06331'),
(1168, 'IGHV4-34', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 31844303, 31845604, 64.13, 1, 'ENSG00000276491', 'P06331'),
(1169, 'IGHV5-10-1', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 39329990, 39725408, 40.55, 8, 'ENSG00000282651', 'A0A0J9YXX1'),
(1170, 'IGKV1-13', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'miRNA', 41923222, 41923297, 57.89, 1, 'ENSG00000282603', 'P0DP09'),
(1171, 'IGKV1-13', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'miRNA', 41923222, 41923297, 57.89, 1, 'ENSG00000253497', 'P0DP09'),
(1172, 'IGKV1-16', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 91232532, 91253925, 51.90, 5, 'ENSG00000282282', 'P04430'),
(1173, 'IGKV1-16', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 91232532, 91253925, 51.90, 5, 'ENSG00000240864', 'P04430'),
(1174, 'IGKV1-8', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 26254104, 26420982, 37.47, 11, 'ENSG00000283113', 'A0A0C4DH67'),
(1175, 'IGKV1-8', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 26254104, 26420982, 37.47, 11, 'ENSG00000240671', 'A0A0C4DH67'),
(1176, 'IGKV2-29', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 51756996, 51757988, 49.65, 1, 'ENSG00000282635', 'A2NJV5'),
(1177, 'IGKV2-29', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 51756996, 51757988, 49.65, 1, 'ENSG00000253998', 'A2NJV5'),
(1178, 'IGKV5-2', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 16313574, 16514026, 43.50, 12, 'ENSG00000282172', 'P06315'),
(1179, 'IGKV5-2', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 16313574, 16514026, 43.50, 12, 'ENSG00000211599', 'P06315'),
(1180, 'IGLC1', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Located in blood microparticle and extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 22886267, 22886379, 43.36, 1, 'ENSG00000211675', 'P0CG04'),
(1181, 'IGLC2', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Located in blood microparticle and extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 35002404, 35092807, 42.92, 2, 'ENSG00000211677', 'P0DOY2'),
(1182, 'IGLC3', 'Predicted to enable antigen binding activity. Predicted to be involved in immunoglobulin mediated immune response. Located in blood microparticle and extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 36214441, 36277042, 43.82, 6, 'ENSG00000211679', 'P0DOY3'),
(1183, 'IGLV1-51', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 5225464, 5229395, 37.64, 6, 'ENSG00000211644', 'P01701'),
(1184, 'IGLV2-11', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 499210, 540200, 49.98, 26, 'ENSG00000211668', 'P01706'),
(1185, 'IGLV2-23', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 57578456, 57593890, 47.31, 8, 'ENSG00000211660', 'P01705'),
(1186, 'IGLV3-27', 'Predicted to enable antigen binding activity. Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 114442299, 115845780, 36.62, 13, 'ENSG00000211658', 'P01718'),
(1187, 'IGLV3-9', 'Predicted to be involved in immune response. Predicted to be located in extracellular region and plasma membrane. Predicted to be part of immunoglobulin complex. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 114743445, 114743525, 34.57, 1, 'ENSG00000211670', 'A0A075B6K5'),
(1188, 'IGSF9B', 'Predicted to enable kinase binding activity. Predicted to be involved in synaptic membrane adhesion. Predicted to act upstream of or within homophilic cell adhesion via plasma membrane adhesion molecules and positive regulation of inhibitory postsynaptic potential. Predicted to be located in dendrite; inhibitory synapse; and neuronal cell body. Predicted to be active in GABA-ergic synapse; neuron projection; and postsynaptic specialization of symmetric synapse. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 25612935, 25624014, 43.42, 1, 'ENSG00000080854', 'Q9UPX0'),
(1189, 'IL15', 'The protein encoded by this gene is a cytokine that regulates T and natural killer cell activation and proliferation. This cytokine and interleukine 2 share many biological activities. They are found to bind common hematopoietin receptor subunits, and may compete for the same receptor, and thus negatively regulate each other\'s activity. The number of CD8+ memory cells is shown to be controlled by a balance between this cytokine and IL2. This cytokine induces the activation of JAK kinases, as well as the phosphorylation and activation of transcription activators STAT3, STAT5, and STAT6. Studies of the mouse counterpart suggested that this cytokine may increase the expression of apoptosis inhibitor BCL2L1/BCL-x(L), possibly through the transcription activation activity of STAT6, and thus prevent apoptosis. Alternatively spliced transcript variants of this gene have been reported. [provided by RefSeq, Feb 2011]', '4', 'protein_coding', 94896949, 94927223, 42.96, 7, 'ENSG00000164136', 'P40933'),
(1190, 'IL17RD', 'This gene encodes a membrane protein belonging to the interleukin-17 receptor (IL-17R) protein family. The encoded protein is a component of the interleukin-17 receptor signaling complex, and the interaction between this protein and IL-17R does not require the interleukin. The gene product also affects fibroblast growth factor signaling, inhibiting or stimulating growth through MAPK/ERK signaling. Alternate splicing generates multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jan 2016]', '3', 'protein_coding', 189952, 190854, 52.49, 1, 'ENSG00000144730', 'Q8NFM7'),
(1191, 'IL2RG', 'The protein encoded by this gene is an important signaling component of many interleukin receptors, including those of interleukin -2, -4, -7 and -21, and is thus referred to as the common gamma chain. Mutations in this gene cause X-linked severe combined immunodeficiency (XSCID), as well as X-linked combined immunodeficiency (XCID), a less severe immunodeficiency disorder. [provided by RefSeq, Mar 2010]', 'X', 'protein_coding', 56994492, 56998447, 54.07, 3, 'ENSG00000147168', 'P31785'),
(1192, 'IL31', 'IL31, which is made principally by activated Th2-type T cells, interacts with a heterodimeric receptor consisting of IL31RA (MIM 609510) and OSMR (MIM 601743) that is constitutively expressed on epithelial cells and keratinocytes. IL31 may be involved in the promotion of allergic skin disorders and in regulating other allergic diseases, such as asthma (Dillon et al., 2004 [PubMed 15184896]).[supplied by OMIM, Mar 2008]', '12', 'miRNA', 3885353, 3885432, 43.75, 1, 'ENSG00000204671', 'Q6EBC2'),
(1193, 'IL3RA', 'The protein encoded by this gene is an interleukin 3 specific subunit of a heterodimeric cytokine receptor. The receptor is comprised of a ligand specific alpha subunit and a signal transducing beta subunit shared by the receptors for interleukin 3 (IL3), colony stimulating factor 2 (CSF2/GM-CSF), and interleukin 5 (IL5). The binding of this protein to IL3 depends on the beta subunit. The beta subunit is activated by the ligand binding, and is required for the biological activities of IL3. This gene and the gene encoding the colony stimulating factor 2 receptor alpha chain (CSF2RA) form a cytokine receptor gene cluster in a X-Y pseudoautosomal region on chromosomes X or Y. Alternatively spliced transcript variants encoding distinct isoforms have been found. [provided by RefSeq, Jun 2012]', 'X, Y', 'protein_coding', 457879, 463247, 47.90, 1, 'ENSG00000292332', 'P26951'),
(1194, 'IL3RA', 'The protein encoded by this gene is an interleukin 3 specific subunit of a heterodimeric cytokine receptor. The receptor is comprised of a ligand specific alpha subunit and a signal transducing beta subunit shared by the receptors for interleukin 3 (IL3), colony stimulating factor 2 (CSF2/GM-CSF), and interleukin 5 (IL5). The binding of this protein to IL3 depends on the beta subunit. The beta subunit is activated by the ligand binding, and is required for the biological activities of IL3. This gene and the gene encoding the colony stimulating factor 2 receptor alpha chain (CSF2RA) form a cytokine receptor gene cluster in a X-Y pseudoautosomal region on chromosomes X or Y. Alternatively spliced transcript variants encoding distinct isoforms have been found. [provided by RefSeq, Jun 2012]', 'X, Y', 'protein_coding', 457879, 463247, 47.90, 1, 'ENSG00000185291', 'P26951'),
(1195, 'IL7R', 'The protein encoded by this gene is a receptor for interleukin 7 (IL7). The function of this receptor requires the interleukin 2 receptor, gamma chain (IL2RG), which is a common gamma chain shared by the receptors of various cytokines, including interleukins 2, 4, 7, 9, and 15. This protein has been shown to play a critical role in V(D)J recombination during lymphocyte development. Defects in this gene may be associated with severe combined immunodeficiency (SCID). Alternatively spliced transcript variants have been found. [provided by RefSeq, Dec 2015]', '5', 'protein_coding', 115189410, 115192868, 62.76, 1, 'ENSG00000168685', 'P16871'),
(1196, 'IL9R', 'The protein encoded by this gene is a cytokine receptor that specifically mediates the biological effects of interleukin 9 (IL9). The functional IL9 receptor complex requires this protein as well as the interleukin 2 receptor, gamma (IL2RG), a common gamma subunit shared by the receptors of many different cytokines. The ligand binding of this receptor leads to the activation of various JAK kinases and STAT proteins, which connect to different biologic responses. This gene is located at the pseudoautosomal regions of X and Y chromosomes. Genetic studies suggested an association of this gene with the development of asthma. Multiple pseudogenes on chromosome 9, 10, 16, and 18 have been described. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2008]', 'X, Y', 'protein_coding', 20756108, 20781032, 42.67, 1, 'ENSG00000292373', 'Q01113'),
(1197, 'IL9R', 'The protein encoded by this gene is a cytokine receptor that specifically mediates the biological effects of interleukin 9 (IL9). The functional IL9 receptor complex requires this protein as well as the interleukin 2 receptor, gamma (IL2RG), a common gamma subunit shared by the receptors of many different cytokines. The ligand binding of this receptor leads to the activation of various JAK kinases and STAT proteins, which connect to different biologic responses. This gene is located at the pseudoautosomal regions of X and Y chromosomes. Genetic studies suggested an association of this gene with the development of asthma. Multiple pseudogenes on chromosome 9, 10, 16, and 18 have been described. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2008]', 'X, Y', 'protein_coding', 20756108, 20781032, 42.67, 1, 'ENSG00000124334', 'Q01113'),
(1198, 'ILVBL', 'The protein encoded by this gene shares similarity with several thiamine pyrophosphate-binding proteins identified in bacteria, yeast, and plants. The highest degree of similarity is found with bacterial acetolactate synthases (AHAS), which are enzymes that catalyze the first step in branched-chain amino acid biosynthesis. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 50934867, 50959140, 40.20, 16, 'ENSG00000105135', 'A1L0T0'),
(1199, 'IMPG2', 'The protein encoded by this gene binds chondroitin sulfate and hyaluronan and is a proteoglycan. The encoded protein plays a role in the organization of the interphotoreceptor matrix and may promote the growth and maintenance of the light-sensitive photoreceptor outer segment. Defects in this gene are a cause of retinitis pigmentosa type 56 and maculopathy, IMPG2-related.[provided by RefSeq, Mar 2011]', '3', 'protein_coding', 69035751, 69079076, 46.21, 5, 'ENSG00000081148', 'Q9BZV3'),
(1200, 'ING3', 'The protein encoded by this gene is similar to ING1, a tumor suppressor protein that can interact with TP53, inhibit cell growth, and induce apoptosis. This protein contains a PHD-finger, which is a common motif in proteins involved in chromatin remodeling. This gene can activate p53 trans-activated promoters, including promoters of p21/waf1 and bax. Overexpression of this gene has been shown to inhibit cell growth and induce apoptosis. Allelic loss and reduced expression of this gene were detected in head and neck cancers. Two alternatively spliced transcript variants encoding different isoforms have been observed. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 5922585, 5922647, 53.97, 1, 'ENSG00000071243', 'Q9NXR8'),
(1201, 'INHBA', 'This gene encodes a member of the TGF-beta (transforming growth factor-beta) superfamily of proteins. The encoded preproprotein is proteolytically processed to generate a subunit of the dimeric activin and inhibin protein complexes. These complexes activate and inhibit, respectively, follicle stimulating hormone secretion from the pituitary gland. The encoded protein also plays a role in eye, tooth and testis development. Elevated expression of this gene may be associated with cancer cachexia in human patients. [provided by RefSeq, Aug 2016]', '7', 'protein_coding', 31465471, 31476757, 55.69, 2, 'ENSG00000122641', 'P08476'),
(1202, 'INO80D', 'Involved in several processes, including chromatin remodeling; regulation of chromosome organization; and regulation of nucleobase-containing compound metabolic process. Part of Ino80 complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 24426545, 24430954, 60.63, 2, 'ENSG00000283510', 'Q53TQ3'),
(1203, 'INO80D', 'Involved in several processes, including chromatin remodeling; regulation of chromosome organization; and regulation of nucleobase-containing compound metabolic process. Part of Ino80 complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 24426545, 24430954, 60.63, 2, 'ENSG00000114933', 'Q53TQ3'),
(1204, 'INPP5A', 'The protein encoded by this gene is a membrane-associated type I inositol 1,4,5-trisphosphate (InsP3) 5-phosphatase. InsP3 5-phosphatases hydrolyze Ins(1,4,5)P3, which mobilizes intracellular calcium and acts as a second messenger mediating cell responses to various stimulation. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 89710299, 89717872, 37.30, 2, 'ENSG00000068383', 'Q14642'),
(1205, 'INPP5D', 'This gene is a member of the inositol polyphosphate-5-phosphatase (INPP5) family and encodes a protein with an N-terminal SH2 domain, an inositol phosphatase domain, and two C-terminal protein interaction domains. Expression of this protein is restricted to hematopoietic cells where its movement from the cytosol to the plasma membrane is mediated by tyrosine phosphorylation. At the plasma membrane, the protein hydrolyzes the 5\' phosphate from phosphatidylinositol (3,4,5)-trisphosphate and inositol-1,3,4,5-tetrakisphosphate, thereby affecting multiple signaling pathways. The protein is also partly localized to the nucleus, where it may be involved in nuclear inositol phosphate signaling processes. Overall, the protein functions as a negative regulator of myeloid cell proliferation and survival. Mutations in this gene are associated with defects and cancers of the immune system. Deficiencies in the encoded protein, SHIP1, have been associated with Inflammatory Bowel Disease types such as Crohn\'s Disease and Ulcerative Colitis. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jul 2020]', '2', 'protein_coding', 58385345, 58399220, 37.47, 4, 'ENSG00000281614', 'Q92835'),
(1206, 'INPP5D', 'This gene is a member of the inositol polyphosphate-5-phosphatase (INPP5) family and encodes a protein with an N-terminal SH2 domain, an inositol phosphatase domain, and two C-terminal protein interaction domains. Expression of this protein is restricted to hematopoietic cells where its movement from the cytosol to the plasma membrane is mediated by tyrosine phosphorylation. At the plasma membrane, the protein hydrolyzes the 5\' phosphate from phosphatidylinositol (3,4,5)-trisphosphate and inositol-1,3,4,5-tetrakisphosphate, thereby affecting multiple signaling pathways. The protein is also partly localized to the nucleus, where it may be involved in nuclear inositol phosphate signaling processes. Overall, the protein functions as a negative regulator of myeloid cell proliferation and survival. Mutations in this gene are associated with defects and cancers of the immune system. Deficiencies in the encoded protein, SHIP1, have been associated with Inflammatory Bowel Disease types such as Crohn\'s Disease and Ulcerative Colitis. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jul 2020]', '2', 'protein_coding', 58385345, 58399220, 37.47, 4, 'ENSG00000168918', 'Q92835'),
(1207, 'INSM1', 'Insulinoma-associated 1 (INSM1) gene is intronless and encodes a protein containing both a zinc finger DNA-binding domain and a putative prohormone domain. This gene is a sensitive marker for neuroendocrine differentiation of human lung tumors. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 48274364, 48274415, 38.46, 1, 'ENSG00000173404', 'Q01101'),
(1208, 'INSM2', 'Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in negative regulation of transcription by RNA polymerase II; neuron differentiation; and regulation of cell cycle process. Predicted to be located in cytoplasm. Predicted to be part of transcription repressor complex. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 266611, 268581, 49.06, 1, 'ENSG00000168348', 'Q96T92'),
(1209, 'INSR', 'This gene encodes a member of the receptor tyrosine kinase family of proteins. The encoded preproprotein is proteolytically processed to generate alpha and beta subunits that form a heterotetrameric receptor. Binding of insulin or other ligands to this receptor activates the insulin signaling pathway, which regulates glucose uptake and release, as well as the synthesis and storage of carbohydrates, lipids and protein. Mutations in this gene underlie the inherited severe insulin resistance syndromes including type A insulin resistance syndrome, Donohue syndrome and Rabson-Mendenhall syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2015]', '19', 'processed_pseudogene', 27945260, 27945415, 42.95, 1, 'ENSG00000171105', 'P06213'),
(1210, 'INTS3', 'The protein encoded by this gene can form a complex with human single-strand DNA binding proteins 1 or 2 (hSSB1 and hSSB2) and other proteins to mediate genome stability and the DNA damage response. The encoded protein is also part of a multiprotein complex that interacts with the C-terminal domain of RNA polymerase II large subunit to help regulate processing of U1 and U2 small nuclear RNAs. [provided by RefSeq, May 2016]', '1', 'miRNA', 1734985, 1735066, 63.41, 1, 'ENSG00000262826', 'Q68E01'),
(1211, 'INTS3', 'The protein encoded by this gene can form a complex with human single-strand DNA binding proteins 1 or 2 (hSSB1 and hSSB2) and other proteins to mediate genome stability and the DNA damage response. The encoded protein is also part of a multiprotein complex that interacts with the C-terminal domain of RNA polymerase II large subunit to help regulate processing of U1 and U2 small nuclear RNAs. [provided by RefSeq, May 2016]', '1', 'miRNA', 1734985, 1735066, 63.41, 1, 'ENSG00000143624', 'Q68E01'),
(1212, 'INTS6-AS1', 'Predicted to be involved in SRP-dependent cotranslational protein targeting to membrane, signal sequence recognition. Predicted to be part of signal recognition particle, endoplasmic reticulum targeting. [provided by Alliance of Genome Resources, May 2025]', '13', 'miRNA', 20502265, 20502342, 46.15, 1, 'ENSG00000236778', ''),
(1213, 'INVS', 'This gene encodes a protein containing multiple ankyrin domains and two IQ calmodulin-binding domains. The encoded protein may function in renal tubular development and function, and in left-right axis determination. This protein interacts with nephrocystin and infers a connection between primary cilia function and left-right axis determination. A similar protein in mice interacts with calmodulin. Mutations in this gene have been associated with nephronophthisis type 2. Multiple transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, May 2012]', '9', 'protein_coding', 74999024, 75110994, 45.85, 11, 'ENSG00000119509', 'Q9Y283'),
(1214, 'IPO11', 'Importins, including IPO11, are a members of the karyopherin/importin-beta family of transport receptors (see KPNB1; 602738) that mediate nucleocytoplasmic transport of protein and RNA cargoes (Plafker and Macara, 2000 [PubMed 11032817]).[supplied by OMIM, Sep 2008]', '5', 'protein_coding', 86958599, 86963258, 61.44, 4, 'ENSG00000086200', 'Q9UI26'),
(1215, 'IQCA1', 'The protein encoded by this gene is a member of the ATPases Associated with diverse cellular Activities (AAA) superfamily. Members of this superfamily, found in all organisms, participate in a large number of cellular processes and contain the ATPase module consisting of an alpha-beta-alpha core domain and the Walker A and B motifs of the P-loop NTPases. Alternative splicing results in multiple transcript variants that encode different protein isoforms. [provided by RefSeq, Jul 2012]', '2', 'protein_coding', 2801008, 2835445, 49.36, 4, 'ENSG00000132321', 'Q86XH1'),
(1216, 'IQGAP3', 'Enables calmodulin binding activity and myosin VI light chain binding activity. Predicted to be involved in mitotic actomyosin contractile ring assembly actin filament organization. Predicted to act upstream of or within several processes, including G1/S transition of mitotic cell cycle; intracellular signaling cassette; and regulation of macromolecule metabolic process. Predicted to be located in cytosol. Predicted to be active in cell cortex. [provided by Alliance of Genome Resources, May 2025]', '1', 'snoRNA', 17962710, 17962946, 49.79, 1, 'ENSG00000291754', 'Q86VI3'),
(1217, 'IQGAP3', 'Enables calmodulin binding activity and myosin VI light chain binding activity. Predicted to be involved in mitotic actomyosin contractile ring assembly actin filament organization. Predicted to act upstream of or within several processes, including G1/S transition of mitotic cell cycle; intracellular signaling cassette; and regulation of macromolecule metabolic process. Predicted to be located in cytosol. Predicted to be active in cell cortex. [provided by Alliance of Genome Resources, May 2025]', '1', 'snoRNA', 17962710, 17962946, 49.79, 1, 'ENSG00000183856', 'Q86VI3'),
(1218, 'IQUB', 'Involved in flagellated sperm motility and mating. Predicted to be located in sperm flagellum. Predicted to be part of radial spoke. Predicted to be active in acrosomal vesicle and motile cilium. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 134546614, 134546711, 51.02, 1, 'ENSG00000164675', 'Q8NA54'),
(1219, 'IRF1', 'The protein encoded by this gene is a transcriptional regulator and tumor suppressor, serving as an activator of genes involved in both innate and acquired immune responses. The encoded protein activates the transcription of genes involved in the body\'s response to viruses and bacteria, playing a role in cell proliferation, apoptosis, the immune response, and DNA damage response. This protein represses the transcription of several other genes. As a tumor suppressor, it both suppresses tumor cell growth and stimulates an immune response against tumor cells. Defects in this gene have been associated with gastric cancer, myelogenous leukemia, and lung cancer. [provided by RefSeq, Aug 2017]', '5', 'protein_coding', 258363, 275291, 49.60, 28, 'ENSG00000125347', 'P10914'),
(1220, 'IRF3', 'This gene encodes a member of the interferon regulatory transcription factor (IRF) family. The encoded protein is found in an inactive cytoplasmic form that upon serine/threonine phosphorylation forms a complex with CREBBP. This complex translocates to the nucleus and activates the transcription of interferons alpha and beta, as well as other interferon-induced genes. The protein plays an important role in the innate immune response against DNA and RNA viruses. Mutations in this gene are associated with Encephalopathy, acute, infection-induced, herpes-specific, 7. [provided by RefSeq, Sep 2020]', '19', 'protein_coding', 66552563, 66566251, 41.69, 8, 'ENSG00000126456', 'Q14653'),
(1221, 'IRS4', 'IRS4 encodes the insulin receptor substrate 4, a cytoplasmic protein that contains many potential tyrosine and serine/threonine phosphorylation sites. Tyrosine-phosphorylated IRS4 protein has been shown to associate with cytoplasmic signalling molecules that contain SH2 domains. The IRS4 protein is phosphorylated by the insulin receptor tyrosine kinase upon receptor stimulation.. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 159800249, 159820704, 48.03, 7, 'ENSG00000133124', 'O14654'),
(1222, 'ISCA2', 'The protein encoded by this gene is an A-type iron-sulfur cluster (ISC) protein found in mitochondria. The encoded protein appears to be involved in the maturation of mitochondrial iron-sulfur proteins. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2012]', '14', 'protein_coding', 12116025, 12267044, 38.54, 7, 'ENSG00000165898', 'Q86U28'),
(1223, 'ISCU', 'This gene encodes a component of the iron-sulfur (Fe-S) cluster scaffold. Fe-S clusters are cofactors that play a role in the function of a diverse set of enzymes, including those that regulate metabolism, iron homeostasis, and oxidative stress response. Alternative splicing results in transcript variants encoding different protein isoforms that localize either to the cytosol or to the mitochondrion. Mutations in this gene have been found in patients with hereditary myopathy with lactic acidosis. A disease-associated mutation in an intron may activate a cryptic splice site, resulting in the production of a splice variant encoding a putatively non-functional protein. A pseudogene of this gene is present on chromosome 1. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 248805, 253362, 56.93, 8, 'ENSG00000136003', 'Q9H1K1'),
(1224, 'ISX', 'Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This gene is a member of the RAXLX homeobox gene family. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 39399778, 39437060, 51.35, 7, 'ENSG00000175329', 'Q2M1V0'),
(1225, 'ITCH', 'This gene encodes a member of the Nedd4 family of HECT domain E3 ubiquitin ligases. HECT domain E3 ubiquitin ligases transfer ubiquitin from E2 ubiquitin-conjugating enzymes to protein substrates, thus targeting specific proteins for lysosomal degradation. The encoded protein plays a role in multiple cellular processes including erythroid and lymphoid cell differentiation and the regulation of immune responses. Mutations in this gene are a cause of syndromic multisystem autoimmune disease. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Mar 2012]', '20', 'protein_coding', 113390515, 113685923, 37.14, 6, 'ENSG00000078747', 'Q96J02'),
(1226, 'ITGA2', 'This gene encodes the alpha subunit of a transmembrane receptor for collagens and related proteins. The encoded protein forms a heterodimer with a beta subunit and mediates the adhesion of platelets and other cell types to the extracellular matrix. Loss of the encoded protein is associated with bleeding disorder platelet-type 9. Antibodies against this protein are found in several immune disorders, including neonatal alloimmune thrombocytopenia. This gene is located adjacent to a related alpha subunit gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '5', 'protein_coding', 4340251, 4420494, 57.43, 3, 'ENSG00000164171', 'P17301'),
(1227, 'ITGAD', 'This gene belongs to the beta-2 integrin family of membrane glycoproteins, which are are composed of non-covalently linked alpha and beta subunits to form a heterodimer. It encodes the alpha subunit of the cell surface heterodimers and is involved in the activation and adhesion functions of leukocytes. The gene is located about 11kb downstream of the integrin subunit alpha X gene, another member of the integrin family. It is expressed in the tissue and circulating myeloid leukocytes. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2015]', '16', 'protein_coding', 35369, 39237, 49.73, 1, 'ENSG00000156886', 'Q13349');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1228, 'ITGAL', 'ITGAL encodes the integrin alpha L chain. Integrins are heterodimeric integral membrane proteins composed of an alpha chain and a beta chain. This I-domain containing alpha integrin combines with the beta 2 chain (ITGB2) to form the integrin lymphocyte function-associated antigen-1 (LFA-1), which is expressed on all leukocytes. LFA-1 plays a central role in leukocyte intercellular adhesion through interactions with its ligands, ICAMs 1-3 (intercellular adhesion molecules 1 through 3), and also functions in lymphocyte costimulatory signaling. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000005844', 'P20701'),
(1229, 'ITGAM', 'This gene encodes the integrin alpha M chain. Integrins are heterodimeric integral membrane proteins composed of an alpha chain and a beta chain. This I-domain containing alpha integrin combines with the beta 2 chain (ITGB2) to form a leukocyte-specific integrin referred to as macrophage receptor 1 (\'Mac-1\'), or inactivated-C3b (iC3b) receptor 3 (\'CR3\'). The alpha M beta 2 integrin is important in the adherence of neutrophils and monocytes to stimulated endothelium, and also in the phagocytosis of complement coated particles. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]', '16', 'protein_coding', 214931542, 215138626, 37.18, 3, 'ENSG00000169896', 'P11215'),
(1230, 'ITGBL1', 'This gene encodes a beta integrin-related protein that is a member of the EGF-like protein family. The encoded protein contains integrin-like cysteine-rich repeats. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2012]', '13', 'protein_coding', 47322454, 47327588, 51.65, 1, 'ENSG00000198542', 'O95965'),
(1231, 'ITIH5', 'This gene encodes a heavy chain component of one of the inter-alpha-trypsin inhibitor (ITI) family members. ITI proteins are involved in extracellular matrix stabilization and in the prevention of tumor metastasis. They are also structurally related plasma serine protease inhibitors and are composed of a light chain and varying numbers of heavy chains. This family member is thought to function as a tumor suppressor in breast and thyroid cancers. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]', '10', 'protein_coding', 103043998, 103066417, 39.40, 1, 'ENSG00000123243', 'Q86UX2'),
(1232, 'ITIH6', 'The protein encoded by this gene belongs to the interalpha trypsin inhibitor heavy chain (ITIH) family. Interalpha trypsin inhibitor (ITI) is composed of two heavy chains (containing VWA domain) and one light chain. The light chain confers the protease-inhibitor function, while the heavy chains are involved in mediating protein-protein interactions with the components of the extracellular matrix. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 63286808, 63286866, 64.41, 1, 'ENSG00000102313', 'Q6UXX5'),
(1233, 'ITM2B', 'Amyloid precursor proteins are processed by beta-secretase and gamma-secretase to produce beta-amyloid peptides which form the characteristic plaques of Alzheimer disease. This gene encodes a transmembrane protein which is processed at the C-terminus by furin or furin-like proteases to produce a small secreted peptide which inhibits the deposition of beta-amyloid. Mutations which result in extension of the C-terminal end of the encoded protein, thereby increasing the size of the secreted peptide, are associated with two neurogenerative diseases, familial British dementia and familial Danish dementia. [provided by RefSeq, Oct 2009]', '13', 'protein_coding', 23823769, 23826729, 56.64, 1, 'ENSG00000136156', 'Q9Y287'),
(1234, 'ITSN1', 'The protein encoded by this gene is a cytoplasmic membrane-associated protein that indirectly coordinates endocytic membrane traffic with the actin assembly machinery. In addition, the encoded protein may regulate the formation of clathrin-coated vesicles and could be involved in synaptic vesicle recycling. This protein has been shown to interact with dynamin, CDC42, SNAP23, SNAP25, SPIN90, EPS15, EPN1, EPN2, and STN2. Multiple transcript variants encoding different isoforms have been found for this gene, but the full-length nature of only two of them have been characterized so far. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 30240453, 30246759, 58.90, 2, 'ENSG00000205726', 'Q15811'),
(1235, 'JADE1', 'Enables transcription coactivator activity. Contributes to histone acetyltransferase activity. Involved in several processes, including negative regulation of canonical Wnt signaling pathway; regulation of cell growth; and regulation of nucleobase-containing compound metabolic process. Acts upstream of or within negative regulation of G1/S transition of mitotic cell cycle. Located in several cellular components, including cytosol; microtubule organizing center; and nuclear speck. Part of histone acetyltransferase complex. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 131026850, 131350465, 38.71, 17, 'ENSG00000077684', 'Q6IE81'),
(1236, 'JAG1', 'The jagged 1 protein encoded by JAG1 is the human homolog of the Drosophilia jagged protein. Human jagged 1 is the ligand for the receptor notch 1, the latter is involved in signaling processes. Mutations that alter the jagged 1 protein cause Alagille syndrome. Jagged 1 signalling through notch 1 has also been shown to play a role in hematopoiesis. [provided by RefSeq, Nov 2019]', '20', 'protein_coding', 2199, 18974, 47.07, 2, 'ENSG00000101384', 'P78504'),
(1237, 'JAG2', 'The Notch signaling pathway is an intercellular signaling mechanism that is essential for proper embryonic development. Members of the Notch gene family encode transmembrane receptors that are critical for various cell fate decisions. The protein encoded by this gene is one of several ligands that activate Notch and related receptors. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 128832942, 128882494, 49.05, 10, 'ENSG00000184916', 'Q9Y219'),
(1238, 'JAK2', 'This gene encodes a non-receptor tyrosine kinase that plays a central role in cytokine and growth factor signalling. The primary isoform of this protein has an N-terminal FERM domain that is required for erythropoietin receptor association, an SH2 domain that binds STAT transcription factors, a pseudokinase domain and a C-terminal tyrosine kinase domain. Cytokine binding induces autophosphorylation and activation of this kinase. This kinase then recruits and phosphorylates signal transducer and activator of transcription (STAT) proteins. Growth factors like TGF-beta 1 also induce phosphorylation and activation of this kinase and translocation of downstream STAT proteins to the nucleus where they influence gene transcription. Mutations in this gene are associated with numerous inflammatory diseases and malignancies. This gene is a downstream target of the pleiotropic cytokine IL6 that is produced by B cells, T cells, dendritic cells and macrophages to produce an immune response or inflammation. Disregulation of the IL6/JAK2/STAT3 signalling pathways produces increased cellular proliferation and myeloproliferative neoplasms of hematopoietic stem cells. A nonsynonymous mutation in the pseudokinase domain of this gene disrupts the domains inhibitory effect and results in constitutive tyrosine phosphorylation activity and hypersensitivity to cytokine signalling. This gene and the IL6/JAK2/STAT3 signalling pathway is a therapeutic target for the treatment of excessive inflammatory responses to viral infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2020]', '9', 'protein_coding', 137492199, 137494394, 54.64, 1, 'ENSG00000096968', 'O60674'),
(1239, 'JAKMIP2', 'The protein encoded by this gene is reported to be a component of the Golgi matrix. It may act as a golgin protein by negatively regulating transit of secretory cargo and by acting as a structural scaffold of the Golgi. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '5', 'protein_coding', 141636583, 141733987, 37.14, 8, 'ENSG00000176049', 'Q96AA8'),
(1240, 'JAM2', 'Tight junctions represent one mode of cell-to-cell adhesion in epithelial or endothelial cell sheets, forming continuous seals around cells and serving as a physical barrier to prevent solutes and water from passing freely through the paracellular space. The protein encoded by this immunoglobulin superfamily gene member is localized in the tight junctions between high endothelial cells. Unlike other proteins in this family, the this protein is unable to adhere to leukocyte cell lines and only forms weak homotypic interactions. The encoded protein is a member of the junctional adhesion molecule protein family and acts as a receptor for another member of this family. A mutation in an intron of this gene is associated with hemorrhagic destruction of the brain, subependymal calcification, and congenital cataracts. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Apr 2011]', '11', 'protein_coding', 41986831, 42010387, 42.80, 8, 'ENSG00000154721', 'P57087'),
(1241, 'JKAMP', 'Enables ubiquitin protein ligase binding activity. Involved in ERAD pathway. Predicted to be located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 41464300, 41531116, 49.10, 20, 'ENSG00000050130', 'Q9P055'),
(1242, 'JMY', 'Enables microtubule binding activity. Involved in cellular response to starvation. Located in autophagosome membrane; cell leading edge; and cytoplasmic vesicle. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 118280038, 118353003, 36.51, 4, 'ENSG00000152409', 'Q8N9B5'),
(1243, 'JPH2', 'Junctional complexes between the plasma membrane and endoplasmic/sarcoplasmic reticulum are a common feature of all excitable cell types and mediate cross talk between cell surface and intracellular ion channels. The protein encoded by this gene is a component of junctional complexes and is composed of a C-terminal hydrophobic segment spanning the endoplasmic/sarcoplasmic reticulum membrane and a remaining cytoplasmic domain that shows specific affinity for the plasma membrane. This gene is a member of the junctophilin gene family. Alternative splicing has been observed at this locus and two variants encoding distinct isoforms are described. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 514933, 533330, 44.50, 4, 'ENSG00000149596', 'Q9BR39'),
(1244, 'JPT1', 'Located in several cellular components, including intercellular bridge; microtubule cytoskeleton; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 103312289, 103344830, 40.48, 11, 'ENSG00000189159', 'Q9UK76'),
(1245, 'KAT14', 'CSRP2 is a protein containing two LIM domains, which are double zinc finger motifs found in proteins of diverse function. CSRP2 and some related proteins are thought to act as protein adapters, bridging two or more proteins to form a larger protein complex. The protein encoded by this gene binds to one of the LIM domains of CSRP2 and contains an acetyltransferase domain. Although the encoded protein has been detected in the cytoplasm, it is predominantly a nuclear protein. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jun 2011]', '20', 'snoRNA', 100945649, 100945720, 41.67, 1, 'ENSG00000149474', 'Q9H8E8'),
(1246, 'KAT2B', 'CBP and p300 are large nuclear proteins that bind to many sequence-specific factors involved in cell growth and/or differentiation, including c-jun and the adenoviral oncoprotein E1A. The protein encoded by this gene associates with p300/CBP. It has in vitro and in vivo binding activity with CBP and p300, and competes with E1A for binding sites in p300/CBP. It has histone acetyl transferase activity with core histones and nucleosome core particles, indicating that this protein plays a direct role in transcriptional regulation. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 147585438, 147782775, 38.28, 6, 'ENSG00000114166', 'Q92831'),
(1247, 'KAT8', 'This gene encodes a member of the MYST histone acetylase protein family. The encoded protein has a characteristic MYST domain containing an acetyl-CoA-binding site, a chromodomain typical of proteins which bind histones, and a C2HC-type zinc finger. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2012]', '16', 'protein_coding', 100584936, 100599885, 40.78, 2, 'ENSG00000103510', 'Q9H7Z6'),
(1248, 'KATNAL1', 'Enables identical protein binding activity and microtubule severing ATPase activity. Involved in microtubule severing. Located in cytosol; microtubule cytoskeleton; and mitochondrion. Part of katanin complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'miRNA', 54311972, 54312048, 32.47, 1, 'ENSG00000102781', 'Q9BW62'),
(1249, 'KAZALD1', 'This gene encodes a secreted member of the insulin growth factor-binding protein (IGFBP) superfamily. The protein contains an insulin growth factor-binding domain in its N-terminal region, a Kazal-type serine protease inhibitor and follistatin-like domain in its central region, and an immunoglobulin-like domain in its C-terminal region. Studies of the mouse ortholog suggest that this protein may function in bone development and bone regeneration. This gene is hypomethylated and over-expressed in high-grade glioma compared to low-grade glioma, and thus the hypomethylated gene may be associated with cell proliferation and the shorter survival of patients with high-grade glioma. It is also one of numerous genes found to be deleted in a novel 5.54 Mb interstitial deletion, which is associated with multiple congenital anomalies. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '10', 'miRNA', 21825698, 21825785, 44.32, 1, 'ENSG00000107821', 'Q96I82'),
(1250, 'KCNC2', 'The Shaker gene family of Drosophila encodes components of voltage-gated potassium channels and is comprised of four subfamilies. Based on sequence similarity, this gene is similar to one of these subfamilies, namely the Shaw subfamily. The protein encoded by this gene belongs to the delayed rectifier class of channel proteins and is an integral membrane protein that mediates the voltage-dependent potassium ion permeability of excitable membranes. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2012]', '12', 'protein_coding', 94376747, 94390693, 51.80, 20, 'ENSG00000166006', 'Q96PR1'),
(1251, 'KCNE5', 'This gene encodes a member of a family of single pass transmembrane domain proteins that function as ancillary subunits to voltage-gated potassium channels. Members of this family affect diverse processes in potassium channel regulation, including ion selectivity, voltage dependence, and anterograde recycling from the plasma membrane. Variants of this gene are associated with idiopathic ventricular fibrillation and Brugada syndrome. [provided by RefSeq, Nov 2016]', 'X', 'protein_coding', 2236418, 2263520, 41.45, 4, 'ENSG00000176076', 'Q9UJ90'),
(1252, 'KCNJ14', 'Potassium channels are present in most mammalian cells, where they participate in a wide range of physiologic responses. The protein encoded by this gene is an integral membrane protein and inward-rectifier type potassium channel, and probably has a role in controlling the excitability of motor neurons. [provided by RefSeq, Feb 2013]', '19', 'protein_coding', 20115938, 20127355, 57.68, 13, 'ENSG00000182324', 'Q99712'),
(1253, 'KCNJ3', 'Potassium channels are present in most mammalian cells, where they participate in a wide range of physiologic responses. The protein encoded by this gene is an integral membrane protein and inward-rectifier type potassium channel. The encoded protein, which has a greater tendency to allow potassium to flow into a cell rather than out of a cell, is controlled by G-proteins and plays an important role in regulating heartbeat. It associates with three other G-protein-activated potassium channels to form a heteromultimeric pore-forming complex that also couples to neurotransmitter receptors in the brain and whereby channel activation can inhibit action potential firing by hyperpolarizing the plasma membrane. These multimeric G-protein-gated inwardly-rectifying potassium (GIRK) channels may play a role in the pathophysiology of epilepsy, addiction, Down\'s syndrome, ataxia, and Parkinson\'s disease. Alternative splicing results in multiple transcript variants encoding distinct proteins. [provided by RefSeq, May 2012]', '2', 'miRNA', 154347106, 154347247, 45.77, 1, 'ENSG00000162989', 'P48549'),
(1254, 'KCNK15', 'This gene encodes one of the members of the superfamily of potassium channel proteins containing two pore-forming P domains. The product of this gene has not been shown to be a functional channel, however, it may require other non-pore-forming proteins for activity. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 101309450, 101309534, 41.18, 1, 'ENSG00000124249', 'Q9H427'),
(1255, 'KCNK16', 'The protein encoded by this gene belongs to the family of potassium channel proteins containing two pore-forming P domains. This channel is an open rectifier which primarily passes outward current under physiological K+ concentrations. This gene is expressed predominantly in the pancreas and is activated at alkaline pH. Several alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Sep 2008]', '6', 'protein_coding', 73674287, 73698109, 42.19, 3, 'ENSG00000095981', 'Q96T55'),
(1256, 'KCNMB1', 'MaxiK channels are large conductance, voltage and calcium-sensitive potassium channels which are fundamental to the control of smooth muscle tone and neuronal excitability. MaxiK channels can be formed by 2 subunits: the pore-forming alpha subunit and the product of this gene, the modulatory beta subunit. Intracellular calcium regulates the physical association between the alpha and beta subunits. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 31706866, 31710679, 53.38, 1, 'ENSG00000145936', 'Q16558'),
(1257, 'KCNMB4', 'MaxiK channels are large conductance, voltage and calcium-sensitive potassium channels which are fundamental to the control of smooth muscle tone and neuronal excitability. MaxiK channels can be formed by 2 subunits: the pore-forming alpha subunit and the modulatory beta subunit. The protein encoded by this gene is an auxiliary beta subunit which slows activation kinetics, leads to steeper calcium sensitivity, and shifts the voltage range of current activation to more negative potentials than does the beta 1 subunit. [provided by RefSeq, Jul 2008]', '12', 'miRNA', 27920526, 27920611, 50.00, 1, 'ENSG00000135643', 'Q86W47'),
(1258, 'KCNQ1DN', 'Imprinting is a phenomenon in which epigenetic modifications lead to expression or suppression of alleles of some genes based on their parental origin. Wilms tumor-2 (WT2; MIM 194071) is defined by maternal-specific loss of heterozygosity of a critical region on chromosome 11p15.5 that includes several imprinted genes. KCNQ1DN is an imprinted gene located within the WT2 critical region that is expressed from the maternal allele (Xin et al., 2000 [PubMed 11056398]).[supplied by OMIM, Mar 2008]', '11', 'protein_coding', 132537787, 132783480, 54.13, 7, 'ENSG00000278855', 'Q9H478'),
(1259, 'KCNQ1DN', 'Imprinting is a phenomenon in which epigenetic modifications lead to expression or suppression of alleles of some genes based on their parental origin. Wilms tumor-2 (WT2; MIM 194071) is defined by maternal-specific loss of heterozygosity of a critical region on chromosome 11p15.5 that includes several imprinted genes. KCNQ1DN is an imprinted gene located within the WT2 critical region that is expressed from the maternal allele (Xin et al., 2000 [PubMed 11056398]).[supplied by OMIM, Mar 2008]', '11', 'protein_coding', 132537787, 132783480, 54.13, 7, 'ENSG00000237941', 'Q9H478'),
(1260, 'KCNS3', 'Voltage-gated potassium channels form the largest and most diversified class of ion channels and are present in both excitable and nonexcitable cells. Their main functions are associated with the regulation of the resting membrane potential and the control of the shape and frequency of action potentials. The alpha subunits are of 2 types: those that are functional by themselves and those that are electrically silent but capable of modulating the activity of specific functional alpha subunits. The protein encoded by this gene is not functional by itself but can form heteromultimers with member 1 and with member 2 (and possibly other members) of the Shab-related subfamily of potassium voltage-gated channel proteins. This gene belongs to the S subfamily of the potassium channel family. Alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Sep 2013]', '2', 'protein_coding', 53491040, 53504411, 45.45, 14, 'ENSG00000170745', 'Q9BQ31'),
(1261, 'KCTD1', 'This gene encodes a protein containing a BTB (Broad-complex, tramtrack and bric a brac), also known as a POZ (POxvirus and zinc finger) protein-protein interaction domain. The encoded protein negatively regulates the AP-2 family of transcription factors and the Wnt signaling pathway. A mechanism for the modulation of Wnt signaling has been proposed in which the encoded protein enhances ubiquitination and degradation of the beta-catenin protein. Mutations in this gene have been identified in Scalp-ear-nipple (SEN) syndrome. [provided by RefSeq, May 2017]', '18', 'protein_coding', 48232612, 48270357, 37.65, 11, 'ENSG00000134504', 'Q719H9'),
(1262, 'KDELR2', 'Retention of resident soluble proteins in the lumen of the endoplasmic reticulum (ER) is achieved in both yeast and animal cells by their continual retrieval from the cis-Golgi, or a pre-Golgi compartment. Sorting of these proteins is dependent on a C-terminal tetrapeptide signal, usually lys-asp-glu-leu (KDEL) in animal cells, and his-asp-glu-leu (HDEL) in S. cerevisiae. This process is mediated by a receptor that recognizes, and binds the tetrapeptide-containing protein, and returns it to the ER. In yeast, the sorting receptor encoded by a single gene, ERD2, is a seven-transmembrane protein. Unlike yeast, several human homologs of the ERD2 gene, constituting the KDEL receptor gene family, have been described. KDELR2 was the second member of the family to be identified, and it encodes a protein which is 83% identical to the KDELR1 gene product. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 64501214, 64501313, 48.00, 1, 'ENSG00000136240', 'P33947'),
(1263, 'KDM1B', 'Flavin-dependent histone demethylases, such as KDM1B, regulate histone lysine methylation, an epigenetic mark that regulates gene expression and chromatin function (Karytinos et al., 2009 [PubMed 19407342]).[supplied by OMIM, Oct 2009]', '6', 'protein_coding', 111726908, 111766389, 44.53, 12, 'ENSG00000165097', 'Q8NB78'),
(1264, 'KDM4A', 'This gene is a member of the Jumonji domain 2 (JMJD2) family and encodes a protein containing a JmjN domain, a JmjC domain, a JD2H domain, two TUDOR domains, and two PHD-type zinc fingers. This nuclear protein functions as a trimethylation-specific demethylase, converting specific trimethylated histone residues to the dimethylated form, and as a transcriptional repressor. [provided by RefSeq, Apr 2009]', '1', 'protein_coding', 29268009, 29300525, 44.02, 22, 'ENSG00000066135', 'O75164'),
(1265, 'KDM8', 'This gene likely encodes a histone lysine demethylase. Studies of a similar protein in mouse indicate a potential role for this protein as a tumor suppressor. Alternatively spliced transcript variants have been described.[provided by RefSeq, Feb 2009]', '16', 'protein_coding', 29672483, 29681155, 48.48, 2, 'ENSG00000155666', 'Q8N371'),
(1266, 'KHNYN', 'The protein encoded by this gene contains a ribonuclease NYN domain and belongs to the N4BP1 family. The protein is a cofactor for the zinc finger antiviral protein (ZAP protein) which targets viral RNA for degradation and restricts SARS-CoV-2 infection. [provided by RefSeq, Sep 2021]', '14', 'protein_coding', 36487190, 36507221, 52.87, 18, 'ENSG00000100441', 'O15037'),
(1267, 'KIAA1210', 'Predicted to be located in acrosomal vesicle and basal ectoplasmic specialization. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 47078355, 47093314, 43.38, 6, 'ENSG00000250423', 'Q9ULL0'),
(1268, 'KIAA1671', 'Located in cilium; intercellular bridge; and microtubule cytoskeleton. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 93561741, 93847150, 45.08, 12, 'ENSG00000197077', 'Q9BY89'),
(1269, 'KIF11', 'This gene encodes a motor protein that belongs to the kinesin-like protein family. Members of this protein family are known to be involved in various kinds of spindle dynamics. The function of this gene product includes chromosome positioning, centrosome separation and establishing a bipolar spindle during cell mitosis. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 11280841, 11281330, 55.31, 1, 'ENSG00000138160', 'P52732'),
(1270, 'KIF18B', 'Enables cytoskeletal motor activity and kinesin binding activity. Involved in microtubule depolymerization; mitotic cell cycle; and regulation of cell division. Located in cytosol; microtubule; and nuclear lumen. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 14719724, 14722733, 40.20, 1, 'ENSG00000186185', 'Q86Y91'),
(1271, 'KIF26B', 'The protein encoded by this gene is an intracellular motor protein thought to transport organelles along microtubules. The encoded protein is required for kidney development. Elevated levels of this protein have been found in some breast and colorectal cancers. [provided by RefSeq, Mar 2017]', '1', 'miRNA', 36919357, 36919432, 48.68, 1, 'ENSG00000162849', 'Q2KJY2'),
(1272, 'KIF26B', 'The protein encoded by this gene is an intracellular motor protein thought to transport organelles along microtubules. The encoded protein is required for kidney development. Elevated levels of this protein have been found in some breast and colorectal cancers. [provided by RefSeq, Mar 2017]', '1', 'miRNA', 36919357, 36919432, 48.68, 1, 'ENSG00000281216', 'Q2KJY2'),
(1273, 'KIF3A', 'Enables protein phosphatase binding activity; small GTPase binding activity; and spectrin binding activity. Involved in protein localization to cell junction and protein transport. Located in centriole; centrosome; and spindle microtubule. Part of kinesin II complex. Biomarker of silicosis. [provided by Alliance of Genome Resources, May 2025]', '5', 'unprocessed_pseudogene', 55444435, 55445338, 40.93, 1, 'ENSG00000131437', 'Q9Y496'),
(1274, 'KIF5B', 'Enables identical protein binding activity; microtubule binding activity; and microtubule motor activity. Involved in several processes, including natural killer cell mediated cytotoxicity; positive regulation of protein localization to plasma membrane; and vesicle transport along microtubule. Located in several cellular components, including centriolar satellite; cytosol; and nuclear membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 87504875, 87553461, 38.05, 4, 'ENSG00000170759', 'P33176'),
(1275, 'KIF6', 'This gene encodes a member of a family of molecular motors which are involved in intracellular transport of protein complexes, membrane organelles, and messenger ribonucleic acid along microtubules. Kinesins function as homodimeric molecules with two N-terminal head domains that move along microtubules and two C-terminal tail domains that interact with the transported cargo, either directly or indirectly, through adapter molecules. This gene is ubiquitously expressed in coronary arteries and other vascular tissue. A naturally occurring mutation in this gene is associated with coronary heart disease. [provided by RefSeq, May 2017]', '6', 'protein_coding', 56651343, 57029811, 36.24, 12, 'ENSG00000164627', 'Q6ZMV9'),
(1276, 'KIN', 'The protein encoded by this gene is a nuclear protein that forms intranuclear foci during proliferation and is redistributed in the nucleoplasm during the cell cycle. Short-wave ultraviolet light provokes the relocalization of the protein, suggesting its participation in the cellular response to DNA damage. Originally selected based on protein-binding with RecA antibodies, the mouse protein presents a limited similarity with a functional domain of the bacterial RecA protein, a characteristic shared by this human ortholog. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jan 2012]', '10', 'protein_coding', 55623400, 55626192, 40.92, 1, 'ENSG00000151657', 'O60870'),
(1277, 'KIR2DL5B', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'miRNA', 64891355, 64891439, 62.35, 1, 'ENSG00000274707', 'Q8NHK3'),
(1278, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000277885', 'P43631'),
(1279, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000275452', 'P43631'),
(1280, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000276139', 'P43631'),
(1281, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000275737', 'P43631'),
(1282, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000278152', 'P43631'),
(1283, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000274518', 'P43631'),
(1284, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000276425', 'P43631'),
(1285, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000275735', 'P43631'),
(1286, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000278300', 'P43631'),
(1287, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000276258', 'P43631'),
(1288, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000274438', 'P43631'),
(1289, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000284033', 'P43631'),
(1290, 'KIR2DS2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene represents a haplotype-specific family member that encodes a protein with a short cytoplasmic tail. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '19', 'miRNA', 101027100, 101027183, 41.67, 1, 'ENSG00000275253', 'P43631');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1291, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000288206', 'Q14953'),
(1292, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000274563', 'Q14953'),
(1293, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000277650', 'Q14953'),
(1294, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000274243', 'Q14953'),
(1295, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000275047', 'Q14953'),
(1296, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000274739', 'Q14953'),
(1297, 'KIR2DS5', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36306336, 36528637, 51.09, 15, 'ENSG00000276676', 'Q14953'),
(1298, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000288389', 'P43630'),
(1299, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000284213', 'P43630'),
(1300, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000284192', 'P43630'),
(1301, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278809', 'P43630'),
(1302, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278726', 'P43630'),
(1303, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000284295', 'P43630'),
(1304, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000284466', 'P43630'),
(1305, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275629', 'P43630'),
(1306, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000276004', 'P43630'),
(1307, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000276882', 'P43630'),
(1308, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000274722', 'P43630'),
(1309, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278710', 'P43630'),
(1310, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278474', 'P43630'),
(1311, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278656', 'P43630'),
(1312, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278850', 'P43630'),
(1313, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000277982', 'P43630'),
(1314, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275416', 'P43630'),
(1315, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278403', 'P43630'),
(1316, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000273735', 'P43630'),
(1317, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275262', 'P43630'),
(1318, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000276357', 'P43630'),
(1319, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278758', 'P43630'),
(1320, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275511', 'P43630'),
(1321, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278707', 'P43630'),
(1322, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000273911', 'P43630'),
(1323, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275838', 'P43630'),
(1324, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278442', 'P43630'),
(1325, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000277709', 'P43630');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1326, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000276739', 'P43630'),
(1327, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275566', 'P43630'),
(1328, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000277181', 'P43630'),
(1329, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000276424', 'P43630'),
(1330, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275083', 'P43630'),
(1331, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000278361', 'P43630'),
(1332, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000284528', 'P43630'),
(1333, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000275626', 'P43630'),
(1334, 'KIR3DL2', 'Killer cell immunoglobulin-like receptors (KIRs) are transmembrane glycoproteins expressed by natural killer cells and subsets of T cells. The KIR genes are polymorphic and highly homologous and they are found in a cluster on chromosome 19q13.4 within the 1 Mb leukocyte receptor complex (LRC). The gene content of the KIR gene cluster varies among haplotypes, although several \"framework\" genes are found in all haplotypes (KIR3DL3, KIR3DP1, KIR3DL4, KIR3DL2). The KIR proteins are classified by the number of extracellular immunoglobulin domains (2D or 3D) and by whether they have a long (L) or short (S) cytoplasmic domain. KIR proteins with the long cytoplasmic domain transduce inhibitory signals upon ligand binding via an immune tyrosine-based inhibitory motif (ITIM), while KIR proteins with the short cytoplasmic domain lack the ITIM motif and instead associate with the TYRO protein tyrosine kinase binding protein to transduce activating signals. The ligands for several KIR proteins are subsets of HLA class I molecules; thus, KIR proteins are thought to play an important role in regulation of the immune response. This gene is one of the \"framework\" loci that is present on all haplotypes. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]', '19', 'protein_coding', 43879678, 43892013, 57.39, 3, 'ENSG00000240403', 'P43630'),
(1335, 'KLB', 'Enables fibroblast growth factor binding activity and fibroblast growth factor receptor binding activity. Predicted to be involved in carbohydrate metabolic process. Predicted to act upstream of or within positive regulation of MAPKKK cascade by fibroblast growth factor receptor signaling pathway and positive regulation of cell population proliferation. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 133247511, 133247620, 58.18, 1, 'ENSG00000134962', 'Q86Z14'),
(1336, 'KLC4', 'Predicted to enable kinesin binding activity. Predicted to be involved in microtubule-based movement. Predicted to be located in microtubule. Predicted to be part of kinesin complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'IG_C_gene', 22895375, 22895834, 58.26, 1, 'ENSG00000137171', 'Q9NSK0'),
(1337, 'KLF4', 'This gene encodes a protein that belongs to the Kruppel family of transcription factors. The encoded zinc finger protein is required for normal development of the barrier function of skin. The encoded protein is thought to control the G1-to-S transition of the cell cycle following DNA damage by mediating the tumor suppressor gene p53. Mice lacking this gene have a normal appearance but lose weight rapidly, and die shortly after birth due to fluid evaporation resulting from compromised epidermal barrier function. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Sep 2015]', '9', 'protein_coding', 86876388, 86991149, 36.27, 4, 'ENSG00000136826', 'O43474'),
(1338, 'KLF9', 'The protein encoded by this gene is a transcription factor that binds to GC box elements located in the promoter. Binding of the encoded protein to a single GC box inhibits mRNA expression while binding to tandemly repeated GC box elements activates transcription. [provided by RefSeq, Jul 2008]', '9', 'unprocessed_pseudogene', 56775261, 56776206, 45.03, 1, 'ENSG00000119138', 'Q13886'),
(1339, 'KLHL24', 'The protein encoded by this gene is a ubiquitin ligase substrate receptor and is regulated by autoubiquitination. Variations in the translation initiation codon of this gene have been found, which result in an N-terminally truncated but more stable protein due to loss of the autoubiquitination function. The more stable mutant protein causes an increased ubiquitin and degradation of keratin 14, which leads to skin fragility and the potentially life-threatening disease epidermolysis bullosa. The encoded protein is also involved in the regulation of kainate receptors. [provided by RefSeq, Mar 2017]', '3', 'protein_coding', 71964, 86902, 41.60, 9, 'ENSG00000114796', 'Q6TFL4'),
(1340, 'KLHL34', 'Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 44147872, 44343672, 39.92, 2, 'ENSG00000185915', 'Q8N239'),
(1341, 'KLHL36', 'Enables cullin family protein binding activity. Predicted to be involved in protein ubiquitination. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 69504326, 69532508, 50.05, 1, 'ENSG00000135686', 'Q8N4N3'),
(1342, 'KLHL42', 'Contributes to ubiquitin-protein transferase activity. Involved in proteasome-mediated ubiquitin-dependent protein catabolic process; protein polyubiquitination; and regulation of microtubule-based process. Part of Cul3-RING ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 4368227, 4379712, 44.67, 2, 'ENSG00000087448', 'Q9P2K6'),
(1343, 'KLHL8', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Involved in protein ubiquitination and ubiquitin-dependent protein catabolic process. Located in nucleoplasm. Part of Cul3-RING ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 123994163, 123995161, 50.55, 1, 'ENSG00000145332', 'Q9P2G9'),
(1344, 'KLK1', 'Kallikreins are a subgroup of serine proteases having diverse physiological functions. Growing evidence suggests that many kallikreins are implicated in carcinogenesis and some have potential as novel cancer and other disease biomarkers. This gene is one of the fifteen kallikrein subfamily members located in a cluster on chromosome 19. This protein is functionally conserved in its capacity to release the vasoactive peptide, Lys-bradykinin, from low molecular weight kininogen. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 36657568, 36679362, 43.60, 6, 'ENSG00000167748', 'P06870'),
(1345, 'KLK2', 'This gene encodes a member of the grandular kallikrein protein family. Kallikreins are a subgroup of serine proteases that are clustered on chromosome 19. Members of this family are involved in a diverse array of biological functions. The protein encoded by this gene is a highly active trypsin-like serine protease that selectively cleaves at arginine residues. This protein is primarily expressed in prostatic tissue and is responsible for cleaving pro-prostate-specific antigen into its enzymatically active form. This gene is highly expressed in prostate tumor cells and may be a prognostic maker for prostate cancer risk. Alternate splicing results in both coding and non-coding transcript variants. [provided by RefSeq, Jan 2012]', '19', 'protein_coding', 546786, 550554, 45.95, 3, 'ENSG00000167751', 'P20151'),
(1346, 'KLLN', 'The protein encoded by this intronless gene is found in the nucleus, where it can inhibit DNA synthesis and promote S phase arrest coupled to apoptosis. The expression of this DNA binding protein is upregulated by transcription factor p53. [provided by RefSeq, Dec 2012]', '10', 'protein_coding', 37905932, 38025695, 39.07, 3, 'ENSG00000283059', 'B2CW77'),
(1347, 'KLLN', 'The protein encoded by this intronless gene is found in the nucleus, where it can inhibit DNA synthesis and promote S phase arrest coupled to apoptosis. The expression of this DNA binding protein is upregulated by transcription factor p53. [provided by RefSeq, Dec 2012]', '10', 'protein_coding', 37905932, 38025695, 39.07, 3, 'ENSG00000227268', 'B2CW77'),
(1348, 'KMT5C', 'SUV420H2 and the related enzyme SUV420H1 (MIM 610881) function as histone methyltransferases that specifically trimethylate nucleosomal histone H4 (see MIM 602822) on lysine-20 (K20) (Schotta et al., 2004 [PubMed 15145825]).[supplied by OMIM, Dec 2009]', '19', 'protein_coding', 140855883, 141012347, 39.83, 4, 'ENSG00000133247', 'Q86Y97'),
(1349, 'KPNA3', 'The transport of molecules between the nucleus and the cytoplasm in eukaryotic cells is mediated by the nuclear pore complex (NPC), which consists of 60-100 proteins. Small molecules (up to 70 kD) can pass through the nuclear pore by nonselective diffusion while larger molecules are transported by an active process. The protein encoded by this gene belongs to the importin alpha family, and is involved in nuclear protein import. [provided by RefSeq, Jan 2009]', '13', 'protein_coding', 86388, 87368, 38.84, 1, 'ENSG00000102753', 'O00505'),
(1350, 'KPNB1', 'Nucleocytoplasmic transport, a signal- and energy-dependent process, takes place through nuclear pore complexes embedded in the nuclear envelope. The import of proteins containing a nuclear localization signal (NLS) requires the NLS import receptor, a heterodimer of importin alpha and beta subunits also known as karyopherins. Importin alpha binds the NLS-containing cargo in the cytoplasm and importin beta docks the complex at the cytoplasmic side of the nuclear pore complex. In the presence of nucleoside triphosphates and the small GTP binding protein Ran, the complex moves into the nuclear pore complex and the importin subunits dissociate. Importin alpha enters the nucleoplasm with its passenger protein and importin beta remains at the pore. Interactions between importin beta and the FG repeats of nucleoporins are essential in translocation through the pore complex. The protein encoded by this gene is a member of the importin beta family. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2013]', '17', 'protein_coding', 2030137, 2043898, 56.23, 18, 'ENSG00000108424', 'Q14974'),
(1351, 'KRBOX4', 'This encodes a zinc finger protein with an N-terminal KRAB (Kruppel-associated) domain found in transcriptional repressors. This gene is located in a region of the X chromosome thought to be involved in nonsyndromic X-linked cognitive disability. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 14810853, 14934571, 48.45, 7, 'ENSG00000147121', 'Q5JUW0'),
(1352, 'KREMEN1', 'This gene encodes a high-affinity dickkopf homolog 1 (DKK1) transmembrane receptor that functionally cooperates with DKK1 to block wingless (WNT)/beta-catenin signaling. The encoded protein is a component of a membrane complex that modulates canonical WNT signaling through lipoprotein receptor-related protein 6 (LRP6). It contains extracellular kringle, WSC, and CUB domains. Mutations in this gene result in ectodermal dysplasia. This protein has also been found to be a functional receptor for Coxsackievirus A10 and may be an alternative entry receptor for SARS-CoV-2. [provided by RefSeq, Nov 2021]', '22', 'protein_coding', 7688, 28425, 49.85, 5, 'ENSG00000183762', 'Q96MU8'),
(1353, 'KRT1', 'The protein encoded by this gene is a member of the keratin gene family. The type II cytokeratins consist of basic or neutral proteins which are arranged in pairs of heterotypic keratin chains coexpressed during differentiation of simple and stratified epithelial tissues. This type II cytokeratin is specifically expressed in the spinous and granular layers of the epidermis with family member KRT10 and mutations in these genes have been associated with bullous congenital ichthyosiform erythroderma. The type II cytokeratins are clustered in a region of chromosome 12q12-q13. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 108160812, 108222435, 35.23, 6, 'ENSG00000167768', 'P04264'),
(1354, 'KRT2', 'The protein encoded by this gene is a member of the keratin gene family. The type II cytokeratins consist of basic or neutral proteins which are arranged in pairs of heterotypic keratin chains coexpressed during differentiation of simple and stratified epithelial tissues. This type II cytokeratin is expressed largely in the upper spinous layer of epidermal keratinocytes and mutations in this gene have been associated with bullous congenital ichthyosiform erythroderma. The type II cytokeratins are clustered in a region of chromosome 12q12-q13. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 36150922, 36152449, 63.15, 4, 'ENSG00000172867', 'P35908'),
(1355, 'KRT20', 'The protein encoded by this gene is a member of the keratin family. The keratins are intermediate filament proteins responsible for the structural integrity of epithelial cells and are subdivided into cytokeratins and hair keratins. The type I cytokeratins consist of acidic proteins which are arranged in pairs of heterotypic keratin chains. This cytokeratin is a major cellular protein of mature enterocytes and goblet cells and is specifically expressed in the gastric and intestinal mucosa. The type I cytokeratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 94524204, 94656572, 35.35, 98, 'ENSG00000263057', 'P35900'),
(1356, 'KRT20', 'The protein encoded by this gene is a member of the keratin family. The keratins are intermediate filament proteins responsible for the structural integrity of epithelial cells and are subdivided into cytokeratins and hair keratins. The type I cytokeratins consist of acidic proteins which are arranged in pairs of heterotypic keratin chains. This cytokeratin is a major cellular protein of mature enterocytes and goblet cells and is specifically expressed in the gastric and intestinal mucosa. The type I cytokeratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 94524204, 94656572, 35.35, 98, 'ENSG00000171431', 'P35900'),
(1357, 'KRT23', 'The protein encoded by this gene is a member of the keratin family. The keratins are intermediate filament proteins responsible for the structural integrity of epithelial cells and are subdivided into cytokeratins and hair keratins. The type I cytokeratins consist of acidic proteins which are arranged in pairs of heterotypic keratin chains. The type I cytokeratin genes are clustered in a region of chromosome 17q12-q21. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]', '17', 'protein_coding', 30357102, 30370494, 54.56, 7, 'ENSG00000263309', 'Q9C075'),
(1358, 'KRT23', 'The protein encoded by this gene is a member of the keratin family. The keratins are intermediate filament proteins responsible for the structural integrity of epithelial cells and are subdivided into cytokeratins and hair keratins. The type I cytokeratins consist of acidic proteins which are arranged in pairs of heterotypic keratin chains. The type I cytokeratin genes are clustered in a region of chromosome 17q12-q21. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]', '17', 'protein_coding', 30357102, 30370494, 54.56, 7, 'ENSG00000108244', 'Q9C075'),
(1359, 'KRT25', 'This gene encodes a member of the type I (acidic) keratin family, which belongs to the superfamily of intermediate filament (IF) proteins. Keratins are heteropolymeric structural proteins which form the intermediate filament. These filaments, along with actin microfilaments and microtubules, compose the cytoskeleton of epithelial cells. The type I keratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009]', '17', 'protein_coding', 11331012, 11332360, 55.67, 1, 'ENSG00000204897', 'Q7Z3Z0'),
(1360, 'KRT31', 'The protein encoded by this gene is a member of the keratin gene family. As a type I hair keratin, it is an acidic protein which heterodimerizes with type II keratins to form hair and nails. The type I hair keratins are clustered in a region of chromosome 17q12-q21 and have the same direction of transcription. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28904421, 28936526, 52.63, 14, 'ENSG00000292029', 'Q15323'),
(1361, 'KRT31', 'The protein encoded by this gene is a member of the keratin gene family. As a type I hair keratin, it is an acidic protein which heterodimerizes with type II keratins to form hair and nails. The type I hair keratins are clustered in a region of chromosome 17q12-q21 and have the same direction of transcription. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28904421, 28936526, 52.63, 14, 'ENSG00000262993', 'Q15323'),
(1362, 'KRT31', 'The protein encoded by this gene is a member of the keratin gene family. As a type I hair keratin, it is an acidic protein which heterodimerizes with type II keratins to form hair and nails. The type I hair keratins are clustered in a region of chromosome 17q12-q21 and have the same direction of transcription. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28904421, 28936526, 52.63, 14, 'ENSG00000094796', 'Q15323'),
(1363, 'KRT40', 'This gene encodes a member of the type I (acidic) keratin family, which belongs to the superfamily of intermediate filament (IF) proteins. Keratins are heteropolymeric structural proteins which form the intermediate filament. These filaments, along with actin microfilaments and microtubules, compose the cytoskeleton of epithelial cells. The type I keratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009]', '17', 'unprocessed_pseudogene', 132594398, 132595436, 45.43, 1, 'ENSG00000262845', 'Q6A162'),
(1364, 'KRT40', 'This gene encodes a member of the type I (acidic) keratin family, which belongs to the superfamily of intermediate filament (IF) proteins. Keratins are heteropolymeric structural proteins which form the intermediate filament. These filaments, along with actin microfilaments and microtubules, compose the cytoskeleton of epithelial cells. The type I keratin genes are clustered in a region of chromosome 17q12-q21. [provided by RefSeq, Jul 2009]', '17', 'unprocessed_pseudogene', 132594398, 132595436, 45.43, 1, 'ENSG00000204889', 'Q6A162'),
(1365, 'KRTAP1-1', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 36010098, 36010204, 46.73, 1, 'ENSG00000262808', 'Q07627'),
(1366, 'KRTAP1-1', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 36010098, 36010204, 46.73, 1, 'ENSG00000188581', 'Q07627'),
(1367, 'KRTAP1-5', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 19760004, 19760083, 36.25, 1, 'ENSG00000261862', 'Q9BYS1'),
(1368, 'KRTAP1-5', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 19760004, 19760083, 36.25, 1, 'ENSG00000221852', 'Q9BYS1'),
(1369, 'KRTAP10-11', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 26880597, 27121150, 34.23, 5, 'ENSG00000243489', 'P60412'),
(1370, 'KRTAP10-2', 'This gene encodes a member of the high sulfur-type keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. This gene is located in a cluster of similar genes on 21q22.3. Alternatively-spliced transcript variants have been identified. [provided by RefSeq, Jan 2015]', '21', 'protein_coding', 34438934, 34472060, 40.59, 6, 'ENSG00000205445', 'P60368'),
(1371, 'KRTAP10-4', 'This is an intronless gene located in a cluster of related genes on the q arm of chromosome 21. The proteins encoded by these genes form disulfide bonds with cysteine residues in hair keratins, thereby contributing to the structure and stability of hair fibers. [provided by RefSeq, Apr 2014]', '21', 'protein_coding', 100099243, 100302175, 38.92, 7, 'ENSG00000215454', 'P60372'),
(1372, 'KRTAP10-7', 'Enables identical protein binding activity. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 76435229, 76783253, 41.27, 13, 'ENSG00000272804', 'P60409'),
(1373, 'KRTAP12-2', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 17505563, 17591708, 55.22, 13, 'ENSG00000221864', 'P59991'),
(1374, 'KRTAP15-1', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 23525631, 23687220, 41.44, 9, 'ENSG00000186970', 'Q3LI76'),
(1375, 'KRTAP16-1', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 154781213, 154828813, 34.66, 4, 'ENSG00000292043', 'A8MUX0'),
(1376, 'KRTAP16-1', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 154781213, 154828813, 34.66, 4, 'ENSG00000263186', 'A8MUX0'),
(1377, 'KRTAP16-1', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 154781213, 154828813, 34.66, 4, 'ENSG00000212657', 'A8MUX0'),
(1378, 'KRTAP19-3', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 35852695, 35879603, 39.18, 10, 'ENSG00000244025', 'Q7Z4W3'),
(1379, 'KRTAP19-5', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 109433772, 109540896, 37.89, 3, 'ENSG00000186977', 'Q3LI72'),
(1380, 'KRTAP19-7', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 21666000, 21700015, 54.06, 46, 'ENSG00000244362', 'Q3SYF9'),
(1381, 'KRTAP2-2', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 54423561, 54441391, 38.20, 7, 'ENSG00000292054', 'Q9BYT5'),
(1382, 'KRTAP2-2', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 54423561, 54441391, 38.20, 7, 'ENSG00000214518', 'Q9BYT5'),
(1383, 'KRTAP2-4', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37341, 50411, 51.83, 1, 'ENSG00000292035', 'Q9BYR9'),
(1384, 'KRTAP2-4', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37341, 50411, 51.83, 1, 'ENSG00000263091', 'Q9BYR9'),
(1385, 'KRTAP2-4', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37341, 50411, 51.83, 1, 'ENSG00000213417', 'Q9BYR9'),
(1386, 'KRTAP22-1', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 69809968, 69819416, 61.22, 2, 'ENSG00000186924', 'Q3MIV0'),
(1387, 'KRTAP24-1', 'Predicted to enable structural molecule activity. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 17593653, 17612945, 37.73, 6, 'ENSG00000188694', 'Q3LI83'),
(1388, 'KRTAP27-1', 'Predicted to enable structural molecule activity. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 28315003, 28349050, 34.16, 14, 'ENSG00000206107', 'Q3LI81'),
(1389, 'KRTAP3-2', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 1256605, 1256670, 57.58, 1, 'ENSG00000263296', 'Q9BYR7'),
(1390, 'KRTAP3-2', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the high sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 1256605, 1256670, 57.58, 1, 'ENSG00000212900', 'Q9BYR7'),
(1391, 'KRTAP4-12', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 81875194, 82987179, 37.51, 9, 'ENSG00000292039', 'Q9BQ66'),
(1392, 'KRTAP4-12', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 81875194, 82987179, 37.51, 9, 'ENSG00000263236', 'Q9BQ66'),
(1393, 'KRTAP4-12', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 81875194, 82987179, 37.51, 9, 'ENSG00000213416', 'Q9BQ66'),
(1394, 'KRTAP4-9', 'Involved in hair cycle. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37007857, 37130368, 41.83, 9, 'ENSG00000292052', 'Q9BYQ8'),
(1395, 'KRTAP4-9', 'Involved in hair cycle. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37007857, 37130368, 41.83, 9, 'ENSG00000262121', 'Q9BYQ8'),
(1396, 'KRTAP4-9', 'Involved in hair cycle. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 37007857, 37130368, 41.83, 9, 'ENSG00000212722', 'Q9BYQ8'),
(1397, 'KRTAP5-3', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 318926, 319807, 48.64, 1, 'ENSG00000291435', 'Q6L8H2'),
(1398, 'KRTAP5-3', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 318926, 319807, 48.64, 1, 'ENSG00000281191', 'Q6L8H2'),
(1399, 'KRTAP5-3', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 318926, 319807, 48.64, 1, 'ENSG00000275920', 'Q6L8H2'),
(1400, 'KRTAP5-3', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 318926, 319807, 48.64, 1, 'ENSG00000196224', 'Q6L8H2'),
(1401, 'KRTAP5-6', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 100334739, 100361635, 42.95, 7, 'ENSG00000277389', 'Q6L8G9'),
(1402, 'KRTAP5-6', 'Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 100334739, 100361635, 42.95, 7, 'ENSG00000205864', 'Q6L8G9'),
(1403, 'KRTAP7-1', 'Predicted to be located in intermediate filament. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 59574398, 59615774, 42.65, 3, 'ENSG00000274749', 'Q8IUC3'),
(1404, 'KRTAP9-2', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 49687447, 49695984, 40.40, 1, 'ENSG00000262573', 'Q9BYQ4'),
(1405, 'KRTAP9-2', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 49687447, 49695984, 40.40, 1, 'ENSG00000263090', 'Q9BYQ4'),
(1406, 'KRTAP9-2', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 49687447, 49695984, 40.40, 1, 'ENSG00000239886', 'Q9BYQ4'),
(1407, 'KRTAP9-3', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 37905657, 37942822, 55.92, 12, 'ENSG00000292047', 'Q9BYQ3'),
(1408, 'KRTAP9-3', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 37905657, 37942822, 55.92, 12, 'ENSG00000262749', 'Q9BYQ3'),
(1409, 'KRTAP9-3', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 37905657, 37942822, 55.92, 12, 'ENSG00000204873', 'Q9BYQ3'),
(1410, 'KRTAP9-4', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28931965, 28939342, 56.52, 5, 'ENSG00000292049', 'Q9BYQ2'),
(1411, 'KRTAP9-4', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28931965, 28939342, 56.52, 5, 'ENSG00000262896', 'Q9BYQ2');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1412, 'KRTAP9-4', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28931965, 28939342, 56.52, 5, 'ENSG00000262512', 'Q9BYQ2'),
(1413, 'KRTAP9-4', 'This protein is a member of the keratin-associated protein (KAP) family. The KAP proteins form a matrix of keratin intermediate filaments which contribute to the structure of hair fibers. KAP family members appear to have unique, family-specific amino- and carboxyl-terminal regions and are subdivided into three multi-gene families according to amino acid composition: the high sulfur, the ultrahigh sulfur, and the high tyrosine/glycine KAPs. This protein is a member of the ultrahigh sulfur KAP family and the gene is localized to a cluster of KAPs at 17q12-q21. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 28931965, 28939342, 56.52, 5, 'ENSG00000241595', 'Q9BYQ2'),
(1414, 'KY', 'The protein encoded by this gene belongs to the transglutaminase-like superfamily. The protein is involved in the function, maturation and stabilization of the neuromuscular junction and may be required for normal muscle growth. Mutations in this gene are associated with myopathy, myofibrillar, 7. [provided by RefSeq, Apr 2017]', '3', 'protein_coding', 133896438, 133956968, 55.74, 7, 'ENSG00000174611', 'Q8NBH2'),
(1415, 'KYAT1', 'This gene encodes a cytosolic enzyme that is responsible for the metabolism of cysteine conjugates of certain halogenated alkenes and alkanes. This metabolism can form reactive metabolites leading to nephrotoxicity and neurotoxicity. Increased levels of this enzyme have been linked to schizophrenia. Multiple transcript variants that encode different isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 26521996, 26596717, 48.49, 9, 'ENSG00000171097', 'Q16773'),
(1416, 'LAG3', 'Lymphocyte-activation protein 3 belongs to Ig superfamily and contains 4 extracellular Ig-like domains. The LAG3 gene contains 8 exons.  The sequence data, exon/intron organization, and chromosomal localization all indicate a close relationship of LAG3 to CD4. [provided by RefSeq, Jul 2008]', '12', 'miRNA', 2050271, 2050380, 73.64, 1, 'ENSG00000089692', 'P18627'),
(1417, 'LAGE3', 'This gene belongs to the ESO/LAGE gene family, members of which are clustered together on chromosome Xq28, and have similar exon-intron structures. Unlike the other family members which are normally expressed only in testis and activated in a wide range of human tumors, this gene is ubiquitously expressed in somatic tissues. The latter, combined with the finding that it is highly conserved in mouse and rat, suggests that the encoded protein is functionally important. An intronless pseudogene with high sequence similarity to this gene is located on chromosome 9. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 30378106, 30400108, 53.07, 5, 'ENSG00000196976', 'Q14657'),
(1418, 'LAIR1', 'The protein encoded by this gene is an inhibitory receptor found on peripheral mononuclear cells, including natural killer cells, T cells, and B cells. Inhibitory receptors regulate the immune response to prevent lysis of cells recognized as self. The gene is a member of both the immunoglobulin superfamily and the leukocyte-associated inhibitory receptor family. The gene maps to a region of 19q13.4 called the leukocyte receptor cluster, which contains at least 29 genes encoding leukocyte-expressed receptors of the immunoglobulin superfamily. The encoded protein has been identified as an anchor for tyrosine phosphatase SHP-1, and may induce cell death in myeloid leukemias. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '19', 'protein_coding', 104579764, 104590515, 62.78, 3, 'ENSG00000276163', 'Q6GTX8'),
(1419, 'LAIR1', 'The protein encoded by this gene is an inhibitory receptor found on peripheral mononuclear cells, including natural killer cells, T cells, and B cells. Inhibitory receptors regulate the immune response to prevent lysis of cells recognized as self. The gene is a member of both the immunoglobulin superfamily and the leukocyte-associated inhibitory receptor family. The gene maps to a region of 19q13.4 called the leukocyte receptor cluster, which contains at least 29 genes encoding leukocyte-expressed receptors of the immunoglobulin superfamily. The encoded protein has been identified as an anchor for tyrosine phosphatase SHP-1, and may induce cell death in myeloid leukemias. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '19', 'protein_coding', 104579764, 104590515, 62.78, 3, 'ENSG00000274110', 'Q6GTX8'),
(1420, 'LAIR1', 'The protein encoded by this gene is an inhibitory receptor found on peripheral mononuclear cells, including natural killer cells, T cells, and B cells. Inhibitory receptors regulate the immune response to prevent lysis of cells recognized as self. The gene is a member of both the immunoglobulin superfamily and the leukocyte-associated inhibitory receptor family. The gene maps to a region of 19q13.4 called the leukocyte receptor cluster, which contains at least 29 genes encoding leukocyte-expressed receptors of the immunoglobulin superfamily. The encoded protein has been identified as an anchor for tyrosine phosphatase SHP-1, and may induce cell death in myeloid leukemias. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '19', 'protein_coding', 104579764, 104590515, 62.78, 3, 'ENSG00000276053', 'Q6GTX8'),
(1421, 'LAIR1', 'The protein encoded by this gene is an inhibitory receptor found on peripheral mononuclear cells, including natural killer cells, T cells, and B cells. Inhibitory receptors regulate the immune response to prevent lysis of cells recognized as self. The gene is a member of both the immunoglobulin superfamily and the leukocyte-associated inhibitory receptor family. The gene maps to a region of 19q13.4 called the leukocyte receptor cluster, which contains at least 29 genes encoding leukocyte-expressed receptors of the immunoglobulin superfamily. The encoded protein has been identified as an anchor for tyrosine phosphatase SHP-1, and may induce cell death in myeloid leukemias. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '19', 'protein_coding', 104579764, 104590515, 62.78, 3, 'ENSG00000278154', 'Q6GTX8'),
(1422, 'LAIR1', 'The protein encoded by this gene is an inhibitory receptor found on peripheral mononuclear cells, including natural killer cells, T cells, and B cells. Inhibitory receptors regulate the immune response to prevent lysis of cells recognized as self. The gene is a member of both the immunoglobulin superfamily and the leukocyte-associated inhibitory receptor family. The gene maps to a region of 19q13.4 called the leukocyte receptor cluster, which contains at least 29 genes encoding leukocyte-expressed receptors of the immunoglobulin superfamily. The encoded protein has been identified as an anchor for tyrosine phosphatase SHP-1, and may induce cell death in myeloid leukemias. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '19', 'protein_coding', 104579764, 104590515, 62.78, 3, 'ENSG00000167613', 'Q6GTX8'),
(1423, 'LAMA1', 'This gene encodes one of the alpha 1 subunits of laminin. The laminins are a family of extracellular matrix glycoproteins that have a heterotrimeric structure consisting of an alpha, beta and gamma chain. These proteins make up a major component of the basement membrane and have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Mutations in this gene may be associated with Poretti-Boltshauser syndrome. [provided by RefSeq, Sep 2014]', '18', 'misc_RNA', 126408041, 126408410, 38.11, 1, 'ENSG00000101680', 'P25391'),
(1424, 'LAMA4', 'Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the alpha chain isoform laminin, alpha 4. The domain structure of alpha 4 is similar to that of alpha 3, both of which resemble truncated versions of alpha 1 and alpha 2, in that approximately 1,200 residues at the N-terminus (domains IV, V and VI) have been lost. Laminin, alpha 4 contains the C-terminal G domain which distinguishes all alpha chains from the beta and gamma chains. The RNA analysis from adult and fetal tissues revealed developmental regulation of expression, however, the exact function of laminin, alpha 4 is not known. Tissue-specific utilization of alternative polyA-signal has been described in literature. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2011]', '6', 'protein_coding', 10873223, 10882041, 44.69, 1, 'ENSG00000112769', 'Q16363'),
(1425, 'LAMA5', 'This gene encodes one of the vertebrate laminin alpha chains. Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. The protein encoded by this gene is the alpha-5 subunit of of laminin-10 (laminin-511), laminin-11 (laminin-521) and laminin-15 (laminin-523). [provided by RefSeq, Jun 2013]', '20', 'protein_coding', 51174550, 51218333, 41.03, 3, 'ENSG00000130702', 'O15230'),
(1426, 'LAMC2', 'Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins, composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively), have a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the gamma chain isoform laminin, gamma 2. The gamma 2 chain, formerly thought to be a truncated version of beta chain (B2t), is highly homologous to the gamma 1 chain; however, it lacks domain VI, and domains V, IV and III are shorter. It is expressed in several fetal tissues but differently from gamma 1, and is specifically localized to epithelial cells in skin, lung and kidney. The gamma 2 chain together with alpha 3 and beta 3 chains constitute laminin 5 (earlier known as kalinin), which is an integral part of the anchoring filaments that connect epithelial cells to the underlying basement membrane. The epithelium-specific expression of the gamma 2 chain implied its role as an epithelium attachment molecule, and mutations in this gene have been associated with junctional epidermolysis bullosa, a skin disease characterized by blisters due to disruption of the epidermal-dermal junction. Two transcript variants resulting from alternative splicing of the 3\' terminal exon, and encoding different isoforms of gamma 2 chain, have been described. The two variants are differentially expressed in embryonic tissues, however, the biological significance of the two forms is not known. Transcript variants utilizing alternative polyA_signal have also been noted in literature. [provided by RefSeq, Aug 2011]', '1', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000058085', 'Q13753'),
(1427, 'LAMP5', 'Predicted to be involved in establishment of protein localization to organelle. Located in endoplasmic reticulum-Golgi intermediate compartment membrane; endosome membrane; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 46667821, 46759257, 47.59, 9, 'ENSG00000125869', 'Q9UJQ1'),
(1428, 'LARP4B', 'This gene encodes a member of an evolutionarily conserved protein family implicated in RNA metabolism and translation. Members of this family are characterized by the presence of an La motif, which is often located adjacent to one or more RNA recognition motifs (RRM). Together, the two motifs constitute the functional region of the protein and enable its interaction with the RNA substrate. This protein family is divided into five sub-families: the genuine La proteins and four La-related protein (LARP) sub-families. The protein encoded by this gene belongs to LARP sub-family 4. It is a cytoplasmic protein that may play a stimulatory role in translation. [provided by RefSeq, Oct 2012]', '10', 'miRNA', 109313571, 109313625, 67.27, 1, 'ENSG00000107929', 'Q92615'),
(1429, 'LAS1L', 'Enables RNA binding activity. Involved in rRNA processing. Located in nucleolus. Part of MLL1 complex. Implicated in Wilson-Turner syndrome. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 99923266, 99929620, 42.56, 1, 'ENSG00000001497', 'Q9Y4W2'),
(1430, 'LAT2', 'Enables several functions, including neutral L-amino acid transmembrane transporter activity; protein heterodimerization activity; and secondary active transmembrane transporter activity. Involved in L-alanine import across plasma membrane; L-leucine import across plasma membrane; and thyroid hormone transport. Located in basolateral plasma membrane and microvillus membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 96064315, 96064399, 56.47, 1, 'ENSG00000086730', 'Q9GZY6'),
(1431, 'LATS2', 'This gene encodes a serine/threonine protein kinase belonging to the LATS tumor suppressor family. The protein localizes to centrosomes during interphase, and early and late metaphase. It interacts with the centrosomal proteins aurora-A and ajuba and is required for accumulation of gamma-tubulin and spindle formation at the onset of mitosis. It also interacts with a negative regulator of p53 and may function in a positive feedback loop with p53 that responds to cytoskeleton damage. Additionally, it can function as a co-repressor of androgen-responsive gene expression. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 18989307, 19122637, 42.96, 11, 'ENSG00000150457', 'Q9NRM7'),
(1432, 'LCA5L', 'Predicted to be involved in intraciliary transport. Predicted to be active in axoneme. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 53267398, 53349179, 41.88, 7, 'ENSG00000157578', 'O95447'),
(1433, 'LCN2', 'This gene encodes a protein that belongs to the lipocalin family. Members of this family transport small hydrophobic molecules such as lipids, steroid hormones and retinoids. The protein encoded by this gene is a neutrophil gelatinase-associated lipocalin and plays a role in innate immunity by limiting bacterial growth as a result of sequestering iron-containing siderophores. The presence of this protein in blood and urine is an early biomarker of acute kidney injury. This protein is thought to be be involved in multiple cellular processes, including maintenance of skin homeostasis, and suppression of invasiveness and metastasis. Mice lacking this gene are more susceptible to bacterial infection than wild type mice. [provided by RefSeq, Sep 2015]', '9', 'protein_coding', 132617022, 132624275, 37.39, 2, 'ENSG00000148346', 'P80188'),
(1434, 'LCOR', 'LCOR is a transcriptional corepressor widely expressed in fetal and adult tissues that is recruited to agonist-bound nuclear receptors through a single LxxLL motif, also referred to as a nuclear receptor (NR) box (Fernandes et al., 2003 [PubMed 12535528]).[supplied by OMIM, Mar 2008]', '10', 'protein_coding', 76351797, 76559809, 36.48, 18, 'ENSG00000196233', 'Q96JN0'),
(1435, 'LDAH', 'Predicted to enable sterol ester esterase activity. Involved in cholesterol homeostasis and intracellular triglyceride homeostasis. Acts upstream of with a positive effect on lipid droplet fusion. Located in endoplasmic reticulum and lipid droplet. [provided by Alliance of Genome Resources, May 2025]', '2', 'miRNA', 79422169, 79422226, 46.55, 1, 'ENSG00000118961', 'Q9H6V9'),
(1436, 'LDB1', 'Enables LIM domain binding activity; RNA polymerase II-specific DNA-binding transcription factor binding activity; and enzyme binding activity. Involved in negative regulation of DNA-templated transcription and positive regulation of transcription by RNA polymerase II. Located in chromatin. Part of beta-catenin-TCF complex. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 1180561, 1185415, 55.74, 3, 'ENSG00000198728', 'Q86U70'),
(1437, 'LDB3', 'This gene encodes a PDZ domain-containing protein. PDZ motifs are modular protein-protein interaction domains consisting of 80-120 amino acid residues. PDZ domain-containing proteins interact with each other in cytoskeletal assembly or with other proteins involved in targeting and clustering of membrane proteins. The protein encoded by this gene interacts with alpha-actinin-2 through its N-terminal PDZ domain and with protein kinase C via its C-terminal LIM domains. The LIM domain is a cysteine-rich motif defined by 50-60 amino acids containing two zinc-binding modules. This protein also interacts with all three members of the myozenin family. Mutations in this gene have been associated with myofibrillar myopathy and dilated cardiomyopathy. Alternatively spliced transcript variants encoding different isoforms have been identified; all isoforms have N-terminal PDZ domains while only longer isoforms (1, 2 and 5) have C-terminal LIM domains. [provided by RefSeq, Jan 2010]', '10', 'protein_coding', 272827, 273903, 55.25, 1, 'ENSG00000122367', 'O75112'),
(1438, 'LDHA', 'This gene encodes the A subunit of lactate dehydrogenase enzyme which catalyzes the reversible conversion of pyruvate to lactate with the concomitant oxidation of NADH to NAD in anaerobic glycolysis. The protein is found predominantly in skeletal muscle and belongs to the lactate dehydrogenase family. Mutations in this gene have been linked to exertional myoglobinuria. The human genome contains several non-transcribed pseudogenes of this gene. [provided by RefSeq, Sep 2023]', '11', 'protein_coding', 55997357, 56005519, 44.00, 1, 'ENSG00000288299', 'P00338'),
(1439, 'LDHA', 'This gene encodes the A subunit of lactate dehydrogenase enzyme which catalyzes the reversible conversion of pyruvate to lactate with the concomitant oxidation of NADH to NAD in anaerobic glycolysis. The protein is found predominantly in skeletal muscle and belongs to the lactate dehydrogenase family. Mutations in this gene have been linked to exertional myoglobinuria. The human genome contains several non-transcribed pseudogenes of this gene. [provided by RefSeq, Sep 2023]', '11', 'protein_coding', 55997357, 56005519, 44.00, 1, 'ENSG00000134333', 'P00338'),
(1440, 'LETM1', 'This gene encodes a protein that is localized to the inner mitochondrial membrane. The protein functions to maintain the mitochondrial tubular shapes and is required for normal mitochondrial morphology and cellular viability. Mutations in this gene cause Wolf-Hirschhorn syndrome, a complex malformation syndrome caused by the deletion of parts of the distal short arm of chromosome 4. Related pseudogenes have been identified on chromosomes 8, 15 and 19. [provided by RefSeq, Oct 2009]', '4', 'protein_coding', 183253253, 183298504, 42.14, 12, 'ENSG00000168924', 'O95202'),
(1441, 'LGALS3', 'This gene encodes a member of the galectin family of carbohydrate binding proteins. Members of this protein family have an affinity for beta-galactosides. The encoded protein is characterized by an N-terminal proline-rich tandem repeat domain and a single C-terminal carbohydrate recognition domain. This protein can self-associate through the N-terminal domain allowing it to bind to multivalent saccharide ligands. This protein localizes to the extracellular matrix, the cytoplasm and the nucleus. This protein plays a role in numerous cellular functions including apoptosis, innate immunity, cell adhesion and T-cell regulation. The protein exhibits antimicrobial activity against bacteria and fungi. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Oct 2014]', '14', 'protein_coding', 40679273, 40682812, 55.65, 4, 'ENSG00000131981', 'P17931'),
(1442, 'LGALS7', 'The galectins are a family of beta-galactoside-binding proteins implicated in modulating cell-cell and cell-matrix interactions. Differential and in situ hybridization studies indicate that this lectin is specifically expressed in keratinocytes and found mainly in stratified squamous epithelium. A duplicate copy of this gene (GeneID:653499) is found adjacent to, but on the opposite strand on chromosome 19. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 50226292, 50259362, 57.36, 12, 'ENSG00000283082', 'P47929'),
(1443, 'LGALS7', 'The galectins are a family of beta-galactoside-binding proteins implicated in modulating cell-cell and cell-matrix interactions. Differential and in situ hybridization studies indicate that this lectin is specifically expressed in keratinocytes and found mainly in stratified squamous epithelium. A duplicate copy of this gene (GeneID:653499) is found adjacent to, but on the opposite strand on chromosome 19. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 50226292, 50259362, 57.36, 12, 'ENSG00000205076', 'P47929'),
(1444, 'LHFPL6', 'This gene is a member of the lipoma HMGIC fusion partner (LHFP) gene family, which is a subset of the superfamily of tetraspan transmembrane protein encoding genes. This gene is fused to a high-mobility group gene in a translocation-associated lipoma. Mutations in another LHFP-like gene result in deafness in humans and mice. Alternatively spliced transcript variants have been found; however, their full-length nature is not known. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 33589341, 33643926, 41.59, 23, 'ENSG00000183722', 'Q9Y693'),
(1445, 'LIAT1', 'Predicted to enable molecular condensate scaffold activity. Predicted to be involved in membraneless organelle assembly and protein arginylation. Predicted to be located in cytoplasm and nucleus. Predicted to be active in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 84317874, 84502479, 34.93, 6, 'ENSG00000283985', 'Q6ZQX7'),
(1446, 'LIAT1', 'Predicted to enable molecular condensate scaffold activity. Predicted to be involved in membraneless organelle assembly and protein arginylation. Predicted to be located in cytoplasm and nucleus. Predicted to be active in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 84317874, 84502479, 34.93, 6, 'ENSG00000187624', 'Q6ZQX7'),
(1447, 'LIF', 'The protein encoded by this gene is a pleiotropic cytokine with roles in several different systems. It is involved in the induction of hematopoietic differentiation in normal and myeloid leukemia cells, induction of neuronal cell differentiation, regulator of mesenchymal to epithelial conversion during kidney development, and may also have a role in immune tolerance at the maternal-fetal interface. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Mar 2012]', '22', 'protein_coding', 17137336, 17202054, 36.91, 12, 'ENSG00000128342', 'P15018'),
(1448, 'LIG4', 'The protein encoded by this gene is a DNA ligase that joins single-strand breaks in a double-stranded polydeoxynucleotide in an ATP-dependent reaction. This protein is essential for V(D)J recombination and DNA double-strand break (DSB) repair through nonhomologous end joining (NHEJ). This protein forms a complex with the X-ray repair cross complementing protein 4 (XRCC4), and further interacts with the DNA-dependent protein kinase (DNA-PK). Both XRCC4 and DNA-PK are known to be required for NHEJ. The crystal structure of the complex formed by this protein and XRCC4 has been resolved. Defects in this gene are the cause of LIG4 syndrome. Alternatively spliced transcript variants encoding the same protein have been observed. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 44246339, 44262216, 57.00, 4, 'ENSG00000174405', 'P49917'),
(1449, 'LILRA1', 'This gene encodes an activating member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein is predominantly expressed in B cells, interacts with major histocompatibility complex class I ligands, and contributes to the regulation of immune responses. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2013]', '19', 'protein_coding', 52791371, 52803860, 40.45, 3, 'ENSG00000274935', 'O75019'),
(1450, 'LILRA1', 'This gene encodes an activating member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein is predominantly expressed in B cells, interacts with major histocompatibility complex class I ligands, and contributes to the regulation of immune responses. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2013]', '19', 'protein_coding', 52791371, 52803860, 40.45, 3, 'ENSG00000277398', 'O75019'),
(1451, 'LILRA1', 'This gene encodes an activating member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein is predominantly expressed in B cells, interacts with major histocompatibility complex class I ligands, and contributes to the regulation of immune responses. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2013]', '19', 'protein_coding', 52791371, 52803860, 40.45, 3, 'ENSG00000275525', 'O75019'),
(1452, 'LILRA1', 'This gene encodes an activating member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein is predominantly expressed in B cells, interacts with major histocompatibility complex class I ligands, and contributes to the regulation of immune responses. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2013]', '19', 'protein_coding', 52791371, 52803860, 40.45, 3, 'ENSG00000104974', 'O75019'),
(1453, 'LILRA3', 'This gene encodes a member of a family of immunoreceptors that are expressed predominantly in monocytes and B cells, and at lower levels in dendritic cells and natural killer cells. The encoded protein lacks the transmembrane region found in other members of this family. It acts as a soluble receptor for class I major histocompatibility complex (MHC) antigens. Alternatively spliced transcript variants encoding different isoforms have been found. This gene is located in a cluster of related genes on chromosome 19 and is polymorphic in human populations, with many individuals containing a deletion of this genomic region. [provided by RefSeq, Mar 2014]', '19', 'protein_coding', 47561205, 47571908, 51.19, 5, 'ENSG00000273884', 'Q8N6C8'),
(1454, 'LILRA3', 'This gene encodes a member of a family of immunoreceptors that are expressed predominantly in monocytes and B cells, and at lower levels in dendritic cells and natural killer cells. The encoded protein lacks the transmembrane region found in other members of this family. It acts as a soluble receptor for class I major histocompatibility complex (MHC) antigens. Alternatively spliced transcript variants encoding different isoforms have been found. This gene is located in a cluster of related genes on chromosome 19 and is polymorphic in human populations, with many individuals containing a deletion of this genomic region. [provided by RefSeq, Mar 2014]', '19', 'protein_coding', 47561205, 47571908, 51.19, 5, 'ENSG00000276175', 'Q8N6C8'),
(1455, 'LILRA3', 'This gene encodes a member of a family of immunoreceptors that are expressed predominantly in monocytes and B cells, and at lower levels in dendritic cells and natural killer cells. The encoded protein lacks the transmembrane region found in other members of this family. It acts as a soluble receptor for class I major histocompatibility complex (MHC) antigens. Alternatively spliced transcript variants encoding different isoforms have been found. This gene is located in a cluster of related genes on chromosome 19 and is polymorphic in human populations, with many individuals containing a deletion of this genomic region. [provided by RefSeq, Mar 2014]', '19', 'protein_coding', 47561205, 47571908, 51.19, 5, 'ENSG00000275841', 'Q8N6C8'),
(1456, 'LILRA3', 'This gene encodes a member of a family of immunoreceptors that are expressed predominantly in monocytes and B cells, and at lower levels in dendritic cells and natural killer cells. The encoded protein lacks the transmembrane region found in other members of this family. It acts as a soluble receptor for class I major histocompatibility complex (MHC) antigens. Alternatively spliced transcript variants encoding different isoforms have been found. This gene is located in a cluster of related genes on chromosome 19 and is polymorphic in human populations, with many individuals containing a deletion of this genomic region. [provided by RefSeq, Mar 2014]', '19', 'protein_coding', 47561205, 47571908, 51.19, 5, 'ENSG00000278046', 'Q8N6C8'),
(1457, 'LILRA4', 'This gene encodes an immunoglobulin-like cell surface protein that is expressed predominantly on plasmacytoid dendritic cells (PDCs) and modulates the function of these cells in the immune response. Expression of this gene is downregulated by interleukin 3 (IL3). This gene is one of a cluster of highly related genes located at chromosomal region 19q13.4. [provided by RefSeq, Jan 2015]', '19', 'protein_coding', 82359247, 82363775, 56.37, 1, 'ENSG00000276798', 'P59901'),
(1458, 'LILRA4', 'This gene encodes an immunoglobulin-like cell surface protein that is expressed predominantly on plasmacytoid dendritic cells (PDCs) and modulates the function of these cells in the immune response. Expression of this gene is downregulated by interleukin 3 (IL3). This gene is one of a cluster of highly related genes located at chromosomal region 19q13.4. [provided by RefSeq, Jan 2015]', '19', 'protein_coding', 82359247, 82363775, 56.37, 1, 'ENSG00000277092', 'P59901'),
(1459, 'LILRA4', 'This gene encodes an immunoglobulin-like cell surface protein that is expressed predominantly on plasmacytoid dendritic cells (PDCs) and modulates the function of these cells in the immune response. Expression of this gene is downregulated by interleukin 3 (IL3). This gene is one of a cluster of highly related genes located at chromosomal region 19q13.4. [provided by RefSeq, Jan 2015]', '19', 'protein_coding', 82359247, 82363775, 56.37, 1, 'ENSG00000274185', 'P59901'),
(1460, 'LILRA4', 'This gene encodes an immunoglobulin-like cell surface protein that is expressed predominantly on plasmacytoid dendritic cells (PDCs) and modulates the function of these cells in the immune response. Expression of this gene is downregulated by interleukin 3 (IL3). This gene is one of a cluster of highly related genes located at chromosomal region 19q13.4. [provided by RefSeq, Jan 2015]', '19', 'protein_coding', 82359247, 82363775, 56.37, 1, 'ENSG00000239961', 'P59901'),
(1461, 'LILRA6', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 941109, 943346, 61.62, 1, 'ENSG00000275584', 'Q6PI73'),
(1462, 'LILRA6', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 941109, 943346, 61.62, 1, 'ENSG00000275539', 'Q6PI73'),
(1463, 'LILRA6', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 941109, 943346, 61.62, 1, 'ENSG00000277177', 'Q6PI73'),
(1464, 'LILRA6', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 941109, 943346, 61.62, 1, 'ENSG00000274148', 'Q6PI73'),
(1465, 'LILRA6', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 941109, 943346, 61.62, 1, 'ENSG00000244482', 'Q6PI73'),
(1466, 'LILRB2', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'snoRNA', 68627234, 68627375, 52.82, 1, 'ENSG00000276146', 'Q8N423'),
(1467, 'LILRB2', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'snoRNA', 68627234, 68627375, 52.82, 1, 'ENSG00000277751', 'Q8N423'),
(1468, 'LILRB2', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'snoRNA', 68627234, 68627375, 52.82, 1, 'ENSG00000275463', 'Q8N423'),
(1469, 'LILRB2', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'snoRNA', 68627234, 68627375, 52.82, 1, 'ENSG00000274513', 'Q8N423'),
(1470, 'LILRB2', 'This gene is a member of the leukocyte immunoglobulin-like receptor (LIR) family, which is found in a gene cluster at chromosomal region 19q13.4. The encoded protein belongs to the subfamily B class of LIR receptors which contain two or four extracellular immunoglobulin domains, a transmembrane domain, and two to four cytoplasmic immunoreceptor tyrosine-based inhibitory motifs (ITIMs). The receptor is expressed on immune cells where it binds to MHC class I molecules on antigen-presenting cells and transduces a negative signal that inhibits stimulation of an immune response. It is thought to control inflammatory responses and cytotoxicity to help focus the immune response and limit autoreactivity. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '19', 'snoRNA', 68627234, 68627375, 52.82, 1, 'ENSG00000131042', 'Q8N423'),
(1471, 'LIN28B', 'The protein encoded by this gene belongs to the lin-28 family, which is characterized by the presence of a cold-shock domain and a pair of CCHC zinc finger domains. This gene is highly expressed in testis, fetal liver, placenta, and in primary human tumors and cancer cell lines. It is negatively regulated by microRNAs that target sites in the 3\' UTR, and overexpression of this gene in primary tumors is linked to the repression of let-7 family of microRNAs and derepression of let-7 targets, which facilitates cellular transformation. [provided by RefSeq, Jun 2012]', '6', 'protein_coding', 15528192, 15529139, 43.04, 1, 'ENSG00000187772', 'Q6ZN17'),
(1472, 'LINC02210-CRHR1', 'This locus represents naturally occurring readthrough transcription between neighboring genes CRHR1-IT1, CRHR1 intronic transcript 1 (Gene ID: 147081) and CRHR1, corticotropin releasing hormone receptor 1 (Gene ID: 1394) on chromosome 17. The readthrough transcript encodes a protein that shares sequence identity with the product of the CRHR1 gene. [provided by RefSeq, Dec 2016]', '17', 'protein_coding', 108294991, 108526796, 39.14, 7, 'ENSG00000282456', ''),
(1473, 'LINC02210-CRHR1', 'This locus represents naturally occurring readthrough transcription between neighboring genes CRHR1-IT1, CRHR1 intronic transcript 1 (Gene ID: 147081) and CRHR1, corticotropin releasing hormone receptor 1 (Gene ID: 1394) on chromosome 17. The readthrough transcript encodes a protein that shares sequence identity with the product of the CRHR1 gene. [provided by RefSeq, Dec 2016]', '17', 'protein_coding', 108294991, 108526796, 39.14, 7, 'ENSG00000263715', ''),
(1474, 'LIPI', 'The protein encoded by this gene is a phospholipase that hydrolyzes phosphatidic acid to produce lysophosphatidic acid. Defects in this gene are a cause of susceptibility to familial hypertrigliceridemia. This gene is also expressed at high levels in Ewing family tumor cells. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]', '21', 'TEC', 13239543, 13242076, 54.70, 1, 'ENSG00000188992', 'Q6XZB0'),
(1475, 'LIPK', 'Predicted to enable lipoprotein lipase activity. Predicted to be involved in cornification. Predicted to be located in extracellular region. Predicted to be active in intracellular membrane-bounded organelle. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 50234142, 50234212, 47.89, 1, 'ENSG00000204021', 'Q5VXJ0'),
(1476, 'LLCFC1', 'Predicted to be involved in fusion of sperm to egg plasma membrane involved in single fertilization. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 31544444, 31548427, 49.67, 6, 'ENSG00000165131', 'Q96L11'),
(1477, 'LLCFC1', 'Predicted to be involved in fusion of sperm to egg plasma membrane involved in single fertilization. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 31544444, 31548427, 49.67, 6, 'ENSG00000275821', 'Q96L11'),
(1478, 'LMBRD1', 'This gene encodes a lysosomal membrane protein that may be involved in the transport and metabolism of cobalamin. This protein also interacts with the large form of the hepatitis delta antigen and may be required for the nucleocytoplasmic shuttling of the hepatitis delta virus. Mutations in this gene are associated with the vitamin B12 metabolism disorder termed, homocystinuria-megaloblastic anemia complementation type F.[provided by RefSeq, Oct 2009]', '6', 'miRNA', 112780718, 112780788, 46.48, 1, 'ENSG00000168216', 'Q9NUN5'),
(1479, 'LMF2', 'Predicted to be involved in protein maturation. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 117814691, 117832377, 39.20, 6, 'ENSG00000100258', 'Q9BU23'),
(1480, 'LMNTD2', 'Predicted to be a structural constituent of chromatin. Predicted to act upstream of or within positive regulation of mRNA splicing, via spliceosome. Predicted to be active in lamin filament. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 25331, 40317, 59.54, 4, 'ENSG00000275873', 'Q8IXW0'),
(1481, 'LMNTD2', 'Predicted to be a structural constituent of chromatin. Predicted to act upstream of or within positive regulation of mRNA splicing, via spliceosome. Predicted to be active in lamin filament. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 25331, 40317, 59.54, 4, 'ENSG00000185522', 'Q8IXW0'),
(1482, 'LMO7', 'This gene encodes a protein containing a calponin homology (CH) domain, a PDZ domain, and a LIM domain, and may be involved in protein-protein interactions. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene, however, the full-length nature of some variants is not known. [provided by RefSeq, Jan 2009]', '13', 'protein_coding', 25880535, 26171128, 39.73, 20, 'ENSG00000136153', 'Q8WWI1'),
(1483, 'LMTK2', 'The protein encoded by this gene belongs to the protein kinase superfamily and the protein tyrosine kinase family. It contains N-terminal transmembrane helices and a long C-terminal cytoplasmic tail with serine/threonine/tyrosine kinase activity. This protein interacts with several other proteins, such as Inhibitor-2 (Inh2), protein phosphatase-1 (PP1C), p35, and myosin VI. It phosporylates other proteins, and is itself also phosporylated when interacting with cyclin-dependent kinase 5 (cdk5)/p35 complex. This protein involves in nerve growth factor (NGF)-TrkA signalling, and also plays a critical role in endosomal membrane trafficking. Mouse studies suggested an essential role of this protein in spermatogenesis. [provided by RefSeq, Oct 2009]', '7', 'protein_coding', 23356403, 23365752, 54.29, 3, 'ENSG00000164715', 'Q8IWU2'),
(1484, 'LNX2', 'Predicted to enable ubiquitin-protein transferase activity. Predicted to act upstream of or within neural precursor cell proliferation and neuron differentiation. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 21839617, 21885423, 38.50, 3, 'ENSG00000139517', 'Q8N448');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1485, 'LPAR5', 'This gene encodes a member of the rhodopsin class of G protein-coupled transmembrane receptors. This protein transmits extracellular signals from lysophosphatidic acid to cells through heterotrimeric G proteins and mediates numerous cellular processes. Many G protein receptors serve as targets for pharmaceutical drugs. Transcript variants of this gene have been described.[provided by RefSeq, Dec 2008]', '12', 'protein_coding', 6913745, 6923698, 55.33, 14, 'ENSG00000184574', 'Q9H1C0'),
(1486, 'LPCAT2', 'This gene encodes a member of the lysophospholipid acyltransferase family. The encoded enzyme may function in two ways: to catalyze the biosynthesis of platelet-activating factor (1-O-alkyl-2-acetyl-sn-glycero-3-phosphocholine) from 1-O-alkyl-sn-glycero-3-phosphocholine, and to catalyze the synthesis of glycerophospholipid precursors from arachidonyl-CoA and lysophosphatidylcholine. The encoded protein may function in membrane biogenesis and production of platelet-activating factor in inflammatory cells. The enzyme may localize to the endoplasmic reticulum and the Golgi. [provided by RefSeq, Feb 2009]', '16', 'protein_coding', 48962157, 49154527, 39.06, 5, 'ENSG00000087253', 'Q7L5N7'),
(1487, 'LPXN', 'The product encoded by this gene is preferentially expressed in hematopoietic cells and belongs to the paxillin protein family. Similar to other members of this focal-adhesion-associated adaptor-protein family, it has four leucine-rich LD-motifs in the N-terminus and four LIM domains in the C-terminus. It may function in cell type-specific signaling by associating with PYK2, a member of focal adhesion kinase family. As a substrate for a tyrosine kinase in lymphoid cells, this protein may also function in, and be regulated by, tyrosine kinase activity. Alternative splicing results in multiple transcript variants encoding distinct isoforms.[provided by RefSeq, Jan 2009]', '11', 'protein_coding', 74458400, 74460891, 55.98, 14, 'ENSG00000110031', 'O60711'),
(1488, 'LRAT', 'The protein encoded by this gene localizes to the endoplasmic reticulum, where it catalyzes the esterification of all-trans-retinol into all-trans-retinyl ester. This reaction is an important step in vitamin A metabolism in the visual system. Mutations in this gene have been associated with early-onset severe retinal dystrophy and Leber congenital amaurosis 14. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2014]', '4', 'miRNA', 8144994, 8145071, 52.56, 1, 'ENSG00000121207', 'O95237'),
(1489, 'LRP1', 'This gene encodes a member of the low-density lipoprotein receptor family of proteins. The encoded preproprotein is proteolytically processed by furin to generate 515 kDa and 85 kDa subunits that form the mature receptor (PMID: 8546712). This receptor is involved in several cellular processes, including intracellular signaling, lipid homeostasis, and clearance of apoptotic cells. In addition, the encoded protein is necessary for the alpha 2-macroglobulin-mediated clearance of secreted amyloid precursor protein and beta-amyloid, the main component of amyloid plaques found in Alzheimer patients. Expression of this gene decreases with age and has been found to be lower than controls in brain tissue from Alzheimer\'s disease patients. [provided by RefSeq, Oct 2015]', '12', 'protein_coding', 20912, 25493, 48.56, 3, 'ENSG00000123384', 'Q07954'),
(1490, 'LRP6', 'This gene encodes a member of the low density lipoprotein (LDL) receptor gene family. LDL receptors are transmembrane cell surface proteins involved in receptor-mediated endocytosis of lipoprotein and protein ligands. The protein encoded by this gene functions as a receptor or, with Frizzled, a co-receptor for Wnt and thereby transmits the canonical Wnt/beta-catenin signaling cascade. Through its interaction with the Wnt/beta-catenin signaling cascade this gene plays a role in the regulation of cell differentiation, proliferation, and migration and the development of many cancer types. This protein undergoes gamma-secretase dependent RIP- (regulated intramembrane proteolysis) processing but the precise locations of the cleavage sites have not been determined.[provided by RefSeq, Dec 2009]', '12', 'protein_coding', 46985, 48543, 61.19, 3, 'ENSG00000281324', 'O75581'),
(1491, 'LRP6', 'This gene encodes a member of the low density lipoprotein (LDL) receptor gene family. LDL receptors are transmembrane cell surface proteins involved in receptor-mediated endocytosis of lipoprotein and protein ligands. The protein encoded by this gene functions as a receptor or, with Frizzled, a co-receptor for Wnt and thereby transmits the canonical Wnt/beta-catenin signaling cascade. Through its interaction with the Wnt/beta-catenin signaling cascade this gene plays a role in the regulation of cell differentiation, proliferation, and migration and the development of many cancer types. This protein undergoes gamma-secretase dependent RIP- (regulated intramembrane proteolysis) processing but the precise locations of the cleavage sites have not been determined.[provided by RefSeq, Dec 2009]', '12', 'protein_coding', 46985, 48543, 61.19, 3, 'ENSG00000070018', 'O75581'),
(1492, 'LRRC14B', 'The protein encoded by this gene is a leucine-rich repeat containing protein that is a member of the PRAME family. [provided by RefSeq, Apr 2017]', '5', 'protein_coding', 149993118, 150000654, 44.21, 3, 'ENSG00000185028', 'A6NHZ5'),
(1493, 'LRRC3', 'Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 80669503, 80809877, 36.14, 5, 'ENSG00000160233', 'Q9BY71'),
(1494, 'LRRC30', 'Predicted to be involved in intracellular signal transduction. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 51391317, 51515763, 42.72, 18, 'ENSG00000206422', 'A6NM36'),
(1495, 'LRRC37A', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'processed_pseudogene', 54561, 55128, 47.71, 1, 'ENSG00000278742', 'A6NMS7'),
(1496, 'LRRC37A', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'processed_pseudogene', 54561, 55128, 47.71, 1, 'ENSG00000176681', 'A6NMS7'),
(1497, 'LRRC37A2', 'Predicted to enable ATP binding activity; ATP hydrolysis activity; and metal ion binding activity. Predicted to be involved in SNARE complex disassembly and protein transport. Predicted to be located in cytoplasm and membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 119146151, 119160042, 38.34, 5, 'ENSG00000274332', 'A6NM11'),
(1498, 'LRRC37A2', 'Predicted to enable ATP binding activity; ATP hydrolysis activity; and metal ion binding activity. Predicted to be involved in SNARE complex disassembly and protein transport. Predicted to be located in cytoplasm and membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 119146151, 119160042, 38.34, 5, 'ENSG00000277221', 'A6NM11'),
(1499, 'LRRC37A2', 'Predicted to enable ATP binding activity; ATP hydrolysis activity; and metal ion binding activity. Predicted to be involved in SNARE complex disassembly and protein transport. Predicted to be located in cytoplasm and membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 119146151, 119160042, 38.34, 5, 'ENSG00000238083', 'A6NM11'),
(1500, 'LRRC37A3', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 125446650, 125520202, 40.09, 5, 'ENSG00000275621', 'O60309'),
(1501, 'LRRC37A3', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 125446650, 125520202, 40.09, 5, 'ENSG00000176809', 'O60309'),
(1502, 'LRRC56', 'Predicted to be involved in cell projection organization. Predicted to be located in cilium. Implicated in primary ciliary dyskinesia 39. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 116427847, 116477957, 38.55, 1, 'ENSG00000273831', 'Q8IYG6'),
(1503, 'LRRC56', 'Predicted to be involved in cell projection organization. Predicted to be located in cilium. Implicated in primary ciliary dyskinesia 39. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 116427847, 116477957, 38.55, 1, 'ENSG00000161328', 'Q8IYG6'),
(1504, 'LRRC63', 'Predicted to be involved in intracellular signal transduction. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 7112255, 7294414, 47.51, 7, 'ENSG00000173988', 'Q05C16'),
(1505, 'LRRTM2', 'Predicted to enable neurexin family protein binding activity. Predicted to be involved in regulation of postsynaptic density assembly. Predicted to act upstream of or within several processes, including long-term synaptic potentiation; negative regulation of receptor internalization; and positive regulation of synapse assembly. Is active in GABA-ergic synapse and postsynaptic specialization membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 49171898, 49175235, 54.07, 2, 'ENSG00000146006', 'O43300'),
(1506, 'LRTM2', 'Predicted to act upstream of or within positive regulation of synapse assembly. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 99294539, 99307399, 38.07, 1, 'ENSG00000285383', 'Q8N967'),
(1507, 'LRTM2', 'Predicted to act upstream of or within positive regulation of synapse assembly. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 99294539, 99307399, 38.07, 1, 'ENSG00000166159', 'Q8N967'),
(1508, 'LSM5', 'Sm-like proteins were identified in a variety of organisms based on sequence homology with the Sm protein family (see SNRPD2; MIM 601061). Sm-like proteins contain the Sm sequence motif, which consists of 2 regions separated by a linker of variable length that folds as a loop. The Sm-like proteins are thought to form a stable heteromer present in tri-snRNP particles, which are important for pre-mRNA splicing.[supplied by OMIM, Apr 2004]', '7', 'protein_coding', 147390808, 147510068, 41.44, 9, 'ENSG00000106355', 'Q9Y4Y9'),
(1509, 'LSP1', 'This gene encodes an intracellular F-actin binding protein. The protein is expressed in lymphocytes, neutrophils, macrophages, and endothelium and may regulate neutrophil motility, adhesion to fibrinogen matrix proteins, and transendothelial migration. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 5299218, 5316954, 46.21, 3, 'ENSG00000288199', 'P33241'),
(1510, 'LSP1', 'This gene encodes an intracellular F-actin binding protein. The protein is expressed in lymphocytes, neutrophils, macrophages, and endothelium and may regulate neutrophil motility, adhesion to fibrinogen matrix proteins, and transendothelial migration. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 5299218, 5316954, 46.21, 3, 'ENSG00000130592', 'P33241'),
(1511, 'LTB4R', 'Predicted to enable G protein-coupled peptide receptor activity and leukotriene B4 receptor activity. Predicted to be involved in neuropeptide signaling pathway. Predicted to act upstream of or within signal transduction. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'TR_J_gene', 22489488, 22489543, 46.43, 1, 'ENSG00000285456', 'Q15722'),
(1512, 'LTB4R', 'Predicted to enable G protein-coupled peptide receptor activity and leukotriene B4 receptor activity. Predicted to be involved in neuropeptide signaling pathway. Predicted to act upstream of or within signal transduction. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'TR_J_gene', 22489488, 22489543, 46.43, 1, 'ENSG00000213903', 'Q15722'),
(1513, 'LTB4R2', 'Predicted to enable G protein-coupled peptide receptor activity and leukotriene B4 receptor activity. Predicted to be involved in neuropeptide signaling pathway. Predicted to act upstream of or within keratinocyte migration and signal transduction. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 3411608, 3459978, 43.85, 23, 'ENSG00000285203', 'Q9NPC1'),
(1514, 'LTB4R2', 'Predicted to enable G protein-coupled peptide receptor activity and leukotriene B4 receptor activity. Predicted to be involved in neuropeptide signaling pathway. Predicted to act upstream of or within keratinocyte migration and signal transduction. Located in nucleoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 3411608, 3459978, 43.85, 23, 'ENSG00000213906', 'Q9NPC1'),
(1515, 'LTBP2', 'The protein encoded by this gene belongs to the family of latent transforming growth factor (TGF)-beta binding proteins (LTBP), which are extracellular matrix proteins with multi-domain structure. This protein is the largest member of the LTBP family possessing unique regions and with most similarity to the fibrillins. It has thus been suggested that it may have multiple functions: as a member of the TGF-beta latent complex, as a structural component of microfibrils, and a role in cell adhesion. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 78271468, 78327691, 38.17, 3, 'ENSG00000119681', 'Q14767'),
(1516, 'LTC4S', 'The MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) family includes a number of human proteins, several of which are involved the production of leukotrienes. This gene encodes an enzyme that catalyzes the first step in the biosynthesis of cysteinyl leukotrienes, potent biological compounds derived from arachidonic acid. Leukotrienes have been implicated as mediators of anaphylaxis and inflammatory conditions such as human bronchial asthma. This protein localizes to the nuclear envelope and adjacent endoplasmic reticulum. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 28108248, 28527041, 36.80, 12, 'ENSG00000283887', 'Q16873'),
(1517, 'LTC4S', 'The MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) family includes a number of human proteins, several of which are involved the production of leukotrienes. This gene encodes an enzyme that catalyzes the first step in the biosynthesis of cysteinyl leukotrienes, potent biological compounds derived from arachidonic acid. Leukotrienes have been implicated as mediators of anaphylaxis and inflammatory conditions such as human bronchial asthma. This protein localizes to the nuclear envelope and adjacent endoplasmic reticulum. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 28108248, 28527041, 36.80, 12, 'ENSG00000213316', 'Q16873'),
(1518, 'LUC7L2', 'This gene encodes a protein that contains a C2H2-type zinc finger, coiled-coil region and arginine, serine-rich (RS) domain. A similar protein in mouse interacts with sodium channel modifier 1, and the encoded protein may be involved in the recognition of non-consensus splice donor sites in association with the U1 snRNP spliceosomal subunit. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Sep 2011]', '7', 'protein_coding', 95117355, 95197607, 38.99, 4, 'ENSG00000146963', 'Q9Y383'),
(1519, 'LUC7L3', 'This gene encodes a protein with an N-terminal half that contains cysteine/histidine motifs and leucine zipper-like repeats, and the C-terminal half is rich in arginine and glutamate residues (RE domain) and arginine and serine residues (RS domain). This protein localizes with a speckled pattern in the nucleus, and could be involved in the formation of splicesome via the RE and RS domains. Two alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Aug 2009]', '17', 'protein_coding', 83447228, 83618743, 40.80, 6, 'ENSG00000108848', 'O95232'),
(1520, 'LUM', 'This gene encodes a member of the small leucine-rich proteoglycan (SLRP) family that includes decorin, biglycan, fibromodulin, keratocan, epiphycan, and osteoglycin. In these bifunctional molecules, the protein moiety binds collagen fibrils and the highly charged hydrophilic glycosaminoglycans regulate interfibrillar spacings. Lumican is the major keratan sulfate proteoglycan of the cornea but is also distributed in interstitial collagenous matrices throughout the body. Lumican may regulate collagen fibril organization and circumferential growth, corneal transparency, and epithelial cell migration and tissue repair. [provided by RefSeq, Jul 2008]', '12', 'snoRNA', 28723682, 28723753, 41.67, 1, 'ENSG00000139329', 'P51884'),
(1521, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000204421', 'O95867'),
(1522, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000228859', 'O95867'),
(1523, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000236183', 'O95867'),
(1524, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000206398', 'O95867'),
(1525, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000228250', 'O95867'),
(1526, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000235452', 'O95867'),
(1527, 'LY6G6C', 'LY6G6C belongs to a cluster of leukocyte antigen-6 (LY6) genes located in the major histocompatibility complex (MHC) class III region on chromosome 6. Members of the LY6 superfamily typically contain 70 to 80 amino acids, including 8 to 10 cysteines. Most LY6 proteins are attached to the cell surface by a glycosylphosphatidylinositol (GPI) anchor that is directly involved in signal transduction (Mallya et al., 2002 [PubMed 12079290]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 32507820, 33058381, 45.93, 11, 'ENSG00000235925', 'O95867'),
(1528, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000204424', 'Q5SQ64'),
(1529, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000240008', 'Q5SQ64'),
(1530, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000239741', 'Q5SQ64'),
(1531, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000240957', 'Q5SQ64'),
(1532, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000243003', 'Q5SQ64'),
(1533, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000243804', 'Q5SQ64'),
(1534, 'LY6G6F', 'The human G6f protein is a type I transmembrane protein belonging to the immunoglobin (Ig) superfamily, which is comprised of cell-surface proteins involved in the immune system and cellular recognition (de Vet et al., 2003 [PubMed 12852788]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 55993681, 55994625, 45.71, 1, 'ENSG00000241822', 'Q5SQ64'),
(1535, 'LY6G6F-LY6G6D', 'This locus represents naturally occurring readthrough transcription between the neighboring LY6G6F (lymphocyte antigen 6 family member G6F) and LY6G6D (lymphocyte antigen 6 family member G6D) genes on chromosome 6. The readthrough transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jul 2017]', '6', 'unprocessed_pseudogene', 56097244, 56097384, 40.43, 1, 'ENSG00000250641', 'A0A0G2JK05'),
(1536, 'LY6G6F-LY6G6D', 'This locus represents naturally occurring readthrough transcription between the neighboring LY6G6F (lymphocyte antigen 6 family member G6F) and LY6G6D (lymphocyte antigen 6 family member G6D) genes on chromosome 6. The readthrough transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jul 2017]', '6', 'unprocessed_pseudogene', 56097244, 56097384, 40.43, 1, 'ENSG00000259101', 'A0A0G2JK05'),
(1537, 'LY6G6F-LY6G6D', 'This locus represents naturally occurring readthrough transcription between the neighboring LY6G6F (lymphocyte antigen 6 family member G6F) and LY6G6D (lymphocyte antigen 6 family member G6D) genes on chromosome 6. The readthrough transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jul 2017]', '6', 'unprocessed_pseudogene', 56097244, 56097384, 40.43, 1, 'ENSG00000258635', 'A0A0G2JK05'),
(1538, 'LY6G6F-LY6G6D', 'This locus represents naturally occurring readthrough transcription between the neighboring LY6G6F (lymphocyte antigen 6 family member G6F) and LY6G6D (lymphocyte antigen 6 family member G6D) genes on chromosome 6. The readthrough transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Jul 2017]', '6', 'unprocessed_pseudogene', 56097244, 56097384, 40.43, 1, 'ENSG00000258880', 'A0A0G2JK05'),
(1539, 'LY6H', 'Predicted to enable acetylcholine receptor binding activity and acetylcholine receptor inhibitor activity. Predicted to be involved in acetylcholine receptor signaling pathway. Predicted to be located in extracellular region. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 6924463, 6942321, 55.53, 4, 'ENSG00000274488', 'O94772'),
(1540, 'LY6H', 'Predicted to enable acetylcholine receptor binding activity and acetylcholine receptor inhibitor activity. Predicted to be involved in acetylcholine receptor signaling pathway. Predicted to be located in extracellular region. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 6924463, 6942321, 55.53, 4, 'ENSG00000176956', 'O94772'),
(1541, 'LY75', 'Predicted to enable signaling receptor activity. Predicted to be involved in immune response and inflammatory response. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 193398967, 193593111, 38.99, 11, 'ENSG00000054219', 'O60449'),
(1542, 'LYRM4', 'The protein encoded by this gene is found in both mitochondria and the nucleus, where it binds cysteine desulfurase and helps free inorganic sulfur for Fe/S clusters. Disruption of this gene negatively impacts mitochondrial and cytosolic iron homeostasis. [provided by RefSeq, Sep 2016]', '6', 'protein_coding', 59155732, 59206709, 44.35, 9, 'ENSG00000214113', 'Q9HD34'),
(1543, 'LYRM7', 'Inner mitochondrial membrane complex III (CIII) is the main enzyme complex in the mitochondrial respiratory chain, and Rieske Fe-S protein (UQCRFS1) is the last catalytic subunit added to the complex. The protein encoded by this gene is a nuclear-encoded mitochondrial matrix protein that stabilizes UQCRFS1 and chaperones it to the CIII complex. Defects in this gene are a cause of mitochondrial complex III deficiency, nuclear type 8. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jun 2014]', '5', 'miRNA', 45691704, 45691776, 50.68, 1, 'ENSG00000186687', 'Q5U5X0'),
(1544, 'LYSMD1', 'Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 1842, 18600, 47.07, 2, 'ENSG00000163155', 'Q96S90'),
(1545, 'LYSMD3', 'Enables peptidoglycan binding activity. Involved in Golgi organization. Located in Golgi membrane and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 24117153, 24495756, 38.87, 19, 'ENSG00000176018', 'Q7Z3D4'),
(1546, 'LYZ', 'This gene encodes human lysozyme, whose natural substrate is the bacterial cell wall peptidoglycan (cleaving the beta[1-4]glycosidic linkages between N-acetylmuramic acid and N-acetylglucosamine). Lysozyme is one of the antimicrobial agents found in human milk, and is also present in spleen, lung, kidney, white blood cells, plasma, saliva, and tears. The protein has antibacterial activity against a number of bacterial species. Missense mutations in this gene have been identified in heritable renal amyloidosis. [provided by RefSeq, Oct 2014]', '12', 'protein_coding', 38751534, 38785002, 45.01, 3, 'ENSG00000090382', 'P61626'),
(1547, 'LZTS2', 'The protein encoded by this gene belongs to the leucine zipper tumor suppressor family of proteins, which function in transcription regulation and cell cycle control. This family member can repress beta-catenin-mediated transcriptional activation and is a negative regulator of the Wnt signaling pathway. It negatively regulates microtubule severing at centrosomes, and is necessary for central spindle formation and cytokinesis completion. It is implicated in cancer, where it may inhibit cell proliferation and decrease susceptibility to tumor development. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Dec 2015]', '10', 'protein_coding', 64103802, 64107565, 62.54, 2, 'ENSG00000107816', 'Q9BRK4'),
(1548, 'M6PR', 'This gene encodes a receptor for both insulin-like growth factor 2 and mannose 6-phosphate. The binding sites for each ligand are located on different segments of the protein. This receptor has various functions, including in the intracellular trafficking of lysosomal enzymes, the activation of transforming growth factor beta, and the degradation of insulin-like growth factor 2. Mutation or loss of heterozygosity of this gene has been association with risk of hepatocellular carcinoma. The orthologous mouse gene is imprinted and shows exclusive expression from the maternal allele; however, imprinting of the human gene may be polymorphic, as only a minority of individuals showed biased expression from the maternal allele (PMID:8267611). [provided by RefSeq, Nov 2015]', '6', 'protein_coding', 67360328, 67386516, 37.53, 6, 'ENSG00000003056', 'P20645'),
(1549, 'MACROH2A2', 'Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene encodes a replication-independent histone that is a member of the histone H2A family. It replaces conventional H2A histones in a subset of nucleosomes where it represses transcription and may participate in stable X chromosome inactivation. [provided by RefSeq, Oct 2015]', '10', 'unprocessed_pseudogene', 19847367, 19848263, 41.81, 1, 'ENSG00000099284', 'Q9P0M6'),
(1550, 'MAFG', 'Globin gene expression is regulated through nuclear factor erythroid-2 (NFE2) elements located in enhancer-like locus control regions positioned many kb upstream of alpha- and beta-gene clusters (summarized by Blank et al., 1997 [PubMed 9166829]). NFE2 DNA-binding activity consists of a heterodimer containing a ubiquitous small Maf protein (MafF, MIM 604877; MafG; or MafK, MIM 600197) and the tissue-restricted protein p45 NFE2 (MIM 601490). Both subunits are members of the activator protein-1-like superfamily of basic leucine zipper (bZIP) proteins (see MIM 165160).[supplied by OMIM, Mar 2010]', '17', 'protein_coding', 9910606, 10198606, 47.97, 19, 'ENSG00000197063', 'O15525'),
(1551, 'MAFIP', 'This gene was originally thought to be a protein coding gene. However, the encoded protein sequence is highly similar to the C-terminal sequence of the tektin-4 protein, and the transcript sequences of this gene are highly similar to the TEKT4 pseudogenes, which are found on chromosomes 4, 21 and Y, respectively. Therefore, this gene is thought to be another pseudogene of the TEKT4 gene (GeneID:150483). Multiple alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Mar 2012]', '14', 'protein_coding', 53594, 115055, 45.64, 1, 'ENSG00000274847', 'Q8WZ33'),
(1552, 'MAGEA10', 'This gene is a member of the MAGEA gene family. The members of this family encode proteins with 50 to 80% sequence identity to each other. The promoters and first exons of the MAGEA genes show considerable variability, suggesting that the existence of this gene family enables the same function to be expressed under different transcriptional controls. The MAGEA genes are clustered at chromosomal location Xq28. They have been implicated in some hereditary disorders, such as dyskeratosis congenita. Alternative splicing results in multiple transcript variants. Read-through transcription also exists between this gene and the downstream melanoma antigen family A, 5 (MAGEA5) gene.[provided by RefSeq, Oct 2011]', 'X', 'protein_coding', 133233278, 133276024, 50.14, 7, 'ENSG00000124260', 'P43363'),
(1553, 'MAGEB5', 'Predicted to be involved in negative regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 30908207, 30914106, 51.20, 6, 'ENSG00000188408', 'Q9BZ81'),
(1554, 'MAGEC3', 'This gene is a member of the MAGEC gene family. The members of this family are not expressed in normal tissues, except for testis, and are expressed in tumors of various histological types. The MAGEC genes are clustered on chromosome Xq26-q27. Two transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 35699272, 35700984, 50.26, 5, 'ENSG00000165509', 'Q8TD91'),
(1555, 'MAGED2', 'This gene is a member of the MAGED gene family. The MAGED genes are clustered on chromosome Xp11. This gene is located in Xp11.2, a hot spot for X-linked intellectual disability (XLID). Mutations in this gene cause a form of transient antenatal Bartter\'s syndrome. This gene may also be involved in several types of cancer, including breast cancer and melanoma. The protein encoded by this gene is progressively recruited from the cytoplasm to the nucleoplasm during the interphase and after nucleolar stress and is thus thought to play a role in cell cycle regulation. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2017]', 'X', 'protein_coding', 49785660, 49787285, 45.39, 1, 'ENSG00000102316', 'Q9UNF1'),
(1556, 'MAGED4', 'Predicted to be involved in negative regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 67294371, 67360265, 38.54, 14, 'ENSG00000154545', 'Q96JG8'),
(1557, 'MAGED4B', 'This gene is a member of the MAGED gene family. It is expressed only in brain and ovary, and some transcript variants of this gene are specifically expressed in glioma cells. This gene is clustered with other MAGED genes on chromosome Xp11. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, May 2011]', 'X', 'protein_coding', 49727376, 49744437, 34.47, 4, 'ENSG00000187243', 'Q96JG8'),
(1558, 'MAGI3', 'Predicted to enable frizzled binding activity. Predicted to be involved in signal transduction. Predicted to act upstream of or within positive regulation of JUN kinase activity. Located in cell junction. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 2844, 11554, 48.88, 2, 'ENSG00000081026', 'Q5TCQ9'),
(1559, 'MAGOH', 'Drosophila that have mutations in their mago nashi (grandchildless) gene produce progeny with defects in germplasm assembly and germline development. This gene encodes the mammalian mago nashi homolog. In mammals, mRNA expression is not limited to the germ plasm, but is expressed ubiquitously in adult tissues and can be induced by serum stimulation of quiescent fibroblasts. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 66002228, 66004149, 57.86, 8, 'ENSG00000162385', 'P61326'),
(1560, 'MAK', 'The product of this gene is a serine/threonine protein kinase related to kinases involved in cell cycle regulation. Studies of the mouse and rat homologs have localized the kinase to the chromosomes during meiosis in spermatogenesis, specifically to the synaptonemal complex that exists while homologous chromosomes are paired. Mutations in this gene have been associated with ciliary defects resulting in retinitis pigmentosa 62. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 133379261, 133391694, 52.74, 10, 'ENSG00000111837', 'P20794'),
(1561, 'MALT1', 'This gene encodes a caspase-like protease that plays a role in BCL10-induced activation of NF-kappaB. The protein is a component of the CARMA1-BCL10-MALT1 (CBM) signalosome that triggers NF-kappaB signaling and lymphoctye activation following antigen-receptor stimulation. Mutations in this gene result in immunodeficiency 12 (IMD12). This gene has been found to be recurrently rearranged in chromosomal translocations with other genes in mucosa-associated lymphoid tissue lymphomas, including a t(11;18)(q21;q21) translocation with the baculoviral IAP repeat-containing protein 3 (also known as apoptosis inhibitor 2) locus [BIRC3(API2)-MALT1], and a t(14;18)(q32;q21) translocation with the immunoglobulin heavy chain locus (IGH-MALT1). Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, May 2018]', '18', 'unprocessed_pseudogene', 192384, 193338, 36.96, 1, 'ENSG00000172175', 'Q9UDY8'),
(1562, 'MAML1', 'This protein is the human homolog of mastermind, a Drosophila protein that plays a role in the Notch signaling pathway involved in cell-fate determination. There is in vitro evidence that the human homolog forms a complex with the intracellular portion of human Notch receptors and can increase expression of a Notch-induced gene. This evidence supports its proposed function as a transcriptional co-activator in the Notch signaling pathway. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 74671855, 74700960, 45.73, 8, 'ENSG00000283780', 'Q92585'),
(1563, 'MAML1', 'This protein is the human homolog of mastermind, a Drosophila protein that plays a role in the Notch signaling pathway involved in cell-fate determination. There is in vitro evidence that the human homolog forms a complex with the intracellular portion of human Notch receptors and can increase expression of a Notch-induced gene. This evidence supports its proposed function as a transcriptional co-activator in the Notch signaling pathway. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 74671855, 74700960, 45.73, 8, 'ENSG00000161021', 'Q92585'),
(1564, 'MAN1A1', 'This gene encodes a class I mammalian Golgi 1,2-mannosidase which is a type II transmembrane protein. This protein catalyzes the hydrolysis of three terminal mannose residues from peptide-bound Man(9)-GlcNAc(2) oligosaccharides and belongs to family 47 of glycosyl hydrolases. [provided by RefSeq, Jul 2012]', '6', 'protein_coding', 73644881, 73663803, 50.61, 3, 'ENSG00000111885', 'P33908'),
(1565, 'MAP1LC3A', 'MAP1A and MAP1B are microtubule-associated proteins which mediate the physical interactions between microtubules and components of the cytoskeleton. MAP1A and MAP1B each consist of a heavy chain subunit and multiple light chain subunits. The protein encoded by this gene is one of the light chain subunits and can associate with either MAP1A or MAP1B. Two transcript variants encoding different isoforms have been found for this gene. The expression of variant 1 is suppressed in many tumor cell lines, suggesting that may be involved in carcinogenesis. [provided by RefSeq, Feb 2012]', '20', 'unprocessed_pseudogene', 6128610, 6129556, 49.42, 1, 'ENSG00000101460', 'Q9H492'),
(1566, 'MAP1LC3B2', 'Predicted to enable microtubule binding activity; phosphatidylethanolamine binding activity; and ubiquitin protein ligase binding activity. Predicted to be involved in cellular response to nitrogen starvation and macroautophagy. Predicted to be located in several cellular components, including cytoplasmic vesicle; endomembrane system; and microtubule. Predicted to be active in autophagosome membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'IG_V_gene', 6058, 6610, 42.86, 1, 'ENSG00000258102', 'A6NCE7'),
(1567, 'MAP3K14', 'This gene encodes mitogen-activated protein kinase kinase kinase 14, which is a serine/threonine protein-kinase. This kinase binds to TRAF2 and stimulates NF-kappaB activity. It shares sequence similarity with several other MAPKK kinases. It participates in an NF-kappaB-inducing signalling cascade common to receptors of the tumour-necrosis/nerve-growth factor (TNF/NGF) family and to the interleukin-1 type-I receptor. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 170330761, 170330844, 59.52, 1, 'ENSG00000006062', 'Q99558'),
(1568, 'MAP3K14', 'This gene encodes mitogen-activated protein kinase kinase kinase 14, which is a serine/threonine protein-kinase. This kinase binds to TRAF2 and stimulates NF-kappaB activity. It shares sequence similarity with several other MAPKK kinases. It participates in an NF-kappaB-inducing signalling cascade common to receptors of the tumour-necrosis/nerve-growth factor (TNF/NGF) family and to the interleukin-1 type-I receptor. [provided by RefSeq, Jul 2008]', '17', 'miRNA', 170330761, 170330844, 59.52, 1, 'ENSG00000282637', 'Q99558'),
(1569, 'MAP3K20', 'This gene is a member of the MAPKKK family of signal transduction molecules and encodes a protein with an N-terminal kinase catalytic domain, followed by a leucine zipper motif and a sterile-alpha motif (SAM). This magnesium-binding protein forms homodimers and is located in the cytoplasm. The protein mediates gamma radiation signaling leading to cell cycle arrest and activity of this protein plays a role in cell cycle checkpoint regulation in cells. The protein also has pro-apoptotic activity. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 24311450, 24318036, 53.67, 5, 'ENSG00000091436', 'Q9NYL2'),
(1570, 'MAPK13', 'This gene encodes a member of the mitogen-activated protein (MAP) kinase family. MAP kinases act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. The encoded protein is a p38 MAP kinase and is activated by proinflammatory cytokines and cellular stress. Substrates of the encoded protein include the transcription factor ATF2 and the microtubule dynamics regulator stathmin. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Jul 2012]', '6', 'protein_coding', 33235995, 33243311, 49.38, 3, 'ENSG00000156711', 'O15264'),
(1571, 'MAPK15', 'Enables MAP kinase activity and chromatin binding activity. Involved in several processes, including dopamine uptake; protein localization to ciliary transition zone; and regulation of organelle organization. Located in several cellular components, including Golgi apparatus; autophagosome; and microtubule organizing center. Biomarker of breast cancer. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 4669774, 4704337, 48.03, 15, 'ENSG00000181085', 'Q8TD08'),
(1572, 'MAPK15', 'Enables MAP kinase activity and chromatin binding activity. Involved in several processes, including dopamine uptake; protein localization to ciliary transition zone; and regulation of organelle organization. Located in several cellular components, including Golgi apparatus; autophagosome; and microtubule organizing center. Biomarker of breast cancer. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 4669774, 4704337, 48.03, 15, 'ENSG00000274205', 'Q8TD08'),
(1573, 'MAPRE1', 'The protein encoded by this gene was first identified by its binding to the APC protein which is often mutated in familial and sporadic forms of colorectal cancer. This protein localizes to microtubules, especially the growing ends, in interphase cells. During mitosis, the protein is associated with the centrosomes and spindle microtubules. The protein also associates with components of the dynactin complex and the intermediate chain of cytoplasmic dynein. Because of these associations, it is thought that this protein is involved in the regulation of microtubule structures and chromosome stability. This gene is a member of the RP/EB family. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 1428465, 1459470, 63.37, 15, 'ENSG00000101367', 'Q15691'),
(1574, 'MARCHF11', 'MARCH11 is a member of the MARCH family of membrane-bound E3 ubiquitin ligases (EC 6.3.2.19). These enzymes add ubiquitin (see MIM 191339) to target lysines in substrate proteins, thereby signaling their intracellular transport. March11 appears to have a role in ubiquitin-mediated protein sorting in the trans-Golgi network (TGN)-multivesicular body (MVB) transport pathway (Morokuma et al., 2007 [PubMed 17604280]).[supplied by OMIM, Apr 2010]', '5', 'protein_coding', 101354033, 101557321, 38.96, 7, 'ENSG00000183654', 'A6NNE9'),
(1575, 'MARCHF5', 'MARCH5 is a ubiquitin ligase of the mitochondrial outer membrane that plays a role in the control of mitochondrial morphology by regulating mitofusin-2 (MFN2; MIM 608507) and DRP1 (DNM1L; MIM 603850) (Nakamura et al., 2006 [PubMed 16936636]).[supplied by OMIM, Mar 2008]', '10', 'protein_coding', 15086, 18803, 50.35, 4, 'ENSG00000198060', 'Q9NX47'),
(1576, 'MARVELD2', 'The protein encoded by this gene is a membrane protein found at the tight junctions between epithelial cells. The encoded protein helps establish epithelial barriers such as those in the organ of Corti, where these barriers are required for normal hearing. Defects in this gene are a cause of deafness autosomal recessive type 49 (DFNB49). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]', '5', 'protein_coding', 68230595, 68232113, 53.85, 1, 'ENSG00000152939', 'Q8N4S9'),
(1577, 'MARVELD2', 'The protein encoded by this gene is a membrane protein found at the tight junctions between epithelial cells. The encoded protein helps establish epithelial barriers such as those in the organ of Corti, where these barriers are required for normal hearing. Defects in this gene are a cause of deafness autosomal recessive type 49 (DFNB49). Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]', '5', 'protein_coding', 68230595, 68232113, 53.85, 1, 'ENSG00000274671', 'Q8N4S9'),
(1578, 'MAVS', 'This gene encodes an intermediary protein necessary in the virus-triggered beta interferon signaling pathways. It is required for activation of transcription factors which regulate expression of beta interferon and contributes to antiviral innate immunity. [provided by RefSeq, Jul 2020]', '20', 'protein_coding', 18990888, 19034023, 50.00, 15, 'ENSG00000088888', 'Q7Z434'),
(1579, 'MB21D2', 'Enables cadherin binding activity. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 76397518, 76442071, 50.47, 7, 'ENSG00000180611', 'Q8IYB1'),
(1580, 'MBD2', 'DNA methylation is the major modification of eukaryotic genomes and plays an essential role in mammalian development. Human proteins MECP2, MBD1, MBD2, MBD3, and MBD4 comprise a family of nuclear proteins related by the presence in each of a methyl-CpG binding domain (MBD). Each of these proteins, with the exception of MBD3, is capable of binding specifically to methylated DNA. MECP2, MBD1 and MBD2 can also repress transcription from methylated gene promoters. The protein encoded by this gene may function as a mediator of the biological consequences of the methylation signal. It is also reported that the this protein functions as a demethylase to activate transcription, as DNA methylation causes gene silencing. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2011]', '18', 'protein_coding', 19738, 37393, 45.88, 10, 'ENSG00000134046', 'Q9UBB5'),
(1581, 'MBL2', 'This gene encodes the soluble mannose-binding lectin or mannose-binding protein found in serum. The protein encoded belongs to the collectin family and is an important element in the innate immune system. The protein recognizes and binds to mannose and N-acetylglucosamine on many microorganisms, including bacteria, yeast, and viruses including influenza virus, HIV and SARS-CoV. This binding activates the classical complement pathway. Deficiencies of this gene have been associated with susceptibility to autoimmune and infectious diseases. [provided by RefSeq, Jun 2020]', '10', 'transcribed_unprocessed_pseudogene', 411954, 412879, 44.17, 1, 'ENSG00000165471', 'P11226');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1582, 'MBLAC1', 'Enables RNA endonuclease activity and metal ion binding activity. Involved in histone mRNA metabolic process; mRNA 3\'-end processing; and positive regulation of G1/S transition of mitotic cell cycle. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 55339853, 55348121, 64.87, 12, 'ENSG00000214309', 'A4D2B0'),
(1583, 'MBNL2', 'This gene is a member of the muscleblind protein family which was initially described in Drosophila melanogaster. This gene encodes a C3H-type zinc finger protein that modulates alternative splicing of pre-mRNAs. Muscleblind proteins bind specifically to expanded dsCUG RNA but not to normal size CUG repeats and may thereby play a role in the pathophysiology of myotonic dystrophy. Several alternatively spliced transcript variants have been described but the full-length natures of only some have been determined. [provided by RefSeq, Mar 2012]', '13', 'protein_coding', 60751791, 60844274, 39.18, 7, 'ENSG00000139793', 'Q5VZF2'),
(1584, 'MBP', 'This gene encodes the soluble mannose-binding lectin or mannose-binding protein found in serum. The protein encoded belongs to the collectin family and is an important element in the innate immune system. The protein recognizes and binds to mannose and N-acetylglucosamine on many microorganisms, including bacteria, yeast, and viruses including influenza virus, HIV and SARS-CoV. This binding activates the classical complement pathway. Deficiencies of this gene have been associated with susceptibility to autoimmune and infectious diseases. [provided by RefSeq, Jun 2020]', '10', 'snoRNA', 39313819, 39313911, 59.14, 1, 'ENSG00000197971', 'P02686'),
(1585, 'MBTPS2', 'This gene encodes a intramembrane zinc metalloprotease, which is essential in development. This protease functions in the signal protein activation involved in sterol control of transcription and the ER stress response. Mutations in this gene have been associated with ichthyosis follicularis with atrichia and photophobia (IFAP syndrome); IFAP syndrome has been quantitatively linked to a reduction in cholesterol homeostasis and ER stress response.[provided by RefSeq, Aug 2009]', 'X', 'miRNA', 63178500, 63178586, 54.02, 1, 'ENSG00000012174', 'O43462'),
(1586, 'MCHR1', 'The protein encoded by this gene, a member of the G protein-coupled receptor family 1, is an integral plasma membrane protein which binds melanin-concentrating hormone. The encoded protein can inhibit cAMP accumulation and stimulate intracellular calcium flux, and is probably involved in the neuronal regulation of food consumption. Although structurally similar to somatostatin receptors, this protein does not seem to bind somatostatin. [provided by RefSeq, Jul 2008]', '22', 'unprocessed_pseudogene', 21601882, 21602825, 52.44, 1, 'ENSG00000128285', 'Q99705'),
(1587, 'MCRIP1', 'Involved in regulation of epithelial to mesenchymal transition. Located in cytoplasmic stress granule and nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 72708367, 72727629, 38.81, 1, 'ENSG00000288213', 'C9JLW8'),
(1588, 'MCRIP1', 'Involved in regulation of epithelial to mesenchymal transition. Located in cytoplasmic stress granule and nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 72708367, 72727629, 38.81, 1, 'ENSG00000225663', 'C9JLW8'),
(1589, 'ME2', 'This gene encodes a mitochondrial NAD-dependent malic enzyme, a homotetrameric protein, that catalyzes the oxidative decarboxylation of malate to pyruvate. It had previously been weakly linked to a syndrome known as Friedreich ataxia that has since been shown to be the result of mutation in a completely different gene. Certain single-nucleotide polymorphism haplotypes of this gene have been shown to increase the risk for idiopathic generalized epilepsy. Alternatively spliced transcript variants encoding different isoforms found for this gene. [provided by RefSeq, Dec 2009]', '18', 'protein_coding', 20768, 35515, 41.81, 4, 'ENSG00000082212', 'P23368'),
(1590, 'MEA1', 'Predicted to be involved in male gonad development and spermatogenesis. Predicted to be located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 36036913, 36124455, 39.23, 3, 'ENSG00000124733', 'Q16626'),
(1591, 'MED17', 'The activation of gene transcription is a multistep process that is triggered by factors that recognize transcriptional enhancer sites in DNA. These factors work with co-activators to direct transcriptional initiation by the RNA polymerase II apparatus. The protein encoded by this gene is a subunit of the CRSP (cofactor required for SP1 activation) complex, which, along with TFIID, is required for efficient activation by SP1. This protein is also a component of other multisubunit complexes e.g. thyroid hormone receptor-(TR-) associated proteins which interact with TR and facilitate TR function on DNA templates in conjunction with initiation factors and cofactors. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 165772904, 165837462, 32.29, 5, 'ENSG00000042429', 'Q9NVC6'),
(1592, 'MED26', 'The activation of gene transcription is a multistep process that is triggered by factors that recognize transcriptional enhancer sites in DNA. These factors work with co-activators to direct transcriptional initiation by the RNA polymerase II apparatus. The protein encoded by this gene is a subunit of the CRSP (cofactor required for SP1 activation) complex, which, along with TFIID, is required for efficient activation by SP1. This protein is also a component of other multisubunit complexes e.g. thyroid hormone receptor-(TR-) associated proteins which interact with TR and facilitate TR function on DNA templates in conjunction with initiation factors and cofactors. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 24429286, 24441843, 52.18, 6, 'ENSG00000105085', 'O95402'),
(1593, 'MED29', 'MED29 is a subunit of the Mediator complex, a multiprotein coactivator of RNA transcription that interacts with DNA-bound transcriptional activators, RNA polymerase II (see MIM 180660), and general initiation factors (Sato et al., 2003 [PubMed 14576168]).[supplied by OMIM, Aug 2009]', '19', 'miRNA', 100093560, 100093643, 63.10, 1, 'ENSG00000063322', 'Q9NX70'),
(1594, 'MED4', 'This gene encodes a component of the Mediator complex. The Mediator complex interacts with DNA-binding gene-specific transcription factors to modulate transcription by RNA polymerase II. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jul 2012]', '13', 'protein_coding', 39174769, 39183488, 40.53, 5, 'ENSG00000136146', 'Q9NPJ6'),
(1595, 'MEGF9', 'Predicted to be located in basement membrane and membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 136461, 153231, 47.06, 2, 'ENSG00000106780', 'Q9H1U4'),
(1596, 'MEIOC', 'Predicted to be involved in several processes, including germ cell development; germline cell cycle switching, mitotic to meiotic cell cycle; and meiosis I. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'IG_V_gene', 678705, 679222, 52.32, 1, 'ENSG00000180336', 'A2RUB1'),
(1597, 'MEST', 'This gene encodes a member of the alpha/beta hydrolase superfamily. It is imprinted, exhibiting preferential expression from the paternal allele in fetal tissues, and isoform-specific imprinting in lymphocytes. The loss of imprinting of this gene has been linked to certain types of cancer and may be due to promotor switching. The encoded protein may play a role in development. Alternatively spliced transcript variants encoding multiple isoforms have been identified for this gene. Pseudogenes of this gene are located on the short arm of chromosomes 3 and 4, and the long arm of chromosomes 6 and 15. [provided by RefSeq, Dec 2011]', '7', 'protein_coding', 39314698, 39322968, 54.72, 5, 'ENSG00000106484', 'Q5EB52'),
(1598, 'METAP1D', 'The N-terminal methionine excision pathway is an essential process in which the N-terminal methionine is removed from many proteins, thus facilitating subsequent protein modification. In mitochondria, enzymes that catalyze this reaction are celled methionine aminopeptidases (MetAps, or MAPs; EC 3.4.11.18) (Serero et al., 2003 [PubMed 14532271]).[supplied by OMIM, Mar 2008]', '2', 'protein_coding', 79512533, 79516440, 46.26, 1, 'ENSG00000292097', 'Q6UB28'),
(1599, 'METAP1D', 'The N-terminal methionine excision pathway is an essential process in which the N-terminal methionine is removed from many proteins, thus facilitating subsequent protein modification. In mitochondria, enzymes that catalyze this reaction are celled methionine aminopeptidases (MetAps, or MAPs; EC 3.4.11.18) (Serero et al., 2003 [PubMed 14532271]).[supplied by OMIM, Mar 2008]', '2', 'protein_coding', 79512533, 79516440, 46.26, 1, 'ENSG00000172878', 'Q6UB28'),
(1600, 'METAP2', 'The protein encoded by this gene is a member of the methionyl aminopeptidase family. The encoded protein functions both by protecting the alpha subunit of eukaryotic initiation factor 2 from inhibitory phosphorylation and by removing the amino-terminal methionine residue from nascent proteins. Increased expression of this gene is associated with various forms of cancer, and the anti-cancer drugs fumagillin and ovalicin inhibit the protein by irreversibly binding to its active site. Inhibitors of this gene have also been shown to be effective for the treatment of obesity. A pseudogene of this gene is located on chromosome 2. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2015]', '12', 'unitary_pseudogene', 5551662, 5552563, 45.23, 1, 'ENSG00000111142', 'P50579'),
(1601, 'METRNL', 'Predicted to enable hormone activity. Predicted to be involved in several processes, including brown fat cell differentiation; positive regulation of brown fat cell differentiation; and response to muscle activity. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 2655726, 2805017, 36.81, 27, 'ENSG00000176845', 'Q641Q3'),
(1602, 'METRNL', 'Predicted to enable hormone activity. Predicted to be involved in several processes, including brown fat cell differentiation; positive regulation of brown fat cell differentiation; and response to muscle activity. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 2655726, 2805017, 36.81, 27, 'ENSG00000275031', 'Q641Q3'),
(1603, 'METTL1', 'This gene is similar in sequence to the S. cerevisiae YDL201w gene. The gene product contains a conserved S-adenosylmethionine-binding motif and is inactivated by phosphorylation. Alternative splice variants encoding different protein isoforms have been described for this gene. A pseudogene has been identified on chromosome X. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 150716668, 150743646, 42.98, 1, 'ENSG00000037897', 'Q9UBP6'),
(1604, 'METTL15', 'Enables rRNA (cytosine-N4-)-methyltransferase activity. Involved in rRNA base methylation. Is active in mitochondrial matrix. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 136945185, 136946975, 65.27, 6, 'ENSG00000169519', 'A6NJ78'),
(1605, 'METTL27', 'This gene encodes a protein belonging to ubiE/COQ5 methyltransferase family. The gene is deleted in Williams syndrome, a multisystem developmental disorder caused by the deletion of contiguous genes at 7q11.22-q11.23. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 48552172, 48599425, 51.71, 4, 'ENSG00000165171', 'Q8N6F8'),
(1606, 'MEX3C', 'This gene encodes a member of a family of proteins with two K homology (KH) RNA-binding domains and a C-terminal RING-finger domain. The protein interacts with mRNA via the KH domains, and the protein shuttles between the nucleus and cytoplasm. Polymorphisms in this gene may contribute to hypertension. [provided by RefSeq, Oct 2009]', '18', 'protein_coding', 9771153, 10054936, 45.33, 13, 'ENSG00000176624', 'Q5U5Q3'),
(1607, 'MFAP3L', 'Located in several cellular components, including cell junction; nucleoplasm; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'snoRNA', 52713249, 52713370, 45.90, 1, 'ENSG00000198948', 'O75121'),
(1608, 'MFAP4', 'This gene encodes a protein with similarity to a bovine microfibril-associated protein. The protein has binding specificities for both collagen and carbohydrate. It is thought to be an extracellular matrix protein which is involved in cell adhesion or intercellular interactions. The gene is located within the Smith-Magenis syndrome region. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2010]', '17', 'protein_coding', 1688574, 1788674, 39.40, 7, 'ENSG00000166482', 'P55083'),
(1609, 'MFN1', 'The protein encoded by this gene is a mediator of mitochondrial fusion. This protein and mitofusin 2 are homologs of the Drosophila protein fuzzy onion (Fzo). They are mitochondrial membrane proteins that interact with each other to facilitate mitochondrial targeting. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 165233, 169508, 55.29, 3, 'ENSG00000171109', 'Q8IWA4'),
(1610, 'MFSD14B', 'Predicted to enable transmembrane transporter activity. Predicted to be involved in transmembrane transport. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 6618835, 6635960, 51.24, 3, 'ENSG00000148110', 'Q5SR56'),
(1611, 'MGARP', 'Predicted to be involved in several processes, including axonal transport; cellular response to gonadotropin-releasing hormone; and nervous system development. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 276322, 303356, 53.82, 10, 'ENSG00000137463', 'Q8TDB4'),
(1612, 'MGAT3', 'There are believed to be over 100 different glycosyltransferases involved in the synthesis of protein-bound and lipid-bound oligosaccharides. The enzyme encoded by this gene transfers a GlcNAc residue to the beta-linked mannose of the trimannosyl core of N-linked oligosaccharides and produces a bisecting GlcNAc. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Jul 2008]', '22', 'transcribed_unitary_pseudogene', 101624, 103504, 53.59, 1, 'ENSG00000128268', 'Q09327'),
(1613, 'MGMT', 'Alkylating agents are potent carcinogens that can result in cell death, mutation and cancer. The protein encoded by this gene is a DNA repair protein that is involved in cellular defense against mutagenesis and toxicity from alkylating agents. The protein catalyzes transfer of methyl groups from O(6)-alkylguanine and other methylated moieties of the DNA to its own molecule, which repairs the toxic lesions. Methylation of the genes promoter has been associated with several cancer types, including colorectal cancer, lung cancer, lymphoma and glioblastoma. [provided by RefSeq, Sep 2015]', '10', 'protein_coding', 47501854, 47523638, 40.48, 1, 'ENSG00000170430', 'P16455'),
(1614, 'MGST1', 'The MAPEG (Membrane Associated Proteins in Eicosanoid and Glutathione metabolism) family consists of six human proteins, two of which are involved in the production of leukotrienes and prostaglandin E, important mediators of inflammation. Other family members, demonstrating glutathione S-transferase and peroxidase activities, are involved in cellular defense against toxic, carcinogenic, and pharmacologically active electrophilic compounds. This gene encodes a protein that catalyzes the conjugation of glutathione to electrophiles and the reduction of lipid hydroperoxides. This protein is localized to the endoplasmic reticulum and outer mitochondrial membrane where it is thought to protect these membranes from oxidative stress. Several transcript variants, some non-protein coding and some protein coding, have been found for this gene. [provided by RefSeq, May 2012]', '12', 'protein_coding', 89177875, 89295759, 35.29, 3, 'ENSG00000008394', 'P10620'),
(1615, 'MICAL3', 'Enables actin binding activity and molecular adaptor activity. Involved in actin filament depolymerization. Located in several cellular components, including Flemming body; intercellular bridge; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 63123346, 63320128, 42.74, 9, 'ENSG00000243156', 'Q7RTP6'),
(1616, 'MICALL1', 'Enables several functions, including identical protein binding activity; phosphatidic acid binding activity; and small GTPase binding activity. Involved in several processes, including plasma membrane tubulation; protein localization to organelle; and slow endocytic recycling. Located in centriole; cilium; and endosome. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 5148233, 5148322, 57.78, 1, 'ENSG00000100139', 'Q8N3F8'),
(1617, 'MICALL2', 'Enables filamin binding activity. Involved in positive regulation of protein targeting to mitochondrion. Predicted to be located in several cellular components, including bicellular tight junction; recycling endosome; and stress fiber. [provided by Alliance of Genome Resources, May 2025]', '7', 'processed_pseudogene', 139362040, 139362382, 48.40, 1, 'ENSG00000164877', 'Q8IY33'),
(1618, 'MICU1', 'This gene encodes an essential regulator of mitochondrial Ca2+ uptake under basal conditions. The encoded protein interacts with the mitochondrial calcium uniporter, a mitochondrial inner membrane Ca2+ channel, and is essential in preventing mitochondrial Ca2+ overload, which can cause excessive production of reactive oxygen species and cell stress. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Mar 2013]', '10', 'protein_coding', 134599923, 134651636, 48.47, 5, 'ENSG00000107745', 'Q9BPX6'),
(1619, 'MIEF1', 'Enables identical protein binding activity; mitochondrial large ribosomal subunit binding activity; and purine ribonucleotide binding activity. Involved in several processes, including mitochondrial large ribosomal subunit assembly; mitochondrion organization; and regulation of mitochondrion organization. Located in mitochondrial matrix and mitochondrial outer membrane. Implicated in optic atrophy. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 55307989, 55425579, 36.82, 9, 'ENSG00000100335', 'L0R8F8'),
(1620, 'MIER3', 'Predicted to enable histone deacetylase binding activity and transcription corepressor activity. Predicted to be involved in negative regulation of transcription by RNA polymerase II. Located in nucleoplasm. Part of protein-containing complex. Implicated in breast cancer and lung adenocarcinoma. Biomarker of breast cancer; invasive ductal carcinoma; and lung non-small cell carcinoma. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 129094142, 129111189, 58.61, 22, 'ENSG00000155545', 'Q7Z3K6'),
(1621, 'MINPP1', 'This gene encodes multiple inositol polyphosphate phosphatase; an enzyme that removes 3-phosphate from inositol phosphate substrates. It is the only enzyme known to hydrolzye inositol pentakisphosphate and inositol hexakisphosphate. This enzyme also converts 2,3 bisphosphoglycerate (2,3-BPG) to 2-phosphoglycerate; an activity formerly thought to be exclusive to 2,3-BPG synthase/2-phosphatase (BPGM) in the Rapoport-Luebering shunt of the glycolytic pathway.[provided by RefSeq, Sep 2009]', '10', 'IG_V_gene', 106153624, 106154163, 50.56, 1, 'ENSG00000107789', 'Q9UNW1'),
(1622, 'MIPEP', 'The product of this gene performs the final step in processing a specific class of nuclear-encoded proteins targeted to the mitochondrial matrix or inner membrane. This protein is primarily involved in the maturation of oxidative phosphorylation (OXPHOS)-related proteins. This gene may contribute to the functional effects of frataxin deficiency and the clinical manifestations of Friedreich ataxia. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 21622, 22569, 45.36, 1, 'ENSG00000027001', 'Q99797'),
(1623, 'MIR101-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 19146993, 19150292, 64.33, 1, 'ENSG00000199065', ''),
(1624, 'MIR103B2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 106813871, 107000212, 39.76, 4, 'ENSG00000283320', ''),
(1625, 'MIR1184-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 127397138, 127408424, 57.79, 15, 'ENSG00000221190', ''),
(1626, 'MIR1185-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 71212811, 71225516, 50.82, 11, 'ENSG00000221525', ''),
(1627, 'MIR1200', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 24558087, 24672677, 42.38, 4, 'ENSG00000221325', ''),
(1628, 'MIR1224', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'miRNA', 1986384, 1986477, 67.02, 1, 'ENSG00000221120', ''),
(1629, 'MIR1233-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '15', 'protein_coding', 108866898, 108934328, 39.99, 48, 'ENSG00000221649', ''),
(1630, 'MIR1233-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '15', 'protein_coding', 108866898, 108934328, 39.99, 48, 'ENSG00000291989', ''),
(1631, 'MIR124-3', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 77716097, 78031811, 39.18, 2, 'ENSG00000207598', ''),
(1632, 'MIR1261', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 98252379, 98310441, 51.97, 7, 'ENSG00000221586', ''),
(1633, 'MIR1267', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 107163510, 107867496, 37.44, 1, 'ENSG00000221650', ''),
(1634, 'MIR1289-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 74498183, 74612378, 51.68, 6, 'ENSG00000221763', ''),
(1635, 'MIR1293', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 207820, 218775, 59.95, 4, 'ENSG00000221604', ''),
(1636, 'MIR1294', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'lncRNA', 91347686, 91397592, 37.82, 20, 'ENSG00000221430', ''),
(1637, 'MIR1297', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 84648511, 84667686, 51.32, 7, 'ENSG00000283126', ''),
(1638, 'MIR1302-11', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'IG_V_gene', 865340, 865824, 55.88, 1, 'ENSG00000283801', ''),
(1639, 'MIR1321', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 106639814, 106639896, 38.55, 1, 'ENSG00000283435', ''),
(1640, 'MIR1322', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 12166991, 12209228, 55.40, 4, 'ENSG00000284875', ''),
(1641, 'MIR1322', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 12166991, 12209228, 55.40, 4, 'ENSG00000283210', ''),
(1642, 'MIR1323', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000221017', ''),
(1643, 'MIR133A1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 95867048, 95942635, 39.85, 8, 'ENSG00000283927', ''),
(1644, 'MIR133A1HG', 'Predicted to be involved in miRNA-mediated post-transcriptional gene silencing. Predicted to be part of RISC complex. [provided by Alliance of Genome Resources, May 2025]', '18', 'lncRNA', 116398157, 116400606, 39.47, 1, 'ENSG00000265142', ''),
(1645, 'MIR133A2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 28820339, 29505947, 40.72, 6, 'ENSG00000284508', ''),
(1646, 'MIR1343', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 141002610, 141015228, 41.83, 9, 'ENSG00000251862', ''),
(1647, 'MIR135A2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 101728648, 101739472, 36.71, 5, 'ENSG00000207586', ''),
(1648, 'MIR1468', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 177890, 184284, 54.26, 5, 'ENSG00000222532', ''),
(1649, 'MIR146A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. Some of the targets of the encoded miRNA are the transcripts for tumor necrosis factor, interleukin 1 receptor-associated kinase 1, interleukin 1-beta, TNF receptor-associated factor 6, and complement factor H. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2015]', '5', 'protein_coding', 24149173, 24216255, 40.77, 11, 'ENSG00000283733', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1650, 'MIR1538', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 74513463, 74517148, 65.00, 4, 'ENSG00000223109', ''),
(1651, 'MIR154', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 101915010, 102001062, 36.41, 5, 'ENSG00000207978', ''),
(1652, 'MIR155', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 43914852, 44069616, 47.56, 6, 'ENSG00000283904', ''),
(1653, 'MIR16-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'miRNA', 45478266, 45478326, 55.74, 1, 'ENSG00000208006', ''),
(1654, 'MIR17HG', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 84516455, 84819589, 38.37, 5, 'ENSG00000215417', 'Q75NE6'),
(1655, 'MIR181A2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 101127104, 101139247, 50.41, 1, 'ENSG00000207595', ''),
(1656, 'MIR181D', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 36320189, 36386148, 42.30, 12, 'ENSG00000288329', ''),
(1657, 'MIR181D', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 36320189, 36386148, 42.30, 12, 'ENSG00000207585', ''),
(1658, 'MIR188', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 85835780, 85835858, 31.65, 1, 'ENSG00000207768', ''),
(1659, 'MIR1908', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 102637025, 102642791, 57.17, 1, 'ENSG00000284416', ''),
(1660, 'MIR194-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'IG_V_gene', 817052, 817524, 57.72, 1, 'ENSG00000284155', ''),
(1661, 'MIR19A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 52751132, 52790305, 44.96, 4, 'ENSG00000284204', ''),
(1662, 'MIR202', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 163593396, 163736012, 36.30, 3, 'ENSG00000284219', ''),
(1663, 'MIR210', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 3218297, 3526004, 35.00, 12, 'ENSG00000199038', ''),
(1664, 'MIR212', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 1843, 18639, 47.04, 2, 'ENSG00000267195', ''),
(1665, 'MIR219B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 41447830, 41529273, 49.69, 32, 'ENSG00000283335', ''),
(1666, 'MIR22', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 45863989, 45873797, 53.20, 2, 'ENSG00000283824', ''),
(1667, 'MIR2278', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 1188133, 1195260, 36.74, 1, 'ENSG00000252153', ''),
(1668, 'MIR2392', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 77979542, 78138444, 38.79, 2, 'ENSG00000266461', ''),
(1669, 'MIR23A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 49699320, 49792682, 38.63, 2, 'ENSG00000288462', ''),
(1670, 'MIR23A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 49699320, 49792682, 38.63, 2, 'ENSG00000207980', ''),
(1671, 'MIR23AHG', 'Predicted to be involved in miRNA-mediated post-transcriptional gene silencing. Predicted to be part of RISC complex. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 116371, 146260, 42.58, 20, 'ENSG00000288498', ''),
(1672, 'MIR23AHG', 'Predicted to be involved in miRNA-mediated post-transcriptional gene silencing. Predicted to be part of RISC complex. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 116371, 146260, 42.58, 20, 'ENSG00000267519', ''),
(1673, 'MIR25', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 139266165, 139280225, 40.47, 1, 'ENSG00000207547', ''),
(1674, 'MIR2861', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of target mRNAs. This microRNA may play a role in osteoblast differentiation and a mutation in this gene is associated with osteoporosis. Altered expression of this microRNA has been observed in human cancers, with reduced expression seen in cervical cancer, while expression in papillary thyroid carcinoma (PTC) is increased. [provided by RefSeq, Mar 2017]', '9', 'protein_coding', 115613210, 115632992, 37.81, 3, 'ENSG00000284547', ''),
(1675, 'MIR2909', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 2937933, 2963406, 50.02, 5, 'ENSG00000276326', ''),
(1676, 'MIR297', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'miRNA', 6745164, 6745244, 38.27, 1, 'ENSG00000215961', ''),
(1677, 'MIR299', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 128083766, 128189677, 35.24, 8, 'ENSG00000207749', ''),
(1678, 'MIR3130-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 47837692, 47841289, 59.14, 13, 'ENSG00000283469', ''),
(1679, 'MIR3133', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 117753, 132611, 48.96, 1, 'ENSG00000263752', ''),
(1680, 'MIR3144', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 46375627, 46391784, 54.03, 6, 'ENSG00000265725', ''),
(1681, 'MIR3157', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 2460086, 2464963, 62.03, 13, 'ENSG00000266407', ''),
(1682, 'MIR3162', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 24305734, 24312053, 55.70, 5, 'ENSG00000264559', ''),
(1683, 'MIR3177', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 48455574, 48466980, 52.28, 2, 'ENSG00000265820', ''),
(1684, 'MIR3180-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 30423615, 30430030, 53.20, 6, 'ENSG00000265373', ''),
(1685, 'MIR3184', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 6941742, 7117797, 43.13, 8, 'ENSG00000284399', ''),
(1686, 'MIR3199-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 34926903, 34945170, 42.70, 6, 'ENSG00000283225', ''),
(1687, 'MIR3200', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 45954465, 46052759, 34.90, 6, 'ENSG00000264661', ''),
(1688, 'MIR3201', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 96985713, 96994730, 42.73, 4, 'ENSG00000266508', ''),
(1689, 'MIR3202-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 105173297, 105236203, 43.64, 16, 'ENSG00000265176', ''),
(1690, 'MIR320C1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 16546954, 16569204, 39.49, 6, 'ENSG00000221493', ''),
(1691, 'MIR320E', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 183697797, 183812624, 42.59, 6, 'ENSG00000211513', ''),
(1692, 'MIR323A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 101002229, 101024032, 42.39, 11, 'ENSG00000199069', ''),
(1693, 'MIR329-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 125164687, 125433389, 49.03, 14, 'ENSG00000207762', ''),
(1694, 'MIR340', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 55379244, 55384292, 57.10, 1, 'ENSG00000198995', ''),
(1695, 'MIR3609', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 70528940, 70908711, 38.73, 4, 'ENSG00000266019', ''),
(1696, 'MIR361', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 147281943, 147282030, 40.91, 1, 'ENSG00000199051', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1697, 'MIR3617', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 107591091, 107666779, 38.98, 7, 'ENSG00000266071', ''),
(1698, 'MIR3646', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'miRNA', 11654885, 11654941, 49.12, 1, 'ENSG00000266151', ''),
(1699, 'MIR3654', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 89802, 90700, 63.29, 1, 'ENSG00000284135', ''),
(1700, 'MIR3660', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 116909916, 116990353, 44.25, 3, 'ENSG00000264342', ''),
(1701, 'MIR3672', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 205856, 220666, 40.66, 1, 'ENSG00000265456', ''),
(1702, 'MIR3679', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'unprocessed_pseudogene', 56381635, 56382589, 36.96, 1, 'ENSG00000263813', ''),
(1703, 'MIR3685', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 41397808, 41425002, 52.33, 8, 'ENSG00000265917', ''),
(1704, 'MIR3691', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 25686, 28123, 58.74, 5, 'ENSG00000265083', ''),
(1705, 'MIR376B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 21024109, 21036276, 51.69, 12, 'ENSG00000283556', ''),
(1706, 'MIR376C', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 71771655, 71785206, 39.37, 2, 'ENSG00000283279', ''),
(1707, 'MIR378A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 327525, 328424, 39.67, 1, 'ENSG00000199047', ''),
(1708, 'MIR378B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'protein_coding', 183318100, 183320276, 41.85, 1, 'ENSG00000264534', ''),
(1709, 'MIR378I', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'miRNA', 127785918, 127786007, 76.67, 1, 'ENSG00000283829', ''),
(1710, 'MIR384', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 136550878, 136550960, 43.37, 1, 'ENSG00000283242', ''),
(1711, 'MIR3915', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 73353063, 73354276, 54.70, 1, 'ENSG00000263600', ''),
(1712, 'MIR3918', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 73699206, 73719672, 58.84, 14, 'ENSG00000265558', ''),
(1713, 'MIR3926-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 20028179, 20091911, 44.56, 8, 'ENSG00000284855', ''),
(1714, 'MIR3926-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 20028179, 20091911, 44.56, 8, 'ENSG00000283523', ''),
(1715, 'MIR3928', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 2387176, 2440912, 44.74, 4, 'ENSG00000264141', ''),
(1716, 'MIR3935', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 29788609, 29838304, 48.94, 23, 'ENSG00000265281', ''),
(1717, 'MIR3937', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 70022857, 70022923, 44.78, 1, 'ENSG00000263730', ''),
(1718, 'MIR3941', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 35749287, 35758585, 52.56, 2, 'ENSG00000265442', ''),
(1719, 'MIR3943', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 11241213, 11564063, 43.69, 4, 'ENSG00000264069', ''),
(1720, 'MIR3973', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 176680, 177522, 67.26, 4, 'ENSG00000263389', ''),
(1721, 'MIR3974', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 183186238, 183245127, 43.26, 4, 'ENSG00000283665', ''),
(1722, 'MIR412', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 50578959, 50601455, 50.64, 12, 'ENSG00000199012', ''),
(1723, 'MIR423', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 45844881, 45847067, 61.18, 1, 'ENSG00000283935', ''),
(1724, 'MIR424', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 4315611, 4454253, 41.82, 1, 'ENSG00000284231', ''),
(1725, 'MIR4262', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 6074845, 6123214, 42.60, 8, 'ENSG00000265172', ''),
(1726, 'MIR4266', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 56375624, 56376553, 43.87, 1, 'ENSG00000265965', ''),
(1727, 'MIR4275', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'miRNA', 181043403, 181043484, 62.20, 1, 'ENSG00000283275', ''),
(1728, 'MIR4276', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 31259967, 31332892, 43.36, 8, 'ENSG00000265846', ''),
(1729, 'MIR4281', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 64084232, 64338112, 41.74, 13, 'ENSG00000266329', ''),
(1730, 'MIR4289', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 8790795, 8801383, 43.22, 2, 'ENSG00000291518', ''),
(1731, 'MIR4289', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 8790795, 8801383, 43.22, 2, 'ENSG00000265873', ''),
(1732, 'MIR4303', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 123033667, 123035665, 45.47, 1, 'ENSG00000263890', ''),
(1733, 'MIR4305', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 150307, 986812, 40.93, 3, 'ENSG00000264171', ''),
(1734, 'MIR4308', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 101023409, 101348278, 35.79, 9, 'ENSG00000265432', ''),
(1735, 'MIR431', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'miRNA', 130880, 130948, 63.77, 1, 'ENSG00000208001', ''),
(1736, 'MIR4318', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 41311267, 41377030, 37.31, 11, 'ENSG00000266530', ''),
(1737, 'MIR4319', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 30829039, 30829759, 45.77, 1, 'ENSG00000265957', ''),
(1738, 'MIR432', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 80106647, 80346270, 36.82, 6, 'ENSG00000272458', ''),
(1739, 'MIR4320', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 74692, 81260, 49.61, 4, 'ENSG00000283343', ''),
(1740, 'MIR4322', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 34363235, 34511773, 40.54, 19, 'ENSG00000264266', ''),
(1741, 'MIR4324', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'scaRNA', 77089307, 77089493, 48.66, 1, 'ENSG00000265407', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1742, 'MIR4325', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 285979, 305684, 44.29, 3, 'ENSG00000266666', ''),
(1743, 'MIR4326', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 10158301, 10172138, 38.78, 11, 'ENSG00000266104', ''),
(1744, 'MIR4327', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 97221434, 97394120, 40.24, 11, 'ENSG00000265007', ''),
(1745, 'MIR4329', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 101039690, 101039755, 28.79, 1, 'ENSG00000283775', ''),
(1746, 'MIR4432', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'unprocessed_pseudogene', 56335142, 56336068, 36.68, 1, 'ENSG00000266078', ''),
(1747, 'MIR4433A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 110045846, 110199199, 39.87, 8, 'ENSG00000283222', ''),
(1748, 'MIR4437', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 56462469, 56463401, 45.55, 1, 'ENSG00000266705', ''),
(1749, 'MIR4438', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 137423350, 137434406, 62.38, 5, 'ENSG00000265734', ''),
(1750, 'MIR4442', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'unprocessed_pseudogene', 56393729, 56394648, 39.67, 1, 'ENSG00000283971', ''),
(1751, 'MIR4452', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'miRNA', 87300495, 87300583, 56.18, 1, 'ENSG00000266515', ''),
(1752, 'MIR4464', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 63802, 75965, 64.28, 1, 'ENSG00000266760', ''),
(1753, 'MIR4468', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'miRNA', 54607874, 54607957, 30.95, 1, 'ENSG00000284028', ''),
(1754, 'MIR4474', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 35953617, 35959472, 47.71, 3, 'ENSG00000264941', ''),
(1755, 'MIR4477A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 315665, 321332, 30.77, 3, 'ENSG00000276029', ''),
(1756, 'MIR4479', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 136658856, 136672678, 67.31, 7, 'ENSG00000266507', ''),
(1757, 'MIR4482', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 37823382, 37825485, 66.11, 1, 'ENSG00000266852', ''),
(1758, 'MIR4493', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 24336025, 24340022, 55.85, 7, 'ENSG00000265357', ''),
(1759, 'MIR4495', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'lncRNA', 2870033, 2872105, 60.97, 1, 'ENSG00000265861', ''),
(1760, 'MIR4499', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 5289019, 5297053, 49.00, 9, 'ENSG00000263978', ''),
(1761, 'MIR449C', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 179944876, 180007297, 35.05, 2, 'ENSG00000251856', ''),
(1762, 'MIR450B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 70173322, 70187361, 57.16, 11, 'ENSG00000216001', ''),
(1763, 'MIR4518', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 107713221, 107777342, 48.75, 18, 'ENSG00000266305', ''),
(1764, 'MIR4521', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'miRNA', 101240795, 101240872, 39.74, 1, 'ENSG00000283160', ''),
(1765, 'MIR4527', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'TR_J_gene', 22492851, 22492907, 45.61, 1, 'ENSG00000266582', ''),
(1766, 'MIR4530', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 10221433, 10231331, 56.88, 1, 'ENSG00000266559', ''),
(1767, 'MIR454', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 233059967, 233207903, 48.79, 14, 'ENSG00000211514', ''),
(1768, 'MIR455', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 73120569, 73131809, 47.02, 2, 'ENSG00000207726', ''),
(1769, 'MIR4644', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 746, 78058, 49.00, 12, 'ENSG00000266245', ''),
(1770, 'MIR4659A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 90131515, 90192894, 34.98, 10, 'ENSG00000284880', ''),
(1771, 'MIR4659A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 90131515, 90192894, 34.98, 10, 'ENSG00000266038', ''),
(1772, 'MIR4679-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 152727484, 152733735, 40.67, 4, 'ENSG00000283664', ''),
(1773, 'MIR4694', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 70098223, 70132934, 50.25, 4, 'ENSG00000264309', ''),
(1774, 'MIR4699', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 192796815, 192917856, 42.65, 1, 'ENSG00000265227', ''),
(1775, 'MIR4719', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 52787916, 52804044, 38.89, 2, 'ENSG00000266426', ''),
(1776, 'MIR4741', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 48933699, 48944969, 52.54, 4, 'ENSG00000284331', ''),
(1777, 'MIR4745', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'miRNA', 94561789, 94561859, 33.80, 1, 'ENSG00000284361', ''),
(1778, 'MIR4749', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'TR_V_gene', 142511626, 142512127, 49.40, 1, 'ENSG00000265954', ''),
(1779, 'MIR4751', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'miRNA', 158764661, 158764753, 62.37, 1, 'ENSG00000283842', ''),
(1780, 'MIR4758', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 5140330, 5320304, 42.24, 4, 'ENSG00000284074', ''),
(1781, 'MIR4759', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 37103259, 37105664, 44.64, 1, 'ENSG00000266133', ''),
(1782, 'MIR4767', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 240157, 265888, 48.35, 6, 'ENSG00000264268', ''),
(1783, 'MIR4769', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 57060664, 57075432, 45.20, 1, 'ENSG00000263858', ''),
(1784, 'MIR4770', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 50003503, 50003588, 58.14, 1, 'ENSG00000265284', ''),
(1785, 'MIR4774', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 554850, 560738, 66.96, 5, 'ENSG00000265694', ''),
(1786, 'MIR4777', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 87882877, 87888903, 59.30, 1, 'ENSG00000263641', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1787, 'MIR4789', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'IG_C_gene', 22900976, 22901437, 57.36, 1, 'ENSG00000264974', ''),
(1788, 'MIR4789', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'IG_C_gene', 22900976, 22901437, 57.36, 1, 'ENSG00000285406', ''),
(1789, 'MIR4796', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '3', 'protein_coding', 2463728, 2547694, 55.33, 11, 'ENSG00000264623', ''),
(1790, 'MIR4799', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 444319, 497221, 45.61, 19, 'ENSG00000264274', ''),
(1791, 'MIR4803', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 120150898, 120176520, 36.14, 2, 'ENSG00000264099', ''),
(1792, 'MIR484', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 122172029, 122174221, 49.20, 1, 'ENSG00000283736', ''),
(1793, 'MIR487A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'miRNA', 49308797, 49308868, 51.39, 1, 'ENSG00000207742', ''),
(1794, 'MIR487B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 21032818, 21034807, 51.71, 1, 'ENSG00000207754', ''),
(1795, 'MIR490', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 151771045, 151812911, 39.46, 78, 'ENSG00000207597', ''),
(1796, 'MIR493', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 4984390, 5129948, 37.57, 4, 'ENSG00000207989', ''),
(1797, 'MIR5003', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 152759749, 152855378, 43.36, 5, 'ENSG00000265160', ''),
(1798, 'MIR501', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 76919273, 76919360, 27.27, 1, 'ENSG00000211538', ''),
(1799, 'MIR502', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 124619292, 124744378, 52.70, 3, 'ENSG00000272080', ''),
(1800, 'MIR5047', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 663522, 674470, 60.05, 1, 'ENSG00000284368', ''),
(1801, 'MIR507', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 129467190, 129770983, 45.58, 5, 'ENSG00000207969', ''),
(1802, 'MIR509-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 133394094, 133422520, 48.28, 8, 'ENSG00000212013', ''),
(1803, 'MIR510', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 134960378, 135103056, 36.66, 16, 'ENSG00000207641', ''),
(1804, 'MIR513A1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 3014712, 3020071, 56.92, 3, 'ENSG00000207873', ''),
(1805, 'MIR514A2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 147230984, 147231077, 40.43, 1, 'ENSG00000207866', ''),
(1806, 'MIR514A3', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 147258760, 147258850, 43.96, 1, 'ENSG00000207867', ''),
(1807, 'MIR515-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 35673928, 35681159, 52.49, 3, 'ENSG00000207616', ''),
(1808, 'MIR515-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 178237476, 178590393, 50.61, 9, 'ENSG00000207615', ''),
(1809, 'MIR518E', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 45263119, 45317029, 49.97, 6, 'ENSG00000207987', ''),
(1810, 'MIR5197', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 260071, 261000, 38.71, 1, 'ENSG00000266478', ''),
(1811, 'MIR520A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 128161251, 128204383, 52.50, 28, 'ENSG00000207594', ''),
(1812, 'MIR520B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 832361, 849155, 47.06, 2, 'ENSG00000207722', ''),
(1813, 'MIR520C', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 498460, 507265, 50.56, 6, 'ENSG00000207738', ''),
(1814, 'MIR520G', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'miRNA', 31621467, 31621531, 64.62, 1, 'ENSG00000207799', ''),
(1815, 'MIR525', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000207711', ''),
(1816, 'MIR542', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 4932598, 4949251, 43.48, 2, 'ENSG00000207784', ''),
(1817, 'MIR543', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 110099819, 110180004, 36.65, 10, 'ENSG00000212040', ''),
(1818, 'MIR548AD', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'unprocessed_pseudogene', 56296541, 56297471, 41.57, 1, 'ENSG00000265301', ''),
(1819, 'MIR548AI', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 76070, 86899, 55.12, 2, 'ENSG00000265226', ''),
(1820, 'MIR548AM', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 50009722, 50009805, 53.57, 1, 'ENSG00000265144', ''),
(1821, 'MIR548AN', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 147272335, 147272408, 44.59, 1, 'ENSG00000263515', ''),
(1822, 'MIR548AS', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 100379, 173784, 51.36, 4, 'ENSG00000263741', ''),
(1823, 'MIR548E', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 59139475, 59158832, 43.41, 9, 'ENSG00000221214', ''),
(1824, 'MIR548F1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'snoRNA', 100897920, 100897996, 37.66, 1, 'ENSG00000221594', ''),
(1825, 'MIR548G', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 82412, 111099, 62.11, 4, 'ENSG00000221369', ''),
(1826, 'MIR548I2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'misc_RNA', 165609441, 165609679, 46.86, 1, 'ENSG00000221275', ''),
(1827, 'MIR548U', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 129906121, 129929761, 45.09, 4, 'ENSG00000212017', ''),
(1828, 'MIR548W', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 1309218, 1310156, 44.09, 2, 'ENSG00000265005', ''),
(1829, 'MIR550B1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 30558809, 30577766, 39.28, 1, 'ENSG00000283237', ''),
(1830, 'MIR550B2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 174438573, 175810548, 35.22, 12, 'ENSG00000283461', ''),
(1831, 'MIR5571', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 130339888, 130373361, 37.86, 15, 'ENSG00000264824', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1832, 'MIR5579', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 841880, 846265, 37.60, 1, 'ENSG00000266570', ''),
(1833, 'MIR5583-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 62307955, 62367312, 61.89, 14, 'ENSG00000266148', ''),
(1834, 'MIR5586', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 68707440, 68751520, 45.07, 9, 'ENSG00000263629', ''),
(1835, 'MIR5685', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 30064164, 30070457, 59.26, 19, 'ENSG00000264056', ''),
(1836, 'MIR5692B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 47152834, 47176364, 38.67, 2, 'ENSG00000264580', ''),
(1837, 'MIR5693', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 4852356, 5010293, 44.93, 5, 'ENSG00000266072', ''),
(1838, 'MIR5696', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 47823272, 47831541, 49.19, 7, 'ENSG00000264857', ''),
(1839, 'MIR5699', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 1165482, 1195807, 42.84, 11, 'ENSG00000263511', ''),
(1840, 'MIR5700', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 124853417, 124857890, 57.87, 3, 'ENSG00000266099', ''),
(1841, 'MIR5705', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 16232506, 16300806, 45.83, 30, 'ENSG00000263981', ''),
(1842, 'MIR5739', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 84275457, 84340758, 42.61, 2, 'ENSG00000276162', ''),
(1843, 'MIR580', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'unprocessed_pseudogene', 56448668, 56449592, 42.16, 1, 'ENSG00000207756', ''),
(1844, 'MIR588', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 224378698, 224402107, 44.16, 3, 'ENSG00000207632', ''),
(1845, 'MIR6072', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 134917393, 134950115, 42.53, 6, 'ENSG00000278069', ''),
(1846, 'MIR6078', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 122775079, 122834543, 35.79, 13, 'ENSG00000274809', ''),
(1847, 'MIR6082', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 76527586, 76553578, 55.00, 13, 'ENSG00000278502', ''),
(1848, 'MIR6086', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'IG_V_gene', 22668288, 22668806, 57.42, 1, 'ENSG00000283493', ''),
(1849, 'MIR609', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 75053237, 75069483, 41.68, 6, 'ENSG00000208033', ''),
(1850, 'MIR6124', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'miRNA', 16368876, 16368942, 67.16, 1, 'ENSG00000275373', ''),
(1851, 'MIR6125', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 39406930, 39451533, 42.12, 1, 'ENSG00000284360', ''),
(1852, 'MIR6126', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 103704, 123845, 57.52, 8, 'ENSG00000273776', ''),
(1853, 'MIR6132', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 139341587, 139369720, 43.58, 8, 'ENSG00000283923', ''),
(1854, 'MIR616', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '12', 'protein_coding', 26172059, 26184655, 41.16, 2, 'ENSG00000208028', ''),
(1855, 'MIR623', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'protein_coding', 106969831, 106975627, 50.84, 8, 'ENSG00000207719', ''),
(1856, 'MIR625', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 50719565, 50756219, 39.65, 23, 'ENSG00000207781', ''),
(1857, 'MIR632', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 12377563, 12535356, 43.49, 6, 'ENSG00000283774', ''),
(1858, 'MIR634', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 72114595, 72257193, 40.49, 9, 'ENSG00000207943', ''),
(1859, 'MIR636', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 43225629, 43229481, 55.28, 4, 'ENSG00000283805', ''),
(1860, 'MIR645', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 26903372, 26947242, 37.93, 7, 'ENSG00000208018', ''),
(1861, 'MIR6508', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 7859159, 7869037, 37.73, 8, 'ENSG00000275523', ''),
(1862, 'MIR6511A3', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 76949751, 77040384, 40.64, 16, 'ENSG00000276311', ''),
(1863, 'MIR6514', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 54542325, 54571080, 39.92, 8, 'ENSG00000274066', ''),
(1864, 'MIR652', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'unprocessed_pseudogene', 6045807, 6046747, 50.90, 1, 'ENSG00000208013', ''),
(1865, 'MIR653', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 113876777, 114075920, 36.17, 3, 'ENSG00000208014', ''),
(1866, 'MIR656', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 78889764, 79282124, 35.25, 17, 'ENSG00000207959', ''),
(1867, 'MIR659', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 165040, 172260, 34.73, 3, 'ENSG00000207696', ''),
(1868, 'MIR664B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 37196488, 37216193, 38.06, 7, 'ENSG00000284450', ''),
(1869, 'MIR668', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'miRNA', 53556223, 53556341, 42.86, 1, 'ENSG00000276352', ''),
(1870, 'MIR6715A', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '10', 'protein_coding', 12658043, 12728945, 42.05, 23, 'ENSG00000276857', ''),
(1871, 'MIR6718', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 93591573, 93911265, 35.76, 5, 'ENSG00000275518', ''),
(1872, 'MIR6724-4', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'miRNA', 51348567, 51348639, 46.58, 1, 'ENSG00000275692', ''),
(1873, 'MIR6724-4', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'miRNA', 51348567, 51348639, 46.58, 1, 'ENSG00000288308', ''),
(1874, 'MIR6744', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 153892, 166764, 58.25, 3, 'ENSG00000275669', ''),
(1875, 'MIR6748', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 499948, 508771, 50.57, 6, 'ENSG00000284535', ''),
(1876, 'MIR676', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 85903636, 85903707, 41.67, 1, 'ENSG00000211991', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1877, 'MIR6767', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'transcribed_unprocessed_pseudogene', 21115929, 21119758, 59.03, 1, 'ENSG00000283180', ''),
(1878, 'MIR6773', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'miRNA', 62260359, 62260454, 70.83, 1, 'ENSG00000284259', ''),
(1879, 'MIR6776', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 89283694, 89309271, 42.26, 15, 'ENSG00000273606', ''),
(1880, 'MIR6782', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 34704339, 34713439, 55.18, 4, 'ENSG00000275107', ''),
(1881, 'MIR6796', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 101574180, 101594465, 39.69, 8, 'ENSG00000275652', ''),
(1882, 'MIR6805', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 62388634, 62407285, 59.63, 2, 'ENSG00000284029', ''),
(1883, 'MIR6812', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'snoRNA', 54222284, 54222412, 54.26, 1, 'ENSG00000273555', ''),
(1884, 'MIR6814', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 405445, 409147, 68.43, 1, 'ENSG00000275166', ''),
(1885, 'MIR6815', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 7559270, 7666998, 43.72, 9, 'ENSG00000275167', ''),
(1886, 'MIR6815', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 7559270, 7666998, 43.72, 9, 'ENSG00000291767', ''),
(1887, 'MIR6816', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 58549, 99008, 62.40, 8, 'ENSG00000278278', ''),
(1888, 'MIR6832', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 127809, 144581, 47.05, 2, 'ENSG00000274494', ''),
(1889, 'MIR6849', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 270920, 312726, 43.41, 7, 'ENSG00000285088', ''),
(1890, 'MIR6849', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '8', 'protein_coding', 270920, 312726, 43.41, 7, 'ENSG00000283770', ''),
(1891, 'MIR6852', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'miRNA', 122416965, 122417067, 40.78, 1, 'ENSG00000284195', ''),
(1892, 'MIR6862-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'miRNA', 101042977, 101043062, 38.37, 1, 'ENSG00000275429', ''),
(1893, 'MIR6862-2', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'unprocessed_pseudogene', 19796345, 19797293, 46.68, 1, 'ENSG00000278340', ''),
(1894, 'MIR6863', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 42079691, 42084284, 48.41, 3, 'ENSG00000283471', ''),
(1895, 'MIR6871', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 44523639, 44542859, 41.03, 2, 'ENSG00000276169', ''),
(1896, 'MIR6877', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 37696371, 37719761, 41.53, 5, 'ENSG00000273932', ''),
(1897, 'MIR6879', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 30914205, 30926459, 54.92, 13, 'ENSG00000278851', ''),
(1898, 'MIR6883', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '17', 'snoRNA', 17320746, 17320882, 39.42, 1, 'ENSG00000284117', ''),
(1899, 'MIR6889', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 18377, 26129, 44.45, 2, 'ENSG00000274552', ''),
(1900, 'MIR6891', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 103576231, 103587729, 39.32, 12, 'ENSG00000277402', ''),
(1901, 'MIR6895', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 48029383, 48044079, 46.77, 2, 'ENSG00000276575', ''),
(1902, 'MIR7-1', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 86594315, 86815171, 36.16, 9, 'ENSG00000284179', ''),
(1903, 'MIR7109', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 6445953, 6484190, 46.74, 6, 'ENSG00000284082', ''),
(1904, 'MIR7153', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'snoRNA', 100982926, 100982997, 45.83, 1, 'ENSG00000283492', ''),
(1905, 'MIR7154', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 70431936, 70447951, 51.27, 2, 'ENSG00000276639', ''),
(1906, 'MIR7158', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '2', 'protein_coding', 126387, 143165, 47.08, 2, 'ENSG00000276733', ''),
(1907, 'MIR7161', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'protein_coding', 129779949, 129795201, 52.71, 2, 'ENSG00000278571', ''),
(1908, 'MIR7703', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 100089015, 100530437, 39.84, 16, 'ENSG00000285146', ''),
(1909, 'MIR7703', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 100089015, 100530437, 39.84, 16, 'ENSG00000283856', ''),
(1910, 'MIR7843', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 17719571, 17722136, 63.80, 1, 'ENSG00000273830', ''),
(1911, 'MIR7855', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '14', 'protein_coding', 57591174, 57603418, 46.79, 8, 'ENSG00000284269', ''),
(1912, 'MIR802', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '21', 'protein_coding', 47501887, 47656877, 41.15, 5, 'ENSG00000211590', ''),
(1913, 'MIR8052', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 68020916, 68023232, 55.37, 1, 'ENSG00000274999', ''),
(1914, 'MIR8054', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '11', 'protein_coding', 70202835, 70233911, 39.80, 6, 'ENSG00000275868', ''),
(1915, 'MIR8062', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '20', 'protein_coding', 67389809, 67393518, 61.24, 5, 'ENSG00000278159', ''),
(1916, 'MIR8065', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 58281, 69580, 48.77, 6, 'ENSG00000273648', ''),
(1917, 'MIR8066', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 112226367, 112269955, 41.90, 9, 'ENSG00000273882', ''),
(1918, 'MIR8075', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '13', 'unprocessed_pseudogene', 8414780, 8415805, 49.03, 1, 'ENSG00000277942', ''),
(1919, 'MIR8078', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '18', 'protein_coding', 25225, 72124, 42.52, 4, 'ENSG00000277521', ''),
(1920, 'MIR8081', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '9', 'protein_coding', 24952716, 25197448, 48.94, 6, 'ENSG00000277889', ''),
(1921, 'MIR8089', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '5', 'protein_coding', 1811479, 1856156, 53.78, 7, 'ENSG00000277912', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1922, 'MIR877', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '6', 'miRNA', 30503919, 30504001, 55.42, 1, 'ENSG00000216101', ''),
(1923, 'MIR934', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'miRNA', 147213463, 147213591, 44.19, 1, 'ENSG00000216060', ''),
(1924, 'MIR943', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '4', 'protein_coding', 111549722, 111599893, 43.26, 9, 'ENSG00000284587', ''),
(1925, 'MIR98', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 24420931, 24512778, 42.87, 3, 'ENSG00000271886', ''),
(1926, 'MIRLET7B', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '22', 'protein_coding', 68232003, 68300035, 36.18, 19, 'ENSG00000284520', ''),
(1927, 'MIRLET7E', 'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]', '19', 'protein_coding', 135693407, 135704498, 63.42, 4, 'ENSG00000198972', ''),
(1928, 'MIS18A', 'Enables identical protein binding activity. Predicted to be involved in CENP-A containing chromatin assembly and chromosome segregation. Predicted to act upstream of or within pericentric heterochromatin organization and protein localization to chromosome, centromeric region. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 39499432, 39518132, 49.85, 14, 'ENSG00000159055', 'Q9NYP9'),
(1929, 'MISP', 'The protein encoded by this gene is an actin-bundling protein involved in determining cell morphology and mitotic progression. The encoded protein is required for the proper positioning of the mitotic spindle. Two transcript variants, one protein-coding and the other non-protein coding, have been found for this gene. [provided by RefSeq, Feb 2016]', '19', 'protein_coding', 68686951, 68745809, 39.25, 13, 'ENSG00000099812', 'Q8IVT2'),
(1930, 'MKKS', 'This gene encodes a protein which shares sequence similarity with other members of the type II chaperonin family. The encoded protein is a centrosome-shuttling protein and plays an important role in cytokinesis. This protein also interacts with other type II chaperonin members to form a complex known as the BBSome, which involves ciliary membrane biogenesis. This protein is encoded by a downstream open reading frame (dORF). Several upstream open reading frames (uORFs) have been identified, which repress the translation of the dORF, and two of which can encode small mitochondrial membrane proteins. Mutations in this gene have been observed in patients with Bardet-Biedl syndrome type 6, also known as McKusick-Kaufman syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2023]', '20', 'protein_coding', 108207439, 108218368, 40.58, 19, 'ENSG00000125863', 'Q9NPJ1'),
(1931, 'MLC1', 'The function of this gene product is unknown; however, homology to other proteins suggests that it may be an integral membrane transporter. Mutations in this gene have been associated with megalencephalic leukoencephalopathy with subcortical cysts, an autosomal recessive neurological disorder. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 116548105, 116576606, 44.51, 4, 'ENSG00000100427', 'Q15049'),
(1932, 'MLKL', 'This gene belongs to the protein kinase superfamily. The encoded protein contains a protein kinase-like domain; however, is thought to be inactive because it lacks several residues required for activity. This protein plays a critical role in tumor necrosis factor (TNF)-induced necroptosis, a programmed cell death process, via interaction with receptor-interacting protein 3 (RIP3), which is a key signaling molecule in necroptosis pathway. Inhibitor studies and knockdown of this gene inhibited TNF-induced necrosis. High levels of this protein and RIP3 are associated with inflammatory bowel disease in children. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Sep 2015]', '16', 'protein_coding', 94907202, 95296415, 36.45, 8, 'ENSG00000168404', 'Q8NB16'),
(1933, 'MMP11', 'Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP\'s are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. However, the enzyme encoded by this gene is activated intracellularly by furin within the constitutive secretory pathway. Also in contrast to other MMP\'s, this enzyme cleaves alpha 1-proteinase inhibitor but weakly degrades structural proteins of the extracellular matrix. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 53001279, 53061824, 40.86, 3, 'ENSG00000099953', 'P24347'),
(1934, 'MMP11', 'Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP\'s are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. However, the enzyme encoded by this gene is activated intracellularly by furin within the constitutive secretory pathway. Also in contrast to other MMP\'s, this enzyme cleaves alpha 1-proteinase inhibitor but weakly degrades structural proteins of the extracellular matrix. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 53001279, 53061824, 40.86, 3, 'ENSG00000275365', 'P24347'),
(1935, 'MMP14', 'Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP\'s are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. However, the protein encoded by this gene is a member of the membrane-type MMP (MT-MMP) subfamily; each member of this subfamily contains a potential transmembrane domain suggesting that these proteins are expressed at the cell surface rather than secreted. This protein activates MMP2 protein, and this activity may be involved in tumor invasion. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 171378, 217852, 41.46, 9, 'ENSG00000157227', 'P50281'),
(1936, 'MMRN2', 'This gene encodes a protein belonging to the member of elastin microfibril interface-located (EMILIN) protein family. This family member is an extracellular matrix glycoprotein that can interfere with tumor angiogenesis and growth. It serves as a transforming growth factor beta antagonist and can interfere with the VEGF-A/VEGFR2 pathway. A related pseudogene has been identified on chromosome 6. [provided by RefSeq, Aug 2012]', '10', 'protein_coding', 31111223, 31112575, 57.80, 1, 'ENSG00000173269', 'Q9H8L6'),
(1937, 'MNT', 'The Myc/Max/Mad network comprises a group of transcription factors that co-interact to regulate gene-specific transcriptional activation or repression. This gene encodes a protein member of the Myc/Max/Mad network. This protein has a basic-Helix-Loop-Helix-zipper domain (bHLHzip) with which it binds the canonical DNA sequence CANNTG, known as the E box, following heterodimerization with Max proteins. This protein is likely a transcriptional repressor and an antagonist of Myc-dependent transcriptional activation and cell growth. This protein represses transcription by binding to DNA binding proteins at its N-terminal Sin3-interaction domain. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 8940361, 8949761, 42.37, 13, 'ENSG00000070444', 'Q99583'),
(1938, 'MOB1B', 'The protein encoded by this gene is similar to the yeast Mob1 protein. Yeast Mob1 binds Mps1p, a protein kinase essential for spindle pole body duplication and mitotic checkpoint regulation. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]', '4', 'IG_V_gene', 621391, 621932, 50.55, 1, 'ENSG00000173542', 'Q7L9L4'),
(1939, 'MOB2', 'Predicted to enable protein kinase activator activity. Predicted to be involved in signal transduction. Predicted to act upstream of or within actin cytoskeleton organization; positive regulation of neuron projection development; and positive regulation of protein phosphorylation. Located in cytosol; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 350528, 355749, 46.73, 2, 'ENSG00000291442', 'Q70IA6'),
(1940, 'MOB2', 'Predicted to enable protein kinase activator activity. Predicted to be involved in signal transduction. Predicted to act upstream of or within actin cytoskeleton organization; positive regulation of neuron projection development; and positive regulation of protein phosphorylation. Located in cytosol; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 350528, 355749, 46.73, 2, 'ENSG00000182208', 'Q70IA6'),
(1941, 'MOCS3', 'Molybdenum cofactor (MoCo) is necessary for the function of all molybdoenzymes. The protein encoded by this gene adenylates and activates molybdopterin synthase, an enzyme required for biosynthesis of MoCo. This gene contains no introns. A pseudogene of this gene is present on chromosome 14. [provided by RefSeq, Nov 2012]', '20', 'protein_coding', 36385392, 36419015, 46.71, 3, 'ENSG00000124217', 'O95396'),
(1942, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000204655', 'Q16653'),
(1943, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000234096', 'Q16653'),
(1944, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000232641', 'Q16653'),
(1945, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000234623', 'Q16653'),
(1946, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000237834', 'Q16653'),
(1947, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000137345', 'Q16653'),
(1948, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000230885', 'Q16653'),
(1949, 'MOG', 'The product of this gene is a membrane protein expressed on the oligodendrocyte cell surface and the outermost surface of myelin sheaths. Due to this localization, it is a primary target antigen involved in immune-mediated demyelination. This protein may be involved in completion and maintenance of the myelin sheath and in cell-cell communication. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 84495, 90434, 67.09, 5, 'ENSG00000236561', 'Q16653'),
(1950, 'MORC1', 'This gene encodes the human homolog of mouse morc and like the mouse protein it is testis-specific. Mouse studies support a testis-specific function since only male knockout mice are infertile; infertility is the only apparent defect. These studies further support a role for this protein early in spermatogenesis, possibly by affecting entry into apoptosis because testis from knockout mice show greatly increased numbers of apoptotic cells. [provided by RefSeq, Jan 2009]', '3', 'protein_coding', 36383120, 36418644, 40.73, 3, 'ENSG00000114487', 'Q86VD1'),
(1951, 'MORC3', 'This gene encodes a protein that localizes to the nuclear matrix and forms nuclear bodies via an ATP-dependent mechanism. The protein is predicted to have coiled-coil and zinc finger domains and has RNA binding activity. Alternative splicing produces multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Feb 2016]', '21', 'protein_coding', 78598362, 78603552, 51.88, 1, 'ENSG00000159256', 'Q14149'),
(1952, 'MORC4', 'In human, the four current members of the microrchidia (morc) gene family share an N-terminal ATPase-like ATP-binding region and a CW four-cysteine zinc-finger motif. The protein encoded by this gene also has a nuclear matrix binding domain and a two-stranded coiled-coil motif near its C-terminus. This gene is widely expressed at low levels in normal tissues and has elevated expression in placenta and testis. Alternative splicing results in multiple transcript variants encoding distinct proteins. [provided by RefSeq, Jan 2010]', 'X', 'TR_J_gene', 22544071, 22544136, 36.36, 1, 'ENSG00000133131', 'Q8TE76'),
(1953, 'MOSPD2', 'Enables FFAT motif binding activity. Involved in lipid droplet formation; positive regulation of leukocyte chemotaxis; and protein homooligomerization. Located in endoplasmic reticulum membrane; endoplasmic reticulum-endosome membrane contact site; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000130150', 'Q8NHP6'),
(1954, 'MPC1L', 'Predicted to enable pyruvate transmembrane transporter activity. Involved in mitochondrial pyruvate transmembrane transport. Located in mitochondrial inner membrane. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 30496621, 30504841, 40.54, 8, 'ENSG00000238205', 'P0DKB6'),
(1955, 'MPC2', 'Enables identical protein binding activity. Involved in mitochondrial pyruvate transmembrane transport. Located in mitochondrial inner membrane. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 2084, 18878, 47.10, 2, 'ENSG00000143158', 'O95563'),
(1956, 'MPHOSPH8', 'Enables chromatin binding activity; histone reader activity; and methylated histone binding activity. Involved in negative regulation of gene expression via chromosomal CpG island methylation and transposable element silencing by heterochromatin formation. Located in several cellular components, including cytosol; heterochromatin; and nuclear lumen. Part of nucleosome. [provided by Alliance of Genome Resources, May 2025]', '13', 'TR_J_gene', 22488593, 22488652, 45.00, 1, 'ENSG00000196199', 'Q99549'),
(1957, 'MPLKIP', 'The protein encoded by this gene localizes to the centrosome during mitosis and to the midbody during cytokinesis. The protein is phosphorylated by cyclin-dependent kinase 1 during mitosis and subsequently interacts with polo-like kinase 1. The protein is thought to function in regulating mitosis and cytokinesis. Mutations in this gene result in nonphotosensitive trichothiodystrophy. [provided by RefSeq, Nov 2009]', '7', 'protein_coding', 40931919, 41061440, 38.79, 5, 'ENSG00000168303', 'Q8TAP9'),
(1958, 'MPND', 'Predicted to enable metallopeptidase activity and polyubiquitin modification-dependent protein binding activity. Predicted to be involved in double-strand break repair. Predicted to be part of BRCA1-A complex and BRISC complex. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 238238267, 238240662, 70.08, 9, 'ENSG00000008382', 'Q8N594'),
(1959, 'MPP7', 'Chromatin assembly factor I (CAF-I) is required for the assembly of histone octamers onto newly-replicated DNA. CAF-I is composed of three protein subunits, p50, p60, and p150. The protein encoded by this gene corresponds to the p60 subunit and is required for chromatin assembly after replication. The encoded protein is differentially phosphorylated in a cell cycle-dependent manner. In addition, it is normally found in the nucleus except during mitosis, when it is released into the cytoplasm. This protein is a member of the WD-repeat HIR1 family and may also be involved in DNA repair. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 119871832, 119876662, 42.08, 1, 'ENSG00000150054', 'Q13112'),
(1960, 'MPPE1', 'Predicted to enable GPI anchor binding activity; GPI-mannose ethanolamine phosphate phosphodiesterase activity; and manganese ion binding activity. Involved in GPI anchor biosynthetic process. Located in Golgi apparatus and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 37006150, 37019662, 48.79, 3, 'ENSG00000154889', 'Q53F39'),
(1961, 'MPPED1', 'Predicted to enable hydrolase activity. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 533400, 540220, 46.62, 31, 'ENSG00000186732', 'O15442'),
(1962, 'MRGPRE', 'Predicted to enable G protein-coupled receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 52702813, 52735073, 46.11, 21, 'ENSG00000184350', 'Q86SM8'),
(1963, 'MRLN', 'This gene encodes a small peptide that shares structural similarity to the small peptides sarcolipin and phospholamban, which are key regulators of sarcoplasmic reticulum Ca(2+)-ATPases (SERCAs). This protein is thought to have a similar function to these peptides, regulating Ca(2+) reuptake in the sarcoplasmic reticulum by inhibiting the Ca(2+) pump activity of SERCAs. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2015]', '10', 'protein_coding', 8775402, 8963773, 40.44, 10, 'ENSG00000227877', 'P0DMT0'),
(1964, 'MROH5', 'Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '8', 'miRNA', 26953961, 26954043, 31.33, 1, 'ENSG00000291473', 'Q6ZUA9'),
(1965, 'MROH5', 'Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '8', 'miRNA', 26953961, 26954043, 31.33, 1, 'ENSG00000282181', 'Q6ZUA9'),
(1966, 'MROH5', 'Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '8', 'miRNA', 26953961, 26954043, 31.33, 1, 'ENSG00000226807', 'Q6ZUA9'),
(1967, 'MRPL1', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein that belongs to the L1 ribosomal protein family. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 45120510, 45284893, 39.12, 7, 'ENSG00000169288', 'Q9BYD6'),
(1968, 'MRPL10', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein. Sequence analysis identified three transcript variants that encode two different isoforms. A pseudogene corresponding to this gene is found on chromosome 5q. [provided by RefSeq, Nov 2010]', '17', 'protein_coding', 47656197, 47786616, 50.45, 14, 'ENSG00000159111', 'Q7Z7H8'),
(1969, 'MRPL14', 'This nuclear gene encodes a protein component of the 39S subunit of the mitochondrial ribosome. A pseudogene of this gene is found on chromosome 17. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 190646746, 190692766, 37.70, 10, 'ENSG00000180992', 'Q6P1L8'),
(1970, 'MRPL18', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein which is more than twice the size of its E.coli counterpart (EcoL24). Sequence analysis identified two transcript variants that encode the same protein. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 2384073, 2401104, 61.64, 9, 'ENSG00000112110', 'Q9H0U6'),
(1971, 'MRPL22', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein. Deletions in this gene may contribute to the etiology of velo-cardio-facial syndrome and DiGeorge syndrome. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 59086307, 59094786, 37.69, 1, 'ENSG00000082515', 'Q9NWU5'),
(1972, 'MRPL32', 'This nuclear gene encodes a protein component of the 39S subunit of the mitochondrial ribosome. A pseudogene of this gene is found on chromosome 17. [provided by RefSeq, Jan 2016]', '6', 'snoRNA', 49490904, 49490974, 50.70, 1, 'ENSG00000106591', 'Q9BYC8'),
(1973, 'MRPL43', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein. This gene and the gene for a semaphorin class 4 protein (SEMA4G) overlap at map location 10q24.31 and are transcribed in opposite directions. Sequence analysis identified multiple transcript variants encoding at least four different protein isoforms. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 1842, 18618, 47.10, 2, 'ENSG00000055950', 'Q8N983'),
(1974, 'MRPL50', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a putative 39S subunit protein and belongs to the L47P ribosomal protein family. Pseudogenes corresponding to this gene are found on chromosomes 2p, 2q, 5p, and 10q. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 158232236, 158511828, 43.15, 7, 'ENSG00000136897', 'Q8N5N7'),
(1975, 'MRPL52', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein which has no bacterial homolog. Multiple transcript variants encoding different protein isoforms were identified through sequence analysis. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 31706904, 31717918, 52.05, 1, 'ENSG00000172590', 'Q86TS9'),
(1976, 'MRPL53', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 39S subunit protein. A pseudogene corresponding to this gene is found on chromosome 1p. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 75135248, 75168281, 46.27, 12, 'ENSG00000204822', 'Q96EL3'),
(1977, 'MRPL57', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a protein which belongs to an undetermined ribosomal subunit and which seems to be specific to animal mitoribosomes. Pseudogenes corresponding to this gene are found on chromosomes 1p, 1q, 3p, 5q, 8q, 14q, and Y. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 9035033, 9546075, 40.18, 5, 'ENSG00000173141', 'Q9BQC6'),
(1978, 'MRPS31', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. The 28S subunit of the mammalian mitoribosome may play a crucial and characteristic role in translation initiation. This gene encodes a 28S subunit protein that has also been associated with type 1 diabetes; however, its relationship to the etiology of this disease remains to be clarified. Pseudogenes corresponding to this gene have been found on chromosomes 3 and 13. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 240452, 285496, 55.14, 9, 'ENSG00000102738', 'Q92665'),
(1979, 'MRPS33', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. The 28S subunit of the mammalian mitoribosome may play a crucial and characteristic role in translation initiation. This gene encodes a 28S subunit protein that is one of the more highly conserved mitochondrial ribosomal proteins among mammals, Drosophila and C. elegans. Splice variants that differ in the 5\' UTR have been found for this gene; all variants encode the same protein. Pseudogenes corresponding to this gene are found on chromosomes 1q, 4p, 4q, and 20q [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 359628, 497696, 41.73, 2, 'ENSG00000090263', 'Q9Y291'),
(1980, 'MRPS34', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 28S subunit protein. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014]', '16', 'protein_coding', 75979231, 76005999, 50.97, 3, 'ENSG00000074071', 'P82930'),
(1981, 'MRPS35', 'Mammalian mitochondrial ribosomal proteins are encoded by nuclear genes and help in protein synthesis within the mitochondrion. Mitochondrial ribosomes (mitoribosomes) consist of a small 28S subunit and a large 39S subunit. They have an estimated 75% protein to rRNA composition compared to prokaryotic ribosomes, where this ratio is reversed. Another difference between mammalian mitoribosomes and prokaryotic ribosomes is that the latter contain a 5S rRNA. Among different species, the proteins comprising the mitoribosome differ greatly in sequence, and sometimes in biochemical properties, which prevents easy recognition by sequence homology. This gene encodes a 28S subunit protein that has had confusing nomenclature in the literature. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. Pseudogenes corresponding to this gene are found on chromosomes 3p, 5q, and 10q. [provided by RefSeq, Jul 2010]', '12', 'TR_V_gene', 21894433, 21895030, 51.34, 1, 'ENSG00000061794', 'P82673'),
(1982, 'MRRF', 'This gene encodes a ribosome recycling factor, which is a component of the mitochondrial translational machinery. The encoded protein, along with mitochondrial elongation factor 2, functions in ribosomal recycling at the termination of mitochondrial translation by mediating the disassembly of ribosomes from messenger RNA. A pseudogene of this gene has been identified on chromosome X. [provided by RefSeq, Oct 2016]', '9', 'protein_coding', 73052362, 73069755, 40.47, 5, 'ENSG00000148187', 'Q96E11'),
(1983, 'MS4A14', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'miRNA', 68233426, 68233499, 48.65, 1, 'ENSG00000166928', 'Q96JA4'),
(1984, 'MSANTD7', 'Predicted to enable ATP hydrolysis activity; heat shock protein binding activity; and protein folding chaperone. Predicted to be involved in chaperone cofactor-dependent protein refolding and protein refolding. Located in cytosol and ribosome. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 24424565, 24495205, 43.49, 6, 'ENSG00000284024', 'A0A1W2PQ72'),
(1985, 'MSLN', 'This gene encodes a preproprotein that is proteolytically processed to generate two protein products, megakaryocyte potentiating factor and mesothelin. Megakaryocyte potentiating factor functions as a cytokine that can stimulate colony formation of bone marrow megakaryocytes. Mesothelin is a glycosylphosphatidylinositol-anchored cell-surface protein that may function as a cell adhesion protein. This protein is overexpressed in epithelial mesotheliomas, ovarian cancers and in specific squamous cell carcinomas. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Feb 2016]', '16', 'protein_coding', 41780782, 41787452, 55.24, 6, 'ENSG00000102854', 'Q13421'),
(1986, 'MSMB', 'The protein encoded by this gene is a member of the immunoglobulin binding factor family. It is synthesized by the epithelial cells of the prostate gland and secreted into the seminal plasma. This protein has inhibin-like activity. It may have a role as an autocrine paracrine factor in uterine, breast and other female reproductive tissues. The expression of the encoded protein is found to be decreased in prostate cancer. Two alternatively spliced transcript variants encoding different isoforms are described for this gene. The use of alternate polyadenylation sites has been found for this gene. [provided by RefSeq, Jul 2008]', '10', 'unprocessed_pseudogene', 27977150, 27978075, 46.98, 1, 'ENSG00000263639', 'P08118'),
(1987, 'MSRB1', 'The protein encoded by this gene belongs to the methionine-R-sulfoxide reductase B (MsrB) family. Members of this family function as repair enzymes that protect proteins from oxidative stress by catalyzing the reduction of methionine-R-sulfoxides to methionines. This protein is highly expressed in liver and kidney, and is localized to the nucleus and cytosol. It is the only member of the MsrB family that is a selenoprotein, containing a selenocysteine (Sec) residue at its active site. It also has the highest methionine-R-sulfoxide reductase activity compared to other members containing cysteine in place of Sec. Sec is encoded by the UGA codon, which normally signals translation termination. The 3\' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. A pseudogene of this locus has been identified on chromosome 19. [provided by RefSeq, Aug 2017]', '16', 'protein_coding', 62405103, 62423195, 50.41, 2, 'ENSG00000291928', 'Q9NZV6'),
(1988, 'MSRB1', 'The protein encoded by this gene belongs to the methionine-R-sulfoxide reductase B (MsrB) family. Members of this family function as repair enzymes that protect proteins from oxidative stress by catalyzing the reduction of methionine-R-sulfoxides to methionines. This protein is highly expressed in liver and kidney, and is localized to the nucleus and cytosol. It is the only member of the MsrB family that is a selenoprotein, containing a selenocysteine (Sec) residue at its active site. It also has the highest methionine-R-sulfoxide reductase activity compared to other members containing cysteine in place of Sec. Sec is encoded by the UGA codon, which normally signals translation termination. The 3\' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. A pseudogene of this locus has been identified on chromosome 19. [provided by RefSeq, Aug 2017]', '16', 'protein_coding', 62405103, 62423195, 50.41, 2, 'ENSG00000198736', 'Q9NZV6'),
(1989, 'MSS51', 'Predicted to enable zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 54151606, 54224669, 39.18, 5, 'ENSG00000166343', 'Q4VC12'),
(1990, 'MSX1', 'This gene encodes a member of the muscle segment homeobox gene family. The encoded protein functions as a transcriptional repressor during embryogenesis through interactions with components of the core transcription complex and other homeoproteins. It may also have roles in limb-pattern formation, craniofacial development, particularly odontogenesis, and tumor growth inhibition. Mutations in this gene, which was once known as homeobox 7, have been associated with nonsyndromic cleft lip with or without cleft palate 5, Witkop syndrome, Wolf-Hirschom syndrome, and autosomoal dominant hypodontia. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 68843531, 69085182, 43.09, 10, 'ENSG00000163132', 'P28360'),
(1991, 'MT-ATP6', 'Enables proton channel activity. Contributes to proton-transporting ATP synthase activity, rotational mechanism. Involved in proton motive force-driven mitochondrial ATP synthesis and proton transmembrane transport. Located in mitochondrion. Part of proton-transporting ATP synthase complex. Implicated in Leber hereditary optic neuropathy; NARP syndrome; Parkinson\'s disease; multiple sclerosis; and systemic lupus erythematosus. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 8527, 9207, 44.20, 1, 'ENSG00000198899', 'P00846'),
(1992, 'MT-CO1', 'Contributes to cytochrome-c oxidase activity. Predicted to be involved in aerobic respiration; positive regulation of vasoconstriction; and respiratory electron transport chain. Located in mitochondrial membrane. Part of respiratory chain complex IV. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 5904, 7445, 46.24, 1, 'ENSG00000198804', 'P00395'),
(1993, 'MT-CO2', 'Contributes to cytochrome-c oxidase activity. Predicted to be involved in mitochondrial electron transport, cytochrome c to oxygen and positive regulation of vasoconstriction. Located in mitochondrial inner membrane. Part of respiratory chain complex IV. Biomarker of Huntington\'s disease and stomach cancer. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 7586, 8269, 46.20, 1, 'ENSG00000198712', 'P00403'),
(1994, 'MT-CYB', 'Predicted to enable metal ion binding activity. Predicted to contribute to ubiquinol-cytochrome-c reductase activity. Predicted to be involved in mitochondrial electron transport, ubiquinol to cytochrome c. Located in mitochondrial inner membrane. Implicated in ovarian carcinoma and urinary bladder cancer. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 14747, 15887, 46.28, 1, 'ENSG00000198727', 'P00156'),
(1995, 'MT-ND2', 'Enables NADH dehydrogenase (ubiquinone) activity. Involved in mitochondrial electron transport, NADH to ubiquinone and mitochondrial respiratory chain complex I assembly. Located in mitochondrial inner membrane. Part of respiratory chain complex I. Implicated in Leber hereditary optic neuropathy; multiple sclerosis; myocardial infarction; neurodegenerative disease (multiple); and urinary bladder cancer. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 4470, 5511, 42.99, 1, 'ENSG00000198763', 'P03891'),
(1996, 'MT-ND3', 'Enables NADH dehydrogenase (ubiquinone) activity. Involved in mitochondrial electron transport, NADH to ubiquinone. Located in mitochondrial inner membrane. Part of respiratory chain complex I. Implicated in Leber hereditary optic neuropathy; Leigh disease; and Parkinson\'s disease. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 10059, 10404, 40.17, 1, 'ENSG00000198840', 'P03897'),
(1997, 'MT-ND4', 'Enables NADH dehydrogenase (ubiquinone) activity. Involved in mitochondrial electron transport, NADH to ubiquinone and mitochondrial respiratory chain complex I assembly. Located in mitochondrial inner membrane. Part of respiratory chain complex I. Implicated in Leber hereditary optic neuropathy; Parkinson\'s disease; macular degeneration; and schizophrenia. Biomarker of Alzheimer\'s disease. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'protein_coding', 10760, 12137, 44.27, 1, 'ENSG00000198886', 'P03905');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(1998, 'MT-RNR2', 'Enables G protein-coupled receptor binding activity; identical protein binding activity; and receptor antagonist activity. Involved in several processes, including cellular response to amyloid-beta; negative regulation of macromolecule metabolic process; and regulation of defense response. Located in several cellular components, including mitochondrion; perinuclear region of cytoplasm; and sperm midpiece. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'Mt_rRNA', 1671, 3229, 42.81, 1, 'ENSG00000210082', 'Q8IVG9'),
(1999, 'MT-TF', 'Predicted to enable triplet codon-amino acid adaptor activity. Predicted to be involved in translation. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'Mt_tRNA', 577, 647, 40.85, 1, 'ENSG00000210049', ''),
(2000, 'MT-TN', 'Predicted to enable triplet codon-amino acid adaptor activity. Predicted to be involved in translation. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'Mt_tRNA', 5657, 5729, 41.10, 1, 'ENSG00000210135', ''),
(2001, 'MT-TT', 'Predicted to enable triplet codon-amino acid adaptor activity. Predicted to be involved in translation. [provided by Alliance of Genome Resources, May 2025]', 'MT', 'Mt_tRNA', 15888, 15953, 37.88, 1, 'ENSG00000210195', ''),
(2002, 'MT1A', 'This gene is a member of the metallothionein family of genes. Proteins encoded by this gene family are low in molecular weight, are cysteine-rich, lack aromatic residues, and bind divalent heavy metal ions. The conserved cysteine residues co-ordinate metal ions using mercaptide linkages. These proteins act as anti-oxidants, protect against hydroxyl free radicals, are important in homeostatic control of metal in the cell, and play a role in detoxification of heavy metals. Disruption of two metallothionein genes in mouse resulted in defects in protection against heavy metals, oxidative stress, immune reactions, carcinogens, and displayed obesity. [provided by RefSeq, Sep 2017]', '16', 'protein_coding', 18387350, 18468870, 39.19, 2, 'ENSG00000205362', 'P04731'),
(2003, 'MT1X', 'Predicted to enable copper ion binding activity and zinc ion binding activity. Involved in cellular response to cadmium ion; cellular response to erythropoietin; and cellular response to zinc ion. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'miRNA', 100881007, 100881120, 58.77, 1, 'ENSG00000187193', 'P80297'),
(2004, 'MT2A', 'This gene is a member of the metallothionein family of genes. Proteins encoded by this gene family are low in molecular weight, are cysteine-rich, lack aromatic residues, and bind divalent heavy metal ions, altering the intracellular concentration of heavy metals in the cell. These proteins act as anti-oxidants, protect against hydroxyl free radicals, are important in homeostatic control of metal in the cell, and play a role in detoxification of heavy metals. The encoded protein interacts with the protein encoded by the homeobox containing 1 gene in some cell types, controlling intracellular zinc levels, affecting apoptotic and autophagy pathways. Some polymorphisms in this gene are associated with an increased risk of cancer. [provided by RefSeq, Sep 2017]', '16', 'miRNA', 34382069, 34382150, 73.17, 1, 'ENSG00000125148', 'P02795'),
(2005, 'MT3', 'This gene is a member of the metallothionein family of genes. Proteins encoded by this gene family are low in molecular weight, are cysteine-rich, lack aromatic residues, and bind divalent heavy metal ions. This gene family member displays tissue-specific expression, and contains a threonine insert near its N-terminus and a glutamate-rich hexapeptide insert near its C-terminus relative to the proteins encoded by other gene family members. It plays an important role in zinc and copper homeostasis, and is induced under hypoxic conditions. The encoded protein is a growth inhibitory factor, and reduced levels of the protein are observed in the brains of individuals with some metal-linked neurodegenerative disorders such as Alzheimer\'s disease. [provided by RefSeq, Sep 2017]', '16', 'protein_coding', 137964020, 138016364, 36.83, 8, 'ENSG00000087250', 'P25713'),
(2006, 'MTA1', 'This gene encodes a protein that was identified in a screen for genes expressed in metastatic cells, specifically, mammary adenocarcinoma cell lines. Expression of this gene has been correlated with the metastatic potential of at least two types of carcinomas although it is also expressed in many normal tissues. The role it plays in metastasis is unclear. It was initially thought to be the 70kD component of a nucleosome remodeling deacetylase complex, NuRD, but it is more likely that this component is a different but very similar protein. These two proteins are so closely related, though, that they share the same types of domains. These domains include two DNA binding domains, a dimerization domain, and a domain commonly found in proteins that methylate DNA. The profile and activity of this gene product suggest that it is involved in regulating transcription and that this may be accomplished by chromatin remodeling. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2011]', '14', 'protein_coding', 39175183, 39183220, 40.64, 6, 'ENSG00000182979', 'Q13330'),
(2007, 'MTCL2', 'Involved in chromosome segregation. Located in extracellular space; microtubule; and midbody. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 21654690, 21658330, 58.39, 1, 'ENSG00000149639', 'O94964'),
(2008, 'MTFP1', 'MTP18 is a mitochondrial protein and downstream target of the phosphatidylinositol 3-kinase (see PIK3CA, MIM 171834) signaling pathway that plays a role in cell viability and mitochondrial dynamics (Tondera et al., 2004 [PubMed 15155745]).[supplied by OMIM, Mar 2008]', '22', 'protein_coding', 101205977, 101218420, 58.50, 16, 'ENSG00000242114', 'Q9UDX5'),
(2009, 'MTG1', 'Enables GTPase activity. Involved in mitochondrial large ribosomal subunit assembly; regulation of mitochondrial translation; and regulation of respiratory system process. Located in mitochondrial inner membrane; mitochondrial ribosome; and nucleoplasm. Is active in mitochondrial matrix. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 49314, 53277, 55.83, 2, 'ENSG00000148824', 'Q9BT17'),
(2010, 'MTHFD1L', 'The protein encoded by this gene is involved in the synthesis of tetrahydrofolate (THF) in the mitochondrion. THF is important in the de novo synthesis of purines and thymidylate and in the regeneration of methionine from homocysteine. Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jun 2011]', '6', 'protein_coding', 1697195, 1697755, 61.50, 1, 'ENSG00000120254', 'Q6UB35'),
(2011, 'MTREX', 'Enables ATP binding activity and RNA helicase activity. Involved in DNA damage response; RNA catabolic process; and maturation of 5.8S rRNA. Located in nuclear exosome (RNase complex) and nucleoplasm. Part of TRAMP complex and catalytic step 2 spliceosome. Biomarker of amyotrophic lateral sclerosis. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 122271296, 122338159, 45.42, 3, 'ENSG00000039123', 'P42285'),
(2012, 'MTUS2', 'Enables microtubule binding activity and protein homodimerization activity. Located in centrosome; cytoplasmic microtubule; and intercellular bridge. Part of nucleus. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 20221962, 20232365, 46.22, 14, 'ENSG00000132938', 'Q5JR59'),
(2013, 'MUC2', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. The protein polymerizes into a gel of which 80% is composed of oligosaccharide side chains by weight. The protein features a central domain containing tandem repeats rich in threonine and proline that varies between 50 and 115 copies in different individuals. Downregulation of this gene has been observed in patients with Crohn disease and ulcerative colitis. [provided by RefSeq, Oct 2016]', '11', 'protein_coding', 30214006, 30216097, 37.33, 1, 'ENSG00000293532', 'Q02817'),
(2014, 'MUC2', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. The protein polymerizes into a gel of which 80% is composed of oligosaccharide side chains by weight. The protein features a central domain containing tandem repeats rich in threonine and proline that varies between 50 and 115 copies in different individuals. Downregulation of this gene has been observed in patients with Crohn disease and ulcerative colitis. [provided by RefSeq, Oct 2016]', '11', 'protein_coding', 30214006, 30216097, 37.33, 1, 'ENSG00000278466', 'Q02817'),
(2015, 'MUC2', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. The protein polymerizes into a gel of which 80% is composed of oligosaccharide side chains by weight. The protein features a central domain containing tandem repeats rich in threonine and proline that varies between 50 and 115 copies in different individuals. Downregulation of this gene has been observed in patients with Crohn disease and ulcerative colitis. [provided by RefSeq, Oct 2016]', '11', 'protein_coding', 30214006, 30216097, 37.33, 1, 'ENSG00000198788', 'Q02817'),
(2016, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000204544', 'Q5SSG8'),
(2017, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000231350', 'Q5SSG8'),
(2018, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000227506', 'Q5SSG8'),
(2019, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000234137', 'Q5SSG8'),
(2020, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000223590', 'Q5SSG8'),
(2021, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000237962', 'Q5SSG8'),
(2022, 'MUC21', 'This gene encodes a large membrane-bound glycoprotein which is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. The encoded protein contains an N-terminal signal sequence, an extracellular mucin domain, a stem domain, a transmembrane domain, and a C-terminal cytoplasmic tail domain. The mucin domain contains O-glycosylation sites and is polymorphic with isoforms containing a variable number of nonidentical proline-, threonine-, and serine-rich tandem repeats of 15 amino acids each. The aberrent expression of this gene is associated with lung adenocarcinoma. [provided by RefSeq, May 2017]', '6', 'protein_coding', 33358839, 33401561, 44.40, 16, 'ENSG00000206463', 'Q5SSG8'),
(2023, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000237108', 'E2RYF6'),
(2024, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000234063', 'E2RYF6'),
(2025, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000229199', 'E2RYF6'),
(2026, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000234461', 'E2RYF6'),
(2027, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000227132', 'E2RYF6'),
(2028, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000231474', 'E2RYF6'),
(2029, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000261272', 'E2RYF6'),
(2030, 'MUC22', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 123536997, 123557904, 35.56, 25, 'ENSG00000229126', 'E2RYF6'),
(2031, 'MUC6', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. [provided by RefSeq, Dec 2016]', '11', 'protein_coding', 46324124, 46445769, 51.58, 25, 'ENSG00000277281', 'Q6W4X9'),
(2032, 'MUC6', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. [provided by RefSeq, Dec 2016]', '11', 'protein_coding', 46324124, 46445769, 51.58, 25, 'ENSG00000184956', 'Q6W4X9'),
(2033, 'MUC6', 'This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen. [provided by RefSeq, Dec 2016]', '11', 'protein_coding', 46324124, 46445769, 51.58, 25, 'ENSG00000277518', 'Q6W4X9'),
(2034, 'MXRA5', 'This gene encodes one of the matrix-remodelling associated proteins. This protein contains 7 leucine-rich repeats and 12 immunoglobulin-like C2-type domains related to perlecan. This gene has a pseudogene on chromosome Y. [provided by RefSeq, Mar 2010]', 'X', 'protein_coding', 35366694, 35377337, 42.67, 6, 'ENSG00000101825', 'Q9NR99'),
(2035, 'MYBPC1', 'This gene encodes a member of the myosin-binding protein C family. Myosin-binding protein C family members are myosin-associated proteins found in the cross-bridge-bearing zone (C region) of A bands in striated muscle. The encoded protein is the slow skeletal muscle isoform of myosin-binding protein C and plays an important role in muscle contraction by recruiting muscle-type creatine kinase to myosin filaments. Mutations in this gene are associated with distal arthrogryposis type I. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Dec 2011]', '12', 'unprocessed_pseudogene', 50049, 50945, 41.81, 1, 'ENSG00000196091', 'Q00872'),
(2036, 'MYCBP2', 'This gene encodes an E3 ubiquitin-protein ligase and member of the PHR (Phr1/MYCBP2, highwire and RPM-1) family of proteins. The encoded protein plays a role in axon guidance and synapse formation in the developing nervous system. In mammalian cells, this protein regulates the cAMP and mTOR signaling pathways, and may additionally regulate autophagy. Reduced expression of this gene has been observed in acute lymphoblastic leukemia patients and a mutation in this gene has been identified in patients with a rare inherited vision defect. [provided by RefSeq, Mar 2017]', '13', 'protein_coding', 41301525, 41360147, 54.29, 2, 'ENSG00000005810', 'O75592'),
(2037, 'MYCLP1', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 157981863, 158099176, 49.67, 9, 'ENSG00000204053', 'P12525'),
(2038, 'MYG1', 'Enables 3\'-5\'-RNA exonuclease activity. Involved in mRNA processing; mitochondrial RNA metabolic process; and rRNA processing. Located in mitochondrion and nucleus. Is active in mitochondrial matrix; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 83861126, 83886699, 32.67, 2, 'ENSG00000291835', 'Q9HB07'),
(2039, 'MYG1', 'Enables 3\'-5\'-RNA exonuclease activity. Involved in mRNA processing; mitochondrial RNA metabolic process; and rRNA processing. Located in mitochondrion and nucleus. Is active in mitochondrial matrix; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 83861126, 83886699, 32.67, 2, 'ENSG00000139637', 'Q9HB07'),
(2040, 'MYH13', 'Predicted to enable actin filament binding activity and microfilament motor activity. Predicted to be involved in muscle contraction. Predicted to act upstream of or within cellular response to starvation. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 24216857, 24232367, 43.50, 8, 'ENSG00000006788', 'Q9UKX3'),
(2041, 'MYH8', 'Myosins are actin-based motor proteins that function in the generation of mechanical force in eukaryotic cells. Muscle myosins are heterohexamers composed of 2 myosin heavy chains and 2 pairs of nonidentical myosin light chains. This gene encodes a member of the class II or conventional myosin heavy chains, and functions in skeletal muscle contraction. This gene is predominantly expressed in fetal skeletal muscle. This gene is found in a cluster of myosin heavy chain genes on chromosome 17. A mutation in this gene results in trismus-pseudocamptodactyly syndrome. [provided by RefSeq, Sep 2009]', '17', 'protein_coding', 99969351, 100035634, 37.43, 24, 'ENSG00000133020', 'P13535'),
(2042, 'MYL1', 'Myosin is a hexameric ATPase cellular motor protein. It is composed of two heavy chains, two nonphosphorylatable alkali light chains, and two phosphorylatable regulatory light chains. This gene encodes a myosin alkali light chain expressed in fast skeletal muscle. Two transcript variants have been identified for this gene. [provided by RefSeq, Jul 2008]', '2', 'miRNA', 175369540, 175369621, 37.80, 1, 'ENSG00000168530', 'P05976'),
(2043, 'MYL12A', 'This gene encodes a nonsarcomeric myosin regulatory light chain. This protein is activated by phosphorylation and regulates smooth muscle and non-muscle cell contraction. This protein may also be involved in DNA damage repair by sequestering the transcriptional regulator apoptosis-antagonizing transcription factor (AATF)/Che-1 which functions as a repressor of p53-driven apoptosis. Alternate splicing results in multiple transcript variants. A pseudogene of this gene is found on chromosome 8.[provided by RefSeq, Dec 2014]', '18', 'protein_coding', 80547, 119527, 45.82, 8, 'ENSG00000101608', 'P19105'),
(2044, 'MYL12B', 'The activity of nonmuscle myosin II (see MYH9; MIM 160775) is regulated by phosphorylation of a regulatory light chain, such as MRLC2. This phosphorylation results in higher MgATPase activity and the assembly of myosin II filaments (Iwasaki et al., 2001 [PubMed 11942626]).[supplied by OMIM, Mar 2008]', '18', 'protein_coding', 38380027, 38661780, 41.91, 15, 'ENSG00000118680', 'O14950'),
(2045, 'MYL9', 'Myosin, a structural component of muscle, consists of two heavy chains and four light chains. The protein encoded by this gene is a myosin light chain that may regulate muscle contraction by modulating the ATPase activity of myosin heads. The encoded protein binds calcium and is activated by myosin light chain kinase. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 196763, 409417, 37.01, 16, 'ENSG00000101335', 'P24844'),
(2046, 'MYOD1', 'This gene encodes a nuclear protein that belongs to the basic helix-loop-helix family of transcription factors and the myogenic factors subfamily. It regulates muscle cell differentiation by inducing cell cycle arrest, a prerequisite for myogenic initiation. The protein is also involved in muscle regeneration. It activates its own transcription which may stabilize commitment to myogenesis. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 63624087, 63671921, 42.43, 4, 'ENSG00000129152', 'P15172'),
(2047, 'MYOZ1', 'The protein encoded by this gene is primarily expressed in the skeletal muscle, and belongs to the myozenin family. Members of this family function as calcineurin-interacting proteins that help tether calcineurin to the sarcomere of cardiac and skeletal muscle. They play an important role in modulation of calcineurin signaling. [provided by RefSeq, Apr 2012]', '10', 'miRNA', 99124609, 99124696, 28.41, 1, 'ENSG00000177791', 'Q9NP98'),
(2048, 'MYOZ2', 'The protein encoded by this gene belongs to a family of sarcomeric proteins that bind to calcineurin, a phosphatase involved in calcium-dependent signal transduction in diverse cell types. These family members tether calcineurin to alpha-actinin at the z-line of the sarcomere of cardiac and skeletal muscle cells, and thus they are important for calcineurin signaling. Mutations in this gene cause cardiomyopathy familial hypertrophic type 16, a hereditary heart disorder. [provided by RefSeq, Aug 2011]', '4', 'protein_coding', 95117923, 95247225, 39.33, 2, 'ENSG00000172399', 'Q9NPC6'),
(2049, 'MYPN', 'Striated muscle in vertebrates comprises large proteins which must be organized properly to contract efficiently. Z-lines in striated muscle are a sign of this organization, representing the ends of actin thin filaments, titin, nebulin or nebulette and accessory proteins required for structure and function. This gene encodes a protein which interacts with nebulin in skeletal muscle or nebulette in cardiac muscle and alpha-actinin. In addition, this gene product can interact with a protein with the I-band indicating it has a regulatory as well as structural function. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2011]', '10', 'protein_coding', 214614, 215377, 57.20, 1, 'ENSG00000138347', 'Q86TC9'),
(2050, 'MZT1', 'Involved in gamma-tubulin complex localization. Located in centrosome and spindle. Part of gamma-tubulin ring complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 39209116, 39603528, 39.07, 3, 'ENSG00000204899', 'Q08AG7'),
(2051, 'MZT2B', 'Located in cytosol; microtubule cytoskeleton; and nucleoplasm. Part of gamma-tubulin ring complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 81666737, 81673904, 57.05, 6, 'ENSG00000152082', 'Q6NZ67'),
(2052, 'MZT2B', 'Located in cytosol; microtubule cytoskeleton; and nucleoplasm. Part of gamma-tubulin ring complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 81666737, 81673904, 57.05, 6, 'ENSG00000292119', 'Q6NZ67'),
(2053, 'NAA11', 'Enables protein-N-terminal amino-acid acetyltransferase activity. Involved in N-terminal protein amino acid acetylation. Located in several cellular components, including Golgi apparatus; microtubule organizing center; and nucleoplasm. Part of NatA complex. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 26025237, 26025310, 40.54, 1, 'ENSG00000156269', 'Q9BSU3'),
(2054, 'NAA16', 'Enables ribosome binding activity. Involved in N-terminal protein amino acid acetylation; negative regulation of apoptotic process; and protein stabilization. Located in cytosol. Part of NatA complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 18813783, 18814378, 47.15, 1, 'ENSG00000172766', 'Q6N069'),
(2055, 'NAA20', 'NAT5 is a component of N-acetyltransferase complex B (NatB). Human NatB performs cotranslational N(alpha)-terminal acetylation of methionine residues when they are followed by asparagine (Starheim et al., 2008 [PubMed 18570629]).[supplied by OMIM, Apr 2009]', '20', 'protein_coding', 103907189, 104164379, 37.11, 3, 'ENSG00000173418', 'P61599'),
(2056, 'NAA40', 'Enables histone H2A acetyltransferase activity; histone H4 acetyltransferase activity; and protein N-terminal-serine acetyltransferase activity. Predicted to be involved in chromatin remodeling. Predicted to act upstream of or within lipid metabolic process. Located in centriolar satellite; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'processed_pseudogene', 31287510, 31288964, 49.42, 1, 'ENSG00000110583', 'Q86UY6'),
(2057, 'NAALAD2', 'This gene is a member of the N-acetylated alpha-linked acidic dipeptidase (NAALADase) gene family. The representative member of this family is the gene encoding human prostate-specific membrane antigen (PSM), which is a marker of prostatic carcinomas and is the first to be shown to possess NAALADase activity. NAALADase cleaves N-acetyl-L-aspartate-L-glutamate (NAAG), which is a neuropeptide expressed both in the central nervous systems and in the periphery and is thought to function as a neurotransmitter. The product of this gene is a type II integral membrane protein. Transient transfection of this gene confers both NAALADase and dipetidyl peptidase IV activities to mammalian cells. This gene is highly expressed in ovary and testis as well as within discrete brain areas. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2014]', '11', 'protein_coding', 51630313, 52160017, 39.84, 9, 'ENSG00000077616', 'Q9Y3Q0'),
(2058, 'NAALAD2', 'This gene is a member of the N-acetylated alpha-linked acidic dipeptidase (NAALADase) gene family. The representative member of this family is the gene encoding human prostate-specific membrane antigen (PSM), which is a marker of prostatic carcinomas and is the first to be shown to possess NAALADase activity. NAALADase cleaves N-acetyl-L-aspartate-L-glutamate (NAAG), which is a neuropeptide expressed both in the central nervous systems and in the periphery and is thought to function as a neurotransmitter. The product of this gene is a type II integral membrane protein. Transient transfection of this gene confers both NAALADase and dipetidyl peptidase IV activities to mammalian cells. This gene is highly expressed in ovary and testis as well as within discrete brain areas. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2014]', '11', 'protein_coding', 51630313, 52160017, 39.84, 9, 'ENSG00000285516', 'Q9Y3Q0'),
(2059, 'NAALADL2', 'Predicted to act upstream of or within response to bacterium. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 247588, 249197, 58.26, 1, 'ENSG00000177694', 'Q58DX5'),
(2060, 'NAB1', 'Predicted to enable transcription coregulator activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to act upstream of or within endochondral ossification; nervous system development; and regulation of epidermis development. Predicted to be located in nucleoplasm. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 86666785, 86736072, 53.15, 22, 'ENSG00000138386', 'Q13506'),
(2061, 'NACAD', 'Predicted to enable unfolded protein binding activity. Predicted to be involved in protein targeting to membrane. Predicted to be located in nucleus. Predicted to be part of nascent polypeptide-associated complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 50320027, 50328251, 59.39, 6, 'ENSG00000136274', 'O15069'),
(2062, 'NAF1', 'This gene encodes an A20-binding protein which plays a role in autoimmunity and tissue homeostasis through the regulation of nuclear factor kappa-B activation. Mutations in this gene have been associated with psoriatic arthritis, rheumatoid arthritis, and systemic lupus erythematosus. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2011]', '5', 'protein_coding', 1347265, 1354844, 38.25, 2, 'ENSG00000145414', 'Q96HR8'),
(2063, 'NALF1', 'Predicted to contribute to stretch-activated, monoatomic cation-selective, calcium channel activity. Predicted to be involved in calcium ion import across plasma membrane. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 62715541, 62980433, 39.46, 7, 'ENSG00000204442', 'B1AL88'),
(2064, 'NALF2', 'This gene encodes a product belonging to a family of proteins with unknown function. The presence of two transmembrane domains suggests that this protein is a multi-pass membrane protein. [provided by RefSeq, Sep 2011]', 'X', 'protein_coding', 3777504, 3781448, 57.54, 3, 'ENSG00000130054', 'O75949'),
(2065, 'NAMPT', 'This gene encodes a protein that catalyzes the condensation of nicotinamide with 5-phosphoribosyl-1-pyrophosphate to yield nicotinamide mononucleotide, one step in the biosynthesis of nicotinamide adenine dinucleotide. The protein belongs to the nicotinic acid phosphoribosyltransferase (NAPRTase) family and is thought to be involved in many important biological processes, including metabolism, stress response and aging. This gene has a pseudogene on chromosome 10. [provided by RefSeq, Feb 2011]', '7', 'protein_coding', 48761747, 48773648, 45.40, 4, 'ENSG00000105835', 'P43490'),
(2066, 'NANP', 'Enables N-acylneuraminate-9-phosphatase activity. Involved in CMP-N-acetylneuraminate biosynthetic process; N-acetylneuraminate biosynthetic process; and glycosylation. Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 25782915, 25832052, 36.74, 4, 'ENSG00000170191', 'Q8TBE9'),
(2067, 'NAP1L2', 'The protein encoded by this intronless gene is a member of the nucleosome assembly protein (NAP) family. The encoded protein represents a class of tissue-specific factors that interact with chromatin to regulate neuronal cell proliferation. [provided by RefSeq, Jan 2011]', 'X', 'protein_coding', 46159185, 46170980, 38.41, 3, 'ENSG00000186462', 'Q9ULW6'),
(2068, 'NAP1L3', 'This gene is intronless and encodes a member of the nucleosome assembly protein (NAP) family. This gene is linked closely to a region of genes responsible for several X-linked cognitive disability syndromes. [provided by RefSeq, Dec 2010]', 'X', 'protein_coding', 310221, 311920, 52.06, 1, 'ENSG00000186310', 'Q99457'),
(2069, 'NAPRT', 'Nicotinic acid (NA; niacin) is converted by nicotinic acid phosphoribosyltransferase (NAPRT; EC 2.4.2.11) to NA mononucleotide (NaMN), which is then converted to NA adenine dinucleotide (NaAD), and finally to nicotinamide adenine dinucleotide (NAD), which serves as a coenzyme in cellular redox reactions and is an essential component of a variety of processes in cellular metabolism including response to stress (Hara et al., 2007).[supplied by OMIM, Mar 2008]', '8', 'protein_coding', 1823926, 1958641, 50.01, 15, 'ENSG00000147813', 'Q6XQN6'),
(2070, 'NAPRT', 'Nicotinic acid (NA; niacin) is converted by nicotinic acid phosphoribosyltransferase (NAPRT; EC 2.4.2.11) to NA mononucleotide (NaMN), which is then converted to NA adenine dinucleotide (NaAD), and finally to nicotinamide adenine dinucleotide (NAD), which serves as a coenzyme in cellular redox reactions and is an essential component of a variety of processes in cellular metabolism including response to stress (Hara et al., 2007).[supplied by OMIM, Mar 2008]', '8', 'protein_coding', 1823926, 1958641, 50.01, 15, 'ENSG00000278488', 'Q6XQN6'),
(2071, 'NARS1', ' Aminoacyl-tRNA synthetases are a class of enzymes that charge tRNAs with their cognate amino acids.  Asparaginyl-tRNA synthetase is localized to the cytoplasm and belongs to the class II family of tRNA synthetases.  The N-terminal domain represents the signature sequence for the eukaryotic asparaginyl-tRNA synthetases. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 62231922, 62296213, 52.75, 28, 'ENSG00000134440', 'O43776'),
(2072, 'NAT8', 'This gene, isolated using the differential display method to detect tissue-specific genes, is specifically expressed in kidney and liver. The encoded protein shows amino acid sequence similarity to N-acetyltransferases. A similar protein in Xenopus affects cell adhesion and gastrulation movements, and may be localized in the secretory pathway. A highly similar paralog is found in a cluster with this gene. [provided by RefSeq, Sep 2008]', '2', 'protein_coding', 68673688, 68715108, 36.04, 14, 'ENSG00000291479', 'Q9UHE5'),
(2073, 'NAT8', 'This gene, isolated using the differential display method to detect tissue-specific genes, is specifically expressed in kidney and liver. The encoded protein shows amino acid sequence similarity to N-acetyltransferases. A similar protein in Xenopus affects cell adhesion and gastrulation movements, and may be localized in the secretory pathway. A highly similar paralog is found in a cluster with this gene. [provided by RefSeq, Sep 2008]', '2', 'protein_coding', 68673688, 68715108, 36.04, 14, 'ENSG00000144035', 'Q9UHE5'),
(2074, 'NAV3', 'This gene belongs to the neuron navigator family and is expressed predominantly in the nervous system. The encoded protein contains coiled-coil domains and a conserved AAA domain characteristic for ATPases associated with a variety of cellular activities. This gene is similar to unc-53, a Caenorhabditis elegans gene involved in axon guidance. Multiple alternatively spliced transcript variants for this gene have been described but only one has had its full-length nature determined. [provided by RefSeq, Jul 2008]', '12', 'miRNA', 53195411, 53195488, 53.85, 1, 'ENSG00000067798', 'Q8IVL0'),
(2075, 'NBEA', 'This gene encodes a member of a large, diverse group of A-kinase anchor proteins that target the activity of protein kinase A to specific subcellular sites by binding to its type II regulatory subunits. Brain-specific expression and coat protein-like membrane recruitment of a highly similar protein in mouse suggest an involvement in neuronal post-Golgi membrane traffic. Mutations in this gene may be associated with a form of autism. This gene and its expression are frequently disrupted in patients with multiple myeloma. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Additional transcript variants may exist, but their full-length nature has not been determined.[provided by RefSeq, Feb 2011]', '13', 'protein_coding', 26132115, 26222314, 40.13, 7, 'ENSG00000172915', 'Q8NFP9'),
(2076, 'NBPF21P', 'This pseudogene on chromosome 3 is a member of the neuroblastoma breakpoint family (NBPF) which consists of dozens of recently duplicated genes primarily located in segmental duplications on human chromosome 1. This gene family has experienced its greatest expansion within the human lineage and has expanded, to a lesser extent, among primates in general. Members of this gene family are characterized by tandemly repeated copies of DUF1220 protein domains. Gene copy number variations in the human chromosomal region 1q21.1, where most DUF1220 domains are located, have been implicated in a number of developmental and neurogenetic diseases such as microcephaly, macrocephaly, autism, schizophrenia, cognitive disability, congenital heart disease, neuroblastoma, and congenital kidney and urinary tract anomalies. Altered expression of some gene family members is associated with several types of cancer. This gene family contains numerous pseudogenes. [provided by RefSeq, Jun 2013]', '3', 'protein_coding', 7231089, 7232044, 58.26, 1, 'ENSG00000231382', ''),
(2077, 'NCAM2', 'The protein encoded by this gene contains a paired-like homeodomain most similar to that of the Drosophila aristaless gene product. The encoded protein plays a central role in development of the autonomic nervous system. It regulates the expression of tyrosine hydroxylase and dopamine beta-hydroxylase, two catecholaminergic biosynthetic enzymes essential for the differentiation and maintenance of the noradrenergic neurotransmitter phenotype. The encoded protein has also been shown to regulate transcription of the alpha3 nicotinic acetylcholine receptor gene. Mutations in this gene have been associated with autosomal recessive congenital fibrosis of the extraocular muscles. [provided by RefSeq, Jul 2008]', '11', 'unprocessed_pseudogene', 58021, 58938, 47.71, 1, 'ENSG00000154654', 'O15394'),
(2078, 'NCBP1', 'The product of this gene is a component of the nuclear cap-binding protein complex (CBC), which binds to the monomethylated 5\' cap of nascent pre-mRNA in the nucleoplasm. The encoded protein promotes high-affinity mRNA-cap binding and associates with the CTD of RNA polymerase II. The CBC promotes pre-mRNA splicing, 3\'-end processing, RNA nuclear export, and nonsense-mediated mRNA decay. [provided by RefSeq, Jul 2008]', '9', 'TR_J_gene', 22509341, 22509406, 46.97, 1, 'ENSG00000136937', 'Q09161'),
(2079, 'NCOA3', 'The protein encoded by this gene is a nuclear receptor coactivator that interacts with nuclear hormone receptors to enhance their transcriptional activator functions. The encoded protein has histone acetyltransferase activity and recruits p300/CBP-associated factor and CREB binding protein as part of a multisubunit coactivation complex. This protein is initially found in the cytoplasm but is translocated into the nucleus upon phosphorylation. Several transcript variants encoding different isoforms have been found for this gene. In addition, a polymorphic repeat region is found in the C-terminus of the encoded protein. [provided by RefSeq, Mar 2010]', '20', 'protein_coding', 61252261, 61940617, 51.80, 3, 'ENSG00000124151', 'Q9Y6Q9'),
(2080, 'NCOA5', 'This gene encodes a coregulator for the alpha and beta estrogen receptors and the orphan nuclear receptor NR1D2. The protein localizes to the nucleus, and is thought to have both coactivator and corepressor functions. Its interaction with nuclear receptors is independent of the AF2 domain on the receptors, which is known to regulate interaction with other coreceptors. Several alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jan 2017]', '20', 'protein_coding', 123008979, 123058290, 40.73, 4, 'ENSG00000124160', 'Q9HCD5'),
(2081, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000204475', 'O14931'),
(2082, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000236315', 'O14931'),
(2083, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000206430', 'O14931'),
(2084, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000237808', 'O14931'),
(2085, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000225211', 'O14931'),
(2086, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000236979', 'O14931'),
(2087, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000237103', 'O14931'),
(2088, 'NCR3', 'The protein encoded by this gene is a natural cytotoxicity receptor (NCR) that may aid NK cells in the lysis of tumor cells. The encoded protein interacts with CD3-zeta (CD247), a T-cell receptor. A single nucleotide polymorphism in the 5\' untranslated region of this gene has been associated with mild malaria suceptibility. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '6', 'protein_coding', 22946228, 22957161, 45.59, 21, 'ENSG00000223833', 'O14931'),
(2089, 'NDC80', 'This gene encodes a component of the NDC80 kinetochore complex. The encoded protein consists of an N-terminal microtubule binding domain and a C-terminal coiled-coiled domain that interacts with other components of the complex. This protein functions to organize and stabilize microtubule-kinetochore interactions and is required for proper chromosome segregation. [provided by RefSeq, Oct 2011]', '18', 'protein_coding', 93661682, 93730358, 38.56, 7, 'ENSG00000080986', 'O14777'),
(2090, 'NDEL1', 'Enables identical protein binding activity. Involved in chromosome segregation; positive regulation of GTPase activity; and regulation of intracellular protein transport. Located in kinetochore. Biomarker of schizophrenia. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 3146519, 3159865, 44.47, 5, 'ENSG00000166579', 'Q9GZM8'),
(2091, 'NDFIP2', 'Enables WW domain binding activity. Involved in negative regulation of transport and regulation of macromolecule metabolic process. Located in several cellular components, including Golgi apparatus; mitochondrion; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 80014753, 80031774, 40.94, 5, 'ENSG00000102471', 'Q9NV92'),
(2092, 'NDST3', 'This gene encodes a member of the heparan sulfate/heparin GlcNAc N-deacetylase/ N-sulfotransferase family. The encoded enzyme is a type II transmembrane protein that resides in the Golgi apparatus. This monomeric bifunctional enzyme catalyzes the N-deacetylation and N-sulfation of N-acetylglucosamine residues in heparan sulfate and heparin, which are the initial chemical modifications required for the biosynthesis of the functional oligosaccharide sequences that define the specific ligand binding activities of heparan sulfate and heparin. [provided by RefSeq, Nov 2008]', '4', 'protein_coding', 46060991, 46089962, 41.79, 2, 'ENSG00000164100', 'O95803');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2093, 'NDUFA1', 'The human NDUFA1 gene codes for an essential component of complex I of the respiratory chain, which transfers electrons from NADH to ubiquinone. It has been noted that the N-terminal hydrophobic domain has the potential to be folded into an alpha-helix spanning the inner mitochondrial membrane with a C-terminal hydrophilic domain interacting with globular subunits of complex I. The highly conserved two-domain structure suggests that this feature is critical for the protein function and might act as an anchor for the NADH:ubiquinone oxidoreductase complex at the inner mitochondrial membrane. However, the NDUFA1 peptide is one of about 31 components of the \"hydrophobic protein\" (HP) fraction of complex I which is involved in proton translocation. Thus the NDUFA1 peptide may also participate in that function. [provided by RefSeq, Jul 2008]', 'X', 'unprocessed_pseudogene', 124357560, 124358469, 41.32, 1, 'ENSG00000125356', 'O15239'),
(2094, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000276220', 'O95167'),
(2095, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000276766', 'O95167'),
(2096, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000274359', 'O95167'),
(2097, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000275724', 'O95167'),
(2098, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000273642', 'O95167'),
(2099, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000278365', 'O95167'),
(2100, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000277722', 'O95167'),
(2101, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000275605', 'O95167'),
(2102, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000276061', 'O95167'),
(2103, 'NDUFA3', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43076307, 43161719, 49.40, 20, 'ENSG00000170906', 'O95167'),
(2104, 'NDUFA4', 'The protein encoded by this gene belongs to the complex I 9kDa subunit family. Mammalian complex I of mitochondrial respiratory chain is composed of 45 different subunits. This protein has NADH dehydrogenase activity and oxidoreductase activity. It transfers electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 141002594, 141003706, 49.96, 1, 'ENSG00000189043', 'O00483'),
(2105, 'NDUFA5', 'This nuclear gene encodes a conserved protein that comprises the B13 subunit of complex I of the mitochondrial respiratory chain. The encoded protein localizes to the inner mitochondrial membrane, where it is thought to aid in the transfer of electrons from NADH to ubiquinone. Alternative splicing results in multiple transcript variants. There are numerous pseudogenes of this gene on chromosomes 1, 3, 6, 8, 9, 11, 12, and 16. [provided by RefSeq, Apr 2014]', '7', 'protein_coding', 68331174, 68343193, 39.82, 11, 'ENSG00000128609', 'Q16718'),
(2106, 'NDUFA6', 'This gene encodes a member of the LYR family of proteins that contain a highly conserved tripeptide (LYR) motif near the N-terminus. The encoded protein is an accessory subunit of NADH: ubiquinone oxidorerductase (Complex I), which is the largest enzyme of the mitochondrial membrane respiratory chain. Complex I functions in electron transfer from NADH to the respiratory chain. [provided by RefSeq, Oct 2016]', '22', 'miRNA', 2445392, 2445457, 57.58, 1, 'ENSG00000184983', 'P56556'),
(2107, 'NDUFA6', 'This gene encodes a member of the LYR family of proteins that contain a highly conserved tripeptide (LYR) motif near the N-terminus. The encoded protein is an accessory subunit of NADH: ubiquinone oxidorerductase (Complex I), which is the largest enzyme of the mitochondrial membrane respiratory chain. Complex I functions in electron transfer from NADH to the respiratory chain. [provided by RefSeq, Oct 2016]', '22', 'miRNA', 2445392, 2445457, 57.58, 1, 'ENSG00000272765', 'P56556'),
(2108, 'NDUFA6', 'This gene encodes a member of the LYR family of proteins that contain a highly conserved tripeptide (LYR) motif near the N-terminus. The encoded protein is an accessory subunit of NADH: ubiquinone oxidorerductase (Complex I), which is the largest enzyme of the mitochondrial membrane respiratory chain. Complex I functions in electron transfer from NADH to the respiratory chain. [provided by RefSeq, Oct 2016]', '22', 'miRNA', 2445392, 2445457, 57.58, 1, 'ENSG00000281013', 'P56556'),
(2109, 'NDUFA6', 'This gene encodes a member of the LYR family of proteins that contain a highly conserved tripeptide (LYR) motif near the N-terminus. The encoded protein is an accessory subunit of NADH: ubiquinone oxidorerductase (Complex I), which is the largest enzyme of the mitochondrial membrane respiratory chain. Complex I functions in electron transfer from NADH to the respiratory chain. [provided by RefSeq, Oct 2016]', '22', 'miRNA', 2445392, 2445457, 57.58, 1, 'ENSG00000277365', 'P56556'),
(2110, 'NDUFA6', 'This gene encodes a member of the LYR family of proteins that contain a highly conserved tripeptide (LYR) motif near the N-terminus. The encoded protein is an accessory subunit of NADH: ubiquinone oxidorerductase (Complex I), which is the largest enzyme of the mitochondrial membrane respiratory chain. Complex I functions in electron transfer from NADH to the respiratory chain. [provided by RefSeq, Oct 2016]', '22', 'miRNA', 2445392, 2445457, 57.58, 1, 'ENSG00000273397', 'P56556'),
(2111, 'NDUFA8', 'The protein encoded by this gene belongs to the complex I 19 kDa subunit family. Mammalian complex I is composed of 45 different subunits. This protein has NADH dehydrogenase activity and oxidoreductase activity. It plays an important role in transfering electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2015]', '9', 'protein_coding', 37106734, 37172741, 40.77, 6, 'ENSG00000119421', 'P51970'),
(2112, 'NDUFAF4', 'NADH:ubiquinone oxidoreductase (complex I) catalyzes the transfer of electrons from NADH to ubiquinone (coenzyme Q) in the first step of the mitochondrial respiratory chain, resulting in the translocation of protons across the inner mitochondrial membrane. This gene encodes a complex I assembly factor. Mutations in this gene are a cause of mitochondrial complex I deficiency. [provided by RefSeq, Oct 2009]', '6', 'unprocessed_pseudogene', 5898414, 5899404, 48.13, 1, 'ENSG00000123545', 'Q9P032'),
(2113, 'NDUFAF7', 'This gene encodes an assembly factor protein which helps in the assembly and stabilization of Complex I, a large multi-subunit enzyme in the mitochondrial respiratory chain. Complex I is involved in several physiological activities in the cell, including metabolite transport and ATP synthesis. The encoded protein is a methyltransferase which methylates Arg85 of a subunit of Complex I in the early stages of its assembly. A pseudogene related to this gene is located on chromosome 8. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016]', '2', 'protein_coding', 153902531, 153907166, 62.86, 5, 'ENSG00000003509', 'Q7L592'),
(2114, 'NDUFB1', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane and nuclear speck. Part of respiratory chain complex I. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 50126499, 50126563, 46.15, 1, 'ENSG00000183648', 'O75438'),
(2115, 'NDUFB10', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. Implicated in nuclear type mitochondrial complex I deficiency 35. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 58512872, 58831435, 38.00, 3, 'ENSG00000291930', 'O96000'),
(2116, 'NDUFB10', 'Predicted to enable NADH dehydrogenase (ubiquinone) activity. Predicted to be involved in mitochondrial electron transport, NADH to ubiquinone and proton motive force-driven mitochondrial ATP synthesis. Located in mitochondrial inner membrane. Part of respiratory chain complex I. Implicated in nuclear type mitochondrial complex I deficiency 35. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 58512872, 58831435, 38.00, 3, 'ENSG00000140990', 'O96000'),
(2117, 'NDUFB6', 'The protein encoded by this gene is a subunit of the multisubunit NADH:ubiquinone oxidoreductase (complex I). Mammalian complex I is composed of 45 different subunits. It locates at the mitochondrial inner membrane. This protein has NADH dehydrogenase activity and oxidoreductase activity. It transfers electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. Alternative splicing occurs at this locus and three transcript variants encoding distinct isoforms have been identified. [provided by RefSeq, Jan 2011]', '9', 'protein_coding', 1820267, 1836753, 57.80, 12, 'ENSG00000165264', 'O95139'),
(2118, 'NDUFC1', 'The encoded protein is a subunit of the NADH:ubiquinone oxidoreductase (complex I), the first enzyme complex in the electron transport chain located in the inner mitochondrial membrane. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2010]', '4', 'miRNA', 41252992, 41253050, 62.71, 1, 'ENSG00000109390', 'O43677'),
(2119, 'NDUFV2', 'The NADH-ubiquinone oxidoreductase complex (complex I) of the mitochondrial respiratory chain catalyzes the transfer of electrons from NADH to ubiquinone, and consists of at least 43 subunits. The complex is located in the inner mitochondrial membrane. This gene encodes the 24 kDa subunit of complex I, and is involved in electron transfer. Mutations in this gene are implicated in Parkinson\'s disease, bipolar disorder, schizophrenia, and have been found in one case of early onset hypertrophic cardiomyopathy and encephalopathy. A non-transcribed pseudogene of this locus is found on chromosome 19. [provided by RefSeq, Oct 2009]', '18', 'protein_coding', 45702320, 45714559, 45.42, 4, 'ENSG00000178127', 'P19404'),
(2120, 'NEBL', 'This gene encodes a nebulin like protein that is abundantly expressed in cardiac muscle. The encoded protein binds actin and interacts with thin filaments and Z-line associated proteins in striated muscle. This protein may be involved in cardiac myofibril assembly. A shorter isoform of this protein termed LIM nebulette is expressed in non-muscle cells and may function as a component of focal adhesion complexes. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Mar 2010]', '10', 'protein_coding', 186373, 187302, 43.87, 1, 'ENSG00000078114', 'O76041'),
(2121, 'NECAB3', 'The protein encoded by this gene interacts with the amino-terminal domain of the neuron-specific X11-like protein (X11L), inhibits the association of X11L with amyloid precursor protein through a non-competitive mechanism, and abolishes the suppression of beta-amyloid production by X11L. This protein, together with X11L, may play an important role in the regulatory system of amyloid precursor protein metabolism and beta-amyloid generation. The protein is phosphorylated by NIMA-related expressed kinase 2, and localizes to the Golgi apparatus. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 16347142, 16609259, 36.94, 19, 'ENSG00000125967', 'Q96P71'),
(2122, 'NEDD8', 'Enables ubiquitin protein ligase binding activity. Acts upstream of or within protein neddylation. Located in cytosol and nucleoplasm. Biomarker of Parkinson\'s disease and malignant astrocytoma. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 42040036, 42256578, 38.24, 10, 'ENSG00000285246', 'Q15843'),
(2123, 'NEDD8', 'Enables ubiquitin protein ligase binding activity. Acts upstream of or within protein neddylation. Located in cytosol and nucleoplasm. Biomarker of Parkinson\'s disease and malignant astrocytoma. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 42040036, 42256578, 38.24, 10, 'ENSG00000129559', 'Q15843'),
(2124, 'NEDD8-MDP1', 'This locus represents naturally occurring read-through transcription between the neighboring NEDD8 (neural precursor cell expressed, developmentally down-regulated 8) and MDP1 (magnesium-dependent phosphatase 1) genes on chromosome 14. One of the read-through transcripts on this locus encodes a fusion protein that shares sequence identity with the products of each individual gene. [provided by RefSeq, Jul 2016]', '14', 'protein_coding', 1336616, 1382689, 54.54, 3, 'ENSG00000285425', 'E9PL57'),
(2125, 'NEDD8-MDP1', 'This locus represents naturally occurring read-through transcription between the neighboring NEDD8 (neural precursor cell expressed, developmentally down-regulated 8) and MDP1 (magnesium-dependent phosphatase 1) genes on chromosome 14. One of the read-through transcripts on this locus encodes a fusion protein that shares sequence identity with the products of each individual gene. [provided by RefSeq, Jul 2016]', '14', 'protein_coding', 1336616, 1382689, 54.54, 3, 'ENSG00000255526', 'E9PL57'),
(2126, 'NEK11', 'This gene encodes a member of the never in mitosis gene A family of kinases. The encoded protein localizes to the nucleoli, and may function with NEK2A in the S-phase checkpoint. The encoded protein appears to play roles in DNA replication and response to genotoxic stress. Alternatively spliced transcript variants have been described.[provided by RefSeq, Mar 2009]', '3', 'protein_coding', 150000046, 150053142, 43.45, 6, 'ENSG00000114670', 'Q8NG66'),
(2127, 'NELFA', 'This gene is expressed ubiquitously with higher levels in fetal than in adult tissues. It encodes a protein sharing 93% sequence identity with the mouse protein. Wolf-Hirschhorn syndrome (WHS) is a malformation syndrome associated with a hemizygous deletion of the distal short arm of chromosome 4. This gene is mapped to the 165 kb WHS critical region, and may play a role in the phenotype of the WHS or Pitt-Rogers-Danks syndrome. The encoded protein is found to be capable of reacting with HLA-A2-restricted and tumor-specific cytotoxic T lymphocytes, suggesting a target for use in specific immunotherapy for a large number of cancer patients. This protein has also been shown to be a member of the NELF (negative elongation factor) protein complex that participates in the regulation of RNA polymerase II transcription elongation. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 124676, 141434, 47.07, 2, 'ENSG00000185049', 'Q9H3P2'),
(2128, 'NELFCD', 'The NELF complex of proteins interacts with the DSIF protein complex to repress transcriptional elongation by RNA polymerase II. The protein encoded by this gene is an essential part of the NELF complex. Alternative translation initiation site usage results in the formation of two isoforms with different N-termini. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 751112, 764318, 56.63, 2, 'ENSG00000101158', 'Q8IXH7'),
(2129, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000228691', 'Q99519'),
(2130, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000234343', 'Q99519'),
(2131, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000184494', 'Q99519'),
(2132, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000227315', 'Q99519'),
(2133, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000227129', 'Q99519'),
(2134, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000223957', 'Q99519'),
(2135, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000234846', 'Q99519'),
(2136, 'NEU1', 'The protein encoded by this gene is a lysosomal enzyme that cleaves terminal sialic acid residues from substrates such as glycoproteins and glycolipids. In the lysosome, this enzyme is part of a heterotrimeric complex together with beta-galactosidase and cathepsin A (the latter is also referred to as \'protective protein\'). Mutations in this gene can lead to sialidosis, a lysosomal storage disease that can be type 1 (cherry red spot-myoclonus syndrome or normosomatic type), which is late-onset, or type 2 (the dysmorphic type), which occurs at an earlier age with increased severity. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 13361354, 13484365, 36.22, 8, 'ENSG00000204386', 'Q99519'),
(2137, 'NFATC2', 'This gene is a member of the nuclear factor of activated T cells (NFAT) family. The product of this gene is a DNA-binding protein with a REL-homology region (RHR) and an NFAT-homology region (NHR). This protein is present in the cytosol and only translocates to the nucleus upon T cell receptor (TCR) stimulation, where it becomes a member of the nuclear factors of activated T cells transcription complex. This complex plays a central role in inducing gene transcription during the immune response. Alternate transcriptional splice variants encoding different isoforms have been characterized. [provided by RefSeq, Apr 2012]', '20', 'protein_coding', 39077067, 39087743, 54.63, 7, 'ENSG00000101096', 'Q13469'),
(2138, 'NFATC4', 'This gene encodes a member of the nuclear factor of activated T cells (NFAT) protein family. The encoded protein is part of a DNA-binding transcription complex. This complex consists of at least two components: a preexisting cytosolic component that translocates to the nucleus upon T cell receptor stimulation and an inducible nuclear component. NFAT proteins are activated by the calmodulin-dependent phosphatase, calcineurin. The encoded protein plays a role in the inducible expression of cytokine genes in T cells, especially in the induction of interleukin-2 and interleukin-4. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '14', 'protein_coding', 41566835, 41961120, 37.16, 4, 'ENSG00000285485', 'Q14934'),
(2139, 'NFATC4', 'This gene encodes a member of the nuclear factor of activated T cells (NFAT) protein family. The encoded protein is part of a DNA-binding transcription complex. This complex consists of at least two components: a preexisting cytosolic component that translocates to the nucleus upon T cell receptor stimulation and an inducible nuclear component. NFAT proteins are activated by the calmodulin-dependent phosphatase, calcineurin. The encoded protein plays a role in the inducible expression of cytokine genes in T cells, especially in the induction of interleukin-2 and interleukin-4. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]', '14', 'protein_coding', 41566835, 41961120, 37.16, 4, 'ENSG00000100968', 'Q14934'),
(2140, 'NFKBIB', 'The protein encoded by this gene belongs to the NF-kappa-B inhibitor family, which inhibit NF-kappa-B by complexing with, and trapping it in the cytoplasm. Phosphorylation of serine residues on these proteins by kinases marks them for destruction via the ubiquitination pathway, thereby allowing activation of the NF-kappa-B, which translocates to the nucleus to function as a transcription factor. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Jul 2011]', '19', 'protein_coding', 116249964, 116254075, 47.62, 1, 'ENSG00000282905', 'Q15653'),
(2141, 'NFKBIB', 'The protein encoded by this gene belongs to the NF-kappa-B inhibitor family, which inhibit NF-kappa-B by complexing with, and trapping it in the cytoplasm. Phosphorylation of serine residues on these proteins by kinases marks them for destruction via the ubiquitination pathway, thereby allowing activation of the NF-kappa-B, which translocates to the nucleus to function as a transcription factor. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Jul 2011]', '19', 'protein_coding', 116249964, 116254075, 47.62, 1, 'ENSG00000104825', 'Q15653'),
(2142, 'NFX1', 'MHC class II gene expression is controlled primarily at the transcriptional level by transcription factors that bind to the X and Y boxes, two highly conserved elements in the proximal promoter of MHC class II genes. The protein encoded by this gene is a transcriptional repressor capable of binding to the conserved X box motif of HLA-DRA and other MHC class II genes in vitro. The protein may play a role in regulating the duration of an inflammatory response by limiting the period in which class II MHC molecules are induced by IFN-gamma. Three alternative splice variants, each of which encodes a different isoform, have been identified. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 15911302, 16037381, 39.11, 10, 'ENSG00000086102', 'Q12986'),
(2143, 'NFXL1', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 58671465, 58754477, 38.93, 16, 'ENSG00000170448', 'Q6ZNB6'),
(2144, 'NGB', 'This gene encodes an oxygen-binding protein that is distantly related to members of the globin gene family. It is highly conserved among other vertebrates. It is expressed in the central and peripheral nervous system where it may be involved in increasing oxygen availability and providing protection under hypoxic/ischemic conditions. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 170374671, 170389634, 49.37, 2, 'ENSG00000165553', 'Q9NPG2'),
(2145, 'NGLY1', 'This gene encodes an enzyme that catalyzes hydrolysis of an N(4)-(acetyl-beta-D-glucosaminyl) asparagine residue to N-acetyl-beta-D-glucosaminylamine and a peptide containing an aspartate residue. The encoded enzyme may play a role in the proteasome-mediated degradation of misfolded glycoproteins. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Feb 2009]', '3', 'protein_coding', 560404, 564025, 69.11, 8, 'ENSG00000151092', 'Q96IV0'),
(2146, 'NHLRC2', 'Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 7579491, 7582111, 55.89, 4, 'ENSG00000196865', 'Q8NBF2'),
(2147, 'NINJ1', 'The ninjurin protein is upregulated after nerve injury both in dorsal root ganglion neurons and in Schwann cells (Araki and Milbrandt, 1996 [PubMed 8780658]). It demonstrates properties of a homophilic adhesion molecule and promotes neurite outgrowth from primary cultured dorsal root ganglion neurons.[supplied by OMIM, Aug 2009]', '9', 'protein_coding', 4780023, 4823608, 49.01, 3, 'ENSG00000131669', 'Q92982'),
(2148, 'NINL', 'Predicted to enable calcium ion binding activity. Predicted to be involved in microtubule anchoring at centrosome. Located in centrosome. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 25209, 34495, 40.47, 2, 'ENSG00000101004', 'Q9Y2I6'),
(2149, 'NIP7', 'Enables RNA binding activity. Predicted to be involved in ribosomal large subunit biogenesis. Located in cytosol; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 150933, 165052, 48.19, 1, 'ENSG00000132603', 'Q9Y221'),
(2150, 'NIT2', 'Enables omega-amidase activity. Involved in asparagine metabolic process; glutamine metabolic process; and oxaloacetate metabolic process. Located in centrosome and cytosol. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 8039034, 8049134, 55.65, 5, 'ENSG00000114021', 'Q9NQR4'),
(2151, 'NKAPD1', 'Enables identical protein binding activity. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 75646243, 75647975, 55.57, 2, 'ENSG00000150776', 'Q6ZUT1'),
(2152, 'NKD2', 'This gene encodes a member of a family of proteins that function as negative regulators of Wnt receptor signaling through interaction with Dishevelled family members. The encoded protein participates in the delivery of transforming growth factor alpha-containing vesicles to the cell membrane. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Aug 2012]', '5', 'protein_coding', 116443555, 116574823, 40.55, 15, 'ENSG00000276920', 'Q969F2'),
(2153, 'NKD2', 'This gene encodes a member of a family of proteins that function as negative regulators of Wnt receptor signaling through interaction with Dishevelled family members. The encoded protein participates in the delivery of transforming growth factor alpha-containing vesicles to the cell membrane. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Aug 2012]', '5', 'protein_coding', 116443555, 116574823, 40.55, 15, 'ENSG00000145506', 'Q969F2'),
(2154, 'NKX1-2', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in cell differentiation and regulation of transcription by RNA polymerase II. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 37556919, 37566857, 49.25, 6, 'ENSG00000229544', 'Q9UD57'),
(2155, 'NKX2-1', 'This gene encodes a protein initially identified as a thyroid-specific transcription factor. The encoded protein binds to the thyroglobulin promoter and regulates the expression of thyroid-specific genes but has also been shown to regulate the expression of genes involved in morphogenesis. Mutations and deletions in this gene are associated with benign hereditary chorea, choreoathetosis, congenital hypothyroidism, and neonatal respiratory distress, and may be associated with thyroid cancer. Multiple transcript variants encoding different isoforms have been found for this gene. This gene shares the symbol/alias \'TTF1\' with another gene, transcription termination factor 1, which plays a role in ribosomal gene transcription. [provided by RefSeq, Feb 2014]', '14', 'protein_coding', 129423, 146154, 47.00, 2, 'ENSG00000136352', 'P43699'),
(2156, 'NLGN2', 'This gene encodes a member of a family of neuronal cell surface proteins. Members of this family may act as splice site-specific ligands for beta-neurexins and may be involved in the formation and remodeling of central nervous system synapses. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 53028813, 53052057, 40.05, 1, 'ENSG00000283859', 'Q8NFZ4'),
(2157, 'NLGN2', 'This gene encodes a member of a family of neuronal cell surface proteins. Members of this family may act as splice site-specific ligands for beta-neurexins and may be involved in the formation and remodeling of central nervous system synapses. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 53028813, 53052057, 40.05, 1, 'ENSG00000169992', 'Q8NFZ4'),
(2158, 'NLN', 'This gene encodes a member of the metallopeptidase M3 protein family that cleaves neurotensin at the Pro10-Tyr11 bond, leading to the formation of neurotensin(1-10) and neurotensin(11-13). The encoded protein is likely involved in the termination of the neurotensinergic signal in the central nervous system and in the gastrointestinal tract.[provided by RefSeq, Jun 2010]', '5', 'protein_coding', 56451521, 56479104, 40.36, 11, 'ENSG00000123213', 'Q9BYT8'),
(2159, 'NME2', 'Nucleoside diphosphate kinase (NDK) exists as a hexamer composed of \'A\' (encoded by NME1) and \'B\' (encoded by this gene) isoforms. Multiple alternatively spliced transcript variants have been found for this gene. Read-through transcription from the neighboring upstream gene (NME1) generates naturally-occurring transcripts (NME1-NME2) that encode a fusion protein comprised of sequence sharing identity with each individual gene product. [provided by RefSeq, Nov 2010]', '17', 'protein_coding', 66337333, 66345125, 55.88, 9, 'ENSG00000243678', 'P22392'),
(2160, 'NMNAT2', 'This gene product belongs to the nicotinamide mononucleotide adenylyltransferase (NMNAT) enzyme family, members of which catalyze an essential step in NAD (NADP) biosynthetic pathway. Unlike the other human family member, which is localized to the nucleus, and is ubiquitously expressed; this enzyme is cytoplasmic, and is predominantly expressed in the brain. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 75336329, 75364001, 38.36, 18, 'ENSG00000157064', 'Q9BZQ4'),
(2161, 'NMRK1', 'Nicotinamide adenine dinucleotide (NAD+) is essential for life in all organisms, both as a coenzyme for oxidoreductases and as a source of ADP-ribosyl groups used in various reactions. Nicotinic acid and nicotinamide, collectively known as niacin, are the vitamin precursors of NAD+. Nicotinamide riboside kinases, such as NRK1, function to synthesize NAD+ through nicotinamide mononucleotide using nicotinamide riboside as the precursor (Bieganowski and Brenner, 2004 [PubMed 15137942]).[supplied by OMIM, Mar 2008]', '9', 'protein_coding', 33432485, 33479974, 43.33, 10, 'ENSG00000106733', 'Q9NWW6'),
(2162, 'NMT1', 'Myristate, a rare 14-carbon saturated fatty acid, is cotranslationally attached by an amide linkage to the N-terminal glycine residue of cellular and viral proteins with diverse functions. N-myristoyltransferase (NMT; EC 2.3.1.97) catalyzes the transfer of myristate from CoA to proteins. N-myristoylation appears to be irreversible and is required for full expression of the biologic activities of several N-myristoylated proteins, including the alpha subunit of the signal-transducing guanine nucleotide-binding protein (G protein) GO (GNAO1; MIM 139311) (Duronio et al., 1992 [PubMed 1570339]).[supplied by OMIM, Nov 2008]', '17', 'TR_V_gene', 142469537, 142470013, 53.67, 1, 'ENSG00000136448', 'P30419'),
(2163, 'NMT2', 'This gene encodes one of two N-myristoyltransferase proteins. N-terminal myristoylation is a lipid modification that is involved in regulating the function and localization of signaling proteins. The encoded protein catalyzes the addition of a myristoyl group to the N-terminal glycine residue of many signaling proteins, including the human immunodeficiency virus type 1 (HIV-1) proteins, Gag and Nef. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]', '10', 'protein_coding', 77042474, 77327094, 35.75, 29, 'ENSG00000152465', 'O60551'),
(2164, 'NMUR2', 'This gene encodes a protein from the G-protein coupled receptor 1 family. This protein is a receptor for neuromedin U, which is a neuropeptide that is widely distributed in the gut and central nervous system. This receptor plays an important role in the regulation of food intake and body weight. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 7311324, 7315564, 56.28, 16, 'ENSG00000132911', 'Q9GZQ4'),
(2165, 'NOCT', 'The protein encoded by this gene is highly similar to Nocturnin, a gene identified as a circadian clock regulated gene in Xenopus laevis. This protein and Nocturnin protein share similarity with the C-terminal domain of a yeast transcription factor, carbon catabolite repression 4 (CCR4). The mRNA abundance of a similar gene in mouse has been shown to exhibit circadian rhythmicity, which suggests a role for this protein in clock function or as a circadian clock effector. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 100126785, 100128495, 65.69, 2, 'ENSG00000151014', 'Q9UK39'),
(2166, 'NODAL', 'This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate the mature protein, which regulates early embryonic development. This protein is required for maintenance of human embryonic stem cell pluripotency and may play a role in human placental development. Mutations in this gene are associated with heterotaxy, a condition characterized by random orientation of visceral organs with respect to the left-right axis. [provided by RefSeq, Aug 2016]', '10', 'protein_coding', 57499915, 57518404, 43.76, 6, 'ENSG00000156574', 'Q96S42'),
(2167, 'NOL4', 'Predicted to enable RNA binding activity. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 34150987, 34358010, 47.92, 2, 'ENSG00000101746', 'O94818'),
(2168, 'NOLC1', 'Enables molecular function inhibitor activity; protein heterodimerization activity; and protein-macromolecule adaptor activity. Involved in neural crest cell development; neural crest formation; and regulation of translation. Acts upstream of or within axonogenesis and regulation of synaptic plasticity. Located in fibrillar center. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 7428857, 7432820, 55.83, 2, 'ENSG00000166197', 'Q14978'),
(2169, 'NOMO3', 'This gene encodes a protein originally thought to be related to the collagenase gene family. This gene is one of three highly similar genes in a duplicated region on the short arm of chromosome 16. These three genes encode closely related proteins that may have the same function. The protein encoded by one of these genes has been identified as part of a protein complex that participates in the Nodal signaling pathway during vertebrate development. Mutations in ABCC6, which is located nearby, rather than mutations in this gene are associated with pseudoxanthoma elasticum. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 29657002, 29672372, 45.68, 19, 'ENSG00000278087', 'P69849'),
(2170, 'NOMO3', 'This gene encodes a protein originally thought to be related to the collagenase gene family. This gene is one of three highly similar genes in a duplicated region on the short arm of chromosome 16. These three genes encode closely related proteins that may have the same function. The protein encoded by one of these genes has been identified as part of a protein complex that participates in the Nodal signaling pathway during vertebrate development. Mutations in ABCC6, which is located nearby, rather than mutations in this gene are associated with pseudoxanthoma elasticum. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 29657002, 29672372, 45.68, 19, 'ENSG00000103226', 'P69849'),
(2171, 'NONO', 'This gene encodes an RNA-binding protein which plays various roles in the nucleus, including transcriptional regulation and RNA splicing. A rearrangement between this gene and the transcription factor E3 gene has been observed in papillary renal cell carcinoma. Alternatively spliced transcript variants have been described. Pseudogenes exist on Chromosomes 2 and 16. [provided by RefSeq, Feb 2009]', 'X', 'protein_coding', 120217479, 120223400, 50.29, 1, 'ENSG00000147140', 'Q15233'),
(2172, 'NOP10', 'This gene is a member of the H/ACA snoRNPs (small nucleolar ribonucleoproteins) gene family. snoRNPs are involved in various aspects of rRNA processing and modification and have been classified into two families: C/D and H/ACA. The H/ACA snoRNPs also include the DKC1, NOLA1 and NOLA2 proteins. These four H/ACA snoRNP proteins localize to the dense fibrillar components of nucleoli and to coiled (Cajal) bodies in the nucleus. Both 18S rRNA production and rRNA pseudouridylation are impaired if any one of the four proteins is depleted. The four H/ACA snoRNP proteins are also components of the telomerase complex. This gene encodes a protein related to Saccharomyces cerevisiae Nop10p. [provided by RefSeq, Jul 2008]', '15', 'IG_V_gene', 22322472, 22322969, 57.23, 1, 'ENSG00000291996', 'Q9NPE3'),
(2173, 'NOP10', 'This gene is a member of the H/ACA snoRNPs (small nucleolar ribonucleoproteins) gene family. snoRNPs are involved in various aspects of rRNA processing and modification and have been classified into two families: C/D and H/ACA. The H/ACA snoRNPs also include the DKC1, NOLA1 and NOLA2 proteins. These four H/ACA snoRNP proteins localize to the dense fibrillar components of nucleoli and to coiled (Cajal) bodies in the nucleus. Both 18S rRNA production and rRNA pseudouridylation are impaired if any one of the four proteins is depleted. The four H/ACA snoRNP proteins are also components of the telomerase complex. This gene encodes a protein related to Saccharomyces cerevisiae Nop10p. [provided by RefSeq, Jul 2008]', '15', 'IG_V_gene', 22322472, 22322969, 57.23, 1, 'ENSG00000182117', 'Q9NPE3'),
(2174, 'NOP14', 'This gene encodes a protein that plays a role in pre-18s rRNA processing and small ribosomal subunit assembly. The encoded protein may be involved in the regulation of pancreatic cancer cell proliferation and migration. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]', '4', 'protein_coding', 31325804, 31346346, 52.93, 6, 'ENSG00000087269', 'P78316'),
(2175, 'NOP56', 'Nop56p is a yeast nucleolar protein that is part of a complex with the nucleolar proteins Nop58p and fibrillarin. Nop56p is required for assembly of the 60S ribosomal subunit and is involved in pre-rRNA processing. The protein encoded by this gene is similar in sequence to Nop56p and is also found in the nucleolus. Expansion of a GGCCTG repeat from 3-8 copies to 1500-2500 copies in an intron of this gene results in spinocerebellar ataxia 36. Multiple transcript variants encoding several different isoforms have been found for this gene, but the full-length nature of most of them has not been determined. [provided by RefSeq, Jul 2016]', '20', 'TR_V_gene', 21868839, 21869365, 41.94, 1, 'ENSG00000101361', 'O00567'),
(2176, 'NOSTRIN', 'Nitric oxide (NO) is a potent mediator in biologic processes such as neurotransmission, inflammatory response, and vascular homeostasis. NOSTRIN binds the enzyme responsible for NO production, endothelial NO synthase (ENOS; MIM 163729), and triggers the translocation of ENOS from the plasma membrane to vesicle-like subcellular structures, thereby attenuating ENOS-dependent NO production.[supplied by OMIM, Apr 2004]', '2', 'protein_coding', 42766, 155453, 56.63, 31, 'ENSG00000275326', 'Q8IVI9'),
(2177, 'NOSTRIN', 'Nitric oxide (NO) is a potent mediator in biologic processes such as neurotransmission, inflammatory response, and vascular homeostasis. NOSTRIN binds the enzyme responsible for NO production, endothelial NO synthase (ENOS; MIM 163729), and triggers the translocation of ENOS from the plasma membrane to vesicle-like subcellular structures, thereby attenuating ENOS-dependent NO production.[supplied by OMIM, Apr 2004]', '2', 'protein_coding', 42766, 155453, 56.63, 31, 'ENSG00000163072', 'Q8IVI9'),
(2178, 'NOX4', 'This gene encodes a member of the NOX-family of enzymes that functions as the catalytic subunit the NADPH oxidase complex. The encoded protein is localized to non-phagocytic cells where it acts as an oxygen sensor and catalyzes the reduction of molecular oxygen to various reactive oxygen species (ROS). The ROS generated by this protein have been implicated in numerous biological functions including signal transduction, cell differentiation and tumor cell growth. A pseudogene has been identified on the other arm of chromosome 11. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Jan 2009]', '11', 'miRNA', 46709293, 46709345, 54.72, 1, 'ENSG00000086991', 'Q9NPH5'),
(2179, 'NOXA1', 'This gene encodes a protein which activates NADPH oxidases, enzymes which catalyze a reaction generating reactive oxygen species. The encoded protein contains four N-terminal tetratricopeptide domains and a C-terminal Src homology 3 domain. Interaction between the encoded protein and proteins in the oxidase regulatory complex occur via the tetratricopeptide domains. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2011]', '9', 'protein_coding', 150338317, 150341662, 62.31, 5, 'ENSG00000188747', 'Q86UR1'),
(2180, 'NPBWR2', 'The protein encoded by this gene is an integral membrane protein and G protein-coupled receptor. The encoded protein is similar in sequence to another G protein-coupled receptor (GPR7), and it is structurally similar to opioid and somatostatin receptors. This protein binds neuropeptides B and W. This gene is intronless and is expressed primarily in the frontal cortex of the brain. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 190908, 195943, 54.77, 4, 'ENSG00000125522', 'P48146'),
(2181, 'NPBWR2', 'The protein encoded by this gene is an integral membrane protein and G protein-coupled receptor. The encoded protein is similar in sequence to another G protein-coupled receptor (GPR7), and it is structurally similar to opioid and somatostatin receptors. This protein binds neuropeptides B and W. This gene is intronless and is expressed primarily in the frontal cortex of the brain. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 190908, 195943, 54.77, 4, 'ENSG00000277339', 'P48146');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2182, 'NPR3', 'This gene encodes one of three natriuretic peptide receptors. Natriutetic peptides are small peptides which regulate blood volume and pressure, pulmonary hypertension, and cardiac function as well as some metabolic and growth processes. The product of this gene encodes a natriuretic peptide receptor responsible for clearing circulating and extracellular natriuretic peptides through endocytosis of the receptor. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Feb 2011]', '5', 'TR_J_gene', 22495913, 22495966, 44.44, 1, 'ENSG00000113389', 'P17342'),
(2183, 'NPSR1', 'This gene encodes a member of the vasopressin/oxytocin subfamily of G protein-coupled receptors. The encoded membrane protein acts as a receptor for neuropeptide S and affects a variety of cellular processes through its signaling. Increased expression of this gene in ciliated cells of the respiratory epithelium and in bronchial smooth muscle cells is associated with asthma. Polymorphisms in this gene have also been associated with asthma susceptibility, panic disorders, inflammatory bowel disease, and rheumatoid arthritis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014]', '7', 'snRNA', 32203163, 32203269, 46.73, 1, 'ENSG00000187258', 'Q6W5P4'),
(2184, 'NPW', 'The product of this gene is processed into 23- and 30-amino acid neuropeptides that bind and activate two G-protein coupled receptors in the central nervous system. The neuropeptides have been shown to enhance cortisol secretion from adrenal cells through the adenylate cyclase/protein kinase A signaling cascade. The preproprotein is translated using a non-AUG initiation codon that is inferred from analyses of the mouse ortholog. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 87406246, 87493024, 48.41, 5, 'ENSG00000291946', 'Q8N729'),
(2185, 'NPW', 'The product of this gene is processed into 23- and 30-amino acid neuropeptides that bind and activate two G-protein coupled receptors in the central nervous system. The neuropeptides have been shown to enhance cortisol secretion from adrenal cells through the adenylate cyclase/protein kinase A signaling cascade. The preproprotein is translated using a non-AUG initiation codon that is inferred from analyses of the mouse ortholog. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 87406246, 87493024, 48.41, 5, 'ENSG00000183971', 'Q8N729'),
(2186, 'NPY1R', 'This gene belongs to the G-protein-coupled receptor superfamily. The encoded transmembrane protein mediates the function of neuropeptide Y (NPY), a neurotransmitter, and peptide YY (PYY), a gastrointestinal hormone. The encoded receptor undergoes fast agonist-induced internalization through clathrin-coated pits and is subsequently recycled back to the cell membrane. Activation of Y1 receptors may result in mobilization of intracellular calcium and inhibition of adenylate cyclase activity. [provided by RefSeq, Aug 2013]', '4', 'protein_coding', 5069572, 5070497, 43.95, 1, 'ENSG00000164128', 'P25929'),
(2187, 'NPY2R', 'Predicted to enable calcium channel regulator activity and neuropeptide Y receptor activity. Involved in cardiac left ventricle morphogenesis and outflow tract morphogenesis. Located in cilium. Implicated in Huntington\'s disease; morbid obesity; and obesity. Biomarker of peripheral artery disease and temporal lobe epilepsy. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 149732825, 149732890, 53.03, 1, 'ENSG00000185149', 'P49146'),
(2188, 'NPY4R2', 'Enables pancreatic polypeptide receptor activity and peptide hormone binding activity. Involved in G protein-coupled receptor signaling pathway. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 71614, 108092, 63.46, 1, 'ENSG00000288157', 'P0DQD5'),
(2189, 'NPY4R2', 'Enables pancreatic polypeptide receptor activity and peptide hormone binding activity. Involved in G protein-coupled receptor signaling pathway. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 71614, 108092, 63.46, 1, 'ENSG00000264717', 'P0DQD5'),
(2190, 'NPY5R', 'The protein encoded by this gene is a receptor for neuropeptide Y and peptide YY. The encoded protein appears to be involved in regulating food intake, with defects in this gene being associated with eating disorders. Also, the encoded protein is involved in a pathway that protects neuroblastoma cells from chemotherapy-induced cell death, providing a possible therapeutic target against neuroblastoma. Three transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Nov 2015]', '4', 'transcribed_unitary_pseudogene', 1978399, 2007208, 44.09, 5, 'ENSG00000164129', 'Q15761'),
(2191, 'NR1H4', 'This gene encodes a ligand-activated transcription factor that shares structural features in common with nuclear hormone receptor family members. This protein functions as a receptor for bile acids, and when bound to bile acids, binds to DNA and regulates the expression of genes involved in bile acid synthesis and transport. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Feb 2016]', '12', 'protein_coding', 50481543, 50486440, 63.62, 2, 'ENSG00000012504', 'Q96RI1'),
(2192, 'NRARP', 'Involved in Notch signaling pathway. Acts upstream of or within negative regulation of Notch signaling pathway and positive regulation of canonical Wnt signaling pathway. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 36226209, 36239954, 49.58, 9, 'ENSG00000198435', 'Q7Z6K4'),
(2193, 'NRBF2', 'Involved in autophagy. Located in cytoplasm and phosphatidylinositol 3-kinase complex, class III. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 136497079, 136512346, 37.88, 4, 'ENSG00000148572', 'Q96F24'),
(2194, 'NREP', 'Predicted to be involved in axon regeneration; regulation of neuron differentiation; and regulation of transforming growth factor beta receptor signaling pathway. Predicted to be located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 49289638, 49325215, 43.07, 7, 'ENSG00000134986', 'Q16612'),
(2195, 'NRG3', 'This gene is a member of the neuregulin gene family. This gene family encodes ligands for the transmembrane tyrosine kinase receptors ERBB3 and ERBB4 - members of the epidermal growth factor receptor family. Ligand binding activates intracellular signaling cascades and the induction of cellular responses including proliferation, migration, differentiation, and survival or apoptosis. This gene encodes neuregulin 3 (NRG3). NRG3 has been shown to activate the tyrosine phosphorylation of its cognate receptor, ERBB4, and is thought to influence neuroblast proliferation, migration and differentiation by signalling through ERBB4. NRG3 also promotes mammary differentiation during embryogenesis. Linkage studies have implicated this gene as a susceptibility locus for schizophrenia and schizoaffective disorder. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Additional transcript variants have been described but their biological validity has not been verified.[provided by RefSeq, Sep 2009]', '10', 'protein_coding', 40729128, 40771190, 39.20, 4, 'ENSG00000185737', 'P56975'),
(2196, 'NRON', 'Predicted to act upstream of or within negative regulation of dephosphorylation; regulation of calcineurin-NFAT signaling cascade; and regulation of cytokine production. Predicted to be part of ribonucleoprotein complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 472498, 543835, 41.35, 48, 'ENSG00000253079', ''),
(2197, 'NSD2', 'This gene encodes a protein that contains four domains present in other developmental proteins: a PWWP domain, an HMG box, a SET domain, and a PHD-type zinc finger. It is expressed ubiquitously in early development. Wolf-Hirschhorn syndrome (WHS) is a malformation syndrome associated with a hemizygous deletion of the distal short arm of chromosome 4. This gene maps to the 165 kb WHS critical region and has also been involved in the chromosomal translocation t(4;14)(p16.3;q32.3) in multiple myelomas. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. Some transcript variants are nonsense-mediated mRNA (NMD) decay candidates, hence not represented as reference sequences. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 52061827, 52069219, 59.92, 12, 'ENSG00000109685', 'O96028'),
(2198, 'NSF', 'Enables PDZ domain binding activity and ionotropic glutamate receptor binding activity. Involved in intracellular protein transport; positive regulation of protein catabolic process; and positive regulation of receptor recycling. Located in Golgi apparatus; cytosol; and plasma membrane. Implicated in developmental and epileptic encephalopathy 96. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 122739999, 122793831, 39.98, 6, 'ENSG00000278174', 'P46459'),
(2199, 'NSF', 'Enables PDZ domain binding activity and ionotropic glutamate receptor binding activity. Involved in intracellular protein transport; positive regulation of protein catabolic process; and positive regulation of receptor recycling. Located in Golgi apparatus; cytosol; and plasma membrane. Implicated in developmental and epileptic encephalopathy 96. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 122739999, 122793831, 39.98, 6, 'ENSG00000276262', 'P46459'),
(2200, 'NSF', 'Enables PDZ domain binding activity and ionotropic glutamate receptor binding activity. Involved in intracellular protein transport; positive regulation of protein catabolic process; and positive regulation of receptor recycling. Located in Golgi apparatus; cytosol; and plasma membrane. Implicated in developmental and epileptic encephalopathy 96. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 122739999, 122793831, 39.98, 6, 'ENSG00000073969', 'P46459'),
(2201, 'NSUN2', 'This gene encodes a methyltransferase that catalyzes the methylation of cytosine to 5-methylcytosine (m5C) at position 34 of intron-containing tRNA(Leu)(CAA) precursors. This modification is necessary to stabilize the anticodon-codon pairing and correctly translate the mRNA. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Mar 2011]', '5', 'protein_coding', 210290150, 210315174, 34.55, 4, 'ENSG00000037474', 'Q08J23'),
(2202, 'NT5C1B-RDH14', 'This locus represents naturally occurring read-through transcription between the neighboring NT5C1B (5\'-nucleotidase, cytosolic IB) and RDH14 (retinol dehydrogenase 14) genes on chromosome 2. Alternative splicing results in multiple transcript variants, one of which encodes a fusion protein that shares sequence identity with the products of each individual gene. [provided by RefSeq, Nov 2010]', '2', 'protein_coding', 10762723, 10838553, 42.11, 7, 'ENSG00000250741', 'C9J2C7'),
(2203, 'NT5DC1', 'While the exact function of the protein encoded by this gene is not known, it belongs to the 5\'(3\')-deoxyribonucleotidase family. [provided by RefSeq, May 2010]', '6', 'protein_coding', 134986508, 135147963, 36.82, 7, 'ENSG00000178425', 'Q5TFE4'),
(2204, 'NTF3', 'The protein encoded by this gene is a member of the neurotrophin family, that controls survival and differentiation of mammalian neurons. This protein is closely related to both nerve growth factor and brain-derived neurotrophic factor. It may be involved in the maintenance of the adult nervous system, and may affect development of neurons in the embryo when it is expressed in human placenta. NTF3-deficient mice generated by gene targeting display severe movement defects of the limbs. The mature peptide of this protein is identical in all mammals examined including human, pig, rat and mouse. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 93818232, 93850618, 46.26, 3, 'ENSG00000185652', 'P20783'),
(2205, 'NTNG2', 'Predicted to be involved in axonogenesis; regulation of cell projection organization; and regulation of neuron migration. Predicted to be located in plasma membrane. Predicted to be active in Schaffer collateral - CA1 synapse; glutamatergic synapse; and presynaptic active zone membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 64503712, 64533690, 39.55, 7, 'ENSG00000196358', 'Q96CW9'),
(2206, 'NUDT13', 'Predicted to enable NADH pyrophosphatase activity. Predicted to be involved in NADH metabolic process and NADP catabolic process. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 57321502, 57321591, 50.00, 1, 'ENSG00000166321', 'Q86X67'),
(2207, 'NUDT22', 'Enables UDP-sugar diphosphatase activity and metal ion binding activity. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'miRNA', 168582943, 168583018, 34.21, 1, 'ENSG00000149761', 'Q9BRQ3'),
(2208, 'NUDT4', 'The protein encoded by this gene regulates the turnover of diphosphoinositol polyphosphates. The turnover of these high-energy diphosphoinositol polyphosphates represents a molecular switching activity with important regulatory consequences. Molecular switching by diphosphoinositol polyphosphates may contribute to regulating intracellular trafficking. Several alternatively spliced transcript variants have been described, but the full-length nature of some variants has not been determined. Isoforms DIPP2alpha and DIPP2beta are distinguishable from each other solely by DIPP2beta possessing one additional amino acid due to intron boundary skidding in alternate splicing. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 61662929, 61766766, 35.15, 3, 'ENSG00000173598', 'Q9NZJ9'),
(2209, 'NUFIP1', 'This gene encodes a nuclear RNA binding protein that contains a C2H2 zinc finger motif and a nuclear localization signal. This protein is associated with the nuclear matrix in perichromatin fibrils and, in neurons, localizes to the cytoplasm in association with endoplasmic reticulum ribosomes. This protein interacts with the fragile X mental retardation protein (FMRP), the tumor suppressor protein BRCA1, upregulates RNA polymerase II transcription, and is involved in box C/D snoRNP biogenesis. A pseudogene of this gene resides on chromosome 6q12. [provided by RefSeq, Feb 2012]', '13', 'protein_coding', 32238679, 32370321, 43.06, 2, 'ENSG00000083635', 'Q9UHK0'),
(2210, 'NUP107', 'This gene encodes a member of the nucleoporin family. The protein is localized to the nuclear rim and is an essential component of the nuclear pore complex (NPC). All molecules entering or leaving the nucleus either diffuse through or are actively transported by the NPC. Alternate transcriptional splice variants of this gene have been observed but have not been thoroughly characterized. [provided by RefSeq, Jul 2008]', '12', 'miRNA', 130666735, 130666803, 63.77, 1, 'ENSG00000111581', 'P57740'),
(2211, 'NUP188', 'The nuclear pore complex (NPC) is found on the nuclear envelope and forms a gateway that regulates the flow of proteins and RNAs between the cytoplasm and nucleoplasm. The NPC is comprised of approximately 30 distinct proteins collectively known as nucleoporins. Nucleoporins are pore-complex-specific glycoproteins which often have cytoplasmically oriented O-linked N-acetylglucosamine residues and numerous repeats of the pentapeptide sequence XFXFG. However, the nucleoporin protein encoded by this gene does not contain the typical FG repeat sequences found in most vertebrate nucleoporins. This nucleoporin is thought to form part of the scaffold for the central channel of the nuclear pore. [provided by RefSeq, Jan 2013]', '9', 'transcribed_unprocessed_pseudogene', 23659869, 23717295, 49.83, 1, 'ENSG00000095319', 'Q5SRE5'),
(2212, 'NUP58', 'This gene encodes a member of the nucleoporin family that shares 87% sequence identity with rat nucleoporin p58. The protein is localized to the nuclear rim and is a component of the nuclear pore complex (NPC). All molecules entering or leaving the nucleus either diffuse through or are actively transported by the NPC. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '13', 'unprocessed_pseudogene', 29229227, 29230157, 48.55, 1, 'ENSG00000139496', 'Q9BVL2'),
(2213, 'NUP85', 'This gene encodes a protein component of the Nup107-160 subunit of the nuclear pore complex. Nuclear pore complexes are embedded in the nuclear envelope and promote bidirectional transport of macromolecules between the cytoplasm and nucleus. The encoded protein can also bind to the C-terminus of chemokine (C-C motif) receptor 2 (CCR2) and promote chemotaxis of monocytes, thereby participating in the inflammatory response. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]', '17', 'protein_coding', 134455, 144052, 49.15, 1, 'ENSG00000125450', 'Q9BW27'),
(2214, 'NUP93', 'The nuclear pore complex is a massive structure that extends across the nuclear envelope, forming a gateway that regulates the flow of macromolecules between the nucleus and the cytoplasm. Nucleoporins are the main components of the nuclear pore complex in eukaryotic cells. This gene encodes a nucleoporin protein that localizes both to the basket of the pore and to the nuclear entry of the central gated channel of the pore. The encoded protein is a target of caspase cysteine proteases that play a central role in programmed cell death by apoptosis. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2016]', '16', 'protein_coding', 14578, 64819, 47.83, 4, 'ENSG00000102900', 'Q8N1F7'),
(2215, 'NUS1', 'This gene encodes a type I single transmembrane domain receptor, which is a subunit of cis-prenyltransferase, and serves as a specific receptor for the neural and cardiovascular regulator Nogo-B. The encoded protein is essential for dolichol synthesis and protein glycosylation. This gene is highly expressed in non-small cell lung carcinomas as well as estrogen receptor-alpha positive breast cancer cells where it promotes epithelial mesenchymal transition. This gene is associated with the poor prognosis of human hepatocellular carcinoma patients. Naturally occurring mutations in this gene cause a congenital disorder of glycosylation and are associated with epilepsy. A knockout of the orthologous gene in mice causes embryonic lethality before day 6.5. Pseudogenes of this gene have been defined on chromosomes 13 and X. [provided by RefSeq, May 2017]', '6', 'protein_coding', 106863166, 107013634, 40.07, 12, 'ENSG00000153989', 'Q96E22'),
(2216, 'NWD2', 'Predicted to be active in synapse. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 55641062, 55653161, 61.17, 9, 'ENSG00000174145', 'Q9ULI1'),
(2217, 'NXNL2', 'Predicted to be involved in sensory perception of smell and visual perception. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 40010999, 40847158, 40.93, 3, 'ENSG00000291520', 'Q5VZ03'),
(2218, 'NXNL2', 'Predicted to be involved in sensory perception of smell and visual perception. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 40010999, 40847158, 40.93, 3, 'ENSG00000130045', 'Q5VZ03'),
(2219, 'OACYLP', 'This gene is ancient and exists in archaea, bacteria, fungi, worms, flies, and mammals. The gene has been inactivated by mutation and is nonfunctional in humans and chimpanzees. [provided by RefSeq, Oct 2008]', '18', 'protein_coding', 159865080, 159973012, 43.93, 3, 'ENSG00000224367', ''),
(2220, 'OCM', 'This gene is similar to the oncomodulin gene, a high-affinity calcium ion-binding protein that belongs to the superfamily of calmodulin proteins, also known as the EF-hand proteins. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 248562, 256056, 57.76, 17, 'ENSG00000122543', 'P0CE72'),
(2221, 'ODAPH', 'Dental enamel forms the outer cap of teeth and is the hardest substance found in vertebrates. This gene is thought to encode an extracellular matrix acidic phosphoprotein that has a function in enamel mineralization during amelogenesis. Mutations in this gene are associated with recessive hypomineralized amelogenesis imperfecta. [provided by RefSeq, Oct 2012]', '4', 'IG_V_gene', 22697789, 22698407, 60.26, 1, 'ENSG00000174792', 'Q17RF5'),
(2222, 'OGA', 'The dynamic modification of cytoplasmic and nuclear proteins by O-linked N-acetylglucosamine (O-GlcNAc) addition and removal on serine and threonine residues is catalyzed by OGT (MIM 300255), which adds O-GlcNAc, and MGEA5, a glycosidase that removes O-GlcNAc modifications (Gao et al., 2001 [PubMed 11148210]).[supplied by OMIM, Mar 2008]', '10', 'protein_coding', 49782164, 49813953, 39.53, 7, 'ENSG00000198408', 'O60502'),
(2223, 'OGFOD1', 'Enables peptidyl-proline 3-dioxygenase activity. Involved in several processes, including peptidyl-proline hydroxylation; regulation of translational termination; and stress granule assembly. Located in cytoplasmic stress granule; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 131289459, 131359022, 51.73, 3, 'ENSG00000087263', 'Q8N543'),
(2224, 'OGN', 'This gene encodes a member of the small leucine-rich proteoglycan (SLRP) family of proteins. The encoded protein induces ectopic bone formation in conjunction with transforming growth factor beta and may regulate osteoblast differentiation. High expression of the encoded protein may be associated with elevated heart left ventricular mass. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '9', 'protein_coding', 50861568, 50880567, 48.07, 23, 'ENSG00000106809', 'P20774'),
(2225, 'OGN', 'This gene encodes a member of the small leucine-rich proteoglycan (SLRP) family of proteins. The encoded protein induces ectopic bone formation in conjunction with transforming growth factor beta and may regulate osteoblast differentiation. High expression of the encoded protein may be associated with elevated heart left ventricular mass. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '9', 'protein_coding', 50861568, 50880567, 48.07, 23, 'ENSG00000291350', 'P20774'),
(2226, 'OLFM4', 'This gene was originally cloned from human myeloblasts and found to be selectively expressed in inflammed colonic epithelium. This gene encodes a member of the olfactomedin family. The encoded protein is an antiapoptotic factor that promotes tumor growth and is an extracellular matrix glycoprotein that facilitates cell adhesion. [provided by RefSeq, Mar 2011]', '13', 'protein_coding', 203723, 274455, 55.21, 7, 'ENSG00000102837', 'Q6UX06'),
(2227, 'OLFML2A', 'Predicted to enable extracellular matrix binding activity and identical protein binding activity. Predicted to be involved in signal transduction. Predicted to act upstream of or within extracellular matrix organization. Predicted to be located in extracellular matrix and extracellular region. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 134196333, 134201789, 36.25, 1, 'ENSG00000185585', 'Q68BL7'),
(2228, 'OLFML3', 'This gene encodes a member of the olfactomedin-like gene family which also includes genes encoding noelin, tiarin, myocilin, amassin, optimedin, photomedin, and latrophilin. The encoded protein is a secreted extracellular matrix glycoprotein with a C-terminal olfactomedin domain that facilitates protein-protein interactions, cell adhesion, and intercellular interactions. It serves as both a scaffold protein that recruits bone morphogenetic protein 1 to its substrate chordin, and as a vascular tissue remodeler with pro-angiogenic properties. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2017]', '1', 'miRNA', 44408120, 44408203, 38.10, 1, 'ENSG00000116774', 'Q9NRN5'),
(2229, 'OLIG1', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and E-box binding activity. Predicted to be involved in axon development; positive regulation of transcription by RNA polymerase II; and sensory organ development. Predicted to act upstream of or within neuron fate commitment. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 20314238, 20320080, 57.69, 3, 'ENSG00000184221', 'Q8TAK6'),
(2230, 'OLIG3', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in axon development; positive regulation of transcription by RNA polymerase II; and sensory organ development. Predicted to act upstream of or within negative regulation of transcription by RNA polymerase II; spinal cord motor neuron cell fate specification; and spinal cord motor neuron migration. Predicted to be located in chromatin. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 68842197, 68971570, 41.46, 13, 'ENSG00000177468', 'Q7RTU3'),
(2231, 'OLR1', 'This gene encodes a low density lipoprotein receptor that belongs to the C-type lectin superfamily. This gene is regulated through the cyclic AMP signaling pathway. The encoded protein binds, internalizes and degrades oxidized low-density lipoprotein. This protein may be involved in the regulation of Fas-induced apoptosis. This protein may play a role as a scavenger receptor. Mutations of this gene have been associated with atherosclerosis, risk of myocardial infarction, and may modify the risk of Alzheimer\'s disease. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Feb 2010]', '12', 'protein_coding', 76358, 102118, 49.60, 4, 'ENSG00000173391', 'P78380'),
(2232, 'OMA1', 'Enables metalloendopeptidase activity. Involved in several processes, including HRI-mediated signaling; proteolysis; and regulation of mitochondrion organization. Located in mitochondrial inner membrane. Is active in mitochondrial intermembrane space. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 10285666, 10292947, 53.43, 15, 'ENSG00000162600', 'Q96E52'),
(2233, 'OPALIN', 'Predicted to be involved in regulation of oligodendrocyte differentiation. Located in Golgi apparatus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 95578202, 95644706, 41.42, 8, 'ENSG00000197430', 'Q96PE5'),
(2234, 'OPRPN', 'This gene encodes a member of the proline-rich protein family. The encoded protein has multiple proposed functions, including roles in pain suppression, penile erection, and protection of the eye surface. The QRFSR pentapeptide, known as opiorphin, is derived from the N-terminal of this protein. Opiorphin inhibits the enkephalin-inactivating peptidases neprilysin and aminopeptidase N, and this activity is thought to reduce sensitivity to painful stimuli by effecting enkephalin-related activation of opioid-dependent pathways. Opiorphin may also act as an anti-depressant. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]', '4', 'protein_coding', 23454, 30927, 51.32, 10, 'ENSG00000171199', 'Q99935'),
(2235, 'OPTN', 'This gene encodes the coiled-coil containing protein optineurin. Optineurin may play a role in normal-tension glaucoma and adult-onset primary open angle glaucoma. Optineurin interacts with adenovirus E3-14.7K protein and may utilize tumor necrosis factor-alpha or Fas-ligand pathways to mediate apoptosis, inflammation or vasoconstriction. Optineurin may also function in cellular morphogenesis and membrane trafficking, vesicle trafficking, and transcription activation through its interactions with the RAB8, huntingtin, and transcription factor IIIA proteins. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 4630919, 4642294, 54.64, 6, 'ENSG00000123240', 'Q96CV9'),
(2236, 'OR10AK1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 91059, 247523, 39.83, 4, 'ENSG00000254807', ''),
(2237, 'OR10D1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 112422887, 112481017, 39.46, 22, 'ENSG00000196403', ''),
(2238, 'OR10D5P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000213182', ''),
(2239, 'OR10G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 170864875, 170870208, 39.31, 6, 'ENSG00000273586', ''),
(2240, 'OR10G6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 213475, 217824, 60.60, 3, 'ENSG00000198674', 'Q8NH81'),
(2241, 'OR10J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'unprocessed_pseudogene', 58348989, 58349630, 45.33, 1, 'ENSG00000291903', 'P30954'),
(2242, 'OR10J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'unprocessed_pseudogene', 58348989, 58349630, 45.33, 1, 'ENSG00000196184', 'P30954'),
(2243, 'OR10J7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 35904288, 35938779, 41.69, 5, 'ENSG00000291898', ''),
(2244, 'OR10J7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 35904288, 35938779, 41.69, 5, 'ENSG00000248442', ''),
(2245, 'OR10V1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 101387633, 101398618, 35.68, 1, 'ENSG00000172289', 'Q8NGI7'),
(2246, 'OR10V2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'lncRNA', 30193892, 30204310, 56.85, 6, 'ENSG00000255538', ''),
(2247, 'OR10W1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 54566117, 54793315, 43.24, 15, 'ENSG00000172772', 'Q8NGF6'),
(2248, 'OR11H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 136401922, 136410614, 56.53, 11, 'ENSG00000130538', 'Q8NG94'),
(2249, 'OR11H5P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 61409685, 61415849, 44.43, 1, 'ENSG00000291817', ''),
(2250, 'OR11H5P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 61409685, 61415849, 44.43, 1, 'ENSG00000258625', ''),
(2251, 'OR11H7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '14', 'unprocessed_pseudogene', 54692450, 54693359, 41.87, 1, 'ENSG00000258806', 'Q8NGC8'),
(2252, 'OR11H7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '14', 'unprocessed_pseudogene', 54692450, 54693359, 41.87, 1, 'ENSG00000291780', 'Q8NGC8'),
(2253, 'OR1J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '9', 'miRNA', 53685009, 53685091, 42.17, 1, 'ENSG00000136834', 'Q8NGS3'),
(2254, 'OR2A1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 1890490, 1952819, 45.52, 17, 'ENSG00000221970', 'Q8NGT9'),
(2255, 'OR2A1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 1890490, 1952819, 45.52, 17, 'ENSG00000285105', 'Q8NGT9'),
(2256, 'OR2A2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 206783234, 206783308, 54.67, 1, 'ENSG00000285071', 'Q6IF42'),
(2257, 'OR2A2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 206783234, 206783308, 54.67, 1, 'ENSG00000221989', 'Q6IF42'),
(2258, 'OR2B8P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 143716340, 143722458, 64.81, 8, 'ENSG00000182477', 'P59922'),
(2259, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000213911', ''),
(2260, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000224392', ''),
(2261, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000225150', ''),
(2262, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000231320', ''),
(2263, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000224990', ''),
(2264, 'OR2G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 3846799, 3876123, 52.52, 2, 'ENSG00000230874', ''),
(2265, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000204688', 'Q9GZK4');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2266, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000229125', 'Q9GZK4'),
(2267, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000206471', 'Q9GZK4'),
(2268, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000206516', 'Q9GZK4'),
(2269, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000235132', 'Q9GZK4'),
(2270, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000224395', 'Q9GZK4'),
(2271, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000232984', 'Q9GZK4'),
(2272, 'OR2H1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100867387, 100873454, 56.08, 9, 'ENSG00000229408', 'Q9GZK4'),
(2273, 'OR2R1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 159075, 171857, 46.23, 9, 'ENSG00000285299', ''),
(2274, 'OR2R1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 159075, 171857, 46.23, 9, 'ENSG00000236853', ''),
(2275, 'OR2S2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jul 2015]', '9', 'protein_coding', 115931149, 116060891, 36.89, 1, 'ENSG00000278889', 'Q9NQN1'),
(2276, 'OR2T5', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 133034860, 133034915, 48.21, 1, 'ENSG00000273827', 'Q6IEZ7'),
(2277, 'OR2T5', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 133034860, 133034915, 48.21, 1, 'ENSG00000275102', 'Q6IEZ7'),
(2278, 'OR2T5', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 133034860, 133034915, 48.21, 1, 'ENSG00000292007', 'Q6IEZ7'),
(2279, 'OR2T5', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 133034860, 133034915, 48.21, 1, 'ENSG00000203661', 'Q6IEZ7'),
(2280, 'OR2T6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'snoRNA', 115581315, 115581442, 42.97, 1, 'ENSG00000278689', 'Q8NHC8'),
(2281, 'OR2T6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'snoRNA', 115581315, 115581442, 42.97, 1, 'ENSG00000278659', 'Q8NHC8'),
(2282, 'OR2T6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'snoRNA', 115581315, 115581442, 42.97, 1, 'ENSG00000292016', 'Q8NHC8'),
(2283, 'OR2T6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'snoRNA', 115581315, 115581442, 42.97, 1, 'ENSG00000198104', 'Q8NHC8'),
(2284, 'OR2T7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'IG_C_pseudogene', 189563, 190296, 65.80, 1, 'ENSG00000281395', 'P0C7T2'),
(2285, 'OR2T7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'IG_C_pseudogene', 189563, 190296, 65.80, 1, 'ENSG00000281489', 'P0C7T2'),
(2286, 'OR2T7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'IG_C_pseudogene', 189563, 190296, 65.80, 1, 'ENSG00000292023', 'P0C7T2'),
(2287, 'OR2T7', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '1', 'IG_C_pseudogene', 189563, 190296, 65.80, 1, 'ENSG00000227152', 'P0C7T2'),
(2288, 'OR2W2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 100435282, 100436497, 67.27, 5, 'ENSG00000217315', ''),
(2289, 'OR2W4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 44924709, 44947773, 49.30, 4, 'ENSG00000216629', ''),
(2290, 'OR3B1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', 'X', 'miRNA', 101040436, 101040535, 35.00, 1, 'ENSG00000237619', ''),
(2291, 'OR4A17P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 232406844, 232410714, 62.98, 1, 'ENSG00000255196', ''),
(2292, 'OR4A3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 117020211, 117020319, 58.72, 1, 'ENSG00000254464', ''),
(2293, 'OR4A7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 41088451, 41096195, 53.14, 1, 'ENSG00000255349', ''),
(2294, 'OR4A9P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 109451, 155864, 44.42, 5, 'ENSG00000254891', ''),
(2295, 'OR4D7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 20891385, 20892348, 49.38, 1, 'ENSG00000254770', ''),
(2296, 'OR4F1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 96343221, 96359365, 41.61, 6, 'ENSG00000220212', ''),
(2297, 'OR4K1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 1612337, 1677908, 53.10, 9, 'ENSG00000291807', 'Q8NGD4'),
(2298, 'OR4K1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 1612337, 1677908, 53.10, 9, 'ENSG00000155249', 'Q8NGD4'),
(2299, 'OR4K12P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '21', 'unprocessed_pseudogene', 93103, 94048, 45.03, 1, 'ENSG00000218549', ''),
(2300, 'OR4K2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 102596958, 102679958, 37.71, 29, 'ENSG00000291804', 'Q8NGD2'),
(2301, 'OR4K2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 102596958, 102679958, 37.71, 29, 'ENSG00000165762', 'Q8NGD2'),
(2302, 'OR4K6P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 63133247, 63155031, 42.23, 2, 'ENSG00000291799', ''),
(2303, 'OR4K6P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 63133247, 63155031, 42.23, 2, 'ENSG00000228304', ''),
(2304, 'OR4K7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 28367284, 28373091, 59.64, 3, 'ENSG00000265340', ''),
(2305, 'OR4M2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '15', 'protein_coding', 88465778, 88555007, 41.39, 9, 'ENSG00000288194', 'Q8NGB6'),
(2306, 'OR4M2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '15', 'protein_coding', 88465778, 88555007, 41.39, 9, 'ENSG00000274102', 'Q8NGB6'),
(2307, 'OR4N1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'miRNA', 123381440, 123381512, 53.42, 1, 'ENSG00000258564', ''),
(2308, 'OR4N4C', 'Predicted to enable olfactory receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway and detection of chemical stimulus involved in sensory perception of smell. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 53480626, 53487134, 44.98, 5, 'ENSG00000288193', 'A0A096LPK9'),
(2309, 'OR4N4C', 'Predicted to enable olfactory receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway and detection of chemical stimulus involved in sensory perception of smell. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 53480626, 53487134, 44.98, 5, 'ENSG00000279408', 'A0A096LPK9'),
(2310, 'OR4V1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 31281594, 31292534, 51.26, 7, 'ENSG00000254841', ''),
(2311, 'OR51A10P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 120855651, 120894896, 51.61, 14, 'ENSG00000230484', ''),
(2312, 'OR51A3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 10111693, 10115372, 62.34, 2, 'ENSG00000227023', ''),
(2313, 'OR51B3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 108719946, 108736563, 41.07, 2, 'ENSG00000223735', ''),
(2314, 'OR51F4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 10106362, 10112012, 58.50, 8, 'ENSG00000273051', ''),
(2315, 'OR51J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'miRNA', 10230413, 10230485, 75.34, 1, 'ENSG00000184321', 'Q9H342'),
(2316, 'OR52A4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. Although originally considered to be a functional olfactory receptor, this family member is now considered to be pseudogene due to the presence of a C-terminal frameshift compared to other family members; this is also consistent with the Classifier for Olfactory Receptor Pseudogenes (CORP), as described in PMID:16939646. [provided by RefSeq, Jun 2011]', '11', 'protein_coding', 58579063, 58621550, 36.50, 1, 'ENSG00000291915', 'A6NMU1'),
(2317, 'OR52A4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. Although originally considered to be a functional olfactory receptor, this family member is now considered to be pseudogene due to the presence of a C-terminal frameshift compared to other family members; this is also consistent with the Classifier for Olfactory Receptor Pseudogenes (CORP), as described in PMID:16939646. [provided by RefSeq, Jun 2011]', '11', 'protein_coding', 58579063, 58621550, 36.50, 1, 'ENSG00000205494', 'A6NMU1'),
(2318, 'OR52E1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 10115014, 10119918, 59.23, 15, 'ENSG00000273085', 'Q8NGJ3'),
(2319, 'OR52E3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 73695785, 73828316, 38.58, 5, 'ENSG00000291911', ''),
(2320, 'OR52E3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 73695785, 73828316, 38.58, 5, 'ENSG00000226157', ''),
(2321, 'OR52E6', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 143723933, 143738234, 64.47, 3, 'ENSG00000205409', 'Q96RD3'),
(2322, 'OR52H2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 63911230, 63917164, 61.85, 3, 'ENSG00000248553', ''),
(2323, 'OR52J3', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 13726660, 13764556, 39.02, 10, 'ENSG00000291908', 'Q8NH60'),
(2324, 'OR52J3', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 13726660, 13764556, 39.02, 10, 'ENSG00000205495', 'Q8NH60'),
(2325, 'OR52L1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 31633787, 31633858, 52.78, 1, 'ENSG00000183313', 'Q8NGH7'),
(2326, 'OR52Q1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 54153446, 54173986, 43.37, 6, 'ENSG00000277514', ''),
(2327, 'OR52T1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 43558968, 43588237, 49.23, 14, 'ENSG00000233646', ''),
(2328, 'OR52Y1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 17613678, 17682295, 52.51, 10, 'ENSG00000231070', ''),
(2329, 'OR56A7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 31661228, 31666283, 53.98, 8, 'ENSG00000255481', ''),
(2330, 'OR56B1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 50095883, 50159256, 37.41, 1, 'ENSG00000181023', 'Q8NGI3'),
(2331, 'OR56B3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 133348218, 133351426, 50.45, 9, 'ENSG00000180913', ''),
(2332, 'OR5AK1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 41431318, 41440717, 51.01, 16, 'ENSG00000255083', ''),
(2333, 'OR5AL2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 25301524, 25365390, 37.64, 19, 'ENSG00000291848', ''),
(2334, 'OR5AL2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 25301524, 25365390, 37.64, 19, 'ENSG00000255466', ''),
(2335, 'OR5AW1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 29312933, 29319679, 60.13, 4, 'ENSG00000229082', ''),
(2336, 'OR5B10P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 35909490, 35937153, 44.44, 5, 'ENSG00000255096', ''),
(2337, 'OR5B15P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'TR_V_gene', 21797287, 21797886, 45.83, 1, 'ENSG00000273408', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2338, 'OR5B17', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 110478, 247523, 39.53, 3, 'ENSG00000197786', 'Q8NGF7'),
(2339, 'OR5B19P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 28688558, 28735727, 48.48, 14, 'ENSG00000255431', ''),
(2340, 'OR5BP1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 60387362, 60387445, 51.19, 1, 'ENSG00000273255', ''),
(2341, 'OR5BR1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 34306, 40159, 56.49, 2, 'ENSG00000255386', ''),
(2342, 'OR5D13', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jun 2015]', '11', 'protein_coding', 53196792, 53214197, 44.50, 12, 'ENSG00000279761', 'Q8NGL4'),
(2343, 'OR5D14', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 116324448, 116380029, 34.42, 3, 'ENSG00000186113', 'Q8NGL3'),
(2344, 'OR5D17P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 35957108, 35958154, 52.82, 1, 'ENSG00000181837', ''),
(2345, 'OR5D2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 77265483, 77271206, 57.20, 1, 'ENSG00000254457', ''),
(2346, 'OR5F1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 20999509, 21031640, 36.72, 2, 'ENSG00000149133', 'O95221'),
(2347, 'OR5G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 74409, 79783, 47.85, 1, 'ENSG00000281313', ''),
(2348, 'OR5G1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 74409, 79783, 47.85, 1, 'ENSG00000181296', ''),
(2349, 'OR5G3', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 203816, 210831, 48.08, 1, 'ENSG00000281107', 'P0C626'),
(2350, 'OR5G3', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 203816, 210831, 48.08, 1, 'ENSG00000241356', 'P0C626'),
(2351, 'OR5I1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 58183029, 58213562, 47.52, 14, 'ENSG00000167825', 'Q13606'),
(2352, 'OR5J7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 110860582, 110860647, 27.27, 1, 'ENSG00000255217', ''),
(2353, 'OR5L1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jun 2015]', '11', 'protein_coding', 154626945, 154753120, 37.80, 8, 'ENSG00000279395', 'Q8NGL2'),
(2354, 'OR5M2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'IG_V_gene', 106373663, 106374145, 55.69, 1, 'ENSG00000291867', ''),
(2355, 'OR5M2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'IG_V_gene', 106373663, 106374145, 55.69, 1, 'ENSG00000254752', ''),
(2356, 'OR5M4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 21650901, 21704780, 42.44, 10, 'ENSG00000291850', ''),
(2357, 'OR5M4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 21650901, 21704780, 42.44, 10, 'ENSG00000230416', ''),
(2358, 'OR5M6P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1266280, 1309328, 50.50, 4, 'ENSG00000291856', ''),
(2359, 'OR5M6P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1266280, 1309328, 50.50, 4, 'ENSG00000254603', ''),
(2360, 'OR5M8', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 19979961, 20012267, 39.88, 3, 'ENSG00000291862', 'Q8NGP6'),
(2361, 'OR5M8', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 19979961, 20012267, 39.88, 3, 'ENSG00000181371', 'Q8NGP6'),
(2362, 'OR5M9', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 108084207, 108091549, 37.71, 7, 'ENSG00000291852', 'Q8NGP3'),
(2363, 'OR5M9', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 108084207, 108091549, 37.71, 7, 'ENSG00000150269', 'Q8NGP3'),
(2364, 'OR5T1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 624344, 659411, 51.23, 4, 'ENSG00000262784', 'Q8NG75'),
(2365, 'OR5T1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 624344, 659411, 51.23, 4, 'ENSG00000291860', 'Q8NG75'),
(2366, 'OR5T1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 624344, 659411, 51.23, 4, 'ENSG00000181698', 'Q8NG75'),
(2367, 'OR5W1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 54878113, 54878193, 40.74, 1, 'ENSG00000227806', ''),
(2368, 'OR6L1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 26216169, 26218270, 46.10, 1, 'ENSG00000233685', ''),
(2369, 'OR6L2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 25754699, 25781199, 39.49, 3, 'ENSG00000273327', ''),
(2370, 'OR7E105P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 77862830, 77952785, 37.79, 6, 'ENSG00000258423', ''),
(2371, 'OR7E10P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 29536933, 29940033, 38.04, 8, 'ENSG00000285315', ''),
(2372, 'OR7E10P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 29536933, 29940033, 38.04, 8, 'ENSG00000254724', ''),
(2373, 'OR7E110P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 165233, 169508, 55.29, 3, 'ENSG00000235623', ''),
(2374, 'OR7E121P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 55009055, 55030977, 43.13, 9, 'ENSG00000244222', ''),
(2375, 'OR7E140P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '12', 'unprocessed_pseudogene', 58343737, 58344664, 42.46, 1, 'ENSG00000238152', ''),
(2376, 'OR7E149P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 45080437, 45088969, 62.24, 2, 'ENSG00000177586', ''),
(2377, 'OR7E161P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 45388680, 45612165, 36.55, 5, 'ENSG00000285469', ''),
(2378, 'OR7E161P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 45388680, 45612165, 36.55, 5, 'ENSG00000206014', ''),
(2379, 'OR7E37P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Mar 2011]', '13', 'protein_coding', 48488963, 48533256, 40.27, 7, 'ENSG00000215480', ''),
(2380, 'OR7E5P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This family member is believed to be a pseudogene. [provided by RefSeq, Jun 2009]', '11', 'protein_coding', 164735, 169332, 55.98, 7, 'ENSG00000214880', ''),
(2381, 'OR7E89P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 35745600, 35747519, 49.79, 3, 'ENSG00000243792', ''),
(2382, 'OR8A1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 19756703, 19783593, 57.77, 8, 'ENSG00000196119', 'Q8NGG7'),
(2383, 'OR8A2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 413625, 459002, 42.52, 3, 'ENSG00000255510', ''),
(2384, 'OR8B5P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 208160740, 208163589, 43.33, 1, 'ENSG00000255030', ''),
(2385, 'OR8G7P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1140891, 1164186, 52.68, 2, 'ENSG00000276316', ''),
(2386, 'OR8I1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1615059, 1643081, 52.08, 5, 'ENSG00000280712', ''),
(2387, 'OR8I1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1615059, 1643081, 52.08, 5, 'ENSG00000291842', ''),
(2388, 'OR8I1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 1615059, 1643081, 52.08, 5, 'ENSG00000255461', ''),
(2389, 'OR8I4P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 495695, 512489, 47.10, 2, 'ENSG00000254524', ''),
(2390, 'OR8J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57184216, 57199537, 55.06, 4, 'ENSG00000262796', 'Q8NGP2'),
(2391, 'OR8J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57184216, 57199537, 55.06, 4, 'ENSG00000291858', 'Q8NGP2'),
(2392, 'OR8J1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57184216, 57199537, 55.06, 4, 'ENSG00000172487', 'Q8NGP2'),
(2393, 'OR8J2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 2691187, 2741309, 47.38, 15, 'ENSG00000263150', 'Q8NGG1'),
(2394, 'OR8J2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 2691187, 2741309, 47.38, 15, 'ENSG00000291841', 'Q8NGG1'),
(2395, 'OR8J2', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. This olfactory receptor gene is a segregating pseudogene, where some individuals have an allele that encodes a functional olfactory receptor, while other individuals have an allele encoding a protein that is predicted to be non-functional. [provided by RefSeq, Jan 2017]', '11', 'protein_coding', 2691187, 2741309, 47.38, 15, 'ENSG00000254658', 'Q8NGG1'),
(2396, 'OR8K2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57207346, 57212230, 59.75, 15, 'ENSG00000280617', ''),
(2397, 'OR8K2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57207346, 57212230, 59.75, 15, 'ENSG00000291844', ''),
(2398, 'OR8K2P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 57207346, 57212230, 59.75, 15, 'ENSG00000255134', ''),
(2399, 'OR8L1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 22973819, 23005465, 42.58, 4, 'ENSG00000280577', ''),
(2400, 'OR8L1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 22973819, 23005465, 42.58, 4, 'ENSG00000291847', ''),
(2401, 'OR8L1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 22973819, 23005465, 42.58, 4, 'ENSG00000254903', ''),
(2402, 'OR8U1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 70289045, 70330199, 39.07, 3, 'ENSG00000291846', 'Q8NH10'),
(2403, 'OR8U1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 70289045, 70330199, 39.07, 3, 'ENSG00000172199', 'Q8NH10'),
(2404, 'OR8V1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 53567071, 53609907, 44.05, 5, 'ENSG00000230213', ''),
(2405, 'OR9G3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 6118, 12923, 48.02, 1, 'ENSG00000262268', ''),
(2406, 'OR9G3P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 6118, 12923, 48.02, 1, 'ENSG00000181325', ''),
(2407, 'OR9P1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 56904264, 56904353, 40.00, 1, 'ENSG00000239293', ''),
(2408, 'OR9P1P', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 56904264, 56904353, 40.00, 1, 'ENSG00000282335', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2409, 'OR9Q1', 'Olfactory receptors interact with odorant molecules in the nose, to initiate a neuronal response that triggers the perception of a smell. The olfactory receptor proteins are members of a large family of G-protein-coupled receptors (GPCR) arising from single coding-exon genes. Olfactory receptors share a 7-transmembrane domain structure with many neurotransmitter and hormone receptors and are responsible for the recognition and G protein-mediated transduction of odorant signals. The olfactory receptor gene family is the largest in the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 109996368, 110135565, 38.65, 20, 'ENSG00000186509', 'Q8NGQ5'),
(2410, 'ORAI1', 'The protein encoded by this gene is a membrane calcium channel subunit that is activated by the calcium sensor STIM1 when calcium stores are depleted. This type of channel is the primary way for calcium influx into T-cells. Defects in this gene are a cause of immune dysfunction with T-cell inactivation due to calcium entry defect type 1 (IDTICED1). [provided by RefSeq, Sep 2011]', '12', 'protein_coding', 39120167, 39152680, 54.25, 7, 'ENSG00000293604', 'Q96D31'),
(2411, 'ORAI1', 'The protein encoded by this gene is a membrane calcium channel subunit that is activated by the calcium sensor STIM1 when calcium stores are depleted. This type of channel is the primary way for calcium influx into T-cells. Defects in this gene are a cause of immune dysfunction with T-cell inactivation due to calcium entry defect type 1 (IDTICED1). [provided by RefSeq, Sep 2011]', '12', 'protein_coding', 39120167, 39152680, 54.25, 7, 'ENSG00000276045', 'Q96D31'),
(2412, 'ORAI3', 'Enables store-operated calcium channel activity. Involved in store-operated calcium entry. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 58295080, 58406147, 41.93, 10, 'ENSG00000175938', 'Q9BRQ5'),
(2413, 'ORC5', 'The origin recognition complex (ORC) is a highly conserved six subunit protein complex essential for the initiation of the DNA replication in eukaryotic cells. Studies in yeast demonstrated that ORC binds specifically to origins of replication and serves as a platform for the assembly of additional initiation factors such as Cdc6 and Mcm proteins. The protein encoded by this gene is a subunit of the ORC complex. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Oct 2010]', '7', 'TR_V_gene', 21918188, 21918756, 41.12, 1, 'ENSG00000164815', 'O43913'),
(2414, 'OSBP', 'Oxysterol binding protein is an intracellular protein that is believed to transport sterols from lysosomes to the nucleus where the sterol down-regulates the genes for the LDL receptor, HMG-CoA reductase, and HMG synthetase [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 15319452, 15335554, 39.52, 19, 'ENSG00000110048', 'P22059'),
(2415, 'OSBPL2', 'This gene encodes a member of the oxysterol-binding protein (OSBP) family, a group of intracellular lipid receptors. Most members contain an N-terminal pleckstrin homology domain and a highly conserved C-terminal OSBP-like sterol-binding domain, although the encoded protein contains only the sterol-binding domain. In vitro studies have shown that the encoded protein can bind strongly to phosphatic acid and weakly to phosphatidylinositol 3-phosphate, but cannot bind to 25-hydroxycholesterol. The protein associates with the Golgi apparatus. Transcript variants encoding different isoforms have been described. [provided by RefSeq, Sep 2014]', '20', 'unprocessed_pseudogene', 8437403, 8438425, 49.07, 1, 'ENSG00000130703', 'Q9H1P3'),
(2416, 'OSBPL6', 'This gene encodes a member of the oxysterol-binding protein (OSBP) family, a group of intracellular lipid receptors. Most members contain an N-terminal pleckstrin homology domain and a highly conserved C-terminal OSBP-like sterol-binding domain. Transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 87796, 89620, 55.67, 2, 'ENSG00000079156', 'Q9BZF3'),
(2417, 'OSBPL8', 'This gene encodes a member of a family of proteins containing an N-terminal pleckstrin homology domain and a highly conserved C-terminal oxysterol-binding protein-like sterol-binding domain. It binds mutliple lipid-containing molecules, including phosphatidylserine, phosphatidylinositol 4-phosphate (PI4P) and oxysterol, and promotes their exchange between the endoplasmic reticulum and the plasma membrane. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '12', 'unprocessed_pseudogene', 4937371, 4938294, 45.02, 1, 'ENSG00000091039', 'Q9BZF1'),
(2418, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000275644', 'Q8IYS5'),
(2419, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000273511', 'Q8IYS5'),
(2420, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000278533', 'Q8IYS5'),
(2421, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000275551', 'Q8IYS5'),
(2422, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000276982', 'Q8IYS5'),
(2423, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000274703', 'Q8IYS5'),
(2424, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000278378', 'Q8IYS5'),
(2425, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000277088', 'Q8IYS5'),
(2426, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000275736', 'Q8IYS5'),
(2427, 'OSCAR', 'Osteoclasts are multinucleated cells that resorb bone and are essential for bone homeostasis. This gene encodes an osteoclast-associated receptor (OSCAR), which is a member of the leukocyte receptor complex protein family that plays critical roles in the regulation of both innate and adaptive immune responses. The encoded protein may play a role in oxidative stress-mediated atherogenesis as well as monocyte adhesion. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '19', 'protein_coding', 47836902, 48002561, 39.93, 5, 'ENSG00000170909', 'Q8IYS5'),
(2428, 'OTOA', 'The protein encoded by this gene is specifically expressed in the inner ear, and is located at the interface between the apical surface of the inner ear sensory epithelia and their overlying acellular gels. It is prposed that this protein is involved in the attachment of the inner ear acellular gels to the apical surface of the underlying nonsensory cells. Mutations in this gene are associated with autosomal recessive deafness type 22 (DFNB22). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', '16', 'protein_coding', 84118, 85016, 63.29, 1, 'ENSG00000155719', 'Q7RTW8'),
(2429, 'OTUB2', 'This gene encodes one of several deubiquitylating enzymes. Ubiquitin modification of proteins is needed for their stability and function; to reverse the process, deubiquityling enzymes remove ubiquitin. This protein contains an OTU domain and binds Ubal (ubiquitin aldehyde); an active cysteine protease site is present in the OTU domain. [provided by RefSeq, Aug 2011]', '14', 'miRNA', 2076019, 2076089, 61.97, 1, 'ENSG00000277276', 'Q96DC9'),
(2430, 'OTUB2', 'This gene encodes one of several deubiquitylating enzymes. Ubiquitin modification of proteins is needed for their stability and function; to reverse the process, deubiquityling enzymes remove ubiquitin. This protein contains an OTU domain and binds Ubal (ubiquitin aldehyde); an active cysteine protease site is present in the OTU domain. [provided by RefSeq, Aug 2011]', '14', 'miRNA', 2076019, 2076089, 61.97, 1, 'ENSG00000089723', 'Q96DC9'),
(2431, 'OTUD1', 'Deubiquitinating enzymes (DUBs; see MIM 603478) are proteases that specifically cleave ubiquitin (MIM 191339) linkages, negating the action of ubiquitin ligases. DUBA7 belongs to a DUB subfamily characterized by an ovarian tumor (OTU) domain.[supplied by OMIM, May 2008]', '10', 'protein_coding', 64986895, 65062652, 43.81, 8, 'ENSG00000165312', 'Q5VV17'),
(2432, 'OXGR1', 'This gene encodes a G protein-coupled receptor (GPCR) that belongs to the oxoglutarate receptor family within the GPCR superfamily. The encoded protein is activated by the citric acid intermediate, oxoglutarate, as well as several cysteinyl leukotrienes, including leukotrienes E4, C4 and D4, which are implicated in many inflammatory disorders. In mice, a knock-out of this gene leads to middle ear inflammation, changes in the mucosal epithelium, and an increase in fluid behind the eardrum, and is associated with hearing loss. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2016]', '13', 'protein_coding', 16587999, 16712936, 43.51, 5, 'ENSG00000165621', 'Q96P68'),
(2433, 'P2RY11', 'The product of this gene belongs to the family of G-protein coupled receptors. This family has several receptor subtypes with different pharmacological selectivity, which overlaps in some cases, for various adenosine and uridine nucleotides. This receptor is coupled to the stimulation of the phosphoinositide and adenylyl cyclase pathways and behaves as a selective purinoceptor. Naturally occuring read-through transcripts, resulting from intergenic splicing between this gene and an immediately upstream gene (PPAN, encoding peter pan homolog), have been found. The PPAN-P2RY11 read-through mRNA is ubiquitously expressed and encodes a fusion protein that shares identity with each individual gene product. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 31759584, 31760880, 43.56, 2, 'ENSG00000244165', 'Q96G91'),
(2434, 'PABIR2', 'Predicted to enable protein serine/threonine phosphatase inhibitor activity. Predicted to be part of collagen trimer. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 2691187, 2741309, 47.38, 15, 'ENSG00000156504', 'Q7Z309'),
(2435, 'PABPC1L', 'This gene belongs to the polyadenylate-binding protein type-1 family of proteins. Members of this family bind to the polyA tails of mRNAs to regulate mRNA stability and translation. The mouse ortholog of this gene is required for female fertility. In human, expression of a functional protein is regulated by alternative splicing. The protein-coding splice variant for this gene is abundantly expressed in human oocytes, while a noncoding splice variant subject to nonsense-mediated decay is the predominant splice variant expressed in somatic tissues. [provided by RefSeq, Aug 2019]', '20', 'protein_coding', 82321024, 82334074, 63.33, 8, 'ENSG00000101104', 'Q4VXU2'),
(2436, 'PABPC3', 'Messenger RNA stability and translation initiation are extensively under the control of poly(A)-binding proteins (PABP). See PABPC1 (MIM 604679) for background information.[supplied by OMIM, Jul 2002]', '13', 'protein_coding', 88388430, 88478694, 41.25, 6, 'ENSG00000151846', 'Q9H361'),
(2437, 'PABPC4L', 'Predicted to enable mRNA 3\'-UTR binding activity; poly(A) binding activity; and poly(U) RNA binding activity. Predicted to be part of ribonucleoprotein complex. Predicted to be active in cytoplasmic stress granule; cytosol; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 19734, 25971, 35.12, 1, 'ENSG00000254535', 'P0CB38'),
(2438, 'PABPC5', 'This gene encodes a protein that binds to the polyA tail found at the 3\' end of most eukaryotic mRNAs. It is thought to play a role in the regulation of mRNA metabolic processes in the cytoplasm. This gene is located in a gene-poor region within the X-specific 13d-sY43 subinterval of the chromosome Xq21.3/Yp11.2 homology block. It is located close to translocation breakpoints associated with premature ovarian failure, and is therefore a potential candidate gene for this disorder. [provided by RefSeq, May 2010]', 'X', 'protein_coding', 15114984, 15125786, 57.24, 14, 'ENSG00000174740', 'Q96DU9'),
(2439, 'PADI2', 'This gene encodes a member of the peptidyl arginine deiminase family of enzymes, which catalyze the post-translational deimination of proteins by converting arginine residues into citrullines in the presence of calcium ions. The family members have distinct substrate specificities and tissue-specific expression patterns. The type II enzyme is the most widely expressed family member. Known substrates for this enzyme include myelin basic protein in the central nervous system and vimentin in skeletal muscle and macrophages. This enzyme is thought to play a role in the onset and progression of neurodegenerative human disorders, including Alzheimer disease and multiple sclerosis, and it has also been implicated in glaucoma pathogenesis. This gene exists in a cluster with four other paralogous genes. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 2486414, 2500976, 49.62, 6, 'ENSG00000291387', 'Q9Y2J8'),
(2440, 'PADI2', 'This gene encodes a member of the peptidyl arginine deiminase family of enzymes, which catalyze the post-translational deimination of proteins by converting arginine residues into citrullines in the presence of calcium ions. The family members have distinct substrate specificities and tissue-specific expression patterns. The type II enzyme is the most widely expressed family member. Known substrates for this enzyme include myelin basic protein in the central nervous system and vimentin in skeletal muscle and macrophages. This enzyme is thought to play a role in the onset and progression of neurodegenerative human disorders, including Alzheimer disease and multiple sclerosis, and it has also been implicated in glaucoma pathogenesis. This gene exists in a cluster with four other paralogous genes. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 2486414, 2500976, 49.62, 6, 'ENSG00000117115', 'Q9Y2J8'),
(2441, 'PADI3', 'This gene encodes a member of the peptidyl arginine deiminase family of enzymes, which catalyze the post-translational deimination of proteins by converting arginine residues into citrullines in the presence of calcium ions. The family members have distinct substrate specificities and tissue-specific expression patterns. The type III enzyme modulates hair structural proteins, such as filaggrin in the hair follicle and trichohyalin in the inner root sheath, during hair follicle formation. Together with the type I enzyme, this enzyme may also play a role in terminal differentiation of the epidermis. This gene exists in a cluster with four other paralogous genes. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 46762617, 47175699, 49.42, 11, 'ENSG00000280549', 'Q9ULW8'),
(2442, 'PADI3', 'This gene encodes a member of the peptidyl arginine deiminase family of enzymes, which catalyze the post-translational deimination of proteins by converting arginine residues into citrullines in the presence of calcium ions. The family members have distinct substrate specificities and tissue-specific expression patterns. The type III enzyme modulates hair structural proteins, such as filaggrin in the hair follicle and trichohyalin in the inner root sheath, during hair follicle formation. Together with the type I enzyme, this enzyme may also play a role in terminal differentiation of the epidermis. This gene exists in a cluster with four other paralogous genes. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 46762617, 47175699, 49.42, 11, 'ENSG00000142619', 'Q9ULW8'),
(2443, 'PAGE1', 'This gene is a member of the GAGE family. The GAGE genes are expressed in a variety of tumors and in some fetal and reproductive tissues. This gene is strongly expressed in prostate and prostate cancer. It is also expressed in other male and female reproductive tissues including testis, fallopian tube, uterus, and placenta, as well as in testicular cancer and uterine cancer. The protein encoded by this gene shares sequence similarity with other GAGE/PAGE proteins, and also belongs to a family of CT (cancer-testis) antigens. The protein may play a role in benign and malignant prostate diseases. A related pseudogene is located on chromosome 7. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', 'X', 'rRNA', 442732, 442884, 57.52, 1, 'ENSG00000068985', 'O75459'),
(2444, 'PAGE3', 'This gene is a member of family of proteins that are expressed in a variety of tumors and in some fetal and reproductive tissues. Multiple alternatively spliced transcript variants have been observed. [provided by RefSeq, Jan 2015]', 'X', 'protein_coding', 82789, 120568, 56.56, 3, 'ENSG00000204279', 'Q5JUK9'),
(2445, 'PAIP2', 'Enables translation repressor activity. Involved in negative regulation of translational initiation. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 44656404, 44690308, 40.15, 7, 'ENSG00000120727', 'Q9BPZ3'),
(2446, 'PAK1', 'This gene encodes a family member of serine/threonine p21-activating kinases, known as PAK proteins. These proteins are critical effectors that link RhoGTPases to cytoskeleton reorganization and nuclear signaling, and they serve as targets for the small GTP binding proteins Cdc42 and Rac. This specific family member regulates cell motility and morphology. Mutations in this gene have been associated with macrocephaly, seizures, and speech delay. Overexpression of this gene is also reported in many cancer types, and particularly in breast cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2020]', '11', 'protein_coding', 24606480, 24609699, 52.14, 3, 'ENSG00000149269', 'Q13153'),
(2447, 'PAK1IP1', 'Predicted to enable protein kinase inhibitor activity. Involved in regulation of signal transduction by p53 class mediator and ribosomal large subunit biogenesis. Located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 958452, 1002311, 50.89, 2, 'ENSG00000285226', 'Q9NWT1'),
(2448, 'PAK1IP1', 'Predicted to enable protein kinase inhibitor activity. Involved in regulation of signal transduction by p53 class mediator and ribosomal large subunit biogenesis. Located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 958452, 1002311, 50.89, 2, 'ENSG00000111845', 'Q9NWT1'),
(2449, 'PALS1', 'This gene encodes a member of the p55-like subfamily of the membrane-associated guanylate kinase (MAGUK) gene superfamily. The encoded protein participates in the polarization of differentiating cells, has been shown to regulate myelinating Schwann cells (PMID: 20237282), and is one of the components of the Crumbs complex in the retina. Mice which express lower levels of the orthologous protein have retinal degeneration and impaired vision (PMID: 22114289). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2012]', '14', 'protein_coding', 94529173, 94541759, 43.36, 4, 'ENSG00000072415', 'Q8N3R9'),
(2450, 'PALS2', 'Members of the peripheral membrane-associated guanylate kinase (MAGUK) family function in tumor suppression and receptor clustering by forming multiprotein complexes containing distinct sets of transmembrane, cytoskeletal, and cytoplasmic signaling proteins. All MAGUKs contain a PDZ-SH3-GUK core and are divided into 4 subfamilies, DLG-like (see DLG1; MIM 601014), ZO1-like (see TJP1; MIM 601009), p55-like (see MPP1; MIM 305360), and LIN2-like (see CASK; MIM 300172), based on their size and the presence of additional domains. MPP6 is a member of the p55-like MAGUK subfamily (Tseng et al., 2001 [PubMed 11311936]).[supplied by OMIM, Mar 2008]', '7', 'protein_coding', 57645087, 57649944, 55.21, 10, 'ENSG00000105926', 'Q9NZW5'),
(2451, 'PANK4', 'This gene encodes a protein belonging to the pantothenate kinase family. Pantothenate kinase is a key regulatory enzyme in the biosynthesis of coenzyme A (CoA) in bacteria and mammalian cells. It catalyzes the first committed step in the universal biosynthetic pathway leading to CoA and is itself subject to regulation through feedback inhibition by CoA. This family member is most abundant in muscle but is expressed in all tissues. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 75556048, 75565871, 42.69, 5, 'ENSG00000157881', 'Q9NVE7'),
(2452, 'PANK4', 'This gene encodes a protein belonging to the pantothenate kinase family. Pantothenate kinase is a key regulatory enzyme in the biosynthesis of coenzyme A (CoA) in bacteria and mammalian cells. It catalyzes the first committed step in the universal biosynthetic pathway leading to CoA and is itself subject to regulation through feedback inhibition by CoA. This family member is most abundant in muscle but is expressed in all tissues. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 75556048, 75565871, 42.69, 5, 'ENSG00000273494', 'Q9NVE7'),
(2453, 'PAOX', 'Enables polyamine oxidase activity. Involved in polyamine metabolic process and positive regulation of spermidine biosynthetic process. Predicted to be located in cytosol and peroxisomal matrix. Predicted to be active in cytoplasm. Implicated in lung non-small cell carcinoma. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 93677375, 93894840, 40.55, 12, 'ENSG00000148832', 'Q6QHF9'),
(2454, 'PAPPA-AS1', 'Predicted to be located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 156020826, 156025710, 59.75, 15, 'ENSG00000256040', 'Q5QFB9'),
(2455, 'PAQR3', 'This gene encodes a seven-transmembrane protein localized in the Golgi apparatus in mammalian cells. The encoded protein belongs to the progestin and adipoQ receptor (PAQR) family. This protein functions as a tumor suppressor by inhibiting the Raf/MEK/ERK signaling cascade. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2017]', '4', 'protein_coding', 44876299, 44885530, 41.41, 1, 'ENSG00000163291', 'Q6TCH7'),
(2456, 'PAQR4', 'Enables molecular adaptor activity. Involved in protein stabilization and regulation of ceramide biosynthetic process. Located in Golgi membrane. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 53226900, 53238518, 41.43, 4, 'ENSG00000162073', 'Q8N4S7'),
(2457, 'PARD6B', 'This gene is a member of the PAR6 family and encodes a protein with a PSD95/Discs-large/ZO1 (PDZ) domain, an OPR domain and a semi-Cdc42/Rac interactive binding (CRIB) domain. This cytoplasmic protein is involved in asymmetrical cell division and cell polarization processes as a member of a multi-protein complex. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 43607224, 43619629, 52.39, 3, 'ENSG00000124171', 'Q9BYG5'),
(2458, 'PARP4', 'This gene encodes poly(ADP-ribosyl)transferase-like 1 protein, which is capable of catalyzing a poly(ADP-ribosyl)ation reaction. This protein has a catalytic domain which is homologous to that of poly (ADP-ribosyl) transferase, but lacks an N-terminal DNA binding domain which activates the C-terminal catalytic domain of poly (ADP-ribosyl) transferase. Since this protein is not capable of binding DNA directly, its transferase activity may be activated by other factors such as protein-protein interaction mediated by the extensive carboxyl terminus. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 30949068, 30956461, 54.21, 4, 'ENSG00000102699', 'Q9UKK3'),
(2459, 'PARVA', 'This gene encodes a member of the parvin family of actin-binding proteins. Parvins are associated with focal contacts and contain calponin homology domains that bind to actin filaments. The encoded protein is part of the integrin-linked kinase signaling complex and plays a role in cell adhesion, motility and survival. [provided by RefSeq, Dec 2010]', '11', 'miRNA', 154768596, 154768656, 42.62, 1, 'ENSG00000197702', 'Q9NVD7'),
(2460, 'PATL1', 'Enables poly(G) binding activity and poly(U) RNA binding activity. Involved in P-body assembly and deadenylation-dependent decapping of nuclear-transcribed mRNA. Located in CCR4-NOT complex; P-body; and cytosol. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 48303744, 48599436, 38.50, 13, 'ENSG00000166889', 'Q86TB9'),
(2461, 'PATZ1', 'The protein encoded by this gene contains an A-T hook DNA binding motif which usually binds to other DNA binding structures to play an important role in chromatin modeling and transcription regulation. Its Poz domain is thought to function as a site for protein-protein interaction and is required for transcriptional repression, and the zinc-fingers comprise the DNA binding domain. Since the encoded protein has typical features of a transcription factor, it is postulated to be a repressor of gene expression. In small round cell sarcoma, this gene is fused to EWS by a small inversion of 22q, then the hybrid is thought to be translocated (t(1;22)(p36.1;q12). The rearrangement of chromosome 22 involves intron 8 of EWS and exon 1 of this gene creating a chimeric sequence containing the transactivation domain of EWS fused to zinc finger domain of this protein. This is a distinct example of an intra-chromosomal rearrangement of chromosome 22. Four alternatively spliced transcript variants are described for this gene. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 830291, 839007, 48.89, 1, 'ENSG00000100105', 'Q9HBE1'),
(2462, 'PAX3', 'This gene is a member of the paired box (PAX) family of transcription factors. Members of the PAX family typically contain a paired box domain and a paired-type homeodomain. These genes play critical roles during fetal development. Mutations in paired box gene 3 are associated with Waardenburg syndrome, craniofacial-deafness-hand syndrome, and alveolar rhabdomyosarcoma. The translocation t(2;13)(q35;q14), which represents a fusion between PAX3 and the forkhead gene, is a frequent finding in alveolar rhabdomyosarcoma. Alternative splicing results in transcripts encoding isoforms with different C-termini. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000135903', 'P23760'),
(2463, 'PAX5', 'This gene encodes a member of the paired box (PAX) family of transcription factors. The central feature of this gene family is a novel, highly conserved DNA-binding motif, known as the paired box. Paired box transcription factors are important regulators in early development, and alterations in the expression of their genes are thought to contribute to neoplastic transformation. This gene encodes the B-cell lineage specific activator protein that is expressed at early, but not late stages of B-cell differentiation. Its expression has also been detected in developing CNS and testis and so the encoded protein may also play a role in neural development and spermatogenesis. This gene is located at 9p13, which is involved in t(9;14)(p13;q32) translocations recurring in small lymphocytic lymphomas of the plasmacytoid subtype, and in derived large-cell lymphomas. This translocation brings the potent E-mu enhancer of the IgH gene into close proximity of the PAX5 promoter, suggesting that the deregulation of transcription of this gene contributes to the pathogenesis of these lymphomas. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2013]', '9', 'protein_coding', 58266792, 58268260, 46.90, 1, 'ENSG00000196092', 'Q02548'),
(2464, 'PAX9', 'This gene is a member of the paired box (PAX) family of transcription factors. Members of this gene family typically contain a paired box domain, an octapeptide, and a paired-type homeodomain. These genes play critical roles during fetal development and cancer growth. Mice lacking this gene exhibit impaired development of organs, musculature and the skeleton, including absent and abnormally developed teeth, and neonatal lethality. Mutations in the human gene are associated with selective tooth agenesis-3. [provided by RefSeq, Sep 2015]', '14', 'protein_coding', 3566718, 3795557, 37.28, 11, 'ENSG00000198807', 'P55771'),
(2465, 'PAXBP1', 'Predicted to enable DNA binding activity and histone methyltransferase binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within positive regulation of myoblast proliferation and regulation of skeletal muscle satellite cell proliferation. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 347541, 348439, 63.29, 1, 'ENSG00000159086', 'Q9Y5B6'),
(2466, 'PAXBP1', 'Predicted to enable DNA binding activity and histone methyltransferase binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within positive regulation of myoblast proliferation and regulation of skeletal muscle satellite cell proliferation. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 347541, 348439, 63.29, 1, 'ENSG00000263141', 'Q9Y5B6'),
(2467, 'PAXX', 'The protein encoded by this gene plays a role in the nonhomologous end joining (NHEJ) pathway of DNA double-strand break repair. The encoded protein may function to stabilize the Ku70/Ku80 heterodimer to facilitate the assembly and maintain the stability of the NHEJ complex. [provided by RefSeq, Jul 2016]', '9', 'unitary_pseudogene', 102919, 103852, 45.82, 1, 'ENSG00000310560', 'Q9BUH6'),
(2468, 'PBLD', 'Enables identical protein binding activity. Involved in maintenance of gastrointestinal epithelium. Acts upstream of or within with a positive effect on negative regulation of transforming growth factor beta receptor signaling pathway. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 6599239, 6633291, 43.29, 9, 'ENSG00000108187', 'P30039'),
(2469, 'PCAT18', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 41310172, 41324873, 53.54, 3, 'ENSG00000265369', ''),
(2470, 'PCCA', 'The protein encoded by this gene is the alpha subunit of the heterodimeric mitochondrial enzyme Propionyl-CoA carboxylase. PCCA encodes the biotin-binding region of this enzyme. Mutations in either PCCA or PCCB (encoding the beta subunit) lead to an enzyme deficiency resulting in propionic acidemia. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]', '13', 'protein_coding', 43092956, 43107570, 43.62, 11, 'ENSG00000175198', 'P05165'),
(2471, 'PCDH20', 'This gene belongs to the protocadherin gene family, a subfamily of the cadherin superfamily. This gene encodes a protein which contains 6 extracellular cadherin domains, a transmembrane domain and a cytoplasmic tail differing from those of the classical cadherins. Although its specific function is undetermined, the cadherin-related neuronal receptor is thought to play a role in the establishment and function of specific cell-cell connections in the brain. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 100302077, 100352942, 37.95, 4, 'ENSG00000280165', 'Q8N6Y1'),
(2472, 'PCDHA10', 'This gene is a member of the protocadherin alpha gene cluster, one of three related gene clusters tandemly linked on chromosome five that demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The alpha gene cluster is composed of 15 cadherin superfamily genes related to the mouse CNR genes and consists of 13 highly similar and 2 more distantly related coding sequences. The tandem array of 15 N-terminal exons, or variable exons, are followed by downstream C-terminal exons, or constant exons, which are shared by all genes in the cluster. The large, uninterrupted N-terminal exons each encode six cadherin ectodomains while the C-terminal exons encode the cytoplasmic domain. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell connections in the brain. Alternative splicing has been observed and additional variants have been suggested but their full-length nature has yet to be determined. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 38254000, 38256093, 57.35, 1, 'ENSG00000291659', 'Q9Y5I2'),
(2473, 'PCDHA10', 'This gene is a member of the protocadherin alpha gene cluster, one of three related gene clusters tandemly linked on chromosome five that demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The alpha gene cluster is composed of 15 cadherin superfamily genes related to the mouse CNR genes and consists of 13 highly similar and 2 more distantly related coding sequences. The tandem array of 15 N-terminal exons, or variable exons, are followed by downstream C-terminal exons, or constant exons, which are shared by all genes in the cluster. The large, uninterrupted N-terminal exons each encode six cadherin ectodomains while the C-terminal exons encode the cytoplasmic domain. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell connections in the brain. Alternative splicing has been observed and additional variants have been suggested but their full-length nature has yet to be determined. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 38254000, 38256093, 57.35, 1, 'ENSG00000250120', 'Q9Y5I2'),
(2474, 'PCDHA12', 'This gene is a member of the protocadherin alpha gene cluster, one of three related gene clusters tandemly linked on chromosome five that demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The alpha gene cluster is composed of 15 cadherin superfamily genes related to the mouse CNR genes and consists of 13 highly similar and 2 more distantly related coding sequences. The tandem array of 15 N-terminal exons, or variable exons, are followed by downstream C-terminal exons, or constant exons, which are shared by all genes in the cluster. The large, uninterrupted N-terminal exons each encode six cadherin ectodomains while the C-terminal exons encode the cytoplasmic domain. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell connections in the brain. Alternative splicing has been observed and additional variants have been suggested but their full-length nature has yet to be determined. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 54807599, 54816015, 52.36, 13, 'ENSG00000291663', 'Q9UN75'),
(2475, 'PCDHA12', 'This gene is a member of the protocadherin alpha gene cluster, one of three related gene clusters tandemly linked on chromosome five that demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The alpha gene cluster is composed of 15 cadherin superfamily genes related to the mouse CNR genes and consists of 13 highly similar and 2 more distantly related coding sequences. The tandem array of 15 N-terminal exons, or variable exons, are followed by downstream C-terminal exons, or constant exons, which are shared by all genes in the cluster. The large, uninterrupted N-terminal exons each encode six cadherin ectodomains while the C-terminal exons encode the cytoplasmic domain. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell connections in the brain. Alternative splicing has been observed and additional variants have been suggested but their full-length nature has yet to be determined. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 54807599, 54816015, 52.36, 13, 'ENSG00000251664', 'Q9UN75'),
(2476, 'PCDHB11', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 10401009, 10434222, 38.18, 4, 'ENSG00000291690', 'Q9Y5F2'),
(2477, 'PCDHB11', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 10401009, 10434222, 38.18, 4, 'ENSG00000197479', 'Q9Y5F2'),
(2478, 'PCDHB13', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 32009015, 32056425, 37.57, 2, 'ENSG00000291692', 'Q9Y5F0'),
(2479, 'PCDHB13', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 32009015, 32056425, 37.57, 2, 'ENSG00000187372', 'Q9Y5F0'),
(2480, 'PCDHB15', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 64275257, 64409468, 33.53, 6, 'ENSG00000291695', 'Q9Y5E8'),
(2481, 'PCDHB15', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 64275257, 64409468, 33.53, 6, 'ENSG00000113248', 'Q9Y5E8'),
(2482, 'PCDHB18P', 'Predicted to enable calcium ion binding activity. Predicted to be involved in cell adhesion. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 24698355, 24757940, 45.72, 14, 'ENSG00000291703', 'Q96TA0'),
(2483, 'PCDHB18P', 'Predicted to enable calcium ion binding activity. Predicted to be involved in cell adhesion. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 24698355, 24757940, 45.72, 14, 'ENSG00000146001', 'Q96TA0'),
(2484, 'PCDHB5', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 37438102, 37465450, 44.27, 1, 'ENSG00000291677', 'Q9Y5E4');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2485, 'PCDHB5', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. The beta cluster contains 16 genes and 3 pseudogenes, each encoding 6 extracellular cadherin domains and a cytoplasmic tail that deviates from others in the cadherin superfamily. The extracellular domains interact in a homophilic manner to specify differential cell-cell connections. Unlike the alpha and gamma clusters, the transcripts from these genes are made up of only one large exon, not sharing common 3\' exons as expected. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins. Their specific functions are unknown but they most likely play a critical role in the establishment and function of specific cell-cell neural connections. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 37438102, 37465450, 44.27, 1, 'ENSG00000113209', 'Q9Y5E4'),
(2486, 'PCDHB6', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. Unlike the alpha and gamma clusters, the transcripts from these genes do not share common 3\' exons. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell neural connections. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]', '5', 'protein_coding', 59736692, 59756041, 41.50, 11, 'ENSG00000291679', 'Q9Y5E3'),
(2487, 'PCDHB6', 'This gene is a member of the protocadherin beta gene cluster, one of three related gene clusters tandemly linked on chromosome five. The gene clusters demonstrate an unusual genomic organization similar to that of B-cell and T-cell receptor gene clusters. Unlike the alpha and gamma clusters, the transcripts from these genes do not share common 3\' exons. These neural cadherin-like cell adhesion proteins are integral plasma membrane proteins that most likely play a critical role in the establishment and function of specific cell-cell neural connections. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]', '5', 'protein_coding', 59736692, 59756041, 41.50, 11, 'ENSG00000113211', 'Q9Y5E3'),
(2488, 'PCED1A', 'The protein encoded by this gene is a member of the GDSL/SGNH superfamily. Members of this family are hydrolytic enzymes with esterase and lipase activity and broad substrate specificity. This protein belongs to the Pmr5-Cas1p-esterase subfamily in that it contains the catalytic triad comprised of serine, aspartate and histidine and lacks two conserved regions (glycine after strand S2 and GxND motif). A pseudogene of this gene has been identified on the long arm of chromosome 2. Alternative splicing results in multiple transcript variants that encode different protein isoforms. [provided by RefSeq, Sep 2012]', '20', 'protein_coding', 56682470, 56684196, 53.73, 4, 'ENSG00000132635', 'Q9H1Q7'),
(2489, 'PCK1', 'This gene is a main control point for the regulation of gluconeogenesis. The cytosolic enzyme encoded by this gene, along with GTP, catalyzes the formation of phosphoenolpyruvate from oxaloacetate, with the release of carbon dioxide and GDP. The expression of this gene can be regulated by insulin, glucocorticoids, glucagon, cAMP, and diet. Defects in this gene are a cause of cytosolic phosphoenolpyruvate carboxykinase deficiency. A mitochondrial isozyme of the encoded protein also has been characterized. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 5422393, 5426347, 36.13, 1, 'ENSG00000124253', 'P35558'),
(2490, 'PCNA', 'The protein encoded by this gene is found in the nucleus and is a cofactor of DNA polymerase delta. The encoded protein acts as a homotrimer and helps increase the processivity of leading strand synthesis during DNA replication. In response to DNA damage, this protein is ubiquitinated and is involved in the RAD6-dependent DNA repair pathway. Two transcript variants encoding the same protein have been found for this gene. Pseudogenes of this gene have been described on chromosome 4 and on the X chromosome. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 19633659, 19673441, 43.32, 5, 'ENSG00000132646', 'P12004'),
(2491, 'PCNP', 'Involved in proteasome-mediated ubiquitin-dependent protein catabolic process and protein ubiquitination. Located in nuclear body. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 28050993, 28334486, 38.78, 9, 'ENSG00000081154', 'Q8WW12'),
(2492, 'PCNT', 'The protein encoded by this gene binds to calmodulin and is expressed in the centrosome. It is an integral component of the pericentriolar material (PCM). The protein contains a series of coiled-coil domains and a highly conserved PCM targeting motif called the PACT domain near its C-terminus. The protein interacts with the microtubule nucleation component gamma-tubulin and is likely important to normal functioning of the centrosomes, cytoskeleton, and cell-cycle progression. Mutations in this gene cause Seckel syndrome-4 and microcephalic osteodysplastic primordial dwarfism type II. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2015]', '21', 'TR_J_gene', 22497657, 22497718, 54.84, 1, 'ENSG00000160299', 'O95613'),
(2493, 'PCNX1', 'This gene encodes an evolutionarily conserved transmembrane protein similar to the pecanex protein in Drosophila. The fly protein is a component of the Notch signaling pathway, which functions in several developmental processes. [provided by RefSeq, Jul 2016]', '14', 'protein_coding', 1510427, 1612072, 54.44, 11, 'ENSG00000100731', 'Q96RV3'),
(2494, 'PCNX4', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 44550357, 44551505, 66.23, 2, 'ENSG00000126773', 'Q63HM2'),
(2495, 'PCSK2', 'This gene encodes a member of the subtilisin-like proprotein convertase family, which includes proteases that process protein and peptide precursors trafficking through regulated or constitutive branches of the secretory pathway. The protein undergoes an initial autocatalytic processing event and interacts with a neuroendocrine secretory protein in the ER, exits the ER and sorts to secretory granules, where it is cleaved and catalytically activated during intracellular transport. The encoded protease is packaged into and activated in dense core secretory granules and expressed in the neuroendocrine system and brain. This gene encodes one of the seven basic amino acid-specific members which cleave their substrates at single or paired basic residues. It functions in the proteolytic activation of polypeptide hormones and neuropeptides precursors. Single nucleotide polymorphisms in this gene may increase susceptibility to myocardial infarction and type 2 diabetes. This gene may also play a role in tumor development and progression. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jan 2014]', '20', 'protein_coding', 51596514, 51768390, 46.63, 3, 'ENSG00000125851', 'P16519'),
(2496, 'PCTP', 'Enables phosphatidylcholine binding activity and phosphatidylcholine transporter activity. Involved in phospholipid transport. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 102423245, 102432584, 52.83, 4, 'ENSG00000141179', 'Q9UKL6'),
(2497, 'PCYT1B', 'The protein encoded by this gene belongs to the cytidylyltransferase family. It is involved in the regulation of phosphatidylcholine biosynthesis. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 51238724, 51271362, 46.75, 15, 'ENSG00000102230', 'Q9Y5K3'),
(2498, 'PDE3B', 'Enables 3\',5\'-cyclic-AMP phosphodiesterase activity. Involved in several processes, including negative regulation of angiogenesis; negative regulation of lipid catabolic process; and regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction. Located in membrane. Part of guanyl-nucleotide exchange factor complex. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 113854661, 113917194, 35.52, 2, 'ENSG00000152270', 'Q13370'),
(2499, 'PDE6G', 'This gene encodes the gamma subunit of cyclic GMP-phosphodiesterase, which is composed of alpha- and beta- catalytic subunits and two identical, inhibitory gamma subunits. This gene is expressed in rod photoreceptors and functions in the phototransduction signaling cascade. It is also expressed in a variety of other tissues, and has been shown to regulate the c-Src protein kinase and G-protein-coupled receptor kinase 2. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2009]', '17', 'protein_coding', 18858906, 19049358, 42.63, 5, 'ENSG00000185527', 'P18545'),
(2500, 'PDF', 'This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. The protein is expressed in a broad range of cell types, acts as a pleiotropic cytokine and is involved in the stress response program of cells after cellular injury. Increased protein levels are associated with disease states such as tissue hypoxia, inflammation, acute injury and oxidative stress. [provided by RefSeq, Aug 2016]', '19', 'protein_coding', 18892298, 18984114, 45.60, 8, 'ENSG00000258429', 'Q99988'),
(2501, 'PDGFB', 'This gene encodes a member of the protein family comprised of both platelet-derived growth factors (PDGF) and vascular endothelial growth factors (VEGF). The encoded preproprotein is proteolytically processed to generate platelet-derived growth factor subunit B, which can homodimerize, or alternatively, heterodimerize with the related platelet-derived growth factor subunit A. These proteins bind and activate PDGF receptor tyrosine kinases, which play a role in a wide range of developmental processes. Mutations in this gene are associated with meningioma. Reciprocal translocations between chromosomes 22 and 17, at sites where this gene and that for collagen type 1, alpha 1 are located, are associated with dermatofibrosarcoma protuberans, a rare skin tumor. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2015]', '22', 'protein_coding', 23192312, 23194245, 60.50, 6, 'ENSG00000100311', 'P01127'),
(2502, 'PDGFD', 'The protein encoded by this gene is a member of the platelet-derived growth factor family. The four members of this family are mitogenic factors for cells of mesenchymal origin and are characterized by a core motif of eight cysteines, seven of which are found in this factor. This gene product only forms homodimers and, therefore, does not dimerize with the other three family members. It differs from alpha and beta members of this family in having an unusual N-terminal domain, the CUB domain. Two splice variants have been identified for this gene. [provided by RefSeq, Jul 2008]', '11', 'snoRNA', 100951856, 100951933, 42.31, 1, 'ENSG00000170962', 'Q9GZP0'),
(2503, 'PDPR', 'Pyruvate dehydrogenase complex (PDC) catalyzes the oxidative decarboxylation of pyruvate and links glycolysis to the tricarboxylic acid cycle and fatty acid synthesis. The dephosphorylation and reactivation of PDC is catalyzed by pyruvate dehydrogenase phosphatase (PDP). The dimeric PDP has a catalytic subunit and a regulatory subunit. This gene encodes the FAD-containing regulatory subunit of PDP. The encoded protein acts to decrease the sensitivity of the PDP catalytic subunit to magnesium ions. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2017]', '16', 'protein_coding', 180100795, 180209211, 49.98, 7, 'ENSG00000090857', 'Q8NCN5'),
(2504, 'PDRG1', 'Predicted to enable unfolded protein binding activity. Predicted to be involved in protein stabilization. Part of RPAP3/R2TP/prefoldin-like complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 101452593, 101720856, 36.42, 6, 'ENSG00000088356', 'Q9NUG6'),
(2505, 'PDSS2', 'The protein encoded by this gene is an enzyme that synthesizes the prenyl side-chain of coenzyme Q, or ubiquinone, one of the key elements in the respiratory chain. The gene product catalyzes the formation of all trans-polyprenyl pyrophosphates from isopentyl diphosphate in the assembly of polyisoprenoid side chains, the first step in coenzyme Q biosynthesis. Defects in this gene are a cause of coenzyme Q10 deficiency.[provided by RefSeq, Oct 2009]', '6', 'protein_coding', 802503, 804519, 43.33, 1, 'ENSG00000164494', 'Q86YH6'),
(2506, 'PDXK', 'The protein encoded by this gene phosphorylates vitamin B6, a step required for the conversion of vitamin B6 to pyridoxal-5-phosphate, an important cofactor in intermediary metabolism. The encoded protein is cytoplasmic and probably acts as a homodimer. Alternatively spliced transcript variants have been described, but their biological validity has not been determined. [provided by RefSeq, Jul 2008]', '21', 'miRNA', 39447010, 39447069, 48.33, 1, 'ENSG00000160209', 'O00764'),
(2507, 'PELO', 'This gene encodes a protein which contains a conserved nuclear localization signal. The encoded protein may have a role in spermatogenesis, cell cycle control, and in meiotic cell division. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 104753147, 104768494, 52.25, 8, 'ENSG00000152684', 'Q9BRX2'),
(2508, 'PELP1', 'This gene encodes a transcription factor which coactivates transcription of estrogen receptor responsive genes and corepresses genes activated by other hormone receptors or sequence-specific transcription factors. Expression of this gene is regulated by both members of the estrogen receptor family. This gene may be involved in the progression of several types of cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2013]', '17', 'protein_coding', 194387, 208682, 48.09, 1, 'ENSG00000141456', 'Q8IZL8'),
(2509, 'PEMT', 'This gene encodes a membrane-bound protein that is a member of the mucin family. Mucins are O-glycosylated proteins that play an essential role in forming protective mucous barriers on epithelial surfaces. These proteins also play a role in intracellular signaling. This protein is expressed on the apical surface of epithelial cells that line the mucosal surfaces of many different tissues including lung, breast stomach and pancreas. This protein is proteolytically cleaved into alpha and beta subunits that form a heterodimeric complex. The N-terminal alpha subunit functions in cell-adhesion and the C-terminal beta subunit is involved in cell signaling. Overexpression, aberrant intracellular localization, and changes in glycosylation of this protein have been associated with carcinomas. This gene is known to contain a highly polymorphic variable number tandem repeats (VNTR) domain. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Feb 2011]', '1', 'protein_coding', 2528745, 2530263, 68.47, 1, 'ENSG00000133027', 'Q9UBM1'),
(2510, 'PEX14', 'This gene encodes an essential component of the peroxisomal import machinery. The protein is integrated into peroxisome membranes with its C-terminus exposed to the cytosol, and interacts with the cytosolic receptor for proteins containing a PTS1 peroxisomal targeting signal. The protein also functions as a transcriptional corepressor and interacts with a histone deacetylase. A mutation in this gene results in one form of Zellweger syndrome. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 122378966, 122429802, 36.55, 7, 'ENSG00000142655', 'O75381'),
(2511, 'PEX5', 'The product of this gene binds to the C-terminal PTS1-type tripeptide peroxisomal targeting signal (SKL-type) and plays an essential role in peroxisomal protein import. Peroxins (PEXs) are proteins that are essential for the assembly of functional peroxisomes. The peroxisome biogenesis disorders (PBDs) are a group of genetically heterogeneous autosomal recessive, lethal diseases characterized by multiple defects in peroxisome function. The peroxisomal biogenesis disorders are a heterogeneous group with at least 14 complementation groups and with more than 1 phenotype being observed in cases falling into particular complementation groups. Although the clinical features of PBD patients vary, cells from all PBD patients exhibit a defect in the import of one or more classes of peroxisomal matrix proteins into the organelle. Defects in this gene are a cause of neonatal adrenoleukodystrophy (NALD), a cause of Zellweger syndrome (ZWS) as well as may be a cause of infantile Refsum disease (IRD). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Oct 2008]', '12', 'lncRNA', 21565092, 21605081, 46.62, 2, 'ENSG00000288217', 'P50542'),
(2512, 'PEX5', 'The product of this gene binds to the C-terminal PTS1-type tripeptide peroxisomal targeting signal (SKL-type) and plays an essential role in peroxisomal protein import. Peroxins (PEXs) are proteins that are essential for the assembly of functional peroxisomes. The peroxisome biogenesis disorders (PBDs) are a group of genetically heterogeneous autosomal recessive, lethal diseases characterized by multiple defects in peroxisome function. The peroxisomal biogenesis disorders are a heterogeneous group with at least 14 complementation groups and with more than 1 phenotype being observed in cases falling into particular complementation groups. Although the clinical features of PBD patients vary, cells from all PBD patients exhibit a defect in the import of one or more classes of peroxisomal matrix proteins into the organelle. Defects in this gene are a cause of neonatal adrenoleukodystrophy (NALD), a cause of Zellweger syndrome (ZWS) as well as may be a cause of infantile Refsum disease (IRD). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Oct 2008]', '12', 'lncRNA', 21565092, 21605081, 46.62, 2, 'ENSG00000139197', 'P50542'),
(2513, 'PFDN4', 'This gene encodes a member of the prefoldin beta subunit family. The encoded protein is one of six subunits of prefoldin, a molecular chaperone complex that binds and stabilizes newly synthesized polypeptides, thereby allowing them to fold correctly. The complex, consisting of two alpha and four beta subunits, forms a double beta barrel assembly with six protruding coiled-coils. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 5974662, 5974732, 54.93, 1, 'ENSG00000101132', 'Q9NQP4'),
(2514, 'PFDN5', 'This gene encodes a member of the prefoldin alpha subunit family. The encoded protein is one of six subunits of prefoldin, a molecular chaperone complex that binds and stabilizes newly synthesized polypeptides, thereby allowing them to fold correctly. The complex, consisting of two alpha and four beta subunits, forms a double beta barrel assembly with six protruding coiled-coils. The encoded protein may also repress the transcriptional activity of the proto-oncogene c-Myc. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 459939, 563509, 49.19, 7, 'ENSG00000291834', 'Q99471'),
(2515, 'PFDN5', 'This gene encodes a member of the prefoldin alpha subunit family. The encoded protein is one of six subunits of prefoldin, a molecular chaperone complex that binds and stabilizes newly synthesized polypeptides, thereby allowing them to fold correctly. The complex, consisting of two alpha and four beta subunits, forms a double beta barrel assembly with six protruding coiled-coils. The encoded protein may also repress the transcriptional activity of the proto-oncogene c-Myc. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 459939, 563509, 49.19, 7, 'ENSG00000123349', 'Q99471'),
(2516, 'PGAM2', 'Phosphoglycerate mutase (PGAM) catalyzes the reversible reaction of 3-phosphoglycerate (3-PGA) to 2-phosphoglycerate (2-PGA) in the glycolytic pathway. The PGAM is a dimeric enzyme containing, in different tissues, different proportions of a slow-migrating muscle (MM) isozyme, a fast-migrating brain (BB) isozyme, and a hybrid form (MB). This gene encodes muscle-specific PGAM subunit. Mutations in this gene cause muscle phosphoglycerate mutase eficiency, also known as glycogen storage disease X. [provided by RefSeq, Sep 2009]', '7', 'protein_coding', 7158624, 7411488, 43.68, 11, 'ENSG00000164708', 'P15259'),
(2517, 'PGK2', 'This gene is intronless, arose via retrotransposition of the phosphoglycerate kinase 1 gene, and is expressed specifically in the testis. Initially assumed to be a pseudogene, the encoded protein is actually a functional phosphoglycerate kinase that catalyzes the reversible conversion of 1,3-bisphosphoglycerate to 3-phosphoglycerate, during the Embden-Meyerhof-Parnas pathway of glycolysis, in the later stages of spermatogenesis.[provided by RefSeq, May 2010]', '6', 'protein_coding', 51930436, 52012125, 44.70, 27, 'ENSG00000170950', 'P07205'),
(2518, 'PGLYRP1', 'Enables several functions, including Hsp70 protein binding activity; peptidoglycan binding activity; and peptidoglycan immune receptor activity. Involved in antimicrobial humoral immune response mediated by antimicrobial peptide; defense response to Gram-positive bacterium; and detection of bacterium. Located in extracellular exosome. Is active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 51692786, 51692864, 56.96, 1, 'ENSG00000008438', 'O75594'),
(2519, 'PGM5', 'Phosphoglucomutases (EC 5.2.2.2.), such as PGM5, are phosphotransferases involved in interconversion of glucose-1-phosphate and glucose-6-phosphate. PGM activity is essential in formation of carbohydrates from glucose-6-phosphate and in formation of glucose-6-phosphate from galactose and glycogen (Edwards et al., 1995 [PubMed 8586438]).[supplied by OMIM, Mar 2008]', '9', 'protein_coding', 30989365, 31042815, 36.17, 3, 'ENSG00000154330', 'Q15124'),
(2520, 'PGRMC1', 'This gene encodes a putative membrane-associated progesterone steroid receptor. The protein is expressed predominantly in the liver and kidney. [provided by RefSeq, Mar 2010]', 'X', 'protein_coding', 462371, 477391, 48.84, 1, 'ENSG00000101856', 'O00264'),
(2521, 'PHF19', 'Enables methylated histone binding activity. Involved in negative regulation of gene expression, epigenetic. Located in ESC/E(Z) complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'unprocessed_pseudogene', 124460534, 124461485, 45.38, 1, 'ENSG00000119403', 'Q5T6S3'),
(2522, 'PHF20', 'Predicted to enable DNA binding activity and zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in cytosol; nuclear membrane; and nucleoplasm. Part of MLL1 complex and NSL complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 17579066, 17797124, 38.98, 2, 'ENSG00000025293', 'Q9BVI0'),
(2523, 'PHF3', 'This gene encodes a member of a PHD finger-containing gene family. This gene may function as a transcription factor and may be involved in glioblastomas development. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2014]', '6', 'protein_coding', 126340115, 126348875, 35.60, 3, 'ENSG00000118482', 'Q92576'),
(2524, 'PHF8', 'The protein encoded by this gene is a histone lysine demethylase that preferentially acts on histones in the monomethyl or dimethyl states. The encoded protein requires Fe(2+) ion, 2-oxoglutarate, and oxygen for its catalytic activity. The protein has an N-terminal PHD finger and a central Jumonji C domain. This gene is thought to function as a transcription activator. Defects in this gene are a cause of syndromic X-linked Siderius type intellectual disability (MRXSSD) and over-expression of this gene is associated with several forms of cancer. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2017]', 'X', 'protein_coding', 45344063, 45413619, 39.89, 2, 'ENSG00000172943', 'Q9UPP1'),
(2525, 'PHKA1', 'Phosphorylase kinase is a polymer of 16 subunits, four each of alpha, beta, gamma and delta. The alpha subunit includes the skeletal muscle and hepatic isoforms, and the skeletal muscle isoform is encoded by this gene. The beta subunit is the same in both the muscle and hepatic isoforms, and encoded by one gene. The gamma subunit also includes the skeletal muscle and hepatic isoforms, which are encoded by two different genes. The delta subunit is a calmodulin and can be encoded by three different genes. The gamma subunits contain the active site of the enzyme, whereas the alpha and beta subunits have regulatory functions controlled by phosphorylation. The delta subunit mediates the dependence of the enzyme on calcium concentration. Mutations in this gene cause glycogen storage disease type 9D, also known as X-linked muscle glycogenosis. Alternatively spliced transcript variants encoding different isoforms have been identified in this gene. A pseudogene has been found on chromosome 1.[provided by RefSeq, Feb 2010]', 'X', 'protein_coding', 77247813, 77435034, 39.74, 9, 'ENSG00000067177', 'P46020'),
(2526, 'PHKA2', 'Phosphorylase kinase is a polymer of 16 subunits, four each of alpha, beta, gamma and delta. The alpha subunit includes the skeletal muscle and hepatic isoforms, and the hepatic isoform is encoded by this gene. The beta subunit is the same in both the muscle and hepatic isoforms, and encoded by one gene. The gamma subunit also includes the skeletal muscle and hepatic isoforms, which are encoded by two different genes. The delta subunit is a calmodulin and can be encoded by three different genes. The gamma subunits contain the active site of the enzyme, whereas the alpha and beta subunits have regulatory functions controlled by phosphorylation. The delta subunit mediates the dependence of the enzyme on calcium concentration. Mutations in this gene cause glycogen storage disease type 9A, also known as X-linked liver glycogenosis. Alternatively spliced transcript variants have been reported, but the full-length nature of these variants has not been determined.[provided by RefSeq, Feb 2010]', 'X', 'TR_V_gene', 142482548, 142483019, 55.51, 1, 'ENSG00000044446', 'P46019'),
(2527, 'PHLDA2', 'This gene is located in a cluster of imprinted genes on chromosome 11p15.5, which is considered to be an important tumor suppressor gene region. Alterations in this region may be associated with the Beckwith-Wiedemann syndrome, Wilms tumor, rhabdomyosarcoma, adrenocortical carcinoma, and lung, ovarian, and breast cancer. This gene has been shown to be imprinted, with preferential expression from the maternal allele in placenta and liver. [provided by RefSeq, Oct 2010]', '11', 'protein_coding', 25016612, 25026664, 37.78, 2, 'ENSG00000274538', 'Q53GA4'),
(2528, 'PHLDA2', 'This gene is located in a cluster of imprinted genes on chromosome 11p15.5, which is considered to be an important tumor suppressor gene region. Alterations in this region may be associated with the Beckwith-Wiedemann syndrome, Wilms tumor, rhabdomyosarcoma, adrenocortical carcinoma, and lung, ovarian, and breast cancer. This gene has been shown to be imprinted, with preferential expression from the maternal allele in placenta and liver. [provided by RefSeq, Oct 2010]', '11', 'protein_coding', 25016612, 25026664, 37.78, 2, 'ENSG00000181649', 'Q53GA4'),
(2529, 'PHLPP1', 'This gene encodes a member of the serine/threonine phosphatase family. The encoded protein promotes apoptosis by dephosphorylating and inactivating the serine/threonine kinase Akt, and functions as a tumor suppressor in multiple types of cancer. Increased expression of this gene may also play a role in obesity and type 2 diabetes by interfering with Akt-mediated insulin signaling. [provided by RefSeq, Dec 2011]', '18', 'miRNA', 107531171, 107531248, 44.87, 1, 'ENSG00000081913', 'O60346'),
(2530, 'PIAS2', 'This gene encodes a member of the protein inhibitor of activated STAT family, which function as SUMO E3 ligases and play important roles in many cellular processes by mediating the sumoylation of target proteins. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. Isoforms of the encoded protein enhance the sumoylation of specific target proteins including the p53 tumor suppressor protein, c-Jun, and the androgen receptor. A pseudogene of this gene is located on the short arm of chromosome 4. The symbol MIZ1 has also been associated with ZBTB17 which is a different gene located on chromosome 1. [provided by RefSeq, Aug 2017]', '18', 'protein_coding', 26226445, 26391685, 38.62, 3, 'ENSG00000078043', 'O75928'),
(2531, 'PIEZO2', 'The protein encoded by this gene contains more than thirty transmembrane domains and likely functions as part of mechanically-activated (MA) cation channels. These channels serve to connect mechanical forces to biological signals. The encoded protein quickly adapts MA currents in somatosensory neurons. Defects in this gene are a cause of type 5 distal arthrogryposis. Several alternatively spliced transcript variants of this gene have been described, but their full-length nature is not known. [provided by RefSeq, Feb 2014]', '18', 'protein_coding', 108958248, 109118134, 37.56, 2, 'ENSG00000154864', 'Q9H5I5'),
(2532, 'PIGA', 'This gene encodes a protein required for synthesis of N-acetylglucosaminyl phosphatidylinositol (GlcNAc-PI), the first intermediate in the biosynthetic pathway of GPI anchor. The GPI anchor is a glycolipid found on many blood cells and which serves to anchor proteins to the cell surface. Paroxysmal nocturnal hemoglobinuria, an acquired hematologic disorder, has been shown to result from mutations in this gene. Alternate splice variants have been characterized. A related pseudogene is located on chromosome 12. [provided by RefSeq, Jun 2010]', 'X', 'protein_coding', 48383516, 48393347, 44.95, 2, 'ENSG00000165195', 'P37287'),
(2533, 'PIGG', 'This gene encodes an enzyme involved in glycosylphosphatidylinositol-anchor biosynthesis. The encoded protein, which is localized to the endoplasmic reticulum, is involved in transferring ethanoloamine phosphate to mannose 2 of glycosylphosphatidylinositol species H7 to form species H8. Allelic variants of this gene have been associated with intellectual disability, hypotonia, and early-onset seizures. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016]', '4', 'protein_coding', 10454635, 10489949, 46.21, 6, 'ENSG00000174227', 'Q5H8A4'),
(2534, 'PIGQ', 'This gene is involved in the first step in glycosylphosphatidylinositol (GPI)-anchor biosynthesis. The GPI-anchor is a glycolipid found on many blood cells and serves to anchor proteins to the cell surface. This gene encodes a N-acetylglucosaminyl transferase component that is part of the complex that catalyzes transfer of N-acetylglucosamine (GlcNAc) from UDP-GlcNAc to phosphatidylinositol (PI). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jun 2012]', '16', 'protein_coding', 10722915, 10731129, 47.49, 5, 'ENSG00000007541', 'Q9BRB3'),
(2535, 'PIK3AP1', 'Predicted to enable phosphatidylinositol 3-kinase regulatory subunit binding activity and signaling receptor binding activity. Predicted to be involved in positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction; regulation of inflammatory response; and regulation of innate immune response. Predicted to be located in membrane. Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 33657087, 33674463, 59.54, 22, 'ENSG00000155629', 'Q6ZUJ8'),
(2536, 'PIK3C3', 'Enables 1-phosphatidylinositol-3-kinase activity. Involved in several processes, including early endosome to late endosome transport; macroautophagy; and phosphatidylinositol-3-phosphate biosynthetic process. Acts upstream of or within autophagy and positive regulation of protein lipidation. Located in autolysosome; late endosome; and midbody. Part of phosphatidylinositol 3-kinase complex, class III. [provided by Alliance of Genome Resources, May 2025]', '18', 'miRNA', 50014598, 50014683, 55.81, 1, 'ENSG00000078142', 'Q8NEB9'),
(2537, 'PIK3IP1', 'Enables phosphatidylinositol 3-kinase inhibitor activity. Involved in negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction. Predicted to be located in plasma membrane. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 176065841, 176182818, 34.28, 8, 'ENSG00000100100', 'Q96FE7'),
(2538, 'PIN1', 'Peptidyl-prolyl cis/trans isomerases (PPIases) catalyze the cis/trans isomerization of peptidyl-prolyl peptide bonds. This gene encodes one of the PPIases, which specifically binds to phosphorylated ser/thr-pro motifs to catalytically regulate the post-phosphorylation conformation of its substrates. The conformational regulation catalyzed by this PPIase has a profound impact on key proteins involved in the regulation of cell growth, genotoxic and other stress responses, the immune response, induction and maintenance of pluripotency, germ cell development, neuronal differentiation, and survival. This enzyme also plays a key role in the pathogenesis of Alzheimer\'s disease and many cancers. Multiple alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Jun 2011]', '19', 'protein_coding', 175421, 176597, 48.94, 1, 'ENSG00000127445', 'Q13526'),
(2539, 'PIP4K2A', 'Phosphatidylinositol-5,4-bisphosphate, the precursor to second messengers of the phosphoinositide signal transduction pathways, is thought to be involved in the regulation of secretion, cell proliferation, differentiation, and motility. The protein encoded by this gene is one of a family of enzymes capable of catalyzing the phosphorylation of phosphatidylinositol-5-phosphate on the fourth hydroxyl of the myo-inositol ring to form phosphatidylinositol-5,4-bisphosphate. The amino acid sequence of this enzyme does not show homology to other kinases, but the recombinant protein does exhibit kinase activity. This gene is a member of the phosphatidylinositol-5-phosphate 4-kinase family. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 10931943, 10940153, 37.54, 6, 'ENSG00000150867', 'P48426'),
(2540, 'PIP4K2B', 'The protein encoded by this gene catalyzes the phosphorylation of phosphatidylinositol-5-phosphate on the fourth hydroxyl of the myo-inositol ring to form phosphatidylinositol-5,4-bisphosphate. This gene is a member of the phosphatidylinositol-5-phosphate 4-kinase family. The encoded protein sequence does not show similarity to other kinases, but the protein does exhibit kinase activity. Additionally, the encoded protein interacts with p55 TNF receptor. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 158856558, 158956747, 46.03, 5, 'ENSG00000277292', 'P78356'),
(2541, 'PIP4K2B', 'The protein encoded by this gene catalyzes the phosphorylation of phosphatidylinositol-5-phosphate on the fourth hydroxyl of the myo-inositol ring to form phosphatidylinositol-5,4-bisphosphate. This gene is a member of the phosphatidylinositol-5-phosphate 4-kinase family. The encoded protein sequence does not show similarity to other kinases, but the protein does exhibit kinase activity. Additionally, the encoded protein interacts with p55 TNF receptor. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 158856558, 158956747, 46.03, 5, 'ENSG00000276293', 'P78356'),
(2542, 'PIP4K2C', 'Enables 1-phosphatidylinositol-4-phosphate 5-kinase activity and identical protein binding activity. Involved in 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate biosynthetic process; negative regulation of insulin receptor signaling pathway; and positive regulation of autophagosome assembly. Located in several cellular components, including autophagosome; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 67619920, 67651708, 42.07, 4, 'ENSG00000166908', 'Q8TBX8'),
(2543, 'PIP4P1', 'TMEM55B catalyzes the degradation of phosphatidylinositol 4,5-bisphosphate (PtdIns-4,5-P2) by removing the 4-phosphate (Ungewickell et al., 2005 [PubMed 16365287]).[supplied by OMIM, Mar 2008]', '14', 'protein_coding', 96364608, 96513926, 46.43, 3, 'ENSG00000291802', 'Q86T03'),
(2544, 'PIP4P1', 'TMEM55B catalyzes the degradation of phosphatidylinositol 4,5-bisphosphate (PtdIns-4,5-P2) by removing the 4-phosphate (Ungewickell et al., 2005 [PubMed 16365287]).[supplied by OMIM, Mar 2008]', '14', 'protein_coding', 96364608, 96513926, 46.43, 3, 'ENSG00000165782', 'Q86T03'),
(2545, 'PIR', 'This gene encodes a member of the cupin superfamily. The encoded protein is an Fe(II)-containing nuclear protein expressed in all tissues of the body and concentrated within dot-like subnuclear structures. Interactions with nuclear factor I/CCAAT box transcription factor as well as B cell lymphoma 3-encoded oncoprotein suggest the encoded protein may act as a transcriptional cofactor and be involved in the regulation of DNA transcription and replication. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 85277396, 85379717, 35.73, 2, 'ENSG00000087842', 'O00625'),
(2546, 'PIWIL1', 'This gene encodes a member of the PIWI subfamily of Argonaute proteins, evolutionarily conserved proteins containing both PAZ and Piwi motifs that play important roles in stem cell self-renewal, RNA silencing, and translational regulation in diverse organisms. The encoded protein may play a role as an intrinsic regulator of the self-renewal capacity of germline and hematopoietic stem cells. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010]', '12', 'protein_coding', 3401215, 3409364, 46.54, 5, 'ENSG00000275051', 'Q96J94'),
(2547, 'PIWIL1', 'This gene encodes a member of the PIWI subfamily of Argonaute proteins, evolutionarily conserved proteins containing both PAZ and Piwi motifs that play important roles in stem cell self-renewal, RNA silencing, and translational regulation in diverse organisms. The encoded protein may play a role as an intrinsic regulator of the self-renewal capacity of germline and hematopoietic stem cells. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2010]', '12', 'protein_coding', 3401215, 3409364, 46.54, 5, 'ENSG00000125207', 'Q96J94'),
(2548, 'PJA2', 'Enables protein kinase A catalytic subunit binding activity; protein kinase A regulatory subunit binding activity; and ubiquitin protein ligase activity. Involved in several processes, including proteasome-mediated ubiquitin-dependent protein catabolic process; regulation of macrophage activation; and regulation of signal transduction. Located in centriolar satellite; cytoplasm; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 21238874, 21303083, 43.21, 10, 'ENSG00000198961', 'O43164'),
(2549, 'PKIG', 'This gene encodes a member of the protein kinase inhibitor family. Studies of a similar protein in mice suggest that this protein acts as a potent competitive cAMP-dependent protein kinase inhibitor, and is a predominant form of inhibitor in various tissues. The encoded protein may be involved in osteogenesis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 8013567, 8064396, 39.98, 6, 'ENSG00000168734', 'Q9Y2B9'),
(2550, 'PKN3', 'Predicted to enable protein serine/threonine kinase activity. Involved in epithelial cell migration. Predicted to be located in Golgi apparatus; cytosol; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 32384603, 32454769, 41.71, 3, 'ENSG00000160447', 'Q6P5Z2'),
(2551, 'PKNOX2', 'Homeodomain proteins are sequence-specific transcription factors that share a highly conserved DNA-binding domain and play fundamental roles in cell proliferation, differentiation, and death. PKNOX2 belongs to the TALE (3-amino acid loop extension) class of homeodomain proteins characterized by a 3-amino acid extension between alpha helices 1 and 2 within the homeodomain (Imoto et al., 2001 [PubMed 11549286]).[supplied by OMIM, Oct 2009]', '11', 'protein_coding', 5491964, 5493999, 43.27, 1, 'ENSG00000165495', 'Q96KN3'),
(2552, 'PLA1A', 'The protein encoded by this gene is a phospholipase that hydrolyzes fatty acids at the sn-1 position of phosphatidylserine and 1-acyl-2-lysophosphatidylserine. This secreted protein hydrolyzes phosphatidylserine in liposomes. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2011]', '3', 'protein_coding', 124822386, 124929983, 34.37, 31, 'ENSG00000144837', 'Q53H76'),
(2553, 'PLA2G3', 'This gene encodes a protein that belongs to the secreted phospholipase A2 family, whose members include the bee venom enzyme. The encoded enzyme functions in lipid metabolism and catalyzes the calcium-dependent hydrolysis of the sn-2 acyl bond of phospholipids to release arachidonic acid and lysophospholipids. This enzyme acts as a negative regulator of ciliogenesis, and may play a role in cancer development by stimulating tumor cell growth and angiogenesis. This gene is associated with oxidative stress, and polymorphisms in this gene are linked to risk for Alzheimer\'s disease. [provided by RefSeq, Apr 2014]', '22', 'protein_coding', 49659569, 49665875, 57.17, 31, 'ENSG00000100078', 'Q9NZ20'),
(2554, 'PLA2G5', 'This gene is a member of the secretory phospholipase A2 family. It is located in a tightly-linked cluster of secretory phospholipase A2 genes on chromosome 1. The encoded enzyme catalyzes the hydrolysis of membrane phospholipids to generate lysophospholipids and free fatty acids including arachidonic acid. It preferentially hydrolyzes linoleoyl-containing phosphatidylcholine substrates. Secretion of this enzyme is thought to induce inflammatory responses in neighboring cells. Alternatively spliced transcript variants have been found, but their full-length nature has not been determined. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 30983718, 30989903, 50.08, 2, 'ENSG00000127472', 'P39877'),
(2555, 'PLA2G6', 'The protein encoded by this gene is an A2 phospholipase, a class of enzyme that catalyzes the release of fatty acids from phospholipids. The encoded protein may play a role in phospholipid remodelling, arachidonic acid release, leukotriene and prostaglandin synthesis, fas-mediated apoptosis, and transmembrane ion flux in glucose-stimulated B-cells. Several transcript variants encoding multiple isoforms have been described, but the full-length nature of only three of them have been determined to date. [provided by RefSeq, Dec 2010]', '22', 'protein_coding', 49806866, 49817376, 52.28, 18, 'ENSG00000184381', 'O60733'),
(2556, 'PLAA', 'Predicted to enable ubiquitin binding activity. Involved in cellular response to lipopolysaccharide; macroautophagy; and positive regulation of phospholipase A2 activity. Located in cytoplasm; extracellular exosome; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 19523024, 19527545, 63.73, 4, 'ENSG00000137055', 'Q9Y263'),
(2557, 'PLAAT2', 'The protein encoded by this gene has both phospholipase and acyltransferase activities and acts as a tumor suppressor. The encoded protein can hydrolyze dipalmitoylated phosphatidylcholine (PC) to palmitic acid and lyso-PC. In addition, this protein can catalyze the N-acylation of phosphatidylethanolamine and can catalyze the O-acylation of lyso-PC to form PC. [provided by RefSeq, Jul 2016]', '11', 'protein_coding', 1831, 18625, 47.11, 2, 'ENSG00000133328', 'Q9NWW9'),
(2558, 'PLAC1', 'Involved in placenta development. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 2486414, 2500976, 49.62, 6, 'ENSG00000170965', 'Q9HBJ0'),
(2559, 'PLAU', 'This gene encodes a secreted serine protease that converts plasminogen to plasmin. The encoded preproprotein is proteolytically processed to generate A and B polypeptide chains. These chains associate via a single disulfide bond to form the catalytically inactive high molecular weight urokinase-type plasminogen activator (HMW-uPA). HMW-uPA can be further processed into the catalytically active low molecular weight urokinase-type plasminogen activator (LMW-uPA). This low molecular weight form does not bind to the urokinase-type plasminogen activator receptor. Mutations in this gene may be associated with Quebec platelet disorder and late-onset Alzheimer\'s disease. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]', '10', 'protein_coding', 42176539, 42335877, 38.57, 9, 'ENSG00000122861', 'P00749'),
(2560, 'PLCB1', 'The protein encoded by this gene catalyzes the formation of inositol 1,4,5-trisphosphate and diacylglycerol from phosphatidylinositol 4,5-bisphosphate. This reaction uses calcium as a cofactor and plays an important role in the intracellular transduction of many extracellular signals. This gene is activated by two G-protein alpha subunits, alpha-q and alpha-11. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 195575977, 195737702, 38.48, 9, 'ENSG00000182621', 'Q9NQ66'),
(2561, 'PLCG1', 'The protein encoded by this gene catalyzes the formation of inositol 1,4,5-trisphosphate and diacylglycerol from phosphatidylinositol 4,5-bisphosphate. This reaction uses calcium as a cofactor and plays an important role in the intracellular transduction of receptor-mediated tyrosine kinase activators. For example, when activated by SRC, the encoded protein causes the Ras guanine nucleotide exchange factor RasGRP1 to translocate to the Golgi, where it activates Ras. Also, this protein has been shown to be a major substrate for heparin-binding growth factor 1 (acidic fibroblast growth factor)-activated tyrosine kinase. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 128809700, 128875224, 44.20, 14, 'ENSG00000124181', 'P19174'),
(2562, 'PLCH2', 'PLCH2 is a member of the PLC-eta family of the phosphoinositide-specific phospholipase C (PLC) superfamily of enzymes that cleave PtdIns(4,5) P2 to generate second messengers inositol 1,4,5-trisphosphate and diacylglycerol (Zhou et al., 2005 [PubMed 16107206]).[supplied by OMIM, Jun 2009]', '1', 'protein_coding', 92574105, 92655395, 40.82, 5, 'ENSG00000149527', 'O75038'),
(2563, 'PLCH2', 'PLCH2 is a member of the PLC-eta family of the phosphoinositide-specific phospholipase C (PLC) superfamily of enzymes that cleave PtdIns(4,5) P2 to generate second messengers inositol 1,4,5-trisphosphate and diacylglycerol (Zhou et al., 2005 [PubMed 16107206]).[supplied by OMIM, Jun 2009]', '1', 'protein_coding', 92574105, 92655395, 40.82, 5, 'ENSG00000276429', 'O75038'),
(2564, 'PLCXD1', 'This gene is the most terminal protein-coding gene in the pseudoautosomal (PAR) region on chromosomes X and Y. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Mar 2010]', 'X, Y', 'protein_coding', 24763069, 24813492, 36.06, 5, 'ENSG00000292344', 'Q9NUJ7'),
(2565, 'PLCXD1', 'This gene is the most terminal protein-coding gene in the pseudoautosomal (PAR) region on chromosomes X and Y. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Mar 2010]', 'X, Y', 'protein_coding', 24763069, 24813492, 36.06, 5, 'ENSG00000182378', 'Q9NUJ7'),
(2566, 'PLEKHA1', 'This gene encodes a pleckstrin homology domain-containing adapter protein. The encoded protein is localized to the plasma membrane where it specifically binds phosphatidylinositol 3,4-bisphosphate. This protein may be involved in the formation of signaling complexes in the plasma membrane. Polymorphisms in this gene are associated with age-related macular degeneration. Alternate splicing results in multiple transcript variants. A pseudogene of this gene is found on chromosome 5.[provided by RefSeq, Sep 2010]', '10', 'protein_coding', 56819573, 56820516, 41.53, 1, 'ENSG00000107679', 'Q9HB21');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2567, 'PLEKHG2', 'The protein encoded by this gene is a RhoGTPase that can activate CDC42 by promoting exchange of GDP for GTP on CDC42. The encoded protein is activated by binding to the beta and gamma subunits of heterotrimeric guanine nucleotide-binding protein. Defects in this gene have been associated with leukodystrophy and acquired microcephaly with or without dystonia. [provided by RefSeq, May 2017]', '19', 'protein_coding', 68827531, 68895432, 45.18, 5, 'ENSG00000090924', 'Q9H7P9'),
(2568, 'PLEKHM3', 'Predicted to enable zinc ion binding activity. Predicted to be involved in myoblast differentiation. Predicted to be located in Golgi apparatus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 22169589, 22202528, 44.68, 2, 'ENSG00000178385', 'Q6ZWE6'),
(2569, 'PLOD3', 'The protein encoded by this gene is a membrane-bound homodimeric enzyme that is localized to the cisternae of the rough endoplasmic reticulum. The enzyme (cofactors iron and ascorbate) catalyzes the hydroxylation of lysyl residues in collagen-like peptides. The resultant hydroxylysyl groups are attachment sites for carbohydrates in collagen and thus are critical for the stability of intermolecular crosslinks. Some patients with Ehlers-Danlos syndrome type VIB have deficiencies in lysyl hydroxylase activity. [provided by RefSeq, Jul 2008]', '7', 'miRNA', 104218789, 104218883, 48.42, 1, 'ENSG00000106397', 'O60568'),
(2570, 'PLP2', 'This gene encodes an integral membrane protein that localizes to the endoplasmic reticulum in colonic epithelial cells. The encoded protein can multimerize and may function as an ion channel. A polymorphism in the promoter of this gene may be linked to an increased risk of X-linked cognitive disability. A pseudogene of this gene is found on chromosome 5. [provided by RefSeq, Jan 2010]', 'X', 'protein_coding', 112070663, 112087529, 34.63, 11, 'ENSG00000102007', 'Q04941'),
(2571, 'PLPP7', 'Predicted to enable sphingosine-1-phosphate phosphatase activity. Predicted to act upstream of or within negative regulation of myotube differentiation. Predicted to be located in endoplasmic reticulum membrane and nucleus. Predicted to be active in membrane and nuclear envelope. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 17178790, 17258235, 48.88, 30, 'ENSG00000160539', 'Q8NBV4'),
(2572, 'PMEPA1', 'This gene encodes a transmembrane protein that contains a Smad interacting motif (SIM). Expression of this gene is induced by androgens and transforming growth factor beta, and the encoded protein suppresses the androgen receptor and transforming growth factor beta signaling pathways though interactions with Smad proteins. Overexpression of this gene may play a role in multiple types of cancer. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Dec 2011]', '20', 'protein_coding', 7750962, 7787993, 39.94, 5, 'ENSG00000124225', 'Q969W9'),
(2573, 'PNKP', 'This locus represents a gene involved in DNA repair. In response to ionizing radiation or oxidative damage, the protein encoded by this locus catalyzes 5\' phosphorylation and 3\' dephosphorylation of nucleic acids. Mutations at this locus have been associated with microcephaly, seizures, and developmental delay.[provided by RefSeq, Sep 2010]', '19', 'protein_coding', 47649476, 47685505, 43.15, 16, 'ENSG00000039650', 'Q96T60'),
(2574, 'PNLDC1', 'Enables poly(A)-specific ribonuclease activity. Involved in nuclear-transcribed mRNA poly(A) tail shortening; piRNA processing; and spermatogenesis. Located in endoplasmic reticulum. Implicated in spermatogenic failure 57. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 148588, 149289, 48.86, 1, 'ENSG00000146453', 'Q8NA58'),
(2575, 'PNLIPRP3', 'Predicted to enable lipoprotein lipase activity and phospholipase A1 activity. Predicted to be involved in several processes, including cholesterol homeostasis; high-density lipoprotein particle remodeling; and triglyceride catabolic process. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '10', 'lncRNA', 133534, 159776, 46.87, 3, 'ENSG00000203837', 'Q17RR3'),
(2576, 'PNN', 'Enables RNA binding activity. Predicted to be involved in cell adhesion and mRNA splicing, via spliceosome. Predicted to act upstream of or within cell-cell adhesion. Located in exon-exon junction complex and nuclear speck. Part of catalytic step 2 spliceosome. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 15604381, 15681679, 37.86, 13, 'ENSG00000100941', 'Q9H307'),
(2577, 'PNPLA3', 'The protein encoded by this gene is a triacylglycerol lipase that mediates triacylglycerol hydrolysis in adipocytes. The encoded protein, which appears to be membrane bound, may be involved in the balance of energy usage/storage in adipocytes. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 20728731, 20773961, 41.53, 3, 'ENSG00000100344', 'Q9NST1'),
(2578, 'PNPLA5', 'This gene is a member of the patatin-like phospholipase family; its encoded protein has been shown to inhibit transacylation. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Apr 2010]', '22', 'protein_coding', 39377698, 39428528, 44.40, 23, 'ENSG00000100341', 'Q7Z6Z6'),
(2579, 'PODNL1', 'An extracellular matrix structural constituent. Is active in collagen-containing extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 32733899, 32771792, 38.11, 9, 'ENSG00000288195', 'Q6PEZ8'),
(2580, 'PODNL1', 'An extracellular matrix structural constituent. Is active in collagen-containing extracellular matrix. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 32733899, 32771792, 38.11, 9, 'ENSG00000132000', 'Q6PEZ8'),
(2581, 'POF1B', 'Premature ovarian failure (POF) is characterized by primary or secondary amenorrhea in women less than 40 years old. Two POF susceptibility regions called \"POF1\" and \"POF2\" have been identified by breakpoint mapping of X-autosome translocations. POF1 extends from Xq21-qter while POF2 extends from Xq13.3 to Xq21.1. This gene, POF1B, resides in the POF2 region. This gene is expressed at trace levels in mouse prenatal ovary and is barely detectable or absent from adult ovary, in human and in the mouse respectively. This gene\'s expression is restricted to epithelia with its highest expression in the epidermis, and oro-pharyngeal and gastro-intestinal tracts. The protein encoded by this gene binds non-muscle actin filaments. The role this gene may play in the etiology of premature ovarian failure remains to be determined. [provided by RefSeq, Jan 2010]', 'X', 'protein_coding', 106900063, 106903341, 51.27, 2, 'ENSG00000124429', 'Q8WVV4'),
(2582, 'POGLUT1', 'This gene encodes a protein with both O-glucosyltransferase and O-xylosyltransferase activity which localizes to the lumen of the endoplasmic reticulum. This protein has a carboxy-terminal KTEL motif which is predicted to function as an endoplasmic reticulum retention signal. This gene is an essential regulator of Notch signalling and likely plays a role in cell fate and tissue formation during development. It may also play a role in the pathogenesis of leukemia. Mutations in this gene have been associated with the autosomal dominant genodermatosis Dowling-Degos disease 4. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]', '3', 'protein_coding', 34254552, 34355566, 44.94, 7, 'ENSG00000163389', 'Q8NBL1'),
(2583, 'POLA1', 'This gene encodes the catalytic subunit of DNA polymerase, which together with a regulatory and two primase subunits, forms the DNA polymerase alpha complex. The catalytic subunit plays an essential role in the initiation of DNA replication. [provided by RefSeq, Mar 2010]', 'X', 'miRNA', 8186945, 8187004, 41.67, 1, 'ENSG00000101868', 'P09884'),
(2584, 'POLR1C', 'The protein encoded by this gene is a subunit of both RNA polymerase I and RNA polymerase III complexes. The encoded protein is part of the Pol core element. Mutations in this gene have been associated with Treacher Collins syndrome (TCS) and hypomyelinating leukodystrophy 11. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', '6', 'protein_coding', 6772512, 6778455, 57.37, 4, 'ENSG00000171453', 'O15160'),
(2585, 'POLR3F', 'The protein encoded by this gene is one of more than a dozen subunits forming eukaryotic RNA polymerase III (RNA Pol III), which transcribes 5S ribosomal RNA and tRNA genes. This protein has been shown to bind both TFIIIB90 and TBP, two subunits of RNA polymerase III transcription initiation factor IIIB (TFIIIB). Unlike most of the other RNA Pol III subunits, the encoded protein is unique to this polymerase. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]', '20', 'protein_coding', 36605779, 36612557, 49.02, 7, 'ENSG00000132664', 'Q9H1D9'),
(2586, 'POMP', 'This gene encodes a member of the 5\'-nucleotidase family of enzymes that catalyze the dephosphorylation of nucleoside 5\'-monophosphates. The encoded protein is the type 1 isozyme of pyrimidine 5\' nucleotidase and catalyzes the dephosphorylation of pyrimidine 5\' monophosphates. Mutations in this gene are a cause of hemolytic anemia due to uridine 5-prime monophosphate hydrolase deficiency. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and pseudogenes of this gene are located on the long arm of chromosomes 3 and 4. [provided by RefSeq, Mar 2012]', '7', 'protein_coding', 86026146, 86100173, 45.43, 8, 'ENSG00000132963', 'Q9Y244'),
(2587, 'POMZP3', 'This gene appears to have resulted from a fusion of DNA sequences derived from 2 distinct loci, specifically through the duplication of two internal exons from the POM121 gene and four 3\' exons from the ZP3 gene. The 5\' end of this gene is similar to the 5` coding region of the POM121 gene which encodes an integral nuclear pore membrane protein. However, the protein encoded by this gene lacks the nuclear pore localization motif. The 3\' end of this gene is similar to the last 4 exons of the zona pellucida glycoprotein 3 (ZP3) gene and the encoded protein retains one zona pellucida domain. Multiple protein isoforms are encoded by transcript variants of this gene. [provided by RefSeq, Jul 2008]', '7', 'unprocessed_pseudogene', 59531754, 59532695, 50.42, 1, 'ENSG00000146707', 'Q6PJE2'),
(2588, 'POMZP3', 'This gene appears to have resulted from a fusion of DNA sequences derived from 2 distinct loci, specifically through the duplication of two internal exons from the POM121 gene and four 3\' exons from the ZP3 gene. The 5\' end of this gene is similar to the 5` coding region of the POM121 gene which encodes an integral nuclear pore membrane protein. However, the protein encoded by this gene lacks the nuclear pore localization motif. The 3\' end of this gene is similar to the last 4 exons of the zona pellucida glycoprotein 3 (ZP3) gene and the encoded protein retains one zona pellucida domain. Multiple protein isoforms are encoded by transcript variants of this gene. [provided by RefSeq, Jul 2008]', '7', 'unprocessed_pseudogene', 59531754, 59532695, 50.42, 1, 'ENSG00000276464', 'Q6PJE2'),
(2589, 'POP5', 'Enables ribonuclease P RNA binding activity. Contributes to ribonuclease P activity. Involved in tRNA 5\'-leader removal. Located in nucleolus. Part of multimeric ribonuclease P complex and ribonuclease MRP complex. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 43371636, 43376335, 54.00, 4, 'ENSG00000167272', 'Q969H6'),
(2590, 'POP7', 'Enables ribonuclease P RNA binding activity. Contributes to ribonuclease P activity. Involved in tRNA 5\'-leader removal. Located in nucleolus. Part of multimeric ribonuclease P complex and ribonuclease MRP complex. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 87160103, 87240314, 38.23, 8, 'ENSG00000172336', 'O75817'),
(2591, 'POSTN', 'This gene encodes a secreted extracellular matrix protein that functions in tissue development and regeneration, including wound healing, and ventricular remodeling following myocardial infarction. The encoded protein binds to integrins to support adhesion and migration of epithelial cells. This protein plays a role in cancer stem cell maintenance and metastasis. Mice lacking this gene exhibit cardiac valve disease, and skeletal and dental defects. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Sep 2015]', '13', 'protein_coding', 170964, 172037, 40.41, 1, 'ENSG00000133110', 'Q15063'),
(2592, 'POT1', 'This gene is a member of the telombin family and encodes a nuclear protein involved in telomere maintenance. Specifically, this protein functions as a member of a multi-protein complex that binds to the TTAGGG repeats of telomeres, regulating telomere length and protecting chromosome ends from illegitimate recombination, catastrophic chromosome instability, and abnormal chromosome segregation. Increased transcriptional expression of this gene is associated with stomach carcinogenesis and its progression. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 65014160, 65022184, 55.28, 6, 'ENSG00000128513', 'Q9NUX5'),
(2593, 'POU2F1', 'The OCT1 transcription factor was among the first identified members of the POU transcription factor family (summarized by Sturm et al., 1993 [PubMed 8314572]). Members of this family contain the POU domain, a 160-amino acid region necessary for DNA binding to the octameric sequence ATGCAAAT.[supplied by OMIM, Jul 2010]', '1', 'protein_coding', 113255835, 113275572, 43.06, 2, 'ENSG00000143190', 'P14859'),
(2594, 'POU2F3', 'This gene encodes a member of the POU domain family of transcription factors. POU domain transcription factors bind to a specific octamer DNA motif and regulate cell type-specific differentiation pathways. The encoded protein is primarily expressed in the epidermis, and plays a critical role in keratinocyte proliferation and differentiation. The encoded protein is also a candidate tumor suppressor protein, and aberrant promoter methylation of this gene may play a role in cervical cancer. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Sep 2011]', '11', 'protein_coding', 112074086, 112085150, 40.81, 9, 'ENSG00000137709', 'Q9UKI9'),
(2595, 'POU4F1', 'This gene encodes a member of the POU-IV class of neural transcription factors. This protein is expressed in a subset of retinal ganglion cells and may be involved in the developing sensory nervous system. This protein may also promote the growth of cervical tumors. A translocation of this gene is associated with some adult acute myeloid leukemias. [provided by RefSeq, Mar 2012]', '13', 'protein_coding', 43719094, 43762307, 57.26, 19, 'ENSG00000152192', 'Q01851'),
(2596, 'PP2D1', 'Predicted to enable protein serine/threonine phosphatase activity. Predicted to be involved in negative regulation of MAPK cascade and signal transduction. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'miRNA', 92490163, 92490220, 34.48, 1, 'ENSG00000183977', 'A8MPX8'),
(2597, 'PPA1', 'The protein encoded by this gene is a member of the inorganic pyrophosphatase (PPase) family. PPases catalyze the hydrolysis of pyrophosphate to inorganic phosphate, which is important for the phosphate metabolism of cells. Studies of a similar protein in bovine suggested a cytoplasmic localization of this enzyme. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 94402541, 94536332, 38.52, 3, 'ENSG00000180817', 'Q15181'),
(2598, 'PPAN', 'The protein encoded by this gene is an evolutionarily conserved protein similar to yeast SSF1 as well as to the gene product of the Drosophila gene peter pan (ppan). SSF1 is known to be involved in the second step of mRNA splicing. Both SSF1 and ppan are essential for cell growth and proliferation. Exogenous expression of this gene was reported to reduce the anchorage-independent growth of some tumor cells. Read-through transcription of this gene with P2RY11/P2Y(11), an adjacent downstream gene that encodes an ATP receptor, has been found. These read-through transcripts are ubiquitously present and up-regulated during granulocyte differentiation. [provided by RefSeq, Nov 2010]', '19', 'protein_coding', 31064105, 31104757, 54.79, 24, 'ENSG00000130810', 'Q9NQ55'),
(2599, 'PPFIA1', 'The protein encoded by this gene is a member of the LAR protein-tyrosine phosphatase-interacting protein (liprin) family. Liprins interact with members of LAR family of transmembrane protein tyrosine phosphatases, which are known to be important for axon guidance and mammary gland development. This protein binds to the intracellular membrane-distal phosphatase domain of tyrosine phosphatase LAR, and appears to localize LAR to cell focal adhesions. This interaction may regulate the disassembly of focal adhesion and thus help orchestrate cell-matrix interactions. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 44285838, 44298648, 63.87, 5, 'ENSG00000288198', 'Q13136'),
(2600, 'PPFIA1', 'The protein encoded by this gene is a member of the LAR protein-tyrosine phosphatase-interacting protein (liprin) family. Liprins interact with members of LAR family of transmembrane protein tyrosine phosphatases, which are known to be important for axon guidance and mammary gland development. This protein binds to the intracellular membrane-distal phosphatase domain of tyrosine phosphatase LAR, and appears to localize LAR to cell focal adhesions. This interaction may regulate the disassembly of focal adhesion and thus help orchestrate cell-matrix interactions. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 44285838, 44298648, 63.87, 5, 'ENSG00000131626', 'Q13136'),
(2601, 'PPFIBP1', 'The protein encoded by this gene is a member of the LAR protein-tyrosine phosphatase-interacting protein (liprin) family. Liprins interact with members of LAR family of transmembrane protein tyrosine phosphatases, which are known to be important for axon guidance and mammary gland development. It has been proposed that liprins are multivalent proteins that form complex structures and act as scaffolds for the recruitment and anchoring of LAR family of tyrosine phosphatases. This protein was found to interact with S100A4, a calcium-binding protein related to tumor invasiveness and metastasis. In vitro experiment demonstrated that the interaction inhibited the phosphorylation of this protein by protein kinase C and protein kinase CK2. Alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 113297093, 113303376, 49.82, 7, 'ENSG00000110841', 'Q86W92'),
(2602, 'PPIAL4A', 'Predicted to enable cyclosporin A binding activity and peptidyl-prolyl cis-trans isomerase activity. Predicted to be involved in protein folding. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'snoRNA', 119787353, 119787484, 36.36, 1, 'ENSG00000291978', 'Q9Y536'),
(2603, 'PPIAL4A', 'Predicted to enable cyclosporin A binding activity and peptidyl-prolyl cis-trans isomerase activity. Predicted to be involved in protein folding. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'snoRNA', 119787353, 119787484, 36.36, 1, 'ENSG00000263353', 'Q9Y536'),
(2604, 'PPIL2', 'This gene is a member of the cyclophilin family of peptidylprolyl isomerases. The cyclophilins are a highly conserved ubiquitous family, members of which play an important role in protein folding, immunosuppression by cyclosporin A, and infection of HIV-1 virions. This protein interacts with the proteinase inhibitor eglin c and is localized in the nucleus. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Dec 2015]', '22', 'protein_coding', 53022, 224537, 40.80, 6, 'ENSG00000100023', 'Q13356'),
(2605, 'PPM1A', 'The protein encoded by this gene is a member of the PP2C family of Ser/Thr protein phosphatases. PP2C family members are known to be negative regulators of cell stress response pathways. This phosphatase dephosphorylates, and negatively regulates the activities of, MAP kinases and MAP kinase kinases. It has been shown to inhibit the activation of p38 and JNK kinase cascades induced by environmental stresses. This phosphatase can also dephosphorylate cyclin-dependent kinases, and thus may be involved in cell cycle control. Overexpression of this phosphatase is reported to activate the expression of the tumor suppressor gene TP53/p53, which leads to G2/M cell cycle arrest and apoptosis. Three alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Jul 2008]', '14', 'miRNA', 35453954, 35454097, 41.67, 1, 'ENSG00000100614', 'P35813'),
(2606, 'PPM1H', 'Enables identical protein binding activity and phosphoprotein phosphatase activity. Predicted to be involved in signal transduction. Located in cytoplasm and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 18744026, 18779349, 45.41, 5, 'ENSG00000111110', 'Q9ULR3'),
(2607, 'PPM1L', 'The protein encoded by this gene is a magnesium or manganese-requiring phosphatase that is involved in several signaling pathways. The encoded protein downregulates apoptosis signal-regulating kinase 1, a protein that initiates a signaling cascade that leads to apoptosis when cells are subjected to cytotoxic stresses. This protein also is an endoplasmic reticulum transmembrane protein that helps regulate ceramide transport from the endoplasmic reticulum to the Golgi apparatus. Finally, this gene may be involved in adiposity since it is upregulated in adipose tissues. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2015]', '3', 'protein_coding', 144646, 160374, 40.87, 4, 'ENSG00000163590', 'Q5SGD2'),
(2608, 'PPP1R14B', 'Predicted to enable protein serine/threonine phosphatase inhibitor activity. Predicted to be involved in innate immune response. Predicted to be located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 143119771, 143124014, 57.61, 1, 'ENSG00000173457', 'Q96C90'),
(2609, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000146112', 'Q6NYC8'),
(2610, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000230341', 'Q6NYC8'),
(2611, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000225060', 'Q6NYC8'),
(2612, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000234000', 'Q6NYC8'),
(2613, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000231247', 'Q6NYC8'),
(2614, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000229998', 'Q6NYC8'),
(2615, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000206485', 'Q6NYC8'),
(2616, 'PPP1R18', 'Protein phosphatase-1 (PP1; see MIM 176875) interacts with regulatory subunits that target the enzyme to different cellular locations and change its activity toward specific substrates. Phostensin is a regulatory subunit that targets PP1 to F-actin (see MIM 102610) cytoskeleton (Kao et al., 2007 [PubMed 17374523]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000236428', 'Q6NYC8'),
(2617, 'PPP1R27', 'Enables phosphatase binding activity. [provided by Alliance of Genome Resources, May 2025]', '17', 'rRNA', 8395607, 8395759, 57.52, 1, 'ENSG00000288186', 'Q86WC6'),
(2618, 'PPP1R27', 'Enables phosphatase binding activity. [provided by Alliance of Genome Resources, May 2025]', '17', 'rRNA', 8395607, 8395759, 57.52, 1, 'ENSG00000182676', 'Q86WC6'),
(2619, 'PPP1R35', 'Predicted to enable phosphatase binding activity and protein phosphatase inhibitor activity. Involved in positive regulation of centriole elongation. Located in centriole and centrosome. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 55388181, 55388242, 69.35, 1, 'ENSG00000160813', 'Q8TAP8'),
(2620, 'PPP1R36', 'Predicted to enable phosphatase binding activity and protein phosphatase inhibitor activity. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 24573518, 24576250, 53.71, 2, 'ENSG00000165807', 'Q96LQ0'),
(2621, 'PPP1R3A', 'The glycogen-associated form of protein phosphatase-1 (PP1) derived from skeletal muscle is a heterodimer composed of a 37-kD catalytic subunit and a 124-kD targeting and regulatory subunit. This gene encodes the regulatory subunit which binds to muscle glycogen with high affinity, thereby enhancing dephosphorylation of glycogen-bound substrates for PP1 such as glycogen synthase and glycogen phosphorylase kinase. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 109542799, 109614143, 40.32, 14, 'ENSG00000154415', 'Q16821'),
(2622, 'PPP1R3B', 'This gene encodes the catalytic subunit of the serine/theonine phosphatase, protein phosphatase-1. The encoded protein is expressed in liver and skeletal muscle tissue and may be involved in regulating glycogen synthesis in these tissues. This gene may be a involved in type 2 diabetes and maturity-onset diabetes of the young. Alternate splicing results in multiple transcript variants that encode the same protein.[provided by RefSeq, Jan 2011]', '8', 'protein_coding', 93784227, 93814963, 39.79, 29, 'ENSG00000285343', 'Q86XI6'),
(2623, 'PPP1R3B', 'This gene encodes the catalytic subunit of the serine/theonine phosphatase, protein phosphatase-1. The encoded protein is expressed in liver and skeletal muscle tissue and may be involved in regulating glycogen synthesis in these tissues. This gene may be a involved in type 2 diabetes and maturity-onset diabetes of the young. Alternate splicing results in multiple transcript variants that encode the same protein.[provided by RefSeq, Jan 2011]', '8', 'protein_coding', 93784227, 93814963, 39.79, 29, 'ENSG00000173281', 'Q86XI6'),
(2624, 'PPP1R3D', 'Phosphorylation of serine and threonine residues in proteins is a crucial step in the regulation of many cellular functions ranging from hormonal regulation to cell division and even short-term memory. The level of phosphorylation is controlled by the opposing actions of protein kinases and protein phosphatases. Protein phosphatase 1 (PP1) is 1 of 4 major serine/threonine-specific protein phosphatases which have been identified in eukaryotic cells. PP1 associates with various regulatory subunits that dictate its subcellular localization and modulate its substrate specificity. Several subunits that target PP1 to glycogen have been identified. This gene encodes a glycogen-targeting subunit of PP1. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 35471405, 35471486, 36.59, 1, 'ENSG00000132825', 'O95685'),
(2625, 'PPP1R9A', 'This gene is imprinted, and located in a cluster of imprinted genes on chromosome 7q12. This gene is transcribed in both neuronal and multiple embryonic tissues, and it is maternally expressed mainly in embryonic skeletal muscle tissues and biallelically expressed in other embryonic tissues. The protein encoded by this gene includes a PDZ domain and a sterile alpha motif (SAM). It is a regulatory subunit of protein phosphatase I, and controls actin cytoskeleton reorganization. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2009]', '7', 'protein_coding', 110609978, 110815958, 39.87, 8, 'ENSG00000158528', 'Q9ULJ8'),
(2626, 'PPP2R1B', 'This gene encodes a constant regulatory subunit of protein phosphatase 2. Protein phosphatase 2 is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The constant regulatory subunit A serves as a scaffolding molecule to coordinate the assembly of the catalytic subunit and a variable regulatory B subunit. This gene encodes a beta isoform of the constant regulatory subunit A. Mutations in this gene have been associated with some lung and colon cancers. Alternatively spliced transcript variants have been described. [provided by RefSeq, Apr 2010]', '11', 'snoRNA', 100975825, 100975896, 37.50, 1, 'ENSG00000137713', 'P30154'),
(2627, 'PPP2R2D', 'Predicted to enable enzyme-substrate adaptor activity and protein phosphatase regulator activity. Predicted to be involved in exit from mitosis and regulation of chromosome segregation. Predicted to be located in cytoplasm. Predicted to be part of protein phosphatase type 2A complex. Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 44939249, 44989471, 38.71, 1, 'ENSG00000175470', 'Q66LE6'),
(2628, 'PPP2R5D', 'The product of this gene belongs to the phosphatase 2A regulatory subunit B family. Protein phosphatase 2A is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The B regulatory subunit might modulate substrate selectivity and catalytic activity. This gene encodes a delta isoform of the regulatory subunit B56 subfamily. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 133471094, 133479127, 61.25, 5, 'ENSG00000112640', 'Q14738'),
(2629, 'PPP3CA', 'Enables several functions, including ATPase binding activity; calmodulin binding activity; and calmodulin-dependent protein phosphatase activity. Involved in several processes, including calcineurin-NFAT signaling cascade; negative regulation of angiotensin-activated signaling pathway; and peptidyl-serine dephosphorylation. Located in cytoplasm; cytoplasmic side of plasma membrane; and dendritic spine. Part of calcineurin complex. Implicated in developmental and epileptic encephalopathy 91. Biomarker of cholangiocarcinoma; focal segmental glomerulosclerosis; and schizophrenia. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 160421855, 160428739, 39.32, 7, 'ENSG00000138814', 'Q08209'),
(2630, 'PPP4R3B', 'Predicted to enable protein phosphatase activator activity. Involved in regulation of double-strand break repair. Located in centrosome; chromatin; and nuclear speck. Part of protein phosphatase 4 complex. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 103245887, 103248016, 39.58, 1, 'ENSG00000275052', 'Q5MIZ7'),
(2631, 'PPP6C', 'This gene encodes the catalytic subunit of protein phosphatase, a component of a signaling pathway regulating cell cycle progression. Splice variants encoding different protein isoforms exist. The pseudogene of this gene is located on chromosome X. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 34634722, 34637844, 57.41, 13, 'ENSG00000119414', 'O00743'),
(2632, 'PQBP1', 'This gene encodes a nuclear polyglutamine-binding protein that is involved with transcription activation. The encoded protein contains a WW domain. Mutations in this gene have been found in patients with Renpenning syndrome 1 and other syndromes with X-linked cognitive disability. Multiple alternatively spliced transcript variants that encode different protein isoforms have been described for this gene.[provided by RefSeq, Nov 2009]', 'X', 'protein_coding', 102709, 116870, 48.19, 3, 'ENSG00000102103', 'O60828'),
(2633, 'PQBP1', 'This gene encodes a nuclear polyglutamine-binding protein that is involved with transcription activation. The encoded protein contains a WW domain. Mutations in this gene have been found in patients with Renpenning syndrome 1 and other syndromes with X-linked cognitive disability. Multiple alternatively spliced transcript variants that encode different protein isoforms have been described for this gene.[provided by RefSeq, Nov 2009]', 'X', 'protein_coding', 102709, 116870, 48.19, 3, 'ENSG00000292208', 'O60828'),
(2634, 'PRAMEF10', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'miRNA', 86716, 86852, 59.85, 1, 'ENSG00000282663', 'O60809'),
(2635, 'PRAMEF10', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'miRNA', 86716, 86852, 59.85, 1, 'ENSG00000187545', 'O60809'),
(2636, 'PRAMEF11', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'lncRNA', 35721, 54769, 57.53, 1, 'ENSG00000281934', 'O60813'),
(2637, 'PRAMEF11', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'lncRNA', 35721, 54769, 57.53, 1, 'ENSG00000239810', 'O60813'),
(2638, 'PRAMEF13', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 49076, 122044, 38.54, 3, 'ENSG00000282741', 'Q5VWM6'),
(2639, 'PRAMEF13', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 49076, 122044, 38.54, 3, 'ENSG00000279169', 'Q5VWM6'),
(2640, 'PRAMEF25', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 239989, 243392, 51.91, 2, 'ENSG00000282566', 'A6NGN4'),
(2641, 'PRAMEF25', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 239989, 243392, 51.91, 2, 'ENSG00000283348', 'A6NGN4'),
(2642, 'PRAMEF25', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 239989, 243392, 51.91, 2, 'ENSG00000276299', 'A6NGN4'),
(2643, 'PRAMEF25', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 239989, 243392, 51.91, 2, 'ENSG00000229571', 'A6NGN4'),
(2644, 'PRAMEF27', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 143028, 165233, 54.14, 13, 'ENSG00000282437', 'A3QJZ7'),
(2645, 'PRAMEF27', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 143028, 165233, 54.14, 13, 'ENSG00000283425', 'A3QJZ7'),
(2646, 'PRAMEF27', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 143028, 165233, 54.14, 13, 'ENSG00000274764', 'A3QJZ7'),
(2647, 'PRAMEF33', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 276322, 303356, 53.82, 10, 'ENSG00000282125', 'A0A0G2JMD5'),
(2648, 'PRAMEF33', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 276322, 303356, 53.82, 10, 'ENSG00000283488', 'A0A0G2JMD5'),
(2649, 'PRAMEF33', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 276322, 303356, 53.82, 10, 'ENSG00000276370', 'A0A0G2JMD5'),
(2650, 'PRAMEF33', 'Predicted to enable ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in proteasome-mediated ubiquitin-dependent protein catabolic process. Predicted to be part of Cul2-RING ubiquitin ligase complex. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 276322, 303356, 53.82, 10, 'ENSG00000237700', 'A0A0G2JMD5'),
(2651, 'PRB1', 'The protein encoded by this gene is similar in sequence and possibly function to the product of the retinoblastoma 1 (RB1) gene. The RB1 gene product is a tumor suppressor protein that appears to be involved in cell cycle regulation, as it is phosphorylated in the S to M phase transition and is dephosphorylated in the G1 phase of the cell cycle. Both the RB1 protein and the product of this gene can form a complex with adenovirus E1A protein and SV40 large T-antigen, with the SV40 large T-antigen binding only to the unphosphorylated form of each protein. In addition, both proteins can inhibit the transcription of cell cycle genes containing E2F binding sites in their promoters. Due to the sequence and biochemical similarities with the RB1 protein, it is thought that the protein encoded by this gene may also be a tumor suppressor. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 77324641, 78213010, 35.89, 16, 'ENSG00000282673', 'P04280'),
(2652, 'PRB1', 'The protein encoded by this gene is similar in sequence and possibly function to the product of the retinoblastoma 1 (RB1) gene. The RB1 gene product is a tumor suppressor protein that appears to be involved in cell cycle regulation, as it is phosphorylated in the S to M phase transition and is dephosphorylated in the G1 phase of the cell cycle. Both the RB1 protein and the product of this gene can form a complex with adenovirus E1A protein and SV40 large T-antigen, with the SV40 large T-antigen binding only to the unphosphorylated form of each protein. In addition, both proteins can inhibit the transcription of cell cycle genes containing E2F binding sites in their promoters. Due to the sequence and biochemical similarities with the RB1 protein, it is thought that the protein encoded by this gene may also be a tumor suppressor. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 77324641, 78213010, 35.89, 16, 'ENSG00000275517', 'P04280'),
(2653, 'PRB1', 'The protein encoded by this gene is similar in sequence and possibly function to the product of the retinoblastoma 1 (RB1) gene. The RB1 gene product is a tumor suppressor protein that appears to be involved in cell cycle regulation, as it is phosphorylated in the S to M phase transition and is dephosphorylated in the G1 phase of the cell cycle. Both the RB1 protein and the product of this gene can form a complex with adenovirus E1A protein and SV40 large T-antigen, with the SV40 large T-antigen binding only to the unphosphorylated form of each protein. In addition, both proteins can inhibit the transcription of cell cycle genes containing E2F binding sites in their promoters. Due to the sequence and biochemical similarities with the RB1 protein, it is thought that the protein encoded by this gene may also be a tumor suppressor. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 77324641, 78213010, 35.89, 16, 'ENSG00000251655', 'P04280'),
(2654, 'PRCD', 'This gene is predominantly expressed in the retina, and mutations in this gene are the cause of autosomal recessive retinal degeneration in both humans and dogs. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Mar 2010]', '17', 'protein_coding', 559557, 562224, 67.09, 8, 'ENSG00000214140', 'Q00LT1'),
(2655, 'PRDM13', 'Predicted to enable histone methyltransferase activity. Predicted to be involved in regulation of gene expression. Predicted to act upstream of or within negative regulation of transcription by RNA polymerase II and neurogenesis. Predicted to be active in nucleus. Implicated in pontocerebellar hypoplasia. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 70133616, 70176799, 33.35, 5, 'ENSG00000112238', 'Q9H4Q3'),
(2656, 'PRDM15', 'Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific; RNA polymerase II cis-regulatory region sequence-specific DNA binding activity; and promoter-specific chromatin binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II; regulation of signal transduction; and regulation of stem cell division. Located in nuclear body. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 83877205, 83882474, 47.44, 2, 'ENSG00000141956', 'P57071'),
(2657, 'PRDM5', 'The protein encoded by this gene is a transcription factor of the PR-domain protein family. It contains a PR-domain and multiple zinc finger motifs. Transcription factors of the PR-domain family are known to be involved in cell differentiation and tumorigenesis. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 77273, 81393, 53.97, 1, 'ENSG00000138738', 'Q9NQX1'),
(2658, 'PRF1', 'This gene encodes a protein with structural similarities to complement component C9 that is important in immunity. This protein forms membrane pores that allow the release of granzymes and subsequent cytolysis of target cells. Whether pore formation occurs in the plasma membrane of target cells or in an endosomal membrane inside target cells is subject to debate. Mutations in this gene are associated with a variety of human disease including diabetes, multiple sclerosis, lymphomas, autoimmune lymphoproliferative syndrome (ALPS), aplastic anemia, and familial hemophagocytic lymphohistiocytosis type 2 (FHL2), a rare and lethal autosomal recessive disorder of early childhood. [provided by RefSeq, Aug 2017]', '10', 'protein_coding', 100879753, 100903722, 51.40, 1, 'ENSG00000180644', 'P14222'),
(2659, 'PRICKLE4', 'C6ORF49 is a member of the LIM domain protein family (Teufel et al., 2005 [PubMed 15702247]).[supplied by OMIM, Mar 2008]', '6', 'miRNA', 7147860, 7147937, 82.05, 1, 'ENSG00000278224', 'Q2TBC4'),
(2660, 'PRIMPOL', 'This gene encodes a DNA primase-polymerase that belongs to a superfamily of archaeao-eukaryotic primases. Members of this family have primase activity, catalyzing the synthesis of short RNA primers that serve as starting points for DNA synthesis, as well as DNA polymerase activity. The encoded protein facilitates DNA damage tolerance by mediating uninterrupted fork progression after UV irradiation and reinitiating DNA synthesis. An allelic variant in this gene is associated with myopia 22. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016]', '4', 'protein_coding', 143603210, 143609773, 60.13, 7, 'ENSG00000164306', 'Q96LW4'),
(2661, 'PRKD1', 'The protein encoded by this gene is a serine/threonine protein kinase involved in many cellular processes, including Golgi body membrane integrity and transport, cell migration and differentiation, MAPK8/JNK1 and Ras pathway signaling, MAPK1/3 (ERK1/2) pathway signaling, cell survival, and regulation of cell shape and adhesion. [provided by RefSeq, Jan 2017]', '14', 'protein_coding', 2059839, 2303785, 45.76, 20, 'ENSG00000184304', 'Q15139');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2662, 'PRKD3', 'This gene belongs to the multigene protein kinase D family of serine/threonine kinases, which bind diacylglycerol and phorbol esters. Members of this family are characterized by an N-terminal regulatory domain comprised of a tandem repeat of cysteine-rich zinc-finger motifs and a pleckstrin domain. The C-terminal region contains the catalytic domain and is distantly related to calcium-regulated kinases. Catalytic activity of this enzyme promotes its nuclear localization. This protein has been implicated in a variety of functions including negative regulation of human airway epithelial barrier formation, growth regulation of breast and prostate cancer cells, and vesicle trafficking. [provided by RefSeq, Jan 2015]', '2', 'snoRNA', 62855564, 62855632, 44.93, 1, 'ENSG00000115825', 'O94806'),
(2663, 'PRKRIP1', 'Predicted to enable double-stranded RNA binding activity; protein kinase binding activity; and protein kinase inhibitor activity. Involved in renal system process. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 69048, 77107, 53.03, 5, 'ENSG00000128563', 'Q9H875'),
(2664, 'PRM1', 'Predicted to enable DNA binding activity. Predicted to be involved in chromosome organization and spermatogenesis. Predicted to act upstream of or within nucleus organization and spermatid development. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 43040777, 43075095, 52.00, 19, 'ENSG00000175646', 'P04553'),
(2665, 'PRM3', 'Predicted to enable DNA binding activity. Predicted to be involved in flagellated sperm motility. Predicted to be located in chromosome and nucleus. Predicted to be part of nucleosome. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 70052544, 70112282, 45.85, 33, 'ENSG00000178257', 'Q9NNZ6'),
(2666, 'PRNP', 'The protein encoded by this gene is a membrane glycosylphosphatidylinositol-anchored glycoprotein that tends to aggregate into rod-like structures. The encoded protein contains a highly unstable region of five tandem octapeptide repeats. This gene is found on chromosome 20, approximately 20 kbp upstream of a gene which encodes a biochemically and structurally similar protein to the one encoded by this gene. Mutations in the repeat region as well as elsewhere in this gene have been associated with Creutzfeldt-Jakob disease, fatal familial insomnia, Gerstmann-Straussler disease, Huntington disease-like 1, and kuru. An overlapping open reading frame has been found for this gene that encodes a smaller, structurally unrelated protein, AltPrp. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]', '20', 'protein_coding', 97633668, 97673748, 37.74, 5, 'ENSG00000171867', 'P04156'),
(2667, 'PRNT', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 46211928, 46277115, 36.65, 12, 'ENSG00000180259', 'Q86SH4'),
(2668, 'PROCR', 'The protein encoded by this gene is a receptor for activated protein C, a serine protease activated by and involved in the blood coagulation pathway. The encoded protein is an N-glycosylated type I membrane protein that enhances the activation of protein C. Mutations in this gene have been associated with venous thromboembolism and myocardial infarction, as well as with late fetal loss during pregnancy. The encoded protein may also play a role in malarial infection and has been associated with cancer. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 88747225, 88754785, 35.62, 3, 'ENSG00000101000', 'Q9UNN8'),
(2669, 'PROK2', 'This gene encodes a protein expressed in the suprachiasmatic nucleus (SCN) circadian clock that may function as the output component of the circadian clock. The secreted form of the encoded protein may also serve as a chemoattractant for neuronal precursor cells in the olfactory bulb. Proteins from other vertebrates which are similar to this gene product were isolated based on homology to snake venom and secretions from frog skin, and have been shown to have diverse functions. Mutations in this gene are associated with Kallmann syndrome 4. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 175440036, 175444182, 46.64, 1, 'ENSG00000163421', 'Q9HC23'),
(2670, 'PROKR2', 'Prokineticins are secreted proteins that can promote angiogenesis and induce strong gastrointestinal smooth muscle contraction. The protein encoded by this gene is an integral membrane protein and G protein-coupled receptor for prokineticins. The encoded protein is similar in sequence to GPR73, another G protein-coupled receptor for prokineticins. [provided by RefSeq, Jul 2008]', '20', 'unitary_pseudogene', 55302, 55825, 61.07, 1, 'ENSG00000101292', 'Q8NFJ6'),
(2671, 'PROM2', 'This gene encodes a member of the prominin family of pentaspan membrane glycoproteins. The encoded protein localizes to basal epithelial cells and may be involved in the organization of plasma membrane microdomains. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]', '2', 'protein_coding', 75979240, 76000586, 49.22, 5, 'ENSG00000292084', 'Q8N271'),
(2672, 'PROM2', 'This gene encodes a member of the prominin family of pentaspan membrane glycoproteins. The encoded protein localizes to basal epithelial cells and may be involved in the organization of plasma membrane microdomains. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]', '2', 'protein_coding', 75979240, 76000586, 49.22, 5, 'ENSG00000155066', 'Q8N271'),
(2673, 'PROSER1', 'This gene encodes a conserved protein containing proline and serine rich regions. These regions may be important in protein-protein interactions. [provided by RefSeq, Aug 2012]', '13', 'protein_coding', 26917922, 26967088, 38.46, 2, 'ENSG00000120685', 'Q86XN7'),
(2674, 'PRPH2', 'The protein encoded by this gene is a member of the transmembrane 4 superfamily, also known as the tetraspanin family. Most of these members are cell-surface proteins that are characterized by the presence of four hydrophobic domains. The proteins mediate signal transduction events that play a role in the regulation of cell development, activation, growth and motility. This encoded protein is a cell surface glycoprotein found in the outer segment of both rod and cone photoreceptor cells. It may function as an adhesion molecule involved in stabilization and compaction of outer segment disks or in the maintenance of the curvature of the rim. This protein is essential for disk morphogenesis. Defects in this gene are associated with both central and peripheral retinal degenerations. Some of the various phenotypically different disorders are autosomal dominant retinitis pigmentosa, progressive macular degeneration, macular dystrophy and retinitis pigmentosa digenic. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 616577, 626078, 63.08, 11, 'ENSG00000112619', 'P23942'),
(2675, 'PRR11', 'Involved in regulation of cell cycle. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 641689, 641778, 57.78, 1, 'ENSG00000068489', 'Q96HE9'),
(2676, 'PRR20A', 'This gene is one of five identical loci in a cluster on chromosome 13q21.1. The predicted protein is proline-rich and contains several dopamine D4 receptor signatures and PRINTS domains. [provided by RefSeq, Oct 2008]', '13', 'protein_coding', 39570753, 39689735, 56.05, 5, 'ENSG00000204919', 'P86496'),
(2677, 'PRR20E', 'This gene is one of five identical loci in a cluster on chromosome 13q21.1. The predicted protein is proline-rich and contains several dopamine D4 receptor signatures and PRINTS domains. [provided by RefSeq, Oct 2008]', '13', 'protein_coding', 57167197, 57170218, 49.77, 1, 'ENSG00000234278', 'P86478'),
(2678, 'PRR27', 'Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 760734, 768865, 66.92, 8, 'ENSG00000187533', 'Q6MZM9'),
(2679, 'PRR27', 'Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 760734, 768865, 66.92, 8, 'ENSG00000282868', 'Q6MZM9'),
(2680, 'PRR5L', 'Enables ubiquitin protein ligase binding activity. Involved in several processes, including TORC2 signaling; regulation of fibroblast migration; and regulation of primary metabolic process. Part of TORC2 complex. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 151239967, 151358559, 43.87, 5, 'ENSG00000135362', 'Q6MZQ0'),
(2681, 'PRSS12', 'This gene encodes a member of the trypsin family of serine proteases and contains a signal peptide, a proline-rich region, a Kringle domain, four scavenger receptor cysteine-rich domains, and a trypsin-like serine protease domain. The protein, sometimes referred to as neurotrypsin or motopsin, is secreted from neuronal cells and localizes to the synaptic cleft. Studies in mice show that this protein cleaves a protein, agrin, that is important for the formation and maintenance of exitatory synapses. Defects in this gene cause a form of autosomal recessive cognitive impairment (MRT1). [provided by RefSeq, Jul 2017]', '4', 'protein_coding', 39412669, 39428415, 56.66, 16, 'ENSG00000164099', 'P56730'),
(2682, 'PRSS2', 'This gene belongs to the trypsin family of serine proteases and encodes anionic trypsinogen. It is part of a cluster of trypsinogen genes that are located within the T cell receptor beta locus. Enzymes of this family cleave peptide bonds that follow lysine or arginine residues. This protein is found at high levels in pancreatic juice and its upregulation is a characteristic feature of pancreatitis. This protein has also been found to activate pro-urokinase in ovarian tumors, suggesting a function in tumor invasion. In addition, this enzyme is able to cleave across the type II collagen triple helix in rheumatoid arthritis synovitis tissue, potentially participating in the degradation of type II collagen-rich cartilage matrix. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Jan 2015]', '7', 'protein_coding', 120600811, 120714470, 38.18, 1, 'ENSG00000282049', 'P07478'),
(2683, 'PRSS2', 'This gene belongs to the trypsin family of serine proteases and encodes anionic trypsinogen. It is part of a cluster of trypsinogen genes that are located within the T cell receptor beta locus. Enzymes of this family cleave peptide bonds that follow lysine or arginine residues. This protein is found at high levels in pancreatic juice and its upregulation is a characteristic feature of pancreatitis. This protein has also been found to activate pro-urokinase in ovarian tumors, suggesting a function in tumor invasion. In addition, this enzyme is able to cleave across the type II collagen triple helix in rheumatoid arthritis synovitis tissue, potentially participating in the degradation of type II collagen-rich cartilage matrix. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Jan 2015]', '7', 'protein_coding', 120600811, 120714470, 38.18, 1, 'ENSG00000275896', 'P07478'),
(2684, 'PRSS35', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 147617386, 147672044, 38.71, 4, 'ENSG00000146250', 'Q8N3Z0'),
(2685, 'PRSS37', 'Predicted to enable serine-type endopeptidase activity. Involved in positive regulation of acrosome reaction and regulation of protein processing. Located in acrosomal vesicle. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 100884483, 100884576, 52.13, 1, 'ENSG00000165076', 'A4D1T9'),
(2686, 'PRSS51', 'Predicted to enable serine-type endopeptidase activity. Predicted to be involved in proteolysis. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 104455295, 104762014, 46.04, 6, 'ENSG00000292298', 'A0A1B0GVH4'),
(2687, 'PRSS51', 'Predicted to enable serine-type endopeptidase activity. Predicted to be involved in proteolysis. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 104455295, 104762014, 46.04, 6, 'ENSG00000253649', 'A0A1B0GVH4'),
(2688, 'PRSS8', 'This gene encodes a member of the peptidase S1 or chymotrypsin family of serine proteases. The encoded preproprotein is proteolytically processed to generate light and heavy chains that associate via a disulfide bond to form the heterodimeric enzyme. This enzyme is highly expressed in prostate epithelia and is one of several proteolytic enzymes found in seminal fluid. This protease exhibits trypsin-like substrate specificity, cleaving protein substrates at the carboxyl terminus of lysine or arginine residues. The encoded protease partially mediates proteolytic activation of the epithelial sodium channel, a regulator of sodium balance, and may also play a role in epithelial barrier formation. [provided by RefSeq, Feb 2016]', '16', 'miRNA', 39664034, 39664135, 44.12, 1, 'ENSG00000052344', 'Q16651'),
(2689, 'PRX', 'The protein encoded by this gene is a member of the thiol-specific antioxidant protein family. This protein is a bifunctional enzyme with two distinct active sites. It is involved in redox regulation of the cell; it can reduce H(2)O(2) and short chain organic, fatty acid, and phospholipid hydroperoxides. It may play a role in the regulation of phospholipid turnover as well as in protection against oxidative injury. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 57519163, 57519259, 40.21, 1, 'ENSG00000105227', 'Q9BXM0'),
(2690, 'PSAT1', 'This gene encodes a member of the class-V pyridoxal-phosphate-dependent aminotransferase family. The encoded protein is a phosphoserine aminotransferase and decreased expression may be associated with schizophrenia. Mutations in this gene are also associated with phosphoserine aminotransferase deficiency. Alternative splicing results in multiple transcript variants. Pseudogenes of this gene have been defined on chromosomes 1, 3, and 8. [provided by RefSeq, Jul 2013]', '9', 'protein_coding', 19351368, 19447711, 46.97, 3, 'ENSG00000135069', 'Q9Y617'),
(2691, 'PSENEN', 'Presenilins, which are components of the gamma-secretase protein complex, are required for intramembranous processing of some type I transmembrane proteins, such as the Notch proteins and the beta-amyloid precursor protein. Signaling by Notch receptors mediates a wide range of developmental cell fates. Processing of the beta-amyloid precursor protein generates neurotoxic amyloid beta peptides, the major component of senile plaques associated with Alzheimer\'s disease. This gene encodes a protein that is required for Notch pathway signaling, and for the activity and accumulation of gamma-secretase. Mutations resulting in haploinsufficiency for this gene cause familial acne inversa-2 (ACNINV2). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '19', 'protein_coding', 44044640, 44069456, 53.20, 22, 'ENSG00000205155', 'Q9NZ42'),
(2692, 'PSG6', 'This gene is a member of the pregnancy-specific glycoprotein (PSG) gene family. The PSG genes are a subgroup of the carcinoembryonic antigen (CEA) family of immunoglobulin-like genes, and are found in a gene cluster at 19q13.1-q13.2 telomeric to another cluster of CEA-related genes. The PSG genes are expressed by placental trophoblasts and released into the maternal circulation during pregnancy, and are thought to be essential for maintenance of normal pregnancy. The protein encoded by this gene contains the Arg-Gly-Asp tripeptide associated with cellular adhesion and recognition. Alternative splicing results in multiple transcript variants and protein isoforms. [provided by RefSeq, Jul 2012]', '19', 'protein_coding', 42766, 231794, 55.87, 35, 'ENSG00000170848', 'Q00889'),
(2693, 'PSMD10', 'This gene encodes a subunit of the PA700/19S complex, which is the regulatory component of the 26S proteasome. The 26S proteosome complex is required for ubiquitin-dependent protein degradation. This protein is a non-ATPase subunit that may be involved in protein-protein interactions. Aberrant expression of this gene may paly a role in tumorigenesis. Two transcripts encoding different isoforms have been described. Pseudogenes have been identified on chromosomes 3 and 20.[provided by RefSeq, Mar 2011]', 'X', 'protein_coding', 141241463, 141242517, 49.76, 1, 'ENSG00000101843', 'O75832'),
(2694, 'PSMD5', 'The 26S proteasome is a multicatalytic proteinase complex with a highly ordered structure composed of 2 complexes, a 20S core and a 19S regulator. The 20S core is composed of 4 rings of 28 non-identical subunits; 2 rings are composed of 7 alpha subunits and 2 rings are composed of 7 beta subunits. The 19S regulator is composed of a base, which contains 6 ATPase subunits and 2 non-ATPase subunits, and a lid, which contains up to 10 non-ATPase subunits. Proteasomes are distributed throughout eukaryotic cells at a high concentration and cleave peptides in an ATP/ubiquitin-dependent process in a non-lysosomal pathway. This gene encodes a non-ATPase subunit of the 19S regulator base that functions as a chaperone protein during 26S proteasome assembly. [provided by RefSeq, Jul 2012]', '9', 'protein_coding', 5985892, 5986985, 50.18, 1, 'ENSG00000095261', 'Q16401'),
(2695, 'PSMG1', 'Enables molecular adaptor activity. Involved in chaperone-mediated protein complex assembly. Located in several cellular components, including Golgi apparatus; endoplasmic reticulum; and nucleoplasm. Part of protein folding chaperone complex. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 45306840, 45317824, 54.84, 5, 'ENSG00000183527', 'O95456'),
(2696, 'PSMG2', 'Enables molecular adaptor activity. Involved in chaperone-mediated protein complex assembly. Located in nucleus. Part of protein folding chaperone complex. Implicated in proteosome-associated autoinflammatory syndrome 4. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 163110073, 163166890, 36.34, 6, 'ENSG00000128789', 'Q969U7'),
(2697, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000204538', 'Q9UIG4'),
(2698, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000224544', 'Q9UIG4'),
(2699, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000232127', 'Q9UIG4'),
(2700, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000227246', 'Q9UIG4'),
(2701, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000206459', 'Q9UIG4'),
(2702, 'PSORS1C2', 'Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 92121969, 92172145, 39.58, 5, 'ENSG00000234605', 'Q9UIG4'),
(2703, 'PTCHD1', 'This gene encodes a membrane protein with a patched domain. The encoded protein is similar to Drosophila proteins which act as receptors for the morphogen sonic hedgehog. Deletions in this gene, which is located on the X chromosome, are associated with intellectual disability and autism (PMID: 21091464, PMID: 20844286). [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 31944337, 31952046, 49.33, 1, 'ENSG00000165186', 'Q96NR3'),
(2704, 'PTCRA', 'The protein encoded by this gene is a single-pass type I membrane protein that is found in immmature but not mature T-cells. Along with TCRB and CD3 complex, the encoded protein forms the pre-T-cell receptor complex, which regulates early T-cell development. Four transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jul 2011]', '6', 'protein_coding', 255019, 256175, 55.06, 1, 'ENSG00000171611', 'Q6ISU1'),
(2705, 'PTEN', 'This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2015]', '10', 'protein_coding', 251983, 284900, 51.69, 3, 'ENSG00000284792', 'P60484'),
(2706, 'PTEN', 'This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Feb 2015]', '10', 'protein_coding', 251983, 284900, 51.69, 3, 'ENSG00000171862', 'P60484'),
(2707, 'PTF1A', 'This gene encodes a protein that is a component of the pancreas transcription factor 1 complex (PTF1) and is known to have a role in mammalian pancreatic development. The protein plays a role in determining whether cells allocated to the pancreatic buds continue towards pancreatic organogenesis or revert back to duodenal fates. The protein is thought to be involved in the maintenance of exocrine pancreas-specific gene expression including elastase 1 and amylase. Mutations in this gene cause cerebellar agenesis and loss of expression is seen in ductal type pancreas cancers. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 1609915, 1613897, 56.26, 1, 'ENSG00000168267', 'Q7RTS3'),
(2708, 'PTGES2', 'The protein encoded by this gene is a membrane-associated prostaglandin E synthase, which catalyzes the conversion of prostaglandin H2 to prostaglandin E2. This protein also has been shown to activate the transcription regulated by a gamma-interferon-activated transcription element (GATE). Multiple transcript variants have been found for this gene. [provided by RefSeq, Jun 2009]', '9', 'protein_coding', 73394828, 73417566, 40.78, 3, 'ENSG00000148334', 'Q9H7Z7'),
(2709, 'PTGES3L-AARSD1', 'This locus represents naturally occurring readthrough transcription between the neighboring PTGES3L (prostaglandin E synthase 3 (cytosolic)-like) and AARSD1(alanyl-tRNA synthetase domain containing 1) genes on chromosome 17. The readthrough transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, May 2012]', '17', 'miRNA', 71973, 72110, 31.16, 1, 'ENSG00000108825', 'B3KSP9'),
(2710, 'PTGFRN', 'Predicted to be involved in myoblast fusion involved in skeletal muscle regeneration. Predicted to act upstream of or within lipid droplet organization. Located in cell surface. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 77131270, 77142734, 54.74, 3, 'ENSG00000134247', 'Q9P2B2'),
(2711, 'PTGR1', 'This gene encodes an enzyme that is involved in the inactivation of the chemotactic factor, leukotriene B4. The encoded protein specifically catalyzes the NADP+ dependent conversion of leukotriene B4 to 12-oxo-leukotriene B4. A pseudogene of this gene is found on chromosome 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2009]', '9', 'protein_coding', 91102629, 91111494, 35.07, 3, 'ENSG00000106853', 'Q14914'),
(2712, 'PTHLH', 'The protein encoded by this gene is a member of the parathyroid hormone family. This hormone, via its receptor, PTHR1, regulates endochondral bone development and epithelial-mesenchymal interactions during the formation of the mammary glands and teeth. It is responsible for most cases of humoral hypercalcemia of malignancy, and mutations in this gene are associated with brachydactyly type E2 (BDE2). Alternatively spliced transcript variants have been found for this gene. There is also evidence for alternative translation initiation from non-AUG (CUG and GUG) start sites, downstream of the initiator AUG codon, resulting in nuclear forms of this hormone. [provided by RefSeq, Nov 2013]', '12', 'protein_coding', 100802565, 100827523, 54.77, 10, 'ENSG00000087494', 'P12272'),
(2713, 'PTK7', 'This gene encodes a member of the receptor protein tyrosine kinase family of proteins that transduce extracellular signals across the cell membrane. The encoded protein lacks detectable catalytic tyrosine kinase activity, is involved in the Wnt signaling pathway and plays a role in multiple cellular processes including polarity and adhesion. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jul 2012]', '6', 'protein_coding', 57768471, 57772119, 52.12, 8, 'ENSG00000112655', 'Q13308'),
(2714, 'PTP4A3', 'This gene encodes a member of the protein-tyrosine phosphatase family. Protein tyrosine phosphatases are cell signaling molecules that play regulatory roles in a variety of cellular processes. Studies of this class of protein tyrosine phosphatase in mice demonstrates that they are prenylated in vivo, suggesting their association with cell plasma membrane. The encoded protein may enhance cell proliferation, and overexpression of this gene has been implicated in tumor metastasis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '8', 'protein_coding', 45212995, 45683688, 42.88, 3, 'ENSG00000291470', 'O75365'),
(2715, 'PTP4A3', 'This gene encodes a member of the protein-tyrosine phosphatase family. Protein tyrosine phosphatases are cell signaling molecules that play regulatory roles in a variety of cellular processes. Studies of this class of protein tyrosine phosphatase in mice demonstrates that they are prenylated in vivo, suggesting their association with cell plasma membrane. The encoded protein may enhance cell proliferation, and overexpression of this gene has been implicated in tumor metastasis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '8', 'protein_coding', 45212995, 45683688, 42.88, 3, 'ENSG00000184489', 'O75365'),
(2716, 'PTP4A3', 'This gene encodes a member of the protein-tyrosine phosphatase family. Protein tyrosine phosphatases are cell signaling molecules that play regulatory roles in a variety of cellular processes. Studies of this class of protein tyrosine phosphatase in mice demonstrates that they are prenylated in vivo, suggesting their association with cell plasma membrane. The encoded protein may enhance cell proliferation, and overexpression of this gene has been implicated in tumor metastasis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '8', 'protein_coding', 45212995, 45683688, 42.88, 3, 'ENSG00000275575', 'O75365'),
(2717, 'PTPN1', 'The protein encoded by this gene is the founding member of the protein tyrosine phosphatase (PTP) family, which was isolated and identified based on its enzymatic activity and amino acid sequence. PTPs catalyze the hydrolysis of the phosphate monoesters specifically on tyrosine residues. Members of the PTP family share a highly conserved catalytic motif, which is essential for the catalytic activity. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP has been shown to act as a negative regulator of insulin signaling by dephosphorylating the phosphotryosine residues of insulin receptor kinase. This PTP was also reported to dephosphorylate epidermal growth factor receptor kinase, as well as JAK2 and TYK2 kinases, which implicated the role of this PTP in cell growth control, and cell response to interferon stimulation. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 56392371, 56406362, 41.14, 8, 'ENSG00000196396', 'P18031'),
(2718, 'PTPN13', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP is a large intracellular protein. It has a catalytic PTP domain at its C-terminus and two major structural domains: a region with five PDZ domains and a FERM domain that binds to plasma membrane and cytoskeletal elements. This PTP was found to interact with, and dephosphorylate, Fas receptor and IkappaBalpha through the PDZ domains. This suggests it has a role in Fas mediated programmed cell death. This PTP was also shown to interact with GTPase-activating protein, and thus may function as a regulator of Rho signaling pathways. Four alternatively spliced transcript variants, which encode distinct proteins, have been reported. [provided by RefSeq, Oct 2008]', '4', 'protein_coding', 100725705, 100738224, 55.89, 5, 'ENSG00000163629', 'Q12923'),
(2719, 'PTPN2', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. Members of the PTP family share a highly conserved catalytic motif, which is essential for the catalytic activity. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. Epidermal growth factor receptor and the adaptor protein Shc were reported to be substrates of this PTP, which suggested the roles in growth factor mediated cell signaling. Multiple alternatively spliced transcript variants encoding different isoforms have been found. Two highly related but distinctly processed pseudogenes that localize to chromosomes 1 and 13, respectively, have been reported. [provided by RefSeq, May 2011]', '18', 'protein_coding', 28003274, 28100592, 43.22, 3, 'ENSG00000175354', 'P17706'),
(2720, 'PTPN21', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an N-terminal domain, similar to cytoskeletal- associated proteins including band 4.1, ezrin, merlin, and radixin. This PTP was shown to specially interact with BMX/ETK, a member of Tec tyrosine kinase family characterized by a multimodular structures including PH, SH3, and SH2 domains. The interaction of this PTP with BMX kinase was found to increase the activation of STAT3, but not STAT2 kinase. Studies of the similar gene in mice suggested the possible roles of this PTP in liver regeneration and spermatogenesis. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 3190350, 3204685, 54.35, 4, 'ENSG00000070778', 'Q16825'),
(2721, 'PTPN4', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This protein contains a C-terminal PTP domain and an N-terminal domain homologous to the band 4.1 superfamily of cytoskeletal-associated proteins. This PTP has been shown to interact with glutamate receptor delta 2 and epsilon subunits, and is thought to play a role in signalling downstream of the glutamate receptors through tyrosine dephosphorylation. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 89995110, 90035533, 35.67, 4, 'ENSG00000088179', 'P29074'),
(2722, 'PTPRA', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus represents a receptor-type PTP. This PTP has been shown to dephosphorylate and activate Src family tyrosine kinases, and is implicated in the regulation of integrin signaling, cell adhesion and proliferation. Three alternatively spliced variants of this gene, which encode two distinct isoforms, have been reported. [provided by RefSeq, Jul 2008]', '20', 'lncRNA', 469509, 472705, 49.58, 2, 'ENSG00000132670', 'P18433'),
(2723, 'PTPRC', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitosis, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus is classified as a receptor type PTP. This PTP has been shown to be an essential regulator of T- and B-cell antigen receptor signaling. It functions through either direct interaction with components of the antigen receptor complexes, or by activating various Src family kinases required for the antigen receptor signaling. This PTP also suppresses JAK kinases, and thus functions as a regulator of cytokine receptor signaling. Alternatively spliced transcripts variants of this gene, which encode distinct isoforms, have been reported. [provided by RefSeq, Jun 2012]', '1', 'protein_coding', 14057457, 14132490, 39.05, 13, 'ENSG00000262418', 'P08575'),
(2724, 'PTPRC', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitosis, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus is classified as a receptor type PTP. This PTP has been shown to be an essential regulator of T- and B-cell antigen receptor signaling. It functions through either direct interaction with components of the antigen receptor complexes, or by activating various Src family kinases required for the antigen receptor signaling. This PTP also suppresses JAK kinases, and thus functions as a regulator of cytokine receptor signaling. Alternatively spliced transcripts variants of this gene, which encode distinct isoforms, have been reported. [provided by RefSeq, Jun 2012]', '1', 'protein_coding', 14057457, 14132490, 39.05, 13, 'ENSG00000081237', 'P08575'),
(2725, 'PTPRG', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem intracytoplasmic catalytic domains, and thus represents a receptor-type PTP. The extracellular region of this PTP contains a carbonic anhydrase-like (CAH) domain, which is also found in the extracellular region of PTPRBETA/ZETA. This gene is located in a chromosomal region that is frequently deleted in renal cell carcinoma and lung carcinoma, thus is thought to be a candidate tumor suppressor gene. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 56563627, 56567868, 44.39, 1, 'ENSG00000144724', 'P23470'),
(2726, 'PTPRM', 'The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem catalytic domains, and thus represents a receptor-type PTP. The extracellular region contains a meprin-A5 antigen-PTP mu (MAM) domain, an Ig-like domain and four fibronectin type III-like repeats. This PTP has been shown to mediate cell-cell aggregation through the interaction with another molecule of this PTP on an adjacent cell. This PTP can interact with scaffolding protein RACK1/GNB2L1, which may be necessary for the downstream signaling in response to cell-cell adhesion. Alternative splicing results in multiple transcripts encoding distinct isoforms. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 8113184, 8262687, 40.30, 19, 'ENSG00000173482', 'P28827'),
(2727, 'PTS', 'The enzyme encoded by this gene catalyzes the elimination of inorganic triphosphate from dihydroneopterin triphosphate, which is the second and irreversible step in the biosynthesis of tetrahydrobiopterin from GTP. Tetrahydrobiopterin, also known as BH(4), is an essential cofactor and regulator of various enzyme activities, including enzymes involved in serotonin biosynthesis and NO synthase activity. Mutations in this gene result in hyperphenylalaninemia. [provided by RefSeq, Oct 2008]', '11', 'protein_coding', 23701055, 23766475, 44.60, 8, 'ENSG00000150787', 'Q03393'),
(2728, 'PTTG1', 'The encoded protein is a homolog of yeast securin proteins, which prevent separins from promoting sister chromatid separation. It is an anaphase-promoting complex (APC) substrate that associates with a separin until activation of the APC. The gene product has transforming activity in vitro and tumorigenic activity in vivo, and the gene is highly expressed in various tumors. The gene product contains 2 PXXP motifs, which are required for its transforming and tumorigenic activities, as well as for its stimulation of basic fibroblast growth factor expression. It also contains a destruction box (D box) that is required for its degradation by the APC. The acidic C-terminal region of the encoded protein can act as a transactivation domain. The gene product is mainly a cytosolic protein, although it partially localizes in the nucleus. Three transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Sep 2013]', '5', 'protein_coding', 2873420, 3039076, 42.52, 7, 'ENSG00000164611', 'O95997'),
(2729, 'PTTG1IP', 'This gene encodes a single-pass type I integral membrane protein, which binds to pituitary tumor-transforming 1 protein (PTTG1), and facilitates translocation of PTTG1 into the nucleus. Coexpression of this protein and PTTG1 induces transcriptional activation of basic fibroblast growth factor. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2013]', '21', 'protein_coding', 56730118, 56850286, 42.43, 21, 'ENSG00000183255', 'P53801'),
(2730, 'PTTG1IP2', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 31318495, 31529814, 36.04, 23, 'ENSG00000251154', 'P0DTF9'),
(2731, 'PTX3', 'This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]', '3', 'protein_coding', 58526871, 58578220, 38.55, 5, 'ENSG00000163661', 'P26022'),
(2732, 'PURA', 'This gene product is a sequence-specific, single-stranded DNA-binding protein. It binds preferentially to the single strand of the purine-rich element termed PUR, which is present at origins of replication and in gene flanking regions in a variety of eukaryotes from yeasts through humans. Thus, it is implicated in the control of both DNA replication and transcription. Deletion of this gene has been associated with myelodysplastic syndrome and acute myelogenous leukemia. [provided by RefSeq, Jul 2008]', '5', 'miRNA', 2692861, 2692919, 71.19, 1, 'ENSG00000185129', 'Q00577'),
(2733, 'PURB', 'This gene product is a sequence-specific, single-stranded DNA-binding protein. It binds preferentially to the single strand of the purine-rich element termed PUR, which is present at origins of replication and in gene flanking regions in a variety of eukaryotes from yeasts through humans. Thus, it is implicated in the control of both DNA replication and transcription. Deletion of this gene has been associated with myelodysplastic syndrome and acute myelogenous leukemia. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 31134807, 31140508, 56.98, 1, 'ENSG00000146676', 'Q96QR8'),
(2734, 'PVALB', 'This gene encodes a member of the globin superfamily and is predominantly expressed in skeletal and cardiac muscles. The encoded protein forms a monomeric globular haemoprotein that is primarily responsible for the storage and facilitated transfer of oxygen from the cell membrane to the mitochondria. This protein also plays a role in regulating physiological levels of nitric oxide. Multiple transcript variants encoding distinct isoforms exist for this gene. [provided by RefSeq, May 2020]', '22', 'protein_coding', 18155329, 18223854, 42.05, 5, 'ENSG00000100362', 'P20472'),
(2735, 'PVALB', 'This gene encodes a member of the globin superfamily and is predominantly expressed in skeletal and cardiac muscles. The encoded protein forms a monomeric globular haemoprotein that is primarily responsible for the storage and facilitated transfer of oxygen from the cell membrane to the mitochondria. This protein also plays a role in regulating physiological levels of nitric oxide. Multiple transcript variants encoding distinct isoforms exist for this gene. [provided by RefSeq, May 2020]', '22', 'protein_coding', 18155329, 18223854, 42.05, 5, 'ENSG00000274665', 'P20472'),
(2736, 'PYCARD', 'This gene encodes an adaptor protein that is composed of two protein-protein interaction domains: a N-terminal PYRIN-PAAD-DAPIN domain (PYD) and a C-terminal caspase-recruitment domain (CARD). The PYD and CARD domains are members of the six-helix bundle death domain-fold superfamily that mediates assembly of large signaling complexes in the inflammatory and apoptotic signaling pathways via the activation of caspase. In normal cells, this protein is localized to the cytoplasm; however, in cells undergoing apoptosis, it forms ball-like aggregates near the nuclear periphery. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 95473520, 95515839, 39.23, 11, 'ENSG00000103490', 'Q9ULZ3'),
(2737, 'PYCR3', 'This gene encodes a protein that belongs to the pyrroline-5-carboxylate reductase family of enzymes. Members of this family catalyze the final step in proline biosynthesis, converting pyrroline-5-carboxylate to proline. Glutamate and ornithine are precursors in the synthesis of proline. The protein encoded by this gene is a cytoplasmic enzyme involved in the biosynthesis of proline from ornithine. [provided by RefSeq, Aug 2016]', '8', 'protein_coding', 30443631, 30445874, 51.20, 1, 'ENSG00000104524', 'Q53H96'),
(2738, 'PYCR3', 'This gene encodes a protein that belongs to the pyrroline-5-carboxylate reductase family of enzymes. Members of this family catalyze the final step in proline biosynthesis, converting pyrroline-5-carboxylate to proline. Glutamate and ornithine are precursors in the synthesis of proline. The protein encoded by this gene is a cytoplasmic enzyme involved in the biosynthesis of proline from ornithine. [provided by RefSeq, Aug 2016]', '8', 'protein_coding', 30443631, 30445874, 51.20, 1, 'ENSG00000276657', 'Q53H96'),
(2739, 'RAB15', 'Enables G protein activity. Involved in positive regulation of regulated secretory pathway and receptor-mediated endocytosis. Located in cilium; endosome membrane; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 109623700, 109625172, 60.69, 1, 'ENSG00000139998', 'P59190'),
(2740, 'RAB20', 'Predicted to enable GTPase activity. Involved in phagosome acidification and phagosome-lysosome fusion. Located in Golgi apparatus and phagocytic vesicle. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 42346357, 42350997, 53.52, 3, 'ENSG00000139832', 'Q9NX57'),
(2741, 'RAB21', 'This gene belongs to the Rab family of monomeric GTPases, which are involved in the control of cellular membrane traffic. The encoded protein plays a role in the targeted trafficking of integrins via its association with integrin alpha tails. As a consequence, the encoded protein is involved in the regulation of cell adhesion and migration. Expression of this gene is associated with a poor prognosis for glioma patients. This gene is downregulated by the tumor suppressor miR-200b, and miRNA-200b is itself downregulated in glioma tissues. [provided by RefSeq, Nov 2015]', '12', 'protein_coding', 173075435, 173268015, 38.91, 8, 'ENSG00000080371', 'Q9UL25'),
(2742, 'RAB22A', 'The protein encoded by this gene is a member of the RAB family of small GTPases. The GTP-bound form of the encoded protein has been shown to interact with early-endosomal antigen 1, and may be involved in the trafficking of and interaction between endosomal compartments. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 63432304, 63446354, 57.39, 21, 'ENSG00000124209', 'Q9UL26'),
(2743, 'RAB27B', 'Enables guanyl ribonucleotide binding activity; myosin V binding activity; and protein domain specific binding activity. Involved in multivesicular body sorting pathway and positive regulation of exocytosis. Located in Golgi stack and cytoplasmic vesicle. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 20368104, 20370949, 63.18, 1, 'ENSG00000041353', 'O00194'),
(2744, 'RAB28', 'This gene encodes a member of the Rab subfamily of Ras-related small GTPases. The encoded protein may be involved in regulating intracellular trafficking. Alternative splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 9 and X. [provided by RefSeq, Apr 2009]', '4', 'protein_coding', 14028148, 14031551, 51.91, 2, 'ENSG00000157869', 'P51157'),
(2745, 'RAB39B', 'This gene encodes a member of the Rab family of proteins. Rab proteins are small GTPases that are involved in vesicular trafficking. Mutations in this gene are associated with X-linked cognitive disability. [provided by RefSeq, Aug 2013]', 'X', 'unprocessed_pseudogene', 131341570, 131342425, 47.08, 1, 'ENSG00000155961', 'Q96DA2'),
(2746, 'RAB3A', 'Enables GTPase activity and myosin V binding activity. Involved in several processes, including acrosomal vesicle exocytosis; plasma membrane repair; and regulation of plasma membrane repair. Located in perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 31377164, 31390590, 42.85, 4, 'ENSG00000105649', 'P20336'),
(2747, 'RAB3IL1', 'This gene encodes a guanine nucleotide exchange factor for the ras-related protein Rab3A. The encoded protein binds Rab3a and the inositol hexakisphosphate kinase InsP6K1. Alternative splicing results in multiple transcript variants. A related pseudogene has been identified on chromosome 7. [provided by RefSeq, Nov 2012]', '11', 'protein_coding', 33731657, 33792269, 49.23, 7, 'ENSG00000167994', 'Q8TBN0');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2748, 'RAB4B-EGLN2', 'This locus represents naturally occurring read-through transcription between the neighboring RAB4B (RAB4B, member RAS oncogene family) and EGLN2 (egl nine homolog 2) genes on chromosome 19. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is thus unlikely to produce a protein product. [provided by RefSeq, Feb 2011]', '19', 'protein_coding', 1270186, 1277539, 36.29, 1, 'ENSG00000171570', ''),
(2749, 'RAB5IF', 'Involved in mitochondrial respirasome assembly and multi-pass transmembrane protein insertion into ER membrane. Located in mitochondrion. Part of multi-pass translocon complex. Implicated in craniofacial dysmorphism, skeletal anomalies, and impaired intellectual development syndrome 2. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 183321764, 183322426, 51.13, 1, 'ENSG00000101084', 'Q9BUV8'),
(2750, 'RAB6C', 'Enables GTPase activity. Involved in mitotic cell cycle; regulation of centrosome duplication; and response to xenobiotic stimulus. Located in Golgi apparatus; microtubule organizing center; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 49859882, 49878351, 59.53, 31, 'ENSG00000292140', 'Q9H0N0'),
(2751, 'RAB6C', 'Enables GTPase activity. Involved in mitotic cell cycle; regulation of centrosome duplication; and response to xenobiotic stimulus. Located in Golgi apparatus; microtubule organizing center; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 49859882, 49878351, 59.53, 31, 'ENSG00000222014', 'Q9H0N0'),
(2752, 'RABEP2', 'Predicted to enable GTPase activator activity and growth factor activity. Involved in regulation of cilium assembly. Located in Golgi apparatus; cytosol; and microtubule organizing center. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 2832332, 2841959, 43.07, 5, 'ENSG00000177548', 'Q9H5N1'),
(2753, 'RAD18', 'The protein encoded by this gene is highly similar to S. cerevisiae DNA damage repair protein Rad18. Yeast Rad18 functions through its interaction with Rad6, which is an ubiquitin-conjugating enzyme required for post-replication repair of damaged DNA. Similar to its yeast counterpart, this protein is able to interact with the human homolog of yeast Rad6 protein through a conserved ring-finger motif.  Mutation of this motif results in defective replication of UV-damaged DNA and hypersensitivity to multiple mutagens. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 35658290, 35661511, 52.82, 6, 'ENSG00000070950', 'Q9NS91'),
(2754, 'RADX', 'Enables single-stranded DNA binding activity. Involved in negative regulation of double-strand break repair via homologous recombination. Located in nuclear speck and replication fork. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 113648804, 113658198, 56.35, 1, 'ENSG00000147231', 'Q6NSI4'),
(2755, 'RAI2', 'Retinoic acid plays a critical role in development, cellular growth, and differentiation. The specific function of this retinoic acid-induced gene has not yet been determined but it may play a role in development. The chromosomal location of this gene designates it to be a candidate for diseases such as Nance-Horan syndrome, sensorineural deafness, non-specific X-linked cognitive disability, oral-facial-digital syndrome, and Fried syndrome. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]', 'X', 'protein_coding', 78297125, 78330093, 45.60, 2, 'ENSG00000131831', 'Q9Y5P3'),
(2756, 'RALGAPA2', 'Predicted to enable GTPase activator activity and protein heterodimerization activity. Predicted to be involved in activation of GTPase activity. Predicted to act upstream of or within Ral protein signal transduction; regulation of exocyst localization; and regulation of protein localization. Located in cytosol and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 158609707, 158609790, 52.38, 1, 'ENSG00000188559', 'Q2PPJ7'),
(2757, 'RALGAPB', 'Enables protein heterodimerization activity. Predicted to be involved in Ral protein signal transduction and activation of GTPase activity. Predicted to act upstream of or within regulation of exocyst localization and regulation of protein localization. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 101052446, 101052525, 41.25, 1, 'ENSG00000170471', 'Q86X10'),
(2758, 'RALGDS', 'Guanine nucleotide dissociation stimulators (GDSs, or exchange factors), such as RALGDS, are effectors of Ras-related GTPases (see MIM 190020) that participate in signaling for a variety of cellular processes.[supplied by OMIM, Nov 2010]', '9', 'protein_coding', 1033456, 1044170, 52.61, 5, 'ENSG00000160271', 'Q12967'),
(2759, 'RALY', 'This gene encodes a member of the heterogeneous nuclear ribonucleoprotein (hnRNP) gene family. This protein may play a role in pre-mRNA splicing and in embryonic development. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Sep 2011]', '20', 'protein_coding', 46803218, 46920160, 38.59, 16, 'ENSG00000125970', 'Q9UKM9'),
(2760, 'RANBP1', 'This gene encodes a protein that forms a complex with Ras-related nuclear protein (Ran) and metabolizes guanoside triphosphate (GTP). This complex participates in the regulation of the cell cycle by controlling transport of proteins and nucleic acids into the nucleus. There are multiple pseudogenes for this gene on chromosomes 9, 12, 17, and X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '22', 'protein_coding', 624344, 659411, 51.23, 4, 'ENSG00000099901', 'P43487'),
(2761, 'RANBP9', 'This gene encodes a protein that binds RAN, a small GTP binding protein belonging to the RAS superfamily that is essential for the translocation of RNA and proteins through the nuclear pore complex. The protein encoded by this gene has also been shown to interact with several other proteins, including met proto-oncogene, homeodomain interacting protein kinase 2, androgen receptor, and cyclin-dependent kinase 11. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 19678288, 19718979, 47.45, 10, 'ENSG00000010017', 'Q96S59'),
(2762, 'RAP2A', 'Enables GTPase activity; guanyl ribonucleotide binding activity; and magnesium ion binding activity. Involved in several processes, including microvillus assembly; positive regulation of protein autophosphorylation; and regulation of dendrite morphogenesis. Acts upstream of or within establishment of protein localization. Located in plasma membrane and recycling endosome membrane. Is active in Schaffer collateral - CA1 synapse. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 288197, 346679, 38.72, 7, 'ENSG00000125249', 'P10114'),
(2763, 'RARRES2', 'This gene encodes a secreted chemotactic protein that initiates chemotaxis via the ChemR23 G protein-coupled seven-transmembrane domain ligand. Expression of this gene is upregulated by the synthetic retinoid tazarotene and occurs in a wide variety of tissues. The active protein has several roles, including that as an adipokine and as an antimicrobial protein with activity against bacteria and fungi. [provided by RefSeq, Nov 2014]', '7', 'protein_coding', 155290227, 155293683, 32.08, 1, 'ENSG00000106538', 'Q99969'),
(2764, 'RASAL3', 'This gene belongs to the Ras GTPase-activating proteins (RasGAP) family and encodes a protein with pleckstrin homology (PH), C2, and Ras GTPase-activation protein (RasGAP) domains. This protein is localized near or at the plasma membrane when expressed exogenously. Reduced expression of this gene in some cell lines resulted in increased levels of the active form of Ras (Ras-GTP), suggesting that this gene may play a role in negatively regulating the Ras signaling pathway. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2017]', '19', 'protein_coding', 56589074, 56591088, 61.39, 6, 'ENSG00000105122', 'Q86YV0'),
(2765, 'RASGEF1C', 'Predicted to enable guanyl-nucleotide exchange factor activity. Predicted to be involved in Ras protein signal transduction. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 59275156, 59318649, 41.55, 2, 'ENSG00000146090', 'Q8N431'),
(2766, 'RASGRF2', 'RAS GTPases cycle between an inactive GDP-bound state and an active GTP-bound state. This gene encodes a calcium-regulated nucleotide exchange factor activating both RAS and RAS-related protein, RAC1, through the exchange of bound GDP for GTP, thereby, coordinating the signaling of distinct mitogen-activated protein kinase pathways. [provided by RefSeq, Oct 2011]', '5', 'protein_coding', 25096136, 25099254, 39.47, 1, 'ENSG00000113319', 'O14827'),
(2767, 'RASL10A', 'Predicted to enable GTPase activity. Predicted to be involved in small GTPase-mediated signal transduction. Predicted to be located in nucleolus and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 66205317, 66420543, 42.21, 6, 'ENSG00000100276', 'Q92737'),
(2768, 'RASSF2', 'This gene encodes a protein that contains a Ras association domain. Similar to its cattle and sheep counterparts, this gene is located near the prion gene. Two alternatively spliced transcripts encoding the same isoform have been reported. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 46447292, 46497422, 41.17, 7, 'ENSG00000101265', 'P50749'),
(2769, 'RASSF7', 'Predicted to be involved in apoptotic process and signal transduction. Located in centriolar satellite. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 155258235, 155264491, 40.80, 1, 'ENSG00000273859', 'Q02833'),
(2770, 'RASSF7', 'Predicted to be involved in apoptotic process and signal transduction. Located in centriolar satellite. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 155258235, 155264491, 40.80, 1, 'ENSG00000099849', 'Q02833'),
(2771, 'RB1', 'The protein encoded by this gene is a negative regulator of the cell cycle and was the first tumor suppressor gene found. The encoded protein also stabilizes constitutive heterochromatin to maintain the overall chromatin structure. The active, hypophosphorylated form of the protein binds transcription factor E2F1. Defects in this gene are a cause of childhood cancer retinoblastoma (RB), bladder cancer, and osteogenic sarcoma. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 78997564, 79236038, 41.48, 10, 'ENSG00000139687', 'P06400'),
(2772, 'RBBP9', 'The protein encoded by this gene is a retinoblastoma binding protein that may play a role in the regulation of cell proliferation and differentiation. Two alternatively spliced transcript variants of this gene with identical predicted protein products have been reported, one of which is a nonsense-mediated decay candidate. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 50081124, 50115959, 44.49, 23, 'ENSG00000089050', 'O75884'),
(2773, 'RBCK1', 'Enables several functions, including identical protein binding activity; transcription coactivator activity; and ubiquitin binding activity. Involved in several processes, including positive regulation of canonical NF-kappaB signal transduction; protein linear polyubiquitination; and regulation of DNA-binding transcription factor activity. Part of LUBAC complex. Implicated in glycogen storage disease. [provided by Alliance of Genome Resources, May 2025]', '20', 'snoRNA', 23396446, 23396516, 33.80, 1, 'ENSG00000125826', 'Q9BYM8'),
(2774, 'RBM10', 'This gene encodes a nuclear protein that belongs to a family proteins that contain an RNA-binding motif. The encoded protein associates with hnRNP proteins and may be involved in regulating alternative splicing. Defects in this gene are the cause of the X-linked recessive disorder, TARP syndrome. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Mar 2011]', 'X', 'miRNA', 53707453, 53707539, 44.83, 1, 'ENSG00000182872', 'P98175'),
(2775, 'RBM12', 'This gene encodes a protein that contains several RNA-binding motifs, potential transmembrane domains, and proline-rich regions. This gene and the gene for copine I overlap at map location 20q11.21. Alternative splicing in the 5\' UTR results in four transcript variants. All variants encode the same protein. [provided by RefSeq, Nov 2010]', '20', 'protein_coding', 36833269, 37034268, 49.30, 18, 'ENSG00000244462', 'Q9NTZ6'),
(2776, 'RBM19', 'This gene encodes a nucleolar protein that contains six RNA-binding motifs. The encoded protein may be involved in regulating ribosome biogenesis. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Apr 2009]', '12', 'protein_coding', 79937467, 79982614, 44.62, 4, 'ENSG00000122965', 'Q9Y4C8'),
(2777, 'RBM39', 'This gene encodes a member of the U2AF65 family of proteins. The encoded protein is found in the nucleus, where it co-localizes with core spliceosomal proteins. It has been shown to play a role in both steroid hormone receptor-mediated transcription and alternative splicing, and it is also a transcriptional coregulator of the viral oncoprotein v-Rel. Multiple transcript variants have been observed for this gene. A related pseudogene has been identified on chromosome X. [provided by RefSeq, Aug 2011]', '20', 'protein_coding', 55509072, 55586666, 35.49, 8, 'ENSG00000131051', 'Q14498'),
(2778, 'RBM46', 'Predicted to enable mRNA binding activity. Predicted to be involved in several processes, including cell cycle switching, mitotic to meiotic cell cycle; meiotic nuclear division; and spermatid differentiation. Predicted to act upstream of or within mRNA stabilization and trophectodermal cell differentiation. Predicted to be located in cytoplasm. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 176213673, 176277571, 36.68, 6, 'ENSG00000151962', 'Q8TBY0'),
(2779, 'RBM48', 'Predicted to enable RNA binding activity. Predicted to be involved in RNA splicing and mRNA processing. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 40798996, 40808434, 59.14, 18, 'ENSG00000127993', 'Q5RL73'),
(2780, 'RBMX', 'This gene belongs to the RBMY gene family which includes candidate Y chromosome spermatogenesis genes. This gene, an active X chromosome homolog of the Y chromosome RBMY gene, is widely expressed whereas the RBMY gene evolved a male-specific function in spermatogenesis. Pseudogenes of this gene, found on chromosomes 1, 4, 9, 11, and 6, were likely derived by retrotransposition from the original gene. Alternatively spliced transcript variants encoding different isoforms have been identified. A snoRNA gene (SNORD61) is found in one of its introns. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 106489745, 106535662, 41.29, 2, 'ENSG00000147274', 'P38159'),
(2781, 'RBMXL3', 'Predicted to enable mRNA binding activity and snRNA binding activity. Predicted to be involved in mRNA splicing, via spliceosome. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 26466209, 26573286, 37.29, 3, 'ENSG00000175718', 'Q8N7X1'),
(2782, 'RBMY1A1', 'This gene encodes a protein containing an RNA-binding motif in the N-terminus and four SRGY (serine, arginine, glycine, tyrosine) boxes in the C-terminus. This protein is thought to function as a splicing regulator during spermatogenesis. Multiple closely related paralogs of this gene are found in a gene cluster in the AZFb azoospermia factor region of chromosome Y. Most of these related copies are thought to be pseudogenes, though several likely encode functional proteins. [provided by RefSeq, Mar 2016]', 'Y', 'lncRNA', 22131944, 22146831, 46.69, 3, 'ENSG00000234414', 'P0DJD3'),
(2783, 'RBMY1E', 'This gene encodes a protein containing an RNA-binding motif in the N-terminus and four SRGY (serine, arginine, glycine, tyrosine) boxes in the C-terminus. Multiple copies of this gene are found in the AZFb azoospermia factor region of chromosome Y and the encoded protein is thought to be involved in spermatogenesis. Most copies of this locus are pseudogenes, although six highly similar copies have full-length ORFs and are considered functional. Four functional copies of this gene are found within inverted repeat IR2; two functional copies of this gene are found in palindrome P3, along with two copies of PTPN13-like, Y-linked. [provided by RefSeq, Jul 2008]', 'Y', 'protein_coding', 9398421, 9401223, 53.26, 5, 'ENSG00000242389', 'A6NEQ0'),
(2784, 'RBPJL', 'This gene encodes a member of the suppressor of hairless protein family. A similar protein in mouse is a transcription factor that binds to DNA sequences almost identical to that bound by the Notch receptor signaling pathway transcription factor recombining binding protein J. The mouse protein has been shown to activate transcription in concert with Epstein-Barr virus nuclear antigen-2. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 69611596, 69715144, 44.82, 3, 'ENSG00000124232', 'Q9UBG7'),
(2785, 'RCAN1', 'The protein encoded by this gene interacts with calcineurin A and inhibits calcineurin-dependent signaling pathways, possibly affecting central nervous system development. This gene is located in the minimal candidate region for the Down syndrome phenotype, and is overexpressed in the brain of Down syndrome fetuses. Chronic overexpression of this gene may lead to neurofibrillary tangles such as those associated with Alzheimer disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2013]', '21', 'snoRNA', 3532117, 3532251, 52.59, 1, 'ENSG00000159200', 'P53805'),
(2786, 'RCBTB2', 'This gene encodes a protein containing two C-terminal BTB/POZ domains that is related to regulator of chromosome condensation (RCC). The encoded protein may act as a guanine nucleotide exchange factor. This gene is observed to be lost or underexpressed in prostate cancers. There is a pseudogene of this gene on chromosome 10. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2013]', '13', 'protein_coding', 34627075, 34657285, 36.98, 1, 'ENSG00000136161', 'O95199'),
(2787, 'RCC2', 'The protein encoded by this gene is a guanine exchange factor that is active on RalA, a small GTPase. The encoded protein and RalA are both essential for proper kinetochore-microtubule function in early mitosis. This protein has been shown to be a biomarker for colorectal cancer. [provided by RefSeq, Oct 2016]', '1', 'protein_coding', 12785478, 12929643, 42.19, 21, 'ENSG00000281540', 'Q9P258'),
(2788, 'RCC2', 'The protein encoded by this gene is a guanine exchange factor that is active on RalA, a small GTPase. The encoded protein and RalA are both essential for proper kinetochore-microtubule function in early mitosis. This protein has been shown to be a biomarker for colorectal cancer. [provided by RefSeq, Oct 2016]', '1', 'protein_coding', 12785478, 12929643, 42.19, 21, 'ENSG00000179051', 'Q9P258'),
(2789, 'RCN3', 'Enables phosphatase binding activity and troponin I binding activity. Predicted to be involved in calcium-mediated signaling. Predicted to be located in cytosol. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 48695554, 48709016, 54.34, 4, 'ENSG00000142552', 'Q96D15'),
(2790, 'RCOR2', 'Predicted to enable transcription corepressor activity. Predicted to be involved in negative regulation of DNA-templated transcription and regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. Predicted to be part of histone deacetylase complex and transcription regulator complex. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 57747727, 57756013, 53.24, 15, 'ENSG00000167771', 'Q8IZ40'),
(2791, 'RDH12', 'The protein encoded by this gene is an NADPH-dependent retinal reductase whose highest activity is toward 9-cis and all-trans-retinol. The encoded enzyme also plays a role in the metabolism of short-chain aldehydes but does not exhibit steroid dehydrogenase activity. Defects in this gene are a cause of Leber congenital amaurosis type 13 and Retinitis Pigmentosa 53. [provided by RefSeq, Sep 2015]', '14', 'protein_coding', 9835257, 9849689, 56.32, 11, 'ENSG00000139988', 'Q96NR8'),
(2792, 'RECK', 'The protein encoded by this gene is a cysteine-rich, extracellular protein with protease inhibitor-like domains whose expression is suppressed strongly in many tumors and cells transformed by various kinds of oncogenes. In normal cells, this membrane-anchored glycoprotein may serve as a negative regulator for matrix metalloproteinase-9, a key enzyme involved in tumor invasion and metastasis. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2015]', '9', 'protein_coding', 75578620, 75663214, 45.48, 13, 'ENSG00000122707', 'O95980'),
(2793, 'REEP3', 'Predicted to enable microtubule binding activity. Involved in mitotic nuclear membrane reassembly. Predicted to be located in endoplasmic reticulum; membrane; and microtubule. Predicted to be active in cytoplasmic microtubule; endoplasmic reticulum membrane; and endoplasmic reticulum tubular network. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 55935456, 55936400, 40.53, 1, 'ENSG00000165476', 'Q6NUK4'),
(2794, 'REG3G', 'This gene encodes a member of the regenerating islet-derived genes (REG)3 protein family. These proteins are secreted, C-type lectins with a carbohydrate recognition domain and N-terminal signal peptide. The protein encoded by this gene is an antimicrobial lectin with activity against Gram-positive bacteria. Alternative splicing results in multiple transcript variants encoding multiple isoforms. [provided by RefSeq, Nov 2014]', '2', 'protein_coding', 18555545, 18589564, 38.33, 2, 'ENSG00000143954', 'Q6UW15'),
(2795, 'REXO2', 'This gene encodes a 3\'-to-5\' exonuclease specific for small (primarily 5 nucleotides or less in length) single-stranded RNA and DNA oligomers. This protein may have a role in DNA repair, replication, and recombination, and in RNA processing and degradation. It may also be involved in resistance of human cells to UV-C-induced cell death through its role in the DNA repair process. [provided by RefSeq, Nov 2011]', '11', 'protein_coding', 91140484, 91183217, 34.90, 20, 'ENSG00000076043', 'Q9Y3B8'),
(2796, 'RFC2', 'This gene encodes a member of the activator 1 small subunits family. The elongation of primed DNA templates by DNA polymerase delta and epsilon requires the action of the accessory proteins, proliferating cell nuclear antigen (PCNA) and replication factor C (RFC). Replication factor C, also called activator 1, is a protein complex consisting of five distinct subunits. This gene encodes the 40 kD subunit, which has been shown to be responsible for binding ATP and may help promote cell survival. Disruption of this gene is associated with Williams syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been described. A pseudogene of this gene has been defined on chromosome 2. [provided by RefSeq, Jul 2013]', '7', 'lncRNA', 70630507, 70670885, 38.00, 6, 'ENSG00000049541', 'P35250'),
(2797, 'RFLNB', 'Enables filamin binding activity. Predicted to be involved in several processes, including actin filament bundle organization; negative regulation of bone mineralization involved in bone maturation; and negative regulation of chondrocyte development. Predicted to act upstream of or within actin cytoskeleton organization and epithelial to mesenchymal transition. Predicted to be located in actin cytoskeleton and cytoplasm. Predicted to be active in actin filament bundle. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 43213130, 43786972, 39.04, 3, 'ENSG00000283979', 'Q8N5W9'),
(2798, 'RFLNB', 'Enables filamin binding activity. Predicted to be involved in several processes, including actin filament bundle organization; negative regulation of bone mineralization involved in bone maturation; and negative regulation of chondrocyte development. Predicted to act upstream of or within actin cytoskeleton organization and epithelial to mesenchymal transition. Predicted to be located in actin cytoskeleton and cytoplasm. Predicted to be active in actin filament bundle. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 43213130, 43786972, 39.04, 3, 'ENSG00000183688', 'Q8N5W9'),
(2799, 'RFPL4AL1', 'Predicted to enable ubiquitin protein ligase activity. Predicted to be involved in innate immune response and regulation of gene expression. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 22836560, 22849041, 57.24, 8, 'ENSG00000229292', 'F8VTS6'),
(2800, 'RFTN1', 'Enables double-stranded RNA binding activity. Involved in B cell receptor signaling pathway; membrane raft assembly; and positive regulation of growth rate. Acts upstream of or within dsRNA transport; response to exogenous dsRNA; and toll-like receptor 3 signaling pathway. Located in endosome; membrane raft; and plasma membrane. Part of protein-containing complex. Biomarker of glaucoma. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 36189124, 36204840, 49.11, 13, 'ENSG00000131378', 'Q14699'),
(2801, 'RFX3', 'This gene is a member of the regulatory factor X gene family, which encodes transcription factors that contain a highly-conserved winged helix DNA binding domain. The protein encoded by this gene is structurally related to regulatory factors X1, X2, X4, and X5. It is a transcriptional activator that can bind DNA as a monomer or as a heterodimer with other RFX family members. Multiple transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Aug 2013]', '9', 'protein_coding', 64945814, 64973226, 52.17, 10, 'ENSG00000080298', 'P48380'),
(2802, 'RFXAP', 'Major histocompatibility (MHC) class II molecules are transmembrane proteins that have a central role in development and control of the immune system. The protein encoded by this gene, along with regulatory factor X-associated ankyrin-containing protein and regulatory factor-5, forms a complex that binds to the X box motif of certain MHC class II gene promoters and activates their transcription. Once bound to the promoter, this complex associates with the non-DNA-binding factor MHC class II transactivator, which controls the cell type specificity and inducibility of MHC class II gene expression. Mutations in this gene have been linked to bare lymphocyte syndrome type II, complementation group D. Transcript variants utilizing different polyA signals have been found for this gene. [provided by RefSeq, Jul 2008]', '13', 'miRNA', 20433778, 20433846, 42.03, 1, 'ENSG00000133111', 'O00287'),
(2803, 'RGN', 'The protein encoded by this gene is a highly conserved, calcium-binding protein, that is preferentially expressed in the liver and kidney. It may have an important role in calcium homeostasis. Studies in rat indicate that this protein may also play a role in aging, as it shows age-associated down-regulation. This gene is part of a gene cluster on chromosome Xp11.3-Xp11.23. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]', 'X', 'protein_coding', 53051991, 53100873, 39.22, 13, 'ENSG00000130988', 'Q15493'),
(2804, 'RGP1', 'Predicted to contribute to GTPase activator activity. Predicted to be involved in NLS-bearing protein import into nucleus. Predicted to be part of nuclear pore. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 1762029, 1777565, 53.51, 12, 'ENSG00000107185', 'Q92546'),
(2805, 'RGR', 'This gene encodes a putative retinal G-protein coupled receptor. The gene is a member of the opsin subfamily of the 7 transmembrane, G-protein coupled receptor 1 family. Like other opsins which bind retinaldehyde, it contains a conserved lysine residue in the seventh transmembrane domain. The protein acts as a photoisomerase to catalyze the conversion of all-trans-retinal to 11-cis-retinal. The reverse isomerization occurs with rhodopsin in retinal photoreceptor cells. The protein is exclusively expressed in tissue adjacent to retinal photoreceptor cells, the retinal pigment epithelium and Mueller cells. This gene may be associated with autosomal recessive and autosomal dominant retinitis pigmentosa (arRP and adRP, respectively). Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 72367340, 72626131, 40.16, 11, 'ENSG00000148604', 'P47804'),
(2806, 'RGS19', 'G proteins mediate a number of cellular processes. The protein encoded by this gene belongs to the RGS (regulators of G-protein signaling) family and specifically interacts with G protein, GAI3. This protein is a guanosine triphosphatase-activating protein that functions to down-regulate Galpha i/Galpha q-linked signaling. Alternatively spliced transcript variants encoding the same protein isoform have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 158968640, 159232659, 43.43, 5, 'ENSG00000171700', 'P49795'),
(2807, 'RHOBTB1', 'The protein encoded by this gene belongs to the Rho family of the small GTPase superfamily. It contains a GTPase domain, a proline-rich region, a tandem of 2 BTB (broad complex, tramtrack, and bric-a-brac) domains, and a conserved C-terminal region. The protein plays a role in small GTPase-mediated signal transduction and the organization of the actin filament system. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Dec 2008]', '10', 'protein_coding', 88759700, 89338528, 34.85, 2, 'ENSG00000072422', 'O94844'),
(2808, 'RHOQ', 'This gene encodes a member of the Rho family of small GTPases, which cycle between inactive GDP-bound and active GTP-bound states and function as molecular switches in signal transduction cascades. Rho proteins promote reorganization of the actin cytoskeleton and regulate cell shape, attachment, and motility. The encoded protein is an important signalling protein for sarcomere assembly and has been shown to play a significant role in the exocytosis of the solute carrier family 2, facilitated glucose transporter member 4 and other proteins, possibly acting as the signal that turns on the membrane fusion machinery. Three related pseudogene have been identified on chromosomes 2 and 14. [provided by RefSeq, Aug 2011]', '2', 'protein_coding', 132186948, 132205759, 61.60, 5, 'ENSG00000119729', 'P17081'),
(2809, 'RIMBP3', 'Predicted to enable benzodiazepine receptor binding activity. Predicted to be involved in fertilization and spermatid development. Predicted to be located in manchette. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 49289206, 49362473, 41.52, 4, 'ENSG00000275793', 'Q9UFD9'),
(2810, 'RIMS3', 'Predicted to enable transmembrane transporter binding activity. Predicted to be a structural constituent of presynaptic active zone. Predicted to be involved in several processes, including regulated exocytosis; regulation of synapse organization; and regulation of synaptic vesicle exocytosis. Predicted to be located in presynaptic active zone. Predicted to be active in several cellular components, including postsynaptic cytosol; presynaptic active zone cytoplasmic component; and presynaptic membrane. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 51752026, 51770531, 43.43, 6, 'ENSG00000117016', 'Q9UJD0'),
(2811, 'RIMS4', 'Predicted to enable transmembrane transporter binding activity. Predicted to be involved in regulation of membrane potential; regulation of synapse organization; and regulation of synaptic vesicle exocytosis. Predicted to be located in synaptic membrane. Predicted to be active in glutamatergic synapse. [provided by Alliance of Genome Resources, May 2025]', '20', 'snoRNA', 101272936, 101273049, 47.37, 1, 'ENSG00000101098', 'Q9H426'),
(2812, 'RIN1', 'Predicted to enable guanyl-nucleotide exchange factor activity and small GTPase binding activity. Predicted to be involved in signal transduction. Predicted to act upstream of or within associative learning; memory; and negative regulation of synaptic plasticity. Located in cytoplasm and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 407498, 432139, 51.27, 20, 'ENSG00000174791', 'Q13671'),
(2813, 'RIOK3', 'This gene was first identified by the similarity of its product to the Aspergillus nidulans SUDD protein. This gene is now recognized as a member of the right open reading frame (RIO) kinase gene family. This gene encodes a serine/threonine kinase that localizes to the cytoplasm and plays a role in the processing of the pre-40 S ribosomal subunit. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2017]', '18', 'protein_coding', 928959, 931969, 58.82, 1, 'ENSG00000101782', 'O14730'),
(2814, 'RIPK1', 'This gene encodes a member of the receptor-interacting protein (RIP) family of serine/threonine protein kinases. The encoded protein plays a role in inflammation and cell death in response to tissue damage, pathogen recognition, and as part of developmental regulation. RIPK1/RIPK3 kinase-mediated necrosis is referred to as necroptosis. Genetic disruption of this gene in mice results in death shortly after birth. [provided by RefSeq, Aug 2017]', '6', 'protein_coding', 708, 4780, 47.46, 3, 'ENSG00000137275', 'Q13546'),
(2815, 'RIPK3', 'The product of this gene is a member of the receptor-interacting protein (RIP) family of serine/threonine protein kinases, and contains a C-terminal domain unique from other RIP family members. The encoded protein is predominantly localized to the cytoplasm, and can undergo nucleocytoplasmic shuttling dependent on novel nuclear localization and export signals. It is a component of the tumor necrosis factor (TNF) receptor-I signaling complex, and can induce apoptosis and weakly activate the NF-kappaB transcription factor. [provided by RefSeq, Jul 2008]', '14', 'TR_D_gene', 22439007, 22439015, 55.56, 1, 'ENSG00000285379', 'Q9Y572'),
(2816, 'RIPK3', 'The product of this gene is a member of the receptor-interacting protein (RIP) family of serine/threonine protein kinases, and contains a C-terminal domain unique from other RIP family members. The encoded protein is predominantly localized to the cytoplasm, and can undergo nucleocytoplasmic shuttling dependent on novel nuclear localization and export signals. It is a component of the tumor necrosis factor (TNF) receptor-I signaling complex, and can induce apoptosis and weakly activate the NF-kappaB transcription factor. [provided by RefSeq, Jul 2008]', '14', 'TR_D_gene', 22439007, 22439015, 55.56, 1, 'ENSG00000129465', 'Q9Y572'),
(2817, 'RIPPLY1', 'This gene encodes a protein similar to a zebrafish protein which acts as a transcriptional repressor in and is required for somite segmentation in zebrafish embryos (PMID: 16326386). Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', 'X', 'protein_coding', 10472288, 10630758, 45.03, 4, 'ENSG00000147223', 'Q0D2K3'),
(2818, 'RIPPLY3', 'Predicted to be involved in embryonic pattern specification and negative regulation of transcription by RNA polymerase II. Predicted to act upstream of or within negative regulation of cell population proliferation. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 11111301, 11123086, 42.97, 3, 'ENSG00000183145', 'P57055'),
(2819, 'RLF', 'This gene encodes a member of the insulin-like hormone superfamily. The encoded protein is mainly produced in gonadal tissues. Studies of the mouse counterpart suggest that this gene may be involved in the development of urogenital tract and female fertility. This protein may also act as a hormone to regulate growth and differentiation of gubernaculum, and thus mediating intra-abdominal testicular descent. Mutations in this gene may lead to cryptorchidism. Alternate splicing results in multiple transcript variants. [provided by RefSeq, May 2012]', '19', 'unprocessed_pseudogene', 119784, 130698, 60.01, 1, 'ENSG00000117000', 'P51460'),
(2820, 'RLIG1', 'Enables RNA ligase (ATP) activity. Involved in response to reactive oxygen species. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 53936777, 54070594, 40.73, 11, 'ENSG00000133641', 'Q8N999'),
(2821, 'RLN3', 'This gene encodes a member of the relaxin family of insulin-like hormones that is expressed predominantly in the brain and plays a role in physiological processes such as stress, memory and appetite regulation. The encoded protein is a precursor that is proteolytically processed to generate a heterodimeric mature form consisting A and B chains interlinked by disulfide bonds. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2015]', '19', 'protein_coding', 49308650, 49444064, 38.37, 12, 'ENSG00000288474', 'Q8WXF3'),
(2822, 'RLN3', 'This gene encodes a member of the relaxin family of insulin-like hormones that is expressed predominantly in the brain and plays a role in physiological processes such as stress, memory and appetite regulation. The encoded protein is a precursor that is proteolytically processed to generate a heterodimeric mature form consisting A and B chains interlinked by disulfide bonds. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2015]', '19', 'protein_coding', 49308650, 49444064, 38.37, 12, 'ENSG00000171136', 'Q8WXF3'),
(2823, 'RMC1', 'This gene encodes a colon cancer associated protein. [provided by RefSeq, Jan 2013]', '18', 'miRNA', 56245520, 56245623, 46.15, 1, 'ENSG00000141452', 'Q96DM3'),
(2824, 'RN7SKP60', 'Predicted to act upstream of or within response to wounding. [provided by Alliance of Genome Resources, May 2025]', '5', 'miRNA', 91350891, 91350972, 42.68, 1, 'ENSG00000252794', ''),
(2825, 'RNA5-8SN2', '45S ribosomal DNA (rDNA) arrays, or clusters, are present on human chromosomes 13, 14, 15, 21 and 22, designated RNR1 through RNR5, respectively. Each cluster consists of multiple 45S rDNA repeat units that vary in number among individuals and chromosomes, with total diploid copy number estimates ranging from 60 to >800 repeat units in a human genome. The 45S rDNA repeat unit encodes a 45S rRNA precursor, transcribed by RNA polymerase I, which is processed to form the 18S, 5.8S and 28S rRNAs. This gene represents a copy of the 5.8S ribosomal RNA on chromosome 21. [provided by RefSeq, Mar 2017]', '21', 'protein_coding', 32268228, 32279049, 40.71, 2, 'ENSG00000278233', ''),
(2826, 'RNA5-8SN3', '45S ribosomal DNA (rDNA) arrays, or clusters, are present on human chromosomes 13, 14, 15, 21 and 22, designated RNR1 through RNR5, respectively. Each cluster consists of multiple 45S rDNA repeat units that vary in number among individuals and chromosomes, with total diploid copy number estimates ranging from 60 to >800 repeat units in a human genome. The 45S rDNA repeat unit encodes a 45S rRNA precursor, transcribed by RNA polymerase I, which is processed to form the 18S, 5.8S and 28S rRNAs. This gene represents a copy of the 5.8S ribosomal RNA on chromosome 21. [provided by RefSeq, Mar 2017]', '21', 'processed_pseudogene', 50172089, 50173181, 46.48, 1, 'ENSG00000275215', ''),
(2827, 'RNA5-8SN3', '45S ribosomal DNA (rDNA) arrays, or clusters, are present on human chromosomes 13, 14, 15, 21 and 22, designated RNR1 through RNR5, respectively. Each cluster consists of multiple 45S rDNA repeat units that vary in number among individuals and chromosomes, with total diploid copy number estimates ranging from 60 to >800 repeat units in a human genome. The 45S rDNA repeat unit encodes a 45S rRNA precursor, transcribed by RNA polymerase I, which is processed to form the 18S, 5.8S and 28S rRNAs. This gene represents a copy of the 5.8S ribosomal RNA on chromosome 21. [provided by RefSeq, Mar 2017]', '21', 'processed_pseudogene', 50172089, 50173181, 46.48, 1, 'ENSG00000288387', ''),
(2828, 'RNA5S1', 'Predicted to be a structural constituent of ribosome. Predicted to be located in ribosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'transcribed_unprocessed_pseudogene', 18128610, 18136886, 35.45, 1, 'ENSG00000285166', ''),
(2829, 'RNA5S1', 'Predicted to be a structural constituent of ribosome. Predicted to be located in ribosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'transcribed_unprocessed_pseudogene', 18128610, 18136886, 35.45, 1, 'ENSG00000199352', ''),
(2830, 'RNA5S16', 'Predicted to be a structural constituent of ribosome. Predicted to be located in ribosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'transcribed_unprocessed_pseudogene', 19567382, 19604120, 36.66, 1, 'ENSG00000284853', ''),
(2831, 'RNA5S16', 'Predicted to be a structural constituent of ribosome. Predicted to be located in ribosome. [provided by Alliance of Genome Resources, May 2025]', '1', 'transcribed_unprocessed_pseudogene', 19567382, 19604120, 36.66, 1, 'ENSG00000202257', ''),
(2832, 'RNASE13', 'Predicted to enable nucleic acid binding activity. Predicted to be involved in defense response to Gram-positive bacterium. Predicted to be located in extracellular region. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 83969748, 83969857, 41.82, 1, 'ENSG00000206150', 'Q5GAN3'),
(2833, 'RNASE3', 'The protein encoded by this gene belongs to the pancreatic ribonuclease family, a subset of the ribonuclease A superfamily. The protein exhibits antimicrobial activity against pathogenic bacteria [provided by RefSeq, Oct 2014]', '14', 'protein_coding', 23598926, 23662911, 44.28, 8, 'ENSG00000169397', 'P12724'),
(2834, 'RNASEH1', 'This gene encodes an endonuclease that specifically degrades the RNA of RNA-DNA hybrids and plays a key role in DNA replication and repair. Alternate in-frame start codon initiation results in the production of alternate isoforms that are directed to the mitochondria or to the nucleus. The production of the mitochondrial isoform is modulated by an upstream open reading frame (uORF). Mutations in this gene have been found in individuals with progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2. Alternative splicing results in additional coding and non-coding transcript variants. Pseudogenes of this gene have been defined on chromosomes 2 and 17. [provided by RefSeq, Jul 2017]', '2', 'protein_coding', 25174802, 25602229, 36.98, 4, 'ENSG00000171865', 'O60930'),
(2835, 'RNF103-CHMP3', 'This locus represents naturally occurring read-through transcription between the neighboring RNF103 (ring finger protein 103) and CHMP3 (charged multivesicular body protein 3) genes. The read-through transcript encodes a fusion protein that shares sequence identity with each individual gene product. [provided by RefSeq, Oct 2011]', '2', 'protein_coding', 10627501, 10694930, 48.13, 3, 'ENSG00000249884', 'A0A140T963'),
(2836, 'RNF125', 'This gene encodes a novel E3 ubiquitin ligase that contains a RING finger domain in the N-terminus and three zinc-binding and one ubiquitin-interacting motif in the C-terminus. As a result of myristoylation, this protein associates with membranes and is primarily localized to intracellular membrane systems. The encoded protein may function as a positive regulator in the T-cell receptor signaling pathway. [provided by RefSeq, Mar 2012]', '18', 'protein_coding', 95943331, 95968720, 43.76, 4, 'ENSG00000101695', 'Q96EQ8'),
(2837, 'RNF144B', 'Enables ubiquitin-protein transferase activity. Involved in negative regulation of apoptotic process and ubiquitin-dependent protein catabolic process. Located in mitochondrial membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 317456, 321656, 43.35, 2, 'ENSG00000137393', 'Q7Z419'),
(2838, 'RNF151', 'Predicted to enable enzyme binding activity; identical protein binding activity; and zinc ion binding activity. Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 96363452, 96387288, 38.94, 7, 'ENSG00000291936', 'Q2KHN1'),
(2839, 'RNF151', 'Predicted to enable enzyme binding activity; identical protein binding activity; and zinc ion binding activity. Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 96363452, 96387288, 38.94, 7, 'ENSG00000179580', 'Q2KHN1'),
(2840, 'RNF183', 'Enables ubiquitin protein ligase activity. Involved in positive regulation of endoplasmic reticulum stress-induced intrinsic apoptotic signaling pathway; protein ubiquitination; and response to endoplasmic reticulum stress. Located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 34179005, 34252523, 42.80, 5, 'ENSG00000165188', 'Q96D59'),
(2841, 'RNF214', 'Predicted to enable ubiquitin-protein transferase activity. [provided by Alliance of Genome Resources, May 2025]', '11', 'unprocessed_pseudogene', 55673536, 55674675, 35.70, 1, 'ENSG00000167257', 'Q8ND24'),
(2842, 'RNF224', 'Predicted to enable zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '9', 'unprocessed_pseudogene', 28033947, 28034855, 47.74, 1, 'ENSG00000233198', 'P0DH78'),
(2843, 'RNF6', 'The protein encoded by this gene contains a RING-H2 finger motif. Deletions and mutations in this gene were detected in esophageal squamous cell carcinoma (ESCC), suggesting that this protein may be a potential tumor suppressor. Studies of the mouse counterpart suggested a role of this protein in the transcription regulation that controls germinal differentiation. Multiple alternatively spliced transcript variants encoding the same protein are observed. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 32783540, 32789201, 39.49, 1, 'ENSG00000127870', 'Q9Y252'),
(2844, 'RNMT', 'Enables RNA binding activity and mRNA 5\'-cap (guanine-N7-)-methyltransferase activity. Involved in 7-methylguanosine mRNA capping. Located in fibrillar center and nucleoplasm. Part of mRNA cap methyltransferase RNMT:RAMAC complex; mRNA capping enzyme complex; and receptor complex. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 79225694, 79326061, 38.12, 3, 'ENSG00000101654', 'O43148'),
(2845, 'RNPC3', 'Two types of spliceosomes catalyze splicing of pre-mRNAs. The major U2-type spliceosome is found in all eukaryotes and removes U2-type introns, which represent more than 99% of pre-mRNA introns. The minor U12-type spliceosome is found in some eukaryotes and removes U12-type introns, which are rare and have distinct splice consensus signals. The U12-type spliceosome consists of several small nuclear RNAs and associated proteins. This gene encodes a 65K protein that is a component of the U12-type spliceosome. This protein contains two RNA recognition motifs (RRMs), suggesting that it may contact one of the small nuclear RNAs of the minor spliceosome. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 37250502, 37324833, 36.91, 10, 'ENSG00000185946', 'Q96LT9'),
(2846, 'RNU1-1', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 137021, 141806, 52.24, 2, 'ENSG00000291389', ''),
(2847, 'RNU1-1', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 137021, 141806, 52.24, 2, 'ENSG00000206652', ''),
(2848, 'RNU1-2', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 1336616, 1382689, 54.54, 3, 'ENSG00000291385', ''),
(2849, 'RNU1-2', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 1336616, 1382689, 54.54, 3, 'ENSG00000207005', ''),
(2850, 'RNU12', 'Predicted to enable pre-mRNA branch point binding activity. Predicted to be involved in negative regulation of neuron apoptotic process. Predicted to act upstream of or within RNA splicing. Predicted to be part of U12 snRNP. Implicated in autosomal recessive spinocerebellar ataxia 33. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 40393766, 40414793, 55.84, 30, 'ENSG00000276027', '');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2851, 'RNU6-2', 'Predicted to enable U4 snRNA binding activity. Predicted to be involved in formation of quadruple SL/U4/U5/U6 snRNP and spliceosomal tri-snRNP complex assembly. Predicted to be part of U4/U6 x U5 tri-snRNP complex and U6 snRNP. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 167916675, 167937072, 37.94, 3, 'ENSG00000207357', ''),
(2852, 'RNU6-8', 'Predicted to enable U4 snRNA binding activity. Predicted to be involved in formation of quadruple SL/U4/U5/U6 snRNP and spliceosomal tri-snRNP complex assembly. Predicted to be part of U4/U6 x U5 tri-snRNP complex and U6 snRNP. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 55811367, 55812476, 46.22, 1, 'ENSG00000202337', ''),
(2853, 'RNVU1-26', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 38601809, 38651035, 48.56, 3, 'ENSG00000273990', ''),
(2854, 'RNVU1-26', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 38601809, 38651035, 48.56, 3, 'ENSG00000275291', ''),
(2855, 'ROMO1', 'The protein encoded by this gene is a mitochondrial membrane protein that is responsible for increasing the level of reactive oxygen species (ROS) in cells. The protein also has antimicrobial activity against a variety of bacteria by inducing bacterial membrane breakage. [provided by RefSeq, Nov 2014]', '20', 'unprocessed_pseudogene', 55957216, 55957630, 36.14, 1, 'ENSG00000125995', 'P60602'),
(2856, 'ROR2', 'The protein encoded by this gene is a member of the NR1 subfamily of nuclear hormone receptors. It can bind as a monomer or as a homodimer to hormone response elements upstream of several genes to enhance the expression of those genes. The encoded protein has been shown to interact with NM23-2, a nucleoside diphosphate kinase involved in organogenesis and differentiation, as well as with NM23-1, the product of a tumor metastasis suppressor candidate gene. Also, it has been shown to aid in the transcriptional regulation of some genes involved in circadian rhythm. Four transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Feb 2014]', '15', 'protein_coding', 74250846, 74291973, 44.27, 7, 'ENSG00000169071', 'Q01974'),
(2857, 'RPEL1', 'The protein encoded by this gene is a member of the phosphatase and actin regulator family of proteins. This family member can bind actin and regulate the reorganization of the actin cytoskeleton. It plays a role in tubule formation and in endothelial cell survival. Polymorphisms in this gene are associated with susceptibility to myocardial infarction, coronary artery disease and cervical artery dissection. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Apr 2016]', '6', 'protein_coding', 137191617, 137204193, 62.56, 4, 'ENSG00000235376', 'Q9C0D0'),
(2858, 'RPF2', 'Enables 5S rRNA binding activity. Involved in protein localization to nucleolus; regulation of signal transduction by p53 class mediator; and ribosomal large subunit biogenesis. Located in chromosome; nucleolus; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 4856253, 4911692, 42.33, 1, 'ENSG00000197498', 'Q9H7B2'),
(2859, 'RPL10', 'This gene encodes a ribosomal protein that is a component of the 60S ribosome subunit. The related protein in chicken can bind to c-Jun and can repress c-Jun-mediated transcriptional activation. Some studies have detected an association between variation in this gene and autism spectrum disorders, though others do not detect this relationship. There are multiple pseudogenes of this gene dispersed throughout the genome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2015]', 'X', 'protein_coding', 7389734, 7404097, 53.68, 4, 'ENSG00000147403', 'P27635'),
(2860, 'RPL17', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein belongs to the L14P family of ribosomal proteins. It is located in the cytoplasm. This gene has been referred to as rpL17 because the encoded protein shares amino acid identity with ribosomal protein L17 from Saccharomyces cerevisiae; however, its official symbol is RPL23. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 75293698, 75433503, 42.30, 11, 'ENSG00000265681', 'P18621'),
(2861, 'RPL22L1', 'Predicted to enable RNA binding activity. Predicted to be a structural constituent of ribosome. Predicted to be involved in cytoplasmic translation. Predicted to be located in ribosome. Predicted to be part of ribonucleoprotein complex. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 87724051, 87731469, 45.11, 43, 'ENSG00000163584', 'Q6P5R6'),
(2862, 'RPL31', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein belongs to the L31E family of ribosomal proteins. It is located in the cytoplasm. Higher levels of expression of this gene in familial adenomatous polyps compared to matched normal tissues have been observed. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 50510321, 50585241, 43.56, 2, 'ENSG00000071082', 'P62899'),
(2863, 'RPL36A', 'Cytoplasmic ribosomes, organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein, which shares sequence similarity with yeast ribosomal protein L44, belongs to the L44E (L36AE) family of ribosomal proteins. Although this gene has been referred to as ribosomal protein L44 (RPL44), its official name is ribosomal protein L36a (RPL36A). This gene and the human gene officially named ribosomal protein L36a-like (RPL36AL) encode nearly identical proteins; however, they are distinct genes. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. Naturally occurring read-through transcription occurs between this locus and the heterogeneous nuclear ribonucleoprotein H2 (H\') gene. [provided by RefSeq, Jan 2011]', 'X', 'miRNA', 90869121, 90869202, 42.68, 1, 'ENSG00000241343', 'P83881'),
(2864, 'RPL37AP8', 'Predicted to enable zinc ion binding activity. Predicted to be a structural constituent of ribosome. Predicted to be involved in translation. Predicted to be located in ribosome. Predicted to be part of cytosolic large ribosomal subunit. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 66552587, 66579135, 42.55, 6, 'ENSG00000176343', 'A6NKH3'),
(2865, 'RPL7A', 'Cytoplasmic ribosomes, organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein belongs to the L7AE family of ribosomal proteins. It can interact with a subclass of nuclear hormone receptors, including thyroid hormone receptor, and inhibit their ability to transactivate by preventing their binding to their DNA response elements. This gene is included in the surfeit gene cluster, a group of very tightly linked genes that do not share sequence similarity. It is co-transcribed with the U24, U36a, U36b, and U36c small nucleolar RNA genes, which are located in its second, fifth, fourth, and sixth introns, respectively. This gene rearranges with the trk proto-oncogene to form the chimeric oncogene trk-2h, which encodes an oncoprotein consisting of the N terminus of ribosomal protein L7a fused to the receptor tyrosine kinase domain of trk. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 37562583, 37598844, 32.94, 10, 'ENSG00000280858', 'P62424'),
(2866, 'RPL7A', 'Cytoplasmic ribosomes, organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein belongs to the L7AE family of ribosomal proteins. It can interact with a subclass of nuclear hormone receptors, including thyroid hormone receptor, and inhibit their ability to transactivate by preventing their binding to their DNA response elements. This gene is included in the surfeit gene cluster, a group of very tightly linked genes that do not share sequence similarity. It is co-transcribed with the U24, U36a, U36b, and U36c small nucleolar RNA genes, which are located in its second, fifth, fourth, and sixth introns, respectively. This gene rearranges with the trk proto-oncogene to form the chimeric oncogene trk-2h, which encodes an oncoprotein consisting of the N terminus of ribosomal protein L7a fused to the receptor tyrosine kinase domain of trk. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '9', 'protein_coding', 37562583, 37598844, 32.94, 10, 'ENSG00000148303', 'P62424'),
(2867, 'RPL9', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 60S subunit. The protein belongs to the L6P family of ribosomal proteins. It is located in the cytoplasm. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]', '4', 'snoRNA', 133350847, 133350912, 40.91, 1, 'ENSG00000163682', 'P32969'),
(2868, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000241779', 'Q9H633'),
(2869, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000241863', 'Q9H633'),
(2870, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000243009', 'Q9H633'),
(2871, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000242726', 'Q9H633'),
(2872, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000239865', 'Q9H633'),
(2873, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000239927', 'Q9H633'),
(2874, 'RPP21', 'RPP21 is a protein subunit of nuclear ribonuclease P, which processes the 5-prime leader sequence of precursor tRNAs (Jarrous et al., 2001 [PubMed 11497433]).[supplied by OMIM, Jan 2009]', '6', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000241370', 'Q9H633'),
(2875, 'RPS11', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a member of the S17P family of ribosomal proteins that is a component of the 40S subunit. This gene is co-transcribed with the small nucleolar RNA gene U35B, which is located in the third intron. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed throughout the genome. [provided by RefSeq, Jul 2012]', '19', 'protein_coding', 45890144, 45898949, 56.09, 40, 'ENSG00000142534', 'P62280'),
(2876, 'RPS17', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of four RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S17E family of ribosomal proteins and is located in the cytoplasm. Mutations in this gene cause Diamond-Blackfan anemia 4. Alternative splicing of this gene results in multiple transcript variants. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Apr 2014]', '15', 'protein_coding', 81539885, 81553961, 61.60, 7, 'ENSG00000278229', 'P08708'),
(2877, 'RPS17', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of four RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S17E family of ribosomal proteins and is located in the cytoplasm. Mutations in this gene cause Diamond-Blackfan anemia 4. Alternative splicing of this gene results in multiple transcript variants. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Apr 2014]', '15', 'protein_coding', 81539885, 81553961, 61.60, 7, 'ENSG00000182774', 'P08708'),
(2878, 'RPS25', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S25E family of ribosomal proteins. It is located in the cytoplasm. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 485154, 693139, 40.51, 3, 'ENSG00000280831', 'P62851'),
(2879, 'RPS25', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S25E family of ribosomal proteins. It is located in the cytoplasm. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 485154, 693139, 40.51, 3, 'ENSG00000118181', 'P62851'),
(2880, 'RPS4Y2', 'The protein encoded by this gene is a ribosomal protein that is highly similar to RPS4Y1. This gene is located in the male-specific region of the Y chromosome. [provided by RefSeq, Aug 2012]', 'Y', 'protein_coding', 9527880, 9530682, 53.34, 4, 'ENSG00000280969', 'Q8TD47'),
(2881, 'RPS6KA4', 'This gene encodes a member of the RSK (ribosomal S6 kinase) family of serine/threonine kinases. This kinase contains 2 non-identical kinase catalytic domains and phosphorylates various substrates, including CREB1 and ATF1. The encoded protein can also phosphorylate histone H3 to regulate certain inflammatory genes. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2016]', '11', 'protein_coding', 142449007, 142578733, 37.11, 22, 'ENSG00000162302', 'O75676'),
(2882, 'RPS7', 'Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S7E family of ribosomal proteins. It is located in the cytoplasm. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 2241003, 2242127, 45.51, 1, 'ENSG00000171863', 'P62081'),
(2883, 'RRAGC', 'This gene encodes a member of the GTR/RAG GTP-binding protein family. The encoded protein is a monomeric guanine nucleotide-binding protein which forms a heterodimer with RRAGA and RRAGB and is primarily localized to the cytoplasm. The encoded protein promotes intracellular localization of the mTOR complex. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2012]', '1', 'protein_coding', 2461634, 2470853, 46.19, 10, 'ENSG00000116954', 'Q9HB90'),
(2884, 'RRBP1', 'This gene encodes a ribosome-binding protein of the endoplasmic reticulum (ER) membrane. Studies suggest that this gene plays a role in ER proliferation, secretory pathways and secretory cell differentiation, and mediation of ER-microtubule interactions. Alternative splicing has been observed and protein isoforms are characterized by regions of N-terminal decapeptide and C-terminal heptad repeats. Splicing of the tandem repeats results in variations in ribosome-binding affinity and secretory function. The full-length nature of variants which differ in repeat length has not been determined. Pseudogenes of this gene have been identified on chromosomes 3 and 7, and RRBP1 has been excluded as a candidate gene in the cause of Alagille syndrome, the result of a mutation in a nearby gene on chromosome 20p12. [provided by RefSeq, Apr 2012]', '20', 'protein_coding', 70114332, 70162537, 47.10, 11, 'ENSG00000125844', 'Q9P2E9'),
(2885, 'RSPO4', 'This gene encodes a member of the R-spondin family of proteins that share a common domain organization consisting of a signal peptide, cysteine-rich/furin-like domain, thrombospondin domain and a C-terminal basic region. The encoded protein may be involved in activation of Wnt/beta-catenin signaling pathways. Mutations in this gene are associated with anonychia congenital. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Sep 2009]', '20', 'protein_coding', 39223359, 39244982, 58.15, 4, 'ENSG00000101282', 'Q2I0M5'),
(2886, 'RTEL1', 'This gene encodes a DNA helicase which functions in the stability, protection and elongation of telomeres and interacts with proteins in the shelterin complex known to protect telomeres during DNA replication. Mutations in this gene have been associated with dyskeratosis congenita and Hoyerall-Hreidarsson syndrome. Read-through transcription of this gene into the neighboring downstream gene, which encodes tumor necrosis factor receptor superfamily, member 6b, generates a non-coding transcript. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Sep 2013]', '20', 'protein_coding', 18567347, 18569563, 44.25, 3, 'ENSG00000258366', 'Q9NZ71'),
(2887, 'RTL1', 'This gene is a retrotransposon-derived, paternally expressed imprinted gene that is highly expressed at the late fetal stage in both the fetus and placenta. It has an overlapping maternally expressed antisense transcript, which contains several microRNAs targeting the transcripts of this gene through an RNA interference (RNAi) mechanism. This gene is essential for maintenance of the fetal capillaries. [provided by RefSeq, Jul 2009]', '14', 'miRNA', 2521689, 2521759, 33.80, 1, 'ENSG00000254656', 'A6NKG5'),
(2888, 'RTL4', 'Predicted to enable nucleic acid binding activity and zinc ion binding activity. Predicted to act upstream of or within cognition and norepinephrine metabolic process. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 85530144, 85654428, 39.33, 11, 'ENSG00000187823', 'Q6ZR62'),
(2889, 'RTL8C', 'Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 167220876, 167427345, 37.96, 17, 'ENSG00000134590', 'A6ZKI3'),
(2890, 'RTN1', 'This gene belongs to the family of reticulon encoding genes. Reticulons are associated with the endoplasmic reticulum, and are involved in neuroendocrine secretion or in membrane trafficking in neuroendocrine cells. This gene is considered to be a specific marker for neurological diseases and cancer, and is a potential molecular target for therapy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2011]', '14', 'miRNA', 53690881, 53690965, 48.24, 1, 'ENSG00000139970', 'Q16799'),
(2891, 'RUBCNL', 'This gene encodes a cysteine-rich protein that contains a putative zinc-RING and/or ribbon domain. The encoded protein is related to Run domain Beclin-1-interacting and cysteine-rich domain-containing protein, which plays a role in endocytic trafficking and autophagy. In cervical cancer cell lines, this gene is expressed at low levels and may function as a tumor suppressor. Promoter hypermethylation of this gene is observed in cervical cancer cell lines and tissue derived from human patients. [provided by RefSeq, Mar 2017]', '13', 'protein_coding', 47370578, 47414498, 44.90, 1, 'ENSG00000102445', 'Q9H714'),
(2892, 'RUNX1', 'Core binding factor (CBF) is a heterodimeric transcription factor that binds to the core element of many enhancers and promoters. The protein encoded by this gene represents the alpha subunit of CBF and is thought to be involved in the development of normal hematopoiesis. Chromosomal translocations involving this gene are well-documented and have been associated with several types of leukemia. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 74434775, 74457944, 49.67, 2, 'ENSG00000159216', 'Q01196'),
(2893, 'RXFP2', 'This gene encodes a member of the GPCR (G protein-coupled, 7-transmembrane receptor) family. Mutations in this gene are associated with cryptorchidism. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Oct 2009]', '13', 'TR_C_gene', 22462932, 22465787, 43.66, 1, 'ENSG00000133105', 'Q8WXD0'),
(2894, 'RXFP3', 'Predicted to enable G protein-coupled peptide receptor activity. Involved in positive regulation of cytokinesis. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 180847611, 180861285, 42.93, 7, 'ENSG00000182631', 'Q9NSD7'),
(2895, 'RXFP3', 'Predicted to enable G protein-coupled peptide receptor activity. Involved in positive regulation of cytokinesis. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 180847611, 180861285, 42.93, 7, 'ENSG00000277069', 'Q9NSD7'),
(2896, 'RXRA', 'Retinoid X receptors (RXRs) and retinoic acid receptors (RARs) are nuclear receptors that mediate the biological effects of retinoids by their involvement in retinoic acid-mediated gene activation. These receptors function as transcription factors by binding as homodimers or heterodimers to specific sequences in the promoters of target genes. The protein encoded by this gene is a member of the steroid and thyroid hormone receptor superfamily of transcriptional regulators. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, May 2014]', '9', 'miRNA', 124692442, 124692551, 50.00, 1, 'ENSG00000186350', 'P19793'),
(2897, 'RYBP', 'Enables nucleic acid binding activity. Involved in chromatin remodeling; positive regulation of apoptotic process; and regulation of primary metabolic process. Located in nucleoplasm. Part of PcG protein complex. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 38805183, 38824955, 46.16, 9, 'ENSG00000281766', 'Q8N488'),
(2898, 'RYBP', 'Enables nucleic acid binding activity. Involved in chromatin remodeling; positive regulation of apoptotic process; and regulation of primary metabolic process. Located in nucleoplasm. Part of PcG protein complex. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 38805183, 38824955, 46.16, 9, 'ENSG00000163602', 'Q8N488'),
(2899, 'S100G', 'This gene encodes calbindin D9K, a vitamin D-dependent calcium-binding protein. This cytosolic protein belongs to a family of calcium-binding proteins that includes calmodulin, parvalbumin, troponin C, and S100 protein. In the intestine, the protein is vitamin D-dependent and its expression correlates with calcium transport activity. The protein may increase Ca2+ absorption by buffering Ca2+ in the cytoplasm and increase ATP-dependent Ca2+ transport in duodenal basolateral membrane vesicles. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 5455417, 5456856, 64.93, 1, 'ENSG00000169906', 'P29377'),
(2900, 'S1PR2', 'This gene encodes a member of the G protein-coupled receptors, as well as the EDG family of proteins. The encoded protein is a receptor for sphingosine 1-phosphate, which participates in cell proliferation, survival, and transcriptional activation. Defects in this gene have been associated with congenital profound deafness. [provided by RefSeq, Mar 2016]', '19', 'protein_coding', 23992773, 24135970, 41.61, 10, 'ENSG00000267534', 'O95136'),
(2901, 'SAA1', 'This gene encodes a member of the serum amyloid A family of apolipoproteins. The encoded preproprotein is proteolytically processed to generate the mature protein. This protein is a major acute phase protein that is highly expressed in response to inflammation and tissue injury. This protein also plays an important role in HDL metabolism and cholesterol homeostasis. High levels of this protein are associated with chronic inflammatory diseases including atherosclerosis, rheumatoid arthritis, Alzheimer\'s disease and Crohn\'s disease. This protein may also be a potential biomarker for certain tumors. Finally, antimicrobial activity against S. aureus and E. coli resides in the N-terminal portion of the mature protein. Alternate splicing results in multiple transcript variants that encode the same protein. A pseudogene of this gene is found on chromosome 11. [provided by RefSeq, Jul 2020]', '11', 'protein_coding', 38962299, 39039723, 41.33, 8, 'ENSG00000288411', 'P0DJI8'),
(2902, 'SAA1', 'This gene encodes a member of the serum amyloid A family of apolipoproteins. The encoded preproprotein is proteolytically processed to generate the mature protein. This protein is a major acute phase protein that is highly expressed in response to inflammation and tissue injury. This protein also plays an important role in HDL metabolism and cholesterol homeostasis. High levels of this protein are associated with chronic inflammatory diseases including atherosclerosis, rheumatoid arthritis, Alzheimer\'s disease and Crohn\'s disease. This protein may also be a potential biomarker for certain tumors. Finally, antimicrobial activity against S. aureus and E. coli resides in the N-terminal portion of the mature protein. Alternate splicing results in multiple transcript variants that encode the same protein. A pseudogene of this gene is found on chromosome 11. [provided by RefSeq, Jul 2020]', '11', 'protein_coding', 38962299, 39039723, 41.33, 8, 'ENSG00000173432', 'P0DJI8'),
(2903, 'SAFB', 'This gene encodes a DNA-binding protein which has high specificity for scaffold or matrix attachment region DNA elements (S/MAR DNA). This protein is thought to be involved in attaching the base of chromatin loops to the nuclear matrix but there is conflicting evidence as to whether this protein is a component of chromatin or a nuclear matrix protein. Scaffold attachment factors are a specific subset of nuclear matrix proteins (NMP) that specifically bind to S/MAR. The encoded protein is thought to serve as a molecular base to assemble a \'transcriptosome complex\' in the vicinity of actively transcribed genes. It is involved in the regulation of heat shock protein 27 transcription, can act as an estrogen receptor co-repressor and is a candidate for breast tumorigenesis. This gene is arranged head-to-head with a similar gene whose product has the same functions. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2011]', '19', 'protein_coding', 96889315, 96897891, 38.15, 3, 'ENSG00000160633', 'Q15424'),
(2904, 'SAMD1', 'Enables chromatin binding activity. Involved in negative regulation of transcription initiation-coupled chromatin remodeling and protein homooligomerization. Is active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 316737, 500722, 43.13, 3, 'ENSG00000288488', 'Q6SPF0'),
(2905, 'SAMD1', 'Enables chromatin binding activity. Involved in negative regulation of transcription initiation-coupled chromatin remodeling and protein homooligomerization. Is active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 316737, 500722, 43.13, 3, 'ENSG00000141858', 'Q6SPF0'),
(2906, 'SAMD4A', 'Sterile alpha motifs (SAMs) in proteins such as SAMD4A are part of an RNA-binding domain that functions as a posttranscriptional regulator by binding to an RNA sequence motif known as the Smaug recognition element, which was named after the Drosophila Smaug protein (Baez and Boccaccio, 2005 [PubMed 16221671]).[supplied by OMIM, Mar 2008]', '14', 'protein_coding', 128149071, 128153453, 60.89, 7, 'ENSG00000020577', 'Q9UPU9'),
(2907, 'SAP30', 'Histone acetylation plays a key role in the regulation of eukaryotic gene expression. Histone acetylation and deacetylation are catalyzed by multisubunit complexes. The protein encoded by this gene is a component of the histone deacetylase complex, which includes SIN3, SAP18, HDAC1, HDAC2, RbAp46, RbAp48, and other polypeptides. This complex is active in deacetylating core histone octamers, but inactive in deacetylating nucleosomal histones. A pseudogene of this gene is located on chromosome 3. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 119468963, 119494708, 41.15, 9, 'ENSG00000164105', 'O75446'),
(2908, 'SART1', 'This gene encodes two proteins, the SART1(800) protein expressed in the nucleus of the majority of proliferating cells, and the SART1(259) protein expressed in the cytosol of epithelial cancers. The SART1(259) protein is translated by the mechanism of -1 frameshifting during posttranscriptional regulation; its full-length sequence is not published yet. The two encoded proteins are thought to be involved in the regulation of proliferation. Both proteins have tumor-rejection antigens. The SART1(259) protein possesses tumor epitopes capable of inducing HLA-A2402-restricted cytotoxic T lymphocytes in cancer patients. This SART1(259) antigen may be useful in specific immunotherapy for cancer patients and may serve as a paradigmatic tool for the diagnosis and treatment of patients with atopy. The SART1(259) protein is found to be essential for the recruitment of the tri-snRNP to the pre-spliceosome in the spliceosome assembly pathway. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 34516395, 34540958, 48.33, 5, 'ENSG00000175467', 'O43290'),
(2909, 'SASH3', 'The protein encoded by this gene contains a Src homology-3 (SH3) domain and a sterile alpha motif (SAM), both of which are found in proteins involved in cell signaling. This protein may function as a signaling adapter protein in lymphocytes.[provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 161416297, 161425870, 47.57, 6, 'ENSG00000122122', 'O75995'),
(2910, 'SATL1', 'Predicted to enable spermidine binding activity. Predicted to be involved in spermidine acetylation. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 60378530, 60417756, 38.93, 11, 'ENSG00000184788', 'Q86VE3'),
(2911, 'SAXO1', 'Enables microtubule binding activity. Involved in several processes, including cold acclimation; positive regulation of cilium assembly; and protein stabilization. Located in microtubule cytoskeleton and sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 129738949, 129769536, 49.74, 3, 'ENSG00000155875', 'Q8IYX7'),
(2912, 'SCAF1', 'Enables RNA polymerase II C-terminal domain binding activity. Predicted to be involved in RNA splicing; mRNA processing; and transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 35302566, 35412031, 43.00, 23, 'ENSG00000126461', 'O14548'),
(2913, 'SCAF8', 'Enables RNA binding activity and RNA polymerase II C-terminal domain phosphoserine binding activity. Involved in negative regulation of termination of RNA polymerase II transcription, poly(A)-coupled and positive regulation of DNA-templated transcription, elongation. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 128702503, 128720916, 57.67, 3, 'ENSG00000213079', 'Q9UPN6'),
(2914, 'SCAND1', 'This gene encodes a SCAN box domain-containing protein. The SCAN domain is a highly conserved, leucine-rich motif of approximately 60 aa originally found within a subfamily of zinc finger proteins. This gene belongs to a family of genes that encode an isolated SCAN domain, but no zinc finger motif. This protein binds to and may regulate the function of the transcription factor myeloid zinc finger 1B. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Jan 2011]', '20', 'protein_coding', 128340527, 128361470, 54.79, 2, 'ENSG00000171222', 'P57086'),
(2915, 'SCAND2P', 'Predicted to enable DNA-binding transcription factor activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100969708, 100990831, 44.49, 4, 'ENSG00000291642', 'Q9GZW5'),
(2916, 'SCAND2P', 'Predicted to enable DNA-binding transcription factor activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 100969708, 100990831, 44.49, 4, 'ENSG00000176700', 'Q9GZW5'),
(2917, 'SCARNA12', 'This gene produces a small nuclear RNA that localizes specifically to Cajal bodies, which are conserved subnuclear organelles that are present in the nucleoplasm. This RNA is processed from an intron of the prohibitin 2 host gene. It includes both an H/ACA box and a C/D box, and is thought to guide the pseudouridylation of residue U46 in the U5 small nuclear RNA. [provided by RefSeq, Jun 2010]', '12', 'TR_V_gene', 142455174, 142455635, 51.95, 1, 'ENSG00000238795', ''),
(2918, 'SCARNA16', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 138123758, 138123821, 54.69, 1, 'ENSG00000275143', ''),
(2919, 'SCARNA5', 'Predicted to be involved in RNA processing. Located in Cajal body. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 105919, 106856, 45.52, 1, 'ENSG00000280652', ''),
(2920, 'SCARNA5', 'Predicted to be involved in RNA processing. Located in Cajal body. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 105919, 106856, 45.52, 1, 'ENSG00000252010', ''),
(2921, 'SCARNA6', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 94334, 106255, 48.26, 11, 'ENSG00000281388', ''),
(2922, 'SCARNA6', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 94334, 106255, 48.26, 11, 'ENSG00000251791', ''),
(2923, 'SCG5', 'This gene encodes a secreted chaperone protein that prevents the aggregation of other secreted proteins, including proteins that are associated with neurodegenerative and metabolic disease. The encoded protein may be best known for its role in the trafficking and activation of prohormone convertase PC2 (encoded by Gene ID: 5126). Phosphorylation of the encoded protein has been shown to have an inhibitory effect on its chaperone function. This gene also produces a ARHGAP11A-SCG5 readthrough transcript and ARHGAP11A-SCG5 protein. [provided by RefSeq, Feb 2019]', '15', 'protein_coding', 21197450, 21214510, 44.99, 5, 'ENSG00000281931', 'P05408'),
(2924, 'SCG5', 'This gene encodes a secreted chaperone protein that prevents the aggregation of other secreted proteins, including proteins that are associated with neurodegenerative and metabolic disease. The encoded protein may be best known for its role in the trafficking and activation of prohormone convertase PC2 (encoded by Gene ID: 5126). Phosphorylation of the encoded protein has been shown to have an inhibitory effect on its chaperone function. This gene also produces a ARHGAP11A-SCG5 readthrough transcript and ARHGAP11A-SCG5 protein. [provided by RefSeq, Feb 2019]', '15', 'protein_coding', 21197450, 21214510, 44.99, 5, 'ENSG00000277614', 'P05408'),
(2925, 'SCG5', 'This gene encodes a secreted chaperone protein that prevents the aggregation of other secreted proteins, including proteins that are associated with neurodegenerative and metabolic disease. The encoded protein may be best known for its role in the trafficking and activation of prohormone convertase PC2 (encoded by Gene ID: 5126). Phosphorylation of the encoded protein has been shown to have an inhibitory effect on its chaperone function. This gene also produces a ARHGAP11A-SCG5 readthrough transcript and ARHGAP11A-SCG5 protein. [provided by RefSeq, Feb 2019]', '15', 'protein_coding', 21197450, 21214510, 44.99, 5, 'ENSG00000166922', 'P05408'),
(2926, 'SCGB1A1', 'This gene encodes a member of the secretoglobin family of small secreted proteins. The encoded protein has been implicated in numerous functions including anti-inflammation, inhibition of phospholipase A2 and the sequestering of hydrophobic ligands. Defects in this gene are associated with a susceptibility to asthma. [provided by RefSeq, May 2010]', '11', 'protein_coding', 54053584, 54112519, 42.31, 5, 'ENSG00000149021', 'P11684'),
(2927, 'SCN4A', 'Voltage-gated sodium channels are transmembrane glycoprotein complexes composed of a large alpha subunit with 24 transmembrane domains and one or more regulatory beta subunits. They are responsible for the generation and propagation of action potentials in neurons and muscle. This gene encodes one member of the sodium channel alpha subunit gene family. It is expressed in skeletal muscle, and mutations in this gene have been linked to several myotonia and periodic paralysis disorders. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000007314', 'P35499'),
(2928, 'SCN4B', 'The protein encoded by this gene is one of several sodium channel beta subunits. These subunits interact with voltage-gated alpha subunits to change sodium channel kinetics. The encoded transmembrane protein forms interchain disulfide bonds with SCN2A. Defects in this gene are a cause of long QT syndrome type 10 (LQT10). Three protein-coding and one non-coding transcript variant have been found for this gene.[provided by RefSeq, Mar 2009]', '11', 'protein_coding', 74471982, 74472687, 58.50, 4, 'ENSG00000177098', 'Q8IWT1'),
(2929, 'SCO2', 'Cytochrome c oxidase (COX) catalyzes the transfer of electrons from cytochrome c to molecular oxygen, which helps to maintain the proton gradient across the inner mitochondrial membrane that is necessary for aerobic ATP production. Human COX is a multimeric protein complex that requires several assembly factors; this gene encodes one of the COX assembly factors. The encoded protein is a metallochaperone that is involved in the biogenesis of cytochrome c oxidase subunit II. Mutations in this gene are associated with fatal infantile encephalocardiomyopathy and myopia 6. [provided by RefSeq, Oct 2014]', '22', 'snoRNA', 100976477, 100976551, 40.00, 1, 'ENSG00000284194', 'O43819'),
(2930, 'SCP2D1', 'Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 60643421, 60658259, 50.38, 5, 'ENSG00000132631', 'Q9UJQ7'),
(2931, 'SCPEP1', 'Predicted to enable serine-type carboxypeptidase activity. Predicted to be involved in negative regulation of blood pressure and retinoic acid metabolic process. Predicted to act upstream of or within blood vessel diameter maintenance. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 131170944, 131205428, 38.04, 4, 'ENSG00000121064', 'Q9HB40'),
(2932, 'SCRG1', 'Scrapie-responsive gene 1 is associated with neurodegenerative changes observed in transmissible spongiform encephalopathies. It may play a role in host response to prion-associated infections. The scrapie responsive protein 1 may be partly included in the membrane or secreted by the cells due to its hydrophobic N-terminus. In addition, the encoded protein can interact with bone marrow stromal cell antigen 1 (BST1) to enhance the differentiation potentials of human mesenchymal stem cells during tissue and bone regeneration. [provided by RefSeq, Jul 2016]', '4', 'protein_coding', 23765834, 23767972, 62.69, 5, 'ENSG00000164106', 'O75711'),
(2933, 'SCRN1', 'This gene likely encodes a member of the secernin family of proteins. A similar protein in rat functions in regulation of exocytosis in mast cells. Alternatively spliced transcript variants have been described. [provided by RefSeq, Mar 2009]', '7', 'protein_coding', 125146573, 125189939, 42.35, 4, 'ENSG00000136193', 'Q12765'),
(2934, 'SCRN2', 'Predicted to enable cysteine-type exopeptidase activity and dipeptidase activity. Predicted to be involved in proteolysis. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 53701227, 53701287, 45.90, 1, 'ENSG00000141295', 'Q96FV2'),
(2935, 'SCRT2', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in negative regulation of extrinsic apoptotic signaling pathway via death domain receptors and negative regulation of transcription by RNA polymerase II. Predicted to act upstream of or within regulation of neuron migration. Predicted to be located in chromatin. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 62938147, 62948475, 48.76, 2, 'ENSG00000215397', 'Q9NQ03'),
(2936, 'SEC11C', 'Enables serine-type endopeptidase activity. Involved in signal peptide processing. Located in endoplasmic reticulum membrane. Part of signal peptidase complex. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 54357906, 54385218, 34.78, 7, 'ENSG00000166562', 'Q9BY50'),
(2937, 'SEC22B', 'The protein encoded by this gene is a member of the SEC22 family of vesicle trafficking proteins. It seems to complex with SNARE and it is thought to play a role in the ER-Golgi protein trafficking. This protein has strong similarity to Mus musculus and Cricetulus griseus proteins.[provided by RefSeq, Sep 2009]', '1', 'protein_coding', 74199652, 74229834, 53.07, 11, 'ENSG00000265808', 'O75396'),
(2938, 'SEC24B', 'The protein encoded by this gene is a member of the SEC24 subfamily of the SEC23/SEC24 family, which is involved in vesicle trafficking. The encoded protein is thought to be a cargo-binding component of the COPII vesicle, and is thought to be involved in the transport of secretory proteins from the endoplasmic reticulum to the Golgi apparatus. Mutations in this gene have been associated with neural tube defects, and are thought to be a result of a disruption in interactions with the protein encoded by the VANGL planar cell polarity protein 2 (VANGL2) gene. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2015]', '4', 'protein_coding', 17493, 22237, 59.16, 2, 'ENSG00000138802', 'O95487'),
(2939, 'SEC24D', 'The protein encoded by this gene is a member of the SEC24 subfamily of the SEC23/SEC24 family, which is involved in vesicle trafficking. The encoded protein has similarity to yeast Sec24p component of COPII. COPII is the coat protein complex responsible for vesicle budding from the ER. This gene product is implicated in the shaping of the vesicle, and also in cargo selection and concentration. Mutations in this gene have been associated with Cole-Carpenter syndrome, a disorder affecting bone formation, resulting in craniofacial malformations and bones that break easily. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2015]', '4', 'miRNA', 95501, 95561, 55.74, 1, 'ENSG00000150961', 'O94855'),
(2940, 'SECISBP2', 'The protein encoded by this gene is one of the essential components of the machinery involved in co-translational insertion of selenocysteine (Sec) into selenoproteins. Sec is encoded by the UGA codon, which normally signals translation termination. The recoding of UGA as Sec codon requires a Sec insertion sequence (SECIS) element; present in the 3\' untranslated regions of eukaryotic selenoprotein mRNAs. This protein specifically binds to the SECIS element, which is stimulated by a Sec-specific translation elongation factor. Mutations in this gene have been associated with reduction in enzymatic activity of type II iodothyronine deiodinase (a selenoprotein) and abnormal thyroid hormone metabolism. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Aug 2017]', '9', 'miRNA', 47587429, 47587505, 55.84, 1, 'ENSG00000187742', 'Q96T21'),
(2941, 'SECTM1', 'This gene encodes a transmembrane and secreted protein with characteristics of a type 1a transmembrane protein. It is found in a perinuclear Golgi-like pattern and thought to be involved in hematopoietic and/or immune system processes. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 1771890, 1773150, 65.50, 3, 'ENSG00000141574', 'Q8WVN6'),
(2942, 'SEL1L2', 'Predicted to be involved in ERAD pathway. Located in cilium and nuclear speck. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 27243968, 27325959, 37.77, 13, 'ENSG00000291539', 'Q5TEA6'),
(2943, 'SEL1L2', 'Predicted to be involved in ERAD pathway. Located in cilium and nuclear speck. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 27243968, 27325959, 37.77, 13, 'ENSG00000101251', 'Q5TEA6'),
(2944, 'SEMA3C', 'This gene encodes a secreted glycoprotein that belongs to the semaphorin class 3 family of neuronal guidance cues. The encoded protein contains an N-terminal sema domain, integrin and immunoglobulin-like domains, and a C-terminal basic domain. Homodimerization and proteolytic cleavage of the C-terminal propeptide are necessary for the function of the encoded protein. It binds a neuropilin co-receptor before forming a heterotrimeric complex with an associated plexin. An increase in the expression of this gene correlates with an increase in cancer cell invasion and adhesion. Naturally occurring mutations in this gene are associated with Hirschsprung disease. [provided by RefSeq, May 2017]', '7', 'protein_coding', 107490106, 107661306, 41.61, 4, 'ENSG00000075223', 'Q99985');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(2945, 'SEMA4G', 'Semaphorins are a large family of conserved secreted and membrane associated proteins which possess a semaphorin (Sema) domain and a PSI domain (found in plexins, semaphorins and integrins) in the N-terminal extracellular portion. Based on sequence and structural similarities, semaphorins are put into eight classes: invertebrates contain classes 1 and 2, viruses have class V, and vertebrates contain classes 3-7. Semaphorins serve as axon guidance ligands via multimeric receptor complexes, some (if not all) containing plexin proteins. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2011]', '10', 'miRNA', 89063336, 89063410, 29.33, 1, 'ENSG00000095539', 'Q9NTN9'),
(2946, 'SEMA5A', 'This gene belongs to the semaphorin gene family that encodes membrane proteins containing a semaphorin domain and several thrombospondin type-1 repeats. Members of this family are involved in axonal guidance during neural development. This gene has been implicated as an autism susceptibility gene.[provided by RefSeq, Jan 2010]', '5', 'protein_coding', 3541430, 3558333, 44.20, 4, 'ENSG00000112902', 'Q13591'),
(2947, 'SEMA6A', 'Predicted to enable identical protein binding activity; signaling receptor binding activity; and transmembrane signaling receptor activity. Involved in cellular response to vascular endothelial growth factor stimulus; negative regulation of cell adhesion involved in sprouting angiogenesis; and negative regulation of signal transduction. Predicted to be located in axon and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 62196115, 62224731, 45.39, 2, 'ENSG00000092421', 'Q9H2E6'),
(2948, 'SENP6', 'Ubiquitin-like molecules (UBLs), such as SUMO1 (UBL1; MIM 601912), are structurally related to ubiquitin (MIM 191339) and can be ligated to target proteins in a similar manner as ubiquitin. However, covalent attachment of UBLs does not result in degradation of the modified proteins. SUMO1 modification is implicated in the targeting of RANGAP1 (MIM 602362) to the nuclear pore complex, as well as in stabilization of I-kappa-B-alpha (NFKBIA; MIM 164008) from degradation by the 26S proteasome. Like ubiquitin, UBLs are synthesized as precursor proteins, with 1 or more amino acids following the C-terminal glycine-glycine residues of the mature UBL protein. Thus, the tail sequences of the UBL precursors need to be removed by UBL-specific proteases, such as SENP6, prior to their conjugation to target proteins (Kim et al., 2000 [PubMed 10799485]). SENPs also display isopeptidase activity for deconjugation of SUMO-conjugated substrates (Lima and Reverter, 2008 [PubMed 18799455]).[supplied by OMIM, Jun 2009]', '6', 'protein_coding', 33642400, 33899861, 42.47, 32, 'ENSG00000112701', 'Q9GZR1'),
(2949, 'SEPHS2', 'This gene encodes an enzyme that catalyzes the production of monoselenophosphate (MSP) from selenide and ATP. MSP is the selenium donor required for synthesis of selenocysteine (Sec), which is co-translationally incorporated into selenoproteins at in-frame UGA codons that normally signal translation termination. The 3\' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. This protein is itself a selenoprotein containing a Sec residue at its active site, suggesting the existence of an autoregulatory mechanism. It is preferentially expressed in tissues implicated in the synthesis of selenoproteins and in sites of blood cell development. A pseudogene for this locus has been identified on chromosome 5. [provided by RefSeq, May 2017]', '16', 'protein_coding', 237967014, 238099412, 45.35, 2, 'ENSG00000179918', 'Q99611'),
(2950, 'SEPTIN10', 'This gene encodes a member of the septin family of cytoskeletal proteins with GTPase activity. This protein localizes to the cytoplasm and nucleus and displays GTP-binding and GTPase activity. A pseudogene for this gene is located on chromosome 8. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2012]', '2', 'snoRNA', 110848546, 110848673, 32.03, 1, 'ENSG00000186522', 'Q9P0V9'),
(2951, 'SEPTIN11', 'SEPT11 belongs to the conserved septin family of filament-forming cytoskeletal GTPases that are involved in a variety of cellular functions including cytokinesis and vesicle trafficking (Hanai et al., 2004 [PubMed 15196925]; Nagata et al., 2004 [PubMed 15485874]).[supplied by OMIM, Jul 2009]', '4', 'protein_coding', 132822141, 132830659, 59.58, 8, 'ENSG00000138758', 'Q9NVA2'),
(2952, 'SEPTIN3', 'This gene belongs to the septin family of GTPases. Members of this family are required for cytokinesis. Expression is upregulated by retinoic acid in a human teratocarcinoma cell line. The specific function of this gene has not been determined. Alternative splicing of this gene results in several transcript variants encoding different isoforms. [provided by RefSeq, May 2018]', '22', 'protein_coding', 32018825, 32088144, 42.26, 7, 'ENSG00000100167', 'Q9UH03'),
(2953, 'SEPTIN8', 'This gene is a member of the septin family of nucleotide binding proteins, originally described in yeast as cell division cycle regulatory proteins. Septins are highly conserved in yeast, Drosophila, and mouse, and appear to regulate cytoskeletal organization. Disruption of septin function disturbs cytokinesis and results in large multinucleate or polyploid cells. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2014]', '5', 'protein_coding', 24213955, 24232352, 44.50, 4, 'ENSG00000164402', 'Q92599'),
(2954, 'SERF1B', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is the centromeric copy which is identical to the telomeric copy. Alternatively spliced transcripts have been documented but it is unclear whether alternative splicing occurs for both the centromeric and telomeric copies of the gene. The gene encodes a protein of unknown function which bears low-level homology with the RNA-binding domain of matrin-cyclophilin, a protein which colocalizes with small nuclear ribonucleoproteins (snRNPs) and the SMN1 gene product. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 128947699, 129007096, 46.07, 11, 'ENSG00000277429', 'O75920'),
(2955, 'SERF1B', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is the centromeric copy which is identical to the telomeric copy. Alternatively spliced transcripts have been documented but it is unclear whether alternative splicing occurs for both the centromeric and telomeric copies of the gene. The gene encodes a protein of unknown function which bears low-level homology with the RNA-binding domain of matrin-cyclophilin, a protein which colocalizes with small nuclear ribonucleoproteins (snRNPs) and the SMN1 gene product. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 128947699, 129007096, 46.07, 11, 'ENSG00000278839', 'O75920'),
(2956, 'SERF1B', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. This gene is the centromeric copy which is identical to the telomeric copy. Alternatively spliced transcripts have been documented but it is unclear whether alternative splicing occurs for both the centromeric and telomeric copies of the gene. The gene encodes a protein of unknown function which bears low-level homology with the RNA-binding domain of matrin-cyclophilin, a protein which colocalizes with small nuclear ribonucleoproteins (snRNPs) and the SMN1 gene product. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 128947699, 129007096, 46.07, 11, 'ENSG00000205572', 'O75920'),
(2957, 'SERHL2', 'Predicted to enable hydrolase activity. Predicted to be located in cytoplasmic vesicle. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 138946724, 139293557, 44.78, 15, 'ENSG00000183569', 'Q9H4I8'),
(2958, 'SERP2', 'Predicted to be involved in endoplasmic reticulum unfolded protein response. Predicted to be located in endoplasmic reticulum membrane. Predicted to be active in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 57428187, 57447420, 44.27, 15, 'ENSG00000151778', 'Q8N6R1'),
(2959, 'SERPINA1', 'The protein encoded by this gene is a serine protease inhibitor belonging to the serpin superfamily whose targets include elastase, plasmin, thrombin, trypsin, chymotrypsin, and plasminogen activator. This protein is produced in the liver, the bone marrow, by lymphocytic and monocytic cells in lymphoid tissue, and by the Paneth cells of the gut. Defects in this gene are associated with chronic obstructive pulmonary disease, emphysema, and chronic liver disease. Several transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Aug 2020]', '14', 'protein_coding', 20017310, 20033655, 40.77, 7, 'ENSG00000277377', 'P01009'),
(2960, 'SERPINA1', 'The protein encoded by this gene is a serine protease inhibitor belonging to the serpin superfamily whose targets include elastase, plasmin, thrombin, trypsin, chymotrypsin, and plasminogen activator. This protein is produced in the liver, the bone marrow, by lymphocytic and monocytic cells in lymphoid tissue, and by the Paneth cells of the gut. Defects in this gene are associated with chronic obstructive pulmonary disease, emphysema, and chronic liver disease. Several transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Aug 2020]', '14', 'protein_coding', 20017310, 20033655, 40.77, 7, 'ENSG00000197249', 'P01009'),
(2961, 'SERPINA5', 'The protein encoded by this gene is a member of the serpin family of proteins, a group of proteins that inhibit serine proteases. This gene is one in a cluster of serpin genes located on the q arm of chromosome 14. This family member is a glycoprotein that can inhibit several serine proteases, including protein C and various plasminogen activators and kallikreins, and it thus plays diverse roles in hemostasis and thrombosis in multiple organs. [provided by RefSeq, Aug 2012]', '14', 'protein_coding', 30778456, 30787205, 57.73, 1, 'ENSG00000188488', 'P05154'),
(2962, 'SERPINB1', 'The protein encoded by this gene is a member of the serpin family of proteinase inhibitors. Members of this family maintain homeostasis by neutralizing overexpressed proteinase activity through their function as suicide substrates. This protein inhibits the neutrophil-derived proteinases neutrophil elastase, cathepsin G, and proteinase-3 and thus protects tissues from damage at inflammatory sites. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '6', 'protein_coding', 75210755, 75289950, 44.58, 5, 'ENSG00000021355', 'P30740'),
(2963, 'SERPINB13', 'The protein encoded by this gene is a member of the serpin family of serine protease inhibitors. The encoded protein inhibits the activity of cathepsin K and is itself transcriptionally repressed by RUNX1. This gene is downregulated in many types of cancer. [provided by RefSeq, Jan 2017]', '18', 'protein_coding', 34658218, 34878332, 39.67, 9, 'ENSG00000197641', 'Q9UIV8'),
(2964, 'SERPINB6', 'The protein encoded by this gene is a member of the serpin (serine proteinase inhibitor) superfamily, and ovalbumin(ov)-serpin subfamily. It was originally discovered as a placental thrombin inhibitor. The mouse homolog was found to be expressed in the hair cells of the inner ear. Mutations in this gene are associated with nonsyndromic progressive hearing loss, suggesting that this serpin plays an important role in the inner ear in the protection against leakage of lysosomal content during stress, and that loss of this protection results in cell death and sensorineural hearing loss. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2010]', '6', 'protein_coding', 139955728, 139965521, 39.88, 1, 'ENSG00000124570', 'P35237'),
(2965, 'SERPINB7', 'This gene encodes a member of a family of proteins which function as protease inhibitors. Expression of this gene is upregulated in IgA nephropathy and mutations have been found to cause palmoplantar keratoderma, Nagashima type. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2014]', '18', 'protein_coding', 301186, 302121, 44.76, 1, 'ENSG00000166396', 'O75635'),
(2966, 'SERPINE1', 'This gene encodes a member of the serine proteinase inhibitor (serpin) superfamily. This member is the principal inhibitor of tissue plasminogen activator (tPA) and urokinase (uPA), and hence is an inhibitor of fibrinolysis. The protein also functions as a component of innate antiviral immunity. Defects in this gene are the cause of plasminogen activator inhibitor-1 deficiency (PAI-1 deficiency), and high concentrations of the gene product are associated with thrombophilia. [provided by RefSeq, Aug 2020]', '7', 'miRNA', 28390982, 28391051, 57.14, 1, 'ENSG00000106366', 'P05121'),
(2967, 'SERPINF1', 'This gene encodes a member of the serpin family that does not display the serine protease inhibitory activity shown by many of the other serpin proteins. The encoded protein is secreted and strongly inhibits angiogenesis. In addition, this protein is a neurotrophic factor involved in neuronal differentiation in retinoblastoma cells. Mutations in this gene were found in individuals with osteogenesis imperfecta, type VI. [provided by RefSeq, Aug 2016]', '17', 'protein_coding', 2571557, 2616635, 36.33, 5, 'ENSG00000132386', 'P36955'),
(2968, 'SERPINF1', 'This gene encodes a member of the serpin family that does not display the serine protease inhibitory activity shown by many of the other serpin proteins. The encoded protein is secreted and strongly inhibits angiogenesis. In addition, this protein is a neurotrophic factor involved in neuronal differentiation in retinoblastoma cells. Mutations in this gene were found in individuals with osteogenesis imperfecta, type VI. [provided by RefSeq, Aug 2016]', '17', 'protein_coding', 2571557, 2616635, 36.33, 5, 'ENSG00000282307', 'P36955'),
(2969, 'SERTAD3', 'The protein encoded by this gene was identified in a yeast two-hybrid assay employing the second subunit of human replication protein A as bait. It is localized to the nucleus and its expression is significantly higher in cancer cell lines compared to normal cell lines. This protein has also been shown to be a strong transcriptional co-activator. Alternative splicing has been observed at this locus and two variants, both encoding the same protein, have been identified. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 161547063, 161702593, 36.14, 9, 'ENSG00000167565', 'Q9UJW9'),
(2970, 'SEZ6L', 'Predicted to be involved in synapse maturation. Predicted to act upstream of or within activation of protein kinase C activity; adult locomotory behavior; and cerebellar Purkinje cell layer development. Predicted to be located in endoplasmic reticulum membrane. Predicted to be active in several cellular components, including glutamatergic synapse; neuronal cell body; and postsynaptic membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 9771434, 9791592, 47.73, 3, 'ENSG00000100095', 'Q9BYH1'),
(2971, 'SF3B2', 'This gene encodes subunit 2 of the splicing factor 3b protein complex. Splicing factor 3b, together with splicing factor 3a and a 12S RNA unit, forms the U2 small nuclear ribonucleoproteins complex (U2 snRNP). The splicing factor 3b/3a complex binds pre-mRNA upstream of the intron\'s branch site in a sequence-independent manner and may anchor the U2 snRNP to the pre-mRNA. Splicing factor 3b is also a component of the minor U12-type spliceosome. Subunit 2 associates with pre-mRNA upstream of the branch site at the anchoring site. Subunit 2 also interacts directly with subunit 4 of the splicing factor 3b complex. Subunit 2 is a highly hydrophilic protein with a proline-rich N-terminus and a glutamate-rich stretch in the C-terminus. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 102712445, 102717733, 50.60, 5, 'ENSG00000087365', 'Q13435'),
(2972, 'SFI1', 'Enables phosphatase binding activity. Located in centriole. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000198089', 'A8K8P3'),
(2973, 'SFMBT2', 'Enables histone binding activity. Involved in negative regulation of gene expression. Located in aggresome; cytosol; and nuclear speck. [provided by Alliance of Genome Resources, May 2025]', '10', 'TR_V_gene', 21736152, 21736982, 47.29, 1, 'ENSG00000198879', 'Q5VUG0'),
(2974, 'SFRP4', 'Secreted frizzled-related protein 4 (SFRP4) is a member of the SFRP family that contains a cysteine-rich domain homologous to the putative Wnt-binding site of Frizzled proteins. SFRPs act as soluble modulators of Wnt signaling. The expression of SFRP4 in ventricular myocardium correlates with apoptosis related gene expression. [provided by RefSeq, Jul 2008]', '7', 'TR_J_gene', 22528527, 22528586, 48.33, 1, 'ENSG00000106483', 'Q6FHJ7'),
(2975, 'SFTPD', 'The protein encoded by this gene is part of the innate immune response, protecting the lungs against inhaled microorganisms and chemicals. The encoded protein may also be involved in surfactant metabolism. [provided by RefSeq, Jul 2015]', '10', 'miRNA', 101055258, 101055323, 59.09, 1, 'ENSG00000133661', 'P35247'),
(2976, 'SGCE', 'This gene encodes the epsilon member of the sarcoglycan family. Sarcoglycans are transmembrane proteins that are components of the dystrophin-glycoprotein complex, which link the actin cytoskeleton to the extracellular matrix. Unlike other family members which are predominantly expressed in striated muscle, the epsilon sarcoglycan is more broadly expressed. Mutations in this gene are associated with myoclonus-dystonia syndrome. This gene is imprinted, with preferential expression from the paternal allele. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. A pseudogene associated with this gene is located on chromosome 2. [provided by RefSeq, Oct 2016]', '7', 'protein_coding', 137241287, 137243707, 63.28, 2, 'ENSG00000127990', 'O43556'),
(2977, 'SGCG', 'This gene encodes gamma-sarcoglycan, one of several sarcolemmal transmembrane glycoproteins that interact with dystrophin. The dystrophin-glycoprotein complex (DGC) spans the sarcolemma and is comprised of dystrophin, syntrophin, alpha- and beta-dystroglycans and sarcoglycans. The DGC provides a structural link between the subsarcolemmal cytoskeleton and the extracellular matrix of muscle cells. Defects in the encoded protein can lead to early onset autosomal recessive muscular dystrophy, in particular limb-girdle muscular dystrophy, type 2C (LGMD2C). [provided by RefSeq, Oct 2008]', '13', 'protein_coding', 26599278, 26618027, 48.38, 2, 'ENSG00000102683', 'Q13326'),
(2978, 'SGK2', 'This gene is a member of the Ser/Thr protein kinase family and encodes a phosphoprotein with a PX (phox homology) domain. The protein phosphorylates several target proteins and has a role in neutral amino acid transport and activation of potassium and chloride channels. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '8', 'miRNA', 100869060, 100869148, 53.93, 1, 'ENSG00000101049', 'Q9HBY8'),
(2979, 'SGO1', 'The protein encoded by this gene is a member of the shugoshin family of proteins. This protein is thought to protect centromeric cohesin from cleavage during mitotic prophase by preventing phosphorylation of a cohesin subunit. Reduced expression of this gene leads to the premature loss of centromeric cohesion, mis-segregation of sister chromatids, and mitotic arrest. Evidence suggests that this protein also protects a small subset of cohesin found along the length of the chromosome arms during mitotic prophase. An isoform lacking exon 6 has been shown to play a role in the cohesion of centrioles (PMID: 16582621 and PMID:18331714). Mutations in this gene have been associated with Chronic Atrial and Intestinal Dysrhythmia (CAID) syndrome, characterized by the co-occurrence of Sick Sinus Syndrome (SSS) and Chronic Intestinal Pseudo-obstruction (CIPO) within the first four decades of life (PMID:25282101). Fibroblast cells from CAID patients exhibited both increased cell proliferation and higher rates of senescence. Pseudogenes of this gene have been found on chromosomes 1 and 7. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2015]', '3', 'protein_coding', 75469302, 75569722, 41.76, 11, 'ENSG00000129810', 'Q5FBB7'),
(2980, 'SGPP1', 'Sphingosine-1-phosphate (S1P) is a bioactive sphingolipid metabolite that regulates diverse biologic processes. SGPP1 catalyzes the degradation of S1P via salvage and recycling of sphingosine into long-chain ceramides (Mandala et al., 2000 [PubMed 10859351]; Le Stunff et al., 2007 [PubMed 17895250]).[supplied by OMIM, Jun 2009]', '14', 'protein_coding', 31090842, 31095980, 55.01, 8, 'ENSG00000285281', 'Q9BX95'),
(2981, 'SGPP1', 'Sphingosine-1-phosphate (S1P) is a bioactive sphingolipid metabolite that regulates diverse biologic processes. SGPP1 catalyzes the degradation of S1P via salvage and recycling of sphingosine into long-chain ceramides (Mandala et al., 2000 [PubMed 10859351]; Le Stunff et al., 2007 [PubMed 17895250]).[supplied by OMIM, Jun 2009]', '14', 'protein_coding', 31090842, 31095980, 55.01, 8, 'ENSG00000126821', 'Q9BX95'),
(2982, 'SH3D19', 'This gene encodes a multiple SH3 domain-containing protein, which interacts with other proteins, such as EBP and members of ADAM family, via the SH3 domains. This protein may be involved in suppression of Ras-induced cellular transformation and Ras-mediated activation of ELK1 by EBP, and regulation of ADAM proteins in the signaling of EGFR-ligand shedding. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', '4', 'protein_coding', 72872025, 72943837, 44.14, 4, 'ENSG00000292165', 'Q5HYK7'),
(2983, 'SH3D19', 'This gene encodes a multiple SH3 domain-containing protein, which interacts with other proteins, such as EBP and members of ADAM family, via the SH3 domains. This protein may be involved in suppression of Ras-induced cellular transformation and Ras-mediated activation of ELK1 by EBP, and regulation of ADAM proteins in the signaling of EGFR-ligand shedding. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]', '4', 'protein_coding', 72872025, 72943837, 44.14, 4, 'ENSG00000109686', 'Q5HYK7'),
(2984, 'SH3GL2', 'Enables identical protein binding activity. Involved in negative regulation of blood-brain barrier permeability; negative regulation of gene expression; and negative regulation of signal transduction. Located in perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 66720893, 66751609, 44.86, 15, 'ENSG00000107295', 'Q99962'),
(2985, 'SH3GL3', 'Enables identical protein binding activity. Predicted to be involved in central nervous system development and signal transduction. Predicted to be located in acrosomal vesicle and early endosome membrane. Predicted to be part of early endosome. Predicted to be active in several cellular components, including glutamatergic synapse; postsynaptic density, intracellular component; and postsynaptic endosome. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 637003, 641000, 55.85, 7, 'ENSG00000291608', 'Q99963'),
(2986, 'SH3GL3', 'Enables identical protein binding activity. Predicted to be involved in central nervous system development and signal transduction. Predicted to be located in acrosomal vesicle and early endosome membrane. Predicted to be part of early endosome. Predicted to be active in several cellular components, including glutamatergic synapse; postsynaptic density, intracellular component; and postsynaptic endosome. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 637003, 641000, 55.85, 7, 'ENSG00000140600', 'Q99963'),
(2987, 'SH3PXD2B', 'This gene encodes an adapter protein that is characterized by a PX domain and four Src homology 3 domains. The encoded protein is required for podosome formation and is involved in cell adhesion and migration of numerous cell types. Mutations in this gene are the cause of Frank-ter Haar syndrome (FTHS), and also Borrone Dermato-Cardio-Skeletal (BDCS) syndrome. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Apr 2015]', '5', 'protein_coding', 46019153, 46023053, 57.65, 1, 'ENSG00000174705', 'A1X283'),
(2988, 'SH3TC2', 'This gene encodes a protein with two N-terminal Src homology 3 (SH3) domains and 10 tetratricopeptide repeat (TPR) motifs, and is a member of a small gene family. The gene product has been proposed to be an adapter or docking molecule. Mutations in this gene result in autosomal recessive Charcot-Marie-Tooth disease type 4C, a childhood-onset neurodegenerative disease characterized by demyelination of motor and sensory neurons. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 39452587, 39458931, 40.71, 15, 'ENSG00000169247', 'Q8TF17'),
(2989, 'SHISA6', 'Predicted to enable ionotropic glutamate receptor binding activity. Predicted to be involved in several processes, including excitatory chemical synaptic transmission; modulation of chemical synaptic transmission; and negative regulation of canonical Wnt signaling pathway. Predicted to be located in asymmetric, glutamatergic, excitatory synapse. Predicted to be part of AMPA glutamate receptor complex. Predicted to be active in dendritic spine membrane; glutamatergic synapse; and postsynaptic density membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 45687025, 45752520, 43.44, 20, 'ENSG00000188803', 'Q6ZSJ9'),
(2990, 'SHMT1', 'This gene encodes the cytosolic form of serine hydroxymethyltransferase, a pyridoxal phosphate-containing enzyme that catalyzes the reversible conversion of serine and tetrahydrofolate to glycine and 5,10-methylene tetrahydrofolate. This reaction provides one-carbon units for synthesis of methionine, thymidylate, and purines in the cytoplasm. This gene is located within the Smith-Magenis syndrome region on chromosome 17. A pseudogene of this gene is located on the short arm of chromosome 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '17', 'unprocessed_pseudogene', 259419, 260343, 42.16, 1, 'ENSG00000284320', 'P34896'),
(2991, 'SHMT1', 'This gene encodes the cytosolic form of serine hydroxymethyltransferase, a pyridoxal phosphate-containing enzyme that catalyzes the reversible conversion of serine and tetrahydrofolate to glycine and 5,10-methylene tetrahydrofolate. This reaction provides one-carbon units for synthesis of methionine, thymidylate, and purines in the cytoplasm. This gene is located within the Smith-Magenis syndrome region on chromosome 17. A pseudogene of this gene is located on the short arm of chromosome 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '17', 'unprocessed_pseudogene', 259419, 260343, 42.16, 1, 'ENSG00000176974', 'P34896'),
(2992, 'SHOX', 'This gene belongs to the paired homeobox family and is located in the pseudoautosomal region 1 (PAR1) of X and Y chromosomes. Defects in this gene are associated with idiopathic growth retardation and in the short stature phenotype of Turner syndrome patients. This gene is highly conserved across species from mammals to fish to flies. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Jul 2008]', 'X, Y', 'protein_coding', 6910686, 7101970, 43.32, 4, 'ENSG00000292354', 'O15266'),
(2993, 'SHOX', 'This gene belongs to the paired homeobox family and is located in the pseudoautosomal region 1 (PAR1) of X and Y chromosomes. Defects in this gene are associated with idiopathic growth retardation and in the short stature phenotype of Turner syndrome patients. This gene is highly conserved across species from mammals to fish to flies. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Jul 2008]', 'X, Y', 'protein_coding', 6910686, 7101970, 43.32, 4, 'ENSG00000185960', 'O15266'),
(2994, 'SHQ1', 'SHQ1 assists in the assembly of H/ACA-box ribonucleoproteins that function in the processing of ribosomal RNAs, modification of spliceosomal small nuclear RNAs, and stabilization of telomerase (see MIM 602322) (Grozdanov et al., 2009 [PubMed 19383767]).[supplied by OMIM, Dec 2010]', '3', 'protein_coding', 43509702, 43562419, 45.68, 16, 'ENSG00000144736', 'Q6PI26'),
(2995, 'SHROOM1', 'SHROOM family members play diverse roles in the development of the nervous system and other tissues (Hagens et al., 2006 [PubMed 16615870]).[supplied by OMIM, Mar 2008]', '5', 'protein_coding', 46541806, 46584688, 43.97, 8, 'ENSG00000164403', 'Q2M3G4'),
(2996, 'SHROOM2', 'This gene represents the human homolog of Xenopus laevis apical protein (APX) gene, which is implicated in amiloride-sensitive sodium channel activity. It is expressed in endothelial cells and facilitates the formation of a contractile network within endothelial cells. Depletion of this gene results in an increase in endothelial sprouting, migration, and angiogenesis. This gene is highly expressed in the retina, and is a strong candidate for ocular albinism type 1 syndrome. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Mar 2016]', 'X', 'protein_coding', 31739526, 31803389, 39.17, 2, 'ENSG00000146950', 'Q13796'),
(2997, 'SHROOM3', 'This gene encodes a PDZ-domain-containing protein that belongs to a family of Shroom-related proteins. This protein may be involved in regulating cell shape in certain tissues. A similar protein in mice is required for proper neurulation. [provided by RefSeq, Jan 2011]', '4', 'protein_coding', 104036640, 104039196, 48.30, 1, 'ENSG00000138771', 'Q8TF72'),
(2998, 'SIGLEC1', 'This gene encodes a member of the immunoglobulin superfamily. The encoded protein is a lectin-like adhesion molecule that binds glycoconjugate ligands on cell surfaces in a sialic acid-dependent manner. It is a type I transmembrane protein expressed only by a subpopulation of macrophages and is involved in mediating cell-cell interactions. The protein plays an important role in multiple human diseases and bacterial and viral infections has been shown to enhance SARS-CoV-2 infection. [provided by RefSeq, Dec 2021]', '20', 'protein_coding', 49364066, 49396089, 48.26, 1, 'ENSG00000088827', 'Q9BZZ2'),
(2999, 'SIGLEC12', 'Sialic acid-binding immunoglobulin-like lectins (SIGLECs) are a family of cell surface proteins belonging to the immunoglobulin superfamily. They mediate protein-carbohydrate interactions by selectively binding to different sialic acid moieties present on glycolipids and glycoproteins. This gene encodes a member of the SIGLEC3-like subfamily of SIGLECs. Members of this subfamily are characterized by an extracellular V-set immunoglobulin-like domain followed by two C2-set immunoglobulin-like domains, and the cytoplasmic tyrosine-based motifs ITIM and SLAM-like. The encoded protein, upon tyrosine phosphorylation, has been shown to recruit the Src homology 2 domain-containing protein-tyrosine phosphatases SHP1 and SHP2. It has been suggested that the protein is involved in the negative regulation of macrophage signaling by functioning as an inhibitory receptor. This gene is located in a cluster with other SIGLEC3-like genes on 19q13.4. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', '19', 'miRNA', 53679003, 53679085, 42.17, 1, 'ENSG00000254521', 'Q96PQ1'),
(3000, 'SIGMAR1', 'This gene encodes a receptor protein that interacts with a variety of psychotomimetic drugs, including cocaine and amphetamines. The receptor is believed to play an important role in the cellular functions of various tissues associated with the endocrine, immune, and nervous systems. As indicated by its previous name, opioid receptor sigma 1 (OPRS1), the product of this gene was erroneously thought to function as an opioid receptor; it is now thought to be a non-opioid receptor. Mutations in this gene has been associated with juvenile amyotrophic lateral sclerosis 16. Alternative splicing of this gene results in transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2013]', '9', 'protein_coding', 1048715, 1091353, 52.44, 10, 'ENSG00000147955', 'Q99720'),
(3001, 'SIL1', 'This gene encodes a resident endoplasmic reticulum (ER), N-linked glycoprotein with an N-terminal ER targeting sequence, 2 putative N-glycosylation sites, and a C-terminal ER retention signal. This protein functions as a nucleotide exchange factor for another unfolded protein response protein. Mutations in this gene have been associated with Marinesco-Sjogren syndrome. Alternate transcriptional splice variants have been characterized. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 2616667, 2623410, 52.21, 1, 'ENSG00000120725', 'Q9H173'),
(3002, 'SIM2', 'This gene represents a homolog of the Drosophila single-minded (sim) gene, which encodes a transcription factor that is a master regulator of neurogenesis. The encoded protein is ubiquitinated by RING-IBR-RING-type E3 ubiquitin ligases, including the parkin RBR E3 ubiquitin protein ligase. This gene maps within the so-called Down syndrome chromosomal region, and is thus thought to contribute to some specific Down syndrome phenotypes. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Sep 2014]', '21', 'protein_coding', 30718, 130818, 39.40, 7, 'ENSG00000159263', 'Q14190'),
(3003, 'SINHCAF', 'Involved in negative regulation of cell migration. Part of Sin3-type complex. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 29730956, 29767011, 52.77, 2, 'ENSG00000276371', 'Q9NP50'),
(3004, 'SINHCAF', 'Involved in negative regulation of cell migration. Part of Sin3-type complex. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 29730956, 29767011, 52.77, 2, 'ENSG00000139146', 'Q9NP50'),
(3005, 'SIRPB1', 'The protein encoded by this gene is a member of the signal-regulatory-protein (SIRP) family, and also belongs to the immunoglobulin superfamily. SIRP family members are receptor-type transmembrane glycoproteins known to be involved in the negative regulation of receptor tyrosine kinase-coupled signaling processes. This protein was found to interact with TYROBP/DAP12, a protein bearing immunoreceptor tyrosine-based activation motifs. This protein was also reported to participate in the recruitment of tyrosine kinase SYK. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2009]', '20', 'protein_coding', 44106590, 44187188, 47.95, 2, 'ENSG00000101307', 'O00241'),
(3006, 'SIRT1', 'This gene encodes a member of the sirtuin family of proteins, homologs to the yeast Sir2 protein. Members of the sirtuin family are characterized by a sirtuin core domain and grouped into four classes. The functions of human sirtuins have not yet been determined; however, yeast sirtuin proteins are known to regulate epigenetic gene silencing and suppress recombination of rDNA. Studies suggest that the human sirtuins may function as intracellular regulatory proteins with mono-ADP-ribosyltransferase activity. The protein encoded by this gene is included in class I of the sirtuin family. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2008]', '10', 'protein_coding', 109130293, 109167249, 42.23, 6, 'ENSG00000096717', 'Q96EB6'),
(3007, 'SIRT2', 'This gene encodes a member of the sirtuin family of proteins, homologs to the yeast Sir2 protein. Members of the sirtuin family are characterized by a sirtuin core domain and grouped into four classes. The functions of human sirtuins have not yet been determined; however, yeast sirtuin proteins are known to regulate epigenetic gene silencing and suppress recombination of rDNA. Studies suggest that the human sirtuins may function as intracellular regulatory proteins with mono-ADP-ribosyltransferase activity. The protein encoded by this gene is included in class I of the sirtuin family. Several transcript variants are resulted from alternative splicing of this gene. [provided by RefSeq, Jul 2010]', '19', 'protein_coding', 5114953, 5126626, 42.54, 2, 'ENSG00000283100', 'Q8IXJ6'),
(3008, 'SIRT2', 'This gene encodes a member of the sirtuin family of proteins, homologs to the yeast Sir2 protein. Members of the sirtuin family are characterized by a sirtuin core domain and grouped into four classes. The functions of human sirtuins have not yet been determined; however, yeast sirtuin proteins are known to regulate epigenetic gene silencing and suppress recombination of rDNA. Studies suggest that the human sirtuins may function as intracellular regulatory proteins with mono-ADP-ribosyltransferase activity. The protein encoded by this gene is included in class I of the sirtuin family. Several transcript variants are resulted from alternative splicing of this gene. [provided by RefSeq, Jul 2010]', '19', 'protein_coding', 5114953, 5126626, 42.54, 2, 'ENSG00000068903', 'Q8IXJ6'),
(3009, 'SIRT7', 'This gene encodes a member of the sirtuin family of proteins, homologs to the yeast Sir2 protein. Members of the sirtuin family are characterized by a sirtuin core domain and grouped into four classes. The functions of human sirtuins have not yet been determined; however, yeast sirtuin proteins are known to regulate epigenetic gene silencing and suppress recombination of rDNA. Studies suggest that the human sirtuins may function as intracellular regulatory proteins with mono-ADP-ribosyltransferase activity. The protein encoded by this gene is included in class IV of the sirtuin family. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 73375101, 73414076, 40.79, 5, 'ENSG00000187531', 'Q9NRC8'),
(3010, 'SIVA1', 'This gene encodes an E3 ubiquitin ligase that regulates cell cycle progression, cell proliferation and apoptosis. The N-terminus of this protein binds to the cytoplasmic tail of the CD27 antigen, a member of the tumor necrosis factor receptor (TNFR) superfamily. In response to UV radiation-induced DNA damage, this protein has been shown to mediate the ubiquitination of proliferating cell nuclear antigen (PCNA), an important step in translesion DNA synthesis. [provided by RefSeq, Sep 2018]', '14', 'protein_coding', 119597875, 119629811, 45.20, 6, 'ENSG00000184990', 'O15304'),
(3011, 'SIX1', 'The protein encoded by this gene is a homeobox protein that is similar to the Drosophila \'sine oculis\' gene product. This gene is found in a cluster of related genes on chromosome 14 and is thought to be involved in limb development. Defects in this gene are a cause of autosomal dominant deafness type 23 (DFNA23) and branchiootic syndrome type 3 (BOS3). [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 44646414, 44647650, 62.41, 1, 'ENSG00000126778', 'Q15475'),
(3012, 'SLC10A2', 'This gene encodes a sodium/bile acid cotransporter. This transporter is the primary mechanism for uptake of intestinal bile acids by apical cells in the distal ileum. Bile acids are the catabolic product of cholesterol metabolism, so this protein is also critical for cholesterol homeostasis. Mutations in this gene cause primary bile acid malabsorption (PBAM); muatations in this gene may also be associated with other diseases of the liver and intestines, such as familial hypertriglyceridemia (FHTG). [provided by RefSeq, Mar 2010]', '13', 'protein_coding', 60245752, 60299087, 36.08, 8, 'ENSG00000125255', 'Q12908'),
(3013, 'SLC12A2', 'The protein encoded by this gene mediates sodium and chloride transport and reabsorption. The encoded protein is a membrane protein and is important in maintaining proper ionic balance and cell volume. This protein is phosphorylated in response to DNA damage. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]', '5', 'protein_coding', 116287380, 116508541, 37.86, 15, 'ENSG00000064651', 'P55011'),
(3014, 'SLC14A1', 'The protein encoded by this gene is a membrane transporter that mediates urea transport in erythrocytes. This gene forms the basis for the Kidd blood group system. [provided by RefSeq, Mar 2009]', '18', 'protein_coding', 566995, 584109, 64.34, 25, 'ENSG00000141469', 'Q13336'),
(3015, 'SLC14A2', 'The protein encoded by this gene belongs to the urea transporter family. In mammalian cells, urea is the chief end product of nitrogen catabolism, and plays an important role in the urinary concentration mechanism. This protein is expressed in the inner medulla of the kidney, and mediates rapid transepithelial urea transport across the inner medullary collecting duct. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2011]', '18', 'protein_coding', 43923792, 43964488, 49.36, 5, 'ENSG00000132874', 'Q15849'),
(3016, 'SLC16A1', 'The protein encoded by this gene is a proton-linked monocarboxylate transporter that catalyzes the movement of many monocarboxylates, such as lactate and pyruvate, across the plasma membrane. Mutations in this gene are associated with erythrocyte lactate transporter defect. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Oct 2009]', '1', 'protein_coding', 64049836, 64072347, 61.58, 17, 'ENSG00000281917', 'P53985'),
(3017, 'SLC16A1', 'The protein encoded by this gene is a proton-linked monocarboxylate transporter that catalyzes the movement of many monocarboxylates, such as lactate and pyruvate, across the plasma membrane. Mutations in this gene are associated with erythrocyte lactate transporter defect. Alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Oct 2009]', '1', 'protein_coding', 64049836, 64072347, 61.58, 17, 'ENSG00000155380', 'P53985'),
(3018, 'SLC17A1', 'Predicted to enable transmembrane transporter activity. Involved in sodium-dependent phosphate transport; urate metabolic process; and urate transport. Located in apical plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 73834590, 73842516, 59.28, 3, 'ENSG00000124568', 'Q14916'),
(3019, 'SLC17A3', 'The protein encoded by this gene is a voltage-driven transporter that excretes intracellular urate and organic anions from the blood into renal tubule cells. Two transcript variants encoding different isoforms have been found for this gene. The longer isoform is a plasma membrane protein with transporter activity while the shorter isoform localizes to the endoplasmic reticulum. [provided by RefSeq, Aug 2012]', '6', 'miRNA', 53697533, 53697617, 47.06, 1, 'ENSG00000124564', 'O00476'),
(3020, 'SLC17A4', 'Phosphate homeostasis is maintained by regulating intake, intestinal absorption, bone deposition and resorption, and renal excretion of phosphate. The central molecule in the control of phosphate excretion from the kidney is the sodium/phosphate cotransporter NPT1 (SLC17A1; MIM 182308), which is located in the renal proximal tubule. NPT1 uses the transmembrane electrochemical potential gradient of sodium to transport phosphate across the cell membrane. SLC17A4 is a similar sodium/phosphate cotransporter in the intestinal mucosa that plays an important role in the absorption of phosphate from the intestine (summary by Shibui et al., 1999 [PubMed 10319585]).[supplied by OMIM, Feb 2011]', '6', 'protein_coding', 35534164, 35537054, 57.49, 1, 'ENSG00000146039', 'Q9Y2C5'),
(3021, 'SLC17A8', 'This gene encodes a vesicular glutamate transporter. The encoded protein transports the neurotransmitter glutamate into synaptic vesicles before it is released into the synaptic cleft. Mutations in this gene are the cause of autosomal-dominant nonsyndromic type 25 deafness. Alternate splicing results in multiple transcript variants.[provided by RefSeq, May 2010]', '12', 'protein_coding', 120505920, 120604074, 39.70, 40, 'ENSG00000179520', 'Q8NDX2'),
(3022, 'SLC18B1', 'This gene encodes a protein, which has high sequence similarity to rat, xenopus and zebrafish proteins. The protein function is unknown. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 110289233, 110738956, 36.01, 13, 'ENSG00000146409', 'Q6NT16'),
(3023, 'SLC23A2', 'The absorption of vitamin C into the body and its distribution to organs requires two sodium-dependent vitamin C transporters. This gene encodes one of the two required transporters and the encoded protein accounts for tissue-specific uptake of vitamin C. Previously, this gene had an official symbol of SLC23A1. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 46831546, 46897076, 39.27, 3, 'ENSG00000089057', 'Q9UGH3'),
(3024, 'SLC25A14', 'Mitochondrial uncoupling proteins (UCP) are members of the larger family of mitochondrial anion carrier proteins (MACP). Uncoupling proteins separate oxidative phosphorylation from ATP synthesis with energy dissipated as heat, also referred to as the mitochondrial proton leak. Uncoupling proteins facilitate the transfer of anions from the inner to the outer mitochondrial membrane and the return transfer of protons from the outer to the inner mitochondrial membrane. They also reduce the mitochondrial membrane potential in mammalian cells. This gene is widely expressed in many tissues with the greatest abundance in brain and testis. Alternative splicing results in multiple transcript variants. A pseudogene of this gene has been defined on chromosome 4. [provided by RefSeq, Aug 2013]', 'X', 'protein_coding', 45684651, 45705019, 41.50, 2, 'ENSG00000102078', 'O95258'),
(3025, 'SLC25A22', 'This gene encodes a mitochondrial glutamate carrier. Mutations in this gene are associated with early infantile epileptic encephalopathy. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Jul 2010]', '11', 'protein_coding', 61953469, 62065881, 52.37, 10, 'ENSG00000177542', 'Q9H936'),
(3026, 'SLC25A26', 'This gene is a member of the mitochondrial carrier family which includes nuclear-encoded transporters localized on the inner mitochondrial membranes. Members of the family transport important small molecules across the mitochondrial inner membrane. This protein is involved in the transport of S-adenosylmethionine (SAM) into the mitochondria. Mutations in this gene are associated with combined oxidative phosphorylation deficiency 28. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Apr 2017]', '3', 'protein_coding', 27977992, 27981449, 41.76, 1, 'ENSG00000282739', 'Q70HW3'),
(3027, 'SLC25A26', 'This gene is a member of the mitochondrial carrier family which includes nuclear-encoded transporters localized on the inner mitochondrial membranes. Members of the family transport important small molecules across the mitochondrial inner membrane. This protein is involved in the transport of S-adenosylmethionine (SAM) into the mitochondria. Mutations in this gene are associated with combined oxidative phosphorylation deficiency 28. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Apr 2017]', '3', 'protein_coding', 27977992, 27981449, 41.76, 1, 'ENSG00000144741', 'Q70HW3'),
(3028, 'SLC25A52', 'This gene is similar to the mitochondrial carrier triple repeat 1 gene on chromosome 9. The gene is intronless and may be an evolving pseudogene; however, it is transcribed and it contains a full-length coding region so it is currently classified as a protein-coding locus. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 139340359, 139423457, 40.36, 10, 'ENSG00000141437', 'Q3SY17'),
(3029, 'SLC25A53', 'Predicted to enable NAD transmembrane transporter activity. Predicted to be involved in NAD transmembrane transport. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 459239, 508641, 39.83, 7, 'ENSG00000269743', 'Q5H9E4');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3030, 'SLC27A3', 'This gene belongs to a family of integral membrane proteins and encodes a protein that is involved in lipid metabolism. The increased expression of this gene in human neural stem cells derived from induced pluripotent stem cells suggests that it plays an important role in early brain development. Naturally occurring mutations in this gene are associated with autism spectrum disorders. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2017]', '1', 'protein_coding', 38111495, 38214778, 51.79, 39, 'ENSG00000263163', 'Q5K4L6'),
(3031, 'SLC27A3', 'This gene belongs to a family of integral membrane proteins and encodes a protein that is involved in lipid metabolism. The increased expression of this gene in human neural stem cells derived from induced pluripotent stem cells suggests that it plays an important role in early brain development. Naturally occurring mutations in this gene are associated with autism spectrum disorders. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2017]', '1', 'protein_coding', 38111495, 38214778, 51.79, 39, 'ENSG00000143554', 'Q5K4L6'),
(3032, 'SLC27A4', 'This gene encodes a member of a family of fatty acid transport proteins, which are involved in translocation of long-chain fatty acids cross the plasma membrane. This protein is expressed at high levels on the apical side of mature enterocytes in the small intestine, and appears to be the principal fatty acid transporter in enterocytes. Clinical studies suggest this gene as a candidate gene for the insulin resistance syndrome. Mutations in this gene have been associated with ichthyosis prematurity syndrome. [provided by RefSeq, Apr 2010]', '9', 'protein_coding', 94561442, 94593118, 49.18, 12, 'ENSG00000167114', 'Q6P1M0'),
(3033, 'SLC28A3', 'Nucleoside transporters, such as SLC28A3, regulate multiple cellular processes, including neurotransmission, vascular tone, adenosine concentration in the vicinity of cell surface receptors, and transport and metabolism of nucleoside drugs. SLC28A3 shows broad specificity for pyrimidine and purine nucleosides (Ritzel et al., 2001 [PubMed 11032837]).[supplied by OMIM, Mar 2008]', '9', 'protein_coding', 3261479, 3278461, 40.74, 5, 'ENSG00000197506', 'Q9HAS3'),
(3034, 'SLC29A3', 'This gene encodes a nucleoside transporter. The encoded protein plays a role in cellular uptake of nucleosides, nucleobases, and their related analogs. Mutations in this gene have been associated with H syndrome, which is characterized by cutaneous hyperpigmentation and hypertrichosis, hepatosplenomegaly, heart anomalies, and hypogonadism. A related disorder, PHID (pigmented hypertrichosis with insulin-dependent diabetes mellitus), has also been associated with mutations at this locus. Alternatively spliced transcript variants have been described.[provided by RefSeq, Mar 2010]', '10', 'protein_coding', 157341598, 157345677, 45.47, 2, 'ENSG00000198246', 'Q9BZD2'),
(3035, 'SLC2A13', 'Enables ATPase binding activity; myo-inositol:proton symporter activity; and protease binding activity. Involved in myo-inositol transport and positive regulation of amyloid-beta formation. Located in cell body; cell projection; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 55751051, 55842830, 40.30, 8, 'ENSG00000151229', 'Q96QE2'),
(3036, 'SLC2A6', 'Hexose transport into mammalian cells is catalyzed by a family of membrane proteins, including SLC2A6, that contain 12 transmembrane domains and a number of critical conserved residues.[supplied by OMIM, Jul 2002]', '9', 'unprocessed_pseudogene', 199990, 200944, 36.96, 1, 'ENSG00000281165', 'Q9UGQ3'),
(3037, 'SLC2A6', 'Hexose transport into mammalian cells is catalyzed by a family of membrane proteins, including SLC2A6, that contain 12 transmembrane domains and a number of critical conserved residues.[supplied by OMIM, Jul 2002]', '9', 'unprocessed_pseudogene', 199990, 200944, 36.96, 1, 'ENSG00000160326', 'Q9UGQ3'),
(3038, 'SLC2A8', 'This gene belongs to the solute carrier 2A family, which includes intracellular glucose transporters. Based on sequence comparison, the glucose transporters are grouped into three classes and this gene is a member of class II. The encoded protein, like other members of the family, contains several conserved residues and motifs and 12 transmembrane domains with both amino and carboxyl ends being on the cytosolic side of the membrane. Alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Nov 2012]', '9', 'protein_coding', 73308276, 73369388, 55.26, 7, 'ENSG00000136856', 'Q9NY64'),
(3039, 'SLC2A9', 'This gene encodes a member of the SLC2A facilitative glucose transporter family. Members of this family play a significant role in maintaining glucose homeostasis. The encoded protein may play a role in the development and survival of chondrocytes in cartilage matrices. Two transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]', '4', 'protein_coding', 55773438, 55774382, 40.32, 1, 'ENSG00000109667', 'Q9NRM0'),
(3040, 'SLC32A1', 'The protein encoded by this gene is an integral membrane protein involved in gamma-aminobutyric acid (GABA) and glycine uptake into synaptic vesicles. The encoded protein is a member of amino acid/polyamine transporter family II. [provided by RefSeq, Jul 2008]', '20', 'IG_V_pseudogene', 343000, 343738, 45.74, 1, 'ENSG00000101438', 'Q9H598'),
(3041, 'SLC33A1', 'The protein encoded by this gene is required for the formation of O-acetylated (Ac) gangliosides. The encoded protein is predicted to contain 6 to 10 transmembrane domains, and a leucine zipper motif in transmembrane domain III. Defects in this gene have been reported to cause spastic paraplegia autosomal dominant type 42 (SPG42) in one Chinese family, but not in similar patients of European descent. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2010]', '3', 'protein_coding', 109372906, 109482086, 39.92, 1, 'ENSG00000169359', 'O00400'),
(3042, 'SLC35A2', 'This gene encodes a member of the nucleotide-sugar transporter family. The encoded protein is a multi-pass membrane protein. It transports UDP-galactose from the cytosol into Golgi vesicles, where it serves as a glycosyl donor for the generation of glycans. Mutations in this gene cause congenital disorder of glycosylation type IIm (CDG2M). Multiple alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Oct 2014]', 'X', 'protein_coding', 42932376, 42948958, 40.68, 4, 'ENSG00000102100', 'P78381'),
(3043, 'SLC35A2', 'This gene encodes a member of the nucleotide-sugar transporter family. The encoded protein is a multi-pass membrane protein. It transports UDP-galactose from the cytosol into Golgi vesicles, where it serves as a glycosyl donor for the generation of glycans. Mutations in this gene cause congenital disorder of glycosylation type IIm (CDG2M). Multiple alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Oct 2014]', 'X', 'protein_coding', 42932376, 42948958, 40.68, 4, 'ENSG00000292209', 'P78381'),
(3044, 'SLC35A3', 'This gene encodes a UDP-N-acetylglucosamine transporter found in the golgi apparatus membrane. In cattle, a missense mutation in this gene causes complex vertebral malformation. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2012]', '1', 'protein_coding', 128504655, 128543874, 45.42, 15, 'ENSG00000117620', 'Q9Y2D2'),
(3045, 'SLC35G1', 'This gene encodes a transmembrane protein which is a member of the drug/metabolite transporter protein superfamily. The encoded protein may play a role in the regulation of calcium levels inside the cell. [provided by RefSeq, Sep 2016]', '10', 'protein_coding', 77824, 86614, 33.76, 4, 'ENSG00000176273', 'Q2M3R5'),
(3046, 'SLC35G5', 'This gene is intronless and probably arose from retrotransposition of a similar gene. It has high sequence similarity to the gene encoding acyl-malonyl condensing enzyme on chromosome 17. [provided by RefSeq, Aug 2011]', '8', 'protein_coding', 85280220, 85301784, 31.67, 1, 'ENSG00000285084', 'Q96KT7'),
(3047, 'SLC35G5', 'This gene is intronless and probably arose from retrotransposition of a similar gene. It has high sequence similarity to the gene encoding acyl-malonyl condensing enzyme on chromosome 17. [provided by RefSeq, Aug 2011]', '8', 'protein_coding', 85280220, 85301784, 31.67, 1, 'ENSG00000177710', 'Q96KT7'),
(3048, 'SLC35G6', 'Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 74073353, 74147843, 40.53, 14, 'ENSG00000284515', 'P0C7Q6'),
(3049, 'SLC35G6', 'Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 74073353, 74147843, 40.53, 14, 'ENSG00000259224', 'P0C7Q6'),
(3050, 'SLC37A2', 'Enables glucose 6-phosphate:phosphate antiporter activity. Involved in glucose-6-phosphate transport and phosphate ion transmembrane transport. Located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 130266863, 130289798, 44.64, 4, 'ENSG00000134955', 'Q8TED4'),
(3051, 'SLC37A3', 'Enables xenobiotic transmembrane transporter activity. Involved in xenobiotic transmembrane transport. Located in endoplasmic reticulum membrane and lysosomal membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 107571161, 107965180, 37.54, 15, 'ENSG00000157800', 'Q8NCC5'),
(3052, 'SLC39A10', 'Zinc is an essential cofactor for hundreds of enzymes. It is involved in protein, nucleic acid, carbohydrate, and lipid metabolism, as well as in the control of gene transcription, growth, development, and differentiation. SLC39A10 belongs to a subfamily of proteins that show structural characteristics of zinc transporters (Taylor and Nicholson, 2003 [PubMed 12659941]).[supplied by OMIM, Mar 2008]', '2', 'protein_coding', 57738013, 57750219, 51.31, 14, 'ENSG00000196950', 'Q9ULF5'),
(3053, 'SLC3A1', 'This gene encodes a type II membrane glycoprotein which is one of the components of the renal amino acid transporter which transports neutral and basic amino acids in the renal tubule and intestinal tract. Mutations and deletions in this gene are associated with cystinuria. Alternatively spliced transcript variants have been described, but their biological validity has not been determined. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 93667883, 93702592, 34.84, 6, 'ENSG00000138079', 'Q07837'),
(3054, 'SLC40A1', 'The protein encoded by this gene is a cell membrane protein that may be involved in iron export from duodenal epithelial cells. Defects in this gene are a cause of hemochromatosis type 4 (HFE4). [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 6638075, 6647433, 51.64, 9, 'ENSG00000138449', 'Q9NP59'),
(3055, 'SLC47A2', 'This gene encodes a protein belonging to a family of transporters involved in excretion of toxic electrolytes, both endogenous and exogenous, through urine and bile. This transporter family shares homology with the bacterial MATE (multidrug and toxin extrusion) protein family responsible for drug resistance. This gene is one of two members of the MATE transporter family located near each other on chromosome 17. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 19036282, 19122454, 48.62, 6, 'ENSG00000180638', 'Q86VL8'),
(3056, 'SLC4A8', 'The protein encoded by this gene is a membrane protein that functions to transport sodium and bicarbonate ions across the cell membrane. The encoded protein is important for pH regulation in neurons. The activity of this protein can be inhibited by 4,4\'-Di-isothiocyanatostilbene-2,2\'-disulfonic acid (DIDS). Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2012]', '12', 'protein_coding', 80742538, 80922359, 35.82, 9, 'ENSG00000050438', 'Q2Y0W8'),
(3057, 'SLC5A12', 'Normal blood lactate is maintained at about 1.5 mM, and little filtered lactate is excreted in urine. Reabsorption of lactate is mediated by the low-affinity Na(+)-coupled lactate transporter SLC5A12 in the initial part of the proximal tubule and by the high-affinity Na(+)-coupled lactate transporter SLC5A8 (MIM 608044) in the distal proximal tubule (Gopal et al., 2007 [PubMed 17692818]).[supplied by OMIM, Dec 2008]', '11', 'protein_coding', 174636920, 174829247, 35.39, 4, 'ENSG00000148942', 'Q1EHB4'),
(3058, 'SLC5A2', 'This gene encodes a member of the sodium glucose cotransporter family which are sodium-dependent glucose transport proteins. The encoded protein is the major cotransporter involved in glucose reabsorption in the kidney. Mutations in this gene are associated with renal glucosuria. Two transcript variants, one protein-coding and one not, have been found for this gene. [provided by RefSeq, Feb 2015]', '16', 'protein_coding', 9463320, 9741037, 45.27, 14, 'ENSG00000140675', 'P31639'),
(3059, 'SLC5A8', 'SLC5A8 has been shown to transport iodide by a passive mechanism (Rodriguez et al., 2002 [PubMed 12107270]) and monocarboxylates and short-chain fatty acids by a sodium-coupled mechanism (Gopal et al., 2004 [PubMed 15322102]). In kidney, SLC5A8 functions as a high-affinity sodium-coupled lactate transporter involved in reabsorption of lactate and maintenance of blood lactate levels (Thangaraju et al., 2006 [PubMed 16873376]).[supplied by OMIM, Dec 2008]', '12', 'miRNA', 4850297, 4850375, 40.51, 1, 'ENSG00000256870', 'Q8N695'),
(3060, 'SLC5A8', 'SLC5A8 has been shown to transport iodide by a passive mechanism (Rodriguez et al., 2002 [PubMed 12107270]) and monocarboxylates and short-chain fatty acids by a sodium-coupled mechanism (Gopal et al., 2004 [PubMed 15322102]). In kidney, SLC5A8 functions as a high-affinity sodium-coupled lactate transporter involved in reabsorption of lactate and maintenance of blood lactate levels (Thangaraju et al., 2006 [PubMed 16873376]).[supplied by OMIM, Dec 2008]', '12', 'miRNA', 4850297, 4850375, 40.51, 1, 'ENSG00000262217', 'Q8N695'),
(3061, 'SLC6A16', 'SLC6A16 shows structural characteristics of an Na(+)- and Cl(-)-dependent neurotransmitter transporter, including 12 transmembrane (TM) domains, intracellular N and C termini, and large extracellular loops containing multiple N-glycosylation sites.[supplied by OMIM, Mar 2008]', '19', 'protein_coding', 76565377, 76586956, 54.09, 9, 'ENSG00000063127', 'Q9GZN6'),
(3062, 'SLC6A17', 'The protein encoded by this gene is a member of the SLC6 family of transporters, which are responsible for the presynaptic uptake of most neurotransmitters. The encoded vesicular transporter is selective for proline, glycine, leucine and alanine. In mouse, the strongest expression of this gene was in cortical and hippocampal tissues where expression increased during embryonic brain development and peaked postnatally. Defects in this gene cause a form of autosomal recessive intellectual disability. [provided by RefSeq, Jul 2017]', '1', 'transcribed_unprocessed_pseudogene', 23918771, 23922169, 54.34, 1, 'ENSG00000197106', 'Q9H1V8'),
(3063, 'SLC6A3', 'This gene encodes a dopamine transporter which is a member of the sodium- and chloride-dependent neurotransmitter transporter family. The 3\' UTR of this gene contains a 40 bp tandem repeat, referred to as a variable number tandem repeat or VNTR, which can be present in 3 to 11 copies. Variation in the number of repeats is associated with idiopathic epilepsy, attention-deficit hyperactivity disorder, dependence on alcohol and cocaine, susceptibility to Parkinson disease and protection against nicotine dependence.[provided by RefSeq, Nov 2009]', '5', 'miRNA', 22933349, 22933438, 81.11, 1, 'ENSG00000276996', 'Q01959'),
(3064, 'SLC6A3', 'This gene encodes a dopamine transporter which is a member of the sodium- and chloride-dependent neurotransmitter transporter family. The 3\' UTR of this gene contains a 40 bp tandem repeat, referred to as a variable number tandem repeat or VNTR, which can be present in 3 to 11 copies. Variation in the number of repeats is associated with idiopathic epilepsy, attention-deficit hyperactivity disorder, dependence on alcohol and cocaine, susceptibility to Parkinson disease and protection against nicotine dependence.[provided by RefSeq, Nov 2009]', '5', 'miRNA', 22933349, 22933438, 81.11, 1, 'ENSG00000142319', 'Q01959'),
(3065, 'SLC9A7', 'This gene encodes a sodium and potassium/ proton antiporter that is a member of the solute carrier family 9 protein family. The encoded protein is primarily localized to the trans-Golgi network and is involved in maintaining pH homeostasis in organelles along the secretory and endocytic pathways. This protein may enhance cell growth of certain breast tumors. This gene is part of a gene cluster on chromosome Xp11.23. A pseudogene of this gene is found on chromosome 12. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Mar 2012]', 'X', 'protein_coding', 69199951, 69214748, 32.22, 2, 'ENSG00000065923', 'Q96T83'),
(3066, 'SLITRK1', 'This gene encodes a member of the SLITRK protein family. Members of this family are integral membrane proteins that are characterized by two N-terminal leucine-rich repeat (LRR) domains and a C-terminal region that shares homology with trk neurotrophin receptors. However, the protein encoded by this gene lacks the region of homology to neurotrophin receptors. This protein is thought to be involved in neurite outgrowth. Mutations in this gene may be associated with Tourette syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '13', 'protein_coding', 54475593, 54651169, 39.92, 4, 'ENSG00000178235', 'Q96PX8'),
(3067, 'SLITRK5', 'Members of the SLITRK family, such as SLITRK5, are integral membrane proteins with 2 N-terminal leucine-rich repeat (LRR) domains similar to those of SLIT proteins (see SLIT1; MIM 603742). Most SLITRKs, including SLITRK5, also have C-terminal regions that share homology with neurotrophin receptors (see NTRK1; MIM 191315). SLITRKs are expressed predominantly in neural tissues and have neurite-modulating activity (Aruga et al., 2003 [PubMed 14557068]).[supplied by OMIM, Mar 2008]', '13', 'protein_coding', 93202894, 93207065, 43.17, 2, 'ENSG00000165300', 'O94991'),
(3068, 'SLX1A-SULT1A3', 'This locus represents naturally occurring read-through transcription between the neighboring SLX1A (SLX1 structure-specific endonuclease subunit homolog A) and SULT1A3 (sulfotransferase family, cytosolic, 1A, phenol-preferring, member 3) genes on the short arm of chromosome 16. A duplicate read-through locus also exists between the SLX1B and SULT1A4 genes located approximately 730 kb upstream on the same chromosome. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is thus unlikely to produce a protein product. [provided by RefSeq, Feb 2017]', '16', 'miRNA', 140434, 140514, 38.27, 1, 'ENSG00000213599', ''),
(3069, 'SMAD2', 'The protein encoded by this gene belongs to the SMAD, a family of proteins similar to the gene products of the Drosophila gene \'mothers against decapentaplegic\' (Mad) and the C. elegans gene Sma. SMAD proteins are signal transducers and transcriptional modulators that mediate multiple signaling pathways. This protein mediates the signal of the transforming growth factor (TGF)-beta, and thus regulates multiple cellular processes, such as cell proliferation, apoptosis, and differentiation. This protein is recruited to the TGF-beta receptors through its interaction with the SMAD anchor for receptor activation (SARA) protein. In response to TGF-beta signal, this protein is phosphorylated by the TGF-beta receptors. The phosphorylation induces the dissociation of this protein with SARA and the association with the family member SMAD4. The association with SMAD4 is important for the translocation of this protein into the nucleus, where it binds to target promoters and forms a transcription repressor complex with other cofactors. This protein can also be phosphorylated by activin type 1 receptor kinase, and mediates the signal from the activin. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, May 2012]', '18', 'protein_coding', 87250, 97094, 37.75, 2, 'ENSG00000175387', 'Q15796'),
(3070, 'SMC1B', 'SMC1L2 belongs to a family of proteins required for chromatid cohesion and DNA recombination during meiosis and mitosis (3:Revenkova et al., 2001 [PubMed 11564881]).[supplied by OMIM, Mar 2008]', '22', 'protein_coding', 64241003, 64244132, 63.61, 6, 'ENSG00000077935', 'Q8NDV3'),
(3071, 'SMCHD1', 'This gene encodes a protein which contains a hinge region domain found in members of the SMC (structural maintenance of chromosomes) protein family. [provided by RefSeq, Dec 2011]', '18', 'protein_coding', 101789, 103953, 58.20, 1, 'ENSG00000101596', 'A6NHR9'),
(3072, 'SMCO1', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'miRNA', 12163683, 12163767, 61.18, 1, 'ENSG00000214097', 'Q147U7'),
(3073, 'SMDT1', 'This gene encodes a core regulatory component of a calcium channel in the mitochondrial inner membrane. [provided by RefSeq, Apr 2017]', '22', 'protein_coding', 146551, 168638, 57.65, 14, 'ENSG00000183172', 'Q9H4I9'),
(3074, 'SMDT1', 'This gene encodes a core regulatory component of a calcium channel in the mitochondrial inner membrane. [provided by RefSeq, Apr 2017]', '22', 'protein_coding', 146551, 168638, 57.65, 14, 'ENSG00000272901', 'Q9H4I9'),
(3075, 'SMDT1', 'This gene encodes a core regulatory component of a calcium channel in the mitochondrial inner membrane. [provided by RefSeq, Apr 2017]', '22', 'protein_coding', 146551, 168638, 57.65, 14, 'ENSG00000274112', 'Q9H4I9'),
(3076, 'SMDT1', 'This gene encodes a core regulatory component of a calcium channel in the mitochondrial inner membrane. [provided by RefSeq, Apr 2017]', '22', 'protein_coding', 146551, 168638, 57.65, 14, 'ENSG00000272835', 'Q9H4I9'),
(3077, 'SMG6', 'This gene encodes a component of the telomerase ribonucleoprotein complex responsible for the replication and maintenance of chromosome ends. The encoded protein also plays a role in the nonsense-mediated mRNA decay (NMD) pathway, providing the endonuclease activity near the premature translation termination codon that is needed to initiate NMD. Alternatively spliced transcript variants encoding distinct protein isoforms have been described. [provided by RefSeq, Feb 2014]', '17', 'protein_coding', 70412941, 70561174, 36.09, 16, 'ENSG00000070366', 'Q86US8'),
(3078, 'SMIM26', 'Enables protein kinase binding activity and transmembrane transporter binding activity. Located in mitochondrial outer membrane. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 110982015, 111028263, 42.58, 4, 'ENSG00000232388', 'A0A096LP01'),
(3079, 'SMIM30', 'Involved in negative regulation of type I interferon-mediated signaling pathway. Located in endoplasmic reticulum membrane and mitochondrial membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 639607, 665893, 55.11, 10, 'ENSG00000214194', 'A4D0T7'),
(3080, 'SMIM33', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'miRNA', 48428, 48500, 57.53, 1, 'ENSG00000288274', 'A0A1B0GW64'),
(3081, 'SMIM33', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'miRNA', 48428, 48500, 57.53, 1, 'ENSG00000283288', 'A0A1B0GW64'),
(3082, 'SMIM6', 'Predicted to act upstream of or within negative regulation of ATPase-coupled calcium transmembrane transporter activity. Predicted to be located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 179591613, 179691272, 36.77, 4, 'ENSG00000259120', 'P0DI80'),
(3083, 'SMN2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. The telomeric and centromeric copies of this gene are nearly identical and encode the same protein. While mutations in the telomeric copy are associated with spinal muscular atrophy, mutations in this gene, the centromeric copy, do not lead to disease. This gene may be a modifier of disease caused by mutation in the telomeric copy. The critical sequence difference between the two genes is a single nucleotide in exon 7, which is thought to be an exon splice enhancer. Note that the nine exons of both the telomeric and centromeric copies are designated historically as exon 1, 2a, 2b, and 3-8. It is thought that gene conversion events may involve the two genes, leading to varying copy numbers of each gene. The full length protein encoded by this gene localizes to both the cytoplasm and the nucleus. Within the nucleus, the protein localizes to subnuclear bodies called gems which are found near coiled bodies containing high concentrations of small ribonucleoproteins (snRNPs). This protein forms heteromeric complexes with proteins such as SIP1 and GEMIN4, and also interacts with several proteins known to be involved in the biogenesis of snRNPs, such as hnRNP U protein and the small nucleolar RNA binding protein. Four transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Sep 2008]', '5', 'protein_coding', 207821288, 208025527, 41.33, 4, 'ENSG00000273772', 'Q16637'),
(3084, 'SMN2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. The telomeric and centromeric copies of this gene are nearly identical and encode the same protein. While mutations in the telomeric copy are associated with spinal muscular atrophy, mutations in this gene, the centromeric copy, do not lead to disease. This gene may be a modifier of disease caused by mutation in the telomeric copy. The critical sequence difference between the two genes is a single nucleotide in exon 7, which is thought to be an exon splice enhancer. Note that the nine exons of both the telomeric and centromeric copies are designated historically as exon 1, 2a, 2b, and 3-8. It is thought that gene conversion events may involve the two genes, leading to varying copy numbers of each gene. The full length protein encoded by this gene localizes to both the cytoplasm and the nucleus. Within the nucleus, the protein localizes to subnuclear bodies called gems which are found near coiled bodies containing high concentrations of small ribonucleoproteins (snRNPs). This protein forms heteromeric complexes with proteins such as SIP1 and GEMIN4, and also interacts with several proteins known to be involved in the biogenesis of snRNPs, such as hnRNP U protein and the small nucleolar RNA binding protein. Four transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Sep 2008]', '5', 'protein_coding', 207821288, 208025527, 41.33, 4, 'ENSG00000277773', 'Q16637'),
(3085, 'SMN2', 'This gene is part of a 500 kb inverted duplication on chromosome 5q13. This duplicated region contains at least four genes and repetitive elements which make it prone to rearrangements and deletions. The repetitiveness and complexity of the sequence have also caused difficulty in determining the organization of this genomic region. The telomeric and centromeric copies of this gene are nearly identical and encode the same protein. While mutations in the telomeric copy are associated with spinal muscular atrophy, mutations in this gene, the centromeric copy, do not lead to disease. This gene may be a modifier of disease caused by mutation in the telomeric copy. The critical sequence difference between the two genes is a single nucleotide in exon 7, which is thought to be an exon splice enhancer. Note that the nine exons of both the telomeric and centromeric copies are designated historically as exon 1, 2a, 2b, and 3-8. It is thought that gene conversion events may involve the two genes, leading to varying copy numbers of each gene. The full length protein encoded by this gene localizes to both the cytoplasm and the nucleus. Within the nucleus, the protein localizes to subnuclear bodies called gems which are found near coiled bodies containing high concentrations of small ribonucleoproteins (snRNPs). This protein forms heteromeric complexes with proteins such as SIP1 and GEMIN4, and also interacts with several proteins known to be involved in the biogenesis of snRNPs, such as hnRNP U protein and the small nucleolar RNA binding protein. Four transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Sep 2008]', '5', 'protein_coding', 207821288, 208025527, 41.33, 4, 'ENSG00000205571', 'Q16637'),
(3086, 'SMPDL3A', 'Enables phosphoric diester hydrolase activity and zinc ion binding activity. Involved in negative regulation of cGAS/STING signaling pathway and nucleoside triphosphate catabolic process. Located in extracellular exosome. Is active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '6', 'miRNA', 136886733, 136886803, 70.42, 1, 'ENSG00000172594', 'Q92484'),
(3087, 'SMR3B', 'Predicted to enable endopeptidase inhibitor activity. Predicted to be involved in cellular response to lipopolysaccharide; negative regulation of peptidase activity; and regulation of sensory perception of pain. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 232109, 232178, 57.14, 1, 'ENSG00000171201', 'P02814'),
(3088, 'SMTN', 'This gene encodes a structural protein that is found exclusively in contractile smooth muscle cells. It associates with stress fibers and constitutes part of the cytoskeleton. This gene is localized to chromosome 22q12.3, distal to the TUPLE1 locus and outside the DiGeorge syndrome deletion. Alternative splicing of this gene results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, May 2011]', '22', 'snoRNA', 49497720, 49497806, 49.43, 1, 'ENSG00000183963', 'P53814'),
(3089, 'SNCG', 'This gene encodes a member of the synuclein family of proteins which are believed to be involved in the pathogenesis of neurodegenerative diseases. Mutations in this gene have also been associated with breast tumor development. [provided by RefSeq, Jan 2010]', '10', 'protein_coding', 107470018, 107475684, 30.76, 3, 'ENSG00000173267', 'O76070'),
(3090, 'SNORA11E', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 135156540, 135171398, 38.82, 2, 'ENSG00000221705', ''),
(3091, 'SNORA11F', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '10', 'lncRNA', 21825487, 21831539, 32.28, 4, 'ENSG00000221164', ''),
(3092, 'SNORA22', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 159789812, 159798436, 43.85, 4, 'ENSG00000206634', ''),
(3093, 'SNORA24', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 115392, 118091, 67.48, 5, 'ENSG00000275994', ''),
(3094, 'SNORA25B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 132750819, 132807241, 48.12, 16, 'ENSG00000202377', ''),
(3095, 'SNORA26', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 124566660, 124582942, 39.99, 4, 'ENSG00000212588', ''),
(3096, 'SNORA27', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 92383268, 92404696, 36.68, 4, 'ENSG00000207051', ''),
(3097, 'SNORA28', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 108589705, 108694840, 35.70, 6, 'ENSG00000272533', ''),
(3098, 'SNORA37', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 20455191, 20457772, 47.06, 20, 'ENSG00000207233', ''),
(3099, 'SNORA40C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 25718944, 25790039, 37.55, 21, 'ENSG00000212587', ''),
(3100, 'SNORA41', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 127111, 136530, 41.71, 3, 'ENSG00000283482', ''),
(3101, 'SNORA41', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 127111, 136530, 41.71, 3, 'ENSG00000207406', ''),
(3102, 'SNORA48B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', 'X', 'miRNA', 39661216, 39661321, 54.72, 1, 'ENSG00000212214', ''),
(3103, 'SNORA51', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 13836587, 13836659, 57.53, 1, 'ENSG00000271798', ''),
(3104, 'SNORA53', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 130486171, 130506465, 44.95, 17, 'ENSG00000212443', ''),
(3105, 'SNORA5A', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 102152176, 102163871, 47.22, 9, 'ENSG00000206838', ''),
(3106, 'SNORA5C', 'Small nucleolar RNAs (snoRNAs) are small noncoding RNAs involved in RNA processing. Box H/ACA snoRNAs, such as SNORA5C, direct the conversion of uridine to pseudouridine at specific residues of ribosomal RNAs or small nuclear RNAs (snRNAs) (Gu et al., 2005).[supplied by OMIM, Mar 2008]', '7', 'protein_coding', 24693873, 24996986, 38.80, 14, 'ENSG00000201772', ''),
(3107, 'SNORA64', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 16599779, 16645817, 38.01, 6, 'ENSG00000291933', ''),
(3108, 'SNORA64', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 16599779, 16645817, 38.01, 6, 'ENSG00000207405', ''),
(3109, 'SNORA67', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 39480412, 39493785, 53.07, 19, 'ENSG00000277985', ''),
(3110, 'SNORA69', 'This gene encodes a member of the family of box ACA small nucleolar RNAs, characterized by a consensus ACA motif positioned 3 nucleotides from the 3\' end of the RNAs and a double-hairpin structure. The box ACA small nucleolar RNAs may play a role in rRNA maturation. This gene is co-transcribed with the ribosomal protein L39 gene. [provided by RefSeq, Oct 2009]', 'X', 'protein_coding', 122264603, 122331337, 41.16, 12, 'ENSG00000206622', ''),
(3111, 'SNORA70G', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 23388493, 23453189, 45.97, 6, 'ENSG00000206650', ''),
(3112, 'SNORA71B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 155854738, 155920406, 36.51, 8, 'ENSG00000235408', ''),
(3113, 'SNORA71C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 110988926, 110989013, 35.23, 1, 'ENSG00000201512', ''),
(3114, 'SNORA75B', 'Predicted to be involved in RNA processing. Predicted to act upstream of or within response to bacterium. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '4', 'miRNA', 172662165, 172662263, 48.48, 1, 'ENSG00000206780', ''),
(3115, 'SNORA9', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 11598470, 11969748, 42.62, 15, 'ENSG00000277184', ''),
(3116, 'SNORD112', 'Small nucleolar RNAs (snoRNAs), like SNORD112, guide the formation of 2-prime O-methylation of ribosomal RNA (rRNA) and U small nuclear RNAs (snRNAs) through a specific RNA duplex at each modification site (Cavaille et al., 2002 [PubMed 12045206]).[supplied by OMIM, Nov 2010]', '14', 'protein_coding', 580380, 582114, 43.46, 1, 'ENSG00000275662', ''),
(3117, 'SNORD113-5', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 75620434, 75859870, 38.67, 28, 'ENSG00000272474', ''),
(3118, 'SNORD113-7', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'miRNA', 153981097, 153981177, 34.57, 1, 'ENSG00000200632', ''),
(3119, 'SNORD113-9', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 163323962, 163344832, 39.51, 7, 'ENSG00000201950', ''),
(3120, 'SNORD114-18', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 26581205, 26641366, 49.04, 3, 'ENSG00000202142', ''),
(3121, 'SNORD114-19', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 50523568, 50526461, 67.21, 5, 'ENSG00000199942', ''),
(3122, 'SNORD114-2', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 101390824, 101396155, 41.82, 10, 'ENSG00000200823', ''),
(3123, 'SNORD114-22', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 670318, 712662, 46.09, 22, 'ENSG00000202293', ''),
(3124, 'SNORD114-25', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 64470129, 64693058, 40.03, 6, 'ENSG00000200612', ''),
(3125, 'SNORD114-28', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 50525752, 50530032, 64.42, 17, 'ENSG00000200480', ''),
(3126, 'SNORD114-30', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 69585426, 69601570, 39.37, 14, 'ENSG00000201318', ''),
(3127, 'SNORD121A', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 183635610, 183684519, 38.98, 15, 'ENSG00000238886', ''),
(3128, 'SNORD125', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 35710676, 35710741, 60.61, 1, 'ENSG00000239127', ''),
(3129, 'SNORD12C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 122458551, 122514907, 49.66, 3, 'ENSG00000209042', ''),
(3130, 'SNORD15B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 37638, 99017, 34.98, 10, 'ENSG00000207445', ''),
(3131, 'SNORD17', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 19479457, 19520612, 48.03, 14, 'ENSG00000212232', ''),
(3132, 'SNORD1B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 31393335, 31426505, 51.78, 3, 'ENSG00000199961', ''),
(3133, 'SNORD20', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 15200, 38320, 46.36, 5, 'ENSG00000207280', ''),
(3134, 'SNORD25', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 67991099, 67996469, 52.45, 4, 'ENSG00000275043', ''),
(3135, 'SNORD28', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '11', 'miRNA', 64340759, 64340839, 51.85, 1, 'ENSG00000274544', ''),
(3136, 'SNORD34', 'Predicted to be involved in RNA processing. Predicted to act upstream of or within glucose metabolic process; insulin secretion; and reactive oxygen species metabolic process. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 74231499, 74254458, 48.96, 13, 'ENSG00000202503', ''),
(3137, 'SNORD35B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 58157907, 58198106, 41.51, 8, 'ENSG00000200530', ''),
(3138, 'SNORD36C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 29194071, 29630751, 39.61, 13, 'ENSG00000281541', ''),
(3139, 'SNORD36C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 29194071, 29630751, 39.61, 13, 'ENSG00000252542', ''),
(3140, 'SNORD3J', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 199814, 200780, 50.57, 1, 'ENSG00000221400', ''),
(3141, 'SNORD4B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'snoRNA', 52063347, 52063474, 55.47, 1, 'ENSG00000238597', ''),
(3142, 'SNORD56', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 13874875, 13875011, 59.85, 1, 'ENSG00000229686', ''),
(3143, 'SNORD58C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 39040604, 39055972, 53.18, 4, 'ENSG00000202093', ''),
(3144, 'SNORD61', 'Small nucleolar RNAs (snoRNAs) are small noncoding RNAs involved in RNA processing. Box C/D class snoRNAs are involved in site-specific 2-prime-O-ribose methylation of preribosomal RNA precursors. This snoRNA is located in an intron of the RNA binding motif protein, X-linked gene (RBMX). [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 100620416, 100642779, 53.90, 13, 'ENSG00000206979', ''),
(3145, 'SNORD65C', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 9363129, 9364721, 52.35, 1, 'ENSG00000212264', ''),
(3146, 'SNORD83B', 'Intronic regions of ribosomal protein genes can harbor noncoding small nucleolar RNAs (snoRNAs), like SNORD83B, which are generated during pre-mRNA processing. snoRNAs form part of the small nucleolar ribonucleoprotein particles (snoRNPs) involved in pre-rRNA processing and modification. snoRNAs of the box C/D class, like SNORD83B, function in 2-prime-O-ribose methylation of rRNAs (Duga et al., 2000 [PubMed 10684968]).[supplied by OMIM, Mar 2008]', '22', 'protein_coding', 68328308, 68531061, 40.29, 7, 'ENSG00000209480', ''),
(3147, 'SNORD88A', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 31530219, 31546608, 47.36, 3, 'ENSG00000221241', ''),
(3148, 'SNORD89', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '2', 'unprocessed_pseudogene', 58387583, 58388486, 40.04, 1, 'ENSG00000212283', ''),
(3149, 'SNORD91B', 'Predicted to be involved in RNA processing. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 140639435, 140645408, 51.32, 14, 'ENSG00000275084', ''),
(3150, 'SNPH', 'Syntaxin-1, synaptobrevin/VAMP, and SNAP25 interact to form the SNARE complex, which is required for synaptic vesicle docking and fusion. The protein encoded by this gene is membrane-associated and inhibits SNARE complex formation by binding free syntaxin-1. Expression of this gene appears to be brain-specific. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2015]', '20', 'protein_coding', 14108813, 14210955, 35.86, 8, 'ENSG00000101298', 'O15079'),
(3151, 'SNRPB', 'The protein encoded by this gene is one of several nuclear proteins that are found in common among U1, U2, U4/U6, and U5 small ribonucleoprotein particles (snRNPs). These snRNPs are involved in pre-mRNA splicing, and the encoded protein may also play a role in pre-mRNA splicing or snRNP structure. Autoantibodies from patients with systemic lupus erythematosus frequently recognize epitopes on the encoded protein. Two transcript variants encoding different isoforms (B and B\') have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'miRNA', 101025732, 101025817, 50.00, 1, 'ENSG00000125835', 'P14678'),
(3152, 'SNRPD3', 'This gene encodes a core component of the spliceosome, which is a nuclear ribonucleoprotein complex that functions in pre-mRNA splicing. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '22', 'protein_coding', 34942270, 35673022, 35.01, 58, 'ENSG00000100028', 'P62318'),
(3153, 'SNTA1', 'Syntrophins are cytoplasmic peripheral membrane scaffold proteins that are components of the dystrophin-associated protein complex. This gene is a member of the syntrophin gene family and encodes the most common syntrophin isoform found in cardiac tissues. The N-terminal PDZ domain of this syntrophin protein interacts with the C-terminus of the pore-forming alpha subunit (SCN5A) of the cardiac sodium channel Nav1.5. This protein also associates cardiac sodium channels with the nitric oxide synthase-PMCA4b (plasma membrane Ca-ATPase subtype 4b) complex in cardiomyocytes. This gene is a susceptibility locus for Long-QT syndrome (LQT) - an inherited disorder associated with sudden cardiac death from arrhythmia - and sudden infant death syndrome (SIDS). This protein also associates with dystrophin and dystrophin-related proteins at the neuromuscular junction and alters intracellular calcium ion levels in muscle tissue. [provided by RefSeq, Jan 2013]', '20', 'miRNA', 94809962, 94810057, 58.33, 1, 'ENSG00000101400', 'Q13424'),
(3154, 'SNTB2', 'Dystrophin is a large, rod-like cytoskeletal protein found at the inner surface of muscle fibers. Dystrophin is missing in Duchenne Muscular Dystrophy patients and is present in reduced amounts in Becker Muscular Dystrophy patients. The protein encoded by this gene is a peripheral membrane protein found associated with dystrophin and dystrophin-related proteins. This gene is a member of the syntrophin gene family, which contains at least two other structurally-related genes. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 164877178, 164898965, 37.86, 5, 'ENSG00000260873', 'Q13425');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3155, 'SNTB2', 'Dystrophin is a large, rod-like cytoskeletal protein found at the inner surface of muscle fibers. Dystrophin is missing in Duchenne Muscular Dystrophy patients and is present in reduced amounts in Becker Muscular Dystrophy patients. The protein encoded by this gene is a peripheral membrane protein found associated with dystrophin and dystrophin-related proteins. This gene is a member of the syntrophin gene family, which contains at least two other structurally-related genes. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 164877178, 164898965, 37.86, 5, 'ENSG00000168807', 'Q13425'),
(3156, 'SNURFL', 'Predicted to be located in nucleus. Predicted to be active in nuclear speck. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 11479155, 11537551, 52.86, 3, 'ENSG00000173954', 'B1AK76'),
(3157, 'SNX13', 'This gene encodes a PHOX domain- and RGS domain-containing protein that belongs to the sorting nexin (SNX) family and the regulator of G protein signaling (RGS) family. The PHOX domain is a phosphoinositide binding domain, and the SNX family members are involved in intracellular trafficking. The RGS family members are regulatory molecules that act as GTPase activating proteins for G alpha subunits of heterotrimeric G proteins. The RGS domain of this protein interacts with G alpha(s), accelerates its GTP hydrolysis, and attenuates G alpha(s)-mediated signaling. Overexpression of this protein delayes lysosomal degradation of the epidermal growth factor receptor. Because of its bifunctional role, this protein may link heterotrimeric G protein signaling and vesicular trafficking. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 80960363, 81230162, 40.98, 6, 'ENSG00000071189', 'Q9Y5W8'),
(3158, 'SNX2', 'This gene belongs to the sorting nexin family whose members contain the phosphoinositide-binding phox (PX) domain. The encoded protein is a component of the retromer complex which plays a role in protein sorting in the endocytic pathway. This protein may form oligomeric complexes with other family members. Alternate splicing results in multiple transcript variants of this gene. Pseudogenes associated with this gene are located on chromosomes 1 and 7. [provided by RefSeq, May 2013]', '5', 'miRNA', 45705102, 45705180, 43.04, 1, 'ENSG00000205302', 'O60749'),
(3159, 'SNX4', 'This gene encodes a member of the sorting nexin family. Members of this family contain a phox (PX) domain, which is a phosphoinositide binding domain, and are involved in intracellular trafficking. This protein associated with the long isoform of the leptin receptor and with receptor tyrosine kinases for platelet-derived growth factor, insulin, and epidermal growth factor in cell cultures, but its function is unknown. This protein may form oligomeric complexes with family members. Two transcript variants, one protein coding and the other non-protein coding, have been found for this gene. [provided by RefSeq, Nov 2012]', '3', 'protein_coding', 101155493, 101210238, 38.28, 1, 'ENSG00000114520', 'O95219'),
(3160, 'SNX9', 'This gene encodes a member of the sorting nexin family. Members of this family contain a phosphoinositide binding domain, and are involved in intracellular trafficking. The encoded protein does not contain a coiled coil region, like some family members, but does contain a SRC homology domain near its N-terminus. The encoded protein is reported to have a variety of interaction partners, including of adaptor protein 2 , dynamin, tyrosine kinase non-receptor 2, Wiskott-Aldrich syndrome-like, and ARP3 actin-related protein 3. The encoded protein is implicated in several stages of intracellular trafficking, including endocytosis, macropinocytosis, and F-actin nucleation. [provided by RefSeq, Jul 2013]', '6', 'protein_coding', 73110375, 73131828, 41.24, 8, 'ENSG00000130340', 'Q9Y5X1'),
(3161, 'SOBP', 'The protein encoded by this gene is a nuclear zinc finger protein that is involved in development of the cochlea. Defects in this gene have also been linked to intellectual disability. [provided by RefSeq, Mar 2011]', '6', 'protein_coding', 108894471, 108989372, 48.85, 3, 'ENSG00000112320', 'A7XYQ1'),
(3162, 'SOCS6', 'The protein encoded by this gene contains a SH2 domain and a CIS homolog domain. The protein thus belongs to the cytokine-induced STAT inhibitor (CIS), also known as suppressor of cytokine signaling (SOCS) or STAT-induced STAT inhibitor (SSI), protein family. CIS family members are known to be cytokine-inducible negative regulators of cytokine signaling. The expression of this gene can be induced by GM-CSF and EPO in hematopoietic cells. A high expression level of this gene was found in factor-independent chronic myelogenous leukemia (CML) and erythroleukemia (HEL) cell lines. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 6772708, 6857366, 47.39, 10, 'ENSG00000170677', 'O14544'),
(3163, 'SOCS7', 'Predicted to enable phosphorylation-dependent protein binding activity; signaling adaptor activity; and ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in insulin receptor signaling pathway; proteasome-mediated ubiquitin-dependent protein catabolic process; and regulation of neuron migration. Predicted to act upstream of or within brain development; fat cell differentiation; and protein ubiquitination. Located in cytosol. Is active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 49933064, 49933137, 67.57, 1, 'ENSG00000274229', 'O14512'),
(3164, 'SOCS7', 'Predicted to enable phosphorylation-dependent protein binding activity; signaling adaptor activity; and ubiquitin-like ligase-substrate adaptor activity. Predicted to be involved in insulin receptor signaling pathway; proteasome-mediated ubiquitin-dependent protein catabolic process; and regulation of neuron migration. Predicted to act upstream of or within brain development; fat cell differentiation; and protein ubiquitination. Located in cytosol. Is active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 49933064, 49933137, 67.57, 1, 'ENSG00000274211', 'O14512'),
(3165, 'SOHLH1', 'This gene encodes one of testis-specific transcription factors which are essential for spermatogenesis, oogenesis and folliculogenesis. This gene is located on chromosome 9. Mutations in this gene are associated with nonobstructive azoospermia. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2013]', '9', 'protein_coding', 152133310, 152138578, 51.30, 6, 'ENSG00000165643', 'Q5JUK2'),
(3166, 'SOX12', 'Members of the SOX family of transcription factors are characterized by the presence of a DNA-binding high mobility group (HMG) domain, homologous to the HMG box of sex-determining region Y (SRY). Forming a subgroup of the HMG domain superfamily, SOX proteins have been implicated in cell fate decisions in a diverse range of developmental processes. SOX transcription factors have diverse tissue-specific expression patterns during early development and have been proposed to act as target-specific transcription factors and/or as chromatin structure regulatory elements. The protein encoded by this gene was identified as a SOX family member based on conserved domains, and its expression in various tissues suggests a role in both differentiation and maintenance of several cell types. [provided by RefSeq, Jan 2013]', '20', 'protein_coding', 50059391, 50085426, 55.49, 5, 'ENSG00000177732', 'O15370'),
(3167, 'SOX7', 'This gene encodes a member of the SOX (SRY-related HMG-box) family of transcription factors involved in the regulation of embryonic development and in the determination of the cell fate. The encoded protein may act as a transcriptional regulator after forming a protein complex with other proteins. The protein may play a role in tumorigenesis. A similar protein in mice is involved in the regulation of the wingless-type MMTV integration site family (Wnt) pathway. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 23095505, 23100456, 44.77, 2, 'ENSG00000285438', 'Q9BT81'),
(3168, 'SOX7', 'This gene encodes a member of the SOX (SRY-related HMG-box) family of transcription factors involved in the regulation of embryonic development and in the determination of the cell fate. The encoded protein may act as a transcriptional regulator after forming a protein complex with other proteins. The protein may play a role in tumorigenesis. A similar protein in mice is involved in the regulation of the wingless-type MMTV integration site family (Wnt) pathway. [provided by RefSeq, Jul 2008]', '8', 'protein_coding', 23095505, 23100456, 44.77, 2, 'ENSG00000171056', 'Q9BT81'),
(3169, 'SP100', 'This gene encodes a subnuclear organelle and major component of the PML (promyelocytic leukemia)-SP100 nuclear bodies. PML and SP100 are covalently modified by the SUMO-1 modifier, which is considered crucial to nuclear body interactions. The encoded protein binds heterochromatin proteins and is thought to play a role in tumorigenesis, immunity, and gene regulation. Alternatively spliced variants have been identified for this gene; one of which encodes a high-mobility group protein. [provided by RefSeq, Aug 2011]', '2', 'protein_coding', 57561080, 57568121, 48.32, 6, 'ENSG00000067066', 'P23497'),
(3170, 'SP7', 'This gene encodes a member of the Sp subfamily of Sp/XKLF transcription factors. Sp family proteins are sequence-specific DNA-binding proteins characterized by an amino-terminal trans-activation domain and three carboxy-terminal zinc finger motifs. This protein is a bone specific transcription factor and is required for osteoblast differentiation and bone formation.[provided by RefSeq, Jul 2010]', '12', 'protein_coding', 20040446, 20154404, 40.07, 5, 'ENSG00000291837', 'Q8TDD2'),
(3171, 'SP7', 'This gene encodes a member of the Sp subfamily of Sp/XKLF transcription factors. Sp family proteins are sequence-specific DNA-binding proteins characterized by an amino-terminal trans-activation domain and three carboxy-terminal zinc finger motifs. This protein is a bone specific transcription factor and is required for osteoblast differentiation and bone formation.[provided by RefSeq, Jul 2010]', '12', 'protein_coding', 20040446, 20154404, 40.07, 5, 'ENSG00000170374', 'Q8TDD2'),
(3172, 'SPAAR', 'Involved in cellular response to amino acid stimulus and negative regulation of TORC1 signaling. Located in late endosome membrane and lysosomal proton-transporting V-type ATPase complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 50502949, 50507702, 67.14, 6, 'ENSG00000235387', 'A0A1B0GVQ0'),
(3173, 'SPACA6-AS1', 'Predicted to be involved in miRNA-mediated post-transcriptional gene silencing. Predicted to be part of RISC complex. [provided by Alliance of Genome Resources, May 2025]', '19', 'snoRNA', 2329016, 2329237, 40.99, 1, 'ENSG00000269959', ''),
(3174, 'SPAG5', 'This gene encodes a protein associated with the mitotic spindle apparatus. The encoded protein may be involved in the functional and dynamic regulation of mitotic spindles. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 44158811, 44184401, 51.57, 5, 'ENSG00000076382', 'Q96R06'),
(3175, 'SPANXA2', 'Temporally regulated transcription and translation of several testis-specific genes is required to initiate the series of molecular and morphological changes in the male germ cell lineage necessary for the formation of mature spermatozoa. This gene is a member of the SPANX family of cancer/testis-associated genes, which are located in a cluster on chromosome X. The SPANX genes encode differentially expressed testis-specific proteins that localize to various subcellular compartments. This particular gene maps to chromosome X in a head-to-head orientation with SPANX family member A1 and appears to be a duplication of that locus. The protein encoded by this gene targets to the nucleus where it associates with nuclear vacuoles and the redundant nuclear envelope. Based on its association with these poorly characterized regions of the sperm nucleus, this protein provides a biochemical marker to study unique structures in spermatazoa while attempting to further define its role in spermatogenesis. [provided by RefSeq, Jul 2008]', 'X', 'miRNA', 16627012, 16627085, 37.84, 1, 'ENSG00000203926', 'Q9NS26'),
(3176, 'SPANXB1', 'Temporally regulated transcription and translation of several testis-specific genes is required to initiate the series of molecular and morphological changes in the male germ cell lineage necessary for the formation of mature spermatozoa. This gene is a member of the SPANX family of cancer/testis-associated genes, which are located in a cluster on chromosome X. The SPANX genes encode differentially expressed testis-specific proteins that localize to various subcellular compartments. This particular family member contains an additional 18 nucleotides in its coding region compared to the other family members in the same gene cluster. This family member is also subject to gene copy number variation. Although the protein encoded by this gene contains consensus nuclear localization signals, the major site for subcellular localization of expressed protein is in the cytoplasmic droplets of ejaculated spermatozoa. This protein provides a biochemical marker for studying the unique structures in spermatazoa, while attempting to further define its role in spermatogenesis. [provided by RefSeq, Apr 2014]', 'X', 'protein_coding', 55883297, 55891810, 36.58, 2, 'ENSG00000227234', 'Q9NS25'),
(3177, 'SPANXC', 'Temporally regulated transcription and translation of several testis-specific genes is required to initiate the series of molecular and morphological changes in the male germ cell lineage necessary for the formation of mature spermatozoa. This gene is a member of the SPANX family, which is located in a gene cluster on chromosome X. The SPANX genes encode differentially expressed testis-specific proteins that localize to various subcellular compartments. This particular gene encodes a protein that localizes to the nucleus and is expressed in highly metastatic cell lines, making the protein a potential diagnostic and prognostic marker. The protein belongs to a family of cancer/testis antigens and represents a potential target for cancer immunotherapy. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 74871362, 75000173, 44.73, 23, 'ENSG00000198573', 'Q9NY87'),
(3178, 'SPATA12', 'This gene is expressed primarily in testis and may play a role in testicular development and spermatogenesis. The encoded protein may be upregulated in response to ultraviolet-C radiation. [provided by RefSeq, Dec 2015]', '3', 'protein_coding', 42846, 43781, 45.51, 1, 'ENSG00000186451', 'Q7Z6I5'),
(3179, 'SPATA31A5', 'Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 222566899, 222656092, 38.07, 2, 'ENSG00000276581', 'Q5VU36'),
(3180, 'SPATA31E1', 'Predicted to be involved in cell differentiation and spermatogenesis. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 107206611, 107244247, 36.95, 4, 'ENSG00000177992', 'Q6ZUB1'),
(3181, 'SPATC1L', 'Enables identical protein binding activity. Predicted to be involved in spermatogenesis. Predicted to act upstream of or within actin polymerization or depolymerization; positive regulation of cAMP-dependent protein kinase activity; and positive regulation of cAMP/PKA signal transduction. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 8455962, 8478330, 36.72, 1, 'ENSG00000160284', 'Q9H0A9'),
(3182, 'SPATC1L', 'Enables identical protein binding activity. Predicted to be involved in spermatogenesis. Predicted to act upstream of or within actin polymerization or depolymerization; positive regulation of cAMP-dependent protein kinase activity; and positive regulation of cAMP/PKA signal transduction. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 8455962, 8478330, 36.72, 1, 'ENSG00000274679', 'Q9H0A9'),
(3183, 'SPEF1', 'Enables actin binding activity. Involved in filopodium assembly and lamellipodium assembly. Located in several cellular components, including basolateral plasma membrane; lamellipodium; and microvillus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 23024, 33840, 61.69, 1, 'ENSG00000101222', 'Q9Y4P9'),
(3184, 'SPEF2', 'Involved in sperm axoneme assembly. Located in cytosol; nuclear body; and sperm flagellum. Implicated in spermatogenic failure 43. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 35701347, 35742312, 40.28, 47, 'ENSG00000152582', 'Q9C093'),
(3185, 'SPEM1', 'Predicted to be involved in flagellated sperm motility and sperm individualization. Predicted to act upstream of or within spermatogenesis. Predicted to be located in membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 57140918, 57143939, 49.77, 1, 'ENSG00000284598', 'Q8N4L4'),
(3186, 'SPEM1', 'Predicted to be involved in flagellated sperm motility and sperm individualization. Predicted to act upstream of or within spermatogenesis. Predicted to be located in membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 57140918, 57143939, 49.77, 1, 'ENSG00000181323', 'Q8N4L4'),
(3187, 'SPEM3', 'Predicted to be involved in flagellated sperm motility and sperm individualization. Predicted to be located in membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 54717860, 54895516, 37.74, 4, 'ENSG00000284090', 'A0A1B0GUW6'),
(3188, 'SPEM3', 'Predicted to be involved in flagellated sperm motility and sperm individualization. Predicted to be located in membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 54717860, 54895516, 37.74, 4, 'ENSG00000283439', 'A0A1B0GUW6'),
(3189, 'SPG7', 'This gene encodes a mitochondrial metalloprotease protein that is a member of the AAA family. Members of this protein family share an ATPase domain and have roles in diverse cellular processes including membrane trafficking, intracellular motility, organelle biogenesis, protein folding, and proteolysis. Mutations in this gene cause autosomal recessive spastic paraplegia 7. Two transcript variants encoding distinct isoforms have been identified. [provided by RefSeq, Mar 2014]', '16', 'protein_coding', 49496365, 49499708, 53.08, 9, 'ENSG00000197912', 'Q9UQ90'),
(3190, 'SPIN1', 'Enables histone reader activity and methylated histone binding activity. Involved in positive regulation of DNA-templated transcription; positive regulation of Wnt signaling pathway; and rRNA transcription. Located in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'miRNA', 147284641, 147284728, 40.91, 1, 'ENSG00000106723', 'Q9H2V7'),
(3191, 'SPINDOC', 'Involved in DNA damage response and negative regulation of DNA-templated transcription. Is active in nucleus and site of DNA damage. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 335059, 352046, 49.62, 28, 'ENSG00000168005', 'Q9BUA3'),
(3192, 'SPIRE1', 'Spire proteins, such as SPIRE1, are highly conserved between species. They belong to the family of Wiskott-Aldrich homology region-2 (WH2) proteins, which are involved in actin organization (Kerkhoff et al., 2001 [PubMed 11747823]).[supplied by OMIM, Mar 2008]', '18', 'protein_coding', 88382959, 88440757, 61.26, 2, 'ENSG00000134278', 'Q08AE8'),
(3193, 'SPIRE2', 'Predicted to enable microtubule binding activity. Involved in establishment of meiotic spindle localization; formin-nucleated actin cable assembly; and positive regulation of double-strand break repair. Predicted to be located in cleavage furrow. Predicted to be active in cell cortex and cytoplasmic vesicle membrane. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 179347709, 179394936, 38.55, 9, 'ENSG00000204991', 'Q8WWL2'),
(3194, 'SPMIP5', 'Predicted to be involved in flagellated sperm motility. Predicted to be located in several cellular components, including cytoskeleton; motile cilium; and nucleus. Predicted to be active in axonemal A tubule inner sheath and sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 36296288, 36465204, 43.76, 20, 'ENSG00000291892', 'Q8WW14'),
(3195, 'SPMIP5', 'Predicted to be involved in flagellated sperm motility. Predicted to be located in several cellular components, including cytoskeleton; motile cilium; and nucleus. Predicted to be active in axonemal A tubule inner sheath and sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 36296288, 36465204, 43.76, 20, 'ENSG00000165863', 'Q8WW14'),
(3196, 'SPMIP7', 'Predicted to be involved in spermatogenesis. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 112175510, 112224699, 40.07, 19, 'ENSG00000164500', 'A4D263'),
(3197, 'SPNS2', 'The protein encoded by this gene is a transporter of sphingosine 1-phosphate, a secreted lipid that is important in cardiovascular, immunological, and neural development. Defects in this gene are a cause of early onset progressive hearing loss. [provided by RefSeq, Jul 2016]', '17', 'protein_coding', 155277463, 155289848, 53.94, 4, 'ENSG00000183018', 'Q8IVW8'),
(3198, 'SPPL2C', 'Enables protein homodimerization activity. Predicted to be involved in several processes, including acrosome assembly; fusion of sperm to egg plasma membrane involved in single fertilization; and membrane protein intracellular domain proteolysis. Located in cytoplasmic side of endoplasmic reticulum membrane and lumenal side of endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 112107931, 112254939, 39.12, 24, 'ENSG00000185294', 'Q8IUH8'),
(3199, 'SPPL2C', 'Enables protein homodimerization activity. Predicted to be involved in several processes, including acrosome assembly; fusion of sperm to egg plasma membrane involved in single fertilization; and membrane protein intracellular domain proteolysis. Located in cytoplasmic side of endoplasmic reticulum membrane and lumenal side of endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 112107931, 112254939, 39.12, 24, 'ENSG00000278194', 'Q8IUH8'),
(3200, 'SPPL2C', 'Enables protein homodimerization activity. Predicted to be involved in several processes, including acrosome assembly; fusion of sperm to egg plasma membrane involved in single fertilization; and membrane protein intracellular domain proteolysis. Located in cytoplasmic side of endoplasmic reticulum membrane and lumenal side of endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 112107931, 112254939, 39.12, 24, 'ENSG00000277106', 'Q8IUH8'),
(3201, 'SPRY3', 'Involved in negative regulation of MAPK cascade. Predicted to be located in cytoplasm and membrane. Predicted to be active in cytosol. [provided by Alliance of Genome Resources, May 2025]', 'X, Y', 'protein_coding', 1842, 18614, 47.14, 2, 'ENSG00000168939', 'O43610'),
(3202, 'SPSB1', 'Enables ubiquitin-like ligase-substrate adaptor activity. Involved in protein ubiquitination and ubiquitin-dependent protein catabolic process. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '1', 'miRNA', 144400277, 144400345, 63.77, 1, 'ENSG00000171621', 'Q96BD6'),
(3203, 'SRBD1', 'Predicted to enable mRNA binding activity. Predicted to be a structural constituent of ribosome. Predicted to be involved in translation. [provided by Alliance of Genome Resources, May 2025]', '2', 'snoRNA', 75404421, 75404566, 47.95, 1, 'ENSG00000068784', 'Q8N5C6'),
(3204, 'SRMS', 'Enables protein tyrosine kinase activity. Involved in peptidyl-tyrosine autophosphorylation and positive regulation of TORC1 signaling. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 75131469, 75176612, 42.19, 7, 'ENSG00000125508', 'Q9H3Y6'),
(3205, 'SRP19', 'Enables 7S RNA binding activity. Contributes to ribosome binding activity. Predicted to be involved in SRP-dependent cotranslational protein targeting to membrane, signal sequence recognition. Located in cytosol; nuclear body; and nucleolus. Part of signal recognition particle, endoplasmic reticulum targeting. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 155382095, 155383801, 64.79, 1, 'ENSG00000153037', 'P09132'),
(3206, 'SRPX2', 'This gene encodes a secreted protein that contains three sushi repeat motifs. The encoded protein may play a role in the development of speech and language centers in the brain. This protein may also be involved in angiogenesis. Mutations in this gene are the cause of bilateral perisylvian polymicrogyria, rolandic epilepsy, speech dyspraxia and cognitive disability. [provided by RefSeq, May 2010]', 'X', 'protein_coding', 118376555, 118418033, 44.37, 4, 'ENSG00000102359', 'O60687'),
(3207, 'SRRM4', 'SRRM4 promotes alternative splicing and inclusion of neural-specific exons in target mRNAs (Calarco et al., 2009 [PubMed 19737518]).[supplied by OMIM, Oct 2009]', '12', 'miRNA', 90016621, 90016720, 38.00, 1, 'ENSG00000139767', 'A7MD48'),
(3208, 'SRSF1', 'This gene encodes a member of the arginine/serine-rich splicing factor protein family. The encoded protein can either activate or repress splicing, depending on its phosphorylation state and its interaction partners. Multiple transcript variants have been found for this gene. There is a pseudogene of this gene on chromosome 13. [provided by RefSeq, Jun 2014]', '17', 'protein_coding', 1496402, 1541763, 42.50, 3, 'ENSG00000136450', 'Q07955'),
(3209, 'SSBP4', 'Predicted to enable single-stranded DNA binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to be located in membrane. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 37082, 50989, 48.15, 1, 'ENSG00000130511', 'Q9BWG4'),
(3210, 'SSRP1', 'The protein encoded by this gene is a subunit of a heterodimer that, along with SUPT16H, forms chromatin transcriptional elongation factor FACT. FACT interacts specifically with histones H2A/H2B to effect nucleosome disassembly and transcription elongation. FACT and cisplatin-damaged DNA may be crucial to the anticancer mechanism of cisplatin. This encoded protein contains a high mobility group box which most likely constitutes the structure recognition element for cisplatin-modified DNA. This protein also functions as a co-activator of the transcriptional activator p63. An alternatively spliced transcript variant of this gene has been described, but its full-length nature is not known. [provided by RefSeq, Jul 2008]', '11', 'miRNA', 134127125, 134127192, 52.94, 1, 'ENSG00000149136', 'Q08945'),
(3211, 'SSTR3', 'This gene encodes a member of the somatostatin receptor protein family. Somatostatins are peptide hormones that regulate diverse cellular functions such as neurotransmission, cell proliferation, and endocrine signaling as well as inhibiting the release of many hormones and other secretory proteins. Somatostatin has two active forms of 14 and 28 amino acids. The biological effects of somatostatins are mediated by a family of G-protein coupled somatostatin receptors that are expressed in a tissue-specific manner. Somatostatin receptors form homodimers and heterodimers with other members of the superfamily as well as with other G-protein coupled receptors and receptor tyrosine kinases. This protein is functionally coupled to adenylyl cyclase. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '22', 'protein_coding', 99981784, 99986765, 56.62, 5, 'ENSG00000278195', 'P32745'),
(3212, 'SSX2B', 'The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneous humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. This gene, and also the SSX1 and SSX4 family members, have been involved in t(X;18)(p11.2;q11.2) translocations that are characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. The encoded hybrid proteins are likely responsible for transforming activity. Alternative splicing of this gene results in multiple transcript variants. This gene also has an identical duplicate, GeneID: 6757, located about 45 kb upstream in the opposite orientation on chromosome X. [provided by RefSeq, Jul 2013]', 'X', 'miRNA', 128392621, 128392708, 55.68, 1, 'ENSG00000268447', 'Q16385'),
(3213, 'SSX4', 'The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneously humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. SSX1, SSX2 and SSX4 genes have been involved in the t(X;18) translocation characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. Chromosome Xp11 contains a segmental duplication resulting in two identical copies of synovial sarcoma, X breakpoint 4, SSX4 and SSX4B, in tail-to-tail orientation. This gene, SSX4, represents the more telomeric copy. Two transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 92560758, 92590393, 37.44, 11, 'ENSG00000268009', 'O60224'),
(3214, 'SSX4B', 'The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneously humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. SSX1, SSX2 and SSX4 genes have been involved in the t(X;18) translocation characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. Chromosome Xp11 contains a segmental duplication resulting in two identical copies of synovial sarcoma, X breakpoint 4, SSX4 and SSX4B, in tail-to-tail orientation. This gene, SSX4B, represents the more centromeric copy. Two transcript variants encoding distinct isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 73423579, 73433561, 42.14, 3, 'ENSG00000269791', 'O60224'),
(3215, 'ST13P4', 'Predicted to enable protein dimerization activity. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 76978827, 77133683, 48.31, 49, 'ENSG00000232150', 'Q8IZP2'),
(3216, 'ST3GAL2', 'The protein encoded by this gene is a type II membrane protein that catalyzes the transfer of sialic acid from CMP-sialic acid to galactose-containing substrates. The encoded protein is normally found in the Golgi but can be proteolytically processed to a soluble form. This protein, which is a member of glycosyltransferase family 29, can use the same acceptor substrates as does sialyltransferase 4A. [provided by RefSeq, Jul 2008]', '16', 'unprocessed_pseudogene', 5903767, 5904448, 40.91, 1, 'ENSG00000157350', 'Q16842'),
(3217, 'ST6GALNAC2', 'ST6GALNAC2 belongs to a family of sialyltransferases that add sialic acids to the nonreducing ends of glycoconjugates. At the cell surface, these modifications have roles in cell-cell and cell-substrate interactions, bacterial adhesion, and protein targeting (Samyn-Petit et al., 2000 [PubMed 10742600]).[supplied by OMIM, Mar 2008]', '17', 'protein_coding', 151159748, 151165948, 46.07, 2, 'ENSG00000070731', 'Q9UJ37'),
(3218, 'ST8SIA5', 'The protein encoded by this gene is a type II membrane protein that may be present in the Golgi apparatus. The encoded protein, which is a member of glycosyltransferase family 29, may be involved in the synthesis of gangliosides GD1c, GT1a, GQ1b, and GT3 from GD1a, GT1b, GM1b, and GD3, respectively. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 27792483, 27794768, 64.26, 1, 'ENSG00000101638', 'O15466'),
(3219, 'ST8SIA6', 'This gene encodes a member of the glycosyltransferase 29 protein family. Members of this protein family synthesize sialylglycoconjugates. Sialylation may contribute to multidrug resistance in cancer cells. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2016]', '10', 'protein_coding', 33070141, 33072413, 64.80, 3, 'ENSG00000148488', 'P61647'),
(3220, 'STAM', 'This gene encodes a member of the signal-transducing adaptor molecule family. These proteins mediate downstream signaling of cytokine receptors and also play a role in ER to Golgi trafficking by interacting with the coat protein II complex. The encoded protein also associates with hepatocyte growth factor-regulated substrate to form the endosomal sorting complex required for transport-0 (ESCRT-0), which sorts ubiquitinated membrane proteins to the ESCRT-1 complex for lysosomal degradation. Alternatively spliced transcript variants have been observed for this gene. [provided by RefSeq, Feb 2011]', '10', 'protein_coding', 5736448, 5738523, 41.33, 1, 'ENSG00000136738', 'Q92783'),
(3221, 'STAMBPL1', 'Enables K63-linked deubiquitinase activity. Involved in cellular response to L-leucine and positive regulation of TORC1 signaling. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 89818179, 89871319, 54.79, 10, 'ENSG00000138134', 'Q96FJ0'),
(3222, 'STARD13', 'This gene encodes a protein which contains an N-terminal sterile alpha motif (SAM) for protein-protein interactions, followed by an ATP/GTP-binding motif, a GTPase-activating protein (GAP) domain, and a C-terminal STAR-related lipid transfer (START) domain. It may be involved in regulation of cytoskeletal reorganization, cell proliferation, and cell motility, and acts as a tumor suppressor in hepatoma cells. The gene is located in a region of chromosome 13 that is associated with loss of heterozygosity in hepatocellular carcinomas. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Aug 2011]', '13', 'unprocessed_pseudogene', 55903341, 55904278, 42.22, 1, 'ENSG00000133121', 'Q9Y3M8'),
(3223, 'STK10', 'This gene encodes a member of the Ste20 family of serine/threonine protein kinases, and is similar to several known polo-like kinase kinases. The protein can associate with and phosphorylate polo-like kinase 1, and overexpression of a kinase-dead version of the protein interferes with normal cell cycle progression. The kinase can also negatively regulate interleukin 2 expression in T-cells via the mitogen activated protein kinase kinase 1 pathway. [provided by RefSeq, Jul 2008]', '5', 'unprocessed_pseudogene', 145891, 146817, 36.68, 1, 'ENSG00000072786', 'O94804'),
(3224, 'STK38L', 'Enables ATP binding activity; magnesium ion binding activity; and protein serine/threonine kinase activity. Involved in intracellular signal transduction and negative regulation of autophagy. Acts upstream of or within protein phosphorylation. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 116100851, 116249497, 36.29, 4, 'ENSG00000211455', 'Q9Y2H1'),
(3225, 'STOML3', 'Predicted to act upstream of or within signal transduction. Predicted to be located in cilium; membrane raft; and neuron projection. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 17226107, 17484578, 41.47, 5, 'ENSG00000133115', 'Q8TAV4'),
(3226, 'STOX1', 'Enables RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in several processes, including cellular response to nitrosative stress; positive regulation of mitotic cell cycle phase transition; and regulation of macromolecule metabolic process. Located in centrosome; cytosol; and nuclear lumen. Implicated in pre-eclampsia. Biomarker of Alzheimer\'s disease. [provided by Alliance of Genome Resources, May 2025]', '10', 'snoRNA', 50799442, 50799532, 70.33, 1, 'ENSG00000165730', 'Q6ZVD7'),
(3227, 'STOX2', 'This gene encodes a Storkhead-box_winged-helix domain containing protein. This protein is differentially expressed in decidual tissue and may be involved in the susceptibility to pre-eclampsia with fetal growth restriction. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Sep 2015]', '4', 'protein_coding', 2014637, 2015957, 57.68, 1, 'ENSG00000173320', 'Q9P2F5'),
(3228, 'STRAP', 'Enables RNA binding activity. Involved in maintenance of gastrointestinal epithelium and spliceosomal snRNP assembly. Acts upstream of with a positive effect on negative regulation of transforming growth factor beta receptor signaling pathway. Located in cytosol. Part of SMN complex. Implicated in adenocarcinoma; colorectal carcinoma; large cell carcinoma; lung carcinoma; and squamous cell neoplasm. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 5392381, 5630700, 39.83, 36, 'ENSG00000023734', 'Q9Y3F4'),
(3229, 'STRIT1', 'Predicted to enable enzyme activator activity. Predicted to be involved in regulation of calcium ion transport and regulation of striated muscle contraction. Predicted to act upstream of or within positive regulation of ATPase-coupled calcium transmembrane transporter activity and positive regulation of calcium ion import. Predicted to be located in sarcoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 118133377, 118152888, 52.64, 6, 'ENSG00000240045', 'P0DN84'),
(3230, 'STX16-NPEPL1', 'This locus represents naturally occurring read-through transcription between the neighboring syntaxin 16 (STX16) and aminopeptidase-like 1 (NPEPL1) genes on chromosome 20. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is thus unlikely to produce a protein product. [provided by RefSeq, Mar 2011]', '20', 'protein_coding', 33924227, 34462774, 41.68, 11, 'ENSG00000254995', 'H3BU86'),
(3231, 'STX1A', 'This gene encodes a member of the syntaxin superfamily. Syntaxins are nervous system-specific proteins implicated in the docking of synaptic vesicles with the presynaptic plasma membrane. Syntaxins possess a single C-terminal transmembrane domain, a SNARE [Soluble NSF (N-ethylmaleimide-sensitive fusion protein)-Attachment protein REceptor] domain (known as H3), and an N-terminal regulatory domain (Habc). Syntaxins bind synaptotagmin in a calcium-dependent fashion and interact with voltage dependent calcium and potassium channels via the C-terminal H3 domain. This gene product is a key molecule in ion channel regulation and synaptic exocytosis. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Sep 2009]', '7', 'protein_coding', 51165435, 51171744, 52.09, 9, 'ENSG00000106089', 'Q16623'),
(3232, 'STXBP4', 'Enables syntaxin binding activity. Involved in several processes, including positive regulation of cell cycle G1/S phase transition; positive regulation of keratinocyte proliferation; and protein stabilization. Located in extracellular exosome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 20965108, 20981360, 59.24, 19, 'ENSG00000166263', 'Q6ZWJ1'),
(3233, 'SUDS3', 'SDS3 is a subunit of the histone deacetylase (see HDAC1; MIM 601241)-dependent SIN3A (MIM 607776) corepressor complex (Fleischer et al., 2003 [PubMed 12724404]).[supplied by OMIM, Mar 2008]', '12', 'protein_coding', 118981541, 119163051, 44.14, 7, 'ENSG00000111707', 'Q9H7L9'),
(3234, 'SUGP2', 'This gene encodes a member of the arginine/serine-rich family of splicing factors. The encoded protein functions in mRNA processing. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2009]', '19', 'protein_coding', 81081945, 81096395, 43.85, 10, 'ENSG00000064607', 'Q8IX01'),
(3235, 'SULF2', 'Heparan sulfate proteoglycans (HSPGs) act as coreceptors for numerous heparin-binding growth factors and cytokines and are involved in cell signaling. Heparan sulfate 6-O-endosulfatases, such as SULF2, selectively remove 6-O-sulfate groups from heparan sulfate. This activity modulates the effects of heparan sulfate by altering binding sites for signaling molecules (Dai et al., 2005 [PubMed 16192265]).[supplied by OMIM, Mar 2008]', '20', 'protein_coding', 100154420, 100158723, 60.48, 10, 'ENSG00000196562', 'Q8IWU5'),
(3236, 'SULT1B1', 'Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. However, the total genomic length of this gene is greater than that of other SULT1 genes. [provided by RefSeq, Jul 2008]', '4', 'TR_C_gene', 22547506, 22552156, 50.20, 1, 'ENSG00000173597', 'O43704'),
(3237, 'SULT1D1P', 'Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. This gene has been inactivated by mutation and is nonfunctional in humans. [provided by RefSeq, Oct 2008]', '4', 'miRNA', 49854591, 49854651, 70.49, 1, 'ENSG00000236959', ''),
(3238, 'SULT2B1', 'Sulfotransferase enzymes catalyze the sulfate conjugation of many hormones, neurotransmitters, drugs, and xenobiotic compounds. These cytosolic enzymes are different in their tissue distributions and substrate specificities. The gene structure (number and length of exons) is similar among family members. This gene sulfates dehydroepiandrosterone but not 4-nitrophenol, a typical substrate for the phenol and estrogen sulfotransferase subfamilies. Two alternatively spliced variants that encode different isoforms have been described. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 39156894, 39167484, 52.16, 5, 'ENSG00000088002', 'O00204'),
(3239, 'SUMO3', 'This gene encodes a protein that is a member of the SUMO (small ubiquitin-like modifier) protein family. It functions in a manner similar to ubiquitin in that it is bound to target proteins as part of a post-translational modification system. However, unlike ubiquitin which targets proteins for degradation, this protein is involved in a variety of cellular processes, such as nuclear transport, transcriptional regulation, apoptosis, and protein stability. It is not active until the last two amino acids of the carboxy-terminus have been cleaved off. Numerous pseudogenes have been reported for this gene. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 8169944, 8171267, 57.18, 1, 'ENSG00000184900', 'P55854'),
(3240, 'SUN2', 'SUN1 (MIM 607723) and SUN2 are inner nuclear membrane (INM) proteins that play a major role in nuclear-cytoplasmic connection by formation of a \'bridge\' across the nuclear envelope, known as the LINC complex, via interaction with the conserved luminal KASH domain of nesprins (e.g., SYNE1; MIM 608441) located in the outer nuclear membrane (ONM). The LINC complex provides a direct connection between the nuclear lamina and the cytoskeleton, which contributes to nuclear positioning and cellular rigidity (summary by Haque et al., 2010 [PubMed 19933576]).[supplied by OMIM, Nov 2010]', '22', 'miRNA', 160485352, 160485450, 45.45, 1, 'ENSG00000100242', 'Q9UH99'),
(3241, 'SUPV3L1', 'Enables several functions, including double-stranded RNA binding activity; helicase activity; and protein homodimerization activity. Involved in several processes, including mitochondrial RNA 3\'-end processing; mitochondrial RNA surveillance; and positive regulation of mitochondrial RNA catabolic process. Located in mitochondrial nucleoid and nucleus. Part of mitochondrial degradosome. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 35717872, 35729483, 47.71, 1, 'ENSG00000156502', 'Q8IYB8'),
(3242, 'SUSD1', 'Predicted to enable calcium ion binding activity. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'unprocessed_pseudogene', 204478, 205397, 39.67, 1, 'ENSG00000106868', 'Q6UWL2'),
(3243, 'SUSD2', 'Involved in negative regulation of cell cycle G1/S phase transition and negative regulation of cell division. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'miRNA', 59595077, 59595158, 48.78, 1, 'ENSG00000099994', 'Q9UGT4'),
(3244, 'SUSD6', 'Involved in DNA damage response. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 43726164, 43727002, 48.63, 2, 'ENSG00000100647', 'Q92537'),
(3245, 'SUV39H1', 'This gene encodes an evolutionarily-conserved protein containing an N-terminal chromodomain and a C-terminal SET domain. The encoded protein is a histone methyltransferase that trimethylates lysine 9 of histone H3, which results in transcriptional gene silencing. Loss of function of this gene disrupts heterochromatin formation and may cause chromosome instability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', 'X', 'protein_coding', 29490, 60215, 47.77, 2, 'ENSG00000101945', 'O43463'),
(3246, 'SVOP', 'Predicted to enable transmembrane transporter activity. Predicted to be involved in transmembrane transport and xenobiotic transport. Predicted to be located in synaptic vesicle. Predicted to be active in synaptic vesicle membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 38748460, 38772520, 44.81, 1, 'ENSG00000166111', 'Q8N4V2'),
(3247, 'SYCE1', 'This gene encodes a member of the synaptonemal complex, which links homologous chromosomes during prophase I of meiosis. The tripartite structure of the complex is highly conserved amongst metazoans. It consists of two lateral elements and a central region formed by transverse elements and a central element. The protein encoded by this gene localizes to the central element and is required for initiation and elongation of the synapsis. Allelic variants of this gene have been associated with premature ovarian failure and spermatogenic failure. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2016]', '10', 'protein_coding', 25833066, 25882286, 38.42, 10, 'ENSG00000171772', 'Q8N0S2'),
(3248, 'SYCP3', 'This gene encodes an essential structural component of the synaptonemal complex. This complex is involved in synapsis, recombination and segregation of meiotic chromosomes. Mutations in this gene are associated with azoospermia in males and susceptibility to pregnancy loss in females. Alternate splicing results in multiple transcript variants that encode the same protein. [provided by RefSeq, May 2010]', '12', 'protein_coding', 72749277, 72861914, 39.56, 8, 'ENSG00000139351', 'Q8IZU3'),
(3249, 'SYN3', 'This gene is a member of the synapsin gene family. Synapsins encode neuronal phosphoproteins which associate with the cytoplasmic surface of synaptic vesicles. Family members are characterized by common protein domains, and they are implicated in synaptogenesis and the modulation of neurotransmitter release, suggesting a potential role in several neuropsychiatric diseases. The protein encoded by this gene shares the synapsin family domain model, with domains A, C, and E exhibiting the highest degree of conservation. The protein contains a unique domain J, located between domains C and E. Based on this gene\'s localization to 22q12.3, a possible schizophrenia susceptibility locus, and the established neurobiological roles of the synapsins, this family member may represent a candidate gene for schizophrenia. The TIMP3 gene is located within an intron of this gene and is transcribed in the opposite direction. Alternative splicing of this gene results in multiple splice variants that encode different isoforms. [provided by RefSeq, Oct 2008]', '22', 'unprocessed_pseudogene', 55979398, 55980103, 47.17, 1, 'ENSG00000185666', 'O14994'),
(3250, 'SYNE2', 'The protein encoded by this gene is a nuclear outer membrane protein that binds cytoplasmic F-actin. This binding tethers the nucleus to the cytoskeleton and aids in the maintenance of the structural integrity of the nucleus. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]', '14', 'protein_coding', 66603874, 66613892, 58.84, 15, 'ENSG00000054654', 'Q8WXH0');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3251, 'SYNJ2', 'The gene is a member of the inositol-polyphosphate 5-phosphatase family. The encoded protein interacts with the ras-related C3 botulinum toxin substrate 1, which causes translocation of the encoded protein to the plasma membrane where it inhibits clathrin-mediated endocytosis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2010]', '6', 'protein_coding', 93934166, 93976739, 52.44, 10, 'ENSG00000078269', 'O15056'),
(3252, 'SYNJ2BP-COX16', 'This locus represents naturally occurring read-through transcription between the neighboring SYNJ2BP (synaptojanin 2 binding protein) and COX16 (COX16 cytochrome c oxidase assembly homolog (S. cerevisiae)) genes on chromosome 14. The read-through transcript produces a fusion protein that shares sequence identity with each individual gene product. Alternate splicing results in multiple transcript variants that encode different isoforms. [provided by RefSeq, Feb 2011]', '14', 'protein_coding', 137507, 148338, 62.78, 3, 'ENSG00000258644', 'A0A087WYV9'),
(3253, 'SYNPO2L', 'Predicted to enable actin binding activity. Predicted to be involved in several processes, including positive regulation of Rho protein signal transduction; positive regulation of stress fiber assembly; and sarcomere organization. Predicted to be located in cytoplasm and cytoskeleton. Predicted to be active in Z disc; actin cytoskeleton; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 64244479, 64246943, 65.52, 3, 'ENSG00000166317', 'Q9H987'),
(3254, 'SYNRG', 'This gene encodes a protein that interacts with the gamma subunit of AP1 clathrin-adaptor complex. The AP1 complex is located at the trans-Golgi network and associates specific proteins with clathrin-coated vesicles. This encoded protein may act to connect the AP1 complex to other proteins. Alternatively spliced transcript variants that encode different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]', '17', 'lncRNA', 51452364, 51554678, 40.16, 42, 'ENSG00000274047', 'Q9UMZ2'),
(3255, 'SYNRG', 'This gene encodes a protein that interacts with the gamma subunit of AP1 clathrin-adaptor complex. The AP1 complex is located at the trans-Golgi network and associates specific proteins with clathrin-coated vesicles. This encoded protein may act to connect the AP1 complex to other proteins. Alternatively spliced transcript variants that encode different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]', '17', 'lncRNA', 51452364, 51554678, 40.16, 42, 'ENSG00000275066', 'Q9UMZ2'),
(3256, 'SYTL5', 'The protein encoded by this gene belongs to the synaptotagmin-like (Slp) protein family, which contains a unique homology domain at the N-terminus, referred to as the Slp homology domain (SHD). The SHD functions as a binding site for Rab27A, which plays a role in protein transport. Expression of this gene is restricted to placenta and liver, suggesting that it might be involved in Rab27A-dependent membrane trafficking in specific tissues. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 44849585, 44873903, 51.00, 7, 'ENSG00000147041', 'Q8TDW5'),
(3257, 'TAAR2', 'Predicted to enable trace-amine receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Predicted to be located in membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 69180234, 69209099, 41.80, 7, 'ENSG00000146378', 'Q9P1P5'),
(3258, 'TAAR4P', 'Predicted to enable 2-phenylethylamine receptor activity and trace-amine receptor activity. Predicted to be involved in behavioral fear response; chemosensory behavior; and sensory perception of chemical stimulus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 262229, 263158, 38.60, 1, 'ENSG00000237449', ''),
(3259, 'TAB1', 'The protein encoded by this gene was identified as a regulator of the MAP kinase kinase kinase MAP3K7/TAK1, which is known to mediate various intracellular signaling pathways, such as those induced by TGF beta, interleukin 1, and WNT-1. This protein interacts and thus activates TAK1 kinase. It has been shown that the C-terminal portion of this protein is sufficient for binding and activation of TAK1, while a portion of the N-terminus acts as a dominant-negative inhibitor of TGF beta, suggesting that this protein may function as a mediator between TGF beta receptors and TAK1. This protein can also interact with and activate the mitogen-activated protein kinase 14 (MAPK14/p38alpha), and thus represents an alternative activation pathway, in addition to the MAPKK pathways, which contributes to the biological responses of MAPK14 to various stimuli. Alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008]', '22', 'rRNA', 221127, 221245, 60.50, 1, 'ENSG00000100324', 'Q15750'),
(3260, 'TAF4', 'Initiation of transcription by RNA polymerase II requires the activities of more than 70 polypeptides. The protein that coordinates these activities is transcription factor IID (TFIID), which binds to the core promoter to position the polymerase properly, serves as the scaffold for assembly of the remainder of the transcription complex, and acts as a channel for regulatory signals. TFIID is composed of the TATA-binding protein (TBP) and a group of evolutionarily conserved proteins known as TBP-associated factors or TAFs. TAFs may participate in basal transcription, serve as coactivators, function in promoter recognition or modify general transcription factors (GTFs) to facilitate complex assembly and transcription initiation. This gene encodes one of the larger subunits of TFIID that has been shown to potentiate transcriptional activation by retinoic acid, thyroid hormone and vitamin D3 receptors. In addition, this subunit interacts with the transcription factor CREB, which has a glutamine-rich activation domain, and binds to other proteins containing glutamine-rich regions. Aberrant binding to this subunit by proteins with expanded polyglutamine regions has been suggested as one of the pathogenetic mechanisms underlying a group of neurodegenerative disorders referred to as polyglutamine diseases. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 94418389, 94518666, 37.07, 7, 'ENSG00000130699', 'O00268'),
(3261, 'TAF4', 'Initiation of transcription by RNA polymerase II requires the activities of more than 70 polypeptides. The protein that coordinates these activities is transcription factor IID (TFIID), which binds to the core promoter to position the polymerase properly, serves as the scaffold for assembly of the remainder of the transcription complex, and acts as a channel for regulatory signals. TFIID is composed of the TATA-binding protein (TBP) and a group of evolutionarily conserved proteins known as TBP-associated factors or TAFs. TAFs may participate in basal transcription, serve as coactivators, function in promoter recognition or modify general transcription factors (GTFs) to facilitate complex assembly and transcription initiation. This gene encodes one of the larger subunits of TFIID that has been shown to potentiate transcriptional activation by retinoic acid, thyroid hormone and vitamin D3 receptors. In addition, this subunit interacts with the transcription factor CREB, which has a glutamine-rich activation domain, and binds to other proteins containing glutamine-rich regions. Aberrant binding to this subunit by proteins with expanded polyglutamine regions has been suggested as one of the pathogenetic mechanisms underlying a group of neurodegenerative disorders referred to as polyglutamine diseases. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 94418389, 94518666, 37.07, 7, 'ENSG00000280529', 'O00268'),
(3262, 'TAF4B', 'TATA binding protein (TBP) and TBP-associated factors (TAFs) participate in the formation of the TFIID protein complex, which is involved in initiation of transcription of genes by RNA polymerase II. This gene encodes a cell type-specific TAF that may be responsible for mediating transcription by a subset of activators in B cells. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jun 2014]', '18', 'TR_J_pseudogene', 22487183, 22487245, 42.86, 1, 'ENSG00000141384', 'Q92750'),
(3263, 'TAF9B', 'Initiation of transcription by RNA polymerase II requires the activities of more than 70 polypeptides. The protein that coordinates these activities is transcription factor IID (TFIID), which binds to the core promoter to position the polymerase properly, serves as the scaffold for assembly of the remainder of the transcription complex, and acts as a channel for regulatory signals. TFIID is composed of the TATA-binding protein (TBP) and a group of evolutionarily conserved proteins known as TBP-associated factors or TAFs. TAFs may participate in basal transcription, serve as coactivators, function in promoter recognition or modify general transcription factors (GTFs) to facilitate complex assembly and transcription initiation. This gene encodes a protein that is similar to one of the small subunits of TFIID, TBP-associated factor 9, and is also a subunit of TFIID. TAF9 and TAF9b share some functions but also have distinct roles in the transcriptional regulatory process. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 108907741, 109021068, 45.17, 5, 'ENSG00000187325', 'Q9HBM6'),
(3264, 'TANC1', 'Predicted to be involved in regulation of postsynapse organization. Predicted to act upstream of or within dendritic spine maintenance; myoblast fusion; and visual learning. Predicted to be located in several cellular components, including axon terminus; neuronal cell body; and postsynaptic density. Predicted to be active in glutamatergic synapse and postsynaptic density, intracellular component. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 133097720, 133149334, 58.92, 15, 'ENSG00000115183', 'Q9C0D5'),
(3265, 'TANGO6', 'Predicted to be involved in protein secretion. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 43623, 51348, 60.46, 6, 'ENSG00000103047', 'Q9C0B7'),
(3266, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000275384', 'B6A8C7'),
(3267, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000276604', 'B6A8C7'),
(3268, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000277178', 'B6A8C7'),
(3269, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000276145', 'B6A8C7'),
(3270, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000275123', 'B6A8C7'),
(3271, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000274889', 'B6A8C7'),
(3272, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000273875', 'B6A8C7'),
(3273, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000275806', 'B6A8C7'),
(3274, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000276355', 'B6A8C7'),
(3275, 'TARM1', 'Enables immunoglobulin receptor binding activity. Involved in negative regulation of CD4-positive, alpha-beta T cell activation. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 141356470, 141367286, 61.69, 1, 'ENSG00000248385', 'B6A8C7'),
(3276, 'TAS2R39', 'The protein encoded by this gene is a bitter taste receptor that detects green tea catechins, soy isoflavones, and theaflavins. The encoded protein is gustducin-linked and may activate alpha gustducin. This gene is intronless. [provided by RefSeq, Dec 2015]', '7', 'protein_coding', 88879734, 88975153, 39.34, 6, 'ENSG00000236398', 'P59534'),
(3277, 'TAS2R42', 'Predicted to enable G protein-coupled receptor activity and bitter taste receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway and detection of chemical stimulus involved in sensory perception of bitter taste. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 69244311, 69327244, 34.61, 14, 'ENSG00000273505', 'Q7RTR8'),
(3278, 'TAS2R42', 'Predicted to enable G protein-coupled receptor activity and bitter taste receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway and detection of chemical stimulus involved in sensory perception of bitter taste. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 69244311, 69327244, 34.61, 14, 'ENSG00000186136', 'Q7RTR8'),
(3279, 'TAS2R45', 'Predicted to enable G protein-coupled receptor activity and bitter taste receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway and detection of chemical stimulus involved in sensory perception of bitter taste. Located in membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 69348381, 69354234, 36.54, 3, 'ENSG00000261936', 'P59539'),
(3280, 'TAS2R46', 'TAS2R46 belongs to the large TAS2R receptor family. TAS2Rs are expressed on the surface of taste receptor cells and mediate the perception of bitterness through a G protein-coupled second messenger pathway (Conte et al., 2002 [PubMed 12584440]). For further information on TAS2Rs, see MIM 604791.[supplied by OMIM, Sep 2009]', '12', 'protein_coding', 30285117, 30289622, 61.30, 14, 'ENSG00000262525', 'P59540'),
(3281, 'TAS2R46', 'TAS2R46 belongs to the large TAS2R receptor family. TAS2Rs are expressed on the surface of taste receptor cells and mediate the perception of bitterness through a G protein-coupled second messenger pathway (Conte et al., 2002 [PubMed 12584440]). For further information on TAS2Rs, see MIM 604791.[supplied by OMIM, Sep 2009]', '12', 'protein_coding', 30285117, 30289622, 61.30, 14, 'ENSG00000278111', 'P59540'),
(3282, 'TAS2R46', 'TAS2R46 belongs to the large TAS2R receptor family. TAS2Rs are expressed on the surface of taste receptor cells and mediate the perception of bitterness through a G protein-coupled second messenger pathway (Conte et al., 2002 [PubMed 12584440]). For further information on TAS2Rs, see MIM 604791.[supplied by OMIM, Sep 2009]', '12', 'protein_coding', 30285117, 30289622, 61.30, 14, 'ENSG00000226761', 'P59540'),
(3283, 'TAS2R5', 'This gene encodes a bitter taste receptor; bitter taste receptors are members of the G protein-coupled receptor superfamily and are specifically expressed by taste receptor cells of the tongue and palate epithelia. Each of these apparently intronless taste receptor genes encodes a 7-transmembrane receptor protein, functioning as a bitter taste receptor. This gene is clustered with another 3 candidate taste receptor genes on chromosome 7 and is genetically linked to loci that influence bitter perception. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 69451465, 69507666, 51.45, 8, 'ENSG00000127366', 'Q9NYW4'),
(3284, 'TAS2R60', 'This gene encodes a member of the bitter taste receptor family which belong to the G protein-coupled receptor superfamily and are predominantly expressed in taste receptor cells of the tongue and palate epithelia. This intronless taste receptor gene encodes a seven-transmembrane receptor protein, functioning as a bitter taste receptor. This gene is clustered together with eight other taste receptor genes on chromosome 7. [provided by RefSeq, Jul 2017]', '7', 'TR_J_gene', 22525263, 22525322, 35.00, 1, 'ENSG00000285341', 'P59551'),
(3285, 'TAS2R60', 'This gene encodes a member of the bitter taste receptor family which belong to the G protein-coupled receptor superfamily and are predominantly expressed in taste receptor cells of the tongue and palate epithelia. This intronless taste receptor gene encodes a seven-transmembrane receptor protein, functioning as a bitter taste receptor. This gene is clustered together with eight other taste receptor genes on chromosome 7. [provided by RefSeq, Jul 2017]', '7', 'TR_J_gene', 22525263, 22525322, 35.00, 1, 'ENSG00000185899', 'P59551'),
(3286, 'TASL', 'Involved in positive regulation of toll-like receptor 7 signaling pathway; positive regulation of toll-like receptor 8 signaling pathway; and regulation of lysosomal lumen pH. Located in cytosol; endolysosome membrane; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', 'X', 'unprocessed_pseudogene', 5314772, 5315684, 43.70, 1, 'ENSG00000120280', 'Q9HAI6'),
(3287, 'TBC1D10A', 'Enables PDZ domain binding activity. Involved in positive regulation of proteolysis and retrograde transport, endosome to Golgi. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 44228729, 44305046, 39.47, 7, 'ENSG00000099992', 'Q9BXI6'),
(3288, 'TBC1D10B', 'Small G proteins of the RAB family (see MIM 179508) function in intracellular vesicle trafficking by switching from the GTP-bound state to the GDP-bound state with the assistance of guanine nucleotide exchange factors (GEFs; see MIM 609700) and GTPase-activating proteins (GAPs). TBC1D10B functions as a GAP for several proteins of the Rab family (Ishibashi et al., 2009 [PubMed 19077034]).[supplied by OMIM, Nov 2010]', '16', 'protein_coding', 119759922, 119984899, 36.97, 12, 'ENSG00000169221', 'Q4KMP7'),
(3289, 'TBC1D12', 'Predicted to enable GTPase activator activity. Predicted to be involved in regulation of autophagosome assembly. Predicted to be located in endosome. Predicted to be active in autophagosome and recycling endosome. [provided by Alliance of Genome Resources, May 2025]', '10', 'snoRNA', 103337849, 103337974, 41.27, 1, 'ENSG00000108239', 'O60347'),
(3290, 'TBC1D13', 'Predicted to enable GTPase activator activity. Predicted to be involved in intracellular protein transport. Predicted to be located in cytosol and membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 50879080, 50954257, 38.58, 19, 'ENSG00000107021', 'Q9NVG8'),
(3291, 'TBC1D22A', 'Enables 14-3-3 protein binding activity and protein homodimerization activity. Predicted to be active in Golgi apparatus. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 60144119, 60169856, 39.65, 11, 'ENSG00000054611', 'Q8WUA7'),
(3292, 'TBC1D23', 'Involved in brain development; retrograde transport, endosome to Golgi; and vesicle tethering to Golgi. Located in WASH complex; cytoplasmic vesicle; and trans-Golgi network. Implicated in pontocerebellar hypoplasia type 11. [provided by Alliance of Genome Resources, May 2025]', '3', 'miRNA', 101023794, 101023856, 38.10, 1, 'ENSG00000036054', 'Q9NUY8'),
(3293, 'TBC1D3B', 'This gene encodes a protein that is similar to TBC1 domain family, member 3. This protein contains a TBC (Tre-2, Bub2p, and Cdc16p) domain, which is found in proteins involved in RAB GTPase signaling and vesicle trafficking. There are multiple copies of this gene located within a cluster of chemokine genes on chromosome 17q. [provided by RefSeq, Apr 2009]', '17', 'protein_coding', 21953288, 21983660, 40.09, 13, 'ENSG00000282750', 'A6NDS4'),
(3294, 'TBC1D3B', 'This gene encodes a protein that is similar to TBC1 domain family, member 3. This protein contains a TBC (Tre-2, Bub2p, and Cdc16p) domain, which is found in proteins involved in RAB GTPase signaling and vesicle trafficking. There are multiple copies of this gene located within a cluster of chemokine genes on chromosome 17q. [provided by RefSeq, Apr 2009]', '17', 'protein_coding', 21953288, 21983660, 40.09, 13, 'ENSG00000278632', 'A6NDS4'),
(3295, 'TBC1D3B', 'This gene encodes a protein that is similar to TBC1 domain family, member 3. This protein contains a TBC (Tre-2, Bub2p, and Cdc16p) domain, which is found in proteins involved in RAB GTPase signaling and vesicle trafficking. There are multiple copies of this gene located within a cluster of chemokine genes on chromosome 17q. [provided by RefSeq, Apr 2009]', '17', 'protein_coding', 21953288, 21983660, 40.09, 13, 'ENSG00000274939', 'A6NDS4'),
(3296, 'TBC1D3B', 'This gene encodes a protein that is similar to TBC1 domain family, member 3. This protein contains a TBC (Tre-2, Bub2p, and Cdc16p) domain, which is found in proteins involved in RAB GTPase signaling and vesicle trafficking. There are multiple copies of this gene located within a cluster of chemokine genes on chromosome 17q. [provided by RefSeq, Apr 2009]', '17', 'protein_coding', 21953288, 21983660, 40.09, 13, 'ENSG00000274808', 'A6NDS4'),
(3297, 'TBC1D3F', 'Predicted to enable GTPase activator activity. Predicted to be located in early endosome membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 9556168, 9556316, 36.91, 1, 'ENSG00000282087', 'A6NER0'),
(3298, 'TBC1D3F', 'Predicted to enable GTPase activator activity. Predicted to be located in early endosome membrane. [provided by Alliance of Genome Resources, May 2025]', '17', 'miRNA', 9556168, 9556316, 36.91, 1, 'ENSG00000275954', 'A6NER0'),
(3299, 'TBCK', 'This gene encodes a protein that contains a protein kinase domain, a Rhodanase-like domain and the Tre-2/Bub2/Cdc16 (TBC) domain. The encoded protein is thought to play a role in actin organization, cell growth and cell proliferation by regulating the mammalian target of the rapamycin (mTOR) signaling pathway. This protein may also be involved in the transcriptional regulation of the components of the mTOR complex. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2014]', '4', 'protein_coding', 40161387, 40240921, 39.69, 1, 'ENSG00000145348', 'Q8TEA7'),
(3300, 'TBL1X', 'The protein encoded by this gene has sequence similarity with members of the WD40 repeat-containing protein family. The WD40 group is a large family of proteins, which appear to have a regulatory function. It is believed that the WD40 repeats mediate protein-protein interactions and members of the family are involved in signal transduction, RNA processing, gene regulation, vesicular trafficking, cytoskeletal assembly and may play a role in the control of cytotypic differentiation. This encoded protein is found as a subunit in corepressor SMRT (silencing mediator for retinoid and thyroid receptors) complex along with histone deacetylase 3 protein. This gene is located adjacent to the ocular albinism gene and it is thought to be involved in the pathogenesis of the ocular albinism with late-onset sensorineural deafness phenotype. Four transcript variants encoding two different isoforms have been found for this gene. This gene is highly similar to the Y chromosome TBL1Y gene. [provided by RefSeq, Nov 2008]', 'X', 'protein_coding', 151156649, 151159749, 51.53, 1, 'ENSG00000101849', 'O60907'),
(3301, 'TBL1Y', 'The protein encoded by this gene has sequence similarity with members of the WD40 repeat-containing protein family. The WD40 group is a large family of proteins, which appear to have a regulatory function. It is believed that the WD40 repeats mediate protein-protein interactions and members of the family are involved in signal transduction, RNA processing, gene regulation, vesicular trafficking, cytoskeletal assembly and may play a role in the control of cytotypic differentiation. This gene is highly similar to TBL1X gene in nucleotide sequence and protein sequence, but the TBL1X gene is located on chromosome X and this gene is on chromosome Y. This gene has three alternatively spliced transcript variants encoding the same protein. [provided by RefSeq, Jul 2008]', 'Y', 'protein_coding', 25622117, 25624902, 41.35, 2, 'ENSG00000092377', 'Q9BQ87'),
(3302, 'TBR1', 'This gene is a member of a conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of numerous developmental processes. In mouse, the ortholog of this gene is expressed in the cerebral cortex, hippocampus, amygdala and olfactory bulb and is thought to play an important role in neuronal migration and axonal projection. In mouse, the C-terminal region of this protein was found to be necessary and sufficient for association with the guanylate kinase domain of calcium/calmodulin-dependent serine protein kinase. [provided by RefSeq, Dec 2015]', '2', 'snoRNA', 45104349, 45104482, 52.24, 1, 'ENSG00000136535', 'Q16650'),
(3303, 'TBX1', 'This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This gene product shares 98% amino acid sequence identity with the mouse ortholog. DiGeorge syndrome (DGS)/velocardiofacial syndrome (VCFS), a common congenital disorder characterized by neural-crest-related developmental defects, has been associated with deletions of chromosome 22q11.2, where this gene has been mapped. Studies using mouse models of DiGeorge syndrome suggest a major role for this gene in the molecular etiology of DGS/VCFS. Several alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 952323, 1003890, 54.70, 6, 'ENSG00000184058', 'O43435'),
(3304, 'TBX22', 'This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. Mutations in this gene have been associated with the inherited X-linked disorder, Cleft palate with ankyloglossia, and it is believed to play a major role in human palatogenesis. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', 'X', 'miRNA', 44970082, 44970166, 52.94, 1, 'ENSG00000122145', 'Q9Y458'),
(3305, 'TBX22', 'This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. Mutations in this gene have been associated with the inherited X-linked disorder, Cleft palate with ankyloglossia, and it is believed to play a major role in human palatogenesis. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', 'X', 'miRNA', 44970082, 44970166, 52.94, 1, 'ENSG00000277800', 'Q9Y458'),
(3306, 'TCAF1', 'Enables transmembrane transporter binding activity. Involved in negative regulation of cell migration; positive regulation of anion channel activity; and positive regulation of protein targeting to membrane. Located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 46888, 74163, 43.20, 7, 'ENSG00000198420', 'Q9Y4C2'),
(3307, 'TCEA2', 'The protein encoded by this gene is found in the nucleus, where it functions as an SII class transcription elongation factor. Elongation factors in this class are responsible for releasing RNA polymerase II ternary complexes from transcriptional arrest at template-encoded arresting sites. The encoded protein has been shown to interact with general transcription factor IIB, a basal transcription factor. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 70907405, 71115382, 36.87, 15, 'ENSG00000171703', 'Q15560'),
(3308, 'TCEA3', 'Predicted to enable DNA binding activity; translation elongation factor activity; and zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 62811996, 62819167, 57.81, 7, 'ENSG00000282941', 'O75764'),
(3309, 'TCEA3', 'Predicted to enable DNA binding activity; translation elongation factor activity; and zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 62811996, 62819167, 57.81, 7, 'ENSG00000204219', 'O75764'),
(3310, 'TCEAL2', 'This gene encodes a member of the transcription elongation factor A (SII)-like (TCEAL) gene family. Members of this family contain TFA domains and may function as nuclear phosphoproteins that modulate transcription in a promoter context-dependent manner. Multiple family members are located on the X chromosome. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 96593315, 96720514, 44.36, 6, 'ENSG00000291823', 'Q9H3H9'),
(3311, 'TCEAL2', 'This gene encodes a member of the transcription elongation factor A (SII)-like (TCEAL) gene family. Members of this family contain TFA domains and may function as nuclear phosphoproteins that modulate transcription in a promoter context-dependent manner. Multiple family members are located on the X chromosome. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 96593315, 96720514, 44.36, 6, 'ENSG00000184905', 'Q9H3H9'),
(3312, 'TCEAL4', 'This gene encodes a member of the transcription elongation factor A (SII)-like (TCEAL) gene family. This family is comprised of nuclear phosphoproteins that modulate transcription in a promoter context-dependent manner. Multiple family members are located on the X chromosome. Alternatively splicing results in multiple transcript variants. There is a pseudogene for this gene on chromosome 13. [provided by RefSeq, Apr 2015]', 'X', 'protein_coding', 41969443, 41998221, 52.30, 7, 'ENSG00000133142', 'Q96EI5'),
(3313, 'TCEANC2', 'Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 178194481, 178402893, 39.34, 9, 'ENSG00000116205', 'Q96MN5'),
(3314, 'TCP1', 'The protein encoded by this gene is a molecular chaperone that is a member of the chaperonin containing TCP1 complex (CCT), also known as the TCP1 ring complex (TRiC). This complex consists of two identical stacked rings, each containing eight different proteins. Unfolded polypeptides enter the central cavity of the complex and are folded in an ATP-dependent manner. The complex folds various proteins, including actin and tubulin. Alternate transcriptional splice variants of this gene, encoding different isoforms, have been characterized. In addition, three pseudogenes that appear to be derived from this gene have been found. [provided by RefSeq, Jun 2010]', '6', 'protein_coding', 66330241, 66336840, 62.23, 8, 'ENSG00000120438', 'P17987'),
(3315, 'TDH', 'Enables L-serine ammonia-lyase activity; identical protein binding activity; and threonine deaminase activity. Predicted to be involved in L-serine catabolic process. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '12', 'miRNA', 121370972, 121371053, 30.49, 1, 'ENSG00000284856', 'Q8IZJ6'),
(3316, 'TDH', 'Enables L-serine ammonia-lyase activity; identical protein binding activity; and threonine deaminase activity. Predicted to be involved in L-serine catabolic process. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '12', 'miRNA', 121370972, 121371053, 30.49, 1, 'ENSG00000154316', 'Q8IZJ6'),
(3317, 'TDH', 'Enables L-serine ammonia-lyase activity; identical protein binding activity; and threonine deaminase activity. Predicted to be involved in L-serine catabolic process. Predicted to be located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '12', 'miRNA', 121370972, 121371053, 30.49, 1, 'ENSG00000293451', 'Q8IZJ6'),
(3318, 'TDO2', 'This gene encodes a heme enzyme that plays a critical role in tryptophan metabolism by catalyzing the first and rate-limiting step of the kynurenine pathway. Increased activity of the encoded protein and subsequent kynurenine production may also play a role in cancer through the suppression of antitumor immune responses, and single nucleotide polymorphisms in this gene may be associated with autism. [provided by RefSeq, Feb 2012]', '4', 'protein_coding', 218220, 236188, 44.83, 2, 'ENSG00000151790', 'P48775'),
(3319, 'TDO2', 'This gene encodes a heme enzyme that plays a critical role in tryptophan metabolism by catalyzing the first and rate-limiting step of the kynurenine pathway. Increased activity of the encoded protein and subsequent kynurenine production may also play a role in cancer through the suppression of antitumor immune responses, and single nucleotide polymorphisms in this gene may be associated with autism. [provided by RefSeq, Feb 2012]', '4', 'protein_coding', 218220, 236188, 44.83, 2, 'ENSG00000262635', 'P48775'),
(3320, 'TDRD1', 'This gene encodes a protein containing a tudor domain that is thought to function in the suppression of transposable elements during spermatogenesis. The related protein in mouse forms a complex with piRNAs and Piwi proteins to promote methylation and silencing of target sequences. This gene was observed to be upregulated by ETS transcription factor ERG in prostate tumors. [provided by RefSeq, Sep 2018]', '10', 'protein_coding', 31870634, 31945000, 46.05, 11, 'ENSG00000095627', 'Q9BXT4'),
(3321, 'TDRD5', 'Predicted to be involved in P granule organization; spermatid development; and transposable element silencing by piRNA-mediated DNA methylation. Predicted to be located in chromatoid body and pi-body. Predicted to be active in synapse. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 44768, 91509, 48.32, 11, 'ENSG00000162782', 'Q8NAT2'),
(3322, 'TECRL', 'The protein encoded by this gene contains a ubiquitin-like domain in the N-terminal region, three transmembrane segments and a C-terminal 3-oxo-5-alpha steroid 4-dehydrogenase domain. The protein belongs to the steroid 5-alpha reductase family. Mutations in this gene result in ventricular tachycardia, catecholaminergic polymorphic, 3. [provided by RefSeq, Apr 2017]', '4', 'TR_J_gene', 22512852, 22512908, 42.11, 1, 'ENSG00000205678', 'Q5HYJ1'),
(3323, 'TEDC2', 'Predicted to be involved in positive regulation of smoothened signaling pathway. Predicted to be located in centriole and cilium. [provided by Alliance of Genome Resources, May 2025]', '16', 'transcribed_unprocessed_pseudogene', 20702007, 20704490, 69.85, 1, 'ENSG00000162062', 'Q7L2K0'),
(3324, 'TEKT5', 'Predicted to be involved in cilium assembly and flagellated sperm motility. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 56978129, 57006768, 45.28, 16, 'ENSG00000153060', 'Q96M29'),
(3325, 'TEN1', 'The protein encoded by this gene belongs to the tenascin family and teneurin subfamily. It is expressed in the neurons and may function as a cellular signal transducer. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 133061981, 133071861, 59.25, 1, 'ENSG00000257949', 'Q86WV5'),
(3326, 'TEN1-CDK3', 'This locus represents naturally occurring read-through transcription between the neighboring TEN1 telomerase capping complex subunit homolog (S. cerevisiae) and cyclin-dependent kinase 3 (CDK3) genes. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is therefore unlikely to produce a protein product. [provided by RefSeq, Oct 2011]', '17', 'protein_coding', 1946053, 2004535, 38.72, 7, 'ENSG00000261408', ''),
(3327, 'TENM3', 'This gene encodes a member of the teneurin transmembrane protein family. The encoded protein may be involved in the regulation of neuronal development including development of the visual pathway. Mutations in this gene have been associated with microphthalmia and developmental dysplasia of the hip. [provided by RefSeq, Jan 2023]', '4', 'protein_coding', 10390322, 10421950, 38.91, 1, 'ENSG00000218336', 'Q9P273'),
(3328, 'TENT4B', 'Enables guanylyltransferase activity; poly(A) RNA polymerase activity; and telomerase RNA binding activity. Involved in RNA metabolic process; carbohydrate homeostasis; and regulation of nucleobase-containing compound metabolic process. Located in cytosol; nucleolus; and plasma membrane. Part of TRAMP complex. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 24573268, 24694193, 35.88, 9, 'ENSG00000121274', 'Q8NDF8'),
(3329, 'TENT5A', 'Enables poly(A) RNA polymerase activity. Predicted to be involved in mRNA stabilization; positive regulation of bone mineralization; and positive regulation of osteoblast differentiation. Predicted to act upstream of or within response to bacterium. Predicted to be located in cytoplasm. Implicated in lung non-small cell carcinoma; osteoarthritis; and osteogenesis imperfecta type 18. [provided by Alliance of Genome Resources, May 2025]', '6', 'miRNA', 126484631, 126484713, 44.58, 1, 'ENSG00000112773', 'Q96IP4'),
(3330, 'TERF2IP', 'Enables G-rich strand telomeric DNA binding activity and phosphatase binding activity. Involved in several processes, including positive regulation of non-canonical NF-kappaB signal transduction; protection from non-homologous end joining at telomere; and regulation of primary metabolic process. Located in chromosome, telomeric region; cytoplasm; and nuclear body. Part of shelterin complex. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 23768226, 23784316, 57.83, 13, 'ENSG00000166848', 'Q9NYB0'),
(3331, 'TESMIN', 'Metallothionein proteins are highly conserved low-molecular-weight cysteine-rich proteins that are induced by and bind to heavy metal ions and have no enzymatic activity. They may play a central role in the regulation of cell growth and differentiation and are involved in spermatogenesis. This gene encodes a metallothionein-like protein which has been shown to be expressed differentially in mouse testis and ovary. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '11', 'protein_coding', 39391303, 39400641, 52.37, 6, 'ENSG00000132749', 'Q9Y4I5'),
(3332, 'TEX10', 'Located in mitochondrion and nucleoplasm. Part of MLL1 complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 22052430, 22091480, 38.10, 8, 'ENSG00000136891', 'Q9NXF1'),
(3333, 'TEX11', 'This gene is X-linked and is expressed in only male germ cells. Two alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', 'X', 'unprocessed_pseudogene', 124158421, 124159356, 51.28, 1, 'ENSG00000120498', 'Q8IYF3'),
(3334, 'TEX13C', 'Predicted to enable mRNA binding activity. [provided by Alliance of Genome Resources, May 2025]', 'X', 'miRNA', 72516820, 72516898, 54.43, 1, 'ENSG00000282815', 'A0A0J9YWL9'),
(3335, 'TEX13D', 'Predicted to enable mRNA binding activity. [provided by Alliance of Genome Resources, May 2025]', 'X', 'protein_coding', 70597348, 70602759, 56.54, 4, 'ENSG00000282419', 'A0A0J9YY54'),
(3336, 'TEX19', 'Predicted to enable piRNA binding activity. Involved in transposable element silencing. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'transcribed_unprocessed_pseudogene', 251876, 262088, 53.10, 1, 'ENSG00000182459', 'Q8NA77'),
(3337, 'TEX22', 'Predicted to be located in acrosomal vesicle. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 7080214, 7092540, 51.55, 16, 'ENSG00000226174', 'C9J3V5'),
(3338, 'TEX29', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '13', 'snoRNA', 49489245, 49489308, 35.94, 1, 'ENSG00000153495', 'Q8N6K0'),
(3339, 'TEX55', 'Located in sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 73631612, 73641474, 47.83, 1, 'ENSG00000163424', 'Q96M34'),
(3340, 'TFAM', 'This gene encodes a key mitochondrial transcription factor containing two high mobility group motifs. The encoded protein also functions in mitochondrial DNA replication and repair. Sequence polymorphisms in this gene are associated with Alzheimer\'s and Parkinson\'s diseases. There are pseudogenes for this gene on chromosomes 6, 7, and 11. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]', '10', 'protein_coding', 58358124, 58359069, 41.75, 1, 'ENSG00000108064', 'Q00059'),
(3341, 'TFF2', 'Members of the trefoil family are characterized by having at least one copy of the trefoil motif, a 40-amino acid domain that contains three conserved disulfides. They are stable secretory proteins expressed in gastrointestinal mucosa. Their functions are not defined, but they may protect the mucosa from insults, stabilize the mucus layer and affect healing of the epithelium. The encoded protein inhibits gastric acid secretion. This gene and two other related trefoil family member genes are found in a cluster on chromosome 21. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 44531785, 44624247, 44.32, 8, 'ENSG00000160181', 'Q03403'),
(3342, 'TFR2', 'This gene encodes a single-pass type II membrane protein, which is a member of the transferrin receptor-like family. This protein mediates cellular uptake of transferrin-bound iron, and may be involved in iron metabolism, hepatocyte function and erythrocyte differentiation. Mutations in this gene have been associated with hereditary hemochromatosis type III. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, May 2011]', '7', 'protein_coding', 38094334, 38150293, 39.11, 7, 'ENSG00000106327', 'Q9UP52'),
(3343, 'TGIF1', 'The protein encoded by this gene is a member of the three-amino acid loop extension (TALE) superclass of atypical homeodomains. TALE homeobox proteins are highly conserved transcription regulators. This particular homeodomain binds to a previously characterized retinoid X receptor responsive element from the cellular retinol-binding protein II promoter. In addition to its role in inhibiting 9-cis-retinoic acid-dependent RXR alpha transcription activation of the retinoic acid responsive element, the protein is an active transcriptional co-repressor of SMAD2 and may participate in the transmission of nuclear signals during development and in the adult. Mutations in this gene are associated with holoprosencephaly type 4, which is a structural anomaly of the brain. Alternative splicing has been observed at this locus and multiple splice variants encoding distinct isoforms are described. [provided by RefSeq, Jul 2013]', '18', 'TR_J_gene', 22493925, 22493990, 51.52, 1, 'ENSG00000177426', 'Q15583'),
(3344, 'TGM3', 'Transglutaminases are enzymes that catalyze the crosslinking of proteins by epsilon-gamma glutamyl lysine isopeptide bonds. While the primary structure of transglutaminases is not conserved, they all have the same amino acid sequence at their active sites and their activity is calcium-dependent. The protein encoded by this gene consists of two polypeptide chains activated from a single precursor protein by proteolysis. The encoded protein is involved the later stages of cell envelope formation in the epidermis and hair follicle. [provided by RefSeq, Jul 2008]', '20', 'snoRNA', 100989130, 100989201, 47.22, 1, 'ENSG00000125780', 'Q08188'),
(3345, 'TGOLN2', 'This gene encodes a type I integral membrane protein that is localized to the trans-Golgi network, a major sorting station for secretory and membrane proteins. The encoded protein cycles between early endosomes and the trans-Golgi network, and may play a role in exocytic vesicle formation. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Oct 2011]', '2', 'protein_coding', 149592512, 149595314, 56.97, 1, 'ENSG00000292253', 'O43493'),
(3346, 'TGOLN2', 'This gene encodes a type I integral membrane protein that is localized to the trans-Golgi network, a major sorting station for secretory and membrane proteins. The encoded protein cycles between early endosomes and the trans-Golgi network, and may play a role in exocytic vesicle formation. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Oct 2011]', '2', 'protein_coding', 149592512, 149595314, 56.97, 1, 'ENSG00000152291', 'O43493'),
(3347, 'THAP5', 'Enables protease binding activity. Involved in negative regulation of cell cycle and negative regulation of transcription by RNA polymerase II. Located in chromatin and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 41955234, 42087830, 35.33, 16, 'ENSG00000177683', 'Q7Z6K1'),
(3348, 'THBD', 'The protein encoded by this intronless gene is an endothelial-specific type I membrane receptor that binds thrombin. This binding results in the activation of protein C, which degrades clotting factors Va and VIIIa and reduces the amount of thrombin generated. Mutations in this gene are a cause of thromboembolic disease, also known as inherited thrombophilia. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 151053815, 151057897, 57.31, 5, 'ENSG00000178726', 'P07204'),
(3349, 'THNSL1', 'Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 113122799, 113149529, 48.54, 7, 'ENSG00000185875', 'Q8IYQ7'),
(3350, 'THRB', 'The protein encoded by this gene is a nuclear hormone receptor for triiodothyronine. It is one of the several receptors for thyroid hormone, and has been shown to mediate the biological activities of thyroid hormone. Knockout studies in mice suggest that the different receptors, while having certain extent of redundancy, may mediate different functions of thyroid hormone. Mutations in this gene are known to be a cause of generalized thyroid hormone resistance (GTHR), a syndrome characterized by goiter and high levels of circulating thyroid hormone (T3-T4), with normal or slightly elevated thyroid stimulating hormone (TSH). Several alternatively spliced transcript variants encoding the same protein have been observed for this gene. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 133553901, 133569835, 51.63, 5, 'ENSG00000151090', 'P10828'),
(3351, 'TIAL1', 'The protein encoded by this gene is a member of a family of RNA-binding proteins, has three RNA recognition motifs (RRMs), and binds adenine and uridine-rich elements in mRNA and pre-mRNAs of a wide range of genes. It regulates various activities including translational control, splicing and apoptosis. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. The different isoforms have been show to function differently with respect to post-transcriptional silencing. [provided by RefSeq, Jul 2008]', '10', 'snoRNA', 76561109, 76561192, 40.48, 1, 'ENSG00000151923', 'Q01085'),
(3352, 'TIGD6', 'The protein encoded by this gene belongs to the tigger subfamily of the pogo superfamily of DNA-mediated transposons in humans. These proteins are related to DNA transposons found in fungi and nematodes, and more distantly to the Tc1 and mariner transposases. They are also very similar to the major mammalian centromere protein B. [provided by RefSeq, Oct 2009]', '5', 'protein_coding', 55769118, 55773420, 41.92, 1, 'ENSG00000164296', 'Q17RP2');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3353, 'TIMM21', 'Involved in mitochondrial cytochrome c oxidase assembly; mitochondrial respiratory chain complex I assembly; and protein import into mitochondrial matrix. Located in mitochondrion. Part of TIM23 mitochondrial import inner membrane translocase complex. [provided by Alliance of Genome Resources, May 2025]', '18', 'IG_V_gene', 106349283, 106349792, 56.47, 1, 'ENSG00000075336', 'Q9BVV7'),
(3354, 'TIMM50', 'This gene encodes a subunit of the TIM23 inner mitochondrial membrane translocase complex. The encoded protein functions as the receptor subunit that recognizes the mitochondrial targeting signal, or presequence, on protein cargo that is destined for the mitochondrial inner membrane and matrix. This protein may also play a role in maintaining the membrane permeability barrier, and knockdown of this gene in human cells results in the release of cytochrome c and apoptosis. [provided by RefSeq, Jul 2016]', '19', 'protein_coding', 111290851, 111320566, 38.26, 10, 'ENSG00000105197', 'Q3ZCQ8'),
(3355, 'TIMP3', 'This gene belongs to the TIMP gene family. The proteins encoded by this gene family are inhibitors of the matrix metalloproteinases, a group of peptidases involved in degradation of the extracellular matrix (ECM). Expression of this gene is induced in response to mitogenic stimulation and this netrin domain-containing protein is localized to the ECM. Mutations in this gene have been associated with the autosomal dominant disorder Sorsby\'s fundus dystrophy. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 32801705, 32863041, 47.17, 1, 'ENSG00000100234', 'P35625'),
(3356, 'TINAG', 'This gene encodes a glycoprotein that is restricted within the kidney to the basement membranes underlying the epithelium of Bowman\'s capsule and proximal and distal tubules. Autoantibodies against this protein are found in sera of patients with tubulointerstital nephritis, membranous nephropathy and anti-glomerular basement membrane nephritis. Ontogeny studies suggest that the expression of this antigen is developmentally regulated in a precise spatial and temporal pattern throughout nephrogenesis. [provided by RefSeq, Nov 2011]', '6', 'protein_coding', 7420324, 7421632, 60.28, 1, 'ENSG00000137251', 'Q9UJW2'),
(3357, 'TLCD4', 'Predicted to be involved in lipid homeostasis. Predicted to be located in membrane. Predicted to be active in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '1', 'TR_V_pseudogene', 142440883, 142441325, 47.18, 1, 'ENSG00000152078', 'Q96MV1'),
(3358, 'TLCD4-RWDD3', 'This locus represents naturally occurring read-through transcription between the neighboring TMEM56 (transmembrane protein 56) and RWDD3 (RWD domain containing 3) genes on chromosome 1. The read-through transcript encodes a protein that shares sequence identity with the upstream gene product, but it contains a distinct C-terminus due to frameshifts versus the downstream gene coding sequence. [provided by RefSeq, Dec 2010]', '1', 'protein_coding', 50152911, 50235310, 41.51, 4, 'ENSG00000271092', 'S4R434'),
(3359, 'TLL2', 'This gene encodes an astacin-like zinc-dependent metalloprotease and is a subfamily member of the metzincin family. Unlike other family members, a similar protein in mice does not cleave procollagen C-propeptides or chordin. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 104985775, 105010482, 53.30, 9, 'ENSG00000095587', 'Q9Y6L7'),
(3360, 'TLX2', 'This gene is a member of an orphan homeobox-containing transcription factor family. Studies of the mouse ortholog have shown that the encoded protein is crucial for the development of the enteric nervous system; in humans, loss-of-function may play a role in tumorigenesis of gastrointestinal stromal tumors. [provided by RefSeq, May 2010]', '2', 'protein_coding', 106543, 137288, 47.78, 2, 'ENSG00000115297', 'O43763'),
(3361, 'TLX3', 'The protein encoded by this gene is an orphan homeobox protein that encodes a DNA-binding nuclear transcription factor. A translocation [t(5;14)(q35;q32)] involving this gene is associated with T-cell acute lymphoblastic leukemia (T-ALL) in children and young adults. [provided by RefSeq, Nov 2015]', '5', 'protein_coding', 130293134, 130324180, 46.73, 11, 'ENSG00000164438', 'O43711'),
(3362, 'TM6SF2', 'Enables identical protein binding activity. Involved in regulation of lipid metabolic process. Located in endoplasmic reticulum membrane and endoplasmic reticulum-Golgi intermediate compartment membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 63538489, 63547749, 64.28, 1, 'ENSG00000213996', 'Q9BZW4'),
(3363, 'TM9SF1', 'Predicted to be involved in protein localization to membrane. Predicted to be located in autophagosome membrane; cytoplasmic vesicle; and lysosomal membrane. Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 37204237, 37212477, 58.85, 2, 'ENSG00000285465', 'O15321'),
(3364, 'TM9SF1', 'Predicted to be involved in protein localization to membrane. Predicted to be located in autophagosome membrane; cytoplasmic vesicle; and lysosomal membrane. Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 37204237, 37212477, 58.85, 2, 'ENSG00000100926', 'O15321'),
(3365, 'TM9SF4', 'Involved in several processes, including positive regulation of protein localization; regulation of intracellular pH; and vacuolar proton-transporting V-type ATPase complex assembly. Located in Golgi apparatus and early endosome. [provided by Alliance of Genome Resources, May 2025]', '20', 'snoRNA', 29121, 29254, 55.22, 1, 'ENSG00000101337', 'Q92544'),
(3366, 'TMC2', 'This gene encodes a transmembrane protein that is necesssary for mechanotransduction in cochlear hair cells of the inner ear. Mutations in this gene may underlie hereditary disorders of balance and hearing. [provided by RefSeq, Aug 2015]', '20', 'TR_V_pseudogene', 21829539, 21830070, 43.05, 1, 'ENSG00000149488', 'Q8TDI7'),
(3367, 'TMCO6', 'Predicted to enable nuclear import signal receptor activity. Predicted to be involved in protein import into nucleus. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 117314557, 117413266, 47.97, 14, 'ENSG00000113119', 'Q96DC7'),
(3368, 'TMED10', 'This gene is a member of the EMP24/GP25L/p24 family and encodes a protein with a GOLD domain. This type I membrane protein is localized to the plasma membrane and golgi cisternae and is involved in vesicular protein trafficking. The protein is also a member of a heteromeric secretase complex and regulates the complex\'s gamma-secretase activity without affecting its epsilon-secretase activity. Mutations in this gene have been associated with early-onset familial Alzheimer\'s disease. This gene has a pseudogene on chromosome 8. [provided by RefSeq, Jul 2008]', '14', 'protein_coding', 74148523, 74160531, 42.28, 6, 'ENSG00000170348', 'P49755'),
(3369, 'TMED7', 'Predicted to be involved in Golgi organization; endoplasmic reticulum to Golgi vesicle-mediated transport; and intracellular protein transport. Located in Golgi apparatus; endoplasmic reticulum; and endoplasmic reticulum-Golgi intermediate compartment. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 146539, 164531, 38.32, 12, 'ENSG00000134970', 'Q9Y3B3'),
(3370, 'TMEM100', 'Involved in BMP signaling pathway. Located in several cellular components, including endoplasmic reticulum; perikaryon; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 150078445, 150257438, 40.69, 7, 'ENSG00000166292', 'Q9NV29'),
(3371, 'TMEM106C', 'Predicted to be located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 30602558, 30624012, 48.18, 6, 'ENSG00000134291', 'Q9BVX2'),
(3372, 'TMEM144', 'Predicted to enable carbohydrate transmembrane transporter activity. Predicted to be involved in carbohydrate transmembrane transport. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 25199858, 25207359, 52.09, 2, 'ENSG00000164124', 'Q7Z5S9'),
(3373, 'TMEM14C', 'Predicted to be involved in regulation of heme biosynthetic process. Predicted to act upstream of or within erythrocyte differentiation; mitochondrial transport; and nitrogen compound transport. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '6', 'miRNA', 42950928, 42951014, 34.48, 1, 'ENSG00000284936', 'Q9P0S9'),
(3374, 'TMEM14C', 'Predicted to be involved in regulation of heme biosynthetic process. Predicted to act upstream of or within erythrocyte differentiation; mitochondrial transport; and nitrogen compound transport. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '6', 'miRNA', 42950928, 42951014, 34.48, 1, 'ENSG00000111843', 'Q9P0S9'),
(3375, 'TMEM170A', 'Involved in endoplasmic reticulum tubular network organization; nuclear envelope organization; and nuclear pore complex assembly. Located in endoplasmic reticulum membrane and nuclear envelope. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 31137534, 31139066, 57.60, 1, 'ENSG00000166822', 'Q8WVE7'),
(3376, 'TMEM171', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 136848004, 136880764, 43.01, 12, 'ENSG00000157111', 'Q8WVE6'),
(3377, 'TMEM176A', 'Predicted to act upstream of or within negative regulation of dendritic cell differentiation. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'snoRNA', 33952769, 33952850, 39.02, 1, 'ENSG00000002933', 'Q96HP8'),
(3378, 'TMEM182', 'Predicted to be involved in myotube cell development involved in skeletal muscle regeneration; negative regulation of myoblast differentiation; and negative regulation of myoblast fusion. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 4200055, 4215375, 46.00, 2, 'ENSG00000170417', 'Q6ZP80'),
(3379, 'TMEM184C', 'Predicted to enable transmembrane transporter activity. Predicted to be involved in transmembrane transport. Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 5840928, 5841952, 45.07, 2, 'ENSG00000164168', 'Q9NVA4'),
(3380, 'TMEM185B', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 102106489, 102120368, 55.32, 4, 'ENSG00000226479', 'Q9H7F4'),
(3381, 'TMEM191A', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 32553001, 32573184, 41.42, 3, 'ENSG00000226287', 'Q9H0A3'),
(3382, 'TMEM215', 'Predicted to act upstream of or within negative regulation of retinal cell programmed cell death and sprouting angiogenesis. Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 16916359, 17346152, 36.63, 7, 'ENSG00000188133', 'Q68D42'),
(3383, 'TMEM230', 'This gene encodes a multi-pass transmembrane protein that belongs to the TMEM134/TMEM230 protein family. The encoded protein localizes to secretory and recycling vesicle in the neuron and may be involved in synaptic vesicles trafficking and recycling. Mutations in this gene may be linked to familial Parkinson\'s disease. [provided by RefSeq, Mar 2017]', '20', 'protein_coding', 56208985, 56215222, 35.12, 1, 'ENSG00000089063', 'Q96A57'),
(3384, 'TMEM232', 'Predicted to be involved in flagellated sperm motility and spermatid cytoplasm removal during spermiation of flagellated sperm. Predicted to act upstream of or within maintenance of protein complex location. Predicted to be located in membrane. Predicted to be active in outer dense fiber. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 83962, 131161, 50.86, 4, 'ENSG00000186952', 'C9JQI7'),
(3385, 'TMEM238', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 23503496, 23531822, 45.10, 14, 'ENSG00000233493', 'C9JI98'),
(3386, 'TMEM254', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 101065447, 101065537, 52.75, 1, 'ENSG00000133678', 'Q8TBM7'),
(3387, 'TMEM256-PLSCR3', 'This locus represents naturally occurring read-through transcription between the neighboring chromosome 17 open reading frame 61 (C17orf61) and phospholipid scramblase 3 (PLSCR3) genes. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is unlikely to produce a protein product. [provided by RefSeq, Jan 2011]', '17', 'protein_coding', 47808957, 47931146, 37.52, 12, 'ENSG00000284483', 'K7ENI6'),
(3388, 'TMEM256-PLSCR3', 'This locus represents naturally occurring read-through transcription between the neighboring chromosome 17 open reading frame 61 (C17orf61) and phospholipid scramblase 3 (PLSCR3) genes. The read-through transcript is a candidate for nonsense-mediated mRNA decay (NMD), and is unlikely to produce a protein product. [provided by RefSeq, Jan 2011]', '17', 'protein_coding', 47808957, 47931146, 37.52, 12, 'ENSG00000262481', 'K7ENI6'),
(3389, 'TMEM258', 'Predicted to enable oligosaccharyltransferase complex binding activity. Involved in protein N-linked glycosylation. Located in endoplasmic reticulum and membrane. Part of oligosaccharyltransferase complex A and oligosaccharyltransferase complex B. [provided by Alliance of Genome Resources, May 2025]', '11', 'miRNA', 53671968, 53672040, 46.58, 1, 'ENSG00000134825', 'P61165'),
(3390, 'TMEM42', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 35066136, 35087387, 47.22, 2, 'ENSG00000280516', 'Q69YG0'),
(3391, 'TMEM42', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 35066136, 35087387, 47.22, 2, 'ENSG00000169964', 'Q69YG0'),
(3392, 'TMEM47', 'This gene encodes a member of the PMP22/EMP/claudin protein family. The encoded protein is localized to the ER and the plasma membrane. In dogs, transcripts of this gene exist at high levels in the brain. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 22534854, 22714578, 41.89, 8, 'ENSG00000147027', 'Q9BQJ4'),
(3393, 'TMEM50B', 'Predicted to be involved in late endosome to vacuole transport via multivesicular body sorting pathway. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 105708778, 105847725, 35.68, 11, 'ENSG00000142188', 'P56557'),
(3394, 'TMEM50B', 'Predicted to be involved in late endosome to vacuole transport via multivesicular body sorting pathway. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 105708778, 105847725, 35.68, 11, 'ENSG00000263160', 'P56557'),
(3395, 'TMEM74B', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 434786, 438955, 48.56, 2, 'ENSG00000125895', 'Q9NUR3'),
(3396, 'TMPRSS2', 'This gene encodes a protein that belongs to the serine protease family. The encoded protein contains a type II transmembrane domain, a receptor class A domain, a scavenger receptor cysteine-rich domain and a protease domain. Serine proteases are known to be involved in many physiological and pathological processes. This gene was demonstrated to be up-regulated by androgenic hormones in prostate cancer cells and down-regulated in androgen-independent prostate cancer tissue. The protease domain of this protein is thought to be cleaved and secreted into cell media after autocleavage. This protein also facilitates entry of viruses into host cells by proteolytically cleaving and activating viral envelope glycoproteins. Viruses found to use this protein for cell entry include Influenza virus and the human coronaviruses HCoV-229E, MERS-CoV, SARS-CoV and SARS-CoV-2 (COVID-19 virus). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2020]', '21', 'protein_coding', 21536107, 21604847, 37.09, 1, 'ENSG00000184012', 'O15393'),
(3397, 'TMX1', 'This gene encodes a member of the disulfide isomerase (PDI) family of endoplasmic reticulum (ER) proteins that catalyze protein folding and thiol-disulfide interchange reactions. The encoded protein has an N-terminal ER-signal sequence, a catalytically active thioredoxin domain, and one transmembrane domain. Unlike most members of this gene family, it lacks a C-terminal ER-retention sequence. The mature membrane-bound protein can both oxidize and reduce disulfide bonds and acts selectively on membrane-associated polypeptides. [provided by RefSeq, Jan 2017]', '14', 'protein_coding', 19471296, 20575873, 34.89, 12, 'ENSG00000139921', 'Q9H3N1'),
(3398, 'TMX3', 'This gene encodes a member of the disulfide isomerase (PDI) family of endoplasmic reticulum (ER) proteins that catalyze protein folding and thiol-disulfide interchange reactions. The canonical protein encoded by this gene has an N-terminal ER-signal sequence, a catalytically active thioredoxin domain, one transmembrane domain and a C-terminal ER-retention sequence. This gene is expressed in many tissues but has its highest expression in heart and skeletal muscle. It is expressed in the retinal neuroepithelium and lens epithelium in the developing murine eye and haploinsufficiency of this gene in humans and zebrafish is associated with microphthalmia. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Apr 2017]', '18', 'protein_coding', 661596, 675802, 56.68, 1, 'ENSG00000166479', 'Q96JJ7'),
(3399, 'TNFAIP8L2', 'Predicted to be involved in negative regulation of T cell activation and negative regulation of inflammatory response. Predicted to be located in lysosome and nucleus. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 142939343, 142940868, 53.87, 2, 'ENSG00000163154', 'Q6P589'),
(3400, 'TNFRSF1B', 'The protein encoded by this gene is a member of the TNF-receptor superfamily. This protein and TNF-receptor 1 form a heterocomplex that mediates the recruitment of two anti-apoptotic proteins, c-IAP1 and c-IAP2, which possess E3 ubiquitin ligase activity. The function of IAPs in TNF-receptor signalling is unknown, however, c-IAP1 is thought to potentiate TNF-induced apoptosis by the ubiquitination and degradation of TNF-receptor-associated factor 2, which mediates anti-apoptotic signals. Knockout studies in mice also suggest a role of this protein in protecting neurons from apoptosis by stimulating antioxidative pathways. [provided by RefSeq, Jul 2008]', '1', 'miRNA', 43150895, 43150994, 64.00, 1, 'ENSG00000028137', 'P20333'),
(3401, 'TNFSF9', 'The protein encoded by this gene is a cytokine that belongs to the tumor necrosis factor (TNF) ligand family. This transmembrane cytokine is a bidirectional signal transducer that acts as a ligand for TNFRSF9/4-1BB, which is a costimulatory receptor molecule in T lymphocytes. This cytokine and its receptor are involved in the antigen presentation process and in the generation of cytotoxic T cells. The receptor TNFRSF9/4-1BB is absent from resting T lymphocytes but rapidly expressed upon antigenic stimulation. The ligand encoded by this gene, TNFSF9/4-1BBL, has been shown to reactivate anergic T lymphocytes in addition to promoting T lymphocyte proliferation. This cytokine has also been shown to be required for the optimal CD8 responses in CD8 T cells. This cytokine is expressed in carcinoma cell lines, and is thought to be involved in T cell-tumor cell interaction.[provided by RefSeq, Oct 2008]', '19', 'protein_coding', 23520754, 23557731, 47.68, 17, 'ENSG00000125657', 'P41273'),
(3402, 'TNIK', 'Wnt signaling plays important roles in carcinogenesis and embryonic development. The protein encoded by this gene is a serine/threonine kinase that functions as an activator of the Wnt signaling pathway. Mutations in this gene are associated with an autosomal recessive form of cognitive disability. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2017]', '3', 'protein_coding', 62817, 63773, 47.54, 1, 'ENSG00000154310', 'Q9UKE5'),
(3403, 'TNKS', 'Enables histone binding activity; pentosyltransferase activity; and zinc ion binding activity. Involved in several processes, including positive regulation of canonical Wnt signaling pathway; post-translational protein modification; and regulation of chromosome organization. Acts upstream of or within peptidyl-serine phosphorylation and peptidyl-threonine phosphorylation. Located in several cellular components, including chromosome, telomeric region; mitotic spindle pole; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 32465072, 32496686, 45.12, 1, 'ENSG00000285372', 'O95271'),
(3404, 'TNKS', 'Enables histone binding activity; pentosyltransferase activity; and zinc ion binding activity. Involved in several processes, including positive regulation of canonical Wnt signaling pathway; post-translational protein modification; and regulation of chromosome organization. Acts upstream of or within peptidyl-serine phosphorylation and peptidyl-threonine phosphorylation. Located in several cellular components, including chromosome, telomeric region; mitotic spindle pole; and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 32465072, 32496686, 45.12, 1, 'ENSG00000173273', 'O95271'),
(3405, 'TNMD', 'This gene encodes a protein that is related to chondromodulin-I, which is a cartilage-specific glycoprotein that functions to stimulate chondrocyte growth and to inhibit tube formation of endothelial cells. This protein is also an angiogenesis inhibitor. Genetic variation in this gene is associated with a risk for type 2 diabetes, central obesity and serum levels of systemic immune mediators in a body size-dependent manner. This gene is also a candidate gene for age-related macular degeneration, though a direct link has yet to be demonstrated. [provided by RefSeq, Sep 2009]', 'X', 'protein_coding', 5498697, 5499570, 66.82, 1, 'ENSG00000000005', 'Q9H2S6'),
(3406, 'TOB1', 'This gene encodes a member of the transducer of erbB-2 /B-cell translocation gene protein family. Members of this family are anti-proliferative factors that have the potential to regulate cell growth. The encoded protein may function as a tumor suppressor. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]', '17', 'protein_coding', 32485338, 32495283, 39.74, 10, 'ENSG00000141232', 'P50616'),
(3407, 'TOMM6', 'C6ORF49 is a member of the LIM domain protein family (Teufel et al., 2005 [PubMed 15702247]).[supplied by OMIM, Mar 2008]', '6', 'protein_coding', 123828983, 123877481, 39.21, 5, 'ENSG00000214736', 'Q96B49'),
(3408, 'TOMM70', 'This gene encodes an import receptor of the outer mitochondrial membrane that is part of the translocase of the outer membrane complex. This protein is involved in the import of mitochondrial precursor proteins. The protein interacts with the SARS-CoV and SARS-Cov-2 ORF9b proteins. [provided by RefSeq, Dec 2021]', '3', 'miRNA', 101066724, 101066801, 37.18, 1, 'ENSG00000154174', 'O94826'),
(3409, 'TOX2', 'Enables transcription coactivator activity. Involved in positive regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 21372801, 21654695, 35.07, 40, 'ENSG00000124191', 'Q96NM4'),
(3410, 'TP53INP2', 'The protein encoded by this gene promotes autophagy and is essential for proper autophagosome formation and processing. In addition, the encoded protein can enhance rDNA transcription by helping in the assembly of the POLR1/RNA polymerase I preinitiation complex. Finally, this protein serves as a transcriptional activator for some genes. [provided by RefSeq, Jul 2016]', '20', 'protein_coding', 130379449, 130385674, 43.21, 2, 'ENSG00000078804', 'Q8IXH6'),
(3411, 'TP53TG5', 'Predicted to be involved in intracellular signal transduction and negative regulation of cell growth. Located in chromosome and nucleolus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 44666189, 44666927, 60.49, 1, 'ENSG00000124251', 'Q9Y2B4'),
(3412, 'TPO', 'This gene encodes a membrane-bound glycoprotein. The encoded protein acts as an enzyme and plays a central role in thyroid gland function. The protein functions in the iodination of tyrosine residues in thyroglobulin and phenoxy-ester formation between pairs of iodinated tyrosines to generate the thyroid hormones, thyroxine and triiodothyronine. Mutations in this gene are associated with several disorders of thyroid hormonogenesis, including congenital hypothyroidism, congenital goiter, and thyroid hormone organification defect IIA. Multiple transcript variants encoding distinct isoforms have been identified for this gene, but the full-length nature of some variants has not been determined. [provided by RefSeq, May 2011]', '2', 'protein_coding', 79964582, 79970822, 50.46, 5, 'ENSG00000115705', 'P07202'),
(3413, 'TPP2', 'This gene encodes a mammalian peptidase that, at neutral pH, removes tripeptides from the N terminus of longer peptides. The protein has a specialized function that is essential for some MHC class I antigen presentation. The protein is a high molecular mass serine exopeptidase; the amino acid sequence surrounding the serine residue at the active site is similar to the peptidases of the subtilisin class rather than the trypsin class. [provided by RefSeq, Jul 2008]', '13', 'protein_coding', 35617844, 35814611, 37.81, 16, 'ENSG00000134900', 'P29144'),
(3414, 'TPPP2', 'Enables tubulin binding activity. Involved in regulation of flagellated sperm motility. Located in cytosol. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 138868921, 138875368, 38.26, 4, 'ENSG00000179636', 'P59282'),
(3415, 'TPPP3', 'Enables tubulin binding activity. Involved in decidualization and microtubule bundle formation. Located in microtubule bundle and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 77910690, 78050395, 38.61, 57, 'ENSG00000159713', 'Q9BW30'),
(3416, 'TPRG1', 'Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 40623428, 40624136, 56.70, 1, 'ENSG00000188001', 'Q6ZUI0'),
(3417, 'TPRN', 'This locus encodes a sensory epithelial protein. It was defined by linkage analysis in three Pakistani families to lie between D9S1818 (centromeric) and D9SH6 (telomeric). Mutations at this locus have been associated with autosomal recessive deafness. [provided by RefSeq, Oct 2010]', '9', 'transcribed_unprocessed_pseudogene', 116942, 117899, 41.34, 1, 'ENSG00000176058', 'Q4KMQ1'),
(3418, 'TPST1', 'Enables protein homodimerization activity and protein-tyrosine sulfotransferase activity. Involved in post-translational protein modification. Located in Golgi membrane. Implicated in colorectal adenocarcinoma. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 158201604, 158255416, 37.80, 17, 'ENSG00000169902', 'O60507'),
(3419, 'TPST2', 'The protein encoded by this gene catalyzes the O-sulfation of tyrosine residues within acidic regions of proteins. The encoded protein is a type II integral membrane protein found in the Golgi body. Alternative splicing produces multiple transcript variants encoding distinct isoforms. [provided by RefSeq, May 2018]', '22', 'protein_coding', 32948613, 32969094, 43.69, 9, 'ENSG00000128294', 'O60704'),
(3420, 'TPT1', 'This gene encodes a protein that is a regulator of cellular growth and proliferation. Its mRNA is highly structured and contains an oligopyrimidine tract (5\'-TOP) in its 5\' untranslated region that functions to repress its translation under quiescent conditions. The encoded protein is involved in a variety of cellular pathways, including apoptosis, protein synthesis and cell division. It binds to and stabilizes microtubules, and removal of this protein through phosphorylation is required for progression through mitotic and meiotic cell divisions. This gene is known to play a role in carcinogenesis, and is upregulated in some cancer cells. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Aug 2017]', '13', 'protein_coding', 114234887, 114272723, 41.32, 12, 'ENSG00000133112', 'P13693'),
(3421, 'TPTEP2-CSNK1E', 'This locus represents naturally occurring readthrough transcription between the neighboring LOC400927 (transmembrane phosphoinositide 3-phosphatase and tensin homolog 2 pseudogene) and CSNK1E (casein kinase I isoform epsilon) genes on chromosome 22. The readthrough transcript encodes the same protein as the downstream gene product (casein kinase I isoform epsilon). [provided by RefSeq, Feb 2014]', '22', 'miRNA', 15643294, 15643372, 59.49, 1, 'ENSG00000283900', ''),
(3422, 'TRAC', 'This gene encodes a nuclear receptor co-repressor that mediates transcriptional silencing of certain target genes. The encoded protein is a member of a family of thyroid hormone- and retinoic acid receptor-associated co-repressors. This protein acts as part of a multisubunit complex which includes histone deacetylases to modify chromatin structure that prevents basal transcriptional activity of target genes. Aberrant expression of this gene is associated with certain cancers. Alternate splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Apr 2011]', '12', 'protein_coding', 52644558, 52652211, 49.91, 2, 'ENSG00000277734', 'P01848'),
(3423, 'TRAJ10', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 117675469, 117710727, 37.38, 1, 'ENSG00000211879', 'A0N4Z7'),
(3424, 'TRAJ12', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 146012, 159877, 43.95, 25, 'ENSG00000211877', 'A0A075B6U8'),
(3425, 'TRAJ16', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 64943753, 65126112, 40.38, 19, 'ENSG00000211873', 'A0A075B6V0'),
(3426, 'TRAJ19', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'lncRNA', 4731279, 4740668, 43.25, 4, 'ENSG00000211870', 'A0A075B6Y4'),
(3427, 'TRAJ2', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 9206765, 9405683, 46.56, 6, 'ENSG00000211887', 'A0A075B6U9'),
(3428, 'TRAJ21', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 4686350, 4701590, 41.80, 4, 'ENSG00000211868', 'A0A075B6V2'),
(3429, 'TRAJ22', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 2508537, 2526597, 60.69, 12, 'ENSG00000211867', 'A0A075B6Z0'),
(3430, 'TRAJ26', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 134317098, 134440585, 62.16, 5, 'ENSG00000211863', 'A0A075B6V7'),
(3431, 'TRAJ27', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 36991120, 37007851, 47.36, 4, 'ENSG00000211862', 'A0A075B6W6'),
(3432, 'TRAJ30', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 30601087, 30601382, 45.27, 1, 'ENSG00000211859', 'A0A075B6Y1');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3433, 'TRAJ32', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 30560875, 30561314, 43.86, 1, 'ENSG00000211857', 'A0A075B6X3'),
(3434, 'TRAJ36', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 30501657, 30502141, 48.87, 1, 'ENSG00000276699', 'A0A087WU04'),
(3435, 'TRAJ37', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 30491462, 30492013, 44.57, 1, 'ENSG00000278661', 'A0A087X096'),
(3436, 'TRAJ4', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 154835788, 155026940, 39.69, 7, 'ENSG00000211885', 'A0A075B6Z5'),
(3437, 'TRAJ41', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'miRNA', 30375294, 30375378, 54.12, 1, 'ENSG00000211848', 'A0A075B702'),
(3438, 'TRAJ43', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 30337013, 30337694, 47.36, 1, 'ENSG00000211846', 'A0A075B6V1'),
(3439, 'TRAJ45', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 33290512, 33371157, 42.57, 6, 'ENSG00000211844', 'A0A075B6X0'),
(3440, 'TRAJ46', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 124862130, 124877733, 52.55, 3, 'ENSG00000211843', 'A0A075B6W4'),
(3441, 'TRAJ47', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 52024691, 52033600, 41.96, 1, 'ENSG00000211842', 'A0A075B6Y5'),
(3442, 'TRAJ48', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 58651434, 58715410, 47.70, 2, 'ENSG00000211841', 'A0A075B6V3'),
(3443, 'TRAJ49', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 45372557, 45407889, 45.13, 4, 'ENSG00000211840', 'A0A075B6Y0'),
(3444, 'TRAJ50', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 5623035, 5668478, 47.35, 14, 'ENSG00000211839', 'A0A075B6X7'),
(3445, 'TRAJ51', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 51861969, 51866232, 47.35, 1, 'ENSG00000248366', ''),
(3446, 'TRAJ52', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 5420664, 5490220, 38.91, 3, 'ENSG00000211838', 'A0A075B6W2'),
(3447, 'TRAJ53', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 44328944, 44339402, 60.43, 7, 'ENSG00000211837', 'A0A075B6W9'),
(3448, 'TRAJ6', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 75601509, 75718281, 37.44, 12, 'ENSG00000211883', 'A0A075B6Z3'),
(3449, 'TRAJ61', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 17315421, 17454595, 40.76, 3, 'ENSG00000211831', 'A0A075B708'),
(3450, 'TRAM2', 'TRAM2 is a component of the translocon, a gated macromolecular channel that controls the posttranslational processing of nascent secretory and membrane proteins at the endoplasmic reticulum (ER) membrane.[supplied by OMIM, Jul 2004]', '6', 'protein_coding', 79025686, 79028505, 48.19, 5, 'ENSG00000065308', 'Q15035'),
(3451, 'TRAP1', 'This gene encodes a mitochondrial chaperone protein that is member of the heat shock protein 90 (HSP90) family. The encoded protein has ATPase activity and interacts with tumor necrosis factor type I. This protein may function in regulating cellular stress responses. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jan 2013]', '16', 'protein_coding', 143864436, 143938343, 39.82, 2, 'ENSG00000126602', 'Q12931'),
(3452, 'TRAPPC14', 'Enables alpha-tubulin binding activity. Involved in cilium assembly and regulation of cell population proliferation. Located in several cellular components, including microtubule cytoskeleton; midbody; and plasma membrane. Part of TRAPPII protein complex. Implicated in primary autosomal recessive microcephaly 25. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 139266880, 139302551, 42.39, 12, 'ENSG00000146826', 'Q8WVR3'),
(3453, 'TRAPPC6A', 'This gene encodes a component of the trafficking protein particle complex, which tethers transport vesicles to the cis-Golgi membrane. Loss of expression of the related gene in mouse affects coat and eye pigmentation, suggesting that the encoded protein may be involved in melanosome biogenesis. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Aug 2012]', '19', 'unprocessed_pseudogene', 124054793, 124055707, 48.52, 1, 'ENSG00000007255', 'O75865'),
(3454, 'TRAPPC6B', 'TRAPPC6B is a component of TRAPP complexes, which are tethering complexes involved in vesicle transport (Kummel et al., 2005 [PubMed 16025134]).[supplied by OMIM, Mar 2008]', '14', 'protein_coding', 71396920, 71815947, 52.30, 17, 'ENSG00000182400', 'Q86SZ2'),
(3455, 'TRAV1-2', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 23450403, 23452876, 48.14, 2, 'ENSG00000256553', 'A0A0B4J238'),
(3456, 'TRAV11', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 117232625, 117286454, 42.99, 8, 'ENSG00000256474', ''),
(3457, 'TRAV13-1', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 55258415, 55264916, 34.42, 2, 'ENSG00000211788', 'A0A0B4J241'),
(3458, 'TRAV13-2', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 135136916, 135531218, 38.97, 11, 'ENSG00000211791', 'A0A0B4J235'),
(3459, 'TRAV4', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 73772276, 73780251, 47.53, 4, 'ENSG00000211778', 'A0A0B4J268'),
(3460, 'TRAV5', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 183444635, 183449064, 41.20, 5, 'ENSG00000211779', 'A0A0B4J249'),
(3461, 'TRAV8-1', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 70366290, 70434292, 39.68, 2, 'ENSG00000211782', 'A0A0A6YYK1'),
(3462, 'TRAV8-4', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 48402082, 48411960, 44.94, 2, 'ENSG00000211790', 'P01737');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3463, 'TRAV9-1', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 86935540, 86969481, 55.09, 7, 'ENSG00000211783', 'A0A075B6T8'),
(3464, 'TRBV11-3', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'transcribed_processed_pseudogene', 112891610, 112893079, 47.41, 1, 'ENSG00000276597', 'A0A5A6'),
(3465, 'TRBV11-3', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'transcribed_processed_pseudogene', 112891610, 112893079, 47.41, 1, 'ENSG00000282242', 'A0A5A6'),
(3466, 'TRBV12-2', 'Predicted to be involved in immune system process. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'unprocessed_pseudogene', 57055587, 57056132, 37.36, 1, 'ENSG00000211739', 'A0A8S4QGC6'),
(3467, 'TRBV12-2', 'Predicted to be involved in immune system process. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'unprocessed_pseudogene', 57055587, 57056132, 37.36, 1, 'ENSG00000282326', 'A0A8S4QGC6'),
(3468, 'TRBV2', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 75647773, 75761872, 42.42, 5, 'ENSG00000282568', 'A0A1B0GX68'),
(3469, 'TRBV2', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 75647773, 75761872, 42.42, 5, 'ENSG00000226660', 'A0A1B0GX68'),
(3470, 'TRBV24-1', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 44957992, 45109016, 51.44, 19, 'ENSG00000282730', 'A0A075B6N3'),
(3471, 'TRBV24-1', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 44957992, 45109016, 51.44, 19, 'ENSG00000211750', 'A0A075B6N3'),
(3472, 'TRBV27', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 85003877, 85093315, 37.14, 1, 'ENSG00000282234', 'A0A0K0K1C4'),
(3473, 'TRBV27', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 85003877, 85093315, 37.14, 1, 'ENSG00000211752', 'A0A0K0K1C4'),
(3474, 'TRBV3-1', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 23362182, 23373062, 44.84, 5, 'ENSG00000282624', 'A0A576'),
(3475, 'TRBV3-1', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 23362182, 23373062, 44.84, 5, 'ENSG00000237702', 'A0A576'),
(3476, 'TRBV5-5', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 57325986, 57335892, 50.56, 5, 'ENSG00000211725', 'A0A597'),
(3477, 'TRBV5-5', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 57325986, 57335892, 50.56, 5, 'ENSG00000282577', 'A0A597'),
(3478, 'TRBV5-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 184241405, 184241489, 70.59, 1, 'ENSG00000211728', 'A0A599'),
(3479, 'TRBV5-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 184241405, 184241489, 70.59, 1, 'ENSG00000282098', 'A0A599'),
(3480, 'TRBV6-2', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 99896248, 99946618, 34.70, 12, 'ENSG00000282719', 'A0A0J9YXY3'),
(3481, 'TRBV6-2', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 99896248, 99946618, 34.70, 12, 'ENSG00000283063', 'A0A0J9YXY3'),
(3482, 'TRBV6-4', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'unprocessed_pseudogene', 5566678, 5567616, 42.28, 1, 'ENSG00000282240', 'A0A1B0GX49'),
(3483, 'TRBV6-4', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'unprocessed_pseudogene', 5566678, 5567616, 42.28, 1, 'ENSG00000211713', 'A0A1B0GX49'),
(3484, 'TRBV6-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 151039, 167809, 47.06, 2, 'ENSG00000211724', 'A0A0A6YYG2'),
(3485, 'TRBV6-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 151039, 167809, 47.06, 2, 'ENSG00000282459', 'A0A0A6YYG2'),
(3486, 'TRBV7-4', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 112861188, 112898371, 41.89, 12, 'ENSG00000253409', 'A0A1B0GX95'),
(3487, 'TRBV7-4', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 112861188, 112898371, 41.89, 12, 'ENSG00000282756', 'A0A1B0GX95'),
(3488, 'TRBV7-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 133181409, 133214832, 41.98, 19, 'ENSG00000211727', 'A0A1B0GX31'),
(3489, 'TRBV7-6', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 133181409, 133214832, 41.98, 19, 'ENSG00000282704', 'A0A1B0GX31'),
(3490, 'TRBV7-7', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 135881063, 135893444, 43.39, 3, 'ENSG00000253291', 'A0A0K0K1E9'),
(3491, 'TRBV7-7', 'Predicted to be involved in cell surface receptor signaling pathway. Predicted to be part of T cell receptor complex. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 135881063, 135893444, 43.39, 3, 'ENSG00000282179', 'A0A0K0K1E9'),
(3492, 'TRDC', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 59484443, 59505410, 36.78, 14, 'ENSG00000211829', 'B7Z8K6'),
(3493, 'TRDD2', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 23334849, 23404374, 41.17, 2, 'ENSG00000237235', ''),
(3494, 'TRDMT1', 'This gene encodes a protein responsible for the methylation of aspartic acid transfer RNA, specifically at the cytosine-38 residue in the anticodon loop. This enzyme also possesses residual DNA-(cytosine-C5) methyltransferase activity. While similar in sequence and structure to DNA cytosine methyltransferases, this gene is distinct and highly conserved in its function among taxa. [provided by RefSeq, Jun 2010]', '10', 'protein_coding', 65410592, 65744121, 37.41, 17, 'ENSG00000107614', 'O14717'),
(3495, 'TRDV3', 'T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]', '14', 'protein_coding', 44455510, 44462196, 61.15, 1, 'ENSG00000256590', 'A0JD37'),
(3496, 'TRIM36', 'The protein encoded by this gene is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. Multiple alternatively spliced transcript variants that encode different protein isoforms have been described for this gene. [provided by RefSeq, Jul 2008]', '5', 'protein_coding', 62403972, 62628582, 38.30, 14, 'ENSG00000152503', 'Q9NQ86'),
(3497, 'TRIM42', 'This gene encodes a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, namely a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. [provided by RefSeq, Jul 2008]', '3', 'protein_coding', 78129748, 78139650, 38.33, 2, 'ENSG00000155890', 'Q8IWZ5'),
(3498, 'TRIM49', 'The protein encoded by this gene contains a RING zinc finger, a motif known to be involved in protein-protein interactions. This gene has been found to be preferentially expressed in testis. Related pseudogenes and gene duplicates have also been identified on chromosome 11. [provided by RefSeq, Aug 2010]', '11', 'protein_coding', 143851375, 143902198, 39.07, 7, 'ENSG00000168930', 'P0CI25'),
(3499, 'TRIM51', 'Predicted to enable ubiquitin protein ligase activity. Predicted to be involved in innate immune response and regulation of gene expression. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 115267190, 115296693, 36.07, 7, 'ENSG00000124900', 'Q9BSJ1'),
(3500, 'TRIM6', 'The protein encoded by this gene is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, B-box type 1 and B-box type 2 domain, and a coiled-coil region. The protein localizes to the nucleus, but its specific function has not been identified. This gene is mapped to chromosome 11p15, where it resides within a TRIM gene cluster. Alternative splicing results in multiple transcript variants. A read-through transcript from this gene into the downstream TRIM34 gene has also been observed, which results in a fusion product from these neighboring family members. [provided by RefSeq, Oct 2010]', '11', 'protein_coding', 63938959, 63957319, 50.29, 10, 'ENSG00000121236', 'Q9C030'),
(3501, 'TRIM77', 'Predicted to enable ubiquitin protein ligase activity. Predicted to be involved in innate immune response and regulation of gene expression. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 163343892, 163351934, 38.41, 3, 'ENSG00000214414', 'I1YAP6'),
(3502, 'TRIP6', 'This gene is a member of the zyxin family and encodes a protein with three LIM zinc-binding domains. This protein localizes to focal adhesion sites and along actin stress fibers. Recruitment of this protein to the plasma membrane occurs in a lysophosphatidic acid (LPA)-dependent manner and it regulates LPA-induced cell migration. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized. [provided by RefSeq, Jul 2008]', '7', 'unprocessed_pseudogene', 56188379, 56189251, 38.03, 1, 'ENSG00000087077', 'Q15654'),
(3503, 'TRMT2B', 'This gene encodes a homolog of the TRM2 gene in S. cerevisiae. The yeast gene encodes a tRNA methyltransferase that plays a role in tRNA maturation. The yeast protein also has endo-exonuclease activity and may be involved in DNA double strand break repair. Alternative splicing results in multiple transcripts encoding different isoforms. [provided by RefSeq, Nov 2009]', 'X', 'protein_coding', 106041599, 106316683, 35.90, 19, 'ENSG00000188917', 'Q96GJ1'),
(3504, 'TRMT6', 'This gene encodes a member of the tRNA methyltransferase 6 protein family. A similar protein in yeast is part of a two component methyltransferase, which is involved in the posttranslational modification that produces the modified nucleoside 1-methyladenosine in tRNAs. Modified 1-methyladenosine influences initiator methionine stability and may be involved in the replication of human immunodeficiency virus type 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]', '20', 'protein_coding', 26443107, 26483931, 46.91, 49, 'ENSG00000089195', 'Q9UJA5'),
(3505, 'TRMT9B', 'Enables tRNA methyltransferase activity. Predicted to be involved in tRNA methylation and tRNA wobble uridine modification. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 28300346, 28495145, 41.61, 10, 'ENSG00000284872', 'Q9P272'),
(3506, 'TRMT9B', 'Enables tRNA methyltransferase activity. Predicted to be involved in tRNA methylation and tRNA wobble uridine modification. Predicted to be active in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 28300346, 28495145, 41.61, 10, 'ENSG00000250305', 'Q9P272'),
(3507, 'TRO', 'This gene encodes a membrane protein that mediates cell adhesion between trophoblastic cells and the epithelial cells of the endometrium. The encoded protein participates in cell signalling during embryo implantation, and may also be involved in cancer formation. This gene is located near several other closely related genes on chromosome X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2012]', 'X', 'protein_coding', 1982717, 2041903, 48.30, 15, 'ENSG00000067445', 'Q12816'),
(3508, 'TRPC4AP', 'Enables phosphatase binding activity and ubiquitin-like ligase-substrate adaptor activity. Involved in protein ubiquitination and ubiquitin-dependent protein catabolic process via the C-end degron rule pathway. Located in cytosol. Part of Cul4A-RING E3 ubiquitin ligase complex. Is active in Cul4-RING E3 ubiquitin ligase complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 144373101, 144393242, 57.52, 8, 'ENSG00000100991', 'Q8TEL6'),
(3509, 'TRPC5', 'This gene belongs to the transient receptor family. It encodes one of the seven mammalian TRPC (transient receptor potential channel) proteins. The encoded protein is a multi-pass membrane protein and is thought to form a receptor-activated non-selective calcium permeant cation channel. The protein is active alone or as a heteromultimeric assembly with TRPC1, TRPC3, and TRPC4. It also interacts with multiple proteins including calmodulin, CABP1, enkurin, Na(+)-H+ exchange regulatory factor (NHERF ), interferon-induced GTP-binding protein (MX1), ring finger protein 24 (RNF24), and SEC14 domain and spectrin repeat-containing protein 1 (SESTD1). [provided by RefSeq, May 2010]', 'X', 'protein_coding', 165233, 169508, 55.29, 3, 'ENSG00000072315', 'Q9UL62'),
(3510, 'TRPC6', 'The protein encoded by this gene forms a receptor-activated calcium channel in the cell membrane. The channel is activated by diacylglycerol and is thought to be under the control of a phosphatidylinositol second messenger system. Activation of this channel occurs independently of protein kinase C and is not triggered by low levels of intracellular calcium. Defects in this gene are a cause of focal segmental glomerulosclerosis 2 (FSGS2). [provided by RefSeq, Mar 2009]', '11', 'protein_coding', 52765380, 52772784, 40.11, 3, 'ENSG00000137672', 'Q9Y210'),
(3511, 'TSC22D3', 'This gene encodes the anti-inflammatory protein glucocorticoid (GC)-induced leucine zipper. Expression of this gene stimulated by glucocorticoids and interleukin 10 and it appears to play a key role in the anti-inflammatory and immunosuppressive effects of this steroid. This protein has also been shown to inhibit pro-inflammatory molecules including nuclear factor &#954;B. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]', 'X', 'protein_coding', 33936386, 33938918, 62.14, 1, 'ENSG00000157514', 'Q99576'),
(3512, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000274796', 'Q9BSV6'),
(3513, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000274078', 'Q9BSV6'),
(3514, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000278622', 'Q9BSV6'),
(3515, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000273896', 'Q9BSV6'),
(3516, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000274672', 'Q9BSV6'),
(3517, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000275165', 'Q9BSV6'),
(3518, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000274129', 'Q9BSV6'),
(3519, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000278712', 'Q9BSV6'),
(3520, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000278605', 'Q9BSV6'),
(3521, 'TSEN34', 'This gene encodes a catalytic subunit of the tRNA splicing endonuclease, which catalyzes the removal of introns from precursor tRNAs. The endonuclease complex is also associated with a pre-mRNA 3-prime end processing factor. A mutation in this gene results in the neurological disorder pontocerebellar hypoplasia type 2. Multiple alternatively spliced variants, encoding the same protein, have been identified.[provided by RefSeq, Oct 2009]', '19', 'miRNA', 63786002, 63786087, 44.19, 1, 'ENSG00000170892', 'Q9BSV6'),
(3522, 'TSHZ1', 'This gene encodes a colon cancer antigen that was defined by serological analysis of recombinant cDNA expression libraries. The encoded protein is a member of the teashirt C2H2-type zinc-finger protein family and may be involved in transcriptional regulation of developmental processes. Mutations in this gene may be associated with congenital aural atresia syndrome. [provided by RefSeq, Jan 2012]', '18', 'snoRNA', 100991919, 100991990, 45.83, 1, 'ENSG00000179981', 'Q6ZSZ6'),
(3523, 'TSPAN15', 'The protein encoded by this gene is a member of the transmembrane 4 superfamily, also known as the tetraspanin family. Most of these members are cell-surface proteins that are characterized by the presence of four hydrophobic domains. The proteins mediate signal transduction events that play a role in the regulation of cell development, activation, growth and motility. The use of alternate polyadenylation sites has been found for this gene. [provided by RefSeq, Jul 2008]', '10', 'protein_coding', 6583183, 6604103, 48.30, 3, 'ENSG00000099282', 'O95858'),
(3524, 'TSPAN17', 'This gene encodes a member of the transmembrane 4 superfamily. It is characterized by four tetraspanin transmembrane segments. The function of this gene has not yet been determined. [provided by RefSeq, Mar 2014]', '5', 'protein_coding', 157700387, 157945077, 42.45, 23, 'ENSG00000048140', 'Q96FV3'),
(3525, 'TSPAN31', 'The protein encoded by this gene is a member of the transmembrane 4 superfamily, also known as the tetraspanin family. Most of these members are cell-surface proteins that are characterized by the presence of four hydrophobic domains. The proteins mediate signal transduction events that play a role in the regulation of cell development, activation, growth and motility. This encoded protein is thought to be involved in growth-related cellular processes. This gene is associated with tumorigenesis and osteosarcoma. [provided by RefSeq, Jul 2008]', '12', 'protein_coding', 65996545, 66002176, 48.54, 11, 'ENSG00000135452', 'Q12999'),
(3526, 'TSPAN5', 'The protein encoded by this gene is a member of the transmembrane 4 superfamily, also known as the tetraspanin family. Most of these members are cell-surface proteins that are characterized by the presence of four hydrophobic domains. The proteins mediate signal transduction events that play a role in the regulation of cell development, activation, growth and motility. [provided by RefSeq, Jul 2008]', '4', 'miRNA', 65018505, 65018570, 69.70, 1, 'ENSG00000168785', 'P62079'),
(3527, 'TSPY10', 'Predicted to enable chromatin binding activity and histone binding activity. Predicted to be involved in sex differentiation. Predicted to be located in cytoplasm. Predicted to be active in chromatin and nucleus. [provided by Alliance of Genome Resources, May 2025]', 'Y', 'protein_coding', 8669, 20226, 51.45, 16, 'ENSG00000236424', 'P0CW01'),
(3528, 'TSPY3', 'Predicted to enable chromatin binding activity and histone binding activity. Predicted to be involved in sex differentiation. Predicted to be located in cytoplasm. Predicted to be active in chromatin and nucleus. [provided by Alliance of Genome Resources, May 2025]', 'Y', 'protein_coding', 32311018, 32393012, 46.41, 3, 'ENSG00000228927', 'P0CV98'),
(3529, 'TSPYL1', 'The protein encoded by this gene is found in the nucleolus and is similar to that of a family of genes on the Y-chromosome. This gene is intronless. Defects in this gene are a cause of sudden infant death with dysgenesis of the testes syndrome (SIDDT). [provided by RefSeq, Dec 2009]', '6', 'unitary_pseudogene', 107826, 108766, 56.32, 1, 'ENSG00000189241', 'Q9H0U9'),
(3530, 'TSPYL4', 'Predicted to enable chromatin binding activity and histone binding activity. Predicted to be involved in nucleosome assembly. Predicted to be active in chromatin and nucleus. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 29192774, 29921586, 43.51, 7, 'ENSG00000187189', 'Q9UJ04'),
(3531, 'TST', 'This is one of two neighboring genes encoding similar proteins that each contain two rhodanese domains. The encoded protein is localized to the mitochondria and catalyzes the conversion of thiosulfate and cyanide to thiocyanate and sulfite. In addition, the protein interacts with 5S ribosomal RNA and facilitates its import into the mitochondria. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2012]', '22', 'protein_coding', 26045384, 26045869, 57.61, 1, 'ENSG00000128311', 'Q16762'),
(3532, 'TTC39C', 'Predicted to be involved in cilium assembly and otolith morphogenesis. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 173384701, 173406380, 37.62, 2, 'ENSG00000168234', 'Q8N584'),
(3533, 'TTC5', 'Enables ribosome binding activity. Involved in positive regulation of mRNA catabolic process. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 119153458, 119369435, 40.76, 3, 'ENSG00000136319', 'Q8N0Z6'),
(3534, 'TTC5', 'Enables ribosome binding activity. Involved in positive regulation of mRNA catabolic process. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 119153458, 119369435, 40.76, 3, 'ENSG00000291781', 'Q8N0Z6'),
(3535, 'TTC7B', 'Involved in phosphatidylinositol phosphate biosynthetic process and protein localization to plasma membrane. Located in cytosol and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 15882387, 15903478, 36.48, 6, 'ENSG00000165914', 'Q86TV6'),
(3536, 'TTLL2', 'Predicted to enable tubulin binding activity and tubulin-glutamic acid ligase activity. Predicted to be involved in microtubule cytoskeleton organization. Predicted to be active in ciliary basal body. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 18137228, 18188387, 42.63, 13, 'ENSG00000120440', 'Q9BWV7'),
(3537, 'TTLL8', 'Predicted to enable protein-glycine ligase activity, initiating. Predicted to be involved in several processes, including axoneme assembly; flagellated sperm motility; and protein polyglycylation. Predicted to act upstream of or within cilium assembly. Predicted to be located in cilium. Predicted to be active in axoneme; microtubule cytoskeleton; and sperm flagellum. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 67701886, 67734451, 41.06, 4, 'ENSG00000138892', 'A6PVC2'),
(3538, 'TTLL9', 'Predicted to enable tubulin binding activity and tubulin-glutamic acid ligase activity. Predicted to be involved in flagellated sperm motility and microtubule cytoskeleton organization. Predicted to act upstream of or within protein polyglutamylation. Predicted to be located in cytoplasm; microtubule; and motile cilium. Predicted to be active in ciliary basal body. [provided by Alliance of Genome Resources, May 2025]', '20', 'miRNA', 97995383, 97995448, 43.94, 1, 'ENSG00000131044', 'Q3SXZ7'),
(3539, 'TTR', 'This gene encodes one of the three prealbumins, which include alpha-1-antitrypsin, transthyretin and orosomucoid. The encoded protein, transthyretin, is a homo-tetrameric carrier protein, which transports thyroid hormones in the plasma and cerebrospinal fluid. It is also involved in the transport of retinol (vitamin A) in the plasma by associating with retinol-binding protein. The protein may also be involved in other intracellular processes including proteolysis, nerve regeneration, autophagy and glucose homeostasis. Mutations in this gene are associated with amyloid deposition, predominantly affecting peripheral nerves or the heart, while a small percentage of the gene mutations are non-amyloidogenic. The mutations are implicated in the etiology of several diseases, including amyloidotic polyneuropathy, euthyroid hyperthyroxinaemia, amyloidotic vitreous opacities, cardiomyopathy, oculoleptomeningeal amyloidosis, meningocerebrovascular amyloidosis and carpal tunnel syndrome. [provided by RefSeq, Aug 2017]', '18', 'protein_coding', 38861422, 38872249, 58.27, 3, 'ENSG00000118271', 'P02766'),
(3540, 'TTTY13', 'Predicted to be located in membrane. [provided by Alliance of Genome Resources, May 2025]', 'Y', 'protein_coding', 442285, 531866, 40.41, 6, 'ENSG00000184991', 'Q9BZ97'),
(3541, 'TTTY17B', 'There are three copies of this gene on chromosome Y. This record represents the middle copy, which is within the P1 palindrome. [provided by RefSeq, Jul 2008]', 'Y', 'lncRNA', 19567313, 19606274, 36.42, 48, 'ENSG00000227439', ''),
(3542, 'TTTY4B', 'There are three copies of this gene on chromosome Y. This record represents the middle copy, which is within the P1 palindrome. [provided by RefSeq, Jul 2008]', 'Y', 'protein_coding', 60573, 64567, 55.27, 1, 'ENSG00000235412', ''),
(3543, 'TTTY6B', 'There are two copies of this locus in the P3 palindrome on chromosome Y. This record represents the more centromeric copy. [provided by RefSeq, Jul 2008]', 'Y', 'protein_coding', 30281309, 30282950, 42.51, 1, 'ENSG00000131548', ''),
(3544, 'TUBB1', 'This gene encodes a beta tubulin protein. This protein forms a dimer with alpha tubulin and acts as a structural component of microtubules. Mutations in this gene cause cortical dysplasia, complex, with other brain malformations 6. Alternative splicing results in multiple splice variants. There are multiple pseudogenes for this gene on chromosomes 1, 6, 7, 8, 9, and 13. [provided by RefSeq, Jun 2014]', '6', 'protein_coding', 35919734, 36385317, 35.12, 7, 'ENSG00000101162', 'Q9H4B7'),
(3545, 'TUBB4B', 'Enables double-stranded RNA binding activity. Predicted to be involved in microtubule cytoskeleton organization and mitotic cell cycle. Located in axonemal microtubule; intercellular bridge; and mitotic spindle. Implicated in Leber congenital amaurosis with early-onset deafness. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 21663968, 21761935, 44.79, 9, 'ENSG00000188229', 'P68371'),
(3546, 'TUBB8', 'The protein encoded by this gene represents the primary beta-tubulin subunit of oocytes and the early embryo. Defects in this gene, which is primate-specific, are a cause of oocyte maturation defect 2 and infertility. [provided by RefSeq, Mar 2016]', '10', 'protein_coding', 660362, 664146, 50.54, 7, 'ENSG00000261456', 'Q3ZCM7'),
(3547, 'TUBE1', 'This gene encodes a member of the tubulin superfamily. This protein localizes to the centriolar sub-distal appendages that are associated with the older of the two centrioles after centrosome duplication. This protein plays a central role in organization of the microtubules during centriole duplication. A pseudogene of this gene is found on chromosome 5.[provided by RefSeq, Jan 2009]', '6', 'miRNA', 30289794, 30289890, 50.52, 1, 'ENSG00000074935', 'Q9UJT0'),
(3548, 'TUBGCP3', 'Enables gamma-tubulin binding activity. Predicted to be involved in meiotic cell cycle; microtubule cytoskeleton organization; and mitotic cell cycle. Located in cytoplasm and microtubule cytoskeleton. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 26454910, 26657401, 42.61, 11, 'ENSG00000126216', 'Q96CW5'),
(3549, 'TUBGCP3', 'Enables gamma-tubulin binding activity. Predicted to be involved in meiotic cell cycle; microtubule cytoskeleton organization; and mitotic cell cycle. Located in cytoplasm and microtubule cytoskeleton. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 26454910, 26657401, 42.61, 11, 'ENSG00000282003', 'Q96CW5'),
(3550, 'TULP4', 'Predicted to be involved in protein ubiquitination. Located in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 57089982, 57170306, 45.00, 7, 'ENSG00000130338', 'Q9NRJ4'),
(3551, 'TUSC2', 'Predicted to be involved in inflammatory response and regulation of mitochondrial membrane potential. Predicted to act upstream of or within several processes, including natural killer cell differentiation; neutrophil-mediated killing of gram-negative bacterium; and regulation of cytokine production. Predicted to be active in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 85092284, 85243911, 37.26, 6, 'ENSG00000114383', 'O75896'),
(3552, 'TUT4', 'Enables RNA uridylyltransferase activity. Involved in RNA metabolic process; stem cell population maintenance; and transposable element silencing by mRNA destabilization. Located in cytoplasmic ribonucleoprotein granule; cytosol; and nucleolus. Implicated in liver benign neoplasm. Biomarker of breast cancer. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 146464, 155977, 63.04, 9, 'ENSG00000134744', 'Q5TAX3'),
(3553, 'TWIST2', 'The protein encoded by this gene is a basic helix-loop-helix type transcription factor and shares similarity with Twist. This protein may inhibit osteoblast maturation and maintain cells in a preosteoblast phenotype during osteoblast development. This gene may be upregulated in certain cancers. Mutations in this gene cause focal facial dermal dysplasia 3, Setleis type. Two transcript variants encoding the same protein have been found. [provided by RefSeq, Apr 2014]', '2', 'TR_J_gene', 22542199, 22542261, 49.21, 1, 'ENSG00000288335', 'Q8WVJ9'),
(3554, 'TWIST2', 'The protein encoded by this gene is a basic helix-loop-helix type transcription factor and shares similarity with Twist. This protein may inhibit osteoblast maturation and maintain cells in a preosteoblast phenotype during osteoblast development. This gene may be upregulated in certain cancers. Mutations in this gene cause focal facial dermal dysplasia 3, Setleis type. Two transcript variants encoding the same protein have been found. [provided by RefSeq, Apr 2014]', '2', 'TR_J_gene', 22542199, 22542261, 49.21, 1, 'ENSG00000233608', 'Q8WVJ9'),
(3555, 'TXLNGY', 'Predicted to enable syntaxin binding activity. [provided by Alliance of Genome Resources, May 2025]', 'Y', 'IG_V_gene', 10649400, 10649835, 53.67, 1, 'ENSG00000131002', 'Q9BZA5'),
(3556, 'TXNL4A', 'The protein encoded by this gene is a member of the U5 small ribonucleoprotein particle (snRNP), and is involved in pre-mRNA splicing. This protein contains a thioredoxin-like fold and it is expected to interact with multiple proteins. Protein-protein interactions have been observed with the polyglutamine tract-binding protein 1 (PQBP1). Mutations in both the coding region and promoter region of this gene have been associated with Burn-McKeown syndrome, which is a rare disorder characterized by craniofacial dysmorphisms, cardiac defects, hearing loss, and bilateral choanal atresia. A pseudogene of this gene is found on chromosome 2. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2015]', '18', 'TR_J_gene', 22539073, 22539134, 45.16, 1, 'ENSG00000141759', 'P83876'),
(3557, 'TXNRD2', 'The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes a mitochondrial form important for scavenging reactive oxygen species in mitochondria. It functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3\' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternatively spliced transcript variants encoding different isoforms, including a few localized in the cytosol and some lacking the C-terminal Sec residue, have been found for this gene. [provided by RefSeq, Jun 2017]', '22', 'protein_coding', 63586989, 63604639, 38.32, 6, 'ENSG00000184470', 'Q9NNW7'),
(3558, 'TYMP', 'This gene encodes an angiogenic factor which promotes angiogenesis in vivo and stimulates the in vitro growth of a variety of endothelial cells. It has a highly restricted target cell specificity acting only on endothelial cells. Mutations in this gene have been associated with mitochondrial neurogastrointestinal encephalomyopathy. Multiple alternatively spliced transcript variants have been identified. [provided by RefSeq, Apr 2012]', '22', 'snoRNA', 100986057, 100986128, 47.22, 1, 'ENSG00000025708', 'P19971'),
(3559, 'TYR', 'The enzyme encoded by this gene catalyzes the first 2 steps, and at least 1 subsequent step, in the conversion of tyrosine to melanin. The enzyme has both tyrosine hydroxylase and dopa oxidase catalytic activities, and requires copper for function. Mutations in this gene result in oculocutaneous albinism, and nonpathologic polymorphisms result in skin pigmentation variation. The human genome contains a pseudogene similar to the 3\' half of this gene. [provided by RefSeq, Oct 2008]', '11', 'unprocessed_pseudogene', 322986, 323922, 45.89, 1, 'ENSG00000077498', 'P14679'),
(3560, 'TYSND1', 'This gene encodes a protease that removes the N-terminal peroxisomal targeting signal (PTS2) from proteins produced in the cytosol, thereby facilitating their import into the peroxisome. The encoded protein is also capable of removing the C-terminal peroxisomal targeting signal (PTS1) from proteins in the peroxisomal matrix. The full-length protein undergoes self-cleavage to produce shorter, potentially inactive, peptides. Alternative splicing results in multiple transcript variants for this gene. [provided by RefSeq, Jan 2013]', '10', 'protein_coding', 113116718, 113118560, 36.68, 4, 'ENSG00000156521', 'Q2T9J0'),
(3561, 'U2AF1', 'This gene belongs to the splicing factor SR family of genes. U2 auxiliary factor, comprising a large and a small subunit, is a non-snRNP protein required for the binding of U2 snRNP to the pre-mRNA branch site. This gene encodes the small subunit which plays a critical role in both constitutive and enhancer-dependent RNA splicing by directly mediating interactions between the large subunit and proteins bound to the enhancers. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]', '21', 'protein_coding', 31376777, 31414912, 36.91, 2, 'ENSG00000160201', 'Q01081'),
(3562, 'UBAP1', 'This gene is a member of the UBA domain family, whose members include proteins having connections to ubiquitin and the ubiquitination pathway. The ubiquitin associated domain is thought to be a non-covalent ubiquitin binding domain consisting of a compact three helix bundle. This particular protein originates from a gene locus in a refined region on chromosome 9 undergoing loss of heterozygosity in nasopharyngeal carcinoma (NPC). Taking into account its cytogenetic location, this UBA domain family member is being studies as a putative target for mutation in nasopharyngeal carcinomas. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2010]', '9', 'protein_coding', 57092583, 57148369, 54.03, 10, 'ENSG00000165006', 'Q9NZ09');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3563, 'UBE2F-SCLY', 'This locus represents naturally occurring read-through transcription between the neighboring UBE2F (ubiquitin-conjugating enzyme E2F) and SCLY (selenocysteine lyase) genes on chromosome 2. The read-through transcript is a candidate for non-sense mediated mRNA decay (NMD), and is unlikely to produce a protein product. [provided by RefSeq, Feb 2011]', '2', 'protein_coding', 41667168, 41705834, 40.65, 5, 'ENSG00000258984', 'H3BSR4'),
(3564, 'UBE2L3', 'The modification of proteins with ubiquitin is an important cellular mechanism for targeting abnormal or short-lived proteins for degradation. Ubiquitination involves at least three classes of enzymes: ubiquitin-activating enzymes (E1s), ubiquitin-conjugating enzymes (E2s) and ubiquitin-protein ligases (E3s). This gene encodes a member of the E2 ubiquitin-conjugating enzyme family. This enzyme is demonstrated to participate in the ubiquitination of p53, c-Fos, and the NF-kB precursor p105 in vitro. Several alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2009]', '22', 'snoRNA', 44985378, 44985510, 49.62, 1, 'ENSG00000185651', 'P68036'),
(3565, 'UBE2S', 'This gene encodes a member of the ubiquitin-conjugating enzyme family. The encoded protein is able to form a thiol ester linkage with ubiquitin in a ubiquitin activating enzyme-dependent manner, a characteristic property of ubiquitin carrier proteins. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 140875302, 141012347, 39.53, 3, 'ENSG00000108106', 'Q16763'),
(3566, 'UBE2V1', 'Ubiquitin-conjugating E2 enzyme variant proteins constitute a distinct subfamily within the E2 protein family. They have sequence similarity to other ubiquitin-conjugating enzymes but lack the conserved cysteine residue that is critical for the catalytic activity of E2s. The protein encoded by this gene is located in the nucleus and can cause transcriptional activation of the human FOS proto-oncogene. It is thought to be involved in the control of differentiation by altering cell cycle behavior. Alternatively spliced transcript variants encoding multiple isoforms have been described for this gene, and multiple pseudogenes of this gene have been identified. Co-transcription of this gene and the neighboring upstream gene generates a rare transcript (Kua-UEV), which encodes a fusion protein comprised of sequence sharing identity with each individual gene product. [provided by RefSeq, Apr 2012]', '20', 'scaRNA', 221065, 221340, 47.46, 1, 'ENSG00000244687', 'Q13404'),
(3567, 'UBL7', 'Enables protein-macromolecule adaptor activity. Involved in antiviral innate immune response. Is active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 102995311, 103018488, 30.37, 4, 'ENSG00000288408', 'Q96S82'),
(3568, 'UBL7', 'Enables protein-macromolecule adaptor activity. Involved in antiviral innate immune response. Is active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '15', 'protein_coding', 102995311, 103018488, 30.37, 4, 'ENSG00000138629', 'Q96S82'),
(3569, 'UBN2', 'Predicted to be involved in chromatin organization. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000157741', 'Q6ZU65'),
(3570, 'UBQLN2', 'This gene encodes an ubiquitin-like protein (ubiquilin) that shares high degree of similarity with related products in yeast, rat and frog. Ubiquilins contain a N-terminal ubiquitin-like domain and a C-terminal ubiquitin-associated domain. They physically associate with both proteasomes and ubiquitin ligases; and thus, are thought to functionally link the ubiquitination machinery to the proteasome to affect in vivo protein degradation. This ubiquilin has also been shown to bind the ATPase domain of the Hsp70-like Stch protein. [provided by RefSeq, Oct 2009]', 'X', 'protein_coding', 47983356, 48071658, 43.64, 8, 'ENSG00000188021', 'Q9UHD9'),
(3571, 'UBR7', 'This gene encodes a UBR box-containing protein that belongs to the E3 ubiquitin ligase family. The protein also contains a plant homeodomain (PHD) in the C-terminus. In mammals, the encoded protein recognizes N-degrons, the destabilizing N-terminal residues of short-lived proteins, which results in ubiquitinylation, and proteolysis via the proteasome. [provided by RefSeq, Jul 2016]', '14', 'protein_coding', 50862223, 50867978, 50.17, 3, 'ENSG00000012963', 'Q8N806'),
(3572, 'UBR7', 'This gene encodes a UBR box-containing protein that belongs to the E3 ubiquitin ligase family. The protein also contains a plant homeodomain (PHD) in the C-terminus. In mammals, the encoded protein recognizes N-degrons, the destabilizing N-terminal residues of short-lived proteins, which results in ubiquitinylation, and proteolysis via the proteasome. [provided by RefSeq, Jul 2016]', '14', 'protein_coding', 50862223, 50867978, 50.17, 3, 'ENSG00000278787', 'Q8N806'),
(3573, 'UCMA', 'This gene encodes a chondrocyte-specific, highly charged protein that is abundantly expressed in the upper immature zone of fetal and juvenile epiphyseal cartilage. The encoded protein undergoes proteolytic processing to generate a mature protein that is secreted into the extracellular matrix. The glutamic acid residues in the encoded protein undergo gamma carboxylation in a vitamin K-dependent manner. Undercarboxylation of the encoded protein is associated with osteoarthritis in humans. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jul 2015]', '10', 'protein_coding', 12132417, 12134099, 50.21, 1, 'ENSG00000165623', 'Q8WVF2'),
(3574, 'UGDH', 'The protein encoded by this gene converts UDP-glucose to UDP-glucuronate and thereby participates in the biosynthesis of glycosaminoglycans such as hyaluronan, chondroitin sulfate, and heparan sulfate. These glycosylated compounds are common components of the extracellular matrix and likely play roles in signal transduction, cell migration, and cancer growth and metastasis. The expression of this gene is up-regulated by transforming growth factor beta and down-regulated by hypoxia. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010]', '4', 'protein_coding', 57459785, 57472268, 55.27, 8, 'ENSG00000109814', 'O60701'),
(3575, 'UGT1A9', 'This gene encodes a UDP-glucuronosyltransferase, an enzyme of the glucuronidation pathway that transforms small lipophilic molecules, such as steroids, bilirubin, hormones, and drugs, into water-soluble, excretable metabolites. This gene is part of a complex locus that encodes several UDP-glucuronosyltransferases. The locus includes thirteen unique alternate first exons followed by four common exons. Four of the alternate first exons are considered pseudogenes. Each of the remaining nine 5\' exons may be spliced to the four common exons, resulting in nine proteins with different N-termini and identical C-termini. Each first exon encodes the substrate binding site, and is regulated by its own promoter. The enzyme encoded by this gene is active on phenols. [provided by RefSeq, Jul 2008]', '2', 'miRNA', 16309879, 16309966, 77.27, 1, 'ENSG00000241119', 'O60656'),
(3576, 'UGT2B11', 'Enables glucuronosyltransferase activity. Involved in estrogen metabolic process and xenobiotic metabolic process. Predicted to be located in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 32306, 48880, 53.15, 1, 'ENSG00000213759', 'P06133'),
(3577, 'UGT2B7', 'The protein encoded by this gene belongs to the UDP-glycosyltransferase (UGT) family. UGTs serve a major role in the conjugation and subsequent elimination of potentially toxic xenobiotics and endogenous compounds. This protein is localized in the microsome membrane, and has unique specificity for 3,4-catechol estrogens and estriol, suggesting that it may play an important role in regulating the level and activity of these potent estrogen metabolites. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2017]', '4', 'protein_coding', 33103137, 33350630, 40.86, 9, 'ENSG00000171234', 'P16662'),
(3578, 'UPK3B', 'UPK3B is a minor component of the apical plaques of mammalian urothelium that binds and dimerizes with uroplakin-1b (UPK1B; MIM 602380), one of the major conserved urothelium membrane proteins. The other major conserved integral membrane proteins of urothelial plaques are UPK1A (MIM 611557), UPK2 (MIM 611558), and UPK3A (MIM 611559) (Deng et al., 2002 [PubMed 12446744]).[supplied by OMIM, Mar 2008]', '7', 'protein_coding', 73552190, 73565667, 37.11, 8, 'ENSG00000243566', 'Q9BT76'),
(3579, 'UPK3B', 'UPK3B is a minor component of the apical plaques of mammalian urothelium that binds and dimerizes with uroplakin-1b (UPK1B; MIM 602380), one of the major conserved urothelium membrane proteins. The other major conserved integral membrane proteins of urothelial plaques are UPK1A (MIM 611557), UPK2 (MIM 611558), and UPK3A (MIM 611559) (Deng et al., 2002 [PubMed 12446744]).[supplied by OMIM, Mar 2008]', '7', 'protein_coding', 73552190, 73565667, 37.11, 8, 'ENSG00000276184', 'Q9BT76'),
(3580, 'UPK3BL1', 'Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 1716523, 1738599, 57.66, 14, 'ENSG00000267368', 'B0FP48'),
(3581, 'UPRT', 'This gene encodes uracil phosphoribosyltransferase, which catalyzes the conversion of uracil and 5-phosphoribosyl-1-R-diphosphate to uridine monophosphate (UMP). This reaction is an important part of nucleotide metabolism, specifically the pyrimidine salvage pathway. The enzyme localizes to the nucleus and cytoplasm. The protein is a potential target for rational design of drugs to treat parasitic infections and cancer. [provided by RefSeq, Nov 2009]', 'X', 'protein_coding', 387254, 391020, 51.92, 2, 'ENSG00000094841', 'Q96BW1'),
(3582, 'UQCC1', 'This gene encodes a transmembrane protein that is structurally similar to the mouse basic fibroblast growth factor repressed ZIC-binding protein. In mouse this protein may be involved in fibroblast growth factor regulated growth control. In humans, polymorphisms in this gene are associated with variation in human height and osteoarthritis. Alternate splicing results in multiple transcript variants. [provided by RefSeq, May 2010]', '20', 'transcribed_unprocessed_pseudogene', 158853755, 158854576, 48.42, 1, 'ENSG00000101019', 'Q9NVA1'),
(3583, 'UQCR10', 'UCRC is a subunit of mitochondrial complex III (ubiquinol-cytochrome c reductase; EC 1.10.2.2), which forms the middle segment of the respiratory chain of the inner mitochondrial membrane (Schagger et al., 1995 [PubMed 8592474]).[supplied by OMIM, Mar 2008]', '22', 'transcribed_unprocessed_pseudogene', 469727, 472094, 55.07, 1, 'ENSG00000184076', 'Q9UDW1'),
(3584, 'UQCRC2', 'The protein encoded by this gene is located in the mitochondrion, where it is part of the ubiquinol-cytochrome c reductase complex (also known as complex III). This complex constitutes a part of the mitochondrial respiratory chain. Defects in this gene are a cause of mitochondrial complex III deficiency nuclear type 5. [provided by RefSeq, Jul 2015]', '16', 'snoRNA', 100938187, 100938264, 41.03, 1, 'ENSG00000140740', 'P22695'),
(3585, 'URB1', 'Enables RNA binding activity. Predicted to be involved in maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) and maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA). Located in fibrillar center. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 5142911, 5197503, 39.25, 2, 'ENSG00000142207', 'O60287'),
(3586, 'USP15', 'This gene encodes a member of the ubiquitin specific protease (USP) family of deubiquitinating enzymes. USP enzymes play critical roles in ubiquitin-dependent processes through polyubiquitin chain disassembly and hydrolysis of ubiquitin-substrate bonds. The encoded protein associates with the COP9 signalosome, and also plays a role in transforming growth factor beta signalling through deubiquitination of receptor-activated SMAD transcription factors. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 2. [provided by RefSeq, Nov 2011]', '12', 'protein_coding', 117542445, 117549546, 56.25, 4, 'ENSG00000135655', 'Q9Y4E8'),
(3587, 'USP17L30', 'Enables RNA binding activity; cysteine-type deubiquitinase activity; and hyaluronic acid binding activity. Involved in positive regulation of epithelial cell apoptotic process and protein deubiquitination involved in ubiquitin-dependent protein catabolic process. Located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '4', 'protein_coding', 40028260, 40056231, 39.66, 5, 'ENSG00000228856', 'Q0WX57'),
(3588, 'USP17L4', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 122779716, 122784074, 37.03, 2, 'ENSG00000284781', 'A6NCW7'),
(3589, 'USP17L4', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 122779716, 122784074, 37.03, 2, 'ENSG00000236125', 'A6NCW7'),
(3590, 'USP17L7', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 45534196, 45547662, 44.91, 2, 'ENSG00000284991', 'P0C7H9'),
(3591, 'USP17L7', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 45534196, 45547662, 44.91, 2, 'ENSG00000226430', 'P0C7H9'),
(3592, 'USP17L8', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 42950526, 42964498, 48.21, 19, 'ENSG00000285497', 'P0C7I0'),
(3593, 'USP17L8', 'Predicted to enable cysteine-type deubiquitinase activity. Predicted to be involved in regulation of apoptotic process and regulation of protein stability. Predicted to be located in endoplasmic reticulum. Predicted to be active in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 42950526, 42964498, 48.21, 19, 'ENSG00000237038', 'P0C7I0'),
(3594, 'USP44', 'The protein encoded by this gene is a protease that functions as a deubiquitinating enzyme. The encoded protein is thought to help regulate the spindle assembly checkpoint by preventing early anaphase onset. This protein specifically deubiquitinates CDC20, which stabilizes the anaphase promoting complex/cyclosome. [provided by RefSeq, Dec 2016]', '12', 'protein_coding', 119135832, 119187789, 36.63, 1, 'ENSG00000136014', 'Q9H0E7'),
(3595, 'UTF1', 'The protein encoded by this gene is a leucine zipper-containing transcriptional coactivator that may link the upstream activator ATF2 with the basal transcription complex. The encoded protein is closely associated with chromatin and is required for the proper differentiation of embryonic carcinoma and embryonic stem cells. Found nearly exclusively in pluripotent cells, this protein can also serve as a transcriptional repressor. [provided by RefSeq, Nov 2015]', '10', 'protein_coding', 867859, 915058, 59.36, 24, 'ENSG00000171794', 'Q5T230'),
(3596, 'UTP14A', 'This gene encodes a member of the uridine triphosphate 14 family. As an essential component of a large ribonucleoprotein complex bound to the U3 small nucleolar RNA, the encoded protein is involved in ribosome biogenesis and 18S rRNA synthesis. An autosomal retrotransposed copy of this X-linked gene exists on chromosome 13. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Oct 2009]', 'X', 'protein_coding', 70137981, 70146700, 53.28, 4, 'ENSG00000156697', 'Q9BVJ6'),
(3597, 'UTP14C', 'Predicted to be involved in several processes, including meiotic cell cycle; rRNA processing; and spermatogenesis. Located in cytosol and nucleolus. [provided by Alliance of Genome Resources, May 2025]', '13', 'TR_V_gene', 22469041, 22469698, 43.77, 1, 'ENSG00000253797', 'Q5TAP6'),
(3598, 'UTP20', 'UTP20 is a component of the U3 small nucleolar RNA (snoRNA) (SNORD3A; MIM 180710) protein complex (U3 snoRNP) and is involved in 18S rRNA processing (Wang et al., 2007 [PubMed 17498821]).[supplied by OMIM, Jun 2009]', '12', 'protein_coding', 1407569, 1435687, 58.75, 11, 'ENSG00000120800', 'O75691'),
(3599, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000137411', 'Q5ST30'),
(3600, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000234032', 'Q5ST30'),
(3601, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000236178', 'Q5ST30'),
(3602, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000230985', 'Q5ST30'),
(3603, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000206476', 'Q5ST30'),
(3604, 'VARS2', 'Aminoacyl-tRNA synthetases catalyze the aminoacylation of tRNA by their cognate amino acid. Because of their central role in linking amino acids with nucleotide triplets contained in tRNAs, aminoacyl-tRNA synthetases are thought to be among the first proteins that appeared in evolution. The protein encoded by this gene belongs to class-I aminoacyl-tRNA synthetase family and is located in the class III region of the major histocompatibility complex. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 75040077, 75209839, 34.22, 10, 'ENSG00000223494', 'Q5ST30'),
(3605, 'VAV1', 'This gene is a member of the VAV gene family. The VAV proteins are guanine nucleotide exchange factors (GEFs) for Rho family GTPases that activate pathways leading to actin cytoskeletal rearrangements and transcriptional alterations. The encoded protein is important in hematopoiesis, playing a role in T-cell and B-cell development and activation. The encoded protein has been identified as the specific binding partner of Nef proteins from HIV-1. Coexpression and binding of these partners initiates profound morphological changes, cytoskeletal rearrangements and the JNK/SAPK signaling cascade, leading to increased levels of viral transcription and replication. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Apr 2012]', '19', 'protein_coding', 3764107, 3841375, 48.85, 2, 'ENSG00000141968', 'P15498'),
(3606, 'VAV3', 'This gene is a member of the VAV gene family. The VAV proteins are guanine nucleotide exchange factors (GEFs) for Rho family GTPases that activate pathways leading to actin cytoskeletal rearrangements and transcriptional alterations. This gene product acts as a GEF preferentially for RhoG, RhoA, and to a lesser extent, RAC1, and it associates maximally with the nucleotide-free states of these GTPases. Alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 220660, 237454, 47.09, 2, 'ENSG00000134215', 'Q9UKW4'),
(3607, 'VCX2', 'This gene belongs to the VCX/Y gene family, which has multiple members on both X and Y chromosomes that are expressed exclusively in male germ cells. The VCX gene cluster is polymorphic in terms of copy number; different individuals may have a different number of VCX genes. This gene contains two copies of a 30 nt tandem repeat. Deletion of a nearby member of this family was implicated in cognitive disability. [provided by RefSeq, Feb 2015]', 'X', 'protein_coding', 17800049, 17861337, 45.61, 6, 'ENSG00000177504', 'Q9H322'),
(3608, 'VCX3A', 'This gene belongs to the VCX/Y gene family, which has multiple members on both X and Y chromosomes, and all are expressed exclusively in male germ cells. The X-linked members are clustered on chromosome Xp22 and Y-linked members are two identical copies of the gene within a palindromic region on Yq11. The family members share a high degree of sequence identity, with the exception that a 30-bp unit is tandemly repeated in X-linked members but occurs only once in Y-linked members. The VCX gene cluster is polymorphic in terms of copy number; different individuals may have a different number of VCX genes. VCX/Y genes encode small and highly charged proteins of unknown function. The presence of a putative bipartite nuclear localization signal suggests that VCX/Y members are nuclear proteins. This gene contains 8 repeats of the 30-bp unit. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 71878453, 71884561, 33.75, 1, 'ENSG00000169059', 'Q9NNX9'),
(3609, 'VCY1B', 'The protein encoded by this gene is a member of a family of human VCX/Y genes. This gene family has multiple members on both X and Y chromosomes, and all are expressed exclusively in male germ cells. Members of the VCX/Y family share a high degree of sequence identity, with the exception that a 30-bp unit is tandemly repeated in X-linked members but occurs only once in Y-linked members. VCX/Y genes encode small and highly charged proteins of unknown function. This gene encodes a small, positively charged protein. The presence of a putative bipartite nuclear localization signal suggests that this gene encodes a nuclear protein. The genome has two identical copies of this gene within a palindromic region; this record represents the more telomeric copy. [provided by RefSeq, Jul 2008]', 'Y', 'lncRNA', 18135449, 18137029, 34.28, 1, 'ENSG00000129862', 'O14598'),
(3610, 'VEGFD', 'The protein encoded by this gene is a member of the platelet-derived growth factor/vascular endothelial growth factor (PDGF/VEGF) family and is active in angiogenesis, lymphangiogenesis, and endothelial cell growth. This secreted protein undergoes a complex proteolytic maturation, generating multiple processed forms which bind and activate VEGFR-2 and VEGFR-3 receptors. This protein is structurally and functionally similar to vascular endothelial growth factor C. Read-through transcription has been observed between this locus and the upstream PIR (GeneID 8544) locus. [provided by RefSeq, Feb 2011]', 'X', 'protein_coding', 47365496, 47384293, 49.61, 2, 'ENSG00000165197', 'O43915'),
(3611, 'VENTXP2', 'Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This pseudogene is a member of the VENT homeobox gene family [provided by RefSeq, Mar 2013]', '13', 'protein_coding', 43528744, 43812337, 42.97, 10, 'ENSG00000215037', ''),
(3612, 'VGLL3', 'Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 2835314, 2841190, 57.70, 6, 'ENSG00000206538', 'A8MV65'),
(3613, 'VIM', 'This gene encodes a type III intermediate filament protein. Intermediate filaments, along with microtubules and actin microfilaments, make up the cytoskeleton. The encoded protein is responsible for maintaining cell shape and integrity of the cytoplasm, and stabilizing cytoskeletal interactions. This protein is involved in neuritogenesis and cholesterol transport and functions as an organizer of a number of other critical proteins involved in cell attachment, migration, and signaling. Bacterial and viral pathogens have been shown to attach to this protein on the host cell surface. Mutations in this gene are associated with congenital cataracts in human patients. [provided by RefSeq, Aug 2017]', '10', 'protein_coding', 63813456, 63827716, 52.78, 3, 'ENSG00000026025', 'P08670'),
(3614, 'VKORC1', 'This gene encodes the catalytic subunit of the vitamin K epoxide reductase complex, which is responsible for the reduction of inactive vitamin K 2,3-epoxide to active vitamin K in the endoplasmic reticulum membrane. Vitamin K is a required co-factor for carboxylation of glutamic acid residues by vitamin K-dependent gamma-carboxylase in blood-clotting enzymes. Allelic variation in this gene is associated with vitamin k-dependent clotting factors combined deficiency of 2, and increased resistance or sensitivity to warfarin, an inhibitor of vitamin K epoxide reductase. Pseudogenes of this gene are located on chromosomes 1 and X. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2015]', '16', 'lncRNA', 161127, 170657, 43.41, 3, 'ENSG00000167397', 'Q9BQB6'),
(3615, 'VLDLR', 'The low density lipoprotein receptor (LDLR) gene family consists of cell surface proteins involved in receptor-mediated endocytosis of specific ligands. This gene encodes a lipoprotein receptor that is a member of the LDLR family and plays important roles in VLDL-triglyceride metabolism and the reelin signaling pathway. Mutations in this gene cause VLDLR-associated cerebellar hypoplasia. Alternative splicing generates multiple transcript variants encoding distinct isoforms for this gene. [provided by RefSeq, Aug 2009]', '9', 'protein_coding', 69050881, 69153150, 40.67, 9, 'ENSG00000147852', 'P98155'),
(3616, 'VPS37D', 'Predicted to be involved in protein targeting to membrane; protein targeting to vacuole; and ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway. Located in extracellular exosome. Part of ESCRT I complex. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 8509678, 8514167, 62.76, 9, 'ENSG00000176428', 'Q86XT2'),
(3617, 'VPS53', 'Involved in endocytic recycling and retrograde transport, endosome to Golgi. Acts upstream of or within lysosomal transport. Located in several cellular components, including Golgi apparatus; perinuclear region of cytoplasm; and recycling endosome. Part of EARP complex and GARP complex. Implicated in pontocerebellar hypoplasia type 2E. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 8822621, 8887001, 44.23, 9, 'ENSG00000283883', 'Q5VIR6'),
(3618, 'VPS53', 'Involved in endocytic recycling and retrograde transport, endosome to Golgi. Acts upstream of or within lysosomal transport. Located in several cellular components, including Golgi apparatus; perinuclear region of cytoplasm; and recycling endosome. Part of EARP complex and GARP complex. Implicated in pontocerebellar hypoplasia type 2E. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 8822621, 8887001, 44.23, 9, 'ENSG00000141252', 'Q5VIR6'),
(3619, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000275577', 'Q6UX27'),
(3620, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000276066', 'Q6UX27'),
(3621, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000275330', 'Q6UX27'),
(3622, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000274953', 'Q6UX27'),
(3623, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000276363', 'Q6UX27'),
(3624, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000276159', 'Q6UX27'),
(3625, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000277607', 'Q6UX27'),
(3626, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000274887', 'Q6UX27'),
(3627, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000275962', 'Q6UX27'),
(3628, 'VSTM1', 'Predicted to enable cytokine activity. Predicted to be involved in immune response-regulating signaling pathway. Predicted to be located in extracellular space and membrane. Predicted to be active in plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42984553, 43012342, 49.86, 10, 'ENSG00000189068', 'Q6UX27'),
(3629, 'VSTM2A', 'Predicted to enable identical protein binding activity. Involved in several processes, including positive regulation of brown fat cell differentiation; positive regulation of lipid storage; and positive regulation of white fat cell proliferation. Predicted to be located in extracellular region. Predicted to be active in membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 62792702, 62792771, 48.57, 1, 'ENSG00000170419', 'Q8TAG5'),
(3630, 'VSTM5', 'Predicted to be involved in several processes, including filopodium assembly; positive regulation of excitatory synapse assembly; and protein homooligomerization. Predicted to be located in membrane. Predicted to be active in axon; dendrite; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 11273199, 11273629, 61.95, 1, 'ENSG00000214376', 'A8MXK1'),
(3631, 'VTI1B', 'Enables SNARE binding activity and chloride channel inhibitor activity. Involved in regulation of protein localization to plasma membrane. Located in several cellular components, including endosome membrane; lysosomal membrane; and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 24189149, 24195687, 50.99, 12, 'ENSG00000100568', 'Q9UEU0'),
(3632, 'VTN', 'The protein encoded by this gene functions in part as an adhesive glycoprotein. Differential expression of this protein can promote either cell adhesion or migration as it links cells to the extracellular matrix through a variety of ligands. These ligands include integrins, plasminogen activator inhibitor-1, and urokinase plasminogen activator receptor. This secreted protein can be present in the plasma as a monomer or dimer and forms a multimer in the extracellular matrix of several tissues. This protein also inhibits the membrane-damaging effect of the terminal cytolytic complement pathway and binds to several serpin serine protease inhibitors. This protein can also promote extracellular matrix degradation and thus plays a role in tumorigenesis. It is involved in a variety of other biological processes such as the regulation of the coagulation pathway, wound healing, and tissue remodeling. The heparin-binding domain of this protein give it anti-microbial properties. It is also a lipid binding protein that forms a principal component of high density lipoprotein. [provided by RefSeq, Aug 2020]', '17', 'protein_coding', 167325071, 167359503, 43.63, 4, 'ENSG00000109072', 'P04004'),
(3633, 'VWA8', 'Predicted to enable ATP hydrolysis activity. Located in mitochondrion and peroxisome. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 162411, 178897, 57.80, 12, 'ENSG00000102763', 'A3KMH1'),
(3634, 'VWC2', 'This gene encodes a secreted bone morphogenic protein antagonist. The encoded protein is possibly involved in neural function and development and may have a role in cell adhesion.[provided by RefSeq, Oct 2009]', '7', 'TR_J_gene', 22523600, 22523654, 32.73, 1, 'ENSG00000188730', 'Q2TAL6'),
(3635, 'WAPL', 'Enables ATP-dependent protein-DNA unloader activity. Involved in several processes, including negative regulation of DNA replication; negative regulation of sister chromatid cohesion; and protein localization to chromatin. Located in several cellular components, including intercellular bridge; mitotic spindle; and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 147782595, 147782668, 43.24, 1, 'ENSG00000062650', 'Q7Z5K2'),
(3636, 'WASF1', 'The protein encoded by this gene, a member of the Wiskott-Aldrich syndrome protein (WASP)-family, plays a critical role downstream of Rac, a Rho-family small GTPase, in regulating the actin cytoskeleton required for membrane ruffling. It has been shown to associate with an actin nucleation core Arp2/3 complex while enhancing actin polymerization in vitro. Wiskott-Aldrich syndrome is a disease of the immune system, likely due to defects in regulation of actin cytoskeleton. Multiple alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]', '6', 'miRNA', 55172262, 55172353, 46.74, 1, 'ENSG00000112290', 'Q92558'),
(3637, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000231402', ''),
(3638, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000225954', ''),
(3639, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000231309', ''),
(3640, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000229103', ''),
(3641, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000234980', ''),
(3642, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000231433', ''),
(3643, 'WASF5P', 'This gene is a pseudogene belonging to the family of genes encoding Wiskott-Aldrich syndrome (WAS) proteins, which are involved in the transmission of signals to the actin cytoskeleton. Wiskott-Aldrich syndrome is a disease of the immune system. This pseudogene, which is apparently not transcribed, resembles the gene encoding the WAS protein family member 3, which is located on chromosome 13. [provided by RefSeq, Jul 2008]', '6', 'protein_coding', 11444375, 11466177, 41.96, 12, 'ENSG00000225403', ''),
(3644, 'WASH6P', 'Predicted to enable alpha-tubulin binding activity and gamma-tubulin binding activity. Predicted to be involved in Arp2/3 complex-mediated actin nucleation; endosomal transport; and exocytosis. Predicted to be located in early endosome membrane. Predicted to be part of WASH complex. Predicted to be active in early endosome and recycling endosome. [provided by Alliance of Genome Resources, May 2025]', 'X, Y', 'protein_coding', 14056227, 14056958, 61.89, 1, 'ENSG00000292372', 'Q9NQA3'),
(3645, 'WASH6P', 'Predicted to enable alpha-tubulin binding activity and gamma-tubulin binding activity. Predicted to be involved in Arp2/3 complex-mediated actin nucleation; endosomal transport; and exocytosis. Predicted to be located in early endosome membrane. Predicted to be part of WASH complex. Predicted to be active in early endosome and recycling endosome. [provided by Alliance of Genome Resources, May 2025]', 'X, Y', 'protein_coding', 14056227, 14056958, 61.89, 1, 'ENSG00000182484', 'Q9NQA3'),
(3646, 'WASHC4', 'This gene encodes a component of the WASH complex, which functions in the intracellular transport of endosomes. Mutations in this gene have been detected in individuals with autosomal recessive cognitive disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2014]', '12', 'protein_coding', 57623409, 57633239, 56.64, 16, 'ENSG00000136051', 'Q2M389'),
(3647, 'WASL', 'This gene encodes a member of the Wiskott-Aldrich syndrome (WAS) protein family. Wiskott-Aldrich syndrome proteins share similar domain structure, and associate with a variety of signaling molecules to alter the actin cytoskeleton. The encoded protein is highly expressed in neural tissues, and interacts with several proteins involved in cytoskeletal organization, including cell division control protein 42 (CDC42) and the actin-related protein-2/3 (ARP2/3) complex. The encoded protein may be involved in the formation of long actin microspikes, and in neurite extension. [provided by RefSeq, Jul 2013]', '7', 'protein_coding', 86374736, 86423109, 39.88, 10, 'ENSG00000106299', 'O00401'),
(3648, 'WBP1', 'This gene encodes a component of the oligosaccharyltransferase complex which catalyzes the transfer of high-mannose oligosaccharides to asparagine residues on nascent polypeptides in the lumen of the rough endoplasmic reticulum. The protein complex co-purifies with ribosomes. The product of this gene is also implicated in the processing of advanced glycation endproducts (AGEs), which form from non-enzymatic reactions between sugars and proteins or lipids and are associated with aging and hyperglycemia. [provided by RefSeq, Jul 2008]', '1', 'protein_coding', 44062727, 44065567, 58.43, 1, 'ENSG00000239779', 'Q96G27'),
(3649, 'WDR11', 'This gene encodes a protein that binds to the insulin receptor substrate 1 protein and regulates glucose transporter translocation in skeletal muscle cells. The encoded protein may also regulate growth and survival of pancreatic beta cells. Elevated copy number of this gene may be associated with melanoma severity and the encoded protein may promote melanoma metastasis in human patients. [provided by RefSeq, Oct 2016]', '6', 'protein_coding', 67269358, 67319953, 36.87, 8, 'ENSG00000120008', 'Q9BZH6'),
(3650, 'WDR17', 'This gene encodes a WD repeat-containing protein. It is abundantly expressed in retina and testis, and is thought to be a candidate gene for retinal disease. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Nov 2009]', '4', 'protein_coding', 290261, 320516, 49.08, 3, 'ENSG00000150627', 'Q8IZU2'),
(3651, 'WDR18', 'This gene encodes a member of the WD repeat protein family. WD repeats are minimally conserved regions of approximately 40 amino acids typically bracketed by gly-his and trp-asp (GH-WD), which may facilitate formation of heterotrimeric or multiprotein complexes. Members of this family are involved in a variety of cellular processes, including cell cycle progression, signal transduction, apoptosis, and gene regulation. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 119571802, 119597029, 38.16, 13, 'ENSG00000065268', 'Q9BV38'),
(3652, 'WDR38', 'Predicted to act upstream of or within hematopoietic progenitor cell differentiation. Predicted to be part of transferase complex. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 121667946, 121696995, 38.94, 11, 'ENSG00000136918', 'Q5JTN6'),
(3653, 'WDR59', 'Predicted to enable signaling adaptor activity. Involved in cellular response to amino acid starvation and positive regulation of TORC1 signaling. Located in cytosol and lysosomal membrane. Part of GATOR2 complex. [provided by Alliance of Genome Resources, May 2025]', '16', 'transcribed_unprocessed_pseudogene', 1123809, 1133912, 47.03, 1, 'ENSG00000103091', 'Q6PJI9'),
(3654, 'WDR7', 'This gene encodes a member of the WD repeat protein family. WD repeats are minimally conserved regions of approximately 40 amino acids typically bracketed by gly-his and trp-asp (GH-WD) that may facilitate formation of heterotrimeric or multiprotein complexes. Members of this family are involved in a variety of cellular processes, including cell cycle progression, signal transduction, apoptosis, and gene regulation. The encoded protein forms the beta subunit of rabconnectin-3 and binds directly with Rab3A GDP/GTP exchange protein and indirectly with Rab3A GDP/GTP activating protein; these proteins are regulators of Rab3 small G protein family members involved in control of the calcium-dependant exocytosis of neurotransmitters. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]', '18', 'protein_coding', 33851100, 34224952, 34.88, 14, 'ENSG00000091157', 'Q9Y4E6'),
(3655, 'WDR73', 'The protein encoded by this gene is thought to contain multiple WD40 repeats. WD40 repeats are motifs that contain 40-60 amino acids, and usually end with Trp-Asp (WD). This protein is found in the cytoplasm during interphase, but accumulates at the spindle poles and astral microtubules during mitosis. Reduced expression of this gene results in abnormalities in the size and morphology of the nucleus. Mutations in this gene have been associated with Galloway-Mowat syndrome PMID: 25466283), which is a rare autosomal recessive disorder that affects both the central nervous system and kidneys. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2015]', '15', 'protein_coding', 21176658, 21179084, 49.86, 1, 'ENSG00000291629', 'Q6P4I2'),
(3656, 'WDR73', 'The protein encoded by this gene is thought to contain multiple WD40 repeats. WD40 repeats are motifs that contain 40-60 amino acids, and usually end with Trp-Asp (WD). This protein is found in the cytoplasm during interphase, but accumulates at the spindle poles and astral microtubules during mitosis. Reduced expression of this gene results in abnormalities in the size and morphology of the nucleus. Mutations in this gene have been associated with Galloway-Mowat syndrome PMID: 25466283), which is a rare autosomal recessive disorder that affects both the central nervous system and kidneys. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2015]', '15', 'protein_coding', 21176658, 21179084, 49.86, 1, 'ENSG00000177082', 'Q6P4I2'),
(3657, 'WDR75', 'Enables RNA binding activity. Involved in positive regulation of rRNA processing; positive regulation of transcription by RNA polymerase I; and ribosomal small subunit biogenesis. Located in nucleolus. Part of small-subunit processome. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 50742050, 50801935, 36.42, 15, 'ENSG00000115368', 'Q8IWA0');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3658, 'WDR81', 'This gene encodes a multi-domain transmembrane protein which is predominantly expressed in the brain and is thought to play a role in endolysosomal trafficking. Mutations in this gene are associated with an autosomal recessive form of a syndrome exhibiting cerebellar ataxia, cognitive disability, and disequilibrium (CAMRQ2). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2017]', '17', 'processed_pseudogene', 98740742, 98741231, 44.29, 1, 'ENSG00000276021', 'Q562E7'),
(3659, 'WDR81', 'This gene encodes a multi-domain transmembrane protein which is predominantly expressed in the brain and is thought to play a role in endolysosomal trafficking. Mutations in this gene are associated with an autosomal recessive form of a syndrome exhibiting cerebellar ataxia, cognitive disability, and disequilibrium (CAMRQ2). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2017]', '17', 'processed_pseudogene', 98740742, 98741231, 44.29, 1, 'ENSG00000167716', 'Q562E7'),
(3660, 'WEE2', 'Predicted to enable protein tyrosine kinase activity. Predicted to be involved in several processes, including female pronucleus assembly; negative regulation of cyclin-dependent protein serine/threonine kinase activity; and regulation of meiotic cell cycle. Located in cytosol; nucleoplasm; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 19930917, 19936757, 36.09, 3, 'ENSG00000263042', 'P0C1S8'),
(3661, 'WEE2', 'Predicted to enable protein tyrosine kinase activity. Predicted to be involved in several processes, including female pronucleus assembly; negative regulation of cyclin-dependent protein serine/threonine kinase activity; and regulation of meiotic cell cycle. Located in cytosol; nucleoplasm; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 19930917, 19936757, 36.09, 3, 'ENSG00000214102', 'P0C1S8'),
(3662, 'WEE2-AS1', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_V_gene', 142500028, 142500534, 56.41, 1, 'ENSG00000261994', ''),
(3663, 'WEE2-AS1', 'Predicted to enable pre-mRNA 5\'-splice site binding activity. Predicted to be involved in mRNA 5\'-splice site recognition. Predicted to be part of U1 snRNP. [provided by Alliance of Genome Resources, May 2025]', '7', 'TR_V_gene', 142500028, 142500534, 56.41, 1, 'ENSG00000228775', ''),
(3664, 'WFDC10B', 'This gene encodes a member of the WAP-type four-disulfide core (WFDC) domain family. The WFDC domain, or WAP signature motif, contains eight cysteines forming four disulfide bonds at the core of the protein, and functions as a protease inhibitor. Most WFDC gene members are localized to chromosome 20q12-q13 in two clusters: centromeric and telomeric. This gene belongs to the telomeric cluster. Two alternatively spliced transcript variants have been found for this gene, and they encode distinct isoforms. [provided by RefSeq, Jul 2008]', '20', 'protein_coding', 8002615, 8020342, 55.58, 2, 'ENSG00000182931', 'Q8IUB3'),
(3665, 'WNT16', 'The WNT gene family consists of structurally related genes which encode secreted signaling proteins. These proteins have been implicated in oncogenesis and in several developmental processes, including regulation of cell fate and patterning during embryogenesis. This gene is a member of the WNT gene family. It contains two transcript variants diverging at the 5\' termini. These two variants are proposed to be the products of separate promoters and not to be splice variants from a single promoter. They are differentially expressed in normal tissues, one of which (variant 2) is expressed at significant levels only in the pancreas, whereas another one (variant 1) is expressed more ubiquitously with highest levels in adult kidney, placenta, brain, heart, and spleen. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 37244743, 37449463, 38.82, 1, 'ENSG00000002745', 'Q9UBV4'),
(3666, 'WTAP', 'The Wilms tumor suppressor gene WT1 appears to play a role in both transcriptional and posttranscriptional regulation of certain cellular genes. This gene encodes a WT1-associating protein, which is a ubiquitously expressed nuclear protein. Like WT1 protein, this protein is localized throughout the nucleoplasm as well as in speckles and partially colocalizes with splicing factors. Alternative splicing of this gene results in several transcript variants encoding three different isoforms. [provided by RefSeq, Jul 2012]', '6', 'protein_coding', 66050729, 66069308, 48.41, 16, 'ENSG00000146457', 'Q15007'),
(3667, 'WTIP', 'Predicted to enable transcription corepressor activity. Involved in several processes, including miRNA-mediated gene silencing by inhibition of translation; negative regulation of hippo signaling; and response to hypoxia. Acts upstream of or within miRNA-mediated post-transcriptional gene silencing. Located in P-body. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 75443054, 75465497, 44.64, 7, 'ENSG00000142279', 'A6NIX2'),
(3668, 'WWC2', 'This gene encodes a member of the WW-and-C2-domain-containing family of proteins. Members of this family have two N-terminal WW domains that mediate binding to target proteins harboring L/PPxY motifs, an internal C2 domain for membrane association, and C-terminal alpha protein kinase C binding sites and class III PDZ domain-interaction motifs. Proteins of this family are able to form homo- and heterodimers and to modulate hippo pathway signaling. [provided by RefSeq, Sep 2016]', '4', 'protein_coding', 103525691, 103555239, 35.03, 9, 'ENSG00000151718', 'Q6AWC2'),
(3669, 'XCL2', 'Predicted to enable CCR chemokine receptor binding activity and chemokine activity. Predicted to be involved in several processes, including antimicrobial humoral immune response mediated by antimicrobial peptide; cell chemotaxis; and chemokine-mediated signaling pathway. Predicted to be located in extracellular region. Predicted to be active in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 8077251, 8968360, 38.19, 32, 'ENSG00000143185', 'Q9UBD3'),
(3670, 'XKRYP7', 'This probable pseudogene is located in the nonrecombining portion of the Y chromosome, and is expressed specifically in testis. It is similar to the XK (X-linked Kell blood group precursor) gene, which encodes a putative membrane transport protein. This gene is present as two identical copies within a palindromic region; this record represents the more telomeric copy. [provided by RefSeq, Nov 2020]', 'Y', 'protein_coding', 17513043, 17593579, 39.97, 5, 'ENSG00000244646', ''),
(3671, 'XKRYP7', 'This probable pseudogene is located in the nonrecombining portion of the Y chromosome, and is expressed specifically in testis. It is similar to the XK (X-linked Kell blood group precursor) gene, which encodes a putative membrane transport protein. This gene is present as two identical copies within a palindromic region; this record represents the more telomeric copy. [provided by RefSeq, Nov 2020]', 'Y', 'protein_coding', 17513043, 17593579, 39.97, 5, 'ENSG00000290725', ''),
(3672, 'XPNPEP2', 'Aminopeptidase P is a hydrolase specific for N-terminal imido bonds, which are common to several collagen degradation products, neuropeptides, vasoactive peptides, and cytokines. Structurally, the enzyme is a member of the \'pita bread fold\' family and occurs in mammalian tissues in both soluble and GPI-anchored membrane-bound forms.  A membrane-bound and soluble form of this enzyme have been identified as products of two separate genes. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 95516560, 95551476, 41.64, 8, 'ENSG00000122121', 'O43895'),
(3673, 'XPO6', 'The protein encoded by this gene is a member of the importin-beta family. Members of this family are regulated by the GTPase Ran to mediate transport of cargo across the nuclear envelope. This protein has been shown to mediate nuclear export of profilin-actin complexes. A pseudogene of this gene is located on the long arm of chromosome 14. Alternative splicing results in multiple transcript variants that encode different protein isoforms. [provided by RefSeq, Aug 2012]', '16', 'protein_coding', 25377263, 25418498, 37.91, 11, 'ENSG00000169180', 'Q96QU8'),
(3674, 'YAE1', 'Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '7', 'protein_coding', 38705742, 38738299, 51.11, 13, 'ENSG00000241127', 'Q9NRH1'),
(3675, 'YDJC', 'Predicted to enable deacetylase activity and magnesium ion binding activity. Predicted to be involved in carbohydrate metabolic process. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 105655461, 105869552, 35.40, 17, 'ENSG00000161179', 'A8MPS7'),
(3676, 'YEATS2', 'Summary: The protein encoded by this gene is a scaffolding subunit of the ATAC complex, which is a complex with acetyltransferase activity on histones H3 and H4. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, May 2017]', '3', 'protein_coding', 122203681, 122208952, 58.88, 6, 'ENSG00000163872', 'Q9ULM3'),
(3677, 'YPEL3', 'Predicted to enable metal ion binding activity. Involved in positive regulation of cellular senescence. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 8204733, 8210600, 52.69, 14, 'ENSG00000090238', 'P61236'),
(3678, 'YPEL4', 'Predicted to enable metal ion binding activity. Predicted to act upstream of or within several processes, including negative regulation of erythrocyte clearance; negative regulation of erythrocyte differentiation; and protein localization to plasma membrane. Predicted to be located in nucleolus. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 36516392, 36521149, 62.30, 4, 'ENSG00000166793', 'Q96NS1'),
(3679, 'ZAP70', 'This gene encodes an enzyme belonging to the protein tyrosine kinase family, and it plays a role in T-cell development and lymphocyte activation. This enzyme, which is phosphorylated on tyrosine residues upon T-cell antigen receptor (TCR) stimulation, functions in the initial step of TCR-mediated signal transduction in combination with the Src family kinases, Lck and Fyn. This enzyme is also essential for thymocyte development. Mutations in this gene cause selective T-cell defect, a severe combined immunodeficiency disease characterized by a selective absence of CD8-positive T-cells. Two transcript variants that encode different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 77447389, 77457278, 47.37, 1, 'ENSG00000291603', 'P43403'),
(3680, 'ZAP70', 'This gene encodes an enzyme belonging to the protein tyrosine kinase family, and it plays a role in T-cell development and lymphocyte activation. This enzyme, which is phosphorylated on tyrosine residues upon T-cell antigen receptor (TCR) stimulation, functions in the initial step of TCR-mediated signal transduction in combination with the Src family kinases, Lck and Fyn. This enzyme is also essential for thymocyte development. Mutations in this gene cause selective T-cell defect, a severe combined immunodeficiency disease characterized by a selective absence of CD8-positive T-cells. Two transcript variants that encode different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]', '2', 'protein_coding', 77447389, 77457278, 47.37, 1, 'ENSG00000115085', 'P43403'),
(3681, 'ZBED1', 'This gene is located in the pseudoautosomal region 1 (PAR1) of X and Y chromosomes. It was earlier identified as a gene with similarity to Ac transposable elements, however, was found not to have transposase activity. Later studies show that this gene product is localized in the nucleus and functions as a transcription factor. It binds to DNA elements found in the promoter regions of several genes related to cell proliferation, such as histone H1, hence may have a role in regulating genes related to cell proliferation. Alternatively spliced transcript variants with different 5\' untranslated region have been found for this gene. [provided by RefSeq, Jan 2010]', 'X, Y', 'protein_coding', 50977, 156623, 43.85, 4, 'ENSG00000292345', 'O96006'),
(3682, 'ZBED1', 'This gene is located in the pseudoautosomal region 1 (PAR1) of X and Y chromosomes. It was earlier identified as a gene with similarity to Ac transposable elements, however, was found not to have transposase activity. Later studies show that this gene product is localized in the nucleus and functions as a transcription factor. It binds to DNA elements found in the promoter regions of several genes related to cell proliferation, such as histone H1, hence may have a role in regulating genes related to cell proliferation. Alternatively spliced transcript variants with different 5\' untranslated region have been found for this gene. [provided by RefSeq, Jan 2010]', 'X, Y', 'protein_coding', 50977, 156623, 43.85, 4, 'ENSG00000214717', 'O96006'),
(3683, 'ZBTB1', 'Enables K63-linked polyubiquitin modification-dependent protein binding activity; protein heterodimerization activity; and protein homodimerization activity. Involved in several processes, including negative regulation of transcription by RNA polymerase II; nucleobase-containing compound biosynthetic process; and protein homooligomerization. Located in nuclear body and nuclear membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 721533, 734214, 45.77, 12, 'ENSG00000126804', 'Q9Y2K1'),
(3684, 'ZBTB14', 'Enables DNA-binding transcription factor activity and transcription cis-regulatory region binding activity. Involved in negative regulation of DNA-templated transcription. Located in aggresome; cytosol; and nuclear lumen. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 36800684, 36819479, 51.49, 6, 'ENSG00000198081', 'O43829'),
(3685, 'ZBTB21', 'Enables several functions, including DNA-binding transcription repressor activity, RNA polymerase II-specific; POZ domain binding activity; and methyl-CpG binding activity. Involved in negative regulation of transcription by RNA polymerase II. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '21', 'protein_coding', 43411196, 43507848, 52.28, 6, 'ENSG00000173276', 'Q9ULJ3'),
(3686, 'ZBTB5', 'Enables DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in negative regulation of transcription by RNA polymerase II. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 89318500, 89359663, 42.36, 9, 'ENSG00000168795', 'O15062'),
(3687, 'ZBTB6', 'Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in negative regulation of transcription by RNA polymerase II; regulation of cytokine production; and regulation of immune system process. Located in mitochondrion and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '9', 'miRNA', 101031987, 101032064, 38.46, 1, 'ENSG00000186130', 'Q15916'),
(3688, 'ZBTB8B', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '1', 'miRNA', 98881650, 98881729, 38.75, 1, 'ENSG00000273274', 'Q8NAP8'),
(3689, 'ZC2HC1B', 'Predicted to enable zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '6', 'unprocessed_pseudogene', 54662580, 54663450, 38.58, 1, 'ENSG00000118491', 'Q5TFG8'),
(3690, 'ZC3H13', 'Enables RNA binding activity. Involved in mRNA processing. Located in nuclear speck. Part of RNA N6-methyladenosine methyltransferase complex. [provided by Alliance of Genome Resources, May 2025]', '13', 'protein_coding', 1561385, 1620061, 43.93, 15, 'ENSG00000123200', 'Q5T200'),
(3691, 'ZC3H7B', 'This gene encodes a protein that contains a tetratricopeptide repeat domain. The encoded protein also interacts with the rotavirus non-structural protein NSP3. [provided by RefSeq, Jul 2008]', '22', 'protein_coding', 2425980, 2505532, 64.61, 8, 'ENSG00000100403', 'Q9UGR2'),
(3692, 'ZCCHC10', 'Predicted to enable nucleic acid binding activity and zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '5', 'miRNA', 39409623, 39409678, 76.79, 1, 'ENSG00000155329', 'Q8TBK6'),
(3693, 'ZCCHC14', 'Predicted to enable nucleic acid binding activity; phosphatidylinositol binding activity; and zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 120851305, 120909524, 36.77, 17, 'ENSG00000140948', 'Q8WYQ9'),
(3694, 'ZDBF2', 'This gene encodes a protein containing DBF4-type zinc finger domains. This gene is imprinted and paternally expressed in lymphocytes but is more stochastically expressed in the placenta. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2015]', '2', 'protein_coding', 245154985, 245709432, 44.80, 4, 'ENSG00000283649', 'Q9HCK1'),
(3695, 'ZDBF2', 'This gene encodes a protein containing DBF4-type zinc finger domains. This gene is imprinted and paternally expressed in lymphocytes but is more stochastically expressed in the placenta. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2015]', '2', 'protein_coding', 245154985, 245709432, 44.80, 4, 'ENSG00000204186', 'Q9HCK1'),
(3696, 'ZDHHC14', 'Enables palmitoyltransferase activity. Involved in peptidyl-L-cysteine S-palmitoylation. Located in endoplasmic reticulum. [provided by Alliance of Genome Resources, May 2025]', '6', 'protein_coding', 130506238, 130668748, 40.91, 3, 'ENSG00000175048', 'Q8IZN3'),
(3697, 'ZDHHC22', 'Predicted to enable protein-cysteine S-palmitoyltransferase activity. Involved in protein localization to plasma membrane and protein palmitoylation. Located in Golgi apparatus; endoplasmic reticulum; and plasma membrane. [provided by Alliance of Genome Resources, May 2025]', '14', 'protein_coding', 45162928, 45178237, 56.86, 5, 'ENSG00000177108', 'Q8N966'),
(3698, 'ZEB1', 'This gene encodes a zinc finger transcription factor. The encoded protein likely plays a role in transcriptional repression of interleukin 2. Mutations in this gene have been associated with posterior polymorphous corneal dystrophy-3 and late-onset Fuchs endothelial corneal dystrophy. Alternatively spliced transcript variants encoding different isoforms have been described.[provided by RefSeq, Mar 2010]', '10', 'protein_coding', 143684452, 143713898, 56.38, 29, 'ENSG00000148516', 'P37275'),
(3699, 'ZFAND4', 'Predicted to enable zinc ion binding activity. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 75156388, 75304885, 39.90, 9, 'ENSG00000172671', 'Q86XD8'),
(3700, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000204644', 'Q9NU63'),
(3701, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000227858', 'Q9NU63'),
(3702, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000226858', 'Q9NU63'),
(3703, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000223852', 'Q9NU63'),
(3704, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000223858', 'Q9NU63'),
(3705, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000206510', 'Q9NU63'),
(3706, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000234669', 'Q9NU63'),
(3707, 'ZFP57', 'The protein encoded by this gene is a zinc finger protein containing a KRAB domain. Studies in mouse suggest that this protein may function as a transcriptional repressor. Mutations in this gene have been associated with transient neonatal diabetes mellitus type 1 (TNDM1).[provided by RefSeq, Sep 2009]', '6', 'protein_coding', 10081317, 10108255, 44.42, 6, 'ENSG00000232099', 'Q9NU63'),
(3708, 'ZFP62', 'Predicted to enable zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 99705036, 99735196, 41.43, 4, 'ENSG00000196670', 'Q8NB50'),
(3709, 'ZFP64', 'Predicted to enable DNA binding activity and zinc ion binding activity. Involved in mesenchymal cell differentiation. Located in cytoplasm and megasporocyte nucleus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 74526645, 74529760, 54.91, 8, 'ENSG00000020256', 'Q9NTW7'),
(3710, 'ZFP82', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 67240713, 67336061, 37.70, 13, 'ENSG00000181007', 'Q8N141'),
(3711, 'ZFP91', 'The protein encoded by this gene is a member of the zinc finger family of proteins. The gene product contains C2H2-type domains, which are the classical zinc finger domains found in numerous nucleic acid-binding proteins. This protein functions as a regulator of the non-canonical NF-kappaB pathway in lymphotoxin-beta receptor signaling. Alternative splicing results in multiple transcript variants. A read-through transcript variant composed of ZFP91 and the downstream CNTF gene sequence has been identified, but it is thought to be non-coding. Read-through transcription of ZFP91 and CNTF has also been observed in mouse. A ZFP91-related pseudogene has also been identified on chromosome 2. [provided by RefSeq, Oct 2010]', '11', 'protein_coding', 904871, 912172, 52.94, 4, 'ENSG00000186660', 'Q96JP5'),
(3712, 'ZFP91-CNTF', 'This gene represents a read-through transcript composed of ZFP91 and CNTF sequence. This transcript is thought to be non-coding because it would be subject to nonsense-mediated mRNA decay (NMD). Read-through transcription of ZFP91 and CNTF has also been observed in mouse. [provided by RefSeq, Aug 2008]', '11', 'miRNA', 3991160, 3991259, 49.00, 1, 'ENSG00000255073', 'A0A0A6YYC7'),
(3713, 'ZFR2', 'Predicted to enable double-stranded RNA binding activity and single-stranded RNA binding activity. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 45016399, 45306209, 41.49, 2, 'ENSG00000105278', 'Q9UPR6'),
(3714, 'ZFX', 'This gene on the X chromosome is structurally similar to a related gene on the Y chromosome. It encodes a member of the krueppel C2H2-type zinc-finger protein family. The full-length protein contains an acidic transcriptional activation domain (AD), a nuclear localization sequence (NLS) and a DNA binding domain (DBD) consisting of 13 C2H2-type zinc fingers. Studies in mouse embryonic and adult hematopoietic stem cells showed that this gene was required as a transcriptional regulator for self-renewal of both stem cell types, but it was dispensable for growth and differentiation of their progeny. Multiple alternatively spliced transcript variants encoding different isoforms have been identified, but the full-length nature of some variants has not been determined. [provided by RefSeq, May 2010]', 'X', 'protein_coding', 42553617, 42574382, 54.45, 10, 'ENSG00000005889', 'P17010'),
(3715, 'ZIC2', 'This gene encodes a member of the ZIC family of C2H2-type zinc finger proteins. This protein functions as a transcriptional repressor and may regulate tissue specific expression of dopamine receptor D1. Expansion of an alanine repeat in the C-terminus of the encoded protein and other mutations in this gene cause holoprosencephaly type 5. Holoprosencephaly is the most common structural anomaly of the human brain. A polyhistidine tract polymorphism in this gene may be associated with increased risk of neural tube defects. This gene is closely linked to a gene encoding zinc finger protein of the cerebellum 5, a related family member on chromosome 13. [provided by RefSeq, Jul 2016]', '13', 'protein_coding', 43529186, 43659488, 43.01, 2, 'ENSG00000043355', 'O95409'),
(3716, 'ZIK1', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 43643860, 43648881, 46.50, 3, 'ENSG00000171649', 'Q3SY52'),
(3717, 'ZKSCAN7', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 44805617, 44818779, 54.29, 6, 'ENSG00000281894', 'Q9P0L1'),
(3718, 'ZKSCAN7', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '3', 'protein_coding', 44805617, 44818779, 54.29, 6, 'ENSG00000196345', 'Q9P0L1'),
(3719, 'ZMAT5', 'Predicted to enable zinc ion binding activity. Predicted to be involved in RNA splicing. Located in nucleoplasm. Part of U12-type spliceosomal complex. [provided by Alliance of Genome Resources, May 2025]', '22', 'protein_coding', 370382, 373791, 49.68, 2, 'ENSG00000100319', 'Q9UDW3'),
(3720, 'ZNF100', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 135794685, 135802787, 42.90, 2, 'ENSG00000274746', 'Q8IYN0'),
(3721, 'ZNF100', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 135794685, 135802787, 42.90, 2, 'ENSG00000197020', 'Q8IYN0'),
(3722, 'ZNF134', 'Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in negative regulation of transcription by RNA polymerase II; regulation of cytokine production; and regulation of immune system process. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 116423112, 117415070, 41.84, 8, 'ENSG00000213762', 'P52741'),
(3723, 'ZNF143', 'Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in positive regulation of snRNA transcription by RNA polymerase II. Predicted to be located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '11', 'protein_coding', 44373665, 44397714, 44.49, 4, 'ENSG00000166478', 'P52747'),
(3724, 'ZNF157', 'This gene product is a likely zinc finger family transcription factor. It contains KRAB-A and KRAB-B domains that act as transcriptional repressors in related proteins, and multiple zinc finger DNA binding motifs and finger linking regions characteristic of the Kruppel family. This gene is part of a gene cluster on chromosome Xp11.23. [provided by RefSeq, Jul 2008]', 'X', 'protein_coding', 69644425, 69827112, 41.93, 12, 'ENSG00000147117', 'P51786'),
(3725, 'ZNF174', 'This gene encodes a protein with three Cys2-His2-type zinc fingers in the carboxy-terminus, a putative nuclear localization signal, and an amino-terminus SCAN box which forms homodimers. This protein is believed to function as a transcriptional repressor. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Dec 2016]', '16', 'protein_coding', 20229402, 20230346, 45.40, 1, 'ENSG00000103343', 'Q15697'),
(3726, 'ZNF195', 'This gene encodes a protein belonging to the Krueppel C2H2-type zinc-finger protein family. These family members are transcription factors that are implicated in a variety of cellular processes. This gene is located near the centromeric border of chromosome 11p15.5, next to an imprinted domain that is associated with maternal-specific loss of heterozygosity in Wilms\' tumors. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]', '11', 'protein_coding', 23235967, 23275108, 41.79, 15, 'ENSG00000005801', 'O14628'),
(3727, 'ZNF217', 'Enables DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in negative regulation of transcription by RNA polymerase II. Located in mitochondrion and nuclear speck. Part of histone deacetylase complex. [provided by Alliance of Genome Resources, May 2025]', '20', 'processed_pseudogene', 152445477, 152447150, 57.11, 1, 'ENSG00000171940', 'O75362'),
(3728, 'ZNF219', 'This gene is a member of the Kruppel-like zinc finger gene family. The encoded protein functions as a transcriptional repressor of the high mobility group nucleosome binding domain 1 protein, which is associated with transcriptionally active chromatin. [provided by RefSeq, Apr 2017]', '14', 'protein_coding', 49488453, 49492508, 42.92, 24, 'ENSG00000165804', 'Q9P2Y4'),
(3729, 'ZNF235', 'This gene product belongs to the zinc finger protein superfamily, members of which are regulatory proteins characterized by nucleic acid-binding zinc finger domains. The encoded protein is a member of the Kruppel family of zinc finger proteins, and contains Kruppel-associated box (KRAB) A and B domains and 15 tandemly arrayed C2H2-type zinc fingers. It is an ortholog of the mouse Zfp93 protein. This gene is located in a cluster of zinc finger genes on 19q13.2. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 110150494, 110202202, 50.59, 2, 'ENSG00000159917', 'Q14590'),
(3730, 'ZNF256', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in negative regulation of DNA-templated transcription. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 5103629, 5260950, 43.39, 9, 'ENSG00000152454', 'Q9Y2P7'),
(3731, 'ZNF257', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 189672, 194905, 40.58, 3, 'ENSG00000197134', 'Q9Y2Q1'),
(3732, 'ZNF268', 'Predicted to enable DNA binding activity and DNA-binding transcription factor activity. Involved in several processes, including positive regulation of non-canonical NF-kappaB signal transduction; regulation of apoptotic process; and regulation of primary metabolic process. Located in cytosol and nucleus. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 61768501, 61792802, 58.49, 8, 'ENSG00000090612', 'Q14587'),
(3733, 'ZNF274', 'This gene encodes a zinc finger protein containing five C2H2-type zinc finger domains, one or two Kruppel-associated box A (KRAB A) domains, and a leucine-rich domain. The encoded protein has been suggested to be a transcriptional repressor. It localizes predominantly to the nucleolus. Alternatively spliced transcript variants encoding different isoforms exist. These variants utilize alternative polyadenylation signals. [provided by RefSeq, Jul 2008]', '19', 'protein_coding', 139015781, 139045939, 44.82, 3, 'ENSG00000171606', 'Q96GC6'),
(3734, 'ZNF280B', 'The protein encoded by this gene is a transcription factor that upregulates expression of MDM2, which negatively regulates p53 expression. This gene is highly expressed in prostate cancer cells, which leads to a reduction in p53 levels and an increase in growth of the cancer cells. Several transcript variants have been found for this gene, but only one of them is protein-coding. [provided by RefSeq, Jan 2015]', '22', 'protein_coding', 128120693, 128128462, 59.55, 21, 'ENSG00000275004', 'Q86YH2'),
(3735, 'ZNF280B', 'The protein encoded by this gene is a transcription factor that upregulates expression of MDM2, which negatively regulates p53 expression. This gene is highly expressed in prostate cancer cells, which leads to a reduction in p53 levels and an increase in growth of the cancer cells. Several transcript variants have been found for this gene, but only one of them is protein-coding. [provided by RefSeq, Jan 2015]', '22', 'protein_coding', 128120693, 128128462, 59.55, 21, 'ENSG00000277421', 'Q86YH2'),
(3736, 'ZNF287', 'This gene encodes a member of the krueppel family of zinc finger proteins, suggesting a role as a transcription factor. Its specific function has not been determined. This gene is located near the Smith-Magenis syndrome region on chromosome 17. [provided by RefSeq, Jul 2008]', '17', 'protein_coding', 72578814, 72714319, 37.87, 5, 'ENSG00000141040', 'Q9HBT7'),
(3737, 'ZNF30', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'lncRNA', 26658954, 26703783, 40.14, 5, 'ENSG00000168661', 'P17039'),
(3738, 'ZNF32', 'Alpha-2-adrenergic receptors are members of the G protein-coupled receptor superfamily. The alpha-2-adrenergic receptors are a type of adrenergic receptors (for adrenaline or epinephrine), which inhibit adenylate cyclase. These receptors include 3 highly homologous subtypes: alpha2A, alpha2B, and alpha2C. They are involved in regulating the release of neurotransmitter molecules from sympathetic nerves and from adrenergic neurons in the central nervous system. The sympathetic nervous system regulates cardiovascular function by activating adrenergic receptors in the heart, blood vessels and kidney. Studies in mouse revealed that both the alpha2A and alpha2C receptor subtypes were required for presynaptic transmitter release from the sympathetic nervous system in the heart and from central noradrenergic neurons. The alpha-2-adrenergic receptors are also involved in catecholamine signaling by extracellular regulated protein kinase 1 and 2 (ERK1/2) pathways. A clear association between the alpha-2-adrenergic receptor and disease has not been yet established. [provided by RefSeq, Sep 2019]', '10', 'protein_coding', 4301, 5033, 62.89, 1, 'ENSG00000169740', 'P17041'),
(3739, 'ZNF326', 'Enables RNA polymerase II complex binding activity. Involved in regulation of DNA-templated transcription elongation and regulation of RNA splicing. Located in nucleoplasm. Part of DBIRD complex. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 66856426, 66932141, 38.33, 7, 'ENSG00000162664', 'Q5BKZ1'),
(3740, 'ZNF341', 'Enables DNA binding activity and DNA-binding transcription activator activity. Predicted to be involved in regulation of DNA-templated transcription. Located in nucleus. Implicated in hyper IgE recurrent infection syndrome 3. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 57614233, 57624724, 46.32, 4, 'ENSG00000131061', 'Q9BYN7'),
(3741, 'ZNF343', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 33993646, 34108308, 44.86, 10, 'ENSG00000088876', 'Q6P1L6'),
(3742, 'ZNF37A', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'miRNA', 93482760, 93482855, 36.46, 1, 'ENSG00000075407', 'P17032'),
(3743, 'ZNF396', 'Enables protein heterodimerization activity; protein homodimerization activity; and sequence-specific double-stranded DNA binding activity. Involved in negative regulation of DNA-templated transcription. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '18', 'unprocessed_pseudogene', 59332757, 59334086, 42.33, 1, 'ENSG00000186496', 'Q96N95'),
(3744, 'ZNF414', 'Predicted to enable DNA-binding transcription factor activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42085526, 42090884, 48.65, 3, 'ENSG00000133250', 'Q96IQ9'),
(3745, 'ZNF420', 'The protein encoded by this gene is a KRAB-type zinc finger protein that negatively-regulates p53-mediated apoptosis. Under stress conditions, the encoded protein is phosphorylated by ATM and dissociates from p53, which activates p53 and initiates apoptosis. [provided by RefSeq, Jul 2016]', '19', 'protein_coding', 113816738, 113966325, 50.43, 7, 'ENSG00000197050', 'Q8TAQ5'),
(3746, 'ZNF428', 'Predicted to enable zinc ion binding activity. Located in mitochondrion. [provided by Alliance of Genome Resources, May 2025]', '19', 'snoRNA', 118279190, 118279320, 45.04, 1, 'ENSG00000131116', 'Q96B54'),
(3747, 'ZNF462', 'The protein encoded by this gene belongs to C2H2-type zinc finger family of proteins. It contains multiple C2H2-type zinc fingers and may be involved in transcriptional regulation. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2016]', '9', 'protein_coding', 87671371, 87696272, 37.00, 2, 'ENSG00000148143', 'Q96JM2'),
(3748, 'ZNF469', 'This gene encodes a zinc-finger protein. Low-percent homology to certain collagens suggests that it may function as a transcription factor or extra-nuclear regulator factor for the synthesis or organization of collagen fibers. Mutations in this gene cause brittle cornea syndrome. [provided by RefSeq, Jul 2008]', '16', 'protein_coding', 732952, 736848, 36.16, 1, 'ENSG00000225614', 'Q96JG9'),
(3749, 'ZNF48', 'Enables identical protein binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 55743122, 55757264, 49.22, 2, 'ENSG00000180035', 'Q96MX3'),
(3750, 'ZNF488', 'Predicted to enable DNA binding activity and zinc ion binding activity. Predicted to be involved in oligodendrocyte development and regulation of DNA-templated transcription. Predicted to act upstream of or within negative regulation of DNA-templated transcription and positive regulation of oligodendrocyte differentiation. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '10', 'protein_coding', 77672404, 77708023, 42.35, 4, 'ENSG00000265763', 'Q96MN9'),
(3751, 'ZNF511-PRAP1', 'This locus represents naturally occurring readthrough transcription between the neighboring ZNF511 (zinc finger protein 511) and PRAP1 (proline-rich acidic protein 1) genes on chromosome 10. The putative readthrough transcript may encode a fusion protein that shares sequence identity with each individual gene product and may be involved in the regulation of gene promoters, particularly those found on transfected plasmids. [provided by RefSeq, Apr 2017]', '10', 'protein_coding', 43011143, 43016868, 54.79, 6, 'ENSG00000283496', 'H7BY64'),
(3752, 'ZNF519', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific; RNA polymerase II cis-regulatory region sequence-specific DNA binding activity; and chromatin binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '18', 'protein_coding', 141836300, 141841487, 41.54, 4, 'ENSG00000175322', 'Q8TB69'),
(3753, 'ZNF540', 'Enables mRNA regulatory element binding translation repressor activity. Involved in negative regulation of DNA-templated transcription and negative regulation of translation. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'processed_pseudogene', 18642782, 18643634, 44.20, 1, 'ENSG00000171817', 'Q8NDQ6'),
(3754, 'ZNF561', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 14919, 15682, 57.20, 1, 'ENSG00000171469', 'Q8N587'),
(3755, 'ZNF585A', 'Predicted to enable zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 58000919, 58007346, 40.14, 9, 'ENSG00000196967', 'Q6P3V2'),
(3756, 'ZNF585B', 'Predicted to enable zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 31355224, 31355316, 59.14, 1, 'ENSG00000245680', 'Q52M93'),
(3757, 'ZNF598', 'Zinc-finger proteins bind nucleic acids and play important roles in various cellular functions, including cell proliferation, differentiation, and apoptosis. This protein and Grb10-interacting GYF protein 2 have been identified as a components of the mammalian 4EHP (m4EHP) complex. The complex is thought to function as a translation repressor in embryonic development. [provided by RefSeq, Oct 2012]', '16', 'protein_coding', 3043622, 3045747, 57.71, 4, 'ENSG00000292232', 'Q86UK7'),
(3758, 'ZNF598', 'Zinc-finger proteins bind nucleic acids and play important roles in various cellular functions, including cell proliferation, differentiation, and apoptosis. This protein and Grb10-interacting GYF protein 2 have been identified as a components of the mammalian 4EHP (m4EHP) complex. The complex is thought to function as a translation repressor in embryonic development. [provided by RefSeq, Oct 2012]', '16', 'protein_coding', 3043622, 3045747, 57.71, 4, 'ENSG00000167962', 'Q86UK7'),
(3759, 'ZNF607', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in mitochondrial inner membrane. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 152637895, 152657542, 48.30, 1, 'ENSG00000198182', 'Q96SK3'),
(3760, 'ZNF611', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 1729002, 1752668, 46.29, 3, 'ENSG00000213020', 'Q8N823'),
(3761, 'ZNF618', 'Enables identical protein binding activity and transcription coregulator binding activity. Predicted to be involved in regulation of DNA-templated transcription. Located in pericentric heterochromatin. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 154477775, 154479281, 63.57, 1, 'ENSG00000157657', 'Q5T7W0'),
(3762, 'ZNF623', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000183309', 'O75123'),
(3763, 'ZNF623', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 44314, 55747, 49.32, 1, 'ENSG00000276340', 'O75123'),
(3764, 'ZNF629', 'Predicted to enable DNA binding activity and zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'miRNA', 120015179, 120015257, 35.44, 1, 'ENSG00000102870', 'Q9UEG4'),
(3765, 'ZNF630', 'This gene encodes a protein containing an N-terminal Kruppel-associated box-containing (KRAB) domain and 13 Kruppel-type C2H2 zinc finger domains. This gene resides on an area of chromosome X that has been implicated in nonsyndromic X-linked cognitive disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2013]', 'X', 'miRNA', 106600928, 106601022, 45.26, 1, 'ENSG00000221994', 'Q2M218'),
(3766, 'ZNF652', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 140401814, 140539856, 38.44, 21, 'ENSG00000198740', 'Q9Y2D9');
INSERT INTO `final_gene_info` (`id`, `Gene`, `Summary`, `Chromosome`, `Gene type`, `Gene start (bp)`, `Gene end (bp)`, `Gene % GC content`, `Transcript count`, `Ensembl_ID`, `UniProt_ID`) VALUES
(3767, 'ZNF658', 'Enables transcription cis-regulatory region binding activity. Involved in cellular response to zinc ion; negative regulation of DNA-templated transcription; and ribosome biogenesis. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'unprocessed_pseudogene', 56165500, 56165921, 41.47, 1, 'ENSG00000274349', 'Q5TYW1'),
(3768, 'ZNF675', 'Enables ubiquitin protein ligase binding activity. Involved in negative regulation of osteoclast differentiation; negative regulation of signal transduction; and regulation of DNA-templated transcription. Located in nucleus and perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 183797692, 184023526, 42.71, 5, 'ENSG00000197372', 'Q8TD23'),
(3769, 'ZNF689', 'Enables histone binding activity. Involved in regulation of transcription by RNA polymerase II. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 1607565, 1608463, 63.29, 1, 'ENSG00000156853', 'C0HLU2'),
(3770, 'ZNF705B', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'miRNA', 66787072, 66787168, 52.58, 1, 'ENSG00000285286', 'P0CI00'),
(3771, 'ZNF705B', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'miRNA', 66787072, 66787168, 52.58, 1, 'ENSG00000215356', 'P0CI00'),
(3772, 'ZNF705G', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 38838198, 38859772, 41.36, 4, 'ENSG00000292386', 'A8MUZ8'),
(3773, 'ZNF705G', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 38838198, 38859772, 41.36, 4, 'ENSG00000274195', 'A8MUZ8'),
(3774, 'ZNF705G', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 38838198, 38859772, 41.36, 4, 'ENSG00000215372', 'A8MUZ8'),
(3775, 'ZNF707', 'Predicted to enable DNA-binding transcription factor activity and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 268410, 280719, 51.44, 5, 'ENSG00000181135', 'Q96C28'),
(3776, 'ZNF707', 'Predicted to enable DNA-binding transcription factor activity and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '8', 'protein_coding', 268410, 280719, 51.44, 5, 'ENSG00000274352', 'Q96C28'),
(3777, 'ZNF714', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'miRNA', 90312742, 90312833, 35.87, 1, 'ENSG00000160352', 'Q96N38'),
(3778, 'ZNF737', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 42796502, 42858106, 45.00, 6, 'ENSG00000237440', 'O75373'),
(3779, 'ZNF740', 'Enables sequence-specific double-stranded DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be located in chromatin. [provided by Alliance of Genome Resources, May 2025]', '12', 'protein_coding', 20779973, 21293011, 39.95, 24, 'ENSG00000139651', 'Q8NDX6'),
(3780, 'ZNF764', 'Enables transcription coregulator activity. Involved in cellular response to glucocorticoid stimulus. Is active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '16', 'protein_coding', 768981, 785805, 47.13, 2, 'ENSG00000169951', 'Q96H86'),
(3781, 'ZNF773', 'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 7188685, 7218574, 42.58, 21, 'ENSG00000152439', 'Q6PK81'),
(3782, 'ZNF780B', 'Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 20998409, 21543329, 34.02, 5, 'ENSG00000281601', 'Q9Y6R6'),
(3783, 'ZNF780B', 'Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 20998409, 21543329, 34.02, 5, 'ENSG00000128000', 'Q9Y6R6'),
(3784, 'ZNF782', 'Predicted to enable DNA binding activity and zinc ion binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '9', 'protein_coding', 38733290, 38794143, 54.74, 19, 'ENSG00000196597', 'Q6ZMW2'),
(3785, 'ZNF804B', 'Predicted to enable zinc ion binding activity. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, May 2025]', '7', 'miRNA', 155383100, 155383198, 56.57, 1, 'ENSG00000182348', 'A4D1E1'),
(3786, 'ZNF81', 'This gene encodes a protein that likely functions as a transcription factor. The protein contains an N-terminal KRAB domain and several C2H2-type zinc finger motifs. Mutations in this gene cause an X-linked form of intellectual disability (MRX45). Microduplication of a region of chromosome X including this gene has also been associated with other forms of intellectual disability. [provided by RefSeq, Jul 2017]', 'X', 'processed_pseudogene', 65550898, 65552421, 65.16, 1, 'ENSG00000197779', 'P51508'),
(3787, 'ZNF85', 'Enables DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 155389, 241012, 45.33, 18, 'ENSG00000105750', 'Q03923'),
(3788, 'ZNF85', 'Enables DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of DNA-templated transcription. Located in nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 155389, 241012, 45.33, 18, 'ENSG00000278091', 'Q03923'),
(3789, 'ZNFX1', 'Enables RNA binding activity. Involved in defense response to bacterium and defense response to virus. Is active in cytoplasmic stress granule. Implicated in immunodeficiency 91. [provided by Alliance of Genome Resources, May 2025]', '20', 'protein_coding', 124445243, 124450035, 54.79, 1, 'ENSG00000124201', 'Q9P2E3'),
(3790, 'ZNG1C', 'Predicted to enable GTP binding activity; hydrolase activity; and metal ion binding activity. Predicted to be located in nucleus. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, May 2025]', '9', 'miRNA', 24143489, 24143565, 59.74, 1, 'ENSG00000196873', 'Q5JTY5'),
(3791, 'ZNHIT3', 'Predicted to enable nuclear thyroid hormone receptor binding activity. Predicted to be involved in box C/D snoRNP assembly; maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA); and snoRNA localization. Located in cytoplasm and nucleus. Implicated in PEHO syndrome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 169986597, 170033031, 42.91, 11, 'ENSG00000278574', 'Q15649'),
(3792, 'ZNHIT3', 'Predicted to enable nuclear thyroid hormone receptor binding activity. Predicted to be involved in box C/D snoRNP assembly; maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA); and snoRNA localization. Located in cytoplasm and nucleus. Implicated in PEHO syndrome. [provided by Alliance of Genome Resources, May 2025]', '17', 'protein_coding', 169986597, 170033031, 42.91, 11, 'ENSG00000273611', 'Q15649'),
(3793, 'ZNRF1', 'This gene encodes an E3 ubiquitin-protein ligase that plays a role in neural-cell differentiation. Overexpression of this gene causes neurite-like elongation. The encoded protein contains both a zinc finger and a RING finger motif and is localized in the endosome/lysosome compartment, indicating that it may be involved in ubiquitin-mediated protein modification, and in synaptic vessicle membranes in neurons. [provided by RefSeq, Feb 2012]', '16', 'miRNA', 112256, 112339, 70.24, 1, 'ENSG00000186187', 'Q8ND25'),
(3794, 'ZNRF4', 'Enables ubiquitin protein ligase activity. Involved in negative regulation of nucleotide-binding activity oligomerization domain containing 2 signaling pathway and proteasome-mediated ubiquitin-dependent protein catabolic process. Is active in endoplasmic reticulum membrane. [provided by Alliance of Genome Resources, May 2025]', '19', 'protein_coding', 56638666, 56640087, 51.05, 1, 'ENSG00000105428', 'Q8WWF5'),
(3795, 'ZP2', 'The zona pellucida is an extracellular matrix that surrounds the oocyte and early embryo. It is composed of three glycoproteins with various functions during fertilization and preimplantation development. The glycosylated mature peptide is one of the structural components of the zona pellucida and functions in secondary binding and penetration of acrosome-reacted spermatozoa. Female mice lacking this gene do not form a stable zona matrix and are sterile. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2014]', '16', 'protein_coding', 101222546, 101320575, 37.60, 1, 'ENSG00000103310', 'Q05996'),
(3796, 'ZP2', 'The zona pellucida is an extracellular matrix that surrounds the oocyte and early embryo. It is composed of three glycoproteins with various functions during fertilization and preimplantation development. The glycosylated mature peptide is one of the structural components of the zona pellucida and functions in secondary binding and penetration of acrosome-reacted spermatozoa. Female mice lacking this gene do not form a stable zona matrix and are sterile. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2014]', '16', 'protein_coding', 101222546, 101320575, 37.60, 1, 'ENSG00000284588', 'Q05996'),
(3797, 'ZP3', 'The zona pellucida is an extracellular matrix that surrounds the oocyte and early embryo. It is composed primarily of three or four glycoproteins with various functions during fertilization and preimplantation development. The protein encoded by this gene is a structural component of the zona pellucida and functions in primary binding and induction of the sperm acrosome reaction. The nascent protein contains a N-terminal signal peptide sequence, a conserved ZP domain, a C-terminal consensus furin cleavage site, and a transmembrane domain. It is hypothesized that furin cleavage results in release of the mature protein from the plasma membrane for subsequent incorporation into the zona pellucida matrix. However, the requirement for furin cleavage in this process remains controversial based on mouse studies. A variation in the last exon of this gene has previously served as the basis for an additional ZP3 locus; however, sequence and literature review reveals that there is only one full-length ZP3 locus in the human genome. Another locus encoding a bipartite transcript designated POMZP3 contains a duplication of the last four exons of ZP3, including the above described variation, and maps closely to this gene. [provided by RefSeq, Jul 2008]', '7', 'protein_coding', 18196784, 18204042, 55.06, 4, 'ENSG00000188372', 'P21754'),
(3798, 'ZRANB3', 'Enables ATP-dependent DNA/DNA annealing activity; DNA endonuclease activity; and K63-linked polyubiquitin modification-dependent protein binding activity. Involved in DNA repair; replication fork reversal; and response to UV. Located in nuclear replication fork and nucleoplasm. [provided by Alliance of Genome Resources, May 2025]', '2', 'protein_coding', 140175156, 140282052, 43.99, 5, 'ENSG00000121988', 'Q5FWF4'),
(3799, 'ZRSR2P1', 'Predicted to enable pre-mRNA 3\'-splice site binding activity. Predicted to be involved in mRNA splicing, via spliceosome. Predicted to be located in nucleus. Predicted to be part of U2AF complex and spliceosomal complex. [provided by Alliance of Genome Resources, May 2025]', '5', 'protein_coding', 157436850, 157443633, 42.70, 1, 'ENSG00000212643', 'Q15695'),
(3800, 'ZSWIM5', 'Predicted to enable zinc ion binding activity. Located in extracellular space. [provided by Alliance of Genome Resources, May 2025]', '1', 'protein_coding', 31588895, 31593006, 51.85, 6, 'ENSG00000162415', 'Q9P217'),
(3801, 'BRCA1', 'BRCA1 (Breast cancer 1, early onset) is a proteincoding tumor suppressor gene involved in DNA damage repair via homologous recombination, transcriptional regulation, and maintenance of genome stability. It forms part of the BRCA1associated genome surveillance complex (BASC)', '17', 'Protein-coding', 43044295, 43125483, 41.05, 27, 'ENSG00000012048', 'P38398');

-- --------------------------------------------------------

--
-- Table structure for table `search_history`
--

CREATE TABLE `search_history` (
  `id` int(10) UNSIGNED NOT NULL,
  `user_id` int(10) UNSIGNED DEFAULT NULL,
  `gene_id` int(10) UNSIGNED DEFAULT NULL,
  `search_time` timestamp NOT NULL DEFAULT current_timestamp()
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;

-- --------------------------------------------------------

--
-- Table structure for table `users`
--

CREATE TABLE `users` (
  `id` int(11) UNSIGNED NOT NULL,
  `Username` varchar(100) NOT NULL,
  `Email` varchar(150) NOT NULL,
  `Password` varchar(255) NOT NULL,
  `created_at` timestamp NOT NULL DEFAULT current_timestamp()
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;

--
-- Dumping data for table `users`
--

INSERT INTO `users` (`id`, `Username`, `Email`, `Password`, `created_at`) VALUES
(1, 'Divyanjali', 'ndivyanjalianju536@gmail.com', '$2y$10$1cLtvqXUCgvdwiX/gvDwb.J7Zz0LdOObE96E6.2LdZCTCJOe2br8y', '2025-08-08 16:54:25'),
(2, 'geetha', 'geethasiva459@gmail.com', '$2y$10$kCfH7TcdblOrwRbiNHCZjeixNjEfeuzQsZMZa7/yJqw17FOtwsC5S', '2025-09-27 09:00:02');

--
-- Indexes for dumped tables
--

--
-- Indexes for table `final_gene_info`
--
ALTER TABLE `final_gene_info`
  ADD PRIMARY KEY (`id`);

--
-- Indexes for table `search_history`
--
ALTER TABLE `search_history`
  ADD PRIMARY KEY (`id`),
  ADD KEY `user_id` (`user_id`),
  ADD KEY `gene_id` (`gene_id`);

--
-- Indexes for table `users`
--
ALTER TABLE `users`
  ADD PRIMARY KEY (`id`),
  ADD UNIQUE KEY `email` (`Email`);

--
-- AUTO_INCREMENT for dumped tables
--

--
-- AUTO_INCREMENT for table `final_gene_info`
--
ALTER TABLE `final_gene_info`
  MODIFY `id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3804;

--
-- AUTO_INCREMENT for table `search_history`
--
ALTER TABLE `search_history`
  MODIFY `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT;

--
-- AUTO_INCREMENT for table `users`
--
ALTER TABLE `users`
  MODIFY `id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;

--
-- Constraints for dumped tables
--

--
-- Constraints for table `search_history`
--
ALTER TABLE `search_history`
  ADD CONSTRAINT `search_history_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE,
  ADD CONSTRAINT `search_history_ibfk_2` FOREIGN KEY (`gene_id`) REFERENCES `final_gene_info` (`id`) ON DELETE CASCADE;
COMMIT;

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
